{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "4a30972b-9c70-42e9-8f23-4a0083410c77",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cobrakbase 0.3.1\n"
     ]
    }
   ],
   "source": [
    "import cobra\n",
    "import modelseedpy\n",
    "import cobrakbase\n",
    "from cobrakbase.kbaseapi_cache import KBaseCache"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "eaea8cd5-456e-4141-92a0-b3316893be19",
   "metadata": {},
   "outputs": [],
   "source": [
    "kbase = KBaseCache(path='/var/data/kbase/cache')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "e1292821-b3c8-476e-b68f-08da5563d62c",
   "metadata": {},
   "outputs": [],
   "source": [
    "genome = kbase.get_from_ws('GCF_000005845.2.RAST_07_14_2022', 12218)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "0c8ef6ed-8aaa-4e1c-a9bb-e7e3bf56c7e4",
   "metadata": {},
   "outputs": [],
   "source": [
    "p_features = []\n",
    "for f in genome.features:\n",
    "    if f.seq and len(f.seq) > 0:\n",
    "        p_features.append({\"id\": f.id, \"protein_translation\": f.seq})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "5b5ffd68-f974-4f4e-9831-04ffe41f0a69",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "cfea8b62-3c2f-4b1b-a185-9dcbbe665f48",
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "type object argument after ** must be a mapping, not str",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m/tmp/ipykernel_3161047/1720006032.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     11\u001b[0m }]\n\u001b[1;32m     12\u001b[0m \u001b[0mparams\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m{\u001b[0m\u001b[0;34m\"features\"\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mfeatures\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m\"stages\"\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mstages\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 13\u001b[0;31m \u001b[0mclient\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcall\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"GenomeAnnotation.run_pipeline\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mparams\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/.local/lib/python3.8/site-packages/modelseedpy/core/rpcclient.py\u001b[0m in \u001b[0;36mcall\u001b[0;34m(self, method, params, token)\u001b[0m\n\u001b[1;32m     73\u001b[0m                 \u001b[0merr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mret\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjson\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     74\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0;34m\"error\"\u001b[0m \u001b[0;32min\u001b[0m \u001b[0merr\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 75\u001b[0;31m                     \u001b[0;32mraise\u001b[0m \u001b[0mServerError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m**\u001b[0m\u001b[0merr\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"error\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     76\u001b[0m                 \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     77\u001b[0m                     \u001b[0;32mraise\u001b[0m \u001b[0mServerError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Unknown\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mret\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtext\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: type object argument after ** must be a mapping, not str"
     ]
    }
   ],
   "source": [
    "from modelseedpy.core.rpcclient import RPCClient\n",
    "client = RPCClient(\"https://tutorial.theseed.org/services/genome_annotation\")\n",
    "stages = [\n",
    "    {\"name\": \"annotate_proteins_kmer_v2\", \"kmer_v2_parameters\": {}},\n",
    "    {\"name\": \"annotate_proteins_kmer_v1\", \"kmer_v1_parameters\": {\"annotate_hypothetical_only\": 1}},\n",
    "    {\"name\": \"annotate_proteins_similarity\", \"similarity_parameters\": {\"annotate_hypothetical_only\": 1}}\n",
    "]\n",
    "features = [{\n",
    "    'id': 'b0002',\n",
    "    'protein_translation': 'MRVLKFGGTSVANAERFLRVADILESNARQGQVATVLSAPAKITNHLVAMIEKTISGQDALPNISDAERIFAELLTGLAAAQPGFPLAQLKTFVDQEFAQIKHVLHGISLLGQCPDSINAALICRGEKMSIAIMAGVLEARGHNVTVIDPVEKLLAVGHYLESTVDIAESTRRIAASRIPADHMVLMAGFTAGNEKGELVVLGRNGSDYSAAVLAACLRADCCEIWTDVDGVYTCDPRQVPDARLLKSMSYQEAMELSYFGAKVLHPRTITPIAQFQIPCLIKNTGNPQAPGTLIGASRDEDELPVKGISNLNNMAMFSVSGPGMKGMVGMAARVFAAMSRARISVVLITQSSSEYSISFCVPQSDCVRAERAMQEEFYLELKEGLLEPLAVTERLAIISVVGDGMRTLRGISAKFFAALARANINIVAIAQGSSERSISVVVNNDDATTGVRVTHQMLFNTDQVIEVFVIGVGGVGGALLEQLKRQQSWLKNKHIDLRVCGVANSKALLTNVHGLNLENWQEELAQAKEPFNLGRLIRLVKEYHLLNPVIVDCTSSQAVADQYADFLREGFHVVTPNKKANTSSMDYYHQLRYAAEKSRRKFLYDTNVGAGLPVIENLQNLLNAGDELMKFSGILSGSLSYIFGKLDEGMSFSEATTLAREMGYTEPDPRDDLSGMDVARKLLILARETGRELELADIEIEPVLPAEFNAEGDVAAFMANLSQLDDLFAARVAKARDEGKVLRYVGNIDEDGVCRVKIAEVDGNDPLFKVKNGENALAFYSHYYQPLPLVLRGYGAGNDVTAAGVFADLLRTLSWKLGV'\n",
    "}]\n",
    "params = [{\"features\": features}, {\"stages\": stages}]\n",
    "client.call(\"GenomeAnnotation.run_pipeline\", params)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "0e4f362d-9da7-4f96-96f3-80f85c22e01f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'aliases': [['gene_synonym', 'ECK0001; JW4367'],\n",
       "  ['gene', 'thrL'],\n",
       "  ['locus_tag', 'b0001'],\n",
       "  ['protein_id', 'NP_414542.1']],\n",
       " 'cdss': ['b0001_CDS_1'],\n",
       " 'db_xrefs': [['GeneID', '944742'], ['EcoGene', 'EG11277']],\n",
       " 'dna_sequence': 'ATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGA',\n",
       " 'dna_sequence_length': 66,\n",
       " 'functions': ['Thr operon leader peptide'],\n",
       " 'id': 'b0001',\n",
       " 'location': [['NC_000913.3', 190, '+', 66]],\n",
       " 'md5': '13fe4258b37e32edda386faa50ec5fdf',\n",
       " 'ontology_terms': {'GO': {'GO:0009088': [0]},\n",
       "  'KO': {'KO:K08278': [2]},\n",
       "  'SSO': {'SSO:000025637': [5, 5]}},\n",
       " 'protein_translation': 'MKRISTTITTTITITTGNGAG',\n",
       " 'protein_translation_length': 21,\n",
       " 'quality': {'hit_count': 13, 'weighted_hit_count': 22.729}}"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genome_data['features'][0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "7779b641-6188-4d15-a3d7-8f6017fbe8c7",
   "metadata": {},
   "outputs": [],
   "source": [
    "genome = kbase.get_from_ws('GCF_000005845.2.RAST_07_14_2022', 12218)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "ae6720f8-6f76-464c-a108-89c6fcc7de30",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>ID</strong></td>\n",
       "                <td>GCF_000005845.2.RAST_07_14_2022</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Scientific Name</strong></td>\n",
       "                <td>Escherichia coli str. K-12 substr. MG1655 (Bacteria)</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Taxonomy</strong></td>\n",
       "                <td>cellular organisms; Bacteria; Proteobacteria; Gammaproteobacteria; Enterobacterales; Enterobacteriaceae; Escherichia; Escherichia coli; Escherichia coli K-12</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Sequence Data</strong></td>\n",
       "                <td>Size: 4641652, Contigs: 1, GC: 0.508</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>CDS</strong></td>\n",
       "                <td>4319</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Features</strong></td>\n",
       "                <td>4319</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>MD5</strong></td>\n",
       "                <td>482a2b04485ec8c4b5f4eaba2c2002da</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<cobrakbase.core.kbasegenome.genome.KBaseGenome at 0x7f0b9e724340>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "0ac79d46-4eda-425b-b489-2d1515e9c9d7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['gene_synonym', 'ECK0001; JW4367'],\n",
       " ['gene', 'thrL'],\n",
       " ['locus_tag', 'b0001'],\n",
       " ['protein_id', 'NP_414542.1']]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genome.features[0].aliases"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "4a00fd9f-dc14-43a2-baa5-6cae0322ac85",
   "metadata": {},
   "outputs": [],
   "source": [
    "from modelseedpy.helpers import get_classifier"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "9ac228ce-9544-4b3e-9ea9-75cc31b4ceb0",
   "metadata": {},
   "outputs": [],
   "source": [
    "genome_classifier = get_classifier('knn_ACNP_RAST_filter')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "37d4cb5f-d296-4c1c-b7d2-3892bcb65d6c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'N'"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genome_classifier.classify(genome)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "8a33edae-9f11-49f9-83cf-9ac5857f15b8",
   "metadata": {},
   "outputs": [],
   "source": [
    "o = kbase.get_object('GCF_000005845.2.RAST_07_14_2022', 12218)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "11ba126a-8aee-40e3-846f-bb7cad333b33",
   "metadata": {},
   "outputs": [],
   "source": [
    "from cobrakbase.core.kbasegenome.genome import KBaseGenome\n",
    "from cobrakbase.core.kbasegenome.genome_cds_feature import KBaseGenomeCDS, KBaseGenomeFeature\n",
    "genome = from_kbase_data(o)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "1704123b-68ba-427e-8376-2ae57fab6f6b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'aliases': [['gene', 'betA'],\n",
       "  ['gene_synonym', 'ECK0309; JW0303'],\n",
       "  ['EC_number', '1.1.99.1'],\n",
       "  ['protein_id', 'NP_414845.1'],\n",
       "  ['locus_tag', 'b0311']],\n",
       " 'cdss': ['b0311_CDS_1'],\n",
       " 'db_xrefs': [['GeneID', '945716'], ['EcoGene', 'EG10109']],\n",
       " 'dna_sequence': 'TTGCAATTTGACTACATCATTATTGGTGCCGGCTCAGCCGGCAACGTTCTCGCTACCCGTCTGACTGAAGATCCGAATACCTCCGTGCTGCTGCTTGAAGCGGGCGGCCCGGACTATCGCTTTGACTTCCGCACCCAGATGCCCGCTGCCCTGGCATTCCCGCTACAGGGTAAACGCTACAACTGGGCCTATGAAACGGAACCTGAACCGTTTATGAATAACCGCCGCATGGAGTGCGGACGCGGTAAAGGTCTGGGTGGATCGTCGCTGATCAACGGCATGTGCTACATCCGTGGCAATGCGCTGGATCTCGATAACTGGGCGCAAGAACCCGGTCTGGAGAACTGGAGCTACCTCGACTGCCTGCCCTACTACCGCAAGGCCGAGACTCGCGATATGGGTGAAAACGACTATCACGGCGGTGATGGCCCGGTGAGCGTCACTACCTCCAAACCCGGCGTCAATCCGCTGTTTGAAGCGATGATTGAAGCGGGCGTGCAGGCGGGCTACCCGCGCACGGACGATCTCAACGGTTATCAGCAGGAAGGTTTTGGTCCGATGGATCGCACCGTCACGCCGCAGGGCCGTCGCGCCAGCACCGCGCGTGGCTATCTCGATCAGGCCAAATCGCGTCCTAACCTGACCATTCGTACTCACGCTATGACCGATCACATCATTTTTGACGGCAAACGCGCGGTGGGCGTCGAATGGCTGGAAGGCGACAGCACCATCCCAACCCGCGCAACGGCCAACAAAGAAGTGCTGTTATGTGCAGGCGCGATTGCCTCACCGCAGATCCTGCAACGCTCCGGCGTCGGCAACGCTGAACTGCTGGCGGAGTTTGATATTCCGCTGGTGCATGAATTACCCGGCGTCGGCGAAAATCTTCAGGATCATCTGGAGATGTATCTGCAATATGAGTGCAAAGAACCGGTTTCCCTCTACCCTGCCCTGCAGTGGTGGAACCAGCCGAAAATCGGTGCGGAGTGGCTGTTTGGCGGCACTGGCGTTGGTGCCAGCAACCACTTTGAAGCAGGTGGATTTATTCGCAGCCGTGAGGAATTTGCGTGGCCGAATATTCAGTACCATTTCCTGCCAGTAGCGATTAACTATAACGGCTCGAATGCAGTGAAAGAGCACGGTTTCCAGTGCCACGTCGGCTCAATGCGCTCGCCAAGCCGTGGGCATGTGCGGATTAAATCCCGCGACCCGCACCAGCATCCGGCGATTCTGTTTAACTACATGTCGCACGAGCAGGACTGGCAGGAGTTCCGCGACGCAATTCGCATCACCCGCGAGATCATGCATCAACCCGCGCTGGATCAGTATCGTGGCCGCGAAATCAGCCCCGGTGTCGAATGCCAGACGGATGAACAGCTCGATGAGTTCGTGCGTAACCACGCCGAAACCGCCTTCCATCCGTGCGGTACCTGCAAAATGGGTTACGACGAGATGTCCGTGGTTGACGGCGAAGGCCGCGTACACGGGTTAGAAGGCCTGCGTGTGGTGGATGCGTCGATTATGCCGCAGATTATCACCGGGAATTTGAACGCCACGACAATTATGATTGGCGAGAAAATAGCGGATATGATTCGTGGACAGGAAGCGCTGCCGAGGAGCACGGCGGGATATTTTGTGGCAAATGGGATGCCGGTGAGAGCGAAAAAATGA',\n",
       " 'dna_sequence_length': 1671,\n",
       " 'functions': ['Choline dehydrogenase (EC 1.1.99.1)'],\n",
       " 'id': 'b0311',\n",
       " 'location': [['NC_000913.3', 327247, '-', 1671]],\n",
       " 'md5': 'fb2f59b05969a7019fd6a5a1a6247a2d',\n",
       " 'ontology_terms': {'EC': {'EC:1.1.3.10': [3],\n",
       "   'EC:1.1.3.13': [3],\n",
       "   'EC:1.1.3.4': [3],\n",
       "   'EC:1.1.3.7': [3],\n",
       "   'EC:1.1.99.18': [3]},\n",
       "  'GO': {'GO:0006578': [0],\n",
       "   'GO:0006970': [0],\n",
       "   'GO:0009274': [0],\n",
       "   'GO:0019866': [0]},\n",
       "  'KO': {'KO:K00108': [2]},\n",
       "  'SSO': {'SSO:000001541': [5]}},\n",
       " 'protein_translation': 'MQFDYIIIGAGSAGNVLATRLTEDPNTSVLLLEAGGPDYRFDFRTQMPAALAFPLQGKRYNWAYETEPEPFMNNRRMECGRGKGLGGSSLINGMCYIRGNALDLDNWAQEPGLENWSYLDCLPYYRKAETRDMGENDYHGGDGPVSVTTSKPGVNPLFEAMIEAGVQAGYPRTDDLNGYQQEGFGPMDRTVTPQGRRASTARGYLDQAKSRPNLTIRTHAMTDHIIFDGKRAVGVEWLEGDSTIPTRATANKEVLLCAGAIASPQILQRSGVGNAELLAEFDIPLVHELPGVGENLQDHLEMYLQYECKEPVSLYPALQWWNQPKIGAEWLFGGTGVGASNHFEAGGFIRSREEFAWPNIQYHFLPVAINYNGSNAVKEHGFQCHVGSMRSPSRGHVRIKSRDPHQHPAILFNYMSHEQDWQEFRDAIRITREIMHQPALDQYRGREISPGVECQTDEQLDEFVRNHAETAFHPCGTCKMGYDEMSVVDGEGRVHGLEGLRVVDASIMPQIITGNLNATTIMIGEKIADMIRGQEALPRSTAGYFVANGMPVRAKK',\n",
       " 'protein_translation_length': 556,\n",
       " 'quality': {'hit_count': 505, 'weighted_hit_count': 1276.102539}}"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[x for x in o['features'] if x['id'] == 'b0311'][0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "209a331f-2995-4a16-8cb4-cbb41ad1e78f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['NC_000913.3', 327247, '-', 1671]]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f.dna_sequence"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "34ff9924-783c-452e-ba3c-0ccb905c31e5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>ID</strong></td>\n",
       "                <td>b0311</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x07fc7c8276ac0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Ontology Terms</strong></td>\n",
       "                <td>RAST; EC; GO; KO; SSO</td>\n",
       "            </tr><tr><td><strong>RAST</strong></td><td>Choline dehydrogenase (EC 1.1.99.1)</td></tr><tr><td><strong>EC</strong></td><td>EC:1.1.3.10; EC:1.1.3.13; EC:1.1.3.4; EC:1.1.3.7; EC:1.1.99.18</td></tr><tr><td><strong>GO</strong></td><td>GO:0006578; GO:0006970; GO:0009274; GO:0019866</td></tr><tr><td><strong>KO</strong></td><td>KO:K00108</td></tr><tr><td><strong>SSO</strong></td><td>SSO:000001541</td></tr><tr>\n",
       "                <td><strong>DNA Sequence Length</strong></td>\n",
       "                <td>1</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Protein Translation Length</strong></td>\n",
       "                <td>1671</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<cobrakbase.core.kbasegenome.genome_cds_feature.KBaseGenomeFeature at 0x7fc7c8276ac0>"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f = genome.features.b0311\n",
    "f"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1d870677-8367-421d-8288-cd62417f2002",
   "metadata": {},
   "outputs": [],
   "source": [
    "genome."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "a0c15994-2738-4a9b-bbd5-a16c18215578",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>ID</strong></td>\n",
       "                <td>GCF_000005845.2.RAST_07_14_2022</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Scientific Name</strong></td>\n",
       "                <td>Escherichia coli str. K-12 substr. MG1655 (Bacteria)</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Sequence Data</strong></td>\n",
       "                <td>Size: 4641652, Contigs: 1, GC: 0.508</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>CDS</strong></td>\n",
       "                <td>4319</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Features</strong></td>\n",
       "                <td>4319</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>MD5</strong></td>\n",
       "                <td>482a2b04485ec8c4b5f4eaba2c2002da</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<__main__.display at 0x7fc7c854b190>"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "bfe55d46-2a3b-4c58-b864-08df3b7a4326",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['assembly_ref', 'cdss', 'contig_ids', 'contig_lengths', 'dna_size', 'domain', 'external_source_origination_date', 'feature_counts', 'features', 'gc_content', 'genetic_code', 'genome_tiers', 'id', 'md5', 'molecule_type', 'mrnas', 'non_coding_features', 'notes', 'num_contigs', 'ontologies_present', 'ontology_events', 'original_source_file_name', 'publications', 'scientific_name', 'source', 'source_id', 'taxon_ref', 'taxonomy', 'warnings'])"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "o.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "646bea39-3438-47cb-96c4-9c9f30f2fa46",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[4641652]"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "o['contig_lengths']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "7551ff5d-3627-4e00-af42-22c3a82c2e91",
   "metadata": {},
   "outputs": [],
   "source": [
    "for f in genome.features:\n",
    "    f"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "478adef5-831e-4adc-9c70-dea05ee67f7a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>ID</strong></td>\n",
       "                <td>b4403</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x07fc7c656d460</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Ontology Terms</strong></td>\n",
       "                <td>RAST; GO; KO; SSO</td>\n",
       "            </tr><tr><td><strong>RAST</strong></td><td>Uncharacterized tRNA/rRNA methyltransferase LasT</td></tr><tr><td><strong>GO</strong></td><td>GO:0009451</td></tr><tr><td><strong>KO</strong></td><td>KO:K02533</td></tr><tr><td><strong>SSO</strong></td><td>SSO:000028712</td></tr><tr>\n",
       "                <td><strong>DNA Sequence Length</strong></td>\n",
       "                <td>1</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Protein Translation Length</strong></td>\n",
       "                <td>687</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<cobrakbase.core.kbasegenome.genome_cds_feature.KBaseGenomeFeature at 0x7fc7c656d460>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "4302c991-eddc-4843-85b0-f975b2d8f02d",
   "metadata": {},
   "outputs": [],
   "source": [
    "def from_kbase_data(kbase_data: dict, info=None, args=None):\n",
    "    genome = KBaseGenome(kbase_data['id'],\n",
    "                         kbase_data['scientific_name'], kbase_data['domain'], kbase_data['genome_tiers'],\n",
    "                         kbase_data['feature_counts'], kbase_data['genetic_code'], kbase_data['dna_size'],\n",
    "                         kbase_data['num_contigs'], kbase_data['molecule_type'], kbase_data['source'],\n",
    "                         kbase_data['md5'], kbase_data['gc_content'], info, args)\n",
    "    features = [KBaseGenomeFeature.from_kbase_data(o) for o in kbase_data['features']]\n",
    "\n",
    "    cdss = [KBaseGenomeCDS.from_kbase_data(o) for o in kbase_data['cdss']]\n",
    "    genome.add_features(features)\n",
    "    genome.add_cdss(cdss)\n",
    "\n",
    "    if 'ontology_events' in kbase_data:\n",
    "        # events = kbase_data['ontology_events']\n",
    "        for feature_data in kbase_data['features']:\n",
    "            feature = genome.features.get_by_id(feature_data['id'])\n",
    "            if 'ontology_terms' in feature_data:\n",
    "                for term in feature_data['ontology_terms']:\n",
    "                    for value in feature_data['ontology_terms'][term]:\n",
    "                        feature.add_ontology_term(term, value)\n",
    "\n",
    "    return genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "7f17d597-fee3-41ab-8d72-c583f00181d8",
   "metadata": {},
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "38f3e191-bbf8-4d00-83b6-9557736b2fbb",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('example_genome.json', 'w') as fh:\n",
    "    fh.write(json.dumps(o))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "bec3ad21-dd1d-4b79-954b-875e09415c25",
   "metadata": {},
   "outputs": [],
   "source": [
    "from modelseedpy.core.msgenome import MSFeature"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "c6e8f321-e107-4b92-97d8-91230a352345",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'aliases': [['gene_synonym', 'ECK0001'],\n",
       "  ['gene', 'thrL'],\n",
       "  ['locus_tag', 'b0001'],\n",
       "  ['protein_id', 'NP_414542.1']],\n",
       " 'cdss': ['b0001_CDS_1'],\n",
       " 'db_xrefs': [['ECOCYC', 'EG11277'],\n",
       "  ['GeneID', '944742'],\n",
       "  ['ASAP', 'ABE-0000006'],\n",
       "  ['EcoGene', 'EG11277']],\n",
       " 'dna_sequence': 'ATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGA',\n",
       " 'dna_sequence_length': 66,\n",
       " 'functions': ['thr operon leader peptide'],\n",
       " 'id': 'b0001',\n",
       " 'location': [['NC_000913.3', 190, '+', 66]],\n",
       " 'md5': 'cd572f0df722f00fc2123c68f381b98b',\n",
       " 'protein_translation': 'MKRISTTITTTITITTGNGAG',\n",
       " 'protein_translation_length': 21}"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "o['features'][0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "936525f6-cf4e-4143-af2b-ba5e2cc82ac5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>ID</strong></td>\n",
       "                <td>b0001</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x07fe0af267d90</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Ontology Terms</strong></td>\n",
       "                <td>RAST</td>\n",
       "            </tr><tr><td><strong>RAST</strong></td><td>thr operon leader peptide</td></tr><tr>\n",
       "                <td><strong>DNA Sequence Length</strong></td>\n",
       "                <td>66</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Protein Translation Length</strong></td>\n",
       "                <td>21</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<__main__.KBaseGenomeFeature at 0x7fe0af267d90>"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import re\n",
    "import hashlib\n",
    "\n",
    "class KBaseGenomeFeature(MSFeature):\n",
    "\n",
    "    def __init__(self, feature_id: str, protein_translation: str, dna_sequence: str, location,\n",
    "                 cdss: list, functions):\n",
    "        self.dna_sequence = dna_sequence\n",
    "        self.location = location\n",
    "        self.cdss = cdss\n",
    "        self.functions = set(functions)\n",
    "        MSFeature.__init__(self, feature_id, protein_translation, '; '.join(self.functions))\n",
    "\n",
    "\n",
    "    # @property\n",
    "    # def seq(self):\n",
    "    #    return self.protein_translation\n",
    "    \n",
    "    @property\n",
    "    def dna_sequence_length(self):\n",
    "        \"\"\"\n",
    "        @return: length of the above\n",
    "        \"\"\"\n",
    "        return len(self.dna_sequence)\n",
    "\n",
    "    @property\n",
    "    def protein_translation_length(self):\n",
    "        \"\"\"\n",
    "        @return: length of the above\n",
    "        \"\"\"\n",
    "        return len(self.seq)\n",
    "    \n",
    "\n",
    "    @property\n",
    "    def md5(self):\n",
    "        return hashlib.md5(self.dna_sequence.encode('utf-8')).hexdigest()\n",
    "\n",
    "    @property\n",
    "    def protein_translation(self):\n",
    "        return self.seq\n",
    "\n",
    "    def __str__(self):\n",
    "        return self.id\n",
    "\n",
    "    @staticmethod\n",
    "    def split_annotation(annotation):\n",
    "        split = set()\n",
    "        for s in annotation:\n",
    "            if s:\n",
    "                for comp in re.split(' / | @ |; ', s):\n",
    "                    split.add(comp)\n",
    "            else:\n",
    "                split.add(s)\n",
    "        return split\n",
    "\n",
    "    @staticmethod\n",
    "    def split_function(function):\n",
    "        split = set()\n",
    "        if function:\n",
    "            for comp in re.split(' / | @ |; ', function):\n",
    "                split.add(comp)\n",
    "        else:\n",
    "            split.add(split)\n",
    "        return split\n",
    "    \n",
    "    @staticmethod\n",
    "    def extract_functions(data):\n",
    "        functions = set()\n",
    "        if 'functions' in data:\n",
    "            if type(data['functions']) == str:\n",
    "                return {data['functions']}\n",
    "            functions = set(data['functions'])\n",
    "        elif 'function' in data:\n",
    "            functions = {data['function']}\n",
    "        return functions\n",
    "    \n",
    "    def _repr_html_(self):\n",
    "        \"\"\"\n",
    "        taken from cobra.core.Model :)\n",
    "        :return:\n",
    "        \"\"\"\n",
    "        ots=\"\"\n",
    "        for term in self.ontology_terms:\n",
    "            ots+=\"<tr><td><strong>{}</strong></td><td>{}</td></tr>\".format(term, '; '.join(self.ontology_terms[term]))\n",
    "        return \"\"\"\n",
    "        <table>\n",
    "            <tr>\n",
    "                <td><strong>ID</strong></td>\n",
    "                <td>{id}</td>\n",
    "            </tr><tr>\n",
    "                <td><strong>Memory address</strong></td>\n",
    "                <td>{address}</td>\n",
    "            </tr><tr>\n",
    "                <td><strong>Ontology Terms</strong></td>\n",
    "                <td>{terms}</td>\n",
    "            </tr>{ots}<tr>\n",
    "                <td><strong>DNA Sequence Length</strong></td>\n",
    "                <td>{len_dna}</td>\n",
    "            </tr><tr>\n",
    "                <td><strong>Protein Translation Length</strong></td>\n",
    "                <td>{len_protein}</td>\n",
    "            </tr>\n",
    "          </table>\"\"\".format(\n",
    "            id=self.id,\n",
    "            address='0x0%x' % id(self),\n",
    "            functions='<br>'.join(self.functions),\n",
    "            len_dna=self.dna_sequence_length,\n",
    "            len_protein=self.protein_translation_length,\n",
    "            terms='; '.join(self.ontology_terms.keys()),\n",
    "            ots=ots)\n",
    "\n",
    "    @staticmethod\n",
    "    def from_kbase_data(kbase_data):\n",
    "        functions = KBaseGenomeFeature.extract_functions(kbase_data)\n",
    "        functions_split = KBaseGenomeFeature.split_annotation(functions)\n",
    "        o = KBaseGenomeFeature(kbase_data['id'],\n",
    "                               kbase_data['protein_translation'], kbase_data['dna_sequence'],\n",
    "                               kbase_data['location'], kbase_data['cdss'], functions)\n",
    "        for f in functions_split:\n",
    "            o.add_ontology_term('RAST', f)\n",
    "        return o\n",
    "\n",
    "    def to_kbase_data(self):\n",
    "        d = {\n",
    "            'id': self.id,\n",
    "            'dna_sequence': self.dna_sequence,\n",
    "            'md5': self.md5,\n",
    "            'cdss': self.cdss,\n",
    "            'functions': list(self.functions),\n",
    "            'functions': list(self.functions),\n",
    "            'location': self.location,\n",
    "            'dna_sequence_length': self.dna_sequence_length,\n",
    "            'protein_translation': self.seq,\n",
    "            'protein_translation_length': self.protein_translation_length,\n",
    "        }\n",
    "        return d\n",
    "f = KBaseGenomeFeature.from_kbase_data(o['features'][0])\n",
    "f"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "id": "720b9dcb-75d0-4088-950d-2886dcb7ad38",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<cobrakbase.core.kbasegenome.genome.KBaseGenome at 0x7fe0ad016910>"
      ]
     },
     "execution_count": 92,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from cobrakbase.core.kbasegenome.genome import KBaseGenome\n",
    "def from_kbase_data(kbase_data: dict):\n",
    "    genome = KBaseGenome(kbase_data['id'], None, None,\n",
    "                        kbase_data['scientific_name'], kbase_data['domain'], kbase_data['genome_tiers'],\n",
    "                        kbase_data['feature_counts'], kbase_data['genetic_code'], kbase_data['dna_size'],\n",
    "                        kbase_data['num_contigs'], kbase_data['molecule_type'], kbase_data['source'],\n",
    "                        kbase_data['md5'], kbase_data['gc_content'])\n",
    "    \n",
    "    return genome\n",
    "from_kbase_data(o)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "id": "ed83c0a6-7d9b-47de-84ab-28ef32676a91",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['assembly_ref', 'cdss', 'contig_ids', 'contig_lengths', 'dna_size', 'domain', 'external_source_origination_date', 'feature_counts', 'features', 'gc_content', 'genbank_handle_ref', 'genetic_code', 'genome_tiers', 'id', 'md5', 'molecule_type', 'mrnas', 'non_coding_features', 'notes', 'num_contigs', 'original_source_file_name', 'publications', 'scientific_name', 'source', 'source_id', 'taxon_assignments', 'taxonomy', 'warnings'])"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "o.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "47d07585-57a6-42b6-a9d9-7bb49e42a04b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'aliases': [['gene', 'thrL'],\n",
       "  ['locus_tag', 'b0001'],\n",
       "  ['gene_synonym', 'ECK0001'],\n",
       "  ['protein_id', 'NP_414542.1']],\n",
       " 'db_xrefs': [['ASAP', 'ABE-0000006'],\n",
       "  ['ECOCYC', 'EG11277'],\n",
       "  ['EcoGene', 'EG11277'],\n",
       "  ['GeneID', '944742'],\n",
       "  ['UniProtKB/Swiss-Prot', 'P0AD86']],\n",
       " 'dna_sequence': 'ATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGA',\n",
       " 'dna_sequence_length': 66,\n",
       " 'functions': ['thr operon leader peptide'],\n",
       " 'id': 'b0001_CDS_1',\n",
       " 'location': [['NC_000913.3', 190, '+', 66]],\n",
       " 'md5': 'cd572f0df722f00fc2123c68f381b98b',\n",
       " 'parent_gene': 'b0001',\n",
       " 'protein_md5': '13fe4258b37e32edda386faa50ec5fdf',\n",
       " 'protein_translation': 'MKRISTTITTTITITTGNGAG',\n",
       " 'protein_translation_length': 21}"
      ]
     },
     "execution_count": 94,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "o['cdss'][0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "id": "0cb9d2e5-4731-41e4-82bd-018f19a26aab",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GO GO:0009088 threonine biosynthetic process\n",
      "KO KO:K08278 acetyltransferase EpsM [EC:2.3.1.-]\n",
      "SSO SSO:000025637 Thr operon leader peptide\n",
      "b0001 ['Thr operon leader peptide']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "GO GO:0009088 threonine biosynthetic process\n",
      "GO GO:0009090 homoserine biosynthetic process\n",
      "KO KO:K12524 bifunctional aspartokinase / homoserine dehydrogenase 1 [EC:2.7.2.4 1.1.1.3]\n",
      "b0002 ['Aspartokinase (EC 2.7.2.4)', 'Homoserine dehydrogenase (EC 1.1.1.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009088 threonine biosynthetic process\n",
      "KO KO:K00872 homoserine kinase [EC:2.7.1.39]\n",
      "SSO SSO:000003486 Homoserine kinase (EC 2.7.1.39)\n",
      "b0003 ['Homoserine kinase (EC 2.7.1.39)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009088 threonine biosynthetic process\n",
      "KO KO:K01733 threonine synthase [EC:4.2.3.1]\n",
      "SSO SSO:000008061 Threonine synthase (EC 4.2.3.1)\n",
      "b0004 ['Threonine synthase (EC 4.2.3.1)']\n",
      "\n",
      "SSO SSO:000028142 Uncharacterized protein YaaX\n",
      "b0005 ['Uncharacterized protein YaaX']\n",
      "\n",
      "KO KO:K09861 uncharacterized protein\n",
      "SSO SSO:000008666 UPF0246 protein YaaA\n",
      "b0006 ['UPF0246 protein YaaA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03310 alanine or glycine:cation symporter, AGCS family\n",
      "SSO SSO:000023145 Putative alanine/glycine transport protein\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b0007 ['Putative alanine/glycine transport protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "KO KO:K00616 transaldolase [EC:2.2.1.2]\n",
      "SSO SSO:000008135 Transaldolase (EC 2.2.1.2)\n",
      "b0008 ['Transaldolase (EC 2.2.1.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03831 molybdopterin adenylyltransferase [EC:2.7.7.75]\n",
      "KO KO:K06999 phospholipase/carboxylesterase\n",
      "SSO SSO:000004940 Molybdopterin adenylyltransferase (EC 2.7.7.75)\n",
      "b0009 ['Molybdopterin adenylyltransferase (EC 2.7.7.75)']\n",
      "\n",
      "KO KO:K07034 uncharacterized protein\n",
      "SSO SSO:000025198 Succinate-acetate/proton symporter SatP\n",
      "b0010 ['Succinate-acetate/proton symporter SatP']\n",
      "\n",
      "SSO SSO:000026563 UPF0174 protein YaaW\n",
      "b0011 ['UPF0174 protein YaaW']\n",
      "\n",
      "SSO SSO:000026678 UPF0412 protein YaaI\n",
      "b0013 ['UPF0412 protein YaaI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K04043 molecular chaperone DnaK\n",
      "SSO SSO:000001504 Chaperone protein DnaK\n",
      "b0014 ['Chaperone protein DnaK']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03686 molecular chaperone DnaJ\n",
      "SSO SSO:000001503 Chaperone protein DnaJ\n",
      "b0015 ['Chaperone protein DnaJ']\n",
      "\n",
      "SSO SSO:000026100 Transposase InsL for insertion sequence element IS186\n",
      "b0016 ['Transposase InsL for insertion sequence element IS186']\n",
      "\n",
      "KO KO:K18919 protein HokC/D\n",
      "SSO SSO:000006986 RelF inactive antibacterial toxin protein\n",
      "b0018 ['RelF inactive antibacterial toxin protein']\n",
      "\n",
      "KO KO:K18919 protein HokC/D\n",
      "SSO SSO:000006986 RelF inactive antibacterial toxin protein\n",
      "b4412 ['RelF inactive antibacterial toxin protein']\n",
      "\n",
      "GO GO:0009268 response to pH\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03313 Na+:H+ antiporter, NhaA family\n",
      "SSO SSO:000005279 Na+/H+ antiporter NhaA type\n",
      "b0019 ['Na+/H+ antiporter NhaA type']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009268 response to pH\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03717 LysR family transcriptional regulator, transcriptional activator of nhaA\n",
      "SSO SSO:000008184 Transcriptional activator NhaR\n",
      "b0020 ['Transcriptional activator NhaR']\n",
      "\n",
      "KO KO:K07480 insertion element IS1 protein InsB\n",
      "SSO SSO:000018188 IS1 protein InsB\n",
      "b0021 ['IS1 protein InsB']\n",
      "\n",
      "SSO SSO:000018187 IS1 protein InsA\n",
      "b0022 ['IS1 protein InsA']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02968 small subunit ribosomal protein S20\n",
      "SSO SSO:000007286 SSU ribosomal protein S20p\n",
      "b0023 ['SSU ribosomal protein S20p']\n",
      "\n",
      "SSO SSO:000028144 Uncharacterized protein YaaY\n",
      "b0024 ['Uncharacterized protein YaaY']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K11753 riboflavin kinase / FMN adenylyltransferase [EC:2.7.1.26 2.7.7.2]\n",
      "b0025 ['FMN adenylyltransferase (EC 2.7.7.2)', 'Riboflavin kinase (EC 2.7.1.26)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01870 isoleucyl-tRNA synthetase [EC:6.1.1.5]\n",
      "SSO SSO:000004056 Isoleucyl-tRNA synthetase (EC 6.1.1.5)\n",
      "b0026 ['Isoleucyl-tRNA synthetase (EC 6.1.1.5)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03101 signal peptidase II [EC:3.4.23.36]\n",
      "SSO SSO:000004502 Lipoprotein signal peptidase (EC 3.4.23.36)\n",
      "b0027 ['Lipoprotein signal peptidase (EC 3.4.23.36)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03774 FKBP-type peptidyl-prolyl cis-trans isomerase SlpA [EC:5.2.1.8]\n",
      "SSO SSO:000002673 FKBP-type peptidyl-prolyl cis-trans isomerase SlpA (EC 5.2.1.8)\n",
      "b0028 ['FKBP-type peptidyl-prolyl cis-trans isomerase SlpA (EC 5.2.1.8)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03527 4-hydroxy-3-methylbut-2-en-1-yl diphosphate reductase [EC:1.17.7.4]\n",
      "SSO SSO:000000364 4-hydroxy-3-methylbut-2-enyl diphosphate reductase (EC 1.17.1.2)\n",
      "b0029 ['4-hydroxy-3-methylbut-2-enyl diphosphate reductase (EC 1.17.7.4)']\n",
      "\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "KO KO:K01239 purine nucleosidase [EC:3.2.2.1]\n",
      "KO KO:K12700 non-specific riboncleoside hydrolase [EC:3.2.-.-]\n",
      "SSO SSO:000020612 Non-specific ribonucleoside hydrolase RihC\n",
      "b0030 ['Non-specific ribonucleoside hydrolase RihC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K00215 4-hydroxy-tetrahydrodipicolinate reductase [EC:1.17.1.8]\n",
      "SSO SSO:000000365 4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)\n",
      "SSO SSO:000029834 dihydrodipicolinate reductase\n",
      "b0031 ['4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)']\n",
      "\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K01956 carbamoyl-phosphate synthase small subunit [EC:6.3.5.5]\n",
      "SSO SSO:000001403 Carbamoyl-phosphate synthase small chain (EC 6.3.5.5)\n",
      "b0032 ['Carbamoyl-phosphate synthase small chain (EC 6.3.5.5)']\n",
      "\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K01955 carbamoyl-phosphate synthase large subunit [EC:6.3.5.5]\n",
      "SSO SSO:000001400 Carbamoyl-phosphate synthase large chain (EC 6.3.5.5)\n",
      "b0033 ['Carbamoyl-phosphate synthase large chain (EC 6.3.5.5)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K08277 transcriptional activator CaiF\n",
      "SSO SSO:000008193 Transcriptional activatory protein CaiF\n",
      "b0034 ['Transcriptional activatory protein CaiF']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042413 carnitine catabolic process\n",
      "KO KO:K08279 carnitine operon protein CaiE\n",
      "SSO SSO:000001438 Carnitine operon protein CaiE\n",
      "b0035 ['Carnitine operon protein CaiE']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042413 carnitine catabolic process\n",
      "KO KO:K01692 enoyl-CoA hydratase [EC:4.2.1.17]\n",
      "KO KO:K08299 crotonobetainyl-CoA hydratase [EC:4.2.1.149]\n",
      "SSO SSO:000001441 Carnitinyl-CoA dehydratase (EC 4.2.1.-)\n",
      "b0036 ['Carnitinyl-CoA dehydratase (EC 4.2.1.149)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02182 carnitine-CoA ligase [EC:6.2.1.48]\n",
      "b0037 ['Crotonobetaine--CoA ligase', 'Carnitine--CoA ligase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042413 carnitine catabolic process\n",
      "KO KO:K08298 L-carnitine CoA-transferase [EC:2.8.3.21]\n",
      "SSO SSO:000001788 Crotonobetainyl-CoA:carnitine CoA-transferase (EC 2.8.3.-)\n",
      "b0038 ['Crotonobetainyl-CoA:carnitine CoA-transferase (EC 2.8.3.21)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006097 glyoxylate cycle\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042413 carnitine catabolic process\n",
      "KO KO:K08297 crotonobetainyl-CoA dehydrogenase [EC:1.3.8.13]\n",
      "SSO SSO:000001787 Crotonobetainyl-CoA dehydrogenase (EC 1.3.99.-)\n",
      "b0039 ['Crotonobetainyl-CoA dehydrogenase (EC 1.3.99.-)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05245 L-carnitine/gamma-butyrobetaine antiporter\n",
      "SSO SSO:000004134 L-carnitine/gamma-butyrobetaine antiporter\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b0040 ['L-carnitine/gamma-butyrobetaine antiporter']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K03521 electron transfer flavoprotein beta subunit\n",
      "SSO SSO:000002437 Electron transfer flavoprotein, beta subunit ecFixA\n",
      "b0041 ['Electron transfer flavoprotein, beta subunit ecFixA']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K03522 electron transfer flavoprotein alpha subunit\n",
      "SSO SSO:000002431 Electron transfer flavoprotein, alpha subunit ecFixB\n",
      "b0042 ['Electron transfer flavoprotein, alpha subunit ecFixB']\n",
      "\n",
      "GO GO:0042413 carnitine catabolic process\n",
      "KO KO:K00313 electron transfer flavoprotein-quinone oxidoreductase [EC:1.5.5.-]\n",
      "SSO SSO:000002446 Electron transfer flavoprotein-quinone oxidoreductase ecFixC\n",
      "b0043 ['Electron transfer flavoprotein-quinone oxidoreductase ecFixC']\n",
      "\n",
      "GO GO:0042413 carnitine catabolic process\n",
      "KO KO:K03855 ferredoxin like protein\n",
      "SSO SSO:000002712 Ferredoxin-like protein ecFixX\n",
      "b0044 ['Ferredoxin-like protein ecFixX']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08368 MFS transporter, putative metabolite transport protein\n",
      "SSO SSO:000023428 Putative metabolite transport protein YaaU\n",
      "b0045 ['Putative metabolite transport protein YaaU']\n",
      "\n",
      "KO KO:K11746 glutathione-regulated potassium-efflux system ancillary protein KefF\n",
      "SSO SSO:000003229 Glutathione-regulated potassium-efflux system ancillary protein KefF\n",
      "b0046 ['Glutathione-regulated potassium-efflux system ancillary protein KefF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015386 potassium:proton antiporter activity\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03455 monovalent cation:H+ antiporter-2, CPA2 family\n",
      "KO KO:K11745 glutathione-regulated potassium-efflux system ancillary protein KefC\n",
      "SSO SSO:000003234 Glutathione-regulated potassium-efflux system protein KefC\n",
      "b0047 ['Glutathione-regulated potassium-efflux system protein KefC']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K00287 dihydrofolate reductase [EC:1.5.1.3]\n",
      "SSO SSO:000002288 Dihydrofolate reductase (EC 1.5.1.3)\n",
      "b0048 ['Dihydrofolate reductase (EC 1.5.1.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01525 bis(5'-nucleosyl)-tetraphosphatase (symmetrical) [EC:3.6.1.41]\n",
      "SSO SSO:000001251 Bis(5'-nucleosyl)-tetraphosphatase, symmetrical (EC 3.6.1.41)\n",
      "SSO SSO:000013078 Diadenosine tetraphosphatase\n",
      "b0049 [\"Bis(5'-nucleosyl)-tetraphosphatase, symmetrical (EC 3.6.1.41)\"]\n",
      "\n",
      "KO KO:K06195 ApaG protein\n",
      "SSO SSO:000001020 ApaG protein\n",
      "b0050 ['ApaG protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K02528 16S rRNA (adenine1518-N6/adenine1519-N6)-dimethyltransferase [EC:2.1.1.182]\n",
      "SSO SSO:000007229 SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)\n",
      "b0051 ['SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "KO KO:K00097 4-hydroxythreonine-4-phosphate dehydrogenase [EC:1.1.1.262]\n",
      "SSO SSO:000000382 4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)\n",
      "b0052 ['4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K03771 peptidyl-prolyl cis-trans isomerase SurA [EC:5.2.1.8]\n",
      "SSO SSO:000037546 Periplasmic chaperone and peptidyl-prolyl cis-trans isomerase of outer membrane proteins SurA (EC 5.2.1.8)\n",
      "b0053 ['Periplasmic chaperone and peptidyl-prolyl cis-trans isomerase of outer membrane proteins SurA (EC 5.2.1.8)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04744 LPS-assembly protein\n",
      "b0054 ['LPS-assembly protein LptD', 'Organic solvent tolerance protein precursor']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K05801 DnaJ like chaperone protein\n",
      "SSO SSO:000002375 DnaJ-like protein DjlA\n",
      "b0055 ['DnaJ-like protein DjlA']\n",
      "\n",
      "SSO SSO:000028145 Uncharacterized protein YabP\n",
      "b4659 ['Uncharacterized protein YabP']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K06177 tRNA pseudouridine32 synthase / 23S rRNA pseudouridine746 synthase [EC:5.4.99.28 5.4.99.29]\n",
      "b0058 ['LSU rRNA pseudouridine(746) synthase (EC 5.4.99.29)', 'tRNA pseudouridine(32) synthase (EC 5.4.99.28)']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03580 ATP-dependent helicase HepA [EC:3.6.4.-]\n",
      "SSO SSO:000006895 RNA polymerase associated protein RapA (EC 3.6.1.-)\n",
      "b0059 ['RNA polymerase associated protein RapA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009314 response to radiation\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K02336 DNA polymerase II [EC:2.7.7.7]\n",
      "SSO SSO:000002078 DNA polymerase II (EC 2.7.7.7)\n",
      "b0060 ['DNA polymerase II (EC 2.7.7.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K03077 L-ribulose-5-phosphate 4-epimerase [EC:5.1.3.4]\n",
      "SSO SSO:000004170 L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)\n",
      "b0061 ['L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01804 L-arabinose isomerase [EC:5.3.1.4]\n",
      "SSO SSO:000018739 L-arabinose isomerase (EC 5.3.1.4)\n",
      "b0062 ['L-arabinose isomerase (EC 5.3.1.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00853 L-ribulokinase [EC:2.7.1.16]\n",
      "SSO SSO:000007156 Ribulokinase (EC 2.7.1.16)\n",
      "b0063 ['Ribulokinase (EC 2.7.1.16)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02099 AraC family transcriptional regulator, arabinose operon regulatory protein\n",
      "SSO SSO:000034675 Arabinose operon regulatory protein AraC\n",
      "b0064 ['Arabinose operon regulatory protein AraC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000013028 DedA family inner membrane protein YabI\n",
      "b0065 ['DedA family inner membrane protein YabI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000007995 Thiamin ABC transporter, ATPase component\n",
      "b0066 ['Thiamin ABC transporter, ATPase component']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02063 thiamine transport system permease protein\n",
      "SSO SSO:000007997 Thiamin ABC transporter, transmembrane component\n",
      "b0067 ['Thiamin ABC transporter, transmembrane component']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02064 thiamine transport system substrate-binding protein\n",
      "SSO SSO:000007996 Thiamin ABC transporter, substrate-binding component\n",
      "b0068 ['Thiamin ABC transporter, substrate-binding component']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K11925 SgrR family transcriptional regulator\n",
      "SSO SSO:000007454 SgrR, sugar-phosphate stress, transcriptional activator of SgrS small RNA\n",
      "b0069 ['SgrR, sugar-phosphate stress, transcriptional activator of SgrS small RNA']\n",
      "\n",
      "SSO SSO:000007799 Sugar-phosphate stress protein SgrT (embedded in SgrS)\n",
      "b4662 ['Sugar-phosphate stress protein SgrT (embedded in SgrS)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015543 obsolete lactose/glucose efflux transporter activity\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03291 MFS transporter, SET family, sugar efflux transporter\n",
      "SSO SSO:000025234 Sugar efflux transporter A\n",
      "b0070 ['Sugar efflux transporter A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K01704 3-isopropylmalate/(R)-2-methylmalate dehydratase small subunit [EC:4.2.1.33 4.2.1.35]\n",
      "SSO SSO:000000307 3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)\n",
      "b0071 ['3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K01703 3-isopropylmalate/(R)-2-methylmalate dehydratase large subunit [EC:4.2.1.33 4.2.1.35]\n",
      "SSO SSO:000000306 3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)\n",
      "b0072 ['3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K00052 3-isopropylmalate dehydrogenase [EC:1.1.1.85]\n",
      "SSO SSO:000000308 3-isopropylmalate dehydrogenase (EC 1.1.1.85)\n",
      "b0073 ['3-isopropylmalate dehydrogenase (EC 1.1.1.85)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K01649 2-isopropylmalate synthase [EC:2.3.3.13]\n",
      "SSO SSO:000000168 2-isopropylmalate synthase (EC 2.3.3.13)\n",
      "b0074 ['2-isopropylmalate synthase (EC 2.3.3.13)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K05806 leu operon leader peptide\n",
      "SSO SSO:000018892 Leu operon leader peptide\n",
      "b0075 ['Leu operon leader peptide']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K05798 LysR family transcriptional regulator, transcriptional activator for leuABCD operon\n",
      "SSO SSO:000019096 LysR family transcriptional activator LeuO\n",
      "b0076 ['LysR family transcriptional activator LeuO']\n",
      "\n",
      "KO KO:K01652 acetolactate synthase I/II/III large subunit [EC:2.2.1.6]\n",
      "SSO SSO:000000685 Acetolactate synthase large subunit (EC 2.2.1.6)\n",
      "b0077 ['Acetolactate synthase large subunit (EC 2.2.1.6)']\n",
      "\n",
      "KO KO:K01653 acetolactate synthase I/III small subunit [EC:2.2.1.6]\n",
      "SSO SSO:000000686 Acetolactate synthase small subunit (EC 2.2.1.6)\n",
      "b0078 ['Acetolactate synthase small subunit (EC 2.2.1.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03435 LacI family transcriptional regulator, fructose operon transcriptional repressor\n",
      "SSO SSO:000016900 Fructose repressor FruR, LacI family\n",
      "b0080 ['Fructose repressor FruR, LacI family']\n",
      "\n",
      "KO KO:K03925 MraZ protein\n",
      "SSO SSO:000039602 Transcriptional regulator MraZ\n",
      "b0081 ['Transcriptional regulator MraZ']\n",
      "\n",
      "GO GO:0042158 lipoprotein biosynthetic process\n",
      "KO KO:K03438 16S rRNA (cytosine1402-N4)-methyltransferase [EC:2.1.1.199]\n",
      "SSO SSO:000033922 16S rRNA (cytosine(1402)-N(4))-methyltransferase (EC 2.1.1.199)\n",
      "b0082 ['16S rRNA (cytosine(1402)-N(4))-methyltransferase (EC 2.1.1.199)']\n",
      "\n",
      "KO KO:K03586 cell division protein FtsL\n",
      "SSO SSO:000001468 Cell division protein FtsL\n",
      "b0083 ['Cell division protein FtsL']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03587 cell division protein FtsI (penicillin-binding protein 3) [EC:3.4.16.4]\n",
      "SSO SSO:000001467 Cell division protein FtsI [Peptidoglycan synthetase] (EC 2.4.1.129)\n",
      "b0084 ['Cell division protein FtsI [Peptidoglycan synthetase] (EC 2.4.1.129)']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01928 UDP-N-acetylmuramoyl-L-alanyl-D-glutamate--2,6-diaminopimelate ligase [EC:6.3.2.13]\n",
      "SSO SSO:000039894 UDP-N-acetylmuramoyl-dipeptide--2,6-diaminopimelate ligase (EC 6.3.2.13)\n",
      "b0085 ['UDP-N-acetylmuramoyl-dipeptide--2,6-diaminopimelate ligase (EC 6.3.2.13)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01929 UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase [EC:6.3.2.10]\n",
      "SSO SSO:000008630 UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase (EC 6.3.2.10)\n",
      "b0086 ['UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase (EC 6.3.2.10)']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01000 phospho-N-acetylmuramoyl-pentapeptide-transferase [EC:2.7.8.13]\n",
      "SSO SSO:000005947 Phospho-N-acetylmuramoyl-pentapeptide-transferase (EC 2.7.8.13)\n",
      "b0087 ['Phospho-N-acetylmuramoyl-pentapeptide-transferase (EC 2.7.8.13)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01925 UDP-N-acetylmuramoylalanine--D-glutamate ligase [EC:6.3.2.9]\n",
      "SSO SSO:000039893 UDP-N-acetylmuramoyl-L-alanine--D-glutamate ligase (EC 6.3.2.9)\n",
      "b0088 ['UDP-N-acetylmuramoyl-L-alanine--D-glutamate ligase (EC 6.3.2.9)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03588 cell division protein FtsW\n",
      "SSO SSO:000037529 Peptidoglycan glycosyltransferase FtsW (EC 2.4.1.129)\n",
      "b0089 ['Peptidoglycan glycosyltransferase FtsW (EC 2.4.1.129)']\n",
      "\n",
      "EC EC:2.4.1.157 Transferred entry: 2.4.1.336 and 2.4.1.337.\n",
      "EC EC:2.4.1.227 Undecaprenyldiphospho-muramoylpentapeptide beta-N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.241 Leucyl endopeptidase.\n",
      "EC EC:2.4.1.46 Monogalactosyldiacylglycerol synthase.\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02563 UDP-N-acetylglucosamine--N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase [EC:2.4.1.227]\n",
      "SSO SSO:000008622 UDP-N-acetylglucosamine--N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase (EC 2.4.1.227)\n",
      "SSO SSO:000020190 N-acetylglucosaminyltransferase\n",
      "b0090 ['UDP-N-acetylglucosamine--N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase (EC 2.4.1.227)']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01924 UDP-N-acetylmuramate--alanine ligase [EC:6.3.2.8]\n",
      "SSO SSO:000039892 UDP-N-acetylmuramate--L-alanine ligase (EC 6.3.2.8)\n",
      "b0091 ['UDP-N-acetylmuramate--L-alanine ligase (EC 6.3.2.8)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01921 D-alanine-D-alanine ligase [EC:6.3.2.4]\n",
      "SSO SSO:000001986 D-alanine--D-alanine ligase (EC 6.3.2.4)\n",
      "b0092 ['D-alanine--D-alanine ligase (EC 6.3.2.4)']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03589 cell division protein FtsQ\n",
      "SSO SSO:000001469 Cell division protein FtsQ\n",
      "b0093 ['Cell division protein FtsQ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03590 cell division protein FtsA\n",
      "SSO SSO:000001465 Cell division protein FtsA\n",
      "b0094 ['Cell division protein FtsA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03531 cell division protein FtsZ\n",
      "SSO SSO:000001472 Cell division protein FtsZ (EC 3.4.24.-)\n",
      "b0095 ['Cell division protein FtsZ']\n",
      "\n",
      "EC EC:3.5.1.108 UDP-3-O-acyl-N-acetylglucosamine deacetylase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02535 UDP-3-O-[3-hydroxymyristoyl] N-acetylglucosamine deacetylase [EC:3.5.1.108]\n",
      "SSO SSO:000008603 UDP-3-O-[3-hydroxymyristoyl] N-acetylglucosamine deacetylase (EC 3.5.1.-)\n",
      "b0096 ['UDP-3-O-[3-hydroxymyristoyl] N-acetylglucosamine deacetylase (EC 3.5.1.108)']\n",
      "\n",
      "KO KO:K13301 secretion monitor\n",
      "SSO SSO:000039122 Secretion monitor SecM\n",
      "b0097 ['Secretion monitor SecM']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03070 preprotein translocase subunit SecA\n",
      "SSO SSO:000006520 Protein translocase subunit SecA\n",
      "b0098 ['Protein translocase subunit SecA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03574 8-oxo-dGTP diphosphatase [EC:3.6.1.55]\n",
      "SSO SSO:000005047 Mutator mutT protein (7,8-dihydro-8-oxoguanine-triphosphatase) (EC 3.6.1.-)\n",
      "b0099 ['Mutator MutT protein (7,8-dihydro-8-oxoguanine-triphosphatase) (EC 3.6.1.-)']\n",
      "\n",
      "KO KO:K09862 uncharacterized protein\n",
      "SSO SSO:000012759 DNA gyrase inhibitor YacG\n",
      "b0101 ['DNA gyrase inhibitor YacG']\n",
      "\n",
      "KO KO:K18778 cell division protein ZapD\n",
      "SSO SSO:000001477 Cell division protein ZapD\n",
      "b0102 ['Cell division protein ZapD']\n",
      "\n",
      "KO KO:K00859 dephospho-CoA kinase [EC:2.7.1.24]\n",
      "SSO SSO:000002258 Dephospho-CoA kinase (EC 2.7.1.24)\n",
      "b0103 ['Dephospho-CoA kinase (EC 2.7.1.24)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009152 purine ribonucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00364 GMP reductase [EC:1.7.1.7]\n",
      "SSO SSO:000002977 GMP reductase (EC 1.7.1.7)\n",
      "b0104 ['GMP reductase (EC 1.7.1.7)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02505 protein transport protein HofC\n",
      "SSO SSO:000008560 Type IV fimbrial assembly protein PilC\n",
      "b0106 ['Type IV fimbrial assembly protein PilC']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K02504 protein transport protein HofB\n",
      "SSO SSO:000008561 Type IV fimbrial assembly, ATPase PilB\n",
      "b0107 ['Type IV fimbrial assembly, ATPase PilB']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K02650 type IV pilus assembly protein PilA\n",
      "KO KO:K02682 prepilin peptidase dependent protein D\n",
      "SSO SSO:000008566 Type IV pilin PilA\n",
      "b0108 ['Type IV pilin PilA']\n",
      "\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "KO KO:K00767 nicotinate-nucleotide pyrophosphorylase (carboxylating) [EC:2.4.2.19]\n",
      "SSO SSO:000006877 Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)\n",
      "b0109 ['Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01447 N-acetylmuramoyl-L-alanine amidase [EC:3.5.1.28]\n",
      "KO KO:K03806 N-acetyl-anhydromuramoyl-L-alanine amidase [EC:3.5.1.28]\n",
      "SSO SSO:000000041 1,6-anhydro-N-acetylmuramyl-L-alanine amidase\n",
      "b0110 ['1,6-anhydro-N-acetylmuramyl-L-alanine amidase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03807 AmpE protein\n",
      "SSO SSO:000010632 AmpE protein\n",
      "b0111 ['AmpE protein']\n",
      "\n",
      "GO GO:0006571 tyrosine biosynthetic process\n",
      "GO GO:0009094 L-phenylalanine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11734 aromatic amino acid transport protein AroP\n",
      "SSO SSO:000010817 Aromatic amino acid transport protein AroP\n",
      "b0112 ['Aromatic amino acid transport protein AroP']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "SSO SSO:000008269 Transcriptional repressor for pyruvate dehydrogenase complex\n",
      "b0113 ['Transcriptional repressor for pyruvate dehydrogenase complex']\n",
      "\n",
      "GO GO:0006086 acetyl-CoA biosynthetic process from pyruvate\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009436 glyoxylate catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00163 pyruvate dehydrogenase E1 component [EC:1.2.4.1]\n",
      "SSO SSO:000006847 Pyruvate dehydrogenase E1 component (EC 1.2.4.1)\n",
      "b0114 ['Pyruvate dehydrogenase E1 component (EC 1.2.4.1)']\n",
      "\n",
      "GO GO:0006086 acetyl-CoA biosynthetic process from pyruvate\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009436 glyoxylate catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00627 pyruvate dehydrogenase E2 component (dihydrolipoamide acetyltransferase) [EC:2.3.1.12]\n",
      "SSO SSO:000002291 Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)\n",
      "b0115 ['Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006086 acetyl-CoA biosynthetic process from pyruvate\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009436 glyoxylate catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00382 dihydrolipoamide dehydrogenase [EC:1.8.1.4]\n",
      "b0116 ['Dihydrolipoamide dehydrogenase of pyruvate dehydrogenase complex (EC 1.8.1.4)', 'Dihydrolipoamide dehydrogenase (EC 1.8.1.4)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000041436 Uncharacterized protein YacH\n",
      "b0117 ['Uncharacterized protein YacH']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K01682 aconitate hydratase 2 / 2-methylisocitrate dehydratase [EC:4.2.1.3 4.2.1.99]\n",
      "b0118 ['Aconitate hydratase 2 (EC 4.2.1.3)', '2-methylisocitrate dehydratase (EC 4.2.1.99)']\n",
      "\n",
      "KO KO:K09910 uncharacterized protein\n",
      "SSO SSO:000026588 UPF0231 protein YacL\n",
      "b0119 ['UPF0231 protein YacL']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "KO KO:K01611 S-adenosylmethionine decarboxylase [EC:4.1.1.50]\n",
      "SSO SSO:000007194 S-adenosylmethionine decarboxylase proenzyme , prokaryotic class 1A (EC 4.1.1.50)\n",
      "b0120 ['S-adenosylmethionine decarboxylase proenzyme (EC 4.1.1.50), prokaryotic class 1A']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "KO KO:K00797 spermidine synthase [EC:2.5.1.16]\n",
      "SSO SSO:000007627 Spermidine synthase (EC 2.5.1.16)\n",
      "b0121 ['Spermidine synthase (EC 2.5.1.16)']\n",
      "\n",
      "SSO SSO:000006246 Predicted chaperone lipoprotein YacC, potentially involved in protein secretion\n",
      "b0122 ['Predicted chaperone lipoprotein YacC, potentially involved in protein secretion']\n",
      "\n",
      "EC EC:1.10.3.2 Laccase.\n",
      "KO KO:K14588 blue copper oxidase\n",
      "SSO SSO:000011395 Blue copper oxidase CueO precursor\n",
      "b0123 ['Blue copper oxidase CueO precursor']\n",
      "\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00117 quinoprotein glucose dehydrogenase [EC:1.1.5.2]\n",
      "SSO SSO:000003122 Glucose dehydrogenase, PQQ-dependent (EC 1.1.5.2)\n",
      "b0124 ['Glucose dehydrogenase, PQQ-dependent (EC 1.1.5.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00760 hypoxanthine phosphoribosyltransferase [EC:2.4.2.8]\n",
      "SSO SSO:000003687 Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)\n",
      "SSO SSO:000043066 hypoxanthine phosphoribosyltransferase (EC 2.4.2.8)\n",
      "b0125 ['Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)']\n",
      "\n",
      "KO KO:K01673 carbonic anhydrase [EC:4.2.1.1]\n",
      "SSO SSO:000001419 Carbonic anhydrase (EC 4.2.1.1)\n",
      "SSO SSO:000001421 Carbonic anhydrase, beta class (EC 4.2.1.1)\n",
      "b0126 ['Carbonic anhydrase, beta class (EC 4.2.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01990 ABC-2 type transport system ATP-binding protein\n",
      "SSO SSO:000040102 Uncharacterized efflux ABC transporter, ATP-binding protein YadG\n",
      "b0127 ['Uncharacterized efflux ABC transporter, ATP-binding protein YadG']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000040107 Uncharacterized efflux ABC transporter, permease protein YadH\n",
      "b0128 ['Uncharacterized efflux ABC transporter, permease protein YadH']\n",
      "\n",
      "GO GO:0009401 phosphoenolpyruvate-dependent sugar phosphotransferase system\n",
      "SSO SSO:000023096 Putative PTS system IIA component yadI (EC 2.7.1.69)\n",
      "b0129 ['Putative PTS system IIA component YadI']\n",
      "\n",
      "EC EC:3.1.1.72 Acetylxylan esterase.\n",
      "EC EC:3.5.1.41 Chitin deacetylase.\n",
      "SSO SSO:000041437 Uncharacterized protein YadE\n",
      "b0130 ['Uncharacterized protein YadE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015937 coenzyme A biosynthetic process\n",
      "KO KO:K01579 aspartate 1-decarboxylase [EC:4.1.1.11]\n",
      "SSO SSO:000001091 Aspartate 1-decarboxylase (EC 4.1.1.11)\n",
      "b0131 ['Aspartate 1-decarboxylase (EC 4.1.1.11)']\n",
      "\n",
      "KO KO:K24088 Unknown\n",
      "b0132 ['Uncharacterized protein YadD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015937 coenzyme A biosynthetic process\n",
      "KO KO:K01918 pantoate--beta-alanine ligase [EC:6.3.2.1]\n",
      "SSO SSO:000005727 Pantoate--beta-alanine ligase (EC 6.3.2.1)\n",
      "b0133 ['Pantoate--beta-alanine ligase (EC 6.3.2.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015937 coenzyme A biosynthetic process\n",
      "KO KO:K00606 3-methyl-2-oxobutanoate hydroxymethyltransferase [EC:2.1.2.11]\n",
      "SSO SSO:000000315 3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)\n",
      "b0134 ['3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000008716 Uncharacterized fimbrial-like protein YadC\n",
      "b0135 ['Uncharacterized fimbrial-like protein YadC']\n",
      "\n",
      "SSO SSO:000008717 Uncharacterized fimbrial-like protein YadK\n",
      "b0136 ['Uncharacterized fimbrial-like protein YadK']\n",
      "\n",
      "SSO SSO:000008718 Uncharacterized fimbrial-like protein YadL\n",
      "b0137 ['Uncharacterized fimbrial-like protein YadL']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000008719 Uncharacterized fimbrial-like protein YadM\n",
      "b0138 ['Uncharacterized fimbrial-like protein YadM']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07347 outer membrane usher protein\n",
      "SSO SSO:000005604 Outer membrane usher protein HtrE\n",
      "SSO SSO:000023473 Putative outer membrane usher protein\n",
      "b0139 ['Outer membrane usher protein HtrE']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15540 chaperone protein EcpD\n",
      "SSO SSO:000002780 Fimbria adhesin EcpD\n",
      "b0140 ['Fimbria adhesin EcpD']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000008720 Uncharacterized fimbrial-like protein YadN\n",
      "b0141 ['Uncharacterized fimbrial-like protein YadN']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K00950 2-amino-4-hydroxy-6-hydroxymethyldihydropteridine diphosphokinase [EC:2.7.6.3]\n",
      "SSO SSO:000000127 2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)\n",
      "b0142 ['2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00970 poly(A) polymerase [EC:2.7.7.19]\n",
      "SSO SSO:000006136 Poly(A) polymerase (EC 2.7.7.19)\n",
      "b0143 ['Poly(A) polymerase (EC 2.7.7.19)']\n",
      "\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01894 glutamyl-Q tRNA(Asp) synthetase [EC:6.1.1.-]\n",
      "SSO SSO:000017292 Glutamyl-Q tRNA(Asp) synthetase\n",
      "b0144 ['Glutamyl-Q tRNA(Asp) synthetase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K06204 DnaK suppressor protein\n",
      "SSO SSO:000006922 RNA polymerase-binding transcription factor DksA\n",
      "b0145 ['RNA polymerase-binding transcription factor DksA']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K06206 sugar fermentation stimulation protein A\n",
      "SSO SSO:000039290 Sugar fermentation stimulation protein SfsA\n",
      "b0146 ['Sugar fermentation stimulation protein SfsA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K01975 RNA 2',3'-cyclic 3'-phosphodiesterase [EC:3.1.4.58]\n",
      "SSO SSO:000000071 2'-5' RNA ligase (EC 6.5.1.-)\n",
      "b0147 [\"2'-5' RNA ligase\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K03579 ATP-dependent helicase HrpB [EC:3.6.4.13]\n",
      "SSO SSO:000010058 ATP-dependent helicase HrpB\n",
      "SSO SSO:000023025 Putative ATP-dependent helicase\n",
      "b0148 ['ATP-dependent helicase HrpB']\n",
      "\n",
      "EC EC:2.4.1.129 Peptidoglycan glycosyltransferase.\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K05365 penicillin-binding protein 1B [EC:2.4.1.129 3.4.16.4]\n",
      "SSO SSO:000005028 Multimodular transpeptidase-transglycosylase (EC 3.4.-.-) (EC 2.4.1.129)\n",
      "b0149 ['Multimodular transpeptidase-transglycosylase (EC 2.4.1.129) (EC 3.4.-.-)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "SSO SSO:000002731 Ferric hydroxamate outer membrane receptor FhuA\n",
      "b0150 ['Ferric hydroxamate outer membrane receptor FhuA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02013 iron complex transport system ATP-binding protein [EC:3.6.3.34]\n",
      "KO KO:K10829 iron-chelate-transporting ATPase [EC:3.6.3.34]\n",
      "SSO SSO:000016698 Ferric hydroxamate ABC transporter (TC 3.A.1.14.3), ATP-binding protein FhuC\n",
      "b0151 ['Ferric hydroxamate ABC transporter (TC 3.A.1.14.3), ATP-binding protein FhuC']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K23227 Unknown\n",
      "SSO SSO:000016699 Ferric hydroxamate ABC transporter (TC 3.A.1.14.3), periplasmic substrate binding protein FhuD\n",
      "b0152 ['Ferric hydroxamate ABC transporter (TC 3.A.1.14.3), periplasmic substrate binding protein FhuD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02015 iron complex transport system permease protein\n",
      "KO KO:K23228 Unknown\n",
      "SSO SSO:000016700 Ferric hydroxamate ABC transporter (TC 3.A.1.14.3), permease component FhuB\n",
      "b0153 ['Ferric hydroxamate ABC transporter (TC 3.A.1.14.3), permease component FhuB']\n",
      "\n",
      "KO KO:K01845 glutamate-1-semialdehyde 2,1-aminomutase [EC:5.4.3.8]\n",
      "SSO SSO:000003163 Glutamate-1-semialdehyde 2,1-aminomutase (EC 5.4.3.8)\n",
      "b0154 ['Glutamate-1-semialdehyde 2,1-aminomutase (EC 5.4.3.8)']\n",
      "\n",
      "KO KO:K03281 chloride channel protein, CIC family\n",
      "SSO SSO:000041674 Voltage-gated H(+)/2Cl(-) exchange transporter ClcA\n",
      "b0155 ['Voltage-gated H(+)/2Cl(-) exchange transporter ClcA']\n",
      "\n",
      "KO KO:K15724 iron-sulfur cluster insertion protein\n",
      "SSO SSO:000036433 Iron-sulfur cluster insertion protein ErpA\n",
      "b0156 ['Iron-sulfur cluster insertion protein ErpA']\n",
      "\n",
      "SSO SSO:000026535 UPF0126 inner membrane protein YadS\n",
      "b0157 ['UPF0126 inner membrane protein YadS']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02016 iron complex transport system substrate-binding protein\n",
      "KO KO:K06858 vitamin B12 transport system substrate-binding protein\n",
      "SSO SSO:000041670 Vitamin B12 ABC transporter (EC 7.6.2.8), substrate-binding protein BtuF\n",
      "b0158 ['Vitamin B12 ABC transporter, substrate-binding protein BtuF']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01243 adenosylhomocysteine nucleosidase [EC:3.2.2.9]\n",
      "b0159 [\"5'-methylthioadenosine nucleosidase (EC 3.2.2.16)\", 'S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01129 dGTPase [EC:3.1.5.1]\n",
      "SSO SSO:000002246 Deoxyguanosinetriphosphate triphosphohydrolase (EC 3.1.5.1)\n",
      "SSO SSO:000013054 Deoxyguanosinetriphosphate triphosphohydrolase (EC 3.1.5.1), subgroup 1\n",
      "b0160 ['Deoxyguanosinetriphosphate triphosphohydrolase (EC 3.1.5.1), subgroup 1']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K04771 serine protease Do [EC:3.4.21.107]\n",
      "KO KO:K04772 serine protease DegQ [EC:3.4.21.-]\n",
      "SSO SSO:000017912 HtrA protease/chaperone protein\n",
      "b0161 ['HtrA protease/chaperone protein']\n",
      "\n",
      "KO KO:K02647 carbohydrate diacid regulator\n",
      "SSO SSO:000007797 Sugar diacid utilization regulator SdaR\n",
      "b0162 ['Sugar diacid utilization regulator SdaR']\n",
      "\n",
      "SSO SSO:000008670 UPF0325 protein YaeH\n",
      "b0163 ['UPF0325 protein YaeH']\n",
      "\n",
      "KO KO:K07098 uncharacterized protein\n",
      "SSO SSO:000038512 Phosphodiesterase YaeI\n",
      "b0164 ['Phosphodiesterase YaeI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K00674 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase [EC:2.3.1.117]\n",
      "SSO SSO:000000074 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)\n",
      "b0166 ['2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K00990 [protein-PII] uridylyltransferase [EC:2.7.7.59]\n",
      "b0167 ['[Protein-PII] uridylyltransferase (EC 2.7.7.59)', '[Protein-PII]-UMP uridylyl-removing enzyme']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K01265 methionyl aminopeptidase [EC:3.4.11.18]\n",
      "SSO SSO:000004833 Methionine aminopeptidase (EC 3.4.11.18)\n",
      "b0168 ['Methionine aminopeptidase (EC 3.4.11.18)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02967 small subunit ribosomal protein S2\n",
      "SSO SSO:000007308 SSU ribosomal protein S2p (SAe)\n",
      "b0169 ['SSU ribosomal protein S2p (SAe)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02357 elongation factor Ts\n",
      "SSO SSO:000008306 Translation elongation factor Ts\n",
      "b0170 ['Translation elongation factor Ts']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K09903 uridylate kinase [EC:2.7.4.22]\n",
      "SSO SSO:000033811 uridylate kinase\n",
      "b0171 ['Uridylate kinase (EC 2.7.4.22)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "GO GO:0009386 translational attenuation\n",
      "KO KO:K02838 ribosome recycling factor\n",
      "SSO SSO:000007148 Ribosome recycling factor\n",
      "b0172 ['Ribosome recycling factor']\n",
      "\n",
      "KO KO:K00099 1-deoxy-D-xylulose-5-phosphate reductoisomerase [EC:1.1.1.267]\n",
      "SSO SSO:000000045 1-deoxy-D-xylulose 5-phosphate reductoisomerase (EC 1.1.1.267)\n",
      "b0173 ['1-deoxy-D-xylulose 5-phosphate reductoisomerase (EC 1.1.1.267)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00806 undecaprenyl diphosphate synthase [EC:2.5.1.31]\n",
      "SSO SSO:000008811 Undecaprenyl diphosphate synthase (EC 2.5.1.31)\n",
      "SSO SSO:000028834 Undecaprenyl pyrophosphate synthase\n",
      "b0174 ['Undecaprenyl diphosphate synthase (EC 2.5.1.31)']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00981 phosphatidate cytidylyltransferase [EC:2.7.7.41]\n",
      "SSO SSO:000005932 Phosphatidate cytidylyltransferase (EC 2.7.7.41)\n",
      "b0175 ['Phosphatidate cytidylyltransferase (EC 2.7.7.41)']\n",
      "\n",
      "KO KO:K11749 regulator of sigma E protease [EC:3.4.24.-]\n",
      "SSO SSO:000018568 Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage\n",
      "b0176 ['Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07277 outer membrane protein insertion porin family\n",
      "SSO SSO:000005595 Outer membrane protein assembly factor YaeT\n",
      "b0177 ['Outer membrane protein assembly factor YaeT']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K06142 outer membrane protein\n",
      "b0178 ['Periplasmic chaperone of outer membrane proteins Skp', 'Outer membrane protein H precursor']\n",
      "\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02536 UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase [EC:2.3.1.191]\n",
      "SSO SSO:000008604 UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase (EC 2.3.1.-)\n",
      "b0179 ['UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase (EC 2.3.1.191)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02372 3-hydroxyacyl-[acyl-carrier-protein] dehydratase [EC:4.2.1.59]\n",
      "SSO SSO:000000290 3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)\n",
      "b0180 ['3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00677 UDP-N-acetylglucosamine acyltransferase [EC:2.3.1.129]\n",
      "SSO SSO:000000799 Acyl-[acyl-carrier-protein]--UDP-N-acetylglucosamine O-acyltransferase (EC 2.3.1.129)\n",
      "b0181 ['Acyl-[acyl-carrier-protein]--UDP-N-acetylglucosamine O-acyltransferase (EC 2.3.1.129)']\n",
      "\n",
      "EC EC:2.4.1.182 Lipid-A-disaccharide synthase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00748 lipid-A-disaccharide synthase [EC:2.4.1.182]\n",
      "SSO SSO:000004460 Lipid-A-disaccharide synthase (EC 2.4.1.182)\n",
      "b0182 ['Lipid-A-disaccharide synthase (EC 2.4.1.182)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K03470 ribonuclease HII [EC:3.1.26.4]\n",
      "SSO SSO:000007081 Ribonuclease HII (EC 3.1.26.4)\n",
      "b0183 ['Ribonuclease HII (EC 3.1.26.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02337 DNA polymerase III subunit alpha [EC:2.7.7.7]\n",
      "SSO SSO:000002079 DNA polymerase III alpha subunit (EC 2.7.7.7)\n",
      "b0184 ['DNA polymerase III alpha subunit (EC 2.7.7.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01962 acetyl-CoA carboxylase carboxyl transferase subunit alpha [EC:6.4.1.2 2.1.3.15]\n",
      "SSO SSO:000000716 Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)\n",
      "b0185 ['Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)']\n",
      "\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01582 lysine decarboxylase [EC:4.1.1.18]\n",
      "SSO SSO:000004599 Lysine decarboxylase 2, constitutive (EC 4.1.1.18)\n",
      "b0186 ['Lysine decarboxylase 2, constitutive (EC 4.1.1.18)']\n",
      "\n",
      "KO KO:K08234 glyoxylase I family protein\n",
      "SSO SSO:000023401 Putative lyase\n",
      "SSO SSO:000028147 Uncharacterized protein YaeR with similarity to glyoxylase family\n",
      "b0187 ['Uncharacterized protein YaeR with similarity to glyoxylase family']\n",
      "\n",
      "KO KO:K04075 tRNA(Ile)-lysidine synthase [EC:6.3.4.19]\n",
      "SSO SSO:000009372 tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)\n",
      "b0188 ['tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K19000 Rho-binding antiterminator\n",
      "SSO SSO:000007044 Rho-specific inhibitor of transcription termination (YaeO)\n",
      "b0189 ['Rho-specific inhibitor of transcription termination (YaeO)']\n",
      "\n",
      "SSO SSO:000026602 UPF0253 protein YaeP\n",
      "b4406 ['UPF0253 protein YaeP']\n",
      "\n",
      "SSO SSO:000029106 YaeQ protein\n",
      "b0190 ['YaeQ protein']\n",
      "\n",
      "KO KO:K15034 ribosome-associated protein\n",
      "SSO SSO:000037535 Peptidyl-tRNA hydrolase ArfB (EC 3.1.1.29)\n",
      "b0191 ['Peptidyl-tRNA hydrolase ArfB (EC 3.1.1.29)']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K06079 copper homeostasis protein (lipoprotein)\n",
      "b0192 ['Copper homeostasis protein CutF precursor', 'Lipoprotein NlpE involeved in surface adhesion']\n",
      "\n",
      "SSO SSO:000020814 putative lipoprotein\n",
      "SSO SSO:000027221 Uncharacterized lipoprotein yaeF precursor\n",
      "b0193 ['Uncharacterized lipoprotein yaeF precursor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01881 prolyl-tRNA synthetase [EC:6.1.1.15]\n",
      "SSO SSO:000006373 Prolyl-tRNA synthetase (EC 6.1.1.15)\n",
      "SSO SSO:000006375 Prolyl-tRNA synthetase , bacterial type (EC 6.1.1.15)\n",
      "b0194 ['Prolyl-tRNA synthetase (EC 6.1.1.15), bacterial type']\n",
      "\n",
      "KO KO:K22900 tRNA (adenine37-N6)-methyltransferase [EC:2.1.1.-]\n",
      "SSO SSO:000036225 tRNA (adenine(37)-N6)-methyltransferase\n",
      "b0195 ['tRNA (adenine(37)-N6)-methyltransferase']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K06080 RcsF protein\n",
      "SSO SSO:000022712 Protein RcsF\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "b0196 ['Protein RcsF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02073 D-methionine transport system substrate-binding protein\n",
      "SSO SSO:000004832 Methionine ABC transporter substrate-binding protein\n",
      "b0197 ['Methionine ABC transporter substrate-binding protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02072 D-methionine transport system permease protein\n",
      "SSO SSO:000019558 Methionine ABC transporter permease protein\n",
      "b0198 ['Methionine ABC transporter permease protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02071 D-methionine transport system ATP-binding protein\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "b0199 ['Methionine ABC transporter ATP-binding protein']\n",
      "\n",
      "KO KO:K03273 D-glycero-D-manno-heptose 1,7-bisphosphate phosphatase [EC:3.1.3.82 3.1.3.83]\n",
      "SSO SSO:000002028 D-glycero-beta-D-manno-heptose-1,7-bisphosphate 7-phosphatase (EC 3.1.3.82)\n",
      "b0200 ['D-glycero-beta-D-manno-heptose-1,7-bisphosphate 7-phosphatase (EC 3.1.3.82)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K06222 2,5-diketo-D-gluconate reductase B [EC:1.1.1.346]\n",
      "b0207 ['Methylglyoxal reductase, dihydroxyacetone producing', '2,5-didehydrogluconate reductase (2-dehydro-L-gulonate-forming) (EC 1.1.1.346)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000039687 Transcriptional regulator YafC, LysR family\n",
      "b0208 ['Transcriptional regulator YafC, LysR family']\n",
      "\n",
      "SSO SSO:000026616 UPF0294 protein YafD\n",
      "b0209 ['UPF0294 protein YafD']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "SSO SSO:000007217 SAM-dependent methyltransferase YafE (UbiE paralog)\n",
      "b0210 ['SAM-dependent methyltransferase YafE (UbiE paralog)']\n",
      "\n",
      "EC EC:3.2.1.14 Chitinase.\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K08307 membrane-bound lytic murein transglycosylase D [EC:4.2.2.-]\n",
      "SSO SSO:000019410 Membrane bound lytic murein transglycosylase D (EC 3.2.1.-)\n",
      "b0211 ['Membrane-bound lytic murein transglycosylase D']\n",
      "\n",
      "GO GO:0009438 methylglyoxal metabolic process\n",
      "KO KO:K01069 hydroxyacylglutathione hydrolase [EC:3.1.2.6]\n",
      "SSO SSO:000003519 Hydroxyacylglutathione hydrolase (EC 3.1.2.6)\n",
      "b0212 ['Hydroxyacylglutathione hydrolase (EC 3.1.2.6)']\n",
      "\n",
      "b0213 ['FIG005121: SAM-dependent methyltransferase (EC 2.1.1.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006401 RNA catabolic process\n",
      "GO GO:0006508 proteolysis\n",
      "KO KO:K03469 ribonuclease HI [EC:3.1.26.4]\n",
      "SSO SSO:000007076 Ribonuclease HI (EC 3.1.26.4)\n",
      "b0214 ['Ribonuclease HI (EC 3.1.26.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K02342 DNA polymerase III subunit epsilon [EC:2.7.7.7]\n",
      "SSO SSO:000002084 DNA polymerase III epsilon subunit (EC 2.7.7.7)\n",
      "b0215 ['DNA polymerase III epsilon subunit (EC 2.7.7.7)']\n",
      "\n",
      "SSO SSO:000018986 Lipoprotein\n",
      "SSO SSO:000027200 Uncharacterized lipoprotein YafT\n",
      "b0217 ['Uncharacterized lipoprotein YafT']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4586 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0218 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4503 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000000885 Aliphatic amidase AmiE (EC 3.5.1.4)\n",
      "b0219 ['Aliphatic amidase AmiE (EC 3.5.1.4)']\n",
      "\n",
      "SSO SSO:000003872 Inhibitor of vertebrate c-type lysozyme, periplasmic => Ivy\n",
      "b0220 ['Inhibitor of vertebrate c-type lysozyme, periplasmic => Ivy']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K06445 acyl-CoA dehydrogenase [EC:1.3.99.-]\n",
      "b0221 ['Acyl-coenzyme A dehydrogenase FadE (EC 1.3.8.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K03271 D-sedoheptulose 7-phosphate isomerase [EC:5.3.1.28]\n",
      "SSO SSO:000012708 D-sedoheptulose 7-phosphate isomerase (EC 5.3.1.28)\n",
      "b0222 ['D-sedoheptulose 7-phosphate isomerase (EC 5.3.1.28)']\n",
      "\n",
      "SSO SSO:000023318 Putative glutamine amidotransferase YafJ\n",
      "b0223 ['Putative glutamine amidotransferase YafJ']\n",
      "\n",
      "SSO SSO:000023066 Putative L,D-transpeptidase YafK\n",
      "b0224 ['Putative L,D-transpeptidase YafK']\n",
      "\n",
      "KO KO:K19157 mRNA interferase YafQ [EC:3.1.-.-]\n",
      "SSO SSO:000009179 mRNA interferase YafQ\n",
      "b0225 ['mRNA interferase YafQ']\n",
      "\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K07473 DNA-damage-inducible protein J\n",
      "SSO SSO:000001016 Antitoxin DinJ (binds YafQ toxin)\n",
      "b0226 ['Antitoxin DinJ (binds YafQ toxin)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K19303 murein DD-endopeptidase [EC:3.4.-.-]\n",
      "SSO SSO:000022470 Probable endopeptidase YafL\n",
      "b0227 ['Probable endopeptidase YafL']\n",
      "\n",
      "KO KO:K07491 putative transposase\n",
      "SSO SSO:000026107 Transposase and inactivated derivatives\n",
      "b0228 ['Transposase and inactivated derivatives']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000002800 Flagellar biosynthesis protein FlhA\n",
      "b0229 ['Flagellar biosynthesis protein FlhA']\n",
      "\n",
      "GO GO:0042330 taxis\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0230 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K02346 DNA polymerase IV [EC:2.7.7.7]\n",
      "SSO SSO:000002086 DNA polymerase IV (EC 2.7.7.7)\n",
      "b0231 ['DNA polymerase IV (EC 2.7.7.7)']\n",
      "\n",
      "KO KO:K19161 antitoxin YafN\n",
      "SSO SSO:000010731 Antitoxin YafN\n",
      "b0232 ['Antitoxin YafN']\n",
      "\n",
      "KO KO:K19160 mRNA interferase YafO [EC:3.1.-.-]\n",
      "SSO SSO:000025724 Toxin YafO\n",
      "b0233 ['Toxin YafO']\n",
      "\n",
      "KO KO:K03830 putative acetyltransferase [EC:2.3.1.-]\n",
      "b0234 ['Uncharacterized acetyltransferase YafP (EC 2.3.1.-)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0235 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02839 peptide chain release factor\n",
      "SSO SSO:000005793 Peptide chain release factor homolog\n",
      "b0236 ['Peptide chain release factor homolog']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01270 dipeptidase D [EC:3.4.13.-]\n",
      "SSO SSO:000001971 Cytosol nonspecific dipeptidase (EC 3.4.13.18)\n",
      "SSO SSO:000042387 Aminoacyl-histidine dipeptidase (Peptidase D) (EC 3.4.13.3)\n",
      "b0237 ['Cytosol nonspecific dipeptidase (EC 3.4.13.18)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00769 xanthine phosphoribosyltransferase [EC:2.4.2.22]\n",
      "SSO SSO:000008962 Xanthine-guanine phosphoribosyltransferase (EC 2.4.2.22)\n",
      "b0238 ['Xanthine-guanine phosphoribosyltransferase (EC 2.4.2.22)']\n",
      "\n",
      "KO KO:K11750 esterase FrsA [EC:3.1.-.-]\n",
      "SSO SSO:000002701 Fermentation/respiration switch protein\n",
      "b0239 ['Fermentation/respiration switch protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009289 pilus\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K11926 sigma factor-binding protein Crl\n",
      "SSO SSO:000001797 Curlin genes transcriptional activator\n",
      "b0240 ['Curlin genes transcriptional activator']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K11929 outer membrane pore protein E\n",
      "SSO SSO:000005579 Outer membrane porin PhoE\n",
      "b0241 ['Outer membrane porin PhoE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "KO KO:K00931 glutamate 5-kinase [EC:2.7.2.11]\n",
      "b0242 ['Glutamate 5-kinase (EC 2.7.2.11)', 'RNA-binding C-terminal domain PUA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "KO KO:K00147 glutamate-5-semialdehyde dehydrogenase [EC:1.2.1.41]\n",
      "SSO SSO:000003030 Gamma-glutamyl phosphate reductase (EC 1.2.1.41)\n",
      "b0243 ['Gamma-glutamyl phosphate reductase (EC 1.2.1.41)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4626 ['hypothetical protein']\n",
      "\n",
      "KO KO:K18837 cytoskeleton-binding toxin CbtA and related proteins\n",
      "SSO SSO:000025726 Toxin YkfI\n",
      "b0245 ['Toxin YkfI']\n",
      "\n",
      "KO KO:K18838 cytoskeleton bundling-enhancing protein CbeA and related proteins\n",
      "SSO SSO:000010732 Antitoxin YafW (binds toxin YkfI)\n",
      "b0246 ['Antitoxin YafW (binds toxin YkfI)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "b4504 ['Uncharacterized protein YkfH']\n",
      "\n",
      "KO KO:K03630 DNA repair protein RadC\n",
      "SSO SSO:000026715 UPF0758 protein YkfG\n",
      "b0247 ['UPF0758 protein YkfG']\n",
      "\n",
      "SSO SSO:000010725 Antirestriction protein klcA\n",
      "b0248 ['Antirestriction protein klcA']\n",
      "\n",
      "KO KO:K18837 cytoskeleton-binding toxin CbtA and related proteins\n",
      "SSO SSO:000026672 UPF0401 protein YkfF\n",
      "b0249 ['UPF0401 protein YkfF']\n",
      "\n",
      "SSO SSO:000028427 Uncharacterized protein YkfB\n",
      "b0250 ['Uncharacterized protein YkfB']\n",
      "\n",
      "SSO SSO:000027201 Uncharacterized lipoprotein YafY\n",
      "b0251 ['Uncharacterized lipoprotein YafY']\n",
      "\n",
      "b4627 ['predicted inner membrane protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4628 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000026658 UPF0380 proteins YafZ and homologs\n",
      "b0252 ['UPF0380 proteins YafZ and homologs']\n",
      "\n",
      "KO KO:K06946 uncharacterized protein\n",
      "SSO SSO:000028426 Uncharacterized protein YkfA\n",
      "b0253 ['Uncharacterized protein YkfA']\n",
      "\n",
      "KO KO:K03566 LysR family transcriptional regulator, glycine cleavage system transcriptional activator\n",
      "SSO SSO:000039623 Transcriptional regulator PerR, LysR family\n",
      "b0254 ['Transcriptional regulator PerR, LysR family']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4587 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07482 transposase, IS30 family\n",
      "SSO SSO:000026099 Transposase InsI for insertion sequence element IS30\n",
      "b0256 ['Transposase InsI for insertion sequence element IS30']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0258 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b0259 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16235 S-methylmethionine transporter\n",
      "SSO SSO:000007206 S-methylmethionine permease\n",
      "b0260 ['S-methylmethionine permease']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "KO KO:K00547 homocysteine S-methyltransferase [EC:2.1.1.10]\n",
      "SSO SSO:000003478 Homocysteine S-methyltransferase (EC 2.1.1.10)\n",
      "b0261 ['Homocysteine S-methyltransferase (EC 2.1.1.10)']\n",
      "\n",
      "KO KO:K02010 iron(III) transport system ATP-binding protein [EC:3.6.3.30]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "b0262 ['Ferric cations import ATP-binding protein fbpC (EC 3.6.3.30)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0263 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07480 insertion element IS1 protein InsB\n",
      "SSO SSO:000018188 IS1 protein InsB\n",
      "b0264 ['IS1 protein InsB']\n",
      "\n",
      "SSO SSO:000018187 IS1 protein InsA\n",
      "b0265 ['IS1 protein InsA']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4708 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4505 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0266 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000028804 Uncharacterized transposase YagA\n",
      "b0267 ['Uncharacterized transposase YagA']\n",
      "\n",
      "KO KO:K01714 4-hydroxy-tetrahydrodipicolinate synthase [EC:4.3.3.7]\n",
      "KO KO:K22397 2-dehydro-3-deoxy-D-pentonate aldolase [EC:4.1.2.28]\n",
      "b0268 ['Uncharacterized lyase YjhH']\n",
      "\n",
      "KO KO:K22396 xylonate dehydratase [EC:4.2.1.82]\n",
      "b0269 ['Uncharacterized dehydratase YagF']\n",
      "\n",
      "KO KO:K03292 glycoside/pentoside/hexuronide:cation symporter, GPH family\n",
      "SSO SSO:000022665 Proposed xyloside/Na+ symporter YagG\n",
      "b0270 ['Proposed xyloside/Na+ symporter YagG']\n",
      "\n",
      "EC EC:3.2.1.145 Galactan 1,3-beta-galactosidase.\n",
      "EC EC:3.2.1.146 Beta-galactofuranosidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.55 Non-reducing end alpha-L-arabinofuranosidase.\n",
      "EC EC:3.2.1.8 Endo-1,4-beta-xylanase.\n",
      "EC EC:3.2.1.99 Arabinan endo-1,5-alpha-L-arabinosidase.\n",
      "KO KO:K01198 xylan 1,4-beta-xylosidase [EC:3.2.1.37]\n",
      "SSO SSO:000029062 Xylan 1,4-beta-xylosidase (EC 3.2.1.37)\n",
      "b0271 ['Xylan 1,4-beta-xylosidase (EC 3.2.1.37)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K13641 IclR family transcriptional regulator, acetate operon repressor\n",
      "KO KO:K19333 IclR family transcriptional regulator, KDG regulon repressor\n",
      "b0272 ['Uncharacterized transcriptional regulator YagI, IclR family']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K00611 ornithine carbamoyltransferase [EC:2.1.3.3]\n",
      "SSO SSO:000005514 Ornithine carbamoyltransferase (EC 2.1.3.3)\n",
      "b0273 ['Ornithine carbamoyltransferase (EC 2.1.3.3)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4688 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07480 insertion element IS1 protein InsB\n",
      "SSO SSO:000018188 IS1 protein InsB\n",
      "b0274 ['IS1 protein InsB']\n",
      "\n",
      "SSO SSO:000018187 IS1 protein InsA\n",
      "b0275 ['IS1 protein InsA']\n",
      "\n",
      "SSO SSO:000023693 Putative uncharacterized protein YagJ\n",
      "b0276 ['Putative uncharacterized protein YagJ']\n",
      "\n",
      "SSO SSO:000028149 Uncharacterized protein YagK\n",
      "b0277 ['Uncharacterized protein YagK']\n",
      "\n",
      "SSO SSO:000028685 Uncharacterized resolvase YagL\n",
      "b0278 ['Uncharacterized resolvase YagL']\n",
      "\n",
      "SSO SSO:000028150 Uncharacterized protein YagM\n",
      "b0279 ['Uncharacterized protein YagM']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0280 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000022651 Prophage CP4-6 integrase\n",
      "b0281 ['Prophage CP4-6 integrase']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4694 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4695 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07402 xanthine dehydrogenase accessory factor\n",
      "SSO SSO:000028151 Uncharacterized protein YagQ\n",
      "b0283 ['Uncharacterized protein YagQ']\n",
      "\n",
      "KO KO:K11177 xanthine dehydrogenase YagR molybdenum-binding subunit [EC:1.17.1.4]\n",
      "SSO SSO:000005834 Periplasmic aromatic aldehyde oxidoreductase, molybdenum binding subunit YagR\n",
      "b0284 ['Periplasmic aromatic aldehyde oxidoreductase, molybdenum binding subunit YagR']\n",
      "\n",
      "KO KO:K11178 xanthine dehydrogenase YagS FAD-binding subunit [EC:1.17.1.4]\n",
      "SSO SSO:000005832 Periplasmic aromatic aldehyde oxidoreductase, FAD binding subunit YagS\n",
      "b0285 ['Periplasmic aromatic aldehyde oxidoreductase, FAD binding subunit YagS']\n",
      "\n",
      "KO KO:K13483 xanthine dehydrogenase YagT iron-sulfur-binding subunit\n",
      "SSO SSO:000005833 Periplasmic aromatic aldehyde oxidoreductase, iron-sulfur subunit YagT\n",
      "b0286 ['Periplasmic aromatic aldehyde oxidoreductase, iron-sulfur subunit YagT']\n",
      "\n",
      "KO KO:K08996 putative membrane protein\n",
      "SSO SSO:000012967 DUF1440 domain-containing membrane protein\n",
      "b0287 ['DUF1440 domain-containing membrane protein']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K06940 uncharacterized protein\n",
      "SSO SSO:000002702 Ferredoxin (EC 1.18.1.3 )\n",
      "b0288 ['Ferredoxin']\n",
      "\n",
      "KO KO:K21968 Mat/Ecp fimbriae periplasmic chaperone\n",
      "SSO SSO:000011539 CFA/I fimbrial chaperone\n",
      "b0289 ['CFA/I fimbrial chaperone']\n",
      "\n",
      "KO KO:K21967 Mat/Ecp fimbriae adhesin\n",
      "SSO SSO:000011541 CFA/I fimbrial minor adhesin\n",
      "b0290 ['CFA/I fimbrial minor adhesin']\n",
      "\n",
      "KO KO:K21966 Mat/Ecp fimbriae outer membrane usher protein\n",
      "SSO SSO:000011542 CFA/I fimbrial subunit C usher protein\n",
      "b0291 ['CFA/I fimbrial subunit C usher protein']\n",
      "\n",
      "KO KO:K21965 Mat/Ecp fimbriae periplasmic chaperone\n",
      "SSO SSO:000011538 CFA/I fimbrial auxiliary subunit\n",
      "b0292 ['CFA/I fimbrial auxiliary subunit']\n",
      "\n",
      "KO KO:K21964 Mat/Ecp fimbriae major subunit\n",
      "SSO SSO:000011540 CFA/I fimbrial major subunit\n",
      "b0293 ['CFA/I fimbrial major subunit']\n",
      "\n",
      "KO KO:K21963 LuxR family transcriptional regulator, Mat/Ecp fimbriae transcriptional regulator\n",
      "SSO SSO:000039580 Transcriptional regulator EcpR\n",
      "b0294 ['Transcriptional regulator EcpR']\n",
      "\n",
      "SSO SSO:000028429 Uncharacterized protein YkgL\n",
      "b0295 ['Uncharacterized protein YkgL']\n",
      "\n",
      "KO KO:K02919 large subunit ribosomal protein L36\n",
      "b4506 ['LSU ribosomal protein L36p', 'LSU ribosomal protein L36p, zinc-independent']\n",
      "\n",
      "KO KO:K02909 large subunit ribosomal protein L31\n",
      "b0296 ['LSU ribosomal protein L31p', 'LSU ribosomal protein L31p, zinc-independent']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4671 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4630 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0297 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026096 Transposase InsE for insertion sequence IS3E\n",
      "b0298 ['Transposase InsE for insertion sequence IS3E']\n",
      "\n",
      "KO KO:K07497 putative transposase\n",
      "b0299 ['Transposase InsF for insertion sequence IS3', 'Mobile element protein']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0300 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4631 ['hypothetical protein']\n",
      "\n",
      "KO KO:K21741 reactive chlorine resistance protein C\n",
      "SSO SSO:000018377 Inner membrane protein RclC associated with response to reactive chlorine species\n",
      "b0301 ['Inner membrane protein RclC associated with response to reactive chlorine species']\n",
      "\n",
      "KO KO:K21740 reactive chlorine resistance protein B\n",
      "SSO SSO:000027737 Uncharacterized protein RclB associated with response to reactive chlorine species\n",
      "b0303 ['Uncharacterized protein RclB associated with response to reactive chlorine species']\n",
      "\n",
      "KO KO:K21739 probable pyridine nucleotide-disulfide oxidoreductase\n",
      "b0304 ['Putative Dihydrolipoamide dehydrogenase (EC 1.8.1.4)', 'Mercuric ion reductase (EC 1.16.1.1)', 'PF00070 family, FAD-dependent NAD(P)-disulphide oxidoreductase']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K21746 AraC family transcriptional regulator, reactive chlorine species (RCS)-specific activator of rcl operon\n",
      "SSO SSO:000023903 Reactive-chlorine-species-specific transcriptional activator RclR, AraC family\n",
      "b0305 ['Reactive-chlorine-species-specific transcriptional activator RclR, AraC family']\n",
      "\n",
      "KO KO:K18928 L-lactate dehydrogenase complex protein LldE\n",
      "SSO SSO:000022100 Predicted L-lactate dehydrogenase, Fe-S oxidoreductase subunit YkgE\n",
      "b0306 ['Predicted L-lactate dehydrogenase, Fe-S oxidoreductase subunit YkgE']\n",
      "\n",
      "KO KO:K18929 L-lactate dehydrogenase complex protein LldF\n",
      "SSO SSO:000022101 Predicted L-lactate dehydrogenase, Iron-sulfur cluster-binding subunit YkgF\n",
      "b0307 ['Predicted L-lactate dehydrogenase, Iron-sulfur cluster-binding subunit YkgF']\n",
      "\n",
      "KO KO:K00782 L-lactate dehydrogenase complex protein LldG\n",
      "SSO SSO:000022103 Predicted L-lactate dehydrogenase, hypothetical protein subunit YkgG\n",
      "b0308 ['Predicted L-lactate dehydrogenase, hypothetical protein subunit YkgG']\n",
      "\n",
      "SSO SSO:000028428 Uncharacterized protein YkgH\n",
      "b0310 ['Uncharacterized protein YkgH']\n",
      "\n",
      "EC EC:1.1.3.10 Pyranose oxidase.\n",
      "EC EC:1.1.3.13 Alcohol oxidase.\n",
      "EC EC:1.1.3.4 Glucose oxidase.\n",
      "EC EC:1.1.3.7 Aryl-alcohol oxidase.\n",
      "EC EC:1.1.99.18 Cellobiose dehydrogenase (acceptor).\n",
      "GO GO:0006578 amino-acid betaine biosynthetic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00108 choline dehydrogenase [EC:1.1.99.1]\n",
      "SSO SSO:000001541 Choline dehydrogenase (EC 1.1.99.1)\n",
      "b0311 ['Choline dehydrogenase (EC 1.1.99.1)']\n",
      "\n",
      "GO GO:0006578 amino-acid betaine biosynthetic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K00130 betaine-aldehyde dehydrogenase [EC:1.2.1.8]\n",
      "SSO SSO:000001198 Betaine aldehyde dehydrogenase (EC 1.2.1.8)\n",
      "b0312 ['Betaine aldehyde dehydrogenase (EC 1.2.1.8)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02167 TetR/AcrR family transcriptional regulator, transcriptional repressor of bet genes\n",
      "SSO SSO:000039554 Transcriptional regulator BetI, TetR family\n",
      "b0313 ['Transcriptional regulator BetI, TetR family']\n",
      "\n",
      "GO GO:0006578 amino-acid betaine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02168 choline/glycine/proline betaine transport protein\n",
      "SSO SSO:000003435 High-affinity choline uptake protein BetT\n",
      "b0314 ['High-affinity choline uptake protein BetT']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K13244 c-di-GMP-specific phosphodiesterase [EC:3.1.4.52]\n",
      "b0315 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeL']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K10972 LysR family transcriptional regulator, transcriptional activator of the allD operon\n",
      "SSO SSO:000019101 LysR family transcriptional regulator YahB\n",
      "b0316 ['LysR family transcriptional regulator YahB']\n",
      "\n",
      "SSO SSO:000028152 Uncharacterized protein YahC\n",
      "b0317 ['Uncharacterized protein YahC']\n",
      "\n",
      "KO KO:K06867 uncharacterized protein\n",
      "SSO SSO:000010666 Ankyrin repeat protein\n",
      "SSO SSO:000023184 Putative ankyrin repeat protein YahD\n",
      "b0318 ['Putative ankyrin repeat protein YahD']\n",
      "\n",
      "SSO SSO:000028153 Uncharacterized protein YahE\n",
      "b0319 ['Uncharacterized protein YahE']\n",
      "\n",
      "KO KO:K02381 FdrA protein\n",
      "SSO SSO:000028154 Uncharacterized protein YahF, possible CoA ligase\n",
      "b0320 ['Uncharacterized protein YahF, possible CoA ligase']\n",
      "\n",
      "SSO SSO:000028155 Uncharacterized protein YahG\n",
      "b0321 ['Uncharacterized protein YahG']\n",
      "\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K00926 carbamate kinase [EC:2.7.2.2]\n",
      "SSO SSO:000034936 Carbamate kinase-like protein YahI\n",
      "b0323 ['Carbamate kinase-like protein YahI']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K01485 cytosine/creatinine deaminase [EC:3.5.4.1 3.5.4.21]\n",
      "SSO SSO:000023248 Putative deaminase YahJ\n",
      "b0324 ['Putative deaminase YahJ']\n",
      "\n",
      "KO KO:K13979 uncharacterized zinc-type alcohol dehydrogenase-like protein [EC:1.-.-.-]\n",
      "b0325 ['Cinnamyl alcohol dehydrogenase/reductase (EC 1.1.1.195)', 'Alcohol dehydrogenase (EC 1.1.1.1)']\n",
      "\n",
      "SSO SSO:000028156 Uncharacterized protein YahL\n",
      "b0326 ['Uncharacterized protein YahL']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0327 ['hypothetical protein']\n",
      "\n",
      "KO KO:K03329 amino acid exporter\n",
      "SSO SSO:000027255 Uncharacterized membrane protein YahN\n",
      "b0328 ['Uncharacterized membrane protein YahN']\n",
      "\n",
      "b0329 ['FIG00639237: hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K02688 transcriptional regulator, propionate catabolism operon regulatory protein\n",
      "SSO SSO:000006398 Propionate catabolism operon regulatory protein PrpR\n",
      "b0330 ['Propionate catabolism operon regulatory protein PrpR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K03417 methylisocitrate lyase [EC:4.1.3.30]\n",
      "SSO SSO:000004883 Methylisocitrate lyase (EC 4.1.3.30)\n",
      "b0331 ['Methylisocitrate lyase (EC 4.1.3.30)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01659 2-methylcitrate synthase [EC:2.3.3.5]\n",
      "SSO SSO:000000195 2-methylcitrate synthase (EC 2.3.3.5)\n",
      "b0333 ['2-methylcitrate synthase (EC 2.3.3.5)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01720 2-methylcitrate dehydratase [EC:4.2.1.79]\n",
      "SSO SSO:000000194 2-methylcitrate dehydratase (EC 4.2.1.79)\n",
      "b0334 ['2-methylcitrate dehydratase (EC 4.2.1.79)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01908 propionyl-CoA synthetase [EC:6.2.1.17]\n",
      "SSO SSO:000006402 Propionate--CoA ligase (EC 6.2.1.17)\n",
      "b0335 ['Propionate--CoA ligase (EC 6.2.1.17)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10974 cytosine permease\n",
      "SSO SSO:000001968 Cytosine permease\n",
      "b0336 ['Cytosine permease']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01485 cytosine/creatinine deaminase [EC:3.5.4.1 3.5.4.21]\n",
      "SSO SSO:000001967 Cytosine deaminase (EC 3.5.4.1)\n",
      "b0337 ['Cytosine deaminase (EC 3.5.4.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K11921 LysR family transcriptional regulator, cyn operon transcriptional activator\n",
      "SSO SSO:000001846 Cyn operon transcriptional activator\n",
      "b0338 ['Cyn operon transcriptional activator']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009440 cyanate catabolic process\n",
      "KO KO:K01673 carbonic anhydrase [EC:4.2.1.1]\n",
      "SSO SSO:000001419 Carbonic anhydrase (EC 4.2.1.1)\n",
      "SSO SSO:000001421 Carbonic anhydrase, beta class (EC 4.2.1.1)\n",
      "b0339 ['Carbonic anhydrase, beta class (EC 4.2.1.1)']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009440 cyanate catabolic process\n",
      "KO KO:K01725 cyanate lyase [EC:4.2.1.104]\n",
      "SSO SSO:000001802 Cyanate hydratase (EC 4.2.1.104)\n",
      "b0340 ['Cyanate hydratase (EC 4.2.1.104)']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009440 cyanate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03449 MFS transporter, CP family, cyanate transporter\n",
      "SSO SSO:000001803 Cyanate transport protein CynX\n",
      "b0341 ['Cyanate transport protein CynX']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00633 galactoside O-acetyltransferase [EC:2.3.1.18]\n",
      "KO KO:K00661 maltose O-acetyltransferase [EC:2.3.1.79]\n",
      "SSO SSO:000003018 Galactoside O-acetyltransferase (EC 2.3.1.18)\n",
      "b0342 ['Galactoside O-acetyltransferase (EC 2.3.1.18)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02532 MFS transporter, OHS family, lactose permease\n",
      "SSO SSO:000004385 Lactose permease\n",
      "b0343 ['Lactose permease']\n",
      "\n",
      "EC EC:3.2.1.146 Beta-galactofuranosidase.\n",
      "EC EC:3.2.1.152 Mannosylglycoprotein endo-beta-mannosidase.\n",
      "EC EC:3.2.1.165 Exo-1,4-beta-D-glucosaminidase.\n",
      "EC EC:3.2.1.23 Beta-galactosidase.\n",
      "EC EC:3.2.1.25 Beta-mannosidase.\n",
      "EC EC:3.2.1.31 Beta-glucuronidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.55 Non-reducing end alpha-L-arabinofuranosidase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01190 beta-galactosidase [EC:3.2.1.23]\n",
      "SSO SSO:000001182 Beta-galactosidase (EC 3.2.1.23)\n",
      "b0344 ['beta-galactosidase (EC 3.2.1.23)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "SSO SSO:000008279 Transcriptional repressor of the lac operon\n",
      "b0345 ['Transcriptional repressor of the lac operon']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K05818 IclR family transcriptional regulator, mhp operon transcriptional activator\n",
      "SSO SSO:000019669 Mhp operon transcriptional activator\n",
      "b0346 ['Mhp operon transcriptional activator']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K05712 3-(3-hydroxy-phenyl)propionate hydroxylase [EC:1.14.13.127]\n",
      "SSO SSO:000009599 3-(3-hydroxy-phenyl)propionate hydroxylase (EC 1.14.13.-)\n",
      "SSO SSO:000034039 3-(3-hydroxyphenyl)propanoate hydroxylase (EC 1.14.13.127)\n",
      "b0347 ['3-(3-hydroxyphenyl)propanoate hydroxylase (EC 1.14.13.127)']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K05713 2,3-dihydroxyphenylpropionate 1,2-dioxygenase [EC:1.13.11.16]\n",
      "SSO SSO:000009501 2,3-dihydroxyphenylpropionate 1,2-dioxygenase (EC 1.13.11.-)\n",
      "SSO SSO:000009606 3-carboxyethylcatechol 2,3-dioxygenase (EC 1.13.11.16)\n",
      "b0348 ['3-carboxyethylcatechol 2,3-dioxygenase (EC 1.13.11.16)']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K05714 2-hydroxy-6-oxonona-2,4-dienedioate hydrolase [EC:3.7.1.14]\n",
      "SSO SSO:000009534 2-hydroxy-6-ketonona-2,4-dienedioic acid hydrolase (EC 3.7.1.-)\n",
      "SSO SSO:000033980 2-hydroxy-6-oxonona-2,4-dienedioate hydrolase (EC 3.7.1.14)\n",
      "b0349 ['2-hydroxy-6-oxonona-2,4-dienedioate hydrolase (EC 3.7.1.14)']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K02554 2-keto-4-pentenoate hydratase [EC:4.2.1.80]\n",
      "SSO SSO:000000176 2-keto-4-pentenoate hydratase (EC 4.2.1.80)\n",
      "b0350 ['2-keto-4-pentenoate hydratase (EC 4.2.1.80)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K04073 acetaldehyde dehydrogenase [EC:1.2.1.10]\n",
      "SSO SSO:000000669 Acetaldehyde dehydrogenase, acetylating, in gene cluster for degradation of phenols, cresols, catechol (EC 1.2.1.10)\n",
      "b0351 ['Acetaldehyde dehydrogenase, acetylating, (EC 1.2.1.10) in gene cluster for degradation of phenols, cresols, catechol']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01666 4-hydroxy 2-oxovalerate aldolase [EC:4.1.3.39]\n",
      "SSO SSO:000000363 4-hydroxy-2-oxovalerate aldolase (EC 4.1.3.39)\n",
      "b0352 ['4-hydroxy-2-oxovalerate aldolase (EC 4.1.3.39)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05819 MFS transporter, AAHS family, 3-hydroxyphenylpropionic acid transporter\n",
      "SSO SSO:000034040 3-(3-hydroxyphenyl)propionate transporter, MFS-type\n",
      "b0353 ['3-(3-hydroxyphenyl)propionate transporter, MFS-type']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K09912 uncharacterized protein\n",
      "SSO SSO:000020650 Nucleoprotein/polynucleotide-associated enzyme\n",
      "b0354 ['Nucleoprotein/polynucleotide-associated enzyme']\n",
      "\n",
      "EC EC:2.3.1.122 Trehalose O-mycolyltransferase.\n",
      "EC EC:2.3.1.20 Diacylglycerol O-acyltransferase.\n",
      "EC EC:3.1.1.1 Carboxylesterase.\n",
      "EC EC:3.1.1.72 Acetylxylan esterase.\n",
      "EC EC:3.1.1.73 Feruloyl esterase.\n",
      "EC EC:3.1.2.12 S-formylglutathione hydrolase.\n",
      "KO KO:K01070 S-formylglutathione hydrolase [EC:3.1.2.12]\n",
      "SSO SSO:000007202 S-formylglutathione hydrolase (EC 3.1.2.12)\n",
      "b0355 ['S-formylglutathione hydrolase (EC 3.1.2.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K00121 S-(hydroxymethyl)glutathione dehydrogenase / alcohol dehydrogenase [EC:1.1.1.284 1.1.1.1]\n",
      "SSO SSO:000007189 S-(hydroxymethyl)glutathione dehydrogenase (EC 1.1.1.284)\n",
      "b0356 ['S-(hydroxymethyl)glutathione dehydrogenase (EC 1.1.1.284)']\n",
      "\n",
      "KO KO:K23239 Unknown\n",
      "SSO SSO:000002928 FrmR: Negative transcriptional regulator of formaldehyde detoxification operon\n",
      "b0357 ['FrmR: Negative transcriptional regulator of formaldehyde detoxification operon']\n",
      "\n",
      "SSO SSO:000009451 (Outer) membrane protein\n",
      "SSO SSO:000038830 Putative uncharacterized protein YaiO\n",
      "b0358 ['Putative uncharacterized protein YaiO']\n",
      "\n",
      "SSO SSO:000023648 Putative transferase\n",
      "b4579 ['Putative transferase']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026094 Transposase InsC for insertion element IS2\n",
      "b0360 ['Transposase InsC for insertion element IS2']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000026095 Transposase InsD for insertion element IS2\n",
      "b0361 ['Transposase InsD for insertion element IS2']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "SSO SSO:000027104 Uncharacterized glycosyltransferase YaiP\n",
      "b0363 ['Uncharacterized glycosyltransferase YaiP']\n",
      "\n",
      "EC EC:3.5.1.89 N-acetylglucosaminylphosphatidylinositol deacetylase.\n",
      "SSO SSO:000027069 Uncharacterized deacetylase YaiS\n",
      "b0364 ['Uncharacterized deacetylase YaiS']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02051 NitT/TauT family transport system substrate-binding protein\n",
      "KO KO:K15551 taurine transport system substrate-binding protein\n",
      "SSO SSO:000039423 Taurine ABC transporter (EC 7.6.2.7), substrate-binding protein TauA\n",
      "b0365 ['Taurine ABC transporter, substrate-binding protein TauA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K02049 NitT/TauT family transport system ATP-binding protein\n",
      "KO KO:K10831 taurine transport system ATP-binding protein [EC:3.6.3.36]\n",
      "SSO SSO:000039421 Taurine ABC transporter (EC 7.6.2.7), ATP-binding protein TauB\n",
      "b0366 ['Taurine ABC transporter, ATP-binding protein TauB']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02050 NitT/TauT family transport system permease protein\n",
      "KO KO:K15552 taurine transport system permease protein\n",
      "SSO SSO:000039422 Taurine ABC transporter (EC 7.6.2.7), permease protein TauC\n",
      "b0367 ['Taurine ABC transporter, permease protein TauC']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K03119 taurine dioxygenase [EC:1.14.11.17]\n",
      "SSO SSO:000000924 Alpha-ketoglutarate-dependent taurine dioxygenase (EC 1.14.11.17)\n",
      "b0368 ['Alpha-ketoglutarate-dependent taurine dioxygenase (EC 1.14.11.17)']\n",
      "\n",
      "KO KO:K01698 porphobilinogen synthase [EC:4.2.1.24]\n",
      "SSO SSO:000006176 Porphobilinogen synthase (EC 4.2.1.24)\n",
      "b0369 ['Porphobilinogen synthase (EC 4.2.1.24)']\n",
      "\n",
      "SSO SSO:000028158 Uncharacterized protein YaiT, interrupted by IS3E\n",
      "b4580 ['Uncharacterized protein YaiT, interrupted by IS3E']\n",
      "\n",
      "KO KO:K07497 putative transposase\n",
      "b0372 ['Transposase InsF for insertion sequence IS3', 'Mobile element protein']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026096 Transposase InsE for insertion sequence IS3E\n",
      "b0373 ['Transposase InsE for insertion sequence IS3E']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000028159 Uncharacterized protein YaiV\n",
      "b0375 ['Uncharacterized protein YaiV']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K18988 serine-type D-Ala-D-Ala carboxypeptidase/endopeptidase [EC:3.4.16.4 3.4.21.-]\n",
      "SSO SSO:000035325 D-alanyl-D-alanine-carboxypeptidase/endopeptidase AmpH\n",
      "b0376 ['D-alanyl-D-alanine-carboxypeptidase/endopeptidase AmpH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K17938 peptide/bleomycin uptake transporter\n",
      "SSO SSO:000024407 SbmA protein\n",
      "b0377 ['SbmA protein']\n",
      "\n",
      "SSO SSO:000027377 Uncharacterized outer membrane protein YaiW\n",
      "b0378 ['Uncharacterized outer membrane protein YaiW']\n",
      "\n",
      "SSO SSO:000018379 Inner membrane protein YaiY\n",
      "b0379 ['Inner membrane protein YaiY']\n",
      "\n",
      "SSO SSO:000028160 Uncharacterized protein YaiZ\n",
      "b0380 ['Uncharacterized protein YaiZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01921 D-alanine-D-alanine ligase [EC:6.3.2.4]\n",
      "SSO SSO:000001986 D-alanine--D-alanine ligase (EC 6.3.2.4)\n",
      "SSO SSO:000001987 D-alanine--D-alanine ligase A (EC 6.3.2.4)\n",
      "b0381 ['D-alanine--D-alanine ligase (EC 6.3.2.4)']\n",
      "\n",
      "SSO SSO:000034652 Anti-adapter protein IraP\n",
      "b0382 ['Anti-adapter protein IraP']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01077 alkaline phosphatase [EC:3.1.3.1]\n",
      "SSO SSO:000000886 Alkaline phosphatase (EC 3.1.3.1)\n",
      "b0383 ['Alkaline phosphatase (EC 3.1.3.1)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0042594 response to starvation\n",
      "SSO SSO:000021611 Phosphate starvation-inducible protein PsiF\n",
      "b0384 ['Phosphate starvation-inducible protein PsiF']\n",
      "\n",
      "KO KO:K18968 diguanylate cyclase [EC:2.7.7.65]\n",
      "b0385 ['Diguanylate cyclase (EC 2.7.7.65) => DgcC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "KO KO:K00286 pyrroline-5-carboxylate reductase [EC:1.5.1.2]\n",
      "SSO SSO:000006834 Pyrroline-5-carboxylate reductase (EC 1.5.1.2)\n",
      "b0386 ['Pyrroline-5-carboxylate reductase (EC 1.5.1.2)']\n",
      "\n",
      "KO KO:K09768 uncharacterized protein\n",
      "SSO SSO:000006454 Protein YaiI\n",
      "b0387 ['Protein YaiI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K00891 shikimate kinase [EC:2.7.1.71]\n",
      "SSO SSO:000007463 Shikimate kinase II (EC 2.7.1.71)\n",
      "SSO SSO:000007464 Shikimate kinase III (EC 2.7.1.71)\n",
      "b0388 ['Shikimate kinase III (EC 2.7.1.71)']\n",
      "\n",
      "SSO SSO:000006453 Protein YaiA\n",
      "b0389 ['Protein YaiA']\n",
      "\n",
      "KO KO:K14591 protein AroM\n",
      "SSO SSO:000001060 AroM protein\n",
      "b0390 ['AroM protein']\n",
      "\n",
      "KO KO:K09913 purine/pyrimidine-nucleoside phosphorylase [EC:2.4.2.1 2.4.2.2]\n",
      "SSO SSO:000038857 Pyrimidine/purine nucleoside phosphorylase PpnP\n",
      "b0391 ['Pyrimidine/purine nucleoside phosphorylase PpnP']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0392 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K03554 recombination associated protein RdgC\n",
      "SSO SSO:000035421 DNA recombination-dependent growth factor RdgC\n",
      "b0393 ['DNA recombination-dependent growth factor RdgC']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000035215 Cryptic sugar kinase Mak\n",
      "b0394 ['Cryptic sugar kinase Mak']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08156 MFS transporter, DHA1 family, arabinose polymer utilization protein\n",
      "b0396 ['Protein AraJ precursor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K03546 DNA repair protein SbcC/Rad50\n",
      "SSO SSO:000002627 Exonuclease sbcC (EC 3.1.11.-)\n",
      "b0397 ['Exonuclease SbcC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K03547 DNA repair protein SbcD/Mre11\n",
      "SSO SSO:000002626 Exonuclease SbcD\n",
      "b0398 ['Exonuclease SbcD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07657 two-component system, OmpR family, phosphate regulon response regulator PhoB\n",
      "SSO SSO:000005922 Phosphate regulon transcriptional regulatory protein PhoB (SphR)\n",
      "b0399 ['Phosphate regulon transcriptional regulatory protein PhoB (SphR)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07636 two-component system, OmpR family, phosphate regulon sensor histidine kinase PhoR [EC:2.7.13.3]\n",
      "SSO SSO:000005921 Phosphate regulon sensor protein PhoR (SphS) (EC 2.7.13.3)\n",
      "b0400 ['Phosphate regulon sensor protein PhoR (SphS) (EC 2.7.13.3)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03311 branched-chain amino acid:cation transporter, LIVCS family\n",
      "SSO SSO:000037126 Na(+)-dependent branched-chain amino acid transporter\n",
      "b0401 ['Na(+)-dependent branched-chain amino acid transporter']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11736 proline-specific permease ProY\n",
      "SSO SSO:000006368 Proline-specific permease proY\n",
      "SSO SSO:000022634 Proline-specific permease\n",
      "b0402 ['Proline-specific permease ProY']\n",
      "\n",
      "EC EC:2.4.1.18 1,4-alpha-glucan branching enzyme.\n",
      "EC EC:2.4.1.19 Cyclomaltodextrin glucanotransferase.\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "EC EC:2.4.1.4 Amylosucrase.\n",
      "EC EC:2.4.1.7 Sucrose phosphorylase.\n",
      "EC EC:2.4.99.16 Starch synthase (maltosyl-transferring).\n",
      "EC EC:3.2.1.1 Alpha-amylase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.116 Glucan 1,4-alpha-maltotriohydrolase.\n",
      "EC EC:3.2.1.133 Glucan 1,4-alpha-maltohydrolase.\n",
      "EC EC:3.2.1.135 Neopullulanase.\n",
      "EC EC:3.2.1.141 4-alpha-D-((1->4)-alpha-D-glucano)trehalose trehalohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.33 Amylo-alpha-1,6-glucosidase.\n",
      "EC EC:3.2.1.41 Pullulanase.\n",
      "EC EC:3.2.1.54 Cyclomaltodextrinase.\n",
      "EC EC:3.2.1.60 Glucan 1,4-alpha-maltotetraohydrolase.\n",
      "EC EC:3.2.1.68 Isoamylase.\n",
      "EC EC:3.2.1.70 Glucan 1,6-alpha-glucosidase.\n",
      "EC EC:3.2.1.93 Alpha,alpha-phosphotrehalase.\n",
      "EC EC:3.2.1.98 Glucan 1,4-alpha-maltohexaosidase.\n",
      "EC EC:5.4.99.11 Isomaltulose synthase.\n",
      "EC EC:5.4.99.15 (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase.\n",
      "EC EC:5.4.99.16 Maltose alpha-D-glucosyltransferase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000019307 Maltodextrin glucosidase (EC 3.2.1.20)\n",
      "b0403 ['Maltodextrin glucosidase (EC 3.2.1.20)']\n",
      "\n",
      "KO KO:K08682 acyl carrier protein phosphodiesterase [EC:3.1.4.14]\n",
      "SSO SSO:000009032 acyl carrier protein phosphodiesterase (EC 3.1.4.14 )\n",
      "b0404 ['Acyl carrier protein phosphodiesterase (EC 3.1.4.14)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K07568 S-adenosylmethionine:tRNA ribosyltransferase-isomerase [EC:2.4.99.17]\n",
      "SSO SSO:000024260 S-adenosylmethionine:tRNA ribosyltransferase-isomerase (EC 2.4.99.17)\n",
      "b0405 ['S-adenosylmethionine:tRNA ribosyltransferase-isomerase (EC 2.4.99.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00773 queuine tRNA-ribosyltransferase [EC:2.4.2.29]\n",
      "SSO SSO:000009398 tRNA-guanine transglycosylase (EC 2.4.2.29)\n",
      "b0406 ['Queuine tRNA-ribosyltransferase (EC 2.4.2.29)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03210 preprotein translocase subunit YajC\n",
      "SSO SSO:000022920 Protein translocase subunit YajC\n",
      "b0407 ['Protein translocase subunit YajC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03072 preprotein translocase subunit SecD\n",
      "SSO SSO:000006523 Protein translocase subunit SecD\n",
      "b0408 ['Protein translocase subunit SecD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03074 preprotein translocase subunit SecF\n",
      "SSO SSO:000006525 Protein translocase subunit SecF\n",
      "b0409 ['Protein translocase subunit SecF']\n",
      "\n",
      "SSO SSO:000028161 Uncharacterized protein YajD\n",
      "b0410 ['Uncharacterized protein YajD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K05517 nucleoside-specific channel-forming protein\n",
      "b0411 ['Nucleoside-specific channel-forming protein Tsx precursor']\n",
      "\n",
      "SSO SSO:000018012 Hypothetical lipoprotein yajI\n",
      "SSO SSO:000020814 putative lipoprotein\n",
      "b0412 ['Hypothetical lipoprotein yajI']\n",
      "\n",
      "KO KO:K07738 transcriptional repressor NrdR\n",
      "SSO SSO:000007106 Ribonucleotide reductase transcriptional regulator NrdR\n",
      "b0413 ['Ribonucleotide reductase transcriptional regulator NrdR']\n",
      "\n",
      "KO KO:K11752 diaminohydroxyphosphoribosylaminopyrimidine deaminase / 5-amino-6-(5-phosphoribosylamino)uracil reductase [EC:3.5.4.26 1.1.1.193]\n",
      "b0414 ['Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)', '5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)']\n",
      "\n",
      "KO KO:K00794 6,7-dimethyl-8-ribityllumazine synthase [EC:2.5.1.78]\n",
      "SSO SSO:000000440 6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)\n",
      "b0415 ['6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03625 N utilization substance protein B\n",
      "SSO SSO:000008178 Transcription termination protein NusB\n",
      "SSO SSO:000039514 Transcription antitermination protein\n",
      "b0416 ['Transcription termination protein NusB']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K00946 thiamine-monophosphate kinase [EC:2.7.4.16]\n",
      "SSO SSO:000008014 Thiamine-monophosphate kinase (EC 2.7.4.16)\n",
      "b0417 ['Thiamine-monophosphate kinase (EC 2.7.4.16)']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01095 phosphatidylglycerophosphatase A [EC:3.1.3.27]\n",
      "SSO SSO:000005939 Phosphatidylglycerophosphatase A (EC 3.1.3.27)\n",
      "b0418 ['Phosphatidylglycerophosphatase A (EC 3.1.3.27)']\n",
      "\n",
      "KO KO:K23107 1-deoxyxylulose-5-phosphate synthase [EC:1.1.-.-]\n",
      "b0419 ['NADPH dependent aldo-keto reductase => YajO']\n",
      "\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K01662 1-deoxy-D-xylulose-5-phosphate synthase [EC:2.2.1.7]\n",
      "SSO SSO:000000046 1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)\n",
      "b0420 ['1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00795 farnesyl diphosphate synthase [EC:2.5.1.1 2.5.1.10]\n",
      "b0421 ['(2E,6E)-farnesyl diphosphate synthase (EC 2.5.1.10)', 'Dimethylallyltransferase (EC 2.5.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K03602 exodeoxyribonuclease VII small subunit [EC:3.1.11.6]\n",
      "SSO SSO:000002624 Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)\n",
      "SSO SSO:000013533 Exonuclease VII small subunit\n",
      "b0422 ['Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03151 tRNA uracil 4-sulfurtransferase [EC:2.8.1.4]\n",
      "b0423 ['tRNA 4-thiouridine synthase (EC 2.8.1.4)', 'Rhodanese-like domain required for thiamine synthesis']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K03152 protein deglycase [EC:3.5.1.124]\n",
      "SSO SSO:000002058 DJ-1/YajL/PfpI superfamily, includes chaperone protein YajL (former ThiJ), parkinsonism-associated protein DJ-1, peptidases PfpI, Hsp31\n",
      "b0424 ['DJ-1/YajL/PfpI superfamily, includes chaperone protein YajL (former ThiJ), parkinsonism-associated protein DJ-1, peptidases PfpI, Hsp31']\n",
      "\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "KO KO:K00077 2-dehydropantoate 2-reductase [EC:1.1.1.169]\n",
      "SSO SSO:000000151 2-dehydropantoate 2-reductase (EC 1.1.1.169)\n",
      "b0425 ['2-dehydropantoate 2-reductase (EC 1.1.1.169)']\n",
      "\n",
      "KO KO:K09767 cyclic-di-GMP-binding protein\n",
      "SSO SSO:000026590 UPF0234 protein Yitk\n",
      "b0426 ['UPF0234 protein Yitk']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "SSO SSO:000018438 Inner membrane transport protein YajR\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b0427 ['Inner membrane transport protein YajR']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02257 heme o synthase [EC:2.5.1.141]\n",
      "SSO SSO:000003369 Heme O synthase, protoheme IX farnesyltransferase COX10-CtaB (EC 2.5.1.-)\n",
      "b0428 ['Heme O synthase, protoheme IX farnesyltransferase, COX10-CtaB']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02300 cytochrome o ubiquinol oxidase subunit IV\n",
      "SSO SSO:000001880 Cytochrome O ubiquinol oxidase subunit IV (EC 1.10.3.-)\n",
      "b0429 ['Cytochrome O ubiquinol oxidase subunit IV (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02299 cytochrome o ubiquinol oxidase subunit III\n",
      "SSO SSO:000001879 Cytochrome O ubiquinol oxidase subunit III (EC 1.10.3.-)\n",
      "b0430 ['Cytochrome O ubiquinol oxidase subunit III (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02298 cytochrome o ubiquinol oxidase subunit I [EC:7.1.1.3]\n",
      "SSO SSO:000001877 Cytochrome O ubiquinol oxidase subunit I (EC 1.10.3.-)\n",
      "b0431 ['Cytochrome O ubiquinol oxidase subunit I (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02297 cytochrome o ubiquinol oxidase subunit II [EC:7.1.1.3]\n",
      "SSO SSO:000001878 Cytochrome O ubiquinol oxidase subunit II (EC 1.10.3.-)\n",
      "b0432 ['Cytochrome O ubiquinol oxidase subunit II (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08218 MFS transporter, PAT family, beta-lactamase induction signal transducer AmpG\n",
      "SSO SSO:000000976 AmpG permease\n",
      "b0433 ['AmpG permease']\n",
      "\n",
      "KO KO:K07286 uncharacterized lipoprotein\n",
      "SSO SSO:000003570 Hypothetical lipoprotein YajG precursor\n",
      "SSO SSO:000020814 putative lipoprotein\n",
      "b0434 ['Hypothetical lipoprotein YajG precursor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K05527 BolA family transcriptional regulator, general stress-responsive regulator\n",
      "SSO SSO:000001463 Cell division protein BolA\n",
      "b0435 ['Cell division protein BolA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03545 trigger factor\n",
      "SSO SSO:000001481 Cell division trigger factor (EC 5.2.1.8)\n",
      "b0436 ['Cell division trigger factor (EC 5.2.1.8)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008233 peptidase activity\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K01358 ATP-dependent Clp protease, protease subunit [EC:3.4.21.92]\n",
      "SSO SSO:000034403 ATP-dependent Clp protease proteolytic subunit ClpP (EC 3.4.21.92)\n",
      "b0437 ['ATP-dependent Clp protease proteolytic subunit ClpP (EC 3.4.21.92)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03544 ATP-dependent Clp protease ATP-binding subunit ClpX\n",
      "SSO SSO:000000564 ATP-dependent Clp protease ATP-binding subunit ClpX\n",
      "b0438 ['ATP-dependent Clp protease ATP-binding subunit ClpX']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008233 peptidase activity\n",
      "KO KO:K01338 ATP-dependent Lon protease [EC:3.4.21.53]\n",
      "SSO SSO:000000620 ATP-dependent protease La Type I (EC 3.4.21.53)\n",
      "b0439 ['ATP-dependent protease La (EC 3.4.21.53) Type I']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03530 DNA-binding protein HU-beta\n",
      "SSO SSO:000002139 DNA-binding protein HU-beta\n",
      "b0440 ['DNA-binding protein HU-beta']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03770 peptidyl-prolyl cis-trans isomerase D [EC:5.2.1.8]\n",
      "SSO SSO:000005812 Peptidyl-prolyl cis-trans isomerase PpiD (EC 5.2.1.8)\n",
      "b0441 ['Peptidyl-prolyl cis-trans isomerase PpiD (EC 5.2.1.8)']\n",
      "\n",
      "KO KO:K02237 competence protein ComEA\n",
      "b0442 ['FIG01200701: possible membrane protein']\n",
      "\n",
      "KO KO:K07107 acyl-CoA thioester hydrolase [EC:3.1.2.-]\n",
      "KO KO:K12500 thioesterase III [EC:3.1.2.-]\n",
      "SSO SSO:000009687 4-hydroxybenzoyl-CoA thioesterase family active site\n",
      "b0443 ['4-hydroxybenzoyl-CoA thioesterase family active site']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K06920 7-cyano-7-deazaguanine synthase [EC:6.3.4.20]\n",
      "SSO SSO:000000462 7-cyano-7-deazaguanine synthase (EC 6.3.4.20)\n",
      "b0444 ['7-cyano-7-deazaguanine synthase (EC 6.3.4.20)']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K11925 SgrR family transcriptional regulator\n",
      "SSO SSO:000028696 Uncharacterized substrate-biding protein YbaE\n",
      "b0445 ['Uncharacterized substrate-biding protein YbaE']\n",
      "\n",
      "KO KO:K11938 HMP-PP phosphatase [EC:3.6.1.-]\n",
      "SSO SSO:000003340 HMP-PP hydrolase (pyridoxal phosphatase) Cof, detected in genetic screen for thiamin metabolic genes\n",
      "b0446 ['HMP-PP hydrolase (pyridoxal phosphatase) Cof, detected in genetic screen for thiamin metabolic genes']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03719 Lrp/AsnC family transcriptional regulator, leucine-responsive regulatory protein\n",
      "KO KO:K05800 Lrp/AsnC family transcriptional regulator\n",
      "SSO SSO:000035432 DNA-binding transcriptional activator DecR, AsnC family\n",
      "b0447 ['DNA-binding transcriptional activator DecR, AsnC family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K18889 ATP-binding cassette, subfamily B, multidrug efflux pump\n",
      "SSO SSO:000013305 Efflux ABC transporter, permease/ATP-binding protein MdlA\n",
      "b0448 ['Efflux ABC transporter, permease/ATP-binding protein MdlA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K18890 ATP-binding cassette, subfamily B, multidrug efflux pump\n",
      "SSO SSO:000013306 Efflux ABC transporter, permease/ATP-binding protein MdlB\n",
      "b0449 ['Efflux ABC transporter, permease/ATP-binding protein MdlB']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K04751 nitrogen regulatory protein P-II 1\n",
      "KO KO:K04752 nitrogen regulatory protein P-II 2\n",
      "SSO SSO:000005356 Nitrogen regulatory protein P-II, GlnK\n",
      "b0450 ['Nitrogen regulatory protein P-II, GlnK']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03320 ammonium transporter, Amt family\n",
      "SSO SSO:000000975 Ammonium transporter\n",
      "b0451 ['Ammonium transporter']\n",
      "\n",
      "KO KO:K10805 acyl-CoA thioesterase II [EC:3.1.2.-]\n",
      "SSO SSO:000000794 Acyl-CoA thioesterase II (EC 3.1.2.-)\n",
      "b0452 ['Acyl-CoA thioesterase II (EC 3.1.2.-)']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0042158 lipoprotein biosynthetic process\n",
      "KO KO:K09914 putative lipoprotein\n",
      "SSO SSO:000020875 Outer membrane lipoprotein\n",
      "SSO SSO:000027202 Uncharacterized lipoprotein YbaY\n",
      "b0453 ['Uncharacterized lipoprotein YbaY']\n",
      "\n",
      "KO KO:K07443 methylated-DNA-protein-cysteine methyltransferase related protein\n",
      "SSO SSO:000002061 DNA base-flipping protein\n",
      "b0454 ['DNA base-flipping protein']\n",
      "\n",
      "SSO SSO:000022815 Protein of unknown function DUF1428\n",
      "b0456 ['Protein of unknown function DUF1428']\n",
      "\n",
      "KO KO:K21090 c-di-GMP phosphodiesterase [EC:3.1.4.52]\n",
      "b0457 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeB']\n",
      "\n",
      "SSO SSO:000018419 Inner membrane protein YlaC\n",
      "b0458 ['Inner membrane protein YlaC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00661 maltose O-acetyltransferase [EC:2.3.1.79]\n",
      "SSO SSO:000004702 Maltose O-acetyltransferase (EC 2.3.1.79)\n",
      "b0459 ['Maltose O-acetyltransferase (EC 2.3.1.79)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K05839 haemolysin expression modulating protein\n",
      "SSO SSO:000017624 Haemolysin expression modulating protein\n",
      "b0460 ['Haemolysin expression modulating protein']\n",
      "\n",
      "KO KO:K19162 hha toxicity modulator TomB\n",
      "SSO SSO:000017782 Hha toxicity modulator TomB\n",
      "b0461 ['Hha toxicity modulator TomB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K18138 multidrug efflux pump\n",
      "SSO SSO:000004970 Multidrug efflux system AcrAB-TolC, inner-membrane proton/drug antiporter AcrB (RND type)\n",
      "b0462 ['Multidrug efflux system AcrAB-TolC, inner-membrane proton/drug antiporter AcrB (RND type)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03585 membrane fusion protein, multidrug efflux system\n",
      "SSO SSO:000004971 Multidrug efflux system AcrAB-TolC, membrane fusion component AcrA\n",
      "b0463 ['Multidrug efflux system AcrAB-TolC, membrane fusion component AcrA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03577 TetR/AcrR family transcriptional regulator, acrAB operon repressor\n",
      "SSO SSO:000008237 Transcriptional regulator of acrAB operon, AcrR\n",
      "SSO SSO:000025979 Transcriptional repressor\n",
      "b0464 ['Transcriptional regulator of acrAB operon, AcrR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05802 potassium-dependent mechanosensitive channel\n",
      "SSO SSO:000019393 Mechanosensitive channel MscK\n",
      "b0465 ['Mechanosensitive channel MscK']\n",
      "\n",
      "SSO SSO:000028163 Uncharacterized protein YbaM\n",
      "b0466 ['Uncharacterized protein YbaM']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K04067 primosomal replication protein N''\n",
      "SSO SSO:000022367 Primosomal replication protein N prime prime\n",
      "SSO SSO:000022368 Primosomal replication protein N''\n",
      "b0467 ['Primosomal replication protein N prime prime']\n",
      "\n",
      "KO KO:K09790 uncharacterized protein\n",
      "SSO SSO:000003881 Inner membrane protein YbaN\n",
      "b0468 ['Inner membrane protein YbaN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00759 adenine phosphoribosyltransferase [EC:2.4.2.7]\n",
      "SSO SSO:000000809 Adenine phosphoribosyltransferase (EC 2.4.2.7)\n",
      "b0469 ['Adenine phosphoribosyltransferase (EC 2.4.2.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02343 DNA polymerase III subunit gamma/tau [EC:2.7.7.7]\n",
      "SSO SSO:000002085 DNA polymerase III subunits gamma and tau (EC 2.7.7.7)\n",
      "b0470 ['DNA polymerase III subunits gamma and tau (EC 2.7.7.7)']\n",
      "\n",
      "KO KO:K09747 uncharacterized protein\n",
      "SSO SSO:000005419 Nucleoid-associated protein YaaK\n",
      "b0471 ['Nucleoid-associated protein YaaK']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K06187 recombination protein RecR\n",
      "SSO SSO:000006959 Recombination protein RecR\n",
      "b0472 ['Recombination protein RecR']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04079 molecular chaperone HtpG\n",
      "SSO SSO:000001508 Chaperone protein HtpG\n",
      "b0473 ['Chaperone protein HtpG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009152 purine ribonucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00939 adenylate kinase [EC:2.7.4.3]\n",
      "SSO SSO:000000821 Adenylate kinase (EC 2.7.4.3)\n",
      "b0474 ['Adenylate kinase (EC 2.7.4.3)']\n",
      "\n",
      "KO KO:K01772 protoporphyrin/coproporphyrin ferrochelatase [EC:4.99.1.1 4.99.1.9]\n",
      "SSO SSO:000002760 Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)\n",
      "b0475 ['Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)']\n",
      "\n",
      "EC EC:3.1.1.13 Sterol esterase.\n",
      "EC EC:3.1.1.3 Triacylglycerol lipase.\n",
      "EC EC:3.1.1.7 Acetylcholinesterase.\n",
      "EC EC:3.1.1.8 Cholinesterase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01066 acetyl esterase [EC:3.1.1.-]\n",
      "SSO SSO:000010155 Acetyl esterase\n",
      "b0476 ['Acetyl esterase (EC 3.1.1.-)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00892 inosine kinase [EC:2.7.1.73]\n",
      "SSO SSO:000009748 Inosine-guanosine kinase (EC 2.7.1.73)\n",
      "b0477 ['Inosine-guanosine kinase (EC 2.7.1.73)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03455 monovalent cation:H+ antiporter-2, CPA2 family\n",
      "SSO SSO:000003880 Inner membrane protein YbaL, KefB/KefC family\n",
      "b0478 ['Inner membrane protein YbaL, KefB/KefC family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K08223 MFS transporter, FSR family, fosmidomycin resistance protein\n",
      "SSO SSO:000016884 Fosmidomycin resistance protein\n",
      "b0479 ['Fosmidomycin resistance protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "KO KO:K11751 5'-nucleotidase / UDP-sugar diphosphatase [EC:3.1.3.5 3.6.1.45]\n",
      "b0480 ['UDP-sugar hydrolase (EC 3.6.1.45)', \"5'-nucleotidase (EC 3.1.3.5)\"]\n",
      "\n",
      "KO KO:K03976 Cys-tRNA(Pro)/Cys-tRNA(Cys) deacylase [EC:3.1.1.-]\n",
      "SSO SSO:000001849 Cys-tRNA(Pro) deacylase YbaK\n",
      "b0481 ['Cys-tRNA(Pro) deacylase YbaK']\n",
      "\n",
      "KO KO:K09973 uncharacterized protein\n",
      "SSO SSO:000025731 TraB family protein\n",
      "SSO SSO:000028164 Uncharacterized protein YbaP\n",
      "b0482 ['Uncharacterized protein YbaP']\n",
      "\n",
      "KO KO:K21498 antitoxin HigA-1\n",
      "SSO SSO:000001018 Antitoxin HigA\n",
      "b0483 ['Antitoxin HigA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015662 ATPase activity, coupled to transmembrane movement of ions, phosphorylative mechanism\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01533 Cu2+-exporting ATPase [EC:3.6.3.4]\n",
      "KO KO:K17686 Cu+-exporting ATPase [EC:3.6.3.54]\n",
      "b0484 ['Lead, cadmium, zinc and mercury transporting ATPase (EC 3.6.3.3) (EC 3.6.3.5)', 'Copper-translocating P-type ATPase (EC 3.6.3.4)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01425 glutaminase [EC:3.5.1.2]\n",
      "SSO SSO:000003165 Glutaminase (EC 3.5.1.2)\n",
      "b0485 ['Glutaminase (EC 3.5.1.2)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018439 Inner membrane transport protein YbaT\n",
      "SSO SSO:000023170 Putative amino acid transporter\n",
      "b0486 ['Inner membrane transport protein YbaT']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K19591 MerR family transcriptional regulator, copper efflux regulator\n",
      "SSO SSO:000012468 Copper resistance transcriptional regulator CueR (MerR family)\n",
      "b0487 ['Copper resistance transcriptional regulator CueR (MerR family)']\n",
      "\n",
      "KO KO:K07340 inner membrane protein\n",
      "SSO SSO:000006616 Putative activity regulator of membrane protease YbbK\n",
      "b0488 ['Putative activity regulator of membrane protease YbbK']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000006449 Protein QmcA (possibly involved in integral membrane quality control)\n",
      "b0489 ['Protein QmcA (possibly involved in integral membrane quality control)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02068 putative ABC transport system ATP-binding protein\n",
      "SSO SSO:000006331 Probable iron export ATP-binding protein FetA\n",
      "b0490 ['Probable iron export ATP-binding protein FetA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K02069 putative ABC transport system permease protein\n",
      "SSO SSO:000006332 Probable iron export permease protein FetB\n",
      "b0491 ['Probable iron export permease protein FetB']\n",
      "\n",
      "KO KO:K03671 thioredoxin 1\n",
      "KO KO:K05838 putative thioredoxin\n",
      "b0492 ['FIG000875: Thioredoxin domain-containing protein EC-YbbN']\n",
      "\n",
      "SSO SSO:000006607 Putative NAD(P)-dependent oxidoreductase EC-YbbO\n",
      "b0493 ['Putative NAD(P)-dependent oxidoreductase EC-YbbO']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10804 acyl-CoA thioesterase I [EC:3.1.2.- 3.1.1.5]\n",
      "SSO SSO:000005026 Multifunctional acyl-CoA thioesterase I, protease I, lysophospholipase L1\n",
      "b0494 ['Multifunctional acyl-CoA thioesterase I, protease I, lysophospholipase L1']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02003 putative ABC transport system ATP-binding protein\n",
      "SSO SSO:000008735 Uncharacterized metabolite ABC transporter in Enterobacteriaceae, ATP-binding protein EC-YbbA\n",
      "b0495 ['Uncharacterized metabolite ABC transporter in Enterobacteriaceae, ATP-binding protein EC-YbbA']\n",
      "\n",
      "KO KO:K02004 putative ABC transport system permease protein\n",
      "SSO SSO:000008736 Uncharacterized metabolite ABC transporter in Enterobacteriaceae, permease protein EC-YbbP\n",
      "b0496 ['Uncharacterized metabolite ABC transporter in Enterobacteriaceae, permease protein EC-YbbP']\n",
      "\n",
      "SSO SSO:000022716 Protein RhsD\n",
      "b0497 ['Protein RhsD']\n",
      "\n",
      "SSO SSO:000028165 Uncharacterized protein YbbC\n",
      "b0498 ['Uncharacterized protein YbbC']\n",
      "\n",
      "SSO SSO:000023705 Putative uncharacterized protein YlbH\n",
      "b0499 ['Putative uncharacterized protein YlbH']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0501 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4632 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0502 ['hypothetical protein']\n",
      "\n",
      "EC EC:2.9.1.3 Unknown\n",
      "KO KO:K06917 tRNA 2-selenouridine synthase [EC:2.9.1.-]\n",
      "SSO SSO:000007385 Selenophosphate-dependent tRNA 2-selenouridine synthase\n",
      "b0503 ['Selenophosphate-dependent tRNA 2-selenouridine synthase']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K10972 LysR family transcriptional regulator, transcriptional activator of the allD operon\n",
      "SSO SSO:000039543 Transcriptional regulator AllS, LysR family\n",
      "b0504 ['Transcriptional regulator AllS, LysR family']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009442 allantoin assimilation pathway\n",
      "KO KO:K01483 ureidoglycolate lyase [EC:4.3.2.3]\n",
      "SSO SSO:000008871 Ureidoglycolate lyase (EC 4.3.2.3)\n",
      "b0505 ['Ureidoglycolate lyase (EC 4.3.2.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K10973 IclR family transcriptional regulator, negative regulator of allantoin and glyoxylate utilization operons\n",
      "SSO SSO:000020465 Negative regulator of allantoin and glyoxylate utilization operons\n",
      "b0506 ['Negative regulator of allantoin and glyoxylate utilization operons']\n",
      "\n",
      "GO GO:0009441 glycolate metabolic process\n",
      "GO GO:0009442 allantoin assimilation pathway\n",
      "KO KO:K01608 tartronate-semialdehyde synthase [EC:4.1.1.47]\n",
      "KO KO:K01652 acetolactate synthase I/II/III large subunit [EC:2.2.1.6]\n",
      "SSO SSO:000003322 Glyoxylate carboligase (EC 4.1.1.47)\n",
      "b0507 ['Glyoxylate carboligase (EC 4.1.1.47)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01816 hydroxypyruvate isomerase [EC:5.3.1.22]\n",
      "SSO SSO:000017988 Hydroxypyruvate isomerase (EC 5.3.1.22)\n",
      "b0508 ['Hydroxypyruvate isomerase (EC 5.3.1.22)']\n",
      "\n",
      "GO GO:0009441 glycolate metabolic process\n",
      "GO GO:0009442 allantoin assimilation pathway\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00020 3-hydroxyisobutyrate dehydrogenase [EC:1.1.1.31]\n",
      "KO KO:K00042 2-hydroxy-3-oxopropionate reductase [EC:1.1.1.60]\n",
      "SSO SSO:000000159 2-hydroxy-3-oxopropionate reductase (EC 1.1.1.60)\n",
      "b0509 ['2-hydroxy-3-oxopropionate reductase (EC 1.1.1.60)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03457 nucleobase:cation symporter-1, NCS1 family\n",
      "KO KO:K10975 allantoin permease\n",
      "SSO SSO:000000898 Allantoin permease\n",
      "b0511 ['Allantoin permease']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009442 allantoin assimilation pathway\n",
      "KO KO:K01466 allantoinase [EC:3.5.2.5]\n",
      "SSO SSO:000000900 Allantoinase (EC 3.5.2.5)\n",
      "b0512 ['Allantoinase (EC 3.5.2.5)']\n",
      "\n",
      "SSO SSO:000038815 Putative purine permease YbbY\n",
      "b0513 ['Putative purine permease YbbY']\n",
      "\n",
      "GO GO:0009441 glycolate metabolic process\n",
      "GO GO:0009442 allantoin assimilation pathway\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00865 glycerate 2-kinase [EC:2.7.1.165]\n",
      "SSO SSO:000003240 Glycerate kinase (EC 2.7.1.31)\n",
      "b0514 ['Glycerate kinase (EC 2.7.1.31)']\n",
      "\n",
      "EC EC:3.5.3.26 (S)-ureidoglycine aminohydrolase.\n",
      "KO KO:K14977 (S)-ureidoglycine aminohydrolase [EC:3.5.3.26]\n",
      "SSO SSO:000008870 Ureidoglycine aminohydrolase\n",
      "SSO SSO:000033887 (S)-ureidoglycine aminohydrolase (EC 3.5.3.26)\n",
      "b0515 ['Ureidoglycine aminohydrolase']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009442 allantoin assimilation pathway\n",
      "KO KO:K02083 allantoate deiminase [EC:3.5.3.9]\n",
      "SSO SSO:000000896 Allantoate amidohydrolase (EC 3.5.3.9)\n",
      "b0516 ['Allantoate amidohydrolase (EC 3.5.3.9)']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009442 allantoin assimilation pathway\n",
      "KO KO:K00073 ureidoglycolate dehydrogenase (NAD+) [EC:1.1.1.350]\n",
      "SSO SSO:000028873 Ureidoglycolate dehydrogenase (EC 1.1.1.154)\n",
      "b0517 ['Ureidoglycolate dehydrogenase (EC 1.1.1.154)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02381 FdrA protein\n",
      "SSO SSO:000019430 Membrane protein FdrA\n",
      "b0518 ['Membrane protein FdrA']\n",
      "\n",
      "SSO SSO:000027438 Uncharacterized protein YlbE\n",
      "b4572 ['Uncharacterized protein YlbE']\n",
      "\n",
      "SSO SSO:000028445 Uncharacterized protein YlbF\n",
      "b0520 ['Uncharacterized protein YlbF']\n",
      "\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K00926 carbamate kinase [EC:2.7.2.2]\n",
      "SSO SSO:000001398 Carbamate kinase (EC 2.7.2.2)\n",
      "b0521 ['Carbamate kinase (EC 2.7.2.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K01589 5-(carboxyamino)imidazole ribonucleotide synthase [EC:6.3.4.18]\n",
      "SSO SSO:000005148 N5-carboxyaminoimidazole ribonucleotide synthase (EC 6.3.4.18)\n",
      "b0522 ['N5-carboxyaminoimidazole ribonucleotide synthase (EC 6.3.4.18)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K01588 5-(carboxyamino)imidazole ribonucleotide mutase [EC:5.4.99.18]\n",
      "SSO SSO:000005147 N5-carboxyaminoimidazole ribonucleotide mutase (EC 5.4.99.18)\n",
      "b0523 ['N5-carboxyaminoimidazole ribonucleotide mutase (EC 5.4.99.18)']\n",
      "\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K03269 UDP-2,3-diacylglucosamine hydrolase [EC:3.6.1.54]\n",
      "SSO SSO:000008598 UDP-2,3-diacylglucosamine diphosphatase (EC 3.6.1.54)\n",
      "b0524 ['UDP-2,3-diacylglucosamine diphosphatase (EC 3.6.1.54)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000005813 Peptidyl-prolyl cis-trans isomerase ppiB (EC 5.2.1.8)\n",
      "b0525 ['Peptidyl-prolyl cis-trans isomerase PpiB (EC 5.2.1.8)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01883 cysteinyl-tRNA synthetase [EC:6.1.1.16]\n",
      "SSO SSO:000001868 Cysteinyl-tRNA synthetase (EC 6.1.1.16)\n",
      "b0526 ['Cysteinyl-tRNA synthetase (EC 6.1.1.16)']\n",
      "\n",
      "KO KO:K07038 inner membrane protein\n",
      "SSO SSO:000018380 Inner membrane protein YbcI\n",
      "b0527 ['Inner membrane protein YbcI']\n",
      "\n",
      "KO KO:K14761 ribosome-associated protein\n",
      "SSO SSO:000040031 Uncharacterized S4 RNA-binding-domain protein YbcJ\n",
      "b0528 ['Uncharacterized S4 RNA-binding-domain protein YbcJ']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K01491 methylenetetrahydrofolate dehydrogenase (NADP+) / methenyltetrahydrofolate cyclohydrolase [EC:1.5.1.5 3.5.4.9]\n",
      "b0529 ['Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)', 'Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "KO KO:K07352 type 1 fimbrial protein\n",
      "SSO SSO:000008713 Uncharacterized fimbrial-like protein SfmA\n",
      "b0530 ['Uncharacterized fimbrial-like protein SfmA']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K07346 fimbrial chaperone protein\n",
      "KO KO:K07353 fimbrial chaperone protein\n",
      "SSO SSO:000006323 Probable fimbrial chaperone SfmC\n",
      "b0531 ['Probable fimbrial chaperone SfmC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07347 outer membrane usher protein\n",
      "KO KO:K07354 outer membrane usher protein\n",
      "SSO SSO:000005606 Outer membrane usher protein SfmD\n",
      "b0532 ['Outer membrane usher protein SfmD']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K07356 fimbrial protein\n",
      "SSO SSO:000008715 Uncharacterized fimbrial-like protein SfmH\n",
      "b0533 ['Uncharacterized fimbrial-like protein SfmH']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "KO KO:K07355 fimbrial-like protein\n",
      "SSO SSO:000008714 Uncharacterized fimbrial-like protein SfmF\n",
      "b0534 ['Uncharacterized fimbrial-like protein SfmF']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009289 pilus\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07688 two-component system, NarL family, response regulator, fimbrial Z protein, FimZ\n",
      "SSO SSO:000008242 Transcriptional regulator of fimbriae expression FimZ (LuxR/UhpA family)\n",
      "b0535 ['Transcriptional regulator of fimbriae expression FimZ (LuxR/UhpA family)']\n",
      "\n",
      "KO KO:K03111 single-strand DNA-binding protein\n",
      "SSO SSO:000021513 Phage integrase\n",
      "b0537 ['Phage integrase']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4633 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0539 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4508 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000038008 Phage exclusion protein ren\n",
      "b0542 ['Phage exclusion protein ren']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026096 Transposase InsE for insertion sequence IS3E\n",
      "b0540 ['Transposase InsE for insertion sequence IS3E']\n",
      "\n",
      "KO KO:K07497 putative transposase\n",
      "b0541 ['Transposase InsF for insertion sequence IS3', 'Mobile element protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03297 small multidrug resistance pump\n",
      "b0543 ['Small multidrug resistance (SMR) efflux transporter => EmrE, broad substrate specificity']\n",
      "\n",
      "KO KO:K06400 site-specific DNA recombinase\n",
      "SSO SSO:000028168 Uncharacterized protein YbcK\n",
      "b0544 ['Uncharacterized protein YbcK']\n",
      "\n",
      "SSO SSO:000026498 UPF0098 protein ybcL precursor\n",
      "b0545 ['UPF0098 protein ybcL precursor']\n",
      "\n",
      "SSO SSO:000028743 Uncharacterized transcriptional regulator YbcM, AraC family\n",
      "b0546 ['Uncharacterized transcriptional regulator YbcM, AraC family']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4588 ['hypothetical protein']\n",
      "\n",
      "b0547 ['Uncharacterized protein YbcN']\n",
      "\n",
      "SSO SSO:000038230 Phage protein NinE\n",
      "b0548 ['Phage protein NinE']\n",
      "\n",
      "SSO SSO:000028169 Uncharacterized protein YbcO\n",
      "b0549 ['Uncharacterized protein YbcO']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K01160 crossover junction endodeoxyribonuclease RusA [EC:3.1.22.4]\n",
      "b0550 ['Holliday junction resolvase', 'Crossover junction endodeoxyribonuclease rusA (EC 3.1.22.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000028446 Uncharacterized protein YlcG\n",
      "b4509 ['Uncharacterized protein YlcG']\n",
      "\n",
      "SSO SSO:000021495 Phage antitermination protein Q\n",
      "b0551 ['Phage antitermination protein Q']\n",
      "\n",
      "KO KO:K16076 outer membrane porin protein LC\n",
      "SSO SSO:000020890 Outer membrane porin OmpD\n",
      "b0553 ['Outer membrane porin OmpD']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b0552 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "SSO SSO:000038049 Phage holin/antiholin component S\n",
      "b0554 ['Phage holin/antiholin component S']\n",
      "\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "KO KO:K01185 lysozyme [EC:3.2.1.17]\n",
      "SSO SSO:000037947 Phage lysozyme R (EC 3.2.1.17)\n",
      "b0555 ['Phage lysozyme R (EC 3.2.1.17)']\n",
      "\n",
      "KO KO:K14744 prophage endopeptidase [EC:3.4.-.-]\n",
      "SSO SSO:000038003 Phage endopeptidase Rz\n",
      "b0556 ['Phage endopeptidase Rz']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000038428 Phage-encoded host cell membrane protein Rz1\n",
      "b4510 ['Phage-encoded host cell membrane protein Rz1']\n",
      "\n",
      "SSO SSO:000038437 Phage-encoded virulence determinant Bor\n",
      "b0557 ['Phage-encoded virulence determinant Bor']\n",
      "\n",
      "SSO SSO:000038245 Phage protein YbcV\n",
      "b0558 ['Phage protein YbcV']\n",
      "\n",
      "SSO SSO:000038246 Phage protein YbcW\n",
      "b0559 ['Phage protein YbcW']\n",
      "\n",
      "SSO SSO:000002149 DNA-damage-inducible protein I\n",
      "b4589 ['DNA-damage-inducible protein I']\n",
      "\n",
      "KO KO:K22014 terminase small subunit\n",
      "SSO SSO:000038040 Phage head, terminase subunit Nu1\n",
      "b0560 ['Phage head, terminase subunit Nu1']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4634 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000038366 Phage tail fiber, tail fiber assembly protein Tfa\n",
      "b0561 ['Phage tail fiber, tail fiber assembly protein Tfa']\n",
      "\n",
      "SSO SSO:000023695 Putative uncharacterized protein YbcY\n",
      "b0562 ['Putative uncharacterized protein YbcY']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0563 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042594 response to starvation\n",
      "SSO SSO:000010199 Acid phosphatase transcriptional regulator AppY, AraC family\n",
      "b0564 ['Acid phosphatase transcriptional regulator AppY, AraC family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K01355 omptin [EC:3.4.23.49]\n",
      "b0565 ['Protease VII (Omptin) precursor (EC 3.4.23.49)', 'Outer membrane protease OmpT']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K11920 AraC family transcriptional regulator\n",
      "SSO SSO:000021921 Porin thermoregulatory protein envY\n",
      "b0566 ['Porin thermoregulatory protein envY']\n",
      "\n",
      "SSO SSO:000028166 Uncharacterized protein YbcH\n",
      "b0567 ['Uncharacterized protein YbcH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K11739 bacteriophage N4 adsorption protein A\n",
      "b0568 ['Bacteriophage N4 adsorption protein A']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11740 bacteriophage N4 adsorption protein B\n",
      "b0569 ['Bacteriophage N4 adsorption protein B']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07644 two-component system, OmpR family, heavy metal sensor histidine kinase CusS [EC:2.7.13.3]\n",
      "SSO SSO:000001771 Copper sensory histidine kinase CusS\n",
      "b0570 ['Copper sensory histidine kinase CusS']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07665 two-component system, OmpR family, copper resistance phosphate regulon response regulator CusR\n",
      "SSO SSO:000001776 Copper-sensing two-component system response regulator CusR\n",
      "b0571 ['Copper-sensing two-component system response regulator CusR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07796 outer membrane protein, Cu(I)/Ag(I) efflux system\n",
      "KO KO:K18139 outer membrane protein, multidrug efflux system\n",
      "SSO SSO:000035198 nicotinate N-methyltransferase (EC 2.1.1.7)\n",
      "b0572 ['Copper/silver efflux RND transporter, outer membrane protein CusC']\n",
      "\n",
      "KO KO:K07810 Cu(I)/Ag(I) efflux system periplasmic protein CusF\n",
      "SSO SSO:000035197 Copper/silver efflux RND transporter, periplasmic protein CusF\n",
      "b0573 ['Copper/silver efflux RND transporter, periplasmic protein CusF']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K07798 membrane fusion protein, Cu(I)/Ag(I) efflux system\n",
      "SSO SSO:000035196 Copper/silver efflux RND transporter, membrane fusion protein CusB\n",
      "b0574 ['Copper/silver efflux RND transporter, membrane fusion protein CusB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07787 Cu(I)/Ag(I) efflux system membrane protein CusA/SilA\n",
      "SSO SSO:000035199 Copper/silver efflux RND transporter, transmembrane protein CusA\n",
      "b0575 ['Copper/silver efflux RND transporter, transmembrane protein CusA']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009094 L-phenylalanine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11732 phenylalanine-specific permease\n",
      "SSO SSO:000021586 Phenylalanine-specific permease\n",
      "b0576 ['Phenylalanine-specific permease']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K16053 miniconductance mechanosensitive channel\n",
      "SSO SSO:000036884 Miniconductance mechanosensitive channel YbdG\n",
      "b0577 ['Miniconductance mechanosensitive channel YbdG']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K10679 nitroreductase / dihydropteridine reductase [EC:1.-.-.- 1.5.1.34]\n",
      "b0578 ['Oxygen-insensitive NAD(P)H nitroreductase (EC 1.-.-.-)', 'Dihydropteridine reductase (EC 1.5.1.34)']\n",
      "\n",
      "SSO SSO:000028170 Uncharacterized protein YbdF\n",
      "b0579 ['Uncharacterized protein YbdF']\n",
      "\n",
      "SSO SSO:000027118 Uncharacterized inner membrane protein YbdJ\n",
      "b0580 ['Uncharacterized inner membrane protein YbdJ']\n",
      "\n",
      "KO KO:K06048 glutamate---cysteine ligase / carboxylate-amine ligase [EC:6.3.2.2 6.3.-.-]\n",
      "SSO SSO:000036102 Glutamate--cysteine ligase-like protein YbdK\n",
      "b0581 ['Glutamate--cysteine ligase-like protein YbdK']\n",
      "\n",
      "KO KO:K18922 protein HokE\n",
      "SSO SSO:000017856 HokE protein\n",
      "b4415 ['HokE protein']\n",
      "\n",
      "SSO SSO:000026100 Transposase InsL for insertion sequence element IS186\n",
      "b0582 ['Transposase InsL for insertion sequence element IS186']\n",
      "\n",
      "GO GO:0009239 enterobactin biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02362 enterobactin synthetase component D [EC:6.3.2.14 2.7.8.-]\n",
      "SSO SSO:000000347 4'-phosphopantetheinyl transferase [enterobactin] siderophore (EC 2.7.8.-)\n",
      "b0583 [\"4'-phosphopantetheinyl transferase (EC 2.7.8.-) [enterobactin] siderophore\"]\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "KO KO:K19611 ferric enterobactin receptor\n",
      "b0584 ['TonB-dependent receptor', 'Outer membrane receptor for ferric enterobactin and colicins B, D']\n",
      "\n",
      "EC EC:2.3.1.122 Trehalose O-mycolyltransferase.\n",
      "EC EC:2.3.1.20 Diacylglycerol O-acyltransferase.\n",
      "EC EC:3.1.1.1 Carboxylesterase.\n",
      "EC EC:3.1.1.72 Acetylxylan esterase.\n",
      "EC EC:3.1.1.73 Feruloyl esterase.\n",
      "EC EC:3.1.2.12 S-formylglutathione hydrolase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K07214 enterochelin esterase and related enzymes\n",
      "SSO SSO:000002547 Enterobactin esterase\n",
      "b0585 ['Enterobactin esterase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K05375 MbtH protein\n",
      "SSO SSO:000004727 MbtH-like NRPS chaperone => YbdZ\n",
      "b4511 ['MbtH-like NRPS chaperone => YbdZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009239 enterobactin biosynthetic process\n",
      "GO GO:0015937 coenzyme A biosynthetic process\n",
      "GO GO:0019184 nonribosomal peptide biosynthetic process\n",
      "KO KO:K02364 enterobactin synthetase component F [EC:6.3.2.14]\n",
      "SSO SSO:000002550 Enterobactin synthetase component F, serine activating enzyme (EC 2.7.7.-)\n",
      "b0586 ['Enterobactin synthetase component F, serine activating enzyme (EC 2.7.7.-)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000002726 Ferric enterobactin uptake protein FepE\n",
      "SSO SSO:000042178 regulator of length of O-antigen component of lipopolysaccharide chains\n",
      "b0587 ['Ferric enterobactin uptake protein FepE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K02013 iron complex transport system ATP-binding protein [EC:3.6.3.34]\n",
      "KO KO:K23188 Unknown\n",
      "SSO SSO:000002723 Ferric enterobactin transport ATP-binding protein FepC (TC 3.A.1.14.2)\n",
      "b0588 ['Ferric enterobactin transport ATP-binding protein FepC (TC 3.A.1.14.2)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02015 iron complex transport system permease protein\n",
      "KO KO:K23187 Unknown\n",
      "SSO SSO:000002725 Ferric enterobactin transport system permease protein FepG (TC 3.A.1.14.2)\n",
      "b0589 ['Ferric enterobactin transport system permease protein FepG (TC 3.A.1.14.2)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02015 iron complex transport system permease protein\n",
      "KO KO:K23186 Unknown\n",
      "SSO SSO:000002724 Ferric enterobactin transport system permease protein FepD (TC 3.A.1.14.2)\n",
      "b0590 ['Ferric enterobactin transport system permease protein FepD (TC 3.A.1.14.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08225 MFS transporter, ENTS family, enterobactin (siderophore) exporter\n",
      "SSO SSO:000009105 enterobactin exporter EntS\n",
      "b0591 ['Enterobactin exporter EntS']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02016 iron complex transport system substrate-binding protein\n",
      "KO KO:K23185 Unknown\n",
      "SSO SSO:000002727 Ferric enterobactin-binding periplasmic protein FepB (TC 3.A.1.14.2)\n",
      "b0592 ['Ferric enterobactin-binding periplasmic protein FepB (TC 3.A.1.14.2)']\n",
      "\n",
      "GO GO:0009239 enterobactin biosynthetic process\n",
      "KO KO:K02361 isochorismate synthase [EC:5.4.4.2]\n",
      "b0593 ['Isochorismate synthase (EC 5.4.4.2)', 'Isochorismate synthase (EC 5.4.4.2) [enterobactin] siderophore', 'Isochorismate synthase (EC 5.4.4.2) of siderophore biosynthesis']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009239 enterobactin biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019184 nonribosomal peptide biosynthetic process\n",
      "KO KO:K02363 2,3-dihydroxybenzoate-AMP ligase [EC:6.3.2.14 2.7.7.58]\n",
      "b0594 ['2,3-dihydroxybenzoate-AMP ligase (EC 2.7.7.58) [enterobactin] siderophore', '2,3-dihydroxybenzoate-AMP ligase (EC 2.7.7.58) of siderophore biosynthesis']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009239 enterobactin biosynthetic process\n",
      "GO GO:0019184 nonribosomal peptide biosynthetic process\n",
      "KO KO:K01252 bifunctional isochorismate lyase / aryl carrier protein [EC:3.3.2.1 6.3.2.14]\n",
      "SSO SSO:000017208 Isochorismatase (EC 3.3.2.1)\n",
      "b0595 ['Isochorismatase (EC 3.3.2.1) [enterobactin] siderophore', 'Apo-aryl carrier domain of EntB', 'Isochorismatase (EC 3.3.2.1) of siderophore biosynthesis']\n",
      "\n",
      "GO GO:0009239 enterobactin biosynthetic process\n",
      "KO KO:K00216 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase [EC:1.3.1.28]\n",
      "SSO SSO:000000080 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase (EC 1.3.1.28)\n",
      "b0596 ['2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase (EC 1.3.1.28) [enterobactin] siderophore', '2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase (EC 1.3.1.28) of siderophore biosynthesis']\n",
      "\n",
      "KO KO:K00216 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase [EC:1.3.1.28]\n",
      "KO KO:K19222 1,4-dihydroxy-2-naphthoyl-CoA hydrolase [EC:3.1.2.28]\n",
      "SSO SSO:000006380 Proofreading thioesterase in enterobactin biosynthesis EntH\n",
      "b0597 ['Proofreading thioesterase in enterobactin biosynthesis EntH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K06200 carbon starvation protein\n",
      "SSO SSO:000001417 Carbon starvation protein A\n",
      "b0598 ['Carbon starvation protein A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000028673 Uncharacterized proteins YbdD and YjiX\n",
      "b4512 ['Uncharacterized proteins YbdD and YjiX']\n",
      "\n",
      "KO KO:K08317 uncharacterized oxidoreductase [EC:1.1.-.-]\n",
      "SSO SSO:000036349 Hydroxycarboxylate dehydrogenase (NADP+) HcxA\n",
      "b0599 ['Hydroxycarboxylate dehydrogenase (NADP+) HcxA']\n",
      "\n",
      "GO GO:0008152 metabolic process\n",
      "KO KO:K14287 methionine transaminase [EC:2.6.1.88]\n",
      "SSO SSO:000003178 Glutamine-dependent 2-keto-4-methylthiobutyrate transaminase\n",
      "SSO SSO:000004834 Methionine aminotransferase, PLP-dependent\n",
      "b0600 ['Glutamine-dependent 2-keto-4-methylthiobutyrate transaminase']\n",
      "\n",
      "SSO SSO:000001601 Co-activator of prophage gene expression IbrB\n",
      "b0601 ['Co-activator of prophage gene expression IbrB']\n",
      "\n",
      "SSO SSO:000001600 Co-activator of prophage gene expression IbrA\n",
      "b0602 ['Co-activator of prophage gene expression IbrA']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000019127 LysR-family transcriptional regulator YbdO\n",
      "b0603 ['LysR-family transcriptional regulator YbdO']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03805 thiol:disulfide interchange protein DsbG\n",
      "SSO SSO:000008036 Thiol:disulfide interchange protein DsbG precursor\n",
      "b0604 ['Thiol:disulfide interchange protein DsbG precursor']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K03386 peroxiredoxin (alkyl hydroperoxide reductase subunit C) [EC:1.11.1.15]\n",
      "SSO SSO:000000891 Alkyl hydroperoxide reductase protein C (EC 1.11.1.15)\n",
      "b0605 ['Alkyl hydroperoxide reductase protein C (EC 1.11.1.15)']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K03387 alkyl hydroperoxide reductase subunit F [EC:1.6.4.-]\n",
      "SSO SSO:000000892 Alkyl hydroperoxide reductase protein F (EC 1.6.4.-)\n",
      "b0606 ['Alkyl hydroperoxide reductase protein F']\n",
      "\n",
      "KO KO:K11932 universal stress protein G\n",
      "SSO SSO:000008827 Universal stress protein G\n",
      "b0607 ['Universal stress protein G']\n",
      "\n",
      "SSO SSO:000028814 Uncharacterized zinc-type alcohol dehydrogenase-like protein YbdR\n",
      "b0608 ['Uncharacterized zinc-type alcohol dehydrogenase-like protein YbdR']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K06140 regulator of nucleoside diphosphate kinase\n",
      "SSO SSO:000006969 Regulator of nucleoside diphosphate kinase\n",
      "b0610 ['Regulator of nucleoside diphosphate kinase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K01166 ribonuclease T2 [EC:3.1.27.1]\n",
      "KO KO:K01169 ribonuclease I (enterobacter ribonuclease) [EC:3.1.27.6]\n",
      "SSO SSO:000007083 Ribonuclease I precursor (EC 3.1.27.6)\n",
      "SSO SSO:000024148 Ribonuclease I\n",
      "b0611 ['Ribonuclease I precursor (EC 3.1.27.6)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K09477 citrate:succinate antiporter\n",
      "SSO SSO:000012245 Citrate Succinate antiporter (TC 2.A.47.3.2)\n",
      "b0612 ['Citrate Succinate antiporter (TC 2.A.47.3.2)']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K05966 triphosphoribosyl-dephospho-CoA synthase [EC:2.4.2.52]\n",
      "SSO SSO:000008385 Triphosphoribosyl-dephospho-CoA synthase (EC 2.4.2.52)\n",
      "b0613 ['Triphosphoribosyl-dephospho-CoA synthase (EC 2.4.2.52)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K05964 holo-ACP synthase [EC:2.7.7.61]\n",
      "SSO SSO:000001582 Citrate lyase holo-[acyl-carrier-protein] synthase (EC 2.7.7.61)\n",
      "b0614 ['Citrate lyase holo-[acyl-carrier-protein] synthase (EC 2.7.7.61)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01643 citrate lyase subunit alpha / citrate CoA-transferase [EC:2.8.3.10]\n",
      "SSO SSO:000001579 Citrate lyase alpha chain (EC 4.1.3.6)\n",
      "b0615 ['Citrate lyase alpha chain (EC 4.1.3.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01644 citrate lyase subunit beta / citryl-CoA lyase [EC:4.1.3.34]\n",
      "SSO SSO:000001580 Citrate lyase beta chain (EC 4.1.3.6)\n",
      "b0616 ['Citrate lyase beta chain (EC 4.1.3.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0042967 acyl-carrier-protein biosynthetic process\n",
      "KO KO:K01646 citrate lyase subunit gamma (acyl carrier protein)\n",
      "SSO SSO:000001581 Citrate lyase gamma chain, acyl carrier protein (EC 4.1.3.6)\n",
      "b0617 ['Citrate lyase gamma chain, acyl carrier protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K01910 [citrate (pro-3S)-lyase] ligase [EC:6.2.1.22]\n",
      "SSO SSO:000009012 [Citrate [pro-3S]-lyase] ligase (EC 6.2.1.22)\n",
      "b0618 ['[Citrate [pro-3S]-lyase] ligase (EC 6.2.1.22)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07700 two-component system, CitB family, cit operon sensor histidine kinase CitA [EC:2.7.13.3]\n",
      "b0619 ['Tricarboxylate transport sensor protein TctE => Citrate response regulator CitA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07702 two-component system, CitB family, response regulator CitB\n",
      "b0620 ['Tricarboxylate transport transcriptional regulator TctD => Citrate response regulator CitB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03326 C4-dicarboxylate transporter, DcuC family\n",
      "SSO SSO:000011491 C4-dicarboxylate transporter DcuC (TC 2.A.61.1.1)\n",
      "b0621 ['C4-dicarboxylate transporter DcuC (TC 2.A.61.1.1)']\n",
      "\n",
      "KO KO:K12973 lipid IVA palmitoyltransferase [EC:2.3.1.251]\n",
      "SSO SSO:000036629 Lipid A palmitoyltransferase PagP (EC 2.3.1.251)\n",
      "b0622 ['Lipid A palmitoyltransferase PagP (EC 2.3.1.251)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03704 cold shock protein (beta-ribbon, CspA family)\n",
      "b0623 ['Cold shock protein of CSP family => CspE (naming convention as in E.coli)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K06199 fluoride exporter\n",
      "SSO SSO:000016856 Fluoride ion transporter CrcB\n",
      "b0624 ['Fluoride ion transporter CrcB']\n",
      "\n",
      "SSO SSO:000023157 Putative amidase\n",
      "b4581 ['Putative amidase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03116 sec-independent protein translocase protein TatA\n",
      "KO KO:K03425 sec-independent protein translocase protein TatE\n",
      "SSO SSO:000008428 Twin-arginine translocation protein TatE\n",
      "b0627 ['Twin-arginine translocation protein TatE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009107 lipoate biosynthetic process\n",
      "KO KO:K03644 lipoyl synthase [EC:2.8.1.8]\n",
      "SSO SSO:000004504 Lipoyl synthase (EC 2.8.1.8)\n",
      "b0628 ['Lipoyl synthase (EC 2.8.1.8)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000019128 LysR-family transcriptional regulator YbeF\n",
      "b0629 ['LysR-family transcriptional regulator YbeF']\n",
      "\n",
      "GO GO:0009107 lipoate biosynthetic process\n",
      "KO KO:K03801 lipoyl(octanoyl) transferase [EC:2.3.1.181]\n",
      "SSO SSO:000005479 Octanoate-[acyl-carrier-protein]-protein-N-octanoyltransferase (EC 2.3.1.181)\n",
      "b0630 ['Octanoate-[acyl-carrier-protein]-protein-N-octanoyltransferase (EC 2.3.1.181)']\n",
      "\n",
      "KO KO:K09158 uncharacterized protein\n",
      "SSO SSO:000006407 Proposed lipoate regulatory protein YbeD\n",
      "b0631 ['Proposed lipoate regulatory protein YbeD']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K07258 serine-type D-Ala-D-Ala carboxypeptidase (penicillin-binding protein 5/6) [EC:3.4.16.4]\n",
      "SSO SSO:000001995 D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)\n",
      "b0632 ['D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03642 rare lipoprotein A\n",
      "SSO SSO:000007420 Septum-associated rare lipoprotein A\n",
      "b0633 ['Septum-associated rare lipoprotein A']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05837 rod shape determining protein RodA\n",
      "SSO SSO:000007180 Rod shape-determining protein RodA\n",
      "b0634 ['Rod shape-determining protein RodA']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K05515 penicillin-binding protein 2 [EC:3.4.16.4]\n",
      "SSO SSO:000037526 Peptidoglycan D,D-transpeptidase MrdA (EC 3.4.16.4)\n",
      "b0635 ['Peptidoglycan D,D-transpeptidase MrdA (EC 3.4.16.4)']\n",
      "\n",
      "KO KO:K00783 23S rRNA (pseudouridine1915-N3)-methyltransferase [EC:2.1.1.177]\n",
      "SSO SSO:000000240 23S rRNA (pseudouridine(1915)-N(3))-methyltransferase (EC 2.1.1.177)\n",
      "b0636 ['23S rRNA (pseudouridine(1915)-N(3))-methyltransferase (EC 2.1.1.177)']\n",
      "\n",
      "KO KO:K09710 ribosome-associated protein\n",
      "SSO SSO:000007134 Ribosomal silencing factor RsfA\n",
      "b0637 ['Ribosomal silencing factor RsfA']\n",
      "\n",
      "KO KO:K02226 alpha-ribazole phosphatase [EC:3.1.3.73]\n",
      "SSO SSO:000000926 Alpha-ribazole-5'-phosphate phosphatase (EC 3.1.3.73)\n",
      "b0638 [\"Alpha-ribazole-5'-phosphate phosphatase (EC 3.1.3.73)\"]\n",
      "\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "KO KO:K00969 nicotinate-nucleotide adenylyltransferase [EC:2.7.7.18]\n",
      "SSO SSO:000005314 Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)\n",
      "b0639 ['Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02340 DNA polymerase III subunit delta [EC:2.7.7.7]\n",
      "SSO SSO:000002083 DNA polymerase III delta subunit (EC 2.7.7.7)\n",
      "b0640 ['DNA polymerase III delta subunit (EC 2.7.7.7)']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042158 lipoprotein biosynthetic process\n",
      "KO KO:K03643 LPS-assembly lipoprotein\n",
      "SSO SSO:000036572 LPS-assembly lipoprotein LptE\n",
      "b0641 ['LPS-assembly lipoprotein LptE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01869 leucyl-tRNA synthetase [EC:6.1.1.4]\n",
      "SSO SSO:000004422 Leucyl-tRNA synthetase (EC 6.1.1.4)\n",
      "b0642 ['Leucyl-tRNA synthetase (EC 6.1.1.4)']\n",
      "\n",
      "SSO SSO:000028172 Uncharacterized protein YbeL\n",
      "b0643 ['Uncharacterized protein YbeL']\n",
      "\n",
      "KO KO:K07126 uncharacterized protein\n",
      "SSO SSO:000028173 Uncharacterized protein YbeQ\n",
      "b0644 ['Uncharacterized protein YbeQ']\n",
      "\n",
      "SSO SSO:000028174 Uncharacterized protein YbeR\n",
      "b0645 ['Uncharacterized protein YbeR']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000026886 Uncharacterized J domain-containing protein YbeS, predicted chaperone\n",
      "b0646 ['Uncharacterized J domain-containing protein YbeS, predicted chaperone']\n",
      "\n",
      "KO KO:K07126 uncharacterized protein\n",
      "SSO SSO:000028175 Uncharacterized protein YbeT\n",
      "b0647 ['Uncharacterized protein YbeT']\n",
      "\n",
      "SSO SSO:000028176 Uncharacterized protein YbeU\n",
      "b0648 ['Uncharacterized protein YbeU']\n",
      "\n",
      "SSO SSO:000026887 Uncharacterized J domain-containing protein YbeV\n",
      "b0649 ['Uncharacterized J domain-containing protein YbeV']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04045 molecular chaperone HscC\n",
      "SSO SSO:000012123 Chaperone protein HscC\n",
      "b0650 ['Chaperone protein HscC']\n",
      "\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "KO KO:K01239 purine nucleosidase [EC:3.2.2.1]\n",
      "KO KO:K01250 pyrimidine-specific ribonucleoside hydrolase [EC:3.2.-.-]\n",
      "SSO SSO:000023761 Pyrimidine-specific ribonucleoside hydrolase RihA\n",
      "b0651 ['Pyrimidine-specific ribonucleoside hydrolase RihA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006532 aspartate biosynthetic process\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "KO KO:K02028 polar amino acid transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10004 glutamate/aspartate transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000036107 Glutamate/aspartate ABC transporter, ATP-binding protein GltL (TC 3.A.1.3.4)\n",
      "b0652 ['Glutamate/aspartate ABC transporter, ATP-binding protein GltL (TC 3.A.1.3.4)']\n",
      "\n",
      "GO GO:0006532 aspartate biosynthetic process\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02029 polar amino acid transport system permease protein\n",
      "KO KO:K10002 glutamate/aspartate transport system permease protein\n",
      "SSO SSO:000036109 Glutamate/aspartate ABC transporter, permease protein GltK (TC 3.A.1.3.4)\n",
      "b0653 ['Glutamate/aspartate ABC transporter, permease protein GltK (TC 3.A.1.3.4)']\n",
      "\n",
      "GO GO:0006532 aspartate biosynthetic process\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10003 glutamate/aspartate transport system permease protein\n",
      "SSO SSO:000036108 Glutamate/aspartate ABC transporter, permease protein GltJ (TC 3.A.1.3.4)\n",
      "b0654 ['Glutamate/aspartate ABC transporter, permease protein GltJ (TC 3.A.1.3.4)']\n",
      "\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02030 polar amino acid transport system substrate-binding protein\n",
      "KO KO:K10001 glutamate/aspartate transport system substrate-binding protein\n",
      "SSO SSO:000036110 Glutamate/aspartate ABC transporter, substrate-binding protein GltI (TC 3.A.1.3.4)\n",
      "b0655 ['Glutamate/aspartate ABC transporter, substrate-binding protein GltI (TC 3.A.1.3.4)']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b0656 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042158 lipoprotein biosynthetic process\n",
      "KO KO:K03820 apolipoprotein N-acyltransferase [EC:2.3.1.269]\n",
      "SSO SSO:000001022 Apolipoprotein N-acyltransferase (EC 2.3.1.-)\n",
      "b0657 ['Apolipoprotein N-acyltransferase', 'Copper homeostasis protein CutE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K06189 magnesium and cobalt transporter\n",
      "SSO SSO:000004629 Magnesium and cobalt efflux protein CorC\n",
      "b0658 ['Magnesium and cobalt efflux protein CorC']\n",
      "\n",
      "KO KO:K07042 probable rRNA maturation factor\n",
      "SSO SSO:000004804 Metal-dependent hydrolase YbeY, involved in rRNA and/or ribosome maturation and assembly\n",
      "b0659 ['Metal-dependent hydrolase YbeY, involved in rRNA and/or ribosome maturation and assembly']\n",
      "\n",
      "KO KO:K06217 phosphate starvation-inducible protein PhoH and related proteins\n",
      "SSO SSO:000005923 Phosphate starvation-inducible protein PhoH, predicted ATPase\n",
      "b0660 ['Phosphate starvation-inducible protein PhoH, predicted ATPase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K06168 tRNA-2-methylthio-N6-dimethylallyladenosine synthase [EC:2.8.4.3]\n",
      "SSO SSO:000009401 tRNA-i(6)A37 methylthiotransferase (EC 2.8.4.3)\n",
      "b0661 ['tRNA-i(6)A37 methylthiotransferase (EC 2.8.4.3)']\n",
      "\n",
      "KO KO:K03184 3-demethoxyubiquinol 3-hydroxylase [EC:1.14.99.60]\n",
      "SSO SSO:000000214 2-polyprenyl-3-methyl-6-methoxy-1,4-benzoquinol hydroxylase\n",
      "b0662 ['2-polyprenyl-3-methyl-6-methoxy-1,4-benzoquinol hydroxylase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006529 asparagine biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01953 asparagine synthase (glutamine-hydrolysing) [EC:6.3.5.4]\n",
      "SSO SSO:000001084 Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)\n",
      "b0674 ['Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)']\n",
      "\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K02566 NagD protein\n",
      "b0675 ['Phosphatase NagD predicted to act in N-acetylglucosamine utilization subsystem']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009386 translational attenuation\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K02565 N-acetylglucosamine repressor\n",
      "SSO SSO:000005106 N-acetylglucosamine-6P-responsive transcriptional repressor NagC, ROK family\n",
      "b0676 ['N-acetylglucosamine-6P-responsive transcriptional repressor NagC, ROK family']\n",
      "\n",
      "EC EC:3.5.1.25 N-acetylglucosamine-6-phosphate deacetylase.\n",
      "EC EC:3.5.1.80 Transferred entry: 3.5.1.25.\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K01443 N-acetylglucosamine-6-phosphate deacetylase [EC:3.5.1.25]\n",
      "SSO SSO:000005105 N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)\n",
      "b0677 ['N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)']\n",
      "\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K02564 glucosamine-6-phosphate deaminase [EC:3.5.99.6]\n",
      "SSO SSO:000003114 Glucosamine-6-phosphate deaminase (EC 3.5.99.6)\n",
      "b0678 ['Glucosamine-6-phosphate deaminase (EC 3.5.99.6)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K02804 PTS system, N-acetylglucosamine-specific IIC component\n",
      "b0679 ['PTS system, N-acetylglucosamine-specific IIC component', 'PTS system, N-acetylglucosamine-specific IIB component (EC 2.7.1.193)', 'PTS system, N-acetylglucosamine-specific IIA component (EC 2.7.1.193)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01886 glutaminyl-tRNA synthetase [EC:6.1.1.18]\n",
      "SSO SSO:000003179 Glutaminyl-tRNA synthetase (EC 6.1.1.18)\n",
      "SSO SSO:000003186 Glutamyl-tRNA synthetase (EC 6.1.1.17)\n",
      "b0680 ['Glutaminyl-tRNA synthetase (EC 6.1.1.18)']\n",
      "\n",
      "SSO SSO:000005580 Outer membrane porin for chitooligosaccharides ChiP\n",
      "b0681 ['Outer membrane porin for chitooligosaccharides ChiP']\n",
      "\n",
      "SSO SSO:000027198 Uncharacterized lipoprotein ChiQ\n",
      "b0682 ['Uncharacterized lipoprotein ChiQ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03711 Fur family transcriptional regulator, ferric uptake regulator\n",
      "SSO SSO:000002739 Ferric uptake regulation protein FUR\n",
      "b0683 ['Ferric uptake regulation protein FUR']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4637 ['hypothetical protein']\n",
      "\n",
      "KO KO:K03839 flavodoxin I\n",
      "SSO SSO:000002850 Flavodoxin 1\n",
      "b0684 ['Flavodoxin 1']\n",
      "\n",
      "GO GO:0009432 SOS response\n",
      "SSO SSO:000008792 Uncharacterized protein ybfE\n",
      "b0685 ['Uncharacterized protein YbfE']\n",
      "\n",
      "KO KO:K01175 esterase [EC:3.1.-.-]\n",
      "SSO SSO:000002562 Esterase ybfF (EC 3.1.-.-)\n",
      "b0686 ['Esterase ybfF (EC 3.1.-.-)']\n",
      "\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03645 negative modulator of initiation of replication\n",
      "SSO SSO:000007421 SeqA protein, negative modulator of initiation of replication\n",
      "b0687 ['SeqA protein, negative modulator of initiation of replication']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01835 phosphoglucomutase [EC:5.4.2.2]\n",
      "SSO SSO:000005967 Phosphoglucomutase (EC 5.4.2.2)\n",
      "b0688 ['Phosphoglucomutase (EC 5.4.2.2)']\n",
      "\n",
      "SSO SSO:000018013 Hypothetical lipoprotein ybfP precursor\n",
      "SSO SSO:000018986 Lipoprotein\n",
      "b0689 ['Hypothetical lipoprotein ybfP precursor']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0691 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4636 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0015496 putrescine:ornithine antiporter activity\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03756 putrescine:ornithine antiporter\n",
      "SSO SSO:000006788 Putrescine/proton symporter, putrescine/ornithine antiporter PotE\n",
      "b0692 ['Putrescine/proton symporter, putrescine/ornithine antiporter PotE']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "SSO SSO:000005516 Ornithine decarboxylase (EC 4.1.1.17)\n",
      "b0693 ['Ornithine decarboxylase (EC 4.1.1.17)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4590 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07667 two-component system, OmpR family, KDP operon response regulator KdpE\n",
      "SSO SSO:000002142 DNA-binding response regulator KdpE\n",
      "b0694 ['DNA-binding response regulator KdpE']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07646 two-component system, OmpR family, sensor histidine kinase KdpD [EC:2.7.13.3]\n",
      "SSO SSO:000005525 Osmosensitive K+ channel histidine kinase KdpD (EC 2.7.3.-)\n",
      "b0695 ['Osmosensitive K+ channel histidine kinase KdpD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015662 ATPase activity, coupled to transmembrane movement of ions, phosphorylative mechanism\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01548 K+-transporting ATPase ATPase C chain [EC:3.6.3.12]\n",
      "SSO SSO:000006198 Potassium-transporting ATPase C chain (TC 3.A.3.7.1) (EC 3.6.3.12)\n",
      "b0696 ['Potassium-transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015662 ATPase activity, coupled to transmembrane movement of ions, phosphorylative mechanism\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01547 K+-transporting ATPase ATPase B chain [EC:3.6.3.12]\n",
      "SSO SSO:000006197 Potassium-transporting ATPase B chain (TC 3.A.3.7.1) (EC 3.6.3.12)\n",
      "b0697 ['Potassium-transporting ATPase B chain (EC 3.6.3.12) (TC 3.A.3.7.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015662 ATPase activity, coupled to transmembrane movement of ions, phosphorylative mechanism\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01546 K+-transporting ATPase ATPase A chain [EC:3.6.3.12]\n",
      "SSO SSO:000006196 Potassium-transporting ATPase A chain (TC 3.A.3.7.1) (EC 3.6.3.12)\n",
      "b0698 ['Potassium-transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)']\n",
      "\n",
      "GO GO:0031225 anchored component of membrane\n",
      "KO KO:K01545 K+-transporting ATPase ATPase F chain\n",
      "b4513 ['Protein KdpF']\n",
      "\n",
      "SSO SSO:000028177 Uncharacterized protein YbfA\n",
      "b0699 ['Uncharacterized protein YbfA']\n",
      "\n",
      "SSO SSO:000022715 Protein RhsC\n",
      "b0700 ['Protein RhsC']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "SSO SSO:000030665 putative membrane protein\n",
      "b0702 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0703 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028178 Uncharacterized protein YbfC\n",
      "SSO SSO:000030757 putative secreted protein\n",
      "b0704 ['Uncharacterized protein YbfC']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4514 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0705 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000017503 H repeat-associated protein, YhhI family\n",
      "b0706 ['H repeat-associated protein, YhhI family']\n",
      "\n",
      "SSO SSO:000011663 FIG143828: Hypothetical protein YbgA\n",
      "b0707 ['FIG143828: Hypothetical protein YbgA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K01669 deoxyribodipyrimidine photo-lyase [EC:4.1.99.3]\n",
      "SSO SSO:000002250 Deoxyribodipyrimidine photolyase (EC 4.1.99.3)\n",
      "b0708 ['Deoxyribodipyrimidine photolyase (EC 4.1.99.3)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03305 proton-dependent oligopeptide transporter, POT family\n",
      "SSO SSO:000013164 Dipeptide/H+ symporter DtpD\n",
      "b0709 ['Dipeptide/H+ symporter DtpD']\n",
      "\n",
      "b0710 ['GTP cyclohydrolase 1 type 2 homolog YbgI']\n",
      "\n",
      "KO KO:K23123 5-oxoprolinase (ATP-hydrolysing) subunit B [EC:3.5.2.9]\n",
      "SSO SSO:000000902 Allophanate hydrolase 2 subunit 1 (EC 3.5.1.54)\n",
      "b0711 ['Allophanate hydrolase 2 subunit 1 (EC 3.5.1.54)']\n",
      "\n",
      "KO KO:K23124 5-oxoprolinase (ATP-hydrolysing) subunit C [EC:3.5.2.9]\n",
      "SSO SSO:000000903 Allophanate hydrolase 2 subunit 2 (EC 3.5.1.54)\n",
      "b0712 ['Allophanate hydrolase 2 subunit 2 (EC 3.5.1.54)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K07160 5-oxoprolinase (ATP-hydrolysing) subunit A [EC:3.5.2.9]\n",
      "SSO SSO:000004379 Lactam utilization protein LamB\n",
      "b0713 ['Lactam utilization protein LamB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K05522 endonuclease VIII [EC:3.2.2.- 4.2.99.18]\n",
      "SSO SSO:000002476 Endonuclease VIII\n",
      "b0714 ['Endonuclease VIII']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07120 uncharacterized protein\n",
      "SSO SSO:000010142 AbrB protein\n",
      "b0715 ['AbrB protein']\n",
      "\n",
      "SSO SSO:000008728 Uncharacterized fimbrial-system-associated protein YbgO\n",
      "SSO SSO:000023304 Putative fimbrial protein\n",
      "b0716 ['Uncharacterized fimbrial-system-associated protein YbgO']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000008709 Uncharacterized fimbrial chaperone YbgP\n",
      "b0717 ['Uncharacterized fimbrial chaperone YbgP']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000008739 Uncharacterized outer membrane usher protein YbgQ\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "b0718 ['Uncharacterized outer membrane usher protein YbgQ']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "SSO SSO:000008721 Uncharacterized fimbrial-like protein YbgD\n",
      "b0719 ['Uncharacterized fimbrial-like protein YbgD']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K01647 citrate synthase [EC:2.3.3.1]\n",
      "SSO SSO:000001584 Citrate synthase (si) (EC 2.3.3.1)\n",
      "b0720 ['Citrate synthase (si) (EC 2.3.3.1)']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00241 succinate dehydrogenase / fumarate reductase, cytochrome b subunit\n",
      "SSO SSO:000007776 Succinate dehydrogenase cytochrome b-556 subunit\n",
      "b0721 ['Succinate dehydrogenase cytochrome b-556 subunit']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00242 succinate dehydrogenase / fumarate reductase, membrane anchor subunit\n",
      "SSO SSO:000007780 Succinate dehydrogenase hydrophobic membrane anchor protein\n",
      "b0722 ['Succinate dehydrogenase hydrophobic membrane anchor protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K00239 succinate dehydrogenase / fumarate reductase, flavoprotein subunit [EC:1.3.5.1 1.3.5.4]\n",
      "SSO SSO:000007779 Succinate dehydrogenase flavoprotein subunit (EC 1.3.99.1)\n",
      "b0723 ['Succinate dehydrogenase flavoprotein subunit (EC 1.3.5.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K00240 succinate dehydrogenase / fumarate reductase, iron-sulfur subunit [EC:1.3.5.1 1.3.5.4]\n",
      "SSO SSO:000007781 Succinate dehydrogenase iron-sulfur protein (EC 1.3.99.1)\n",
      "b0724 ['Succinate dehydrogenase iron-sulfur protein (EC 1.3.5.1)']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "KO KO:K00164 2-oxoglutarate dehydrogenase E1 component [EC:1.2.4.2]\n",
      "SSO SSO:000000206 2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)\n",
      "b0726 ['2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "KO KO:K00658 2-oxoglutarate dehydrogenase E2 component (dihydrolipoamide succinyltransferase) [EC:2.3.1.61]\n",
      "SSO SSO:000002298 Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)\n",
      "b0727 ['Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "KO KO:K01903 succinyl-CoA synthetase beta subunit [EC:6.2.1.5]\n",
      "SSO SSO:000007790 Succinyl-CoA ligase [ADP-forming] beta chain (EC 6.2.1.5)\n",
      "b0728 ['Succinyl-CoA ligase [ADP-forming] beta chain (EC 6.2.1.5)']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "KO KO:K01902 succinyl-CoA synthetase alpha subunit [EC:6.2.1.5]\n",
      "SSO SSO:000007789 Succinyl-CoA ligase [ADP-forming] alpha chain (EC 6.2.1.5)\n",
      "b0729 ['Succinyl-CoA ligase [ADP-forming] alpha chain (EC 6.2.1.5)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03710 GntR family transcriptional regulator\n",
      "KO KO:K11922 GntR family transcriptional regulator, mannosyl-D-glycerate transport/metabolism system repressor\n",
      "SSO SSO:000008244 Transcriptional regulator of succinyl CoA synthetase operon\n",
      "b0730 ['Transcriptional regulator of succinyl CoA synthetase operon']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02768 PTS system, fructose-specific IIA component [EC:2.7.1.202]\n",
      "KO KO:K11200 PTS system, 2-O-A-mannosyl-D-glycerate-specific IIC component\n",
      "b0731 ['PTS system, 2-O-alpha-mannosyl-D-glycerate-specific IIA component (EC 2.7.1.195)', 'PTS system, 2-O-alpha-mannosyl-D-glycerate-specific IIB component (EC 2.7.1.195)', 'PTS system, 2-O-alpha-mannosyl-D-glycerate-specific IIC component']\n",
      "\n",
      "EC EC:3.2.1.113 Mannosyl-oligosaccharide 1,2-alpha-mannosidase.\n",
      "EC EC:3.2.1.114 Mannosyl-oligosaccharide 1,3-1,6-alpha-mannosidase.\n",
      "EC EC:3.2.1.170 Mannosylglycerate hydrolase.\n",
      "EC EC:3.2.1.24 Alpha-mannosidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K15524 mannosylglycerate hydrolase [EC:3.2.1.170]\n",
      "SSO SSO:000000925 Alpha-mannosidase (EC 3.2.1.24)\n",
      "SSO SSO:000004719 Mannosylglycerate hydrolase (EC 3.2.1.170)\n",
      "b0732 ['Mannosylglycerate hydrolase (EC 3.2.1.170)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00425 cytochrome bd ubiquinol oxidase subunit I [EC:7.1.1.7]\n",
      "SSO SSO:000001939 Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)\n",
      "b0733 ['Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00426 cytochrome bd ubiquinol oxidase subunit II [EC:7.1.1.7]\n",
      "SSO SSO:000001940 Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)\n",
      "b0734 ['Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00424 cytochrome bd-I ubiquinol oxidase subunit X [EC:7.1.1.7]\n",
      "SSO SSO:000001941 Cytochrome d ubiquinol oxidase subunit X (EC 1.10.3.-)\n",
      "b4515 ['Cytochrome d ubiquinol oxidase subunit X (EC 1.10.3.-)']\n",
      "\n",
      "SSO SSO:000001845 Cyd operon protein YbgE\n",
      "b0735 ['Cyd operon protein YbgE']\n",
      "\n",
      "KO KO:K07107 acyl-CoA thioester hydrolase [EC:3.1.2.-]\n",
      "SSO SSO:000008083 Tol-Pal system-associated acyl-CoA thioesterase\n",
      "SSO SSO:000010270 Acyl-CoA thioester hydrolase\n",
      "b0736 ['Tol-Pal system-associated acyl-CoA thioesterase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03561 biopolymer transport protein ExbB\n",
      "KO KO:K03562 biopolymer transport protein TolQ\n",
      "SSO SSO:000008082 Tol-Pal system protein TolQ\n",
      "b0737 ['Tol-Pal system protein TolQ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03559 biopolymer transport protein ExbD\n",
      "SSO SSO:000008078 Tol biopolymer transport system, TolR protein\n",
      "b0738 ['Tol biopolymer transport system, TolR protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03646 colicin import membrane protein\n",
      "SSO SSO:000008084 TolA protein\n",
      "b0739 ['TolA protein']\n",
      "\n",
      "EC EC:4.2.2.6 Oligogalacturonide lyase.\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03641 TolB protein\n",
      "SSO SSO:000008080 Tol-Pal system beta propeller repeat protein TolB\n",
      "SSO SSO:000031394 periplasmic protein\n",
      "b0740 ['Tol-Pal system beta propeller repeat protein TolB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03640 peptidoglycan-associated lipoprotein\n",
      "SSO SSO:000008081 Tol-Pal system peptidoglycan-associated lipoprotein PAL\n",
      "SSO SSO:000021401 Peptidoglycan-associated outer membrane lipoprotein\n",
      "b0741 ['Tol-Pal system peptidoglycan-associated lipoprotein PAL']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "SSO SSO:000035036 Cell division coordinator CpoB\n",
      "b0742 ['Cell division coordinator CpoB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "KO KO:K03517 quinolinate synthase [EC:2.5.1.72]\n",
      "SSO SSO:000006878 Quinolinate synthetase (EC 2.5.1.72)\n",
      "b0750 ['Quinolinate synthetase (EC 2.5.1.72)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03811 nicotinamide mononucleotide transporter\n",
      "SSO SSO:000007153 Ribosyl nicotinamide transporter, PnuC-like\n",
      "b0751 ['Ribosyl nicotinamide transporter, PnuC-like']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16264 cobalt-zinc-cadmium efflux system protein\n",
      "SSO SSO:000029192 Zinc transporter ZitB\n",
      "b0752 ['Zinc transporter ZitB']\n",
      "\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "SSO SSO:000030757 putative secreted protein\n",
      "b0753 ['putative secreted protein']\n",
      "\n",
      "GO GO:0009094 L-phenylalanine biosynthetic process\n",
      "KO KO:K01626 3-deoxy-7-phosphoheptulonate synthase [EC:2.5.1.54]\n",
      "SSO SSO:000000169 2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I alpha (EC 2.5.1.54)\n",
      "b0754 ['2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I alpha (EC 2.5.1.54)']\n",
      "\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006096 glycolytic process\n",
      "KO KO:K01834 2,3-bisphosphoglycerate-dependent phosphoglycerate mutase [EC:5.4.2.11]\n",
      "SSO SSO:000005972 Phosphoglycerate mutase (EC 5.4.2.1)\n",
      "b0755 ['Phosphoglycerate mutase (EC 5.4.2.11)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01785 aldose 1-epimerase [EC:5.1.3.3]\n",
      "SSO SSO:000000870 Aldose 1-epimerase (EC 5.1.3.3)\n",
      "b0756 ['Aldose 1-epimerase (EC 5.1.3.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006012 galactose metabolic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00849 galactokinase [EC:2.7.1.6]\n",
      "SSO SSO:000003006 Galactokinase (EC 2.7.1.6)\n",
      "b0757 ['Galactokinase (EC 2.7.1.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006012 galactose metabolic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00965 UDPglucose--hexose-1-phosphate uridylyltransferase [EC:2.7.7.12]\n",
      "SSO SSO:000003012 Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)\n",
      "b0758 ['Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006012 galactose metabolic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01784 UDP-glucose 4-epimerase [EC:5.1.3.2]\n",
      "SSO SSO:000008644 UDP-glucose 4-epimerase (EC 5.1.3.2)\n",
      "b0759 ['UDP-glucose 4-epimerase (EC 5.1.3.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "GO GO:0042888 molybdenum ion transmembrane transporter activity\n",
      "KO KO:K05776 molybdate transport system ATP-binding protein\n",
      "SSO SSO:000023440 Putative molybdenum transport ATP-binding protein modF\n",
      "b0760 ['Putative molybdenum transport ATP-binding protein modF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02019 molybdate transport system regulatory protein\n",
      "b0761 ['DNA-binding domain of ModE', 'Molybdate-binding domain of ModE']\n",
      "\n",
      "SSO SSO:000000740 AcrZ membrane protein associated with AcrAB-TolC multidrug efflux pump\n",
      "b0762 ['AcrZ membrane protein associated with AcrAB-TolC multidrug efflux pump']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02020 molybdate transport system substrate-binding protein\n",
      "SSO SSO:000004935 Molybdenum ABC transporter, substrate-binding protein ModA\n",
      "b0763 ['Molybdenum ABC transporter, substrate-binding protein ModA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02018 molybdate transport system permease protein\n",
      "SSO SSO:000004934 Molybdenum ABC transporter permease protein ModB\n",
      "b0764 ['Molybdenum ABC transporter permease protein ModB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K02017 molybdate transport system ATP-binding protein [EC:3.6.3.29]\n",
      "SSO SSO:000004933 Molybdenum ABC transporter ATP-binding protein ModC\n",
      "b0765 ['Molybdenum ABC transporter ATP-binding protein ModC']\n",
      "\n",
      "KO KO:K21064 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphatase [EC:3.1.3.104]\n",
      "b0766 ['Phosphatase YbhA (EC 3.1.3.-)']\n",
      "\n",
      "KO KO:K07404 6-phosphogluconolactonase [EC:3.1.1.31]\n",
      "SSO SSO:000000457 6-phosphogluconolactonase (EC 3.1.1.31)\n",
      "b0767 ['6-phosphogluconolactonase (EC 3.1.1.31)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K17737 LysR family transcriptional regulator, carnitine catabolism transcriptional activator\n",
      "SSO SSO:000008220 Transcriptional regulator YbhD, LysR family\n",
      "b0768 ['Transcriptional regulator YbhD, LysR family']\n",
      "\n",
      "KO KO:K16514 4-oxalomesaconate tautomerase [EC:5.3.2.8]\n",
      "SSO SSO:000018114 Hypothetical protein YbhH, similar to PrpF protein involved in 2-methylcitrate cycle\n",
      "b0769 ['Hypothetical protein YbhH, similar to PrpF protein involved in 2-methylcitrate cycle']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03319 divalent anion:Na+ symporter, DASS family\n",
      "SSO SSO:000018381 Inner membrane protein YbhI\n",
      "b0770 ['Inner membrane protein YbhI']\n",
      "\n",
      "SSO SSO:000000733 Aconitase family protein YbhJ\n",
      "b0771 ['Aconitase family protein YbhJ']\n",
      "\n",
      "EC EC:3.1.1.11 Pectinesterase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "GO GO:0008152 metabolic process\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K01051 pectinesterase [EC:3.1.1.11]\n",
      "SSO SSO:000005751 Pectinesterase (EC 3.1.1.11)\n",
      "b0772 ['Pectinesterase (EC 3.1.1.11)']\n",
      "\n",
      "SSO SSO:000026499 UPF0098 protein ybhB\n",
      "b0773 ['UPF0098 protein ybhB']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K00833 adenosylmethionine---8-amino-7-oxononanoate aminotransferase [EC:2.6.1.62]\n",
      "SSO SSO:000000818 Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)\n",
      "b0774 ['Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K01012 biotin synthase [EC:2.8.1.6]\n",
      "SSO SSO:000001236 Biotin synthase (EC 2.8.1.6)\n",
      "b0775 ['Biotin synthase (EC 2.8.1.6)']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K00652 8-amino-7-oxononanoate synthase [EC:2.3.1.47]\n",
      "SSO SSO:000000464 8-amino-7-oxononanoate synthase (EC 2.3.1.47)\n",
      "b0776 ['8-amino-7-oxononanoate synthase (EC 2.3.1.47)']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K02169 malonyl-CoA O-methyltransferase [EC:2.1.1.197]\n",
      "SSO SSO:000004697 Malonyl-[acyl-carrier protein] O-methyltransferase (EC 2.1.1.197)\n",
      "b0777 ['Malonyl-[acyl-carrier protein] O-methyltransferase (EC 2.1.1.197)']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K01935 dethiobiotin synthetase [EC:6.3.3.3]\n",
      "SSO SSO:000002264 Dethiobiotin synthetase (EC 6.3.3.3)\n",
      "SSO SSO:000034208 Dethiobiotin synthase BioD (EC 6.3.3.3)\n",
      "b0778 ['Dethiobiotin synthase BioD (EC 6.3.3.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0009314 response to radiation\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03702 excinuclease ABC subunit B\n",
      "SSO SSO:000002612 Excinuclease ABC subunit B\n",
      "b0779 ['Excinuclease ABC subunit B']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b0780 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03639 GTP 3',8-cyclase [EC:4.1.99.22]\n",
      "SSO SSO:000036014 GTP 3',8-cyclase (EC 4.1.99.22)\n",
      "b0781 [\"GTP 3',8-cyclase (EC 4.1.99.22)\"]\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03638 molybdopterin adenylyltransferase [EC:2.7.7.75]\n",
      "SSO SSO:000004936 Molybdenum cofactor biosynthesis protein MoaB\n",
      "b0782 ['Molybdenum cofactor biosynthesis protein MoaB']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03637 cyclic pyranopterin monophosphate synthase [EC:4.6.1.17]\n",
      "SSO SSO:000035240 Cyclic pyranopterin monophosphate synthase (EC 4.6.1.17)\n",
      "b0783 ['Cyclic pyranopterin monophosphate synthase (EC 4.6.1.17)']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03636 sulfur-carrier protein\n",
      "SSO SSO:000004946 Molybdopterin synthase sulfur carrier subunit\n",
      "b0784 ['Molybdopterin synthase sulfur carrier subunit']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03635 molybdopterin synthase catalytic subunit [EC:2.8.1.12]\n",
      "SSO SSO:000004945 Molybdopterin synthase catalytic subunit MoaE (EC 2.8.1.12)\n",
      "b0785 ['Molybdopterin synthase catalytic subunit MoaE (EC 2.8.1.12)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06890 uncharacterized protein\n",
      "SSO SSO:000018382 Inner membrane protein YbhL\n",
      "b0786 ['Inner membrane protein YbhL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000028180 Uncharacterized protein YbhM\n",
      "b0787 ['Uncharacterized protein YbhM']\n",
      "\n",
      "KO KO:K07027 glycosyltransferase 2 family protein\n",
      "SSO SSO:000018383 Inner membrane protein YbhQ\n",
      "b0788 ['Inner membrane protein YbhQ']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "KO KO:K06131 cardiolipin synthase A/B [EC:2.7.8.-]\n",
      "SSO SSO:000035001 Cardiolipin synthase bacterial type ClsB, has minor trans-phosphatidylation activity converting PE to PG\n",
      "b0789 ['Cardiolipin synthase (EC 2.7.8.-) bacterial type ClsB, has minor trans-phosphatidylation activity converting PE to PG']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "SSO SSO:000013400 Endonuclease/exonuclease/phosphatase family protein\n",
      "b0790 ['Endonuclease/exonuclease/phosphatase family protein']\n",
      "\n",
      "SSO SSO:000018383 Inner membrane protein YbhQ\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b0791 ['Inner membrane protein YbhQ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01992 ABC-2 type transport system permease protein\n",
      "SSO SSO:000000511 ABC-type efflux pump permease component YbhR\n",
      "b0792 ['ABC-type efflux pump permease component YbhR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01992 ABC-2 type transport system permease protein\n",
      "SSO SSO:000000512 ABC-type efflux pump permease component YbhS\n",
      "b0793 ['ABC-type efflux pump permease component YbhS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01990 ABC-2 type transport system ATP-binding protein\n",
      "SSO SSO:000000513 ABC-type efflux pump, duplicated ATPase component YbhF\n",
      "b0794 ['ABC-type efflux pump, duplicated ATPase component YbhF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K01993 HlyD family secretion protein\n",
      "SSO SSO:000000510 ABC-type efflux pump membrane fusion component YbhG\n",
      "b0795 ['ABC-type efflux pump membrane fusion component YbhG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K23777 Unknown\n",
      "SSO SSO:000008221 Transcriptional regulator YbiH, TetR family\n",
      "b0796 ['Transcriptional regulator YbiH, TetR family']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K11927 ATP-dependent RNA helicase RhlE [EC:3.6.4.13]\n",
      "SSO SSO:000000597 ATP-dependent RNA helicase RhlE\n",
      "SSO SSO:000009996 ATP dependent RNA helicase\n",
      "b0797 ['ATP-dependent RNA helicase RhlE (EC 3.6.4.13)']\n",
      "\n",
      "KO KO:K09935 uncharacterized protein\n",
      "SSO SSO:000008744 Uncharacterized protein COG3236\n",
      "b0798 ['Uncharacterized protein COG3236']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03722 ATP-dependent DNA helicase DinG [EC:3.6.4.12]\n",
      "SSO SSO:000000614 ATP-dependent helicase DinG/Rad3\n",
      "SSO SSO:000010027 ATP-dependent DNA helicase\n",
      "b0799 ['ATP-dependent helicase DinG/Rad3']\n",
      "\n",
      "SSO SSO:000034648 Anthranilate phosphoribosyltransferase-like protein\n",
      "b0800 ['Anthranilate phosphoribosyltransferase-like protein']\n",
      "\n",
      "KO KO:K13574 uncharacterized oxidoreductase [EC:1.1.1.-]\n",
      "SSO SSO:000023250 Putative dehydrogenase\n",
      "b0801 ['Hydroxycarboxylate dehydrogenase (NADP+) HcxB']\n",
      "\n",
      "KO KO:K12151 multiple stress resistance protein BhsA\n",
      "b0802 ['Uncharacterized periplasmic protein YbiJ']\n",
      "\n",
      "KO KO:K06204 DnaK suppressor protein\n",
      "SSO SSO:000013188 DksA-type zinc finger protein\n",
      "SSO SSO:000028812 Uncharacterized zinc-finger containing protein YbiI\n",
      "b0803 ['Uncharacterized zinc-finger containing protein YbiI']\n",
      "\n",
      "KO KO:K07336 PKHD-type hydroxylase [EC:1.14.11.-]\n",
      "SSO SSO:000021091 PKHD-type hydroxylase YbiX\n",
      "b0804 ['PKHD-type hydroxylase YbiX']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "KO KO:K16090 catecholate siderophore receptor\n",
      "SSO SSO:000002758 Ferrichrome-iron receptor\n",
      "b0805 ['Ferrichrome-iron receptor']\n",
      "\n",
      "KO KO:K13650 MqsR-controlled colanic acid and biofilm protein A\n",
      "b0806 ['Uncharacterized periplasmic protein McbA']\n",
      "\n",
      "KO KO:K06970 23S rRNA (adenine1618-N6)-methyltransferase [EC:2.1.1.181]\n",
      "SSO SSO:000000222 23S rRNA (adenine(1618)-N(6))-methyltransferase (EC 2.1.1.181)\n",
      "b0807 ['23S rRNA (adenine(1618)-N(6))-methyltransferase (EC 2.1.1.181)']\n",
      "\n",
      "KO KO:K22044 moderate conductance mechanosensitive channel\n",
      "b0808 ['Potassium efflux system KefA protein', 'Small-conductance mechanosensitive channel']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "KO KO:K02028 polar amino acid transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10038 glutamine transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000017274 Glutamine ABC transporter, ATP-binding protein GlnQ\n",
      "b0809 ['Glutamine ABC transporter, ATP-binding protein GlnQ']\n",
      "\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02029 polar amino acid transport system permease protein\n",
      "SSO SSO:000017276 Glutamine ABC transporter, permease protein GlnP\n",
      "b0810 ['Glutamine ABC transporter, permease protein GlnP']\n",
      "\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02030 polar amino acid transport system substrate-binding protein\n",
      "KO KO:K10036 glutamine transport system substrate-binding protein\n",
      "SSO SSO:000017277 Glutamine ABC transporter, substrate-binding protein GlnH\n",
      "b0811 ['Glutamine ABC transporter, substrate-binding protein GlnH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K04047 starvation-inducible DNA-binding protein\n",
      "SSO SSO:000002094 DNA protection during starvation protein\n",
      "b0812 ['DNA protection during starvation protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K11939 inner membrane transporter RhtA\n",
      "SSO SSO:000025640 Threonine/homoserine exporter RhtA\n",
      "b0813 ['Threonine/homoserine exporter RhtA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K11934 outer membrane protein X\n",
      "SSO SSO:000037381 Outer membrane protein X\n",
      "b0814 ['Outer membrane protein X precursor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000021649 Phosphoethanolamine transferase OpgE\n",
      "b0815 ['Phosphoethanolamine transferase OpgE']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4705 ['hypothetical protein']\n",
      "\n",
      "KO KO:K11924 DtxR family transcriptional regulator, manganese transport regulator\n",
      "SSO SSO:000019939 Mn-dependent transcriptional regulator MntR\n",
      "b0817 ['Mn-dependent transcriptional regulator MntR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000036383 Inner membrane protein YbiR, putative anion permease\n",
      "b0818 ['Inner membrane protein YbiR, putative anion permease']\n",
      "\n",
      "KO KO:K19235 L,D-transpeptidase YbiS\n",
      "b0819 ['L,D-transpeptidase => YbiS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06158 ATP-binding cassette, subfamily F, member 3\n",
      "SSO SSO:000011184 Bis-ABC ATPase YbiT\n",
      "b0820 ['Bis-ABC ATPase YbiT']\n",
      "\n",
      "SSO SSO:000028181 Uncharacterized protein YbiU\n",
      "b0821 ['Uncharacterized protein YbiU']\n",
      "\n",
      "KO KO:K07757 sugar-phosphatase [EC:3.1.3.23]\n",
      "SSO SSO:000025244 Sugar phosphatase YbiV\n",
      "b0822 ['Sugar phosphatase YbiV']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K00656 formate C-acetyltransferase [EC:2.3.1.54]\n",
      "SSO SSO:000006850 Pyruvate formate-lyase (EC 2.3.1.54)\n",
      "b0823 ['Pyruvate formate-lyase (EC 2.3.1.54)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04069 pyruvate formate lyase activating enzyme [EC:1.97.1.4]\n",
      "SSO SSO:000006851 Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)\n",
      "b0824 ['Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)']\n",
      "\n",
      "KO KO:K08313 fructose-6-phosphate aldolase 1 [EC:4.1.2.-]\n",
      "SSO SSO:000035974 Fructose-6-phosphate aldolase\n",
      "b0825 ['Fructose-6-phosphate aldolase']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K21029 molybdopterin-synthase adenylyltransferase [EC:2.7.7.80]\n",
      "SSO SSO:000004948 Molybdopterin-synthase adenylyltransferase (EC 2.7.7.80)\n",
      "b0826 ['Molybdopterin-synthase adenylyltransferase (EC 2.7.7.80)']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03750 molybdopterin molybdotransferase [EC:2.10.1.1]\n",
      "SSO SSO:000004943 Molybdopterin molybdenumtransferase (EC 2.10.1.1)\n",
      "b0827 ['Molybdopterin molybdenumtransferase (EC 2.10.1.1)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K13051 beta-aspartyl-peptidase (threonine type) [EC:3.4.19.5]\n",
      "b0828 ['Isoaspartyl aminopeptidase (EC 3.4.19.5)', 'Asp-X dipeptidase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02031 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K02032 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K13892 glutathione transport system ATP-binding protein\n",
      "b0829 ['Oligopeptide transport ATP-binding protein', 'Glutathione ABC transporter, ATP-binding protein GsiA']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "KO KO:K13889 glutathione transport system substrate-binding protein\n",
      "b0830 ['Oligopeptide transport substrate-binding protein', 'Glutathione ABC transporter, substrate-binding protein GsiB']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02033 peptide/nickel transport system permease protein\n",
      "KO KO:K13890 glutathione transport system permease protein\n",
      "b0831 ['Oligopeptide transport permease protein OppB-like', 'Glutathione ABC transporter, permease protein GsiC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02034 peptide/nickel transport system permease protein\n",
      "KO KO:K13891 glutathione transport system permease protein\n",
      "b0832 ['Oligopeptide transport permease protein OppC-like', 'Glutathione ABC transporter, permease protein GsiD']\n",
      "\n",
      "b0833 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeI']\n",
      "\n",
      "b0834 ['Diguanylate cyclase (EC 2.7.7.65) => DgcI']\n",
      "\n",
      "KO KO:K14441 ribosomal protein S12 methylthiotransferase [EC:2.8.4.4]\n",
      "SSO SSO:000007131 Ribosomal protein S12p Asp88 (E. coli) methylthiotransferase (EC 2.8.4.4)\n",
      "b0835 ['Ribosomal protein S12p Asp88 (E. coli) methylthiotransferase (EC 2.8.4.4)']\n",
      "\n",
      "KO KO:K19688 biofilm regulator BssR\n",
      "SSO SSO:000011144 Biofilm regulator BssR\n",
      "b0836 ['Biofilm regulator BssR']\n",
      "\n",
      "KO KO:K21430 aldose sugar dehydrogenase [EC:1.1.5.-]\n",
      "SSO SSO:000007618 Soluble aldose sugar dehydrogenase, PQQ-dependent (EC 1.1.5.-)\n",
      "b0837 ['Soluble aldose sugar dehydrogenase, PQQ-dependent (EC 1.1.5.-)']\n",
      "\n",
      "KO KO:K00799 glutathione S-transferase [EC:2.5.1.18]\n",
      "SSO SSO:000003209 Glutathione S-transferase (EC 2.5.1.18)\n",
      "SSO SSO:000008730 Uncharacterized glutathione S-transferase-like protein\n",
      "b0838 ['Uncharacterized glutathione S-transferase-like protein']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K07258 serine-type D-Ala-D-Ala carboxypeptidase (penicillin-binding protein 5/6) [EC:3.4.16.4]\n",
      "SSO SSO:000001995 D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)\n",
      "b0839 ['D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K11534 DeoR family transcriptional regulator, deoxyribose operon repressor\n",
      "SSO SSO:000035467 Deoxyribose operon repressor DeoR\n",
      "b0840 ['Deoxyribose operon repressor DeoR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K19302 undecaprenyl-diphosphatase [EC:3.6.1.27]\n",
      "SSO SSO:000041592 Undecaprenyl-diphosphatase BcrC (EC 3.6.1.27), conveys bacitracin resistance\n",
      "b0841 ['Undecaprenyl-diphosphatase BcrC (EC 3.6.1.27), conveys bacitracin resistance']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K08160 MFS transporter, DHA1 family, multidrug/chloramphenicol efflux transport protein\n",
      "SSO SSO:000036923 Multidrug efflux pump MdfA/Cmr (of MFS type), broad spectrum\n",
      "b0842 ['Multidrug efflux pump MdfA/Cmr (of MFS type), broad spectrum']\n",
      "\n",
      "SSO SSO:000028183 Uncharacterized protein YbjH\n",
      "b0843 ['Uncharacterized protein YbjH']\n",
      "\n",
      "KO KO:K20861 FMN hydrolase / 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphatase [EC:3.1.3.102 3.1.3.104]\n",
      "SSO SSO:000009723 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphatase YbjI\n",
      "b0844 ['5-amino-6-(5-phospho-D-ribitylamino)uracil phosphatase YbjI']\n",
      "\n",
      "SSO SSO:000018384 Inner membrane protein YbjJ\n",
      "b0845 ['Inner membrane protein YbjJ']\n",
      "\n",
      "KO KO:K23778 Unknown\n",
      "SSO SSO:000017604 HTH-type transcriptional regulator RcdA\n",
      "b0846 ['HTH-type transcriptional regulator RcdA']\n",
      "\n",
      "KO KO:K07085 putative transport protein\n",
      "SSO SSO:000023658 Putative transport protein YbjL\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b0847 ['Putative transport protein YbjL']\n",
      "\n",
      "SSO SSO:000018385 Inner membrane protein YbjM\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b0848 ['Inner membrane protein YbjM']\n",
      "\n",
      "KO KO:K03674 glutaredoxin 1\n",
      "SSO SSO:000003199 Glutaredoxin 1\n",
      "b0849 ['Glutaredoxin 1']\n",
      "\n",
      "SSO SSO:000028182 Uncharacterized protein YbjC\n",
      "b0850 ['Uncharacterized protein YbjC']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K10678 nitroreductase [EC:1.-.-.-]\n",
      "KO KO:K19285 FMN reductase (NADPH) [EC:1.5.1.38]\n",
      "SSO SSO:000020988 Oxygen-insensitive NAD(P)H nitroreductase\n",
      "b0851 ['Oxygen-insensitive NADPH nitroreductase (EC 1.-.-.-)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K05844 ribosomal protein S6--L-glutamate ligase [EC:6.3.2.-]\n",
      "SSO SSO:000007133 Ribosomal protein S6--L-glutamate ligase\n",
      "b0852 ['Ribosomal protein S6--L-glutamate ligase']\n",
      "\n",
      "SSO SSO:000028691 Uncharacterized sensory transduction regulator YbjN\n",
      "b0853 ['Uncharacterized sensory transduction regulator YbjN']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0015847 putrescine transport\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K11073 putrescine transport system substrate-binding protein\n",
      "SSO SSO:000006780 Putrescine ABC transporter putrescine-binding protein PotF (TC 3.A.1.11.2)\n",
      "b0854 ['Putrescine ABC transporter putrescine-binding protein PotF (TC 3.A.1.11.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0015847 putrescine transport\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "KO KO:K11076 putrescine transport system ATP-binding protein\n",
      "SSO SSO:000006784 Putrescine transport ATP-binding protein PotG (TC 3.A.1.11.2)\n",
      "b0855 ['Putrescine transport ATP-binding protein PotG (TC 3.A.1.11.2)']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0015847 putrescine transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11071 spermidine/putrescine transport system permease protein\n",
      "KO KO:K11075 putrescine transport system permease protein\n",
      "SSO SSO:000006785 Putrescine transport system permease protein PotH (TC 3.A.1.11.2)\n",
      "b0856 ['Putrescine transport system permease protein PotH (TC 3.A.1.11.2)']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0015847 putrescine transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11070 spermidine/putrescine transport system permease protein\n",
      "KO KO:K11074 putrescine transport system permease protein\n",
      "SSO SSO:000006786 Putrescine transport system permease protein PotI (TC 3.A.1.11.2)\n",
      "b0857 ['Putrescine transport system permease protein PotI (TC 3.A.1.11.2)']\n",
      "\n",
      "SSO SSO:000018386 Inner membrane protein YbjO\n",
      "b0858 ['Inner membrane protein YbjO']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03212 23S rRNA (uracil747-C5)-methyltransferase [EC:2.1.1.189]\n",
      "SSO SSO:000000242 23S rRNA (uracil(747)-C(5))-methyltransferase (EC 2.1.1.189)\n",
      "b0859 ['23S rRNA (uracil(747)-C(5))-methyltransferase (EC 2.1.1.189)']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02030 polar amino acid transport system substrate-binding protein\n",
      "KO KO:K09996 arginine transport system substrate-binding protein\n",
      "SSO SSO:000010789 Arginine ABC transporter, substrate-binding protein ArtJ\n",
      "b0860 ['Arginine ABC transporter, substrate-binding protein ArtJ']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K09998 arginine transport system permease protein\n",
      "SSO SSO:000010786 Arginine ABC transporter, permease protein ArtM\n",
      "b0861 ['Arginine ABC transporter, permease protein ArtM']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02029 polar amino acid transport system permease protein\n",
      "KO KO:K09999 arginine transport system permease protein\n",
      "SSO SSO:000010787 Arginine ABC transporter, permease protein ArtQ\n",
      "b0862 ['Arginine ABC transporter, permease protein ArtQ']\n",
      "\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02030 polar amino acid transport system substrate-binding protein\n",
      "KO KO:K09997 arginine transport system substrate-binding protein\n",
      "SSO SSO:000010788 Arginine ABC transporter, substrate-binding protein ArtI\n",
      "b0863 ['Arginine ABC transporter, substrate-binding protein ArtI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K10000 arginine transport system ATP-binding protein [EC:7.4.2.1]\n",
      "SSO SSO:000010785 Arginine ABC transporter, ATP-binding protein ArtP\n",
      "b0864 ['Arginine ABC transporter, ATP-binding protein ArtP']\n",
      "\n",
      "SSO SSO:000018986 Lipoprotein\n",
      "SSO SSO:000027203 Uncharacterized lipoprotein YbjP\n",
      "b0865 ['Uncharacterized lipoprotein YbjP']\n",
      "\n",
      "SSO SSO:000026550 UPF0145 protein YbjQ\n",
      "b0866 ['UPF0145 protein YbjQ']\n",
      "\n",
      "KO KO:K01447 N-acetylmuramoyl-L-alanine amidase [EC:3.5.1.28]\n",
      "KO KO:K11066 N-acetylmuramoyl-L-alanine amidase [EC:3.5.1.28]\n",
      "b0867 ['N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)', '1,6-anhydro-N-acetylmuramyl-L-alanine amidase']\n",
      "\n",
      "SSO SSO:000026967 Uncharacterized NAD(P)-dependent epimerase/dehydratase family protein YbjS\n",
      "b0868 ['Uncharacterized NAD(P)-dependent epimerase/dehydratase family protein YbjS']\n",
      "\n",
      "SSO SSO:000028184 Uncharacterized protein YbjT\n",
      "b0869 ['Uncharacterized protein YbjT']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006545 glycine biosynthetic process\n",
      "KO KO:K01620 threonine aldolase [EC:4.1.2.48]\n",
      "SSO SSO:000004554 Low-specificity L-threonine aldolase (EC 4.1.2.48)\n",
      "b0870 ['Low-specificity L-threonine aldolase (EC 4.1.2.48)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042867 pyruvate catabolic process\n",
      "KO KO:K00156 pyruvate dehydrogenase (quinone) [EC:1.2.5.1]\n",
      "SSO SSO:000006846 Pyruvate dehydrogenase (quinone) (EC 1.2.5.1)\n",
      "b0871 ['Pyruvate dehydrogenase (quinone) (EC 1.2.5.1)']\n",
      "\n",
      "KO KO:K11933 NADH oxidoreductase Hcr [EC:1.-.-.-]\n",
      "SSO SSO:000020326 NADH oxidoreductase for hydroxylamine reductase\n",
      "b0872 ['NADH oxidoreductase for hydroxylamine reductase']\n",
      "\n",
      "KO KO:K05601 hydroxylamine reductase [EC:1.7.99.1]\n",
      "SSO SSO:000017976 Hydroxylamine reductase (EC 1.7.99.1)\n",
      "b0873 ['Hydroxylamine reductase (EC 1.7.99.1)']\n",
      "\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000036702 Lysine exporter LysO\n",
      "b0874 ['Lysine exporter LysO']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06188 aquaporin Z\n",
      "SSO SSO:000001024 Aquaporin Z\n",
      "b0875 ['Aquaporin Z']\n",
      "\n",
      "KO KO:K07459 putative ATP-dependent endonuclease of the OLD family\n",
      "SSO SSO:000006220 Predicted ATP-dependent endonuclease of the OLD family, YbjD subgroup\n",
      "b0876 ['Predicted ATP-dependent endonuclease of the OLD family, YbjD subgroup']\n",
      "\n",
      "KO KO:K09824 uncharacterized protein\n",
      "SSO SSO:000008936 Virulence factor VirK\n",
      "b0877 ['Virulence factor VirK']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02005 HlyD family secretion protein\n",
      "KO KO:K13888 membrane fusion protein, macrolide-specific efflux system\n",
      "SSO SSO:000004628 Macrolide-specific efflux protein MacA\n",
      "b0878 ['Macrolide-specific efflux protein MacA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02003 putative ABC transport system ATP-binding protein\n",
      "KO KO:K02004 putative ABC transport system permease protein\n",
      "KO KO:K05685 macrolide transport system ATP-binding/permease protein [EC:3.6.3.-]\n",
      "SSO SSO:000004627 Macrolide export ATP-binding/permease protein MacB (EC 3.6.3.-)\n",
      "b0879 ['Macrolide export ATP-binding/permease protein MacB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03704 cold shock protein (beta-ribbon, CspA family)\n",
      "b0880 ['Cold shock protein of CSP family => CspD (naming convention as in E.coli)']\n",
      "\n",
      "KO KO:K06891 ATP-dependent Clp protease adaptor protein ClpS\n",
      "SSO SSO:000000565 ATP-dependent Clp protease adaptor protein ClpS\n",
      "b0881 ['ATP-dependent Clp protease adaptor protein ClpS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03694 ATP-dependent Clp protease ATP-binding subunit ClpA\n",
      "SSO SSO:000000563 ATP-dependent Clp protease ATP-binding subunit ClpA\n",
      "b0882 ['ATP-dependent Clp protease ATP-binding subunit ClpA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02518 translation initiation factor IF-1\n",
      "SSO SSO:000008308 Translation initiation factor 1\n",
      "b0884 ['Translation initiation factor 1']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006508 proteolysis\n",
      "KO KO:K00684 leucyl/phenylalanyl-tRNA---protein transferase [EC:2.3.2.6]\n",
      "SSO SSO:000004426 Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)\n",
      "b0885 ['Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16012 ATP-binding cassette, subfamily C, bacterial CydC\n",
      "b0886 ['Efflux ABC transporter for glutathione/L-cysteine, essential for assembly of bd-type respiratory oxidases => CydC subunit']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16013 ATP-binding cassette, subfamily C, bacterial CydD\n",
      "b0887 ['Efflux ABC transporter for glutathione/L-cysteine, essential for assembly of bd-type respiratory oxidases => CydD subunit']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00384 thioredoxin reductase (NADPH) [EC:1.8.1.9]\n",
      "SSO SSO:000008043 Thioredoxin reductase (EC 1.8.1.9)\n",
      "b0888 ['Thioredoxin reductase (EC 1.8.1.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03719 Lrp/AsnC family transcriptional regulator, leucine-responsive regulatory protein\n",
      "SSO SSO:000004421 Leucine-responsive regulatory protein, regulator for leucine (or lrp) regulon and high-affinity branched-chain amino acid transport system\n",
      "b0889 ['Leucine-responsive regulatory protein, regulator for leucine (or lrp) regulon and high-affinity branched-chain amino acid transport system']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009432 SOS response\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03466 DNA segregation ATPase FtsK/SpoIIIE, S-DNA-T family\n",
      "SSO SSO:000002128 DNA translocase FtsK\n",
      "b0890 ['DNA translocase FtsK']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03634 outer membrane lipoprotein carrier protein\n",
      "SSO SSO:000020878 Outer membrane lipoprotein carrier protein LolA\n",
      "b0891 ['Outer membrane lipoprotein carrier protein LolA']\n",
      "\n",
      "KO KO:K07478 putative ATPase\n",
      "SSO SSO:000038974 Replication-associated recombination protein RarA\n",
      "b0892 ['Replication-associated recombination protein RarA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01875 seryl-tRNA synthetase [EC:6.1.1.11]\n",
      "SSO SSO:000007452 Seryl-tRNA synthetase (EC 6.1.1.11)\n",
      "b0893 ['Seryl-tRNA synthetase (EC 6.1.1.11)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07306 anaerobic dimethyl sulfoxide reductase subunit A [EC:1.8.5.3]\n",
      "SSO SSO:000000978 Anaerobic dimethyl sulfoxide reductase chain A , molybdopterin-binding domain (EC 1.8.5.3)\n",
      "b0894 ['Anaerobic dimethyl sulfoxide reductase chain A (EC 1.8.5.3), molybdopterin-binding domain']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00124 formate dehydrogenase iron-sulfur subunit\n",
      "KO KO:K07307 anaerobic dimethyl sulfoxide reductase subunit B (DMSO reductase iron- sulfur subunit)\n",
      "SSO SSO:000000979 Anaerobic dimethyl sulfoxide reductase chain B , iron-sulfur binding subunit (EC 1.8.5.3)\n",
      "b0895 ['Anaerobic dimethyl sulfoxide reductase chain B (EC 1.8.5.3), iron-sulfur binding subunit']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07308 anaerobic dimethyl sulfoxide reductase subunit C (DMSO reductase anchor subunit)\n",
      "b0896 ['Anaerobic dimethyl sulfoxide reductase chain C (EC 1.8.5.3), anchor subunit']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "SSO SSO:000005299 Nicotinamidase family protein YcaC\n",
      "b0897 ['Nicotinamidase family protein YcaC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08219 MFS transporter, UMF2 family, putative MFS family transporter protein\n",
      "SSO SSO:000017994 Hypothetical MFS-type transporter protein YcaD\n",
      "b0898 ['Hypothetical MFS-type transporter protein YcaD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018374 Inner membrane permease YcaM\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b0899 ['Inner membrane permease YcaM']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000039689 Transcriptional regulator YcaN, LysR family\n",
      "b0900 ['Transcriptional regulator YcaN, LysR family']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K00355 NAD(P)H dehydrogenase (quinone) [EC:1.6.5.2]\n",
      "b0901 ['putative NAD(P)H dehydrogenase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04069 pyruvate formate lyase activating enzyme [EC:1.97.1.4]\n",
      "SSO SSO:000006851 Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)\n",
      "b0902 ['Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00656 formate C-acetyltransferase [EC:2.3.1.54]\n",
      "SSO SSO:000006850 Pyruvate formate-lyase (EC 2.3.1.54)\n",
      "b0903 ['Pyruvate formate-lyase (EC 2.3.1.54)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06212 formate transporter\n",
      "SSO SSO:000035959 Formate efflux transporter FocA\n",
      "b0904 ['Formate efflux transporter FocA']\n",
      "\n",
      "KO KO:K09136 ribosomal protein S12 methylthiotransferase accessory factor\n",
      "SSO SSO:000007132 Ribosomal protein S12p methylthiotransferase accessory factor YcaO\n",
      "b0905 ['Ribosomal protein S12p methylthiotransferase accessory factor YcaO']\n",
      "\n",
      "b0906 ['Putative inner membrane protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006564 L-serine biosynthetic process\n",
      "KO KO:K00831 phosphoserine aminotransferase [EC:2.6.1.52]\n",
      "SSO SSO:000006050 Phosphoserine aminotransferase (EC 2.6.1.52)\n",
      "b0907 ['Phosphoserine aminotransferase (EC 2.6.1.52)']\n",
      "\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K00800 3-phosphoshikimate 1-carboxyvinyltransferase [EC:2.5.1.19]\n",
      "SSO SSO:000034048 3-phosphoshikimate 1-carboxyvinyltransferase (EC 2.5.1.19)\n",
      "b0908 ['3-phosphoshikimate 1-carboxyvinyltransferase (EC 2.5.1.19)']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K07387 metalloprotease [EC:3.4.24.-]\n",
      "b0909 ['Putative heat shock protein YcaL']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00945 CMP/dCMP kinase [EC:2.7.4.25]\n",
      "SSO SSO:000001874 Cytidylate kinase (EC 2.7.4.25)\n",
      "b0910 ['Cytidylate kinase (EC 2.7.4.25)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02945 small subunit ribosomal protein S1\n",
      "SSO SSO:000007283 SSU ribosomal protein S1p\n",
      "b0911 ['SSU ribosomal protein S1p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K05788 integration host factor subunit beta\n",
      "SSO SSO:000009160 integration host factor beta subunit\n",
      "b0912 ['Integration host factor beta subunit']\n",
      "\n",
      "KO KO:K02238 competence protein ComEC\n",
      "SSO SSO:000002072 DNA internalization-related competence protein ComEC/Rec2\n",
      "b0913 ['DNA internalization-related competence protein ComEC/Rec2']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06147 ATP-binding cassette, subfamily B, bacterial\n",
      "KO KO:K11085 ATP-binding cassette, subfamily B, bacterial MsbA [EC:3.6.3.-]\n",
      "SSO SSO:000018959 Lipid A export permease/ATP-binding protein MsbA\n",
      "b0914 ['Lipid A export permease/ATP-binding protein MsbA']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009248 K antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00912 tetraacyldisaccharide 4'-kinase [EC:2.7.1.130]\n",
      "SSO SSO:000007937 Tetraacyldisaccharide 4'-kinase (EC 2.7.1.130)\n",
      "b0915 [\"Tetraacyldisaccharide 4'-kinase (EC 2.7.1.130)\"]\n",
      "\n",
      "KO KO:K09927 uncharacterized protein\n",
      "b0916 ['FIG004798: Putative cytoplasmic protein']\n",
      "\n",
      "KO KO:K09791 uncharacterized protein\n",
      "SSO SSO:000013675 FIG002473: Protein YcaR in KDO2-Lipid A biosynthesis cluster\n",
      "b0917 ['FIG002473: Protein YcaR in KDO2-Lipid A biosynthesis cluster']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009248 K antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00979 3-deoxy-manno-octulosonate cytidylyltransferase (CMP-KDO synthetase) [EC:2.7.7.38]\n",
      "SSO SSO:000000281 3-deoxy-manno-octulosonate cytidylyltransferase (EC 2.7.7.38)\n",
      "b0918 ['3-deoxy-manno-octulosonate cytidylyltransferase (EC 2.7.7.38)']\n",
      "\n",
      "SSO SSO:000028187 Uncharacterized protein YcbJ\n",
      "b0919 ['Uncharacterized protein YcbJ']\n",
      "\n",
      "SSO SSO:000004731 Membrane Protein Functionally coupled to the MukBEF Chromosome Partitioning Mechanism\n",
      "b0920 ['Membrane Protein Functionally coupled to the MukBEF Chromosome Partitioning Mechanism']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "KO KO:K06219 S-adenosylmethionine-dependent methyltransferase\n",
      "b0921 ['tRNA uridine 5-oxyacetic acid(34) methyltransferase (EC 2.1.1.-)']\n",
      "\n",
      "KO KO:K03633 chromosome partition protein MukF\n",
      "SSO SSO:000001569 Chromosome partition protein MukF\n",
      "b0922 ['Chromosome partition protein MukF']\n",
      "\n",
      "KO KO:K03804 chromosome partition protein MukE\n",
      "SSO SSO:000001568 Chromosome partition protein MukE\n",
      "b0923 ['Chromosome partition protein MukE']\n",
      "\n",
      "KO KO:K03632 chromosome partition protein MukB\n",
      "SSO SSO:000001567 Chromosome partition protein MukB\n",
      "b0924 ['Chromosome partition protein MukB']\n",
      "\n",
      "KO KO:K21470 L,D-transpeptidase YcbB\n",
      "SSO SSO:000018720 L,D-transpeptidase YcbB\n",
      "b0925 ['L,D-transpeptidase YcbB']\n",
      "\n",
      "SSO SSO:000013653 FIG001587: exported protein\n",
      "b0926 ['FIG001587: exported protein']\n",
      "\n",
      "KO KO:K01069 hydroxyacylglutathione hydrolase [EC:3.1.2.6]\n",
      "SSO SSO:000003572 Hypothetical metal-binding enzyme, YcbL homolog\n",
      "b0927 ['Hypothetical metal-binding enzyme, YcbL homolog']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006532 aspartate biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00813 aspartate aminotransferase [EC:2.6.1.1]\n",
      "SSO SSO:000001093 Aspartate aminotransferase (EC 2.6.1.1)\n",
      "b0928 ['Aspartate aminotransferase (EC 2.6.1.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000005575 Outer membrane porin OmpF\n",
      "b0929 ['Outer membrane porin OmpF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01893 asparaginyl-tRNA synthetase [EC:6.1.1.22]\n",
      "SSO SSO:000001088 Asparaginyl-tRNA synthetase (EC 6.1.1.22)\n",
      "b0930 ['Asparaginyl-tRNA synthetase (EC 6.1.1.22)']\n",
      "\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K00763 nicotinate phosphoribosyltransferase [EC:6.3.4.21]\n",
      "SSO SSO:000005313 Nicotinate phosphoribosyltransferase (EC 6.3.4.21)\n",
      "b0931 ['Nicotinate phosphoribosyltransferase (EC 6.3.4.21)']\n",
      "\n",
      "KO KO:K01256 aminopeptidase N [EC:3.4.11.2]\n",
      "SSO SSO:000004732 Membrane alanine aminopeptidase N (EC 3.4.11.2)\n",
      "SSO SSO:000010610 Aminopeptidase N\n",
      "b0932 ['Membrane alanine aminopeptidase N (EC 3.4.11.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K15555 sulfonate transport system ATP-binding protein [EC:3.6.3.-]\n",
      "SSO SSO:000010421 Alkanesulfonate ABC transporter ATP-binding protein SsuB\n",
      "b0933 ['Alkanesulfonate ABC transporter ATP-binding protein SsuB']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02050 NitT/TauT family transport system permease protein\n",
      "KO KO:K15554 sulfonate transport system permease protein\n",
      "SSO SSO:000010422 Alkanesulfonate ABC transporter permease protein SsuC\n",
      "b0934 ['Alkanesulfonate ABC transporter permease protein SsuC']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K04091 alkanesulfonate monooxygenase [EC:1.14.14.5]\n",
      "SSO SSO:000035873 FMNH2-dependent alkanesulfonate monooxygenase (EC 1.14.14.5)\n",
      "b0935 ['FMNH2-dependent alkanesulfonate monooxygenase (EC 1.14.14.5)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02051 NitT/TauT family transport system substrate-binding protein\n",
      "KO KO:K15553 sulfonate transport system substrate-binding protein\n",
      "SSO SSO:000010423 Alkanesulfonate ABC transporter substrate-binding protein SsuA\n",
      "b0936 ['Alkanesulfonate ABC transporter substrate-binding protein SsuA']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K00299 FMN reductase [EC:1.5.1.38]\n",
      "SSO SSO:000016585 FMN reductase (NADPH) (EC 1.5.1.38)\n",
      "SSO SSO:000020357 NADPH-dependent FMN reductase\n",
      "b0937 ['FMN reductase (NADPH) (EC 1.5.1.38)']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "SSO SSO:000002783 Fimbrial subunit ElfA\n",
      "b0938 ['Fimbrial subunit ElfA']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K07346 fimbrial chaperone protein\n",
      "SSO SSO:000006326 Probable fimbrial chaperone protein ElfD\n",
      "b0939 ['Probable fimbrial chaperone protein ElfD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07347 outer membrane usher protein\n",
      "SSO SSO:000006336 Probable outer membrane usher protein ElfC\n",
      "b0940 ['Probable outer membrane usher protein ElfC']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000008712 Uncharacterized fimbrial-like protein ElfG\n",
      "b0941 ['Uncharacterized fimbrial-like protein ElfG']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000008722 Uncharacterized fimbrial-like protein YcbU\n",
      "b0942 ['Uncharacterized fimbrial-like protein YcbU']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "SSO SSO:000008723 Uncharacterized fimbrial-like protein YcbV\n",
      "b0943 ['Uncharacterized fimbrial-like protein YcbV']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000008710 Uncharacterized fimbrial chaperone YcbF\n",
      "b0944 ['Uncharacterized fimbrial chaperone YcbF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "KO KO:K00254 dihydroorotate dehydrogenase [EC:1.3.5.2]\n",
      "SSO SSO:000002311 Dihydroorotate dehydrogenase (quinone) (EC 1.3.5.2)\n",
      "b0945 ['Dihydroorotate dehydrogenase (quinone) (EC 1.3.5.2)']\n",
      "\n",
      "KO KO:K18657 cell division protein ZapC\n",
      "SSO SSO:000001476 Cell division protein ZapC\n",
      "b0946 ['Cell division protein ZapC']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K07140 uncharacterized protein\n",
      "SSO SSO:000013224 Flavodoxin reductases (ferredoxin-NADPH reductases) family 1\n",
      "b0947 ['Flavodoxin reductases (ferredoxin-NADPH reductases) family 1']\n",
      "\n",
      "KO KO:K07444 putative N6-adenine-specific DNA methylase [EC:2.1.1.-]\n",
      "KO KO:K12297 23S rRNA (guanine2445-N2)-methyltransferase / 23S rRNA (guanine2069-N7)-methyltransferase [EC:2.1.1.173 2.1.1.264]\n",
      "b0948 ['23S rRNA (guanine(2445)-N(2))-methyltransferase (EC 2.1.1.173)', '23S rRNA (guanine(2069)-N(7))-methyltransferase (EC 2.1.1.264)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K15738 ABC transport system ATP-binding/permease protein\n",
      "SSO SSO:000011183 Bis-ABC ATPase Uup\n",
      "b0949 ['Bis-ABC ATPase Uup']\n",
      "\n",
      "KO KO:K03808 paraquat-inducible protein A\n",
      "b0950 ['Paraquat-inducible protein A']\n",
      "\n",
      "KO KO:K06192 paraquat-inducible protein B\n",
      "b0951 ['Paraquat-inducible protein B']\n",
      "\n",
      "KO KO:K09857 uncharacterized protein\n",
      "b0952 ['Uncharacterized lipoprotein YmbA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03812 ribosome modulation factor\n",
      "SSO SSO:000007145 Ribosome modulation factor\n",
      "b0953 ['Ribosome modulation factor']\n",
      "\n",
      "KO KO:K01716 3-hydroxyacyl-[acyl-carrier protein] dehydratase / trans-2-decenoyl-[acyl-carrier protein] isomerase [EC:4.2.1.59 5.3.3.14]\n",
      "b0954 ['3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabA form (EC 4.2.1.59)', 'Trans-2-decenoyl-[acyl-carrier-protein] isomerase (EC 5.3.3.14)']\n",
      "\n",
      "GO GO:0006508 proteolysis\n",
      "KO KO:K04770 Lon-like ATP-dependent protease [EC:3.4.21.-]\n",
      "SSO SSO:000019038 Lon protease homolog YcbZ\n",
      "SSO SSO:000023495 Putative peptidase\n",
      "b0955 ['Lon protease homolog YcbZ']\n",
      "\n",
      "KO KO:K09911 uncharacterized protein\n",
      "SSO SSO:000023250 Putative dehydrogenase\n",
      "b0956 ['Putative dehydrogenase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K03286 OmpA-OmpF porin, OOP family\n",
      "SSO SSO:000005589 Outer membrane protein A precursor\n",
      "b0957 ['Outer membrane protein A precursor']\n",
      "\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009432 SOS response\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K13053 cell division inhibitor SulA\n",
      "SSO SSO:000035037 Cell division inhibitor SulA\n",
      "b0958 ['Cell division inhibitor SulA']\n",
      "\n",
      "KO KO:K07343 DNA transformation protein and related proteins\n",
      "SSO SSO:000002127 DNA transformation protein TfoX\n",
      "b0959 ['DNA transformation protein TfoX']\n",
      "\n",
      "SSO SSO:000023271 Putative efflux (PET) family inner membrane protein YccS\n",
      "b0960 ['Putative efflux (PET) family inner membrane protein YccS']\n",
      "\n",
      "SSO SSO:000018387 Inner membrane protein YccF\n",
      "b0961 ['Inner membrane protein YccF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K03658 DNA helicase IV [EC:3.6.4.12]\n",
      "SSO SSO:000002070 DNA helicase IV (EC 3.6.4.12)\n",
      "b0962 ['DNA helicase IV (EC 3.6.4.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009438 methylglyoxal metabolic process\n",
      "KO KO:K01734 methylglyoxal synthase [EC:4.2.3.3]\n",
      "SSO SSO:000004880 Methylglyoxal synthase (EC 4.2.3.3)\n",
      "b0963 ['Methylglyoxal synthase (EC 4.2.3.3)']\n",
      "\n",
      "KO KO:K09909 uncharacterized protein\n",
      "b0964 ['UPF0319 protein YccT precursor']\n",
      "\n",
      "KO KO:K06929 uncharacterized protein\n",
      "SSO SSO:000025203 Succinyl-CoA synthetase, alpha subunit-related enzymes\n",
      "b0965 ['Succinyl-CoA synthetase, alpha subunit-related enzymes']\n",
      "\n",
      "KO KO:K11940 heat shock protein HspQ\n",
      "SSO SSO:000036243 Heat shock protein HspQ\n",
      "b0966 ['Heat shock protein HspQ']\n",
      "\n",
      "KO KO:K06969 23S rRNA (cytosine1962-C5)-methyltransferase [EC:2.1.1.191]\n",
      "SSO SSO:000000231 23S rRNA (cytosine(1962)-C(5))-methyltransferase (EC 2.1.1.191)\n",
      "b0967 ['23S rRNA (cytosine(1962)-C(5))-methyltransferase (EC 2.1.1.191)']\n",
      "\n",
      "KO KO:K01512 acylphosphatase [EC:3.6.1.7]\n",
      "SSO SSO:000010292 Acylphosphate phosphohydrolase (EC 3.6.1.7)\n",
      "b0968 ['Acylphosphate phosphohydrolase (EC 3.6.1.7)']\n",
      "\n",
      "KO KO:K11179 tRNA 2-thiouridine synthesizing protein E [EC:2.8.1.-]\n",
      "SSO SSO:000009347 tRNA 2-thiouridine synthesis protein TusE\n",
      "b0969 ['tRNA 2-thiouridine synthesis protein TusE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K19416 modulator of FtsH protease\n",
      "SSO SSO:000023118 Putative TEGT family carrier/transport protein\n",
      "b0970 ['Putative TEGT family carrier/transport protein']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06282 hydrogenase small subunit [EC:1.12.99.6]\n",
      "SSO SSO:000041612 Uptake [NiFe] hydrogenase, small subunit HyaA (EC 1.12.99.6)\n",
      "b0972 ['Uptake [NiFe] hydrogenase, small subunit HyaA (EC 1.12.99.6)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06281 hydrogenase large subunit [EC:1.12.99.6]\n",
      "SSO SSO:000041611 Uptake [NiFe] hydrogenase, large subunit HyaB (EC 1.12.99.6)\n",
      "b0973 ['Uptake [NiFe] hydrogenase, large subunit HyaB (EC 1.12.99.6)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03620 Ni/Fe-hydrogenase 1 B-type cytochrome subunit\n",
      "SSO SSO:000041610 Uptake [NiFe] hydrogenase, cytochrome b subunit HyaC\n",
      "b0974 ['Uptake [NiFe] hydrogenase, cytochrome b subunit HyaC']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K03605 hydrogenase maturation protease [EC:3.4.23.-]\n",
      "SSO SSO:000041609 Uptake [NiFe] hydrogenase maturation protease HyaD\n",
      "b0975 ['Uptake [NiFe] hydrogenase maturation protease HyaD']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K03619 hydrogenase-1 operon protein HyaE\n",
      "SSO SSO:000041607 Uptake [NiFe] hydrogenase maturation factor HyaE\n",
      "b0976 ['Uptake [NiFe] hydrogenase maturation factor HyaE']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K03618 hydrogenase-1 operon protein HyaF\n",
      "SSO SSO:000041608 Uptake [NiFe] hydrogenase maturation maturation factor HyaF\n",
      "b0977 ['Uptake [NiFe] hydrogenase maturation maturation factor HyaF']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00425 cytochrome bd ubiquinol oxidase subunit I [EC:7.1.1.7]\n",
      "SSO SSO:000001939 Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)\n",
      "b0978 ['Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00426 cytochrome bd ubiquinol oxidase subunit II [EC:7.1.1.7]\n",
      "SSO SSO:000001940 Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)\n",
      "b0979 ['Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)']\n",
      "\n",
      "KO KO:K22501 cytochrome bd-II ubiquinol oxidase subunit AppX [EC:7.1.1.7]\n",
      "SSO SSO:000001941 Cytochrome d ubiquinol oxidase subunit X (EC 1.10.3.-)\n",
      "b4592 ['Cytochrome d ubiquinol oxidase subunit X (EC 1.10.3.-)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0042594 response to starvation\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01093 4-phytase / acid phosphatase [EC:3.1.3.26 3.1.3.2]\n",
      "b0980 ['Phosphoanhydride phosphohydrolase (EC 3.1.3.2) (pH 2.5 acid phosphatase) (AP)', '4- phytase (EC 3.1.3.26)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16554 polysaccharide biosynthesis transport protein\n",
      "KO KO:K16692 tyrosine-protein kinase Etk/Wzc [EC:2.7.10.-]\n",
      "SSO SSO:000015062 Tyrosine-protein kinase (EC 2.7.10.2)\n",
      "b0981 ['Tyrosine-protein kinase (EC 2.7.10.2)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K01104 protein-tyrosine phosphatase [EC:3.1.3.48]\n",
      "b0982 ['Low molecular weight protein-tyrosine-phosphatase (EC 3.1.3.48) => Etp']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K01991 polysaccharide biosynthesis/export protein\n",
      "b0983 ['Putative polysaccharide export protein YccZ precursor']\n",
      "\n",
      "b0984 ['Uncharacterized lipoprotein GfcD']\n",
      "\n",
      "b0985 ['Uncharacterized protein GfcC']\n",
      "\n",
      "b0986 ['Uncharacterized lipoprotein YmcC precursor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "b0987 ['Threonine-rich inner membrane protein GfcA']\n",
      "\n",
      "SSO SSO:000018187 IS1 protein InsA\n",
      "b4516 ['IS1 protein InsA']\n",
      "\n",
      "KO KO:K07480 insertion element IS1 protein InsB\n",
      "SSO SSO:000018188 IS1 protein InsB\n",
      "b0988 ['IS1 protein InsB']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "b0989 ['Cold shock protein of CSP family => CspF/CspH (naming convention as in E.coli)']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K03704 cold shock protein (beta-ribbon, CspA family)\n",
      "b0990 ['Cold shock protein of CSP family => CspB/CspG/CspI (naming convention as in E.coli)']\n",
      "\n",
      "b0991 ['Sfa protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "b4517 ['GnsA protein']\n",
      "\n",
      "KO KO:K02574 ferredoxin-type protein NapH\n",
      "SSO SSO:000018009 Hypothetical iron-sulfur cluster binding protein YccM\n",
      "b0992 ['Hypothetical iron-sulfur cluster binding protein YccM']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07647 two-component system, OmpR family, sensor histidine kinase TorS [EC:2.7.13.3]\n",
      "SSO SSO:000008382 Trimethylamine-N-oxide sensor histidine kinase TorS (EC 2.7.13.3)\n",
      "b0993 ['Trimethylamine-N-oxide sensor histidine kinase TorS (EC 2.7.13.3)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K11930 periplasmic protein TorT\n",
      "SSO SSO:000005846 Periplasmic protein TorT\n",
      "b0994 ['Periplasmic protein TorT']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07772 two-component system, OmpR family, torCAD operon response regulator TorR\n",
      "SSO SSO:000008377 Trimethylamine-N-oxide operon transcriptional regulatory protein TorR\n",
      "b0995 ['Trimethylamine-N-oxide operon transcriptional regulatory protein TorR']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03532 trimethylamine-N-oxide reductase (cytochrome c), cytochrome c-type subunit TorC\n",
      "SSO SSO:000008380 Trimethylamine-N-oxide reductase associated c-type cytochrome, TorC\n",
      "b0996 ['Trimethylamine-N-oxide reductase associated c-type cytochrome, TorC']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K07811 trimethylamine-N-oxide reductase (cytochrome c) [EC:1.7.2.3]\n",
      "SSO SSO:000008378 Trimethylamine-N-oxide reductase , TorA (EC 1.7.2.3)\n",
      "b0997 ['Trimethylamine-N-oxide reductase (EC 1.7.2.3), TorA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03533 TorA specific chaperone\n",
      "SSO SSO:000008108 TorA-specific chaperone protein, TorD\n",
      "b0998 ['TorA-specific chaperone protein, TorD']\n",
      "\n",
      "KO KO:K18997 chaperone modulatory protein CbpM\n",
      "SSO SSO:000001511 Chaperone-modulator protein CbpM\n",
      "b0999 ['Chaperone-modulator protein CbpM']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K05516 curved DNA-binding protein\n",
      "SSO SSO:000002374 DnaJ-class molecular chaperone CbpA\n",
      "b1000 ['DnaJ-class molecular chaperone CbpA']\n",
      "\n",
      "SSO SSO:000028190 Uncharacterized protein YccE\n",
      "b1001 ['Uncharacterized protein YccE']\n",
      "\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01085 glucose-1-phosphatase [EC:3.1.3.10]\n",
      "SSO SSO:000003123 Glucose-1-phosphatase (EC 3.1.3.10)\n",
      "b1002 ['Glucose-1-phosphatase (EC 3.1.3.10)']\n",
      "\n",
      "SSO SSO:000028191 Uncharacterized protein YccJ\n",
      "b1003 ['Uncharacterized protein YccJ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03809 NAD(P)H dehydrogenase (quinone) [EC:1.6.5.2]\n",
      "SSO SSO:000005248 NADPH:quinone oxidoreductase (EC 1.6.5.5)\n",
      "b1004 ['NAD(P)H dehydrogenase (quinone), Type IV (EC 1.6.5.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06884 uncharacterized protein\n",
      "SSO SSO:000028453 Uncharacterized protein YmdF\n",
      "b4518 ['Uncharacterized protein YmdF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K09016 putative pyrimidine permease RutG\n",
      "SSO SSO:000006826 Pyrimidine permease RutG\n",
      "SSO SSO:000023758 Pyrimidine permease\n",
      "b1006 ['Pyrimidine permease RutG']\n",
      "\n",
      "KO KO:K09024 flavin reductase [EC:1.5.1.-]\n",
      "SSO SSO:000002678 FMN reductase (NADH) RutF (EC 1.5.1.42)\n",
      "b1007 ['FMN reductase (NADH) RutF (EC 1.5.1.42)']\n",
      "\n",
      "KO KO:K09019 3-hydroxypropanoate dehydrogenase [EC:1.1.1.-]\n",
      "SSO SSO:000006334 Probable malonic semialdehyde reductase RutE (EC 1.1.1.298)\n",
      "b1008 ['Probable malonic semialdehyde reductase RutE (EC 1.1.1.298)']\n",
      "\n",
      "KO KO:K09023 aminoacrylate hydrolase [EC:3.5.1.-]\n",
      "SSO SSO:000006184 Possible hydrolase or acyltransferase RutD in novel pyrimidine catabolism pathway\n",
      "b1009 ['Possible hydrolase or acyltransferase RutD in novel pyrimidine catabolism pathway']\n",
      "\n",
      "KO KO:K09021 aminoacrylate peracid reductase\n",
      "KO KO:K09022 2-iminobutanoate/2-iminopropanoate deaminase [EC:3.5.99.10]\n",
      "b1010 ['Putative aminoacrylate peracid reductase RutC', 'RidA/YER057c/UK114 superfamily protein']\n",
      "\n",
      "KO KO:K09020 ureidoacrylate peracid hydrolase [EC:3.5.1.110]\n",
      "SSO SSO:000005858 Peroxyureidoacrylate/ureidoacrylate amidohydrolase RutB (EC 3.5.1.110)\n",
      "b1011 ['Peroxyureidoacrylate/ureidoacrylate amidohydrolase RutB (EC 3.5.1.110)']\n",
      "\n",
      "KO KO:K09018 pyrimidine oxygenase [EC:1.14.99.46]\n",
      "SSO SSO:000006824 Pyrimidine monooxygenase RutA (EC 1.14.99.46)\n",
      "b1012 ['Pyrimidine monooxygenase RutA (EC 1.14.99.46)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K09017 TetR/AcrR family transcriptional regulator\n",
      "SSO SSO:000008214 Transcriptional regulator RutR of pyrimidine catabolism (TetR family)\n",
      "b1013 ['Transcriptional regulator RutR of pyrimidine catabolism (TetR family)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K13821 RHH-type transcriptional regulator, proline utilization regulon repressor / proline dehydrogenase / delta 1-pyrroline-5-carboxylate dehydrogenase [EC:1.5.5.2 1.2.1.88]\n",
      "b1014 ['Transcriptional repressor of PutA and PutP', 'Proline dehydrogenase (EC 1.5.5.2)', 'Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)']\n",
      "\n",
      "GO GO:0005298 proline:sodium symporter activity\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11928 sodium/proline symporter\n",
      "b1015 ['Proline/sodium symporter PutP (TC 2.A.21.2.1)', 'Propionate/sodium symporter']\n",
      "\n",
      "KO KO:K07243 high-affinity iron transporter\n",
      "SSO SSO:000002762 Ferrous iron transport permease EfeU\n",
      "b4490 ['Ferrous iron transport permease EfeU']\n",
      "\n",
      "KO KO:K07224 iron uptake system component EfeO\n",
      "SSO SSO:000002761 Ferrous iron transport periplasmic protein EfeO, contains peptidase-M75 domain and (frequently) cupredoxin-like domain\n",
      "b1018 ['Ferrous iron transport periplasmic protein EfeO, contains peptidase-M75 domain and (frequently) cupredoxin-like domain']\n",
      "\n",
      "KO KO:K16301 deferrochelatase/peroxidase EfeB [EC:1.11.1.-]\n",
      "SSO SSO:000002764 Ferrous iron transport peroxidase EfeB\n",
      "b1019 ['Ferrous iron transport peroxidase EfeB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K06217 phosphate starvation-inducible protein PhoH and related proteins\n",
      "SSO SSO:000005923 Phosphate starvation-inducible protein PhoH, predicted ATPase\n",
      "b1020 ['Phosphate starvation-inducible protein PhoH, predicted ATPase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K11937 biofilm PGA synthesis protein PgaD\n",
      "SSO SSO:000001215 Biofilm PGA synthesis auxiliary protein PgaD\n",
      "b1021 ['Biofilm PGA synthesis auxiliary protein PgaD']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.80 Ceramide glucosyltransferase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11936 poly-beta-1,6-N-acetyl-D-glucosamine synthase [EC:2.4.1.-]\n",
      "SSO SSO:000001214 Biofilm PGA synthesis N-glycosyltransferase PgaC (EC 2.4.-.-)\n",
      "b1022 ['Biofilm PGA synthesis N-glycosyltransferase PgaC (EC 2.4.-.-)']\n",
      "\n",
      "EC EC:3.1.1.72 Acetylxylan esterase.\n",
      "EC EC:3.5.1.41 Chitin deacetylase.\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11931 poly-beta-1,6-N-acetyl-D-glucosamine N-deacetylase [EC:3.5.1.-]\n",
      "SSO SSO:000001217 Biofilm PGA synthesis deacetylase PgaB (EC 3.-)\n",
      "b1023 ['Biofilm PGA synthesis deacetylase PgaB (EC 3.-)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K11935 biofilm PGA synthesis protein PgaA\n",
      "SSO SSO:000001213 Biofilm PGA outer membrane secretin PgaA\n",
      "b1024 ['Biofilm PGA outer membrane secretin PgaA']\n",
      "\n",
      "b1025 ['Diguanylate cyclase (EC 2.7.7.65) => DgcT']\n",
      "\n",
      "KO KO:K07497 putative transposase\n",
      "b1026 ['Transposase InsF for insertion sequence IS3', 'Mobile element protein']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026096 Transposase InsE for insertion sequence IS3E\n",
      "b1027 ['Transposase InsE for insertion sequence IS3E']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1028 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028192 Uncharacterized protein YcdU\n",
      "b1029 ['Uncharacterized protein YcdU']\n",
      "\n",
      "KO KO:K12972 glyoxylate/hydroxypyruvate reductase [EC:1.1.1.79 1.1.1.81]\n",
      "b1033 ['Glyoxylate/hydroxypyruvate reductase A (EC 1.1.1.79) (EC 1.1.1.81)']\n",
      "\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K04477 putative hydrolase\n",
      "SSO SSO:000000886 Alkaline phosphatase (EC 3.1.3.1)\n",
      "b1034 ['Putative hydrolase YcdX (EC 3.1.-.-)']\n",
      "\n",
      "KO KO:K23349 Unknown\n",
      "SSO SSO:000034633 Anaerobic respiratory reductase chaperone\n",
      "b1035 ['Anaerobic respiratory reductase chaperone']\n",
      "\n",
      "SSO SSO:000018388 Inner membrane protein YcdZ\n",
      "b1036 ['Inner membrane protein YcdZ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K06214 curli production assembly/transport component CsgG\n",
      "SSO SSO:000012517 Curli production assembly/transport component CsgG\n",
      "b1037 ['Curli production assembly/transport component CsgG']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K04338 curli production assembly/transport component CsgF\n",
      "SSO SSO:000012516 Curli production assembly/transport component CsgF\n",
      "b1038 ['Curli production assembly/transport component CsgF']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K04337 curli production assembly/transport component CsgE\n",
      "SSO SSO:000012515 Curli production assembly/transport component CsgE\n",
      "b1039 ['Curli production assembly/transport component CsgE']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K04333 LuxR family transcriptional regulator, csgAB operon transcriptional regulatory protein\n",
      "SSO SSO:000025800 Transcriptional regulator CsgD for 2nd curli operon\n",
      "b1040 ['Transcriptional regulator CsgD for 2nd curli operon']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K04335 minor curlin subunit\n",
      "SSO SSO:000019682 Minor curlin subunit CsgB, nucleation component of curlin monomers\n",
      "b1041 ['Minor curlin subunit CsgB, nucleation component of curlin monomers']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K04334 major curlin subunit\n",
      "SSO SSO:000019282 Major curlin subunit precursor CsgA\n",
      "b1042 ['Major curlin subunit precursor CsgA']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K04336 curli production protein\n",
      "SSO SSO:000023233 Putative curli production protein CsgC\n",
      "b1043 ['Putative curli production protein CsgC']\n",
      "\n",
      "SSO SSO:000028452 Uncharacterized protein YmdA\n",
      "b1044 ['Uncharacterized protein YmdA']\n",
      "\n",
      "KO KO:K23518 Unknown\n",
      "SSO SSO:000020702 O-acetyl-ADP-ribose deacetylase\n",
      "b1045 ['O-acetyl-ADP-ribose deacetylase']\n",
      "\n",
      "KO KO:K06132 cardiolipin synthase C [EC:2.7.8.-]\n",
      "SSO SSO:000035000 Cardiolipin synthase phosphatidylethanolamine-utilizing, bacterial type ClsC\n",
      "b1046 ['Cardiolipin synthase (EC 2.7.8.-) phosphatidylethanolamine-utilizing, bacterial type ClsC']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009250 glucan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11941 glucans biosynthesis protein C [EC:2.1.-.-]\n",
      "SSO SSO:000017201 Glucans biosynthesis protein C (EC 2.1.-.-)\n",
      "b1047 ['Glucans biosynthesis protein C (EC 2.1.-.-)']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009250 glucan biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03670 periplasmic glucans biosynthesis protein\n",
      "b1048 ['Glucans biosynthesis protein G precursor']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03669 membrane glycosyltransferase [EC:2.4.1.-]\n",
      "SSO SSO:000017200 Glucans biosynthesis glucosyltransferase H (EC 2.4.1.-)\n",
      "b1049 ['Glucans biosynthesis glucosyltransferase H']\n",
      "\n",
      "b1050 ['Putative outer membrane lipoprotein']\n",
      "\n",
      "KO KO:K12147 acidic protein MsyB\n",
      "SSO SSO:000010207 Acidic protein msyB\n",
      "b1051 ['Acidic protein msyB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08161 MFS transporter, DHA1 family, multidrug resistance protein\n",
      "SSO SSO:000020101 Multidrug resistance protein MdtG\n",
      "b1053 ['Multidrug resistance protein MdtG']\n",
      "\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02517 Kdo2-lipid IVA lauroyltransferase [EC:2.3.1.241]\n",
      "SSO SSO:000004453 Lipid A biosynthesis lauroyl acyltransferase (EC 2.3.1.-)\n",
      "b1054 ['Lipid A biosynthesis lauroyl acyltransferase (EC 2.3.1.241)']\n",
      "\n",
      "GO GO:0007049 cell cycle\n",
      "KO KO:K07146 UPF0176 protein\n",
      "SSO SSO:000007053 Rhodanese domain protein, Enterobacterial subgroup, YceA homolog\n",
      "SSO SSO:000030672 putative Rhodanese-related sulfurtransferase\n",
      "b1055 ['Rhodanese domain protein, Enterobacterial subgroup, YceA homolog']\n",
      "\n",
      "b1056 ['Protein yceI precursor']\n",
      "\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K12262 cytochrome b561\n",
      "SSO SSO:000012615 Cytochrome b561 homolog 2\n",
      "SSO SSO:000032463 putative cytochrome B561\n",
      "b1057 ['Cytochrome b561 homolog 2']\n",
      "\n",
      "SSO SSO:000028195 Uncharacterized protein YceO\n",
      "b1058 ['Uncharacterized protein YceO']\n",
      "\n",
      "KO KO:K02846 N-methyl-L-tryptophan oxidase [EC:1.5.3.-]\n",
      "SSO SSO:000037000 N-methyl-L-tryptophan oxidase, SolA\n",
      "b1059 ['N-methyl-L-tryptophan oxidase, SolA']\n",
      "\n",
      "KO KO:K12148 biofilm regulator BssS\n",
      "SSO SSO:000011145 Biofilm regulator BssS\n",
      "b1060 ['Biofilm regulator BssS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K12149 DNA-damage-inducible protein I\n",
      "SSO SSO:000002149 DNA-damage-inducible protein I\n",
      "b1061 ['DNA-damage-inducible protein I']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "SSO SSO:000002305 Dihydroorotase (EC 3.5.2.3)\n",
      "b1062 ['Dihydroorotase (EC 3.5.2.3)']\n",
      "\n",
      "b1063 ['Putative lipoprotein yceB precursor']\n",
      "\n",
      "KO KO:K03675 glutaredoxin 2\n",
      "SSO SSO:000003200 Glutaredoxin 2\n",
      "b1064 ['Glutaredoxin 2']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08162 MFS transporter, DHA1 family, multidrug resistance protein\n",
      "SSO SSO:000005023 Multidrug resistance protein MdtH\n",
      "b1065 ['Multidrug resistance protein MdtH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K03790 [ribosomal protein S5]-alanine N-acetyltransferase [EC:2.3.1.267]\n",
      "SSO SSO:000007140 Ribosomal-protein-S5p-alanine acetyltransferase (EC 2.3.1.128)\n",
      "SSO SSO:000041792 [ribosomal protein S5]-alanine N-acetyltransferase (EC 2.3.1.267)\n",
      "b1066 ['Ribosomal-protein-S5p-alanine acetyltransferase (EC 2.3.1.128)']\n",
      "\n",
      "KO KO:K09915 uncharacterized protein\n",
      "SSO SSO:000006513 Protein of unknown function YceH\n",
      "b1067 ['Protein of unknown function YceH']\n",
      "\n",
      "KO KO:K03810 virulence factor\n",
      "SSO SSO:000008935 Virulence factor MviM\n",
      "b1068 ['Virulence factor MviM']\n",
      "\n",
      "KO KO:K03980 putative peptidoglycan lipid II flippase\n",
      "SSO SSO:000035858 Peptidoglycan lipid II flippase MurJ\n",
      "b1069 ['Peptidoglycan lipid II flippase MurJ']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02399 flagella synthesis protein FlgN\n",
      "SSO SSO:000002799 Flagellar biosynthesis protein FlgN\n",
      "b1070 ['Flagellar biosynthesis protein FlgN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02398 negative regulator of flagellin synthesis FlgM\n",
      "SSO SSO:000005288 Negative regulator of flagellin synthesis FlgM (anti-sigma28)\n",
      "b1071 ['Negative regulator of flagellin synthesis FlgM (anti-sigma28)']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02386 flagella basal body P-ring formation protein FlgA\n",
      "SSO SSO:000002793 Flagellar basal-body P-ring formation protein FlgA\n",
      "b1072 ['Flagellar basal-body P-ring formation protein FlgA']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02387 flagellar basal-body rod protein FlgB\n",
      "SSO SSO:000002795 Flagellar basal-body rod protein FlgB\n",
      "b1073 ['Flagellar basal-body rod protein FlgB']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02388 flagellar basal-body rod protein FlgC\n",
      "SSO SSO:000002796 Flagellar basal-body rod protein FlgC\n",
      "b1074 ['Flagellar basal-body rod protein FlgC']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02389 flagellar basal-body rod modification protein FlgD\n",
      "SSO SSO:000002794 Flagellar basal-body rod modification protein FlgD\n",
      "b1075 ['Flagellar basal-body rod modification protein FlgD']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02390 flagellar hook protein FlgE\n",
      "SSO SSO:000002811 Flagellar hook protein FlgE\n",
      "b1076 ['Flagellar hook protein FlgE']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02391 flagellar basal-body rod protein FlgF\n",
      "SSO SSO:000002797 Flagellar basal-body rod protein FlgF\n",
      "b1077 ['Flagellar basal-body rod protein FlgF']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02392 flagellar basal-body rod protein FlgG\n",
      "SSO SSO:000002798 Flagellar basal-body rod protein FlgG\n",
      "b1078 ['Flagellar basal-body rod protein FlgG']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02393 flagellar L-ring protein precursor FlgH\n",
      "SSO SSO:000002787 Flagellar L-ring protein FlgH\n",
      "b1079 ['Flagellar L-ring protein FlgH']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02394 flagellar P-ring protein precursor FlgI\n",
      "SSO SSO:000002789 Flagellar P-ring protein FlgI\n",
      "b1080 ['Flagellar P-ring protein FlgI']\n",
      "\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "EC EC:3.2.1.96 Mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase.\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02395 flagellar protein FlgJ\n",
      "SSO SSO:000002824 Flagellar protein FlgJ [peptidoglycan hydrolase] (EC 3.2.1.-)\n",
      "b1081 ['Flagellar protein FlgJ [peptidoglycan hydrolase]']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02396 flagellar hook-associated protein 1 FlgK\n",
      "SSO SSO:000002812 Flagellar hook-associated protein FlgK\n",
      "b1082 ['Flagellar hook-associated protein FlgK']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02397 flagellar hook-associated protein 3 FlgL\n",
      "SSO SSO:000002813 Flagellar hook-associated protein FlgL\n",
      "b1083 ['Flagellar hook-associated protein FlgL']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K08300 ribonuclease E [EC:3.1.26.12]\n",
      "SSO SSO:000007072 Ribonuclease E (EC 3.1.26.12)\n",
      "b1084 ['Ribonuclease E (EC 3.1.26.12)']\n",
      "\n",
      "SSO SSO:000028196 Uncharacterized protein YceQ\n",
      "b1085 ['Uncharacterized protein YceQ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K06179 23S rRNA pseudouridine955/2504/2580 synthase [EC:5.4.99.24]\n",
      "SSO SSO:000036582 LSU rRNA pseudouridine(955/2504/2580) synthase (EC 5.4.99.24)\n",
      "b1086 ['LSU rRNA pseudouridine(955/2504/2580) synthase (EC 5.4.99.24)']\n",
      "\n",
      "KO KO:K06287 septum formation protein\n",
      "SSO SSO:000019272 Maf-like protein YceF\n",
      "b1087 ['Maf-like protein YceF']\n",
      "\n",
      "KO KO:K07040 uncharacterized protein\n",
      "b1088 ['FIG01269488: protein, clustered with ribosomal protein L32p']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02911 large subunit ribosomal protein L32\n",
      "b1089 ['LSU ribosomal protein L32p', 'LSU ribosomal protein L32p, zinc-independent']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "KO KO:K03621 phosphate acyltransferase [EC:2.3.1.274]\n",
      "SSO SSO:000005931 Phosphate:acyl-ACP acyltransferase PlsX (EC 2.3.1.n2)\n",
      "b1090 ['Phosphate:acyl-ACP acyltransferase PlsX (EC 2.3.1.n2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00648 3-oxoacyl-[acyl-carrier-protein] synthase III [EC:2.3.1.180]\n",
      "SSO SSO:000000335 3-oxoacyl-[acyl-carrier-protein] synthase, KASIII (EC 2.3.1.41)\n",
      "b1091 ['3-oxoacyl-[acyl-carrier-protein] synthase, KASIII (EC 2.3.1.180)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00645 [acyl-carrier-protein] S-malonyltransferase [EC:2.3.1.39]\n",
      "SSO SSO:000004693 Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)\n",
      "b1092 ['Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00059 3-oxoacyl-[acyl-carrier protein] reductase [EC:1.1.1.100]\n",
      "SSO SSO:000000330 3-oxoacyl-[acyl-carrier protein] reductase (EC 1.1.1.100)\n",
      "b1093 ['3-oxoacyl-[acyl-carrier protein] reductase (EC 1.1.1.100), FadG']\n",
      "\n",
      "GO GO:0042967 acyl-carrier-protein biosynthetic process\n",
      "KO KO:K02078 acyl carrier protein\n",
      "SSO SSO:000000749 Acyl carrier protein\n",
      "b1094 ['Acyl carrier protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K09458 3-oxoacyl-[acyl-carrier-protein] synthase II [EC:2.3.1.179]\n",
      "SSO SSO:000000334 3-oxoacyl-[acyl-carrier-protein] synthase, KASII (EC 2.3.1.179)\n",
      "b1095 ['3-oxoacyl-[acyl-carrier-protein] synthase, KASII (EC 2.3.1.179)']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K02619 4-amino-4-deoxychorismate lyase [EC:4.1.3.38]\n",
      "SSO SSO:000000956 Aminodeoxychorismate lyase (EC 4.1.3.38)\n",
      "b1096 ['Aminodeoxychorismate lyase (EC 4.1.3.38)']\n",
      "\n",
      "KO KO:K07082 UPF0755 protein\n",
      "SSO SSO:000036944 Murein endolytic transglycosylase MltG\n",
      "b1097 ['Murein endolytic transglycosylase MltG']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00943 dTMP kinase [EC:2.7.4.9]\n",
      "SSO SSO:000008067 Thymidylate kinase (EC 2.7.4.9)\n",
      "b1098 ['Thymidylate kinase (EC 2.7.4.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02341 DNA polymerase III subunit delta' [EC:2.7.7.7]\n",
      "SSO SSO:000002082 DNA polymerase III delta prime subunit (EC 2.7.7.7)\n",
      "b1099 ['DNA polymerase III delta prime subunit (EC 2.7.7.7)']\n",
      "\n",
      "KO KO:K03424 TatD DNase family protein [EC:3.1.21.-]\n",
      "SSO SSO:000040261 Uncharacterized metal-dependent hydrolase YcfH\n",
      "b1100 ['Uncharacterized metal-dependent hydrolase YcfH']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02779 PTS system, glucose-specific IIC component\n",
      "b1101 ['PTS system, glucose-specific IIC component', 'PTS system, glucose-specific IIB component (EC 2.7.1.199)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "KO KO:K16088 outer-membrane receptor for ferric coprogen and ferric-rhodotorulic acid\n",
      "SSO SSO:000016724 FhuE receptor precursor\n",
      "b1102 ['FhuE receptor precursor']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02503 histidine triad (HIT) family protein\n",
      "SSO SSO:000029116 YcfF/hinT protein: a purine nucleoside phosphoramidase\n",
      "b1103 ['YcfF/hinT protein: a purine nucleoside phosphoramidase']\n",
      "\n",
      "SSO SSO:000029117 YcfL protein: an outer membrane lipoprotein that is part of a salvage cluster\n",
      "b1104 ['YcfL protein: an outer membrane lipoprotein that is part of a salvage cluster']\n",
      "\n",
      "KO KO:K07337 penicillin-binding protein activator\n",
      "SSO SSO:000019002 Lipoprotein YcfM, part of a salvage pathway of unknown substrate\n",
      "b1105 ['Lipoprotein YcfM, part of a salvage pathway of unknown substrate']\n",
      "\n",
      "KO KO:K07251 thiamine kinase [EC:2.7.1.89]\n",
      "SSO SSO:000008009 Thiamine kinase (EC 2.7.1.89)\n",
      "b1106 ['Thiamine kinase (EC 2.7.1.89)', 'Adenosylcobinamide kinase (EC 2.7.1.156)']\n",
      "\n",
      "EC EC:3.2.1.120 Oligoxyloglucan beta-glycosidase.\n",
      "EC EC:3.2.1.126 Coniferin beta-glucosidase.\n",
      "EC EC:3.2.1.21 Beta-glucosidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.45 Glucosylceramidase.\n",
      "EC EC:3.2.1.52 Beta-N-acetylhexosaminidase.\n",
      "EC EC:3.2.1.55 Non-reducing end alpha-L-arabinofuranosidase.\n",
      "EC EC:3.2.1.58 Glucan 1,3-beta-glucosidase.\n",
      "EC EC:3.2.1.74 Glucan 1,4-beta-glucosidase.\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "KO KO:K01207 beta-N-acetylhexosaminidase [EC:3.2.1.52]\n",
      "SSO SSO:000009052 beta-N-acetylglucosaminidase (EC 3.2.1.52)\n",
      "b1107 ['beta-N-acetylglucosaminidase (EC 3.2.1.52)']\n",
      "\n",
      "KO KO:K07000 uncharacterized protein\n",
      "SSO SSO:000029118 YcfP protein: probably an esterase that is part of a salvage cluster\n",
      "b1108 ['YcfP protein: probably an esterase that is part of a salvage cluster']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K03885 NADH dehydrogenase [EC:1.6.99.3]\n",
      "SSO SSO:000005203 NADH dehydrogenase (EC 1.6.99.3)\n",
      "b1109 ['NADH dehydrogenase (EC 1.6.99.3)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K06077 outer membrane lipoprotein SlyB\n",
      "b1110 ['FIG01200701: possible membrane protein']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K16137 TetR/AcrR family transcriptional regulator, transcriptional repressor for nem operon\n",
      "KO KO:K22041 TetR/AcrR family transcriptional regulator, copper-responsive repressor\n",
      "b1111 ['HTH-type transcriptional repressor ComR']\n",
      "\n",
      "KO KO:K12151 multiple stress resistance protein BhsA\n",
      "SSO SSO:000020113 Multiple stress resistance protein BhsA\n",
      "b1112 ['Multiple stress resistance protein BhsA']\n",
      "\n",
      "KO KO:K19236 L,D-transpeptidase YcfS\n",
      "b1113 ['L,D-transpeptidase => YcfS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03723 transcription-repair coupling factor (superfamily II helicase) [EC:3.6.4.-]\n",
      "SSO SSO:000008179 Transcription-repair coupling factor\n",
      "b1114 ['Transcription-repair coupling factor']\n",
      "\n",
      "SSO SSO:000018389 Inner membrane protein YcfT\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b1115 ['Inner membrane protein YcfT']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02004 putative ABC transport system permease protein\n",
      "KO KO:K09808 lipoprotein-releasing system permease protein\n",
      "SSO SSO:000019006 Lipoprotein releasing system transmembrane protein LolC\n",
      "b1116 ['Lipoprotein releasing system transmembrane protein LolC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02003 putative ABC transport system ATP-binding protein\n",
      "KO KO:K09810 lipoprotein-releasing system ATP-binding protein [EC:3.6.3.-]\n",
      "SSO SSO:000019012 Lipoprotein-releasing system ATP-binding protein LolD\n",
      "b1117 ['Lipoprotein-releasing system ATP-binding protein LolD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02004 putative ABC transport system permease protein\n",
      "KO KO:K09808 lipoprotein-releasing system permease protein\n",
      "SSO SSO:000019007 Lipoprotein releasing system transmembrane protein LolE\n",
      "b1118 ['Lipoprotein releasing system transmembrane protein LolE']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K00845 glucokinase [EC:2.7.1.2]\n",
      "KO KO:K00884 N-acetylglucosamine kinase [EC:2.7.1.59]\n",
      "SSO SSO:000005090 N-acetyl-D-glucosamine kinase (EC 2.7.1.59)\n",
      "b1119 ['N-acetyl-D-glucosamine kinase (EC 2.7.1.59)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K12410 NAD-dependent deacetylase [EC:3.5.1.-]\n",
      "SSO SSO:000005189 NAD-dependent protein deacetylase of SIR2 family\n",
      "b1120 ['NAD-dependent protein deacetylase of SIR2 family']\n",
      "\n",
      "SSO SSO:000032519 putative factor\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b1121 ['putative factor']\n",
      "\n",
      "SSO SSO:000018420 Inner membrane protein YmfA\n",
      "b1122 ['Inner membrane protein YmfA']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02055 putative spermidine/putrescine transport system substrate-binding protein\n",
      "KO KO:K11069 spermidine/putrescine transport system substrate-binding protein\n",
      "SSO SSO:000007630 Spermidine/putrescine import ABC transporter substrate-binding protein PotD (TC 3.A.1.11.1)\n",
      "b1123 ['Spermidine/putrescine import ABC transporter substrate-binding protein PotD (TC 3.A.1.11.1)']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11070 spermidine/putrescine transport system permease protein\n",
      "SSO SSO:000007629 Spermidine/putrescine import ABC transporter permease protein PotC (TC 3.A.1.11.1)\n",
      "b1124 ['Spermidine/putrescine import ABC transporter permease protein PotC (TC 3.A.1.11.1)']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11071 spermidine/putrescine transport system permease protein\n",
      "SSO SSO:000024985 Spermidine/putrescine import ABC transporter permease protein PotB (TC 3.A.1.11.1)\n",
      "b1125 ['Spermidine/putrescine import ABC transporter permease protein PotB (TC 3.A.1.11.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K02010 iron(III) transport system ATP-binding protein [EC:3.6.3.30]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "KO KO:K11072 spermidine/putrescine transport system ATP-binding protein [EC:3.6.3.31]\n",
      "SSO SSO:000007628 Spermidine/putrescine import ABC transporter ATP-binding protein PotA (TC 3.A.1.11.1)\n",
      "b1126 ['Spermidine/putrescine import ABC transporter ATP-binding protein PotA (TC 3.A.1.11.1)']\n",
      "\n",
      "KO KO:K01258 tripeptide aminopeptidase [EC:3.4.11.4]\n",
      "SSO SSO:000008384 Tripeptide aminopeptidase (EC 3.4.11.4)\n",
      "SSO SSO:000021359 Peptidase T (EC 3.4.11.-)\n",
      "b1127 ['Tripeptide aminopeptidase (EC 3.4.11.4)']\n",
      "\n",
      "KO KO:K18850 50S ribosomal protein L16 3-hydroxylase [EC:1.14.11.47]\n",
      "SSO SSO:000004222 LSU ribosomal protein L16p arginine hydroxylase\n",
      "b1128 ['LSU ribosomal protein L16p arginine hydroxylase']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07637 two-component system, OmpR family, sensor histidine kinase PhoQ [EC:2.7.13.3]\n",
      "SSO SSO:000007392 Sensor histidine kinase PhoQ (EC 2.7.13.3)\n",
      "b1129 ['Sensor histidine kinase PhoQ (EC 2.7.13.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07660 two-component system, OmpR family, response regulator PhoP\n",
      "SSO SSO:000008259 Transcriptional regulatory protein PhoP\n",
      "SSO SSO:000033090 response regulator in two-component regulatory system with PhoQ\n",
      "b1130 ['Transcriptional regulatory protein PhoP']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K01756 adenylosuccinate lyase [EC:4.3.2.2]\n",
      "SSO SSO:000000823 Adenylosuccinate lyase (EC 4.3.2.2)\n",
      "b1131 ['Adenylosuccinate lyase (EC 4.3.2.2)', 'SAICAR lyase (EC 4.3.2.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07153 high frequency lysogenization protein\n",
      "SSO SSO:000003433 High frequency lysogenization protein HflD\n",
      "b1132 ['High frequency lysogenization protein HflD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00566 tRNA-uridine 2-sulfurtransferase [EC:2.8.1.13]\n",
      "SSO SSO:000009403 tRNA-specific 2-thiouridylase MnmA\n",
      "b1133 ['tRNA-specific 2-thiouridylase MnmA (EC 2.8.1.13)']\n",
      "\n",
      "KO KO:K03574 8-oxo-dGTP diphosphatase [EC:3.6.1.55]\n",
      "KO KO:K12152 phosphatase NudJ [EC:3.6.1.-]\n",
      "SSO SSO:000005456 Nudix-like NDP and NTP phosphohydrolase NudJ\n",
      "b1134 ['Nudix-like NDP and NTP phosphohydrolase NudJ']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K06181 23S rRNA pseudouridine2457 synthase [EC:5.4.99.20]\n",
      "SSO SSO:000036576 LSU rRNA pseudouridine(2457) synthase (EC 5.4.99.20)\n",
      "SSO SSO:000042969 23S rRNA pseudouridine2457 synthase (EC 5.4.99.20)\n",
      "b1135 ['LSU rRNA pseudouridine(2457) synthase (EC 5.4.99.20)']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00031 isocitrate dehydrogenase [EC:1.1.1.42]\n",
      "SSO SSO:000004050 Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)\n",
      "b1136 ['Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)']\n",
      "\n",
      "SSO SSO:000028454 Uncharacterized protein YmfD\n",
      "b1137 ['Uncharacterized protein YmfD']\n",
      "\n",
      "SSO SSO:000028455 Uncharacterized protein YmfE\n",
      "b1138 ['Uncharacterized protein YmfE']\n",
      "\n",
      "b1139 ['Bacteriophage T4 late gene expression blocking protein']\n",
      "\n",
      "KO KO:K21039 enterobacteria phage integrase [EC:2.7.7.- 3.1.-.-]\n",
      "SSO SSO:000021513 Phage integrase\n",
      "b1140 ['Phage integrase']\n",
      "\n",
      "b1141 ['Excisionase-like protein from lambdoid prophage 14']\n",
      "\n",
      "b1143 ['Uncharacterized protein YmfI']\n",
      "\n",
      "SSO SSO:000028456 Uncharacterized protein YmfJ\n",
      "b1144 ['Uncharacterized protein YmfJ']\n",
      "\n",
      "SSO SSO:000021540 Phage repressor protein cI\n",
      "b1145 ['Phage repressor protein cI']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1146 ['hypothetical protein']\n",
      "\n",
      "b1147 ['Uncharacterized protein YmfL']\n",
      "\n",
      "b1148 ['Uncharacterized protein YmfM']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4692 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4693 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000038034 Phage head, head-tail joining protein W\n",
      "b1150 ['Phage head, head-tail joining protein W']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1151 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1152 ['hypothetical protein']\n",
      "\n",
      "b1153 ['Uncharacterized protein YmfQ in lambdoid prophage e14 region']\n",
      "\n",
      "SSO SSO:000038365 Phage tail fiber, side tail fiber protein Stf\n",
      "b1154 ['Phage tail fiber, side tail fiber protein Stf']\n",
      "\n",
      "b1155 ['Tail fiber assembly protein']\n",
      "\n",
      "b1156 ['Phage tail fiber assembly protein', 'Phage tail fiber assembly protein YcfA']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1157 ['hypothetical protein']\n",
      "\n",
      "KO KO:K14060 putative DNA-invertase from lambdoid prophage Rac\n",
      "KO KO:K21528 serine recombinase [EC:3.1.22.- 6.5.1.-]\n",
      "SSO SSO:000021485 Phage DNA invertase\n",
      "b1158 ['Phage DNA invertase']\n",
      "\n",
      "KO KO:K07451 5-methylcytosine-specific restriction enzyme A [EC:3.1.21.-]\n",
      "SSO SSO:000009732 5-methylcytosine-specific restriction enzyme A\n",
      "b1159 ['5-methylcytosine-specific restriction enzyme A (EC 3.1.21.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000004050 Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)\n",
      "b4519 ['Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)']\n",
      "\n",
      "KO KO:K21638 anti-adapter protein IraM\n",
      "SSO SSO:000010679 Anti-adapter protein IraM\n",
      "b1160 ['Anti-adapter protein IraM']\n",
      "\n",
      "SSO SSO:000038245 Phage protein YbcV\n",
      "b1161 ['Phage protein YbcV']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K21972 MerR family transcriptional regulator, repressor of blue light- and temperature-responsive genes\n",
      "SSO SSO:000017610 HTH-type transcriptional repressor YcgE\n",
      "b1162 ['HTH-type transcriptional repressor YcgE']\n",
      "\n",
      "KO KO:K21973 blue light- and temperature-responsive anti-repressor\n",
      "SSO SSO:000011396 Blue light- and temperature-regulated antirepressor BluF\n",
      "b1163 ['Blue light- and temperature-regulated antirepressor BluF']\n",
      "\n",
      "KO KO:K21974 probable RcsB/C two-component-system connector\n",
      "SSO SSO:000022612 Probable two-component-system connector protein YcgZ\n",
      "b1164 ['Probable two-component-system connector protein YcgZ']\n",
      "\n",
      "KO KO:K21975 probable RcsB/C two-component-system connector\n",
      "b1165 ['FIG00644045: hypothetical protein']\n",
      "\n",
      "KO KO:K21976 probable RcsB/C two-component-system connector, global regulator of biofilm formation and acid-resistance\n",
      "SSO SSO:000022611 Probable two-component-system connector protein AriR\n",
      "b1166 ['Probable two-component-system connector protein AriR']\n",
      "\n",
      "SSO SSO:000028457 Uncharacterized protein YmgC\n",
      "b1167 ['Uncharacterized protein YmgC']\n",
      "\n",
      "KO KO:K21090 c-di-GMP phosphodiesterase [EC:3.1.4.52]\n",
      "b1168 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeG']\n",
      "\n",
      "GO GO:0016021 integral component of membrane\n",
      "SSO SSO:000018421 Inner membrane protein YmgF\n",
      "b4520 ['Inner membrane protein YmgF']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4491 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028458 Uncharacterized protein YmgD\n",
      "b1171 ['Uncharacterized protein YmgD']\n",
      "\n",
      "SSO SSO:000026711 UPF0757 protein YmgG\n",
      "b1172 ['UPF0757 protein YmgG']\n",
      "\n",
      "b4593 ['FIG00639461: hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4594 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4521 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03608 cell division topological specificity factor\n",
      "SSO SSO:000001479 Cell division topological specificity factor MinE\n",
      "b1174 ['Cell division topological specificity factor MinE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03609 septum site-determining protein MinD\n",
      "SSO SSO:000007417 Septum site-determining protein MinD\n",
      "b1175 ['Septum site-determining protein MinD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03610 septum site-determining protein MinC\n",
      "SSO SSO:000009306 septum site-determining protein MinC\n",
      "b1176 ['Septum site-determining protein MinC']\n",
      "\n",
      "SSO SSO:000028197 Uncharacterized protein YcgJ\n",
      "b1177 ['Uncharacterized protein YcgJ']\n",
      "\n",
      "SSO SSO:000003875 Inhibitor of vertebrate g-type lysozyme, periplasmic => PliG\n",
      "b1178 ['Inhibitor of vertebrate g-type lysozyme, periplasmic => PliG']\n",
      "\n",
      "KO KO:K09902 uncharacterized protein\n",
      "SSO SSO:000022726 Protein YcgL\n",
      "b1179 ['Protein YcgL']\n",
      "\n",
      "KO KO:K01557 acylpyruvate hydrolase [EC:3.7.1.5]\n",
      "SSO SSO:000040119 Uncharacterized fumarylacetoacetase-like protein YcgM\n",
      "b1180 ['Uncharacterized fumarylacetoacetase-like protein YcgM']\n",
      "\n",
      "KO KO:K09160 uncharacterized protein\n",
      "SSO SSO:000026605 UPF0260 protein YcgN\n",
      "b1181 ['UPF0260 protein YcgN']\n",
      "\n",
      "GO GO:0005576 extracellular region\n",
      "KO KO:K11139 hemolysin E\n",
      "SSO SSO:000017732 Hemolysin E, chromosomal\n",
      "b1182 ['Hemolysin E, chromosomal']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03503 DNA polymerase V [EC:3.4.21.-]\n",
      "SSO SSO:000002556 Error-prone repair protein UmuD (EC 3.4.21.-)\n",
      "b1183 ['Error-prone repair protein UmuD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03502 DNA polymerase V\n",
      "SSO SSO:000002557 Error-prone, lesion bypass DNA polymerase V (UmuC)\n",
      "b1184 ['Error-prone, lesion bypass DNA polymerase V (UmuC)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009289 pilus\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03611 disulfide bond formation protein DsbB\n",
      "SSO SSO:000005850 Periplasmic thiol:disulfide oxidoreductase DsbB, required for DsbA reoxidation\n",
      "b1185 ['Periplasmic thiol:disulfide oxidoreductase DsbB, required for DsbA reoxidation']\n",
      "\n",
      "GO GO:0009268 response to pH\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03314 Na+:H+ antiporter, NhaB family\n",
      "SSO SSO:000005280 Na+/H+ antiporter NhaB\n",
      "b1186 ['Na+/H+ antiporter NhaB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K03603 GntR family transcriptional regulator, negative regulator for fad regulon and positive regulator of fabA\n",
      "b1187 ['Transcriptional regulator for fatty acid degradation FadR, GntR family']\n",
      "\n",
      "KO KO:K06415 stage V sporulation protein R\n",
      "SSO SSO:000013944 FIG004684: SpoVR-like protein\n",
      "b1188 ['FIG004684: SpoVR-like protein']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00285 D-amino-acid dehydrogenase [EC:1.4.5.1]\n",
      "SSO SSO:000002003 D-amino acid dehydrogenase (EC 1.4.99.1)\n",
      "b1189 ['D-amino acid dehydrogenase (EC 1.4.99.6)']\n",
      "\n",
      "GO GO:0006523 alanine biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01775 alanine racemase [EC:5.1.1.1]\n",
      "SSO SSO:000000848 Alanine racemase (EC 5.1.1.1)\n",
      "b1190 ['Alanine racemase (EC 5.1.1.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11105 cell volume regulation protein A\n",
      "SSO SSO:000018668 K(+)/H(+) antiporter NhaP2\n",
      "b1191 ['K(+)/H(+) antiporter NhaP2']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01297 muramoyltetrapeptide carboxypeptidase [EC:3.4.17.13]\n",
      "SSO SSO:000005035 Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)\n",
      "b1192 ['Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08308 membrane-bound lytic murein transglycosylase E [EC:4.2.2.-]\n",
      "SSO SSO:000013368 Endo-type membrane-bound lytic murein transglycosylase A (EC 4.2.2.n2)\n",
      "b1193 ['Endo-type membrane-bound lytic murein transglycosylase A (EC 4.2.2.n2)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K21087 flagellar brake protein\n",
      "SSO SSO:000002809 Flagellar brake protein YcgR\n",
      "b1194 ['Flagellar brake protein YcgR']\n",
      "\n",
      "SSO SSO:000026677 UPF0410 protein YmgE\n",
      "b1195 ['UPF0410 protein YmgE']\n",
      "\n",
      "SSO SSO:000028200 Uncharacterized protein YcgY\n",
      "b1196 ['Uncharacterized protein YcgY']\n",
      "\n",
      "EC EC:3.2.1.28 Alpha,alpha-trehalase.\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01194 alpha,alpha-trehalase [EC:3.2.1.28]\n",
      "SSO SSO:000026169 Periplasmic trehalase (EC 3.2.1.28)\n",
      "b1197 ['Trehalase (EC 3.2.1.28)', 'Periplasmic trehalase (EC 3.2.1.28)']\n",
      "\n",
      "KO KO:K05881 phosphoenolpyruvate---glycerone phosphotransferase subunit DhaM [EC:2.7.1.121]\n",
      "b1198 ['Phosphoenolpyruvate-dihydroxyacetone phosphotransferase (EC 2.7.1.121), subunit DhaM', 'DHA-specific IIA component', 'DHA-specific phosphocarrier protein HPr', 'DHA-specific EI component']\n",
      "\n",
      "KO KO:K05879 phosphoenolpyruvate---glycerone phosphotransferase subunit DhaL [EC:2.7.1.121]\n",
      "SSO SSO:000005960 Phosphoenolpyruvate-dihydroxyacetone phosphotransferase , ADP-binding subunit DhaL (EC 2.7.1.121)\n",
      "b1199 ['Phosphoenolpyruvate-dihydroxyacetone phosphotransferase (EC 2.7.1.121), ADP-binding subunit DhaL']\n",
      "\n",
      "KO KO:K05878 phosphoenolpyruvate---glycerone phosphotransferase subunit DhaK [EC:2.7.1.121]\n",
      "SSO SSO:000005961 Phosphoenolpyruvate-dihydroxyacetone phosphotransferase , dihydroxyacetone binding subunit DhaK (EC 2.7.1.121)\n",
      "b1200 ['Phosphoenolpyruvate-dihydroxyacetone phosphotransferase (EC 2.7.1.121), dihydroxyacetone binding subunit DhaK']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K05880 transcriptional activator for dhaKLM operon\n",
      "SSO SSO:000005963 Phosphoenolpyruvate-dihydroxyacetone phosphotransferase operon regulatory protein DhaR\n",
      "b1201 ['Phosphoenolpyruvate-dihydroxyacetone phosphotransferase operon regulatory protein DhaR']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000023144 Putative adhesin\n",
      "SSO SSO:000028199 Uncharacterized protein YcgV\n",
      "b1202 ['Uncharacterized protein YcgV']\n",
      "\n",
      "KO KO:K06942 ribosome-binding ATPase\n",
      "SSO SSO:000002991 GTP-binding and nucleic acid-binding protein YchF\n",
      "b1203 ['GTP-binding and nucleic acid-binding protein YchF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01056 peptidyl-tRNA hydrolase, PTH1 family [EC:3.1.1.29]\n",
      "SSO SSO:000005814 Peptidyl-tRNA hydrolase (EC 3.1.1.29)\n",
      "b1204 ['Peptidyl-tRNA hydrolase (EC 3.1.1.29)']\n",
      "\n",
      "b1205 ['Putative membrane protein YchH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03321 sulfate permease, SulP family\n",
      "SSO SSO:000023623 Putative sulfate permease\n",
      "b1206 ['Putative sulfate permease']\n",
      "\n",
      "GO GO:0009152 purine ribonucleotide biosynthetic process\n",
      "KO KO:K00948 ribose-phosphate pyrophosphokinase [EC:2.7.6.1]\n",
      "SSO SSO:000007113 Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)\n",
      "b1207 ['Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)']\n",
      "\n",
      "EC EC:2.7.1.148 4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol kinase.\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00919 4-diphosphocytidyl-2-C-methyl-D-erythritol kinase [EC:2.7.1.148]\n",
      "SSO SSO:000000360 4-diphosphocytidyl-2-C-methyl-D-erythritol kinase (EC 2.7.1.148)\n",
      "b1208 ['4-diphosphocytidyl-2-C-methyl-D-erythritol kinase (EC 2.7.1.148)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02494 outer membrane lipoprotein LolB\n",
      "SSO SSO:000037335 Outer membrane lipoprotein component of lipoprotein transport system LolB\n",
      "b1209 ['Outer membrane lipoprotein component of lipoprotein transport system LolB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K02492 glutamyl-tRNA reductase [EC:1.2.1.70]\n",
      "SSO SSO:000003185 Glutamyl-tRNA reductase (EC 1.2.1.70)\n",
      "b1210 ['Glutamyl-tRNA reductase (EC 1.2.1.70)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02835 peptide chain release factor 1\n",
      "SSO SSO:000005788 Peptide chain release factor 1\n",
      "b1211 ['Peptide chain release factor 1']\n",
      "\n",
      "KO KO:K02493 release factor glutamine methyltransferase [EC:2.1.1.297]\n",
      "SSO SSO:000005792 Peptide chain release factor N(5)-glutamine methyltransferase (EC 2.1.1.297)\n",
      "b1212 ['Peptide chain release factor N(5)-glutamine methyltransferase (EC 2.1.1.297)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000013664 FIG002082: Protein SirB2\n",
      "b1213 ['FIG002082: Protein SirB2']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000013679 FIG002708: Protein SirB1\n",
      "b1214 ['FIG002708: Protein SirB1']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0009248 K antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01627 2-dehydro-3-deoxyphosphooctonate aldolase (KDO 8-P synthase) [EC:2.5.1.55]\n",
      "SSO SSO:000000117 2-Keto-3-deoxy-D-manno-octulosonate-8-phosphate synthase (EC 2.5.1.55)\n",
      "b1215 ['2-Keto-3-deoxy-D-manno-octulosonate-8-phosphate synthase (EC 2.5.1.55)']\n",
      "\n",
      "KO KO:K18862 small toxic polypeptide LdrA/B/C/D\n",
      "SSO SSO:000024879 Small inner membrane protein, YmgF family\n",
      "b4419 ['Small inner membrane protein, YmgF family']\n",
      "\n",
      "KO KO:K18862 small toxic polypeptide LdrA/B/C/D\n",
      "SSO SSO:000024879 Small inner membrane protein, YmgF family\n",
      "b4421 ['Small inner membrane protein, YmgF family']\n",
      "\n",
      "KO KO:K18862 small toxic polypeptide LdrA/B/C/D\n",
      "SSO SSO:000024879 Small inner membrane protein, YmgF family\n",
      "b4423 ['Small inner membrane protein, YmgF family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07300 Ca2+:H+ antiporter\n",
      "SSO SSO:000011863 Calcium/proton antiporter\n",
      "b1216 ['Calcium/proton antiporter']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K06197 cation transport regulator\n",
      "SSO SSO:000011996 Cation transport regulator chaB\n",
      "b1217 ['Cation transport regulator chaB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K07232 glutathione-specific gamma-glutamylcyclotransferase [EC:4.3.2.7]\n",
      "SSO SSO:000011995 Cation transport protein chaC\n",
      "b1218 ['Cation transport protein ChaC']\n",
      "\n",
      "KO KO:K06039 uncharacterized protein involved in oxidation of intracellular sulfur\n",
      "SSO SSO:000022727 Protein YchN\n",
      "b1219 ['Protein YchN']\n",
      "\n",
      "KO KO:K13735 adhesin/invasin\n",
      "SSO SSO:000023373 Putative invasin\n",
      "b1220 ['Putative invasin']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K07684 two-component system, NarL family, nitrate/nitrite response regulator NarL\n",
      "SSO SSO:000020537 Nitrate/nitrite response regulator protein NarL\n",
      "b1221 ['Nitrate/nitrite response regulator protein NarL']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07673 two-component system, NarL family, nitrate/nitrite sensor histidine kinase NarX [EC:2.7.13.3]\n",
      "SSO SSO:000020541 Nitrate/nitrite sensor protein NarX\n",
      "b1222 ['Nitrate/nitrite sensor protein NarX']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02575 MFS transporter, NNP family, nitrate/nitrite transporter\n",
      "SSO SSO:000005331 Nitrate/nitrite transporter\n",
      "SSO SSO:000005332 Nitrate/nitrite transporter NarK\n",
      "b1223 ['Nitrate/nitrite transporter NarK']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00370 nitrate reductase / nitrite oxidoreductase, alpha subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007016 Respiratory nitrate reductase alpha chain (EC 1.7.99.4)\n",
      "b1224 ['Respiratory nitrate reductase alpha chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00371 nitrate reductase / nitrite oxidoreductase, beta subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007017 Respiratory nitrate reductase beta chain (EC 1.7.99.4)\n",
      "b1225 ['Respiratory nitrate reductase beta chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00373 nitrate reductase molybdenum cofactor assembly chaperone NarJ/NarW\n",
      "KO KO:K00374 nitrate reductase gamma subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007018 Respiratory nitrate reductase delta chain (EC 1.7.99.4)\n",
      "b1226 ['Respiratory nitrate reductase delta chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00374 nitrate reductase gamma subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007019 Respiratory nitrate reductase gamma chain (EC 1.7.99.4)\n",
      "b1227 ['Respiratory nitrate reductase gamma chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1229 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009152 purine ribonucleotide biosynthetic process\n",
      "KO KO:K01433 formyltetrahydrofolate deformylase [EC:3.5.1.10]\n",
      "SSO SSO:000002919 Formyltetrahydrofolate deformylase (EC 3.5.1.10)\n",
      "b1232 ['Formyltetrahydrofolate deformylase (EC 3.5.1.10)']\n",
      "\n",
      "KO KO:K09858 SEC-C motif domain protein\n",
      "SSO SSO:000026583 UPF0225 protein YchJ\n",
      "b1233 ['UPF0225 protein YchJ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K07001 NTE family protein\n",
      "SSO SSO:000014401 FIG00613342: Bacterial patatin-like phospholipase domain containing protein\n",
      "b1234 ['FIG00613342: Bacterial patatin-like phospholipase domain containing protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K02485 two-component system, response regulator\n",
      "b1235 ['Regulator of RpoS']\n",
      "\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0006012 galactose metabolic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00963 UTP--glucose-1-phosphate uridylyltransferase [EC:2.7.7.9]\n",
      "SSO SSO:000008674 UTP--glucose-1-phosphate uridylyltransferase (EC 2.7.7.9)\n",
      "b1236 ['UTP--glucose-1-phosphate uridylyltransferase (EC 2.7.7.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03746 DNA-binding protein H-NS\n",
      "SSO SSO:000012901 DNA-binding protein H-NS\n",
      "b1237 ['DNA-binding protein H-NS']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00857 thymidine kinase [EC:2.7.1.21]\n",
      "SSO SSO:000008065 Thymidine kinase (EC 2.7.1.21)\n",
      "b1238 ['Thymidine kinase (EC 2.7.1.21)']\n",
      "\n",
      "SSO SSO:000026127 Transposase, Avin_45320 family\n",
      "b4573 ['Transposase, Avin_45320 family']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K04072 acetaldehyde dehydrogenase / alcohol dehydrogenase [EC:1.2.1.10 1.1.1.1]\n",
      "b1241 ['Acetaldehyde dehydrogenase (EC 1.2.1.10)', 'Alcohol dehydrogenase (EC 1.1.1.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K05595 multiple antibiotic resistance protein\n",
      "SSO SSO:000008660 UPF0056 inner membrane protein YchE\n",
      "b1242 ['UPF0056 inner membrane protein YchE']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "KO KO:K15580 oligopeptide transport system substrate-binding protein\n",
      "b1243 ['Oligopeptide ABC transporter, substrate-binding protein OppA (TC 3.A.1.5.1)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02033 peptide/nickel transport system permease protein\n",
      "KO KO:K15581 oligopeptide transport system permease protein\n",
      "b1244 ['Oligopeptide ABC transporter, permease protein OppB (TC 3.A.1.5.1)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02034 peptide/nickel transport system permease protein\n",
      "KO KO:K15582 oligopeptide transport system permease protein\n",
      "b1245 ['Oligopeptide ABC transporter, permease protein OppC (TC 3.A.1.5.1)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K02031 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K15583 oligopeptide transport system ATP-binding protein\n",
      "b1246 ['Oligopeptide ABC transporter, ATP-binding protein OppD (TC 3.A.1.5.1)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K02032 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K10823 oligopeptide transport system ATP-binding protein\n",
      "b1247 ['Oligopeptide ABC transporter, ATP-binding protein OppF (TC 3.A.1.5.1)']\n",
      "\n",
      "KO KO:K09901 uncharacterized protein\n",
      "SSO SSO:000009102 dsDNA mimic protein, highly acidic\n",
      "b1248 ['dsDNA mimic protein, highly acidic']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K06131 cardiolipin synthase A/B [EC:2.7.8.-]\n",
      "SSO SSO:000035002 Cardiolipin synthase, bacterial type ClsA\n",
      "b1249 ['Cardiolipin synthase, bacterial type ClsA']\n",
      "\n",
      "b4595 ['FIG00644304: hypothetical protein']\n",
      "\n",
      "GO GO:0005244 voltage-gated ion channel activity\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10716 voltage-gated potassium channel\n",
      "SSO SSO:000023538 Putative potassium channel protein\n",
      "b1250 ['Putative potassium channel protein']\n",
      "\n",
      "KO KO:K09780 uncharacterized protein\n",
      "SSO SSO:000008979 YciL protein\n",
      "b1251 ['YciL protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03832 periplasmic protein TonB\n",
      "SSO SSO:000039498 TonB-ExbBD energy transducing system, TonB subunit\n",
      "b1252 ['TonB-ExbBD energy transducing system, TonB subunit']\n",
      "\n",
      "KO KO:K10806 acyl-CoA thioesterase YciA [EC:3.1.2.-]\n",
      "SSO SSO:000010277 Acyl-CoA thioesterase YciA, involved in membrane biogenesis\n",
      "b1253 ['Acyl-CoA thioesterase YciA, involved in membrane biogenesis']\n",
      "\n",
      "KO KO:K06190 intracellular septation protein\n",
      "SSO SSO:000018564 Intracellular septation protein IspA\n",
      "b1254 ['Intracellular septation protein IspA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000019436 Membrane protein YciC, linked to IspA\n",
      "b1255 ['Membrane protein YciC, linked to IspA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07275 outer membrane protein\n",
      "SSO SSO:000020913 Outer membrane protein W\n",
      "b1256 ['Outer membrane protein W precursor']\n",
      "\n",
      "SSO SSO:000006455 Protein YciE\n",
      "b1257 ['Protein YciE']\n",
      "\n",
      "SSO SSO:000007759 Stress response diiron-containing protein YciF\n",
      "b1258 ['Stress response diiron-containing protein YciF']\n",
      "\n",
      "KO KO:K06884 uncharacterized protein\n",
      "SSO SSO:000028453 Uncharacterized protein YmdF\n",
      "b1259 ['Uncharacterized protein YmdF']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01695 tryptophan synthase alpha chain [EC:4.2.1.20]\n",
      "SSO SSO:000008396 Tryptophan synthase alpha chain (EC 4.2.1.20)\n",
      "b1260 ['Tryptophan synthase alpha chain (EC 4.2.1.20)']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01696 tryptophan synthase beta chain [EC:4.2.1.20]\n",
      "KO KO:K01817 phosphoribosylanthranilate isomerase [EC:5.3.1.24]\n",
      "SSO SSO:000008397 Tryptophan synthase beta chain (EC 4.2.1.20)\n",
      "b1261 ['Tryptophan synthase beta chain (EC 4.2.1.20)']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01817 phosphoribosylanthranilate isomerase [EC:5.3.1.24]\n",
      "KO KO:K13498 indole-3-glycerol phosphate synthase / phosphoribosylanthranilate isomerase [EC:4.1.1.48 5.3.1.24]\n",
      "b1262 ['Indole-3-glycerol phosphate synthase (EC 4.1.1.48)', 'Phosphoribosylanthranilate isomerase (EC 5.3.1.24)']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00766 anthranilate phosphoribosyltransferase [EC:2.4.2.18]\n",
      "KO KO:K13497 anthranilate synthase/phosphoribosyltransferase [EC:4.1.3.27 2.4.2.18]\n",
      "b1263 ['Anthranilate synthase, amidotransferase component (EC 4.1.3.27)', 'Anthranilate phosphoribosyltransferase (EC 2.4.2.18)']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "KO KO:K01657 anthranilate synthase component I [EC:4.1.3.27]\n",
      "SSO SSO:000000999 Anthranilate synthase, aminase component (EC 4.1.3.27)\n",
      "b1264 ['Anthranilate synthase, aminase component (EC 4.1.3.27)']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000026202 Trp operon leader peptide\n",
      "b1265 ['Trp operon leader peptide']\n",
      "\n",
      "KO KO:K07053 3',5'-nucleoside bisphosphate phosphatase [EC:3.1.3.97]\n",
      "SSO SSO:000021083 PHP domain protein\n",
      "b1266 ['FIG00031715: Predicted metal-dependent phosphoesterases (PHP family)']\n",
      "\n",
      "KO KO:K07566 L-threonylcarbamoyladenylate synthase [EC:2.7.7.87]\n",
      "SSO SSO:000023107 Putative RNA-binding protein\n",
      "SSO SSO:000041719 YciO protein, TsaC/YrdC paralog\n",
      "b1267 ['YciO protein, TsaC/YrdC paralog']\n",
      "\n",
      "SSO SSO:000028201 Uncharacterized protein YciQ\n",
      "b1268 ['Uncharacterized protein YciQ']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K06178 23S rRNA pseudouridine2605 synthase [EC:5.4.99.22]\n",
      "SSO SSO:000036578 LSU rRNA pseudouridine(2605) synthase (EC 5.4.99.22)\n",
      "b1269 ['LSU rRNA pseudouridine(2605) synthase (EC 5.4.99.22)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K19221 cob(I)alamin adenosyltransferase [EC:2.5.1.17]\n",
      "SSO SSO:000001618 Cob(I)alamin adenosyltransferase (EC 2.5.1.17)\n",
      "b1270 ['Cob(I)alamin adenosyltransferase (EC 2.5.1.17)']\n",
      "\n",
      "b1271 ['Putative oxidoreductase']\n",
      "\n",
      "KO KO:K04774 serine protease SohB [EC:3.4.21.-]\n",
      "b1272 ['Possible protease sohB (EC 3.4.21.-)']\n",
      "\n",
      "SSO SSO:000022728 Protein YciN\n",
      "b1273 ['Protein YciN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03168 DNA topoisomerase I [EC:5.99.1.2]\n",
      "SSO SSO:000002113 DNA topoisomerase I (EC 5.99.1.2)\n",
      "b1274 ['DNA topoisomerase I (EC 5.99.1.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0019344 cysteine biosynthetic process\n",
      "KO KO:K13634 LysR family transcriptional regulator, cys regulon transcriptional activator\n",
      "SSO SSO:000001848 Cys regulon transcriptional activator CysB\n",
      "b1275 ['Cys regulon transcriptional activator CysB']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4522 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "b4523 ['FIG00638229: hypothetical protein']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K01681 aconitate hydratase [EC:4.2.1.3]\n",
      "b1276 ['Aconitate hydratase (EC 4.2.1.3)', '2-methylisocitrate dehydratase (EC 4.2.1.99)']\n",
      "\n",
      "KO KO:K01497 GTP cyclohydrolase II [EC:3.5.4.25]\n",
      "SSO SSO:000002985 GTP cyclohydrolase II (EC 3.5.4.25)\n",
      "b1277 ['GTP cyclohydrolase II (EC 3.5.4.25)']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01096 phosphatidylglycerophosphatase B [EC:3.1.3.27 3.1.3.81 3.1.3.4 3.6.1.27]\n",
      "SSO SSO:000005940 Phosphatidylglycerophosphatase B (EC 3.1.3.27)\n",
      "b1278 ['Phosphatidylglycerophosphatase B (EC 3.1.3.27)']\n",
      "\n",
      "KO KO:K08992 lipopolysaccharide assembly protein A\n",
      "SSO SSO:000018435 Inner membrane protein yciS\n",
      "b1279 ['Inner membrane protein yciS']\n",
      "\n",
      "KO KO:K19804 lipopolysaccharide assembly protein B\n",
      "SSO SSO:000036646 Lipopolysaccharide assembly protein LapB\n",
      "b1280 ['Lipopolysaccharide assembly protein LapB']\n",
      "\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01591 orotidine-5'-phosphate decarboxylase [EC:4.1.1.23]\n",
      "SSO SSO:000005522 Orotidine 5'-phosphate decarboxylase (EC 4.1.1.23)\n",
      "b1281 [\"Orotidine 5'-phosphate decarboxylase (EC 4.1.1.23)\"]\n",
      "\n",
      "KO KO:K03113 translation initiation factor 1\n",
      "SSO SSO:000008315 Translation initiation factor SUI1-related protein\n",
      "b1282 ['Translation initiation factor SUI1-related protein']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K04062 osmotically inducible lipoprotein OsmB\n",
      "SSO SSO:000037329 Osmotically-inducible lipoprotein OsmB\n",
      "b1283 ['Osmotically-inducible lipoprotein OsmB']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000008261 Transcriptional regulatory protein YciT\n",
      "b1284 ['Transcriptional regulatory protein YciT']\n",
      "\n",
      "SSO SSO:000026685 UPF0509 protein YciZ\n",
      "b4596 ['UPF0509 protein YciZ']\n",
      "\n",
      "KO KO:K14051 c-di-GMP phosphodiesterase Gmr [EC:3.1.4.52]\n",
      "KO KO:K21023 diguanylate cyclase [EC:2.7.7.65]\n",
      "b1285 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeR']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4672 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K01147 exoribonuclease II [EC:3.1.13.1]\n",
      "SSO SSO:000002632 Exoribonuclease II (EC 3.1.13.1)\n",
      "SSO SSO:000033127 ribonuclease II\n",
      "b1286 ['Exoribonuclease II (EC 3.1.13.1)']\n",
      "\n",
      "SSO SSO:000028202 Uncharacterized protein YciW\n",
      "b1287 ['Uncharacterized protein YciW']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00208 enoyl-[acyl-carrier protein] reductase I [EC:1.3.1.9 1.3.1.10]\n",
      "SSO SSO:000002545 Enoyl-[acyl-carrier-protein] reductase [NADH] (EC 1.3.1.9)\n",
      "b1288 ['Enoyl-[acyl-carrier-protein] reductase [NADH] (EC 1.3.1.9)']\n",
      "\n",
      "SSO SSO:000028203 Uncharacterized protein YcjD\n",
      "b1289 ['Uncharacterized protein YcjD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K19230 cationic peptide transport system ATP-binding protein\n",
      "b1290 ['Peptide ABC transporter, ATP-binding protein SapF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K19229 cationic peptide transport system ATP-binding protein\n",
      "b1291 ['Peptide ABC transporter, ATP-binding protein SapD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02034 peptide/nickel transport system permease protein\n",
      "KO KO:K19228 cationic peptide transport system permease protein\n",
      "b1292 ['Peptide ABC transporter, permease protein SapC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02033 peptide/nickel transport system permease protein\n",
      "KO KO:K19227 cationic peptide transport system permease protein\n",
      "b1293 ['Peptide ABC transporter, permease protein SapB']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "KO KO:K19226 cationic peptide transport system substrate-binding protein\n",
      "b1294 ['Peptide ABC transporter, substrate-binding protein SapA']\n",
      "\n",
      "SSO SSO:000028459 Uncharacterized protein YmjA\n",
      "b1295 ['Uncharacterized protein YmjA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K14052 putrescine importer\n",
      "SSO SSO:000006783 Putrescine importer\n",
      "b1296 ['Putrescine importer']\n",
      "\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K01915 glutamine synthetase [EC:6.3.1.2]\n",
      "KO KO:K09470 gamma-glutamylputrescine synthase [EC:6.3.1.11]\n",
      "SSO SSO:000003035 Gamma-glutamyl-putrescine synthetase (EC 6.3.1.11)\n",
      "SSO SSO:000043778 glutamate---putrescine ligase (EC 6.3.1.11)\n",
      "b1297 ['Gamma-glutamyl-putrescine synthetase (EC 6.3.1.11)']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K07010 putative glutamine amidotransferase\n",
      "KO KO:K09473 gamma-glutamyl-gamma-aminobutyrate hydrolase [EC:3.5.1.94]\n",
      "SSO SSO:000003032 Gamma-glutamyl-GABA hydrolase (EC 3.5.1.94)\n",
      "SSO SSO:000041922 gamma-glutamyl-gamma-aminobutyrate hydrolase (EC 3.5.1.94)\n",
      "b1298 ['Gamma-glutamyl-GABA hydrolase (EC 3.5.1.94)']\n",
      "\n",
      "KO KO:K14056 HTH-type transcriptional regulator, repressor for puuD\n",
      "SSO SSO:000006787 Putrescine utilization regulator\n",
      "b1299 ['Putrescine utilization regulator']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K09472 4-(gamma-glutamylamino)butanal dehydrogenase [EC:1.2.1.99]\n",
      "SSO SSO:000003033 Gamma-glutamyl-aminobutyraldehyde dehydrogenase (EC 1.2.1.-)\n",
      "b1300 ['Gamma-glutamyl-aminobutyraldehyde dehydrogenase (EC 1.2.1.-)']\n",
      "\n",
      "KO KO:K09471 gamma-glutamylputrescine oxidase [EC:1.4.3.-]\n",
      "SSO SSO:000003034 Gamma-glutamyl-putrescine oxidase (EC1.4.3.-)\n",
      "b1301 ['Gamma-glutamyl-putrescine oxidase (EC1.4.3.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K00823 4-aminobutyrate aminotransferase [EC:2.6.1.19]\n",
      "KO KO:K07250 4-aminobutyrate aminotransferase / (S)-3-amino-2-methylpropionate transaminase / 5-aminovalerate transaminase [EC:2.6.1.19 2.6.1.22 2.6.1.48]\n",
      "SSO SSO:000038837 Putrescine-inducible gamma-aminobutyrate:alpha-ketoglutarate aminotransferase (EC 2.6.1.19)\n",
      "b1302 ['Putrescine-inducible gamma-aminobutyrate:alpha-ketoglutarate aminotransferase (EC 2.6.1.19)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03974 psp operon transcriptional activator\n",
      "SSO SSO:000006573 Psp operon transcriptional activator\n",
      "b1303 ['Psp operon transcriptional activator']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03969 phage shock protein A\n",
      "b1304 ['Phage shock protein A', 'Suppressor of sigma54-dependent transcription, PspA-like']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03970 phage shock protein B\n",
      "SSO SSO:000005868 Phage shock protein B\n",
      "b1305 ['Phage shock protein B']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03973 phage shock protein C\n",
      "SSO SSO:000005869 Phage shock protein C\n",
      "b1306 ['Phage shock protein C']\n",
      "\n",
      "KO KO:K03971 phage shock protein D\n",
      "SSO SSO:000005870 Phage shock protein D\n",
      "b1307 ['Phage shock protein D']\n",
      "\n",
      "KO KO:K03972 phage shock protein E\n",
      "SSO SSO:000008052 Thiosulfate:cyanide sulfurtransferase PspE (EC 2.8.1.1)\n",
      "b1308 ['Thiosulfate:cyanide sulfurtransferase PspE (EC 2.8.1.1)']\n",
      "\n",
      "EC EC:2.4.1.18 1,4-alpha-glucan branching enzyme.\n",
      "EC EC:2.4.1.19 Cyclomaltodextrin glucanotransferase.\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "EC EC:2.4.1.4 Amylosucrase.\n",
      "EC EC:2.4.1.7 Sucrose phosphorylase.\n",
      "EC EC:2.4.99.16 Starch synthase (maltosyl-transferring).\n",
      "EC EC:3.2.1.1 Alpha-amylase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.116 Glucan 1,4-alpha-maltotriohydrolase.\n",
      "EC EC:3.2.1.133 Glucan 1,4-alpha-maltohydrolase.\n",
      "EC EC:3.2.1.135 Neopullulanase.\n",
      "EC EC:3.2.1.141 4-alpha-D-((1->4)-alpha-D-glucano)trehalose trehalohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.33 Amylo-alpha-1,6-glucosidase.\n",
      "EC EC:3.2.1.41 Pullulanase.\n",
      "EC EC:3.2.1.54 Cyclomaltodextrinase.\n",
      "EC EC:3.2.1.60 Glucan 1,4-alpha-maltotetraohydrolase.\n",
      "EC EC:3.2.1.68 Isoamylase.\n",
      "EC EC:3.2.1.70 Glucan 1,6-alpha-glucosidase.\n",
      "EC EC:3.2.1.93 Alpha,alpha-phosphotrehalase.\n",
      "EC EC:3.2.1.98 Glucan 1,4-alpha-maltohexaosidase.\n",
      "EC EC:5.4.99.11 Isomaltulose synthase.\n",
      "EC EC:5.4.99.15 (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase.\n",
      "EC EC:5.4.99.16 Maltose alpha-D-glucosyltransferase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "KO KO:K00690 sucrose phosphorylase [EC:2.4.1.7]\n",
      "KO KO:K22597 glucosylglycerate phosphorylase [EC:2.4.1.352]\n",
      "b1309 ['Uncharacterized glycosyltransferase YcjM']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02027 multiple sugar transport system substrate-binding protein\n",
      "SSO SSO:000009926 ABC-type sugar transport system, periplasmic binding protein YcjN\n",
      "b1310 ['ABC-type sugar transport system, periplasmic binding protein YcjN']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02025 multiple sugar transport system permease protein\n",
      "SSO SSO:000018370 Inner membrane ABC transporter permease protein YcjO\n",
      "b1311 ['Inner membrane ABC transporter permease protein YcjO']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02026 multiple sugar transport system permease protein\n",
      "SSO SSO:000018371 Inner membrane ABC transporter permease protein YcjP\n",
      "b1312 ['Inner membrane ABC transporter permease protein YcjP']\n",
      "\n",
      "SSO SSO:000029203 Zinc-type alcohol dehydrogenase YcjQ\n",
      "b1313 ['Zinc-type alcohol dehydrogenase YcjQ']\n",
      "\n",
      "SSO SSO:000025248 Sugar phosphate isomerases/epimerases family protein YcjR\n",
      "b1314 ['Sugar phosphate isomerases/epimerases family protein YcjR']\n",
      "\n",
      "b1315 ['Putative oxidoreductase YcjS (EC 1.-.-.-), NADH-binding']\n",
      "\n",
      "EC EC:2.4.1.216 Trehalose 6-phosphate phosphorylase.\n",
      "EC EC:2.4.1.230 Kojibiose phosphorylase.\n",
      "EC EC:2.4.1.279 Nigerose phosphorylase.\n",
      "EC EC:2.4.1.282 3-O-alpha-D-glucosyl-L-rhamnose phosphorylase.\n",
      "EC EC:2.4.1.64 Alpha,alpha-trehalose phosphorylase.\n",
      "EC EC:2.4.1.8 Maltose phosphorylase.\n",
      "EC EC:3.2.1.107 Protein-glucosylgalactosylhydroxylysine glucosidase.\n",
      "EC EC:3.2.1.28 Alpha,alpha-trehalase.\n",
      "KO KO:K04844 hypothetical glycosyl hydrolase [EC:3.2.1.-]\n",
      "b1316 ['Uncharacterized glycosyl hydrolase YcjT']\n",
      "\n",
      "KO KO:K01838 beta-phosphoglucomutase [EC:5.4.2.6]\n",
      "SSO SSO:000001193 Beta-phosphoglucomutase (EC 5.4.2.6)\n",
      "b1317 ['Beta-phosphoglucomutase (EC 5.4.2.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K10112 multiple sugar transport system ATP-binding protein\n",
      "SSO SSO:000019303 Maltodextrin ABC transporter, ATP-binding protein MsmX\n",
      "b4524 ['Maltodextrin ABC transporter, ATP-binding protein MsmX']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K14053 outer membrane protein G\n",
      "SSO SSO:000005591 Outer membrane protein G precursor\n",
      "b1319 ['Outer membrane protein G precursor']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "SSO SSO:000025866 Transcriptional regulator YcjW, LacI family, possibly involved in maltodextrin utilization pathway\n",
      "b1320 ['Transcriptional regulator YcjW, LacI family, possibly involved in maltodextrin utilization pathway']\n",
      "\n",
      "KO KO:K06918 uncharacterized protein\n",
      "SSO SSO:000012431 Conserved protein YcjX with nucleoside triphosphate hydrolase domain\n",
      "b1321 ['Conserved protein YcjX with nucleoside triphosphate hydrolase domain']\n",
      "\n",
      "KO KO:K08990 putative membrane protein\n",
      "SSO SSO:000019437 Membrane protein YcjF\n",
      "b1322 ['Membrane protein YcjF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03721 transcriptional regulator of aroF, aroG, tyrA and aromatic amino acid transport\n",
      "SSO SSO:000008282 Transcriptional repressor protein TyrR\n",
      "b1323 ['Transcriptional repressor protein TyrR']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K11065 thiol peroxidase, atypical 2-Cys peroxiredoxin [EC:1.11.1.15]\n",
      "SSO SSO:000008032 Thiol peroxidase, Tpx-type (EC 1.11.1.15)\n",
      "b1324 ['Thiol peroxidase, Tpx-type (EC 1.11.1.15)']\n",
      "\n",
      "KO KO:K19802 L-Ala-D/L-Glu epimerase [EC:5.1.1.20]\n",
      "SSO SSO:000004115 L-alanine-DL-glutamate epimerase (EC 5.1.1.n1)\n",
      "b1325 ['L-alanine-DL-glutamate epimerase (EC 5.1.1.n1)']\n",
      "\n",
      "KO KO:K14054 protein MpaA\n",
      "SSO SSO:000003023 Gamma-D-Glutamyl-meso-Diaminopimelate Amidase\n",
      "b1326 ['Gamma-D-Glutamyl-meso-Diaminopimelate Amidase']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4525 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4673 ['hypothetical protein']\n",
      "\n",
      "KO KO:K06889 uncharacterized protein\n",
      "SSO SSO:000013096 Dienelactone hydrolase and related enzymes\n",
      "b1327 ['Dienelactone hydrolase and related enzymes']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K24082 Unknown\n",
      "SSO SSO:000039624 Transcriptional regulator PgrR, LysR family\n",
      "b1328 ['Transcriptional regulator PgrR, LysR family']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "KO KO:K15580 oligopeptide transport system substrate-binding protein\n",
      "SSO SSO:000036945 Murein peptide ABC transporter, substrate-binding protein (requires DppBCDF)\n",
      "b1329 ['Murein peptide ABC transporter, substrate-binding protein (requires DppBCDF)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K16052 MscS family membrane protein\n",
      "SSO SSO:000019394 Mechanosensitive ion channel\n",
      "b1330 ['Mechanosensitive ion channel']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b1331 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "SSO SSO:000027120 Uncharacterized inner membrane protein YnaJ\n",
      "b1332 ['Uncharacterized inner membrane protein YnaJ']\n",
      "\n",
      "KO KO:K14055 universal stress protein E\n",
      "SSO SSO:000008826 Universal stress protein E\n",
      "b1333 ['Universal stress protein E']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K01420 CRP/FNR family transcriptional regulator, anaerobic regulatory protein\n",
      "b1334 ['Fumarate and nitrate reduction regulatory protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006306 DNA methylation\n",
      "KO KO:K00567 methylated-DNA-[protein]-cysteine S-methyltransferase [EC:2.1.1.63]\n",
      "SSO SSO:000004870 Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)\n",
      "b1335 ['Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K12942 aminobenzoyl-glutamate transport protein\n",
      "SSO SSO:000000955 Aminobenzoyl-glutamate transport protein\n",
      "b1336 ['Aminobenzoyl-glutamate transport protein']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K12941 aminobenzoyl-glutamate utilization protein B\n",
      "SSO SSO:000001450 Catalyzes the cleavage of p-aminobenzoyl-glutamate to p-aminobenzoate and glutamate, subunit B\n",
      "b1337 ['Catalyzes the cleavage of p-aminobenzoyl-glutamate to p-aminobenzoate and glutamate, subunit B']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K12940 aminobenzoyl-glutamate utilization protein A\n",
      "SSO SSO:000001449 Catalyzes the cleavage of p-aminobenzoyl-glutamate to p-aminobenzoate and glutamate, subunit A\n",
      "b1338 ['Catalyzes the cleavage of p-aminobenzoyl-glutamate to p-aminobenzoate and glutamate, subunit A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K14057 LysR family transcriptional regulator, regulator of abg operon\n",
      "SSO SSO:000006973 Regulatory protein (induces abgABT, used to catabolize p-aminobenzoyl-glutamate)\n",
      "b1339 ['Regulatory protein (induces abgABT, used to catabolize p-aminobenzoyl-glutamate)']\n",
      "\n",
      "SSO SSO:000022389 Probable DNA endonuclease SmrA\n",
      "b1340 ['Probable DNA endonuclease SmrA']\n",
      "\n",
      "KO KO:K21088 diguanylate cyclase [EC:2.7.7.65]\n",
      "b1341 ['Diguanylate cyclase (EC 2.7.7.65) => DgcM']\n",
      "\n",
      "KO KO:K16074 zinc transporter\n",
      "SSO SSO:000029190 Zinc transport protein ZntB\n",
      "b1342 ['Zinc transport protein ZntB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K05591 ATP-independent RNA helicase DbpA [EC:3.6.4.13]\n",
      "SSO SSO:000010022 ATP-dependent 23S rRNA helicase DbpA\n",
      "b1343 ['ATP-dependent 23S rRNA helicase DbpA']\n",
      "\n",
      "KO KO:K14058 tRNA 2-thiocytidine biosynthesis protein TtcA\n",
      "b1344 ['tRNA-(cytosine32)-2-thiocytidine synthetase TtcA']\n",
      "\n",
      "KO KO:K14059 integrase\n",
      "SSO SSO:000021513 Phage integrase\n",
      "b1345 ['Phage integrase']\n",
      "\n",
      "b1346 ['ydaQ protein']\n",
      "\n",
      "KO KO:K19780 antisense regulator of RalR protein\n",
      "SSO SSO:000021531 Phage protein\n",
      "b1347 ['Phage protein']\n",
      "\n",
      "KO KO:K19779 endodeoxyribonuclease RalR [EC:3.1.-.-]\n",
      "SSO SSO:000038434 Phage-encoded restriction alleviation and modification enhancement protein\n",
      "b1348 ['Phage-encoded restriction alleviation and modification enhancement protein']\n",
      "\n",
      "KO KO:K07455 recombination protein RecT\n",
      "SSO SSO:000006960 Recombinational DNA repair protein RecT (prophage associated)\n",
      "b1349 ['Recombinational DNA repair protein RecT (prophage associated)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K10906 exodeoxyribonuclease VIII [EC:3.1.11.-]\n",
      "SSO SSO:000013527 Exodeoxyribonuclease VIII (EC 3.1.11.-)\n",
      "b1350 ['Exodeoxyribonuclease VIII (EC 3.1.11.-)']\n",
      "\n",
      "b1351 ['Protein RacC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "b4526 ['Uncharacterized phage-associated protein YdaE']\n",
      "\n",
      "SSO SSO:000038066 Phage host-killing protein Kil\n",
      "b1352 ['Phage host-killing protein Kil']\n",
      "\n",
      "SSO SSO:000038354 Phage superinfection exclusion protein B\n",
      "b1353 ['Phage superinfection exclusion protein B']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000038031 Phage head, head-DNA stabilization protein D\n",
      "b4527 ['Phage head, head-DNA stabilization protein D']\n",
      "\n",
      "SSO SSO:000021531 Phage protein\n",
      "b1355 ['Phage protein']\n",
      "\n",
      "b1356 ['Rac prophage repressor']\n",
      "\n",
      "KO KO:K22302 transcriptional repressor of cell division inhibition gene dicB\n",
      "SSO SSO:000021540 Phage repressor protein cI\n",
      "b1357 ['Phage repressor protein cI']\n",
      "\n",
      "SSO SSO:000037926 Phage activator protein cII\n",
      "b1358 ['Phage activator protein cII']\n",
      "\n",
      "SSO SSO:000021486 Phage DNA replication protein O\n",
      "b1359 ['Phage DNA replication protein O']\n",
      "\n",
      "KO KO:K02315 DNA replication protein DnaC\n",
      "KO KO:K10762 putative replication protein\n",
      "b1360 ['Bacteriophage-encoded homolog of DNA replication protein DnaC']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1361 ['hypothetical protein']\n",
      "\n",
      "KO KO:K14744 prophage endopeptidase [EC:3.4.-.-]\n",
      "SSO SSO:000038003 Phage endopeptidase Rz\n",
      "b1362 ['Phage endopeptidase Rz']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000038428 Phage-encoded host cell membrane protein Rz1\n",
      "b4528 ['Phage-encoded host cell membrane protein Rz1']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03498 trk system potassium uptake protein\n",
      "SSO SSO:000008387 Trk potassium uptake system protein TrkG\n",
      "b1363 ['Trk potassium uptake system protein TrkG']\n",
      "\n",
      "SSO SSO:000001565 Chromosome (plasmid) partitioning protein ParB\n",
      "b1365 ['Chromosome (plasmid) partitioning protein ParB']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1366 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1368 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4570 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b1370 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "SSO SSO:000038365 Phage tail fiber, side tail fiber protein Stf\n",
      "b1372 ['Phage tail fiber, side tail fiber protein Stf']\n",
      "\n",
      "SSO SSO:000038366 Phage tail fiber, tail fiber assembly protein Tfa\n",
      "b1373 ['Phage tail fiber, tail fiber assembly protein Tfa']\n",
      "\n",
      "KO KO:K14060 putative DNA-invertase from lambdoid prophage Rac\n",
      "SSO SSO:000039146 Serine recombinase, PinQ/PinR-type\n",
      "b1374 ['Serine recombinase, PinQ/PinR-type']\n",
      "\n",
      "SSO SSO:000028669 Uncharacterized protein, YnaE family\n",
      "b1375 ['Uncharacterized protein, YnaE family']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4638 ['hypothetical protein']\n",
      "\n",
      "KO KO:K14061 universal stress protein F\n",
      "SSO SSO:000009440 universal stress protein F\n",
      "b1376 ['Universal stress protein F']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K09476 outer membrane pore protein F\n",
      "KO KO:K14062 outer membrane protein N\n",
      "SSO SSO:000005576 Outer membrane porin OmpN\n",
      "b1377 ['Outer membrane porin OmpN']\n",
      "\n",
      "KO KO:K03737 pyruvate-ferredoxin/flavodoxin oxidoreductase [EC:1.2.7.1 1.2.7.-]\n",
      "SSO SSO:000006856 Pyruvate-flavodoxin oxidoreductase (EC 1.2.7.-)\n",
      "b1378 ['Pyruvate-flavodoxin oxidoreductase']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K09712 uncharacterized protein\n",
      "SSO SSO:000028210 Uncharacterized protein YdbJ\n",
      "b4529 ['Uncharacterized protein YdbJ']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K03668 heat shock protein HslJ\n",
      "SSO SSO:000017653 Heat shock protein HslJ\n",
      "b1379 ['Heat shock protein HslJ']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K03778 D-lactate dehydrogenase [EC:1.1.1.28]\n",
      "SSO SSO:000002031 D-lactate dehydrogenase (EC 1.1.1.28)\n",
      "b1380 ['D-lactate dehydrogenase (EC 1.1.1.28)']\n",
      "\n",
      "SSO SSO:000023716 Putative uncharacterized protein ydbH\n",
      "b1381 ['Putative uncharacterized protein ydbH']\n",
      "\n",
      "SSO SSO:000018986 Lipoprotein\n",
      "SSO SSO:000028661 Uncharacterized protein ynbE\n",
      "b1382 ['Uncharacterized protein YnbE']\n",
      "\n",
      "KO KO:K09978 uncharacterized protein\n",
      "SSO SSO:000023717 Putative uncharacterized protein ydbL, may be related to amine metabolism\n",
      "b1383 ['Putative uncharacterized protein ydbL, may be related to amine metabolism']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K14063 AraC family transcriptional regulator, positive regulator of tynA and feaB\n",
      "b1384 ['regulatory protein for 2-phenylethylamine catabolism']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "KO KO:K00146 phenylacetaldehyde dehydrogenase [EC:1.2.1.39]\n",
      "SSO SSO:000005890 Phenylacetaldehyde dehydrogenase (EC 1.2.1.39)\n",
      "b1385 ['Phenylacetaldehyde dehydrogenase (EC 1.2.1.39)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K00276 primary-amine oxidase [EC:1.4.3.21]\n",
      "SSO SSO:000020024 Monoamine oxidase\n",
      "b1386 ['Monoamine oxidase (1.4.3.4)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02618 oxepin-CoA hydrolase / 3-oxo-5,6-dehydrosuberyl-CoA semialdehyde dehydrogenase [EC:3.3.2.12 1.2.1.91]\n",
      "b1387 ['3-oxo-5,6-dehydrosuberyl-CoA semialdehyde dehydrogenase (EC 1.2.1.91)', '2-oxepin-2(3H)-ylideneacetyl-CoA hydrolase (EC 3.3.2.12)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02609 ring-1,2-phenylacetyl-CoA epoxidase subunit PaaA [EC:1.14.13.149]\n",
      "SSO SSO:000000027 1,2-phenylacetyl-CoA epoxidase, subunit A (EC 1.14.13.149)\n",
      "b1388 ['1,2-phenylacetyl-CoA epoxidase, subunit A (EC 1.14.13.149)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02610 ring-1,2-phenylacetyl-CoA epoxidase subunit PaaB\n",
      "SSO SSO:000000028 1,2-phenylacetyl-CoA epoxidase, subunit B (EC 1.14.13.149)\n",
      "b1389 ['1,2-phenylacetyl-CoA epoxidase, subunit B (EC 1.14.13.149)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02611 ring-1,2-phenylacetyl-CoA epoxidase subunit PaaC [EC:1.14.13.149]\n",
      "SSO SSO:000000029 1,2-phenylacetyl-CoA epoxidase, subunit C (EC 1.14.13.149)\n",
      "b1390 ['1,2-phenylacetyl-CoA epoxidase, subunit C (EC 1.14.13.149)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02612 ring-1,2-phenylacetyl-CoA epoxidase subunit PaaD\n",
      "SSO SSO:000000030 1,2-phenylacetyl-CoA epoxidase, subunit D (EC 1.14.13.149)\n",
      "b1391 ['1,2-phenylacetyl-CoA epoxidase, subunit D (EC 1.14.13.149)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02613 ring-1,2-phenylacetyl-CoA epoxidase subunit PaaE\n",
      "SSO SSO:000000031 1,2-phenylacetyl-CoA epoxidase, subunit E (EC 1.14.13.149)\n",
      "b1392 ['1,2-phenylacetyl-CoA epoxidase, subunit E (EC 1.14.13.149)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01692 enoyl-CoA hydratase [EC:4.2.1.17]\n",
      "SSO SSO:000000079 2,3-dehydroadipyl-CoA hydratase (EC 4.2.1.17)\n",
      "b1393 ['2,3-dehydroadipyl-CoA hydratase (EC 4.2.1.17)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01692 enoyl-CoA hydratase [EC:4.2.1.17]\n",
      "KO KO:K15866 2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA isomerase [EC:5.3.3.18]\n",
      "SSO SSO:000000026 1,2-epoxyphenylacetyl-CoA isomerase (EC 5.3.3.18)\n",
      "b1394 ['1,2-epoxyphenylacetyl-CoA isomerase (EC 5.3.3.18)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00074 3-hydroxybutyryl-CoA dehydrogenase [EC:1.1.1.157]\n",
      "SSO SSO:000000291 3-hydroxyadipyl-CoA dehydrogenase\n",
      "b1395 ['3-hydroxyadipyl-CoA dehydrogenase']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02614 acyl-CoA thioesterase [EC:3.1.2.-]\n",
      "SSO SSO:000000800 Acyl-coenzyme A thioesterase PaaD (Pse.pu.) (E. coli PaaI)\n",
      "b1396 ['Acyl-coenzyme A thioesterase PaaD (Pse.pu.) (E. coli PaaI)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00626 acetyl-CoA C-acetyltransferase [EC:2.3.1.9]\n",
      "KO KO:K00632 acetyl-CoA acyltransferase [EC:2.3.1.16]\n",
      "KO KO:K02615 3-oxo-5,6-didehydrosuberyl-CoA/3-oxoadipyl-CoA thiolase [EC:2.3.1.223 2.3.1.174]\n",
      "SSO SSO:000000339 3-oxoadipyl-CoA thiolase @ 3-oxo-5,6-dehydrosuberyl-CoA thiolase (EC 2.3.1.223) (EC 2.3.1.174)\n",
      "b1397 ['3-oxoadipyl-CoA thiolase (EC 2.3.1.174)', '3-oxo-5,6-dehydrosuberyl-CoA thiolase (EC 2.3.1.223)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01912 phenylacetate-CoA ligase [EC:6.2.1.30]\n",
      "SSO SSO:000005892 Phenylacetate-coenzyme A ligase (EC 6.2.1.30)\n",
      "b1398 ['Phenylacetate-coenzyme A ligase (EC 6.2.1.30)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02616 phenylacetic acid degradation operon negative regulatory protein\n",
      "SSO SSO:000005893 Phenylacetic acid degradation operon negative regulatory protein PaaX\n",
      "b1399 ['Phenylacetic acid degradation operon negative regulatory protein PaaX']\n",
      "\n",
      "KO KO:K02617 phenylacetic acid degradation protein\n",
      "SSO SSO:000005895 Phenylacetic acid degradation protein PaaY\n",
      "b1400 ['Phenylacetic acid degradation protein PaaY']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4674 ['hypothetical protein']\n",
      "\n",
      "KO KO:K12516 putative surface-exposed virulence protein\n",
      "SSO SSO:000011718 COG3210: Large exoproteins involved in heme utilization or adhesion\n",
      "b4492 ['COG3210: Large exoproteins involved in heme utilization or adhesion']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000026095 Transposase InsD for insertion element IS2\n",
      "b1402 ['Transposase InsD for insertion element IS2']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026094 Transposase InsC for insertion element IS2\n",
      "b1403 ['Transposase InsC for insertion element IS2']\n",
      "\n",
      "KO KO:K07482 transposase, IS30 family\n",
      "SSO SSO:000026099 Transposase InsI for insertion sequence element IS30\n",
      "b1404 ['Transposase InsI for insertion sequence element IS30']\n",
      "\n",
      "KO KO:K05275 pyridoxine 4-dehydrogenase [EC:1.1.1.65]\n",
      "SSO SSO:000038850 Pyridoxine 4-dehydrogenase PhxI (EC 1.1.1.65)\n",
      "b1406 ['Pyridoxine 4-dehydrogenase PhxI (EC 1.1.1.65)']\n",
      "\n",
      "SSO SSO:000028209 Uncharacterized protein YdbD\n",
      "b1407 ['Uncharacterized protein YdbD']\n",
      "\n",
      "SSO SSO:000011529 CDP-alcohol phosphatidyltransferase family protein\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b1408 ['CDP-alcohol phosphatidyltransferase family protein']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00981 phosphatidate cytidylyltransferase [EC:2.7.7.41]\n",
      "SSO SSO:000005932 Phosphatidate cytidylyltransferase (EC 2.7.7.41)\n",
      "b1409 ['Phosphatidate cytidylyltransferase (EC 2.7.7.41)']\n",
      "\n",
      "SSO SSO:000009716 Putative esterase\n",
      "SSO SSO:000023386 Putative lipase in cluster with Phosphatidate cytidylyltransferase\n",
      "b1410 ['Putative lipase in cluster with Phosphatidate cytidylyltransferase']\n",
      "\n",
      "SSO SSO:000024546 Ser/Thr and Tyr protein phosphatase (dual specificity)\n",
      "b1411 ['Ser/Thr and Tyr protein phosphatase (dual specificity)']\n",
      "\n",
      "KO KO:K01118 FMN-dependent NADH-azoreductase [EC:1.7.1.17]\n",
      "SSO SSO:000016592 FMN-dependent NADH-azoreductase (EC 1.7.1.6)\n",
      "b1412 ['FMN-dependent NADH-azoreductase (EC 1.7.1.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K03578 ATP-dependent helicase HrpA [EC:3.6.4.13]\n",
      "KO KO:K21843 tetratricopeptide repeat protein 7\n",
      "SSO SSO:000010055 ATP-dependent helicase (hrpA)\n",
      "SSO SSO:000023025 Putative ATP-dependent helicase\n",
      "b1413 ['ATP-dependent helicase HrpA']\n",
      "\n",
      "SSO SSO:000040032 Uncharacterized SAM-binding protein YdcF\n",
      "b1414 ['Uncharacterized SAM-binding protein YdcF']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009438 methylglyoxal metabolic process\n",
      "GO GO:0009441 glycolate metabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K07248 lactaldehyde dehydrogenase / glycolaldehyde dehydrogenase [EC:1.2.1.22 1.2.1.21]\n",
      "b1415 ['Aldehyde dehydrogenase A (EC 1.2.1.22)', 'Glycolaldehyde dehydrogenase (EC 1.2.1.21)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4493 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K12262 cytochrome b561\n",
      "SSO SSO:000012612 Cytochrome b561\n",
      "b1418 ['Cytochrome b561']\n",
      "\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "SSO SSO:000028211 Uncharacterized protein YdcA\n",
      "b1419 ['Uncharacterized protein YdcA']\n",
      "\n",
      "KO KO:K18921 protein HokB\n",
      "SSO SSO:000036299 Hok/Gef-like protein\n",
      "b4428 ['Hok/Gef-like protein']\n",
      "\n",
      "SSO SSO:000023953 Regulatory protein MokB\n",
      "b1420 ['Regulatory protein MokB']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03406 methyl-accepting chemotaxis protein\n",
      "KO KO:K05876 methyl-accepting chemotaxis protein III, ribose and galactose sensor receptor\n",
      "SSO SSO:000019574 Methyl-accepting chemotaxis protein III (ribose and galactose chemoreceptor protein)\n",
      "b1421 ['Methyl-accepting chemotaxis protein III (ribose and galactose chemoreceptor protein)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02623 LysR family transcriptional regulator, pca operon transcriptional activator\n",
      "SSO SSO:000004584 LysR family transcriptional regulator YdcI\n",
      "b1422 ['LysR family transcriptional regulator YdcI']\n",
      "\n",
      "SSO SSO:000028214 Uncharacterized protein YdcJ\n",
      "b1423 ['Uncharacterized protein YdcJ']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03670 periplasmic glucans biosynthesis protein\n",
      "b1424 ['Glucans biosynthesis protein D precursor']\n",
      "\n",
      "KO KO:K09794 uncharacterized protein\n",
      "SSO SSO:000028213 Uncharacterized protein YdcH\n",
      "b1426 ['Uncharacterized protein YdcH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K03817 ribosomal-protein-serine acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000007137 Ribosomal-protein-L7p-serine acetyltransferase (EC 2.3.1.-)\n",
      "b1427 ['Ribosomal-protein-L7p-serine acetyltransferase']\n",
      "\n",
      "SSO SSO:000014075 FIG005189: putative transferase clustered with tellurite resistance proteins TehA/TehB\n",
      "b1428 ['FIG005189: putative transferase clustered with tellurite resistance proteins TehA/TehB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03304 tellurite resistance protein\n",
      "SSO SSO:000025509 Tellurite resistance protein TehA\n",
      "b1429 ['Tellurite resistance protein TehA']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K16868 tellurite methyltransferase [EC:2.1.1.265]\n",
      "SSO SSO:000007922 Tellurite methyltransferase (EC 2.1.1.265)\n",
      "b1430 ['Tellurite methyltransferase (EC 2.1.1.265)']\n",
      "\n",
      "SSO SSO:000018986 Lipoprotein\n",
      "SSO SSO:000027236 Uncharacterized membrane lipoprotein clustered with tellurite resistance proteins TehA/TehB\n",
      "b1431 ['Uncharacterized membrane lipoprotein clustered with tellurite resistance proteins TehA/TehB']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4578 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07496 putative transposase\n",
      "SSO SSO:000023666 Putative transposase InsQ for insertion sequence element IS609\n",
      "b1432 ['Putative transposase InsQ for insertion sequence element IS609']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05782 benzoate membrane transport protein\n",
      "SSO SSO:000011059 Benzoate transport protein\n",
      "b1433 ['Benzoate transport protein']\n",
      "\n",
      "KO KO:K23779 Unknown\n",
      "b1434 ['Transcriptional regulator yidN, Cro/CI family']\n",
      "\n",
      "KO KO:K08303 putative protease [EC:3.4.-.-]\n",
      "SSO SSO:000023495 Putative peptidase\n",
      "b1435 ['Uncharacterized protease YdcP']\n",
      "\n",
      "SSO SSO:000028464 Uncharacterized protein YncJ\n",
      "b1436 ['Uncharacterized protein YncJ']\n",
      "\n",
      "KO KO:K07339 mRNA interferase HicA [EC:3.1.-.-]\n",
      "SSO SSO:000006333 Probable mRNA interferase HicA\n",
      "b4532 ['Probable mRNA interferase HicA']\n",
      "\n",
      "KO KO:K18843 antitoxin HicB\n",
      "SSO SSO:000001017 Antitoxin HicB\n",
      "b1438 ['Antitoxin HicB']\n",
      "\n",
      "GO GO:0004009 obsolete ATP-binding cassette (ABC) transporter activity\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "b1439 ['DNA-binding transcriptional regulator, MocR family', 'aminotransferase domain']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02055 putative spermidine/putrescine transport system substrate-binding protein\n",
      "SSO SSO:000009832 ABC transporter periplasmic-binding protein YdcS\n",
      "b1440 ['ABC transporter periplasmic-binding protein YdcS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000009816 ABC transporter ATP-binding protein YdcT\n",
      "b1441 ['ABC transporter ATP-binding protein YdcT']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000009834 ABC transporter permease protein YdcU\n",
      "b1442 ['ABC transporter permease protein YdcU']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000009835 ABC transporter permease protein YdcV\n",
      "b1443 ['ABC transporter permease protein YdcV']\n",
      "\n",
      "KO KO:K00137 aminobutyraldehyde dehydrogenase [EC:1.2.1.19]\n",
      "SSO SSO:000000352 4-aminobutyraldehyde dehydrogenase (EC 1.2.1.19)\n",
      "b1444 ['4-aminobutyraldehyde dehydrogenase (EC 1.2.1.19)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4598 ['hypothetical protein']\n",
      "\n",
      "KO KO:K18839 toxin GhoT\n",
      "SSO SSO:000005523 Orphan toxin OrtT\n",
      "b1445 ['Orphan toxin OrtT']\n",
      "\n",
      "SSO SSO:000008764 Uncharacterized protein YdcY (not antitoxin to OrtT)\n",
      "b1446 ['Uncharacterized protein YdcY (not antitoxin to OrtT)']\n",
      "\n",
      "KO KO:K09936 bacterial/archaeal transporter family-2 protein\n",
      "SSO SSO:000036384 Inner membrane protein YdcZ\n",
      "b1447 ['Inner membrane protein YdcZ']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03823 phosphinothricin acetyltransferase [EC:2.3.1.183]\n",
      "SSO SSO:000004118 L-amino acid N-acyltransferase MnaT\n",
      "b1448 ['L-amino acid N-acyltransferase MnaT']\n",
      "\n",
      "KO KO:K23256 Unknown\n",
      "SSO SSO:000006690 Putative oxidoreductase YncB\n",
      "b1449 ['Putative oxidoreductase YncB']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K13654 GntR family transcriptional regulator, colanic acid and biofilm gene transcriptional regulator\n",
      "SSO SSO:000039702 Transcriptional regulator YncC, GntR family\n",
      "b1450 ['Transcriptional regulator YncC, GntR family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "b1451 ['Probable tonB-dependent receptor yncD precursor']\n",
      "\n",
      "SSO SSO:000028462 Uncharacterized protein YncE\n",
      "b1452 ['Uncharacterized protein YncE']\n",
      "\n",
      "GO GO:0006529 asparagine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11738 L-asparagine permease\n",
      "SSO SSO:000018742 L-asparagine permease\n",
      "b1453 ['L-asparagine permease']\n",
      "\n",
      "KO KO:K11208 GST-like protein\n",
      "SSO SSO:000008697 Uncharacterized GST-like protein yncG\n",
      "SSO SSO:000017311 Glutathione-S-transferase homolog\n",
      "b1454 ['Uncharacterized GST-like protein yncG']\n",
      "\n",
      "SSO SSO:000028463 Uncharacterized protein YncH\n",
      "b1455 ['Uncharacterized protein YncH']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1456 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028212 Uncharacterized protein YdcD\n",
      "b1457 ['Uncharacterized protein YdcD']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1459 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000017503 H repeat-associated protein, YhhI family\n",
      "b1460 ['H repeat-associated protein, YhhI family']\n",
      "\n",
      "KO KO:K01821 4-oxalocrotonate tautomerase [EC:5.3.2.6]\n",
      "SSO SSO:000009695 4-oxalocrotonate tautomerase\n",
      "SSO SSO:000025494 Tautomerase PptA\n",
      "b1461 ['Tautomerase PptA']\n",
      "\n",
      "SSO SSO:000020292 NAD(P)H-flavin oxidoreductase\n",
      "b1462 ['NAD(P)H-flavin oxidoreductase']\n",
      "\n",
      "GO GO:0008152 metabolic process\n",
      "KO KO:K00675 N-hydroxyarylamine O-acetyltransferase [EC:2.3.1.118]\n",
      "SSO SSO:000005134 N-hydroxyarylamine O-acetyltransferase (EC 2.3.1.118)\n",
      "b1463 ['N-hydroxyarylamine O-acetyltransferase (EC 2.3.1.118)']\n",
      "\n",
      "KO KO:K06998 trans-2,3-dihydro-3-hydroxyanthranilate isomerase [EC:5.3.3.17]\n",
      "SSO SSO:000005876 Phenazine biosynthesis protein PhzF like\n",
      "b1464 ['Phenazine biosynthesis protein PhzF like']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00374 nitrate reductase gamma subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007019 Respiratory nitrate reductase gamma chain (EC 1.7.99.4)\n",
      "b1465 ['Respiratory nitrate reductase gamma chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00373 nitrate reductase molybdenum cofactor assembly chaperone NarJ/NarW\n",
      "KO KO:K00374 nitrate reductase gamma subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007018 Respiratory nitrate reductase delta chain (EC 1.7.99.4)\n",
      "b1466 ['Respiratory nitrate reductase delta chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00371 nitrate reductase / nitrite oxidoreductase, beta subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007017 Respiratory nitrate reductase beta chain (EC 1.7.99.4)\n",
      "b1467 ['Respiratory nitrate reductase beta chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00370 nitrate reductase / nitrite oxidoreductase, alpha subunit [EC:1.7.5.1 1.7.99.-]\n",
      "SSO SSO:000007016 Respiratory nitrate reductase alpha chain (EC 1.7.99.4)\n",
      "b1468 ['Respiratory nitrate reductase alpha chain (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02575 MFS transporter, NNP family, nitrate/nitrite transporter\n",
      "SSO SSO:000005331 Nitrate/nitrite transporter\n",
      "SSO SSO:000037201 Nitrate/nitrite transporter NarU\n",
      "b1469 ['Nitrate/nitrite transporter NarU']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "SSO SSO:000023326 Putative glycoportein\n",
      "b1471 ['Putative glycoportein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1472 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000021459 Permease of the drug/metabolite transporter (DMT) superfamily\n",
      "b1473 ['Permease of the drug/metabolite transporter (DMT) superfamily']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K08348 formate dehydrogenase-N, alpha subunit [EC:1.17.5.3]\n",
      "b1474 ['Formate dehydrogenase N alpha subunit (EC 1.2.1.2)', 'selenocysteine-containing']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00124 formate dehydrogenase iron-sulfur subunit\n",
      "KO KO:K08349 formate dehydrogenase-N, beta subunit\n",
      "SSO SSO:000002878 Formate dehydrogenase N beta subunit (EC 1.2.1.2)\n",
      "b1475 ['Formate dehydrogenase N beta subunit (EC 1.2.1.2)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00127 formate dehydrogenase subunit gamma\n",
      "KO KO:K08350 formate dehydrogenase-N, gamma subunit\n",
      "SSO SSO:000002879 Formate dehydrogenase N gamma subunit (EC 1.2.1.2)\n",
      "b1476 ['Formate dehydrogenase N gamma subunit (EC 1.2.1.2)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K21498 antitoxin HigA-1\n",
      "SSO SSO:000001018 Antitoxin HigA\n",
      "b1477 ['Antitoxin HigA']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K13953 alcohol dehydrogenase, propanol-preferring [EC:1.1.1.1]\n",
      "SSO SSO:000000860 Alcohol dehydrogenase (EC 1.1.1.1)\n",
      "b1478 ['Alcohol dehydrogenase (EC 1.1.1.1)']\n",
      "\n",
      "GO GO:0006094 gluconeogenesis\n",
      "KO KO:K00027 malate dehydrogenase (oxaloacetate-decarboxylating) [EC:1.1.1.38]\n",
      "KO KO:K00029 malate dehydrogenase (oxaloacetate-decarboxylating)(NADP+) [EC:1.1.1.40]\n",
      "SSO SSO:000005188 NAD-dependent malic enzyme (EC 1.1.1.38)\n",
      "b1479 ['NAD-dependent malic enzyme (EC 1.1.1.38)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02972 stationary-phase-induced ribosome-associated protein\n",
      "SSO SSO:000007732 Stationary-phase-induced ribosome-associated protein\n",
      "b1480 ['Stationary-phase-induced ribosome-associated protein']\n",
      "\n",
      "SSO SSO:000011050 Bdm protein\n",
      "b1481 ['Bdm protein']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K04063 osmotically inducible protein OsmC\n",
      "SSO SSO:000005856 Peroxiredoxin OsmC (EC 1.11.1.15)\n",
      "b1482 ['Peroxiredoxin OsmC (EC 1.11.1.15)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000035308 D,D-dipeptide ABC transporter, ATP-binding protein DdpF\n",
      "b1483 ['D,D-dipeptide ABC transporter, ATP-binding protein DdpF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02031 peptide/nickel transport system ATP-binding protein\n",
      "SSO SSO:000035307 D,D-dipeptide ABC transporter, ATP-binding protein DdpD\n",
      "b1484 ['D,D-dipeptide ABC transporter, ATP-binding protein DdpD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02034 peptide/nickel transport system permease protein\n",
      "SSO SSO:000035310 D,D-dipeptide ABC transporter, permease protein DdpC\n",
      "b1485 ['D,D-dipeptide ABC transporter, permease protein DdpC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02033 peptide/nickel transport system permease protein\n",
      "SSO SSO:000035309 D,D-dipeptide ABC transporter, permease protein DdpB\n",
      "b1486 ['D,D-dipeptide ABC transporter, permease protein DdpB']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "SSO SSO:000035311 D,D-dipeptide ABC transporter, substrate-binding protein DdpA\n",
      "b1487 ['D,D-dipeptide ABC transporter, substrate-binding protein DdpA']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K08641 zinc D-Ala-D-Ala dipeptidase [EC:3.4.13.22]\n",
      "SSO SSO:000001996 D-alanyl-D-alanine dipeptidase (EC 3.4.13.22)\n",
      "b1488 ['D-alanyl-D-alanine dipeptidase (EC 3.4.13.22)']\n",
      "\n",
      "KO KO:K13243 c-di-GMP-specific phosphodiesterase [EC:3.1.4.52]\n",
      "KO KO:K21023 diguanylate cyclase [EC:2.7.7.65]\n",
      "b1489 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => Heme-regulated, oxygen sensor PdeO']\n",
      "\n",
      "KO KO:K13069 diguanylate cyclase [EC:2.7.7.65]\n",
      "b1490 ['Diguanylate cyclase (EC 2.7.7.65) => Globin-coupled heme-based oxygen sensor DgcO']\n",
      "\n",
      "SSO SSO:000011658 COG1649 predicted glycoside hydrolase\n",
      "b1491 ['COG1649 predicted glycoside hydrolase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K20265 glutamate:GABA antiporter\n",
      "SSO SSO:000006327 Probable glutamate/gamma-aminobutyrate antiporter\n",
      "b1492 ['Probable glutamate/gamma-aminobutyrate antiporter']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006520 cellular amino acid metabolic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01580 glutamate decarboxylase [EC:4.1.1.15]\n",
      "SSO SSO:000003148 Glutamate decarboxylase (EC 4.1.1.15)\n",
      "b1493 ['Glutamate decarboxylase (EC 4.1.1.15)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K07263 zinc protease [EC:3.4.24.-]\n",
      "SSO SSO:000022618 Probable zinc protease pqqL (EC 3.4.99.-)\n",
      "b1494 ['Probable zinc protease pqqL (EC 3.4.99.-)']\n",
      "\n",
      "SSO SSO:000023123 Putative TonB dependent outer membrane receptor\n",
      "SSO SSO:000028215 Uncharacterized protein YddB\n",
      "b1495 ['Uncharacterized protein YddB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02471 vitamin B12/bleomycin/antimicrobial peptide transport system ATP-binding/permease protein\n",
      "SSO SSO:000009817 ABC transporter ATP-binding protein YddA\n",
      "b1496 ['ABC transporter ATP-binding protein YddA']\n",
      "\n",
      "KO KO:K06871 uncharacterized protein\n",
      "SSO SSO:000016956 GALNS arylsulfatase regulator (Fe-S oxidoreductase)\n",
      "b1497 ['GALNS arylsulfatase regulator (Fe-S oxidoreductase)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K01138 uncharacterized sulfatase [EC:3.1.6.-]\n",
      "SSO SSO:000020181 N-acetylgalactosamine 6-sulfate sulfatase (GALNS)\n",
      "b1498 ['N-acetylgalactosamine 6-sulfate sulfatase (GALNS)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000025867 Transcriptional regulator YdeO, AraC family\n",
      "b1499 ['Transcriptional regulator YdeO, AraC family']\n",
      "\n",
      "KO KO:K23236 Unknown\n",
      "SSO SSO:000026323 Two-component-system connector protein SafA\n",
      "b1500 ['Two-component-system connector protein SafA']\n",
      "\n",
      "SSO SSO:000006654 Putative formate dehydrogenase oxidoreductase protein\n",
      "b1501 ['Putative formate dehydrogenase oxidoreductase protein']\n",
      "\n",
      "KO KO:K07350 minor fimbrial subunit\n",
      "SSO SSO:000031003 mannose-specific adhesin FimH\n",
      "b1502 ['mannose-specific adhesin FimH']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07349 minor fimbrial subunit\n",
      "SSO SSO:000026326 Type 1 fimbrae adaptor subunit FimG\n",
      "b1503 ['Type 1 fimbrae adaptor subunit FimG']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "KO KO:K07348 minor fimbrial subunit\n",
      "SSO SSO:000027082 Uncharacterized fimbrial-like protein YdeS\n",
      "b1504 ['Uncharacterized fimbrial-like protein YdeS']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1505 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000023839 Transcriptional regulator, AraC family\n",
      "b1506 ['Transcriptional regulator, AraC family']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006259 DNA metabolic process\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K07154 serine/threonine-protein kinase HipA [EC:2.7.11.1]\n",
      "b1507 ['Toxin HigB', 'Protein kinase domain of HipA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006259 DNA metabolic process\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K15773 HTH-type transcriptional regulator / antitoxin HipB\n",
      "b1508 ['HipB protein', 'Antitoxin HigA']\n",
      "\n",
      "KO KO:K12678 autotransporter family porin\n",
      "SSO SSO:000027204 Uncharacterized lipoprotein YdeK\n",
      "b4696 ['Uncharacterized lipoprotein YdeK']\n",
      "\n",
      "KO KO:K11216 autoinducer-2 kinase [EC:2.7.1.189]\n",
      "SSO SSO:000001124 Autoinducer 2 (AI-2) kinase LsrK (EC 2.7.1.-)\n",
      "b1511 ['Autoinducer 2 (AI-2) kinase LsrK (EC 2.7.1.-)']\n",
      "\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K11531 lsr operon transcriptional repressor\n",
      "SSO SSO:000004557 LsrR, transcriptional repressor of lsr operon\n",
      "b1512 ['LsrR, transcriptional repressor of lsr operon']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10558 AI-2 transport system ATP-binding protein\n",
      "SSO SSO:000034720 Autoinducer 2 (AI-2) ABC transporter, dimeric ATP-binding protein\n",
      "b1513 ['Autoinducer 2 (AI-2) ABC transporter, dimeric ATP-binding protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10556 AI-2 transport system permease protein\n",
      "SSO SSO:000034721 Autoinducer 2 (AI-2) ABC transporter, permease protein LsrC\n",
      "b1514 ['Autoinducer 2 (AI-2) ABC transporter, permease protein LsrC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10557 AI-2 transport system permease protein\n",
      "SSO SSO:000034722 Autoinducer 2 (AI-2) ABC transporter, permease protein LsrD\n",
      "b1515 ['Autoinducer 2 (AI-2) ABC transporter, permease protein LsrD']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10555 AI-2 transport system substrate-binding protein\n",
      "SSO SSO:000034723 Autoinducer 2 (AI-2) ABC transporter, substrate-binding protein\n",
      "b1516 ['Autoinducer 2 (AI-2) ABC transporter, substrate-binding protein']\n",
      "\n",
      "KO KO:K08321 3-hydroxy-5-phosphonooxypentane-2,4-dione thiolase [EC:2.3.1.245]\n",
      "SSO SSO:000034055 3-hydroxy-5-phosphonooxypentane-2,4-dione thiolase (EC 2.3.1.245)\n",
      "b1517 ['3-hydroxy-5-phosphonooxypentane-2,4-dione thiolase (EC 2.3.1.245)']\n",
      "\n",
      "KO KO:K11530 (4S)-4-hydroxy-5-phosphonooxypentane-2,3-dione isomerase [EC:5.3.1.32]\n",
      "SSO SSO:000033858 (4S)-4-hydroxy-5-phosphonooxypentane-2,3-dione isomerase (EC 5.3.1.32)\n",
      "b1518 ['(4S)-4-hydroxy-5-phosphonooxypentane-2,3-dione isomerase (EC 5.3.1.32)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00598 trans-aconitate 2-methyltransferase [EC:2.1.1.144]\n",
      "SSO SSO:000008131 Trans-aconitate 2-methyltransferase (EC 2.1.1.144)\n",
      "b1519 ['Trans-aconitate 2-methyltransferase (EC 2.1.1.144)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K08994 ion channel-forming bestrophin family protein\n",
      "SSO SSO:000026570 UPF0187 protein YneE\n",
      "b1520 ['UPF0187 protein YneE']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00041 tagaturonate reductase [EC:1.1.1.58]\n",
      "SSO SSO:000000937 Altronate oxidoreductase (EC 1.1.1.58)\n",
      "b1521 ['Altronate oxidoreductase (EC 1.1.1.58)']\n",
      "\n",
      "b1522 ['Diguanylate cyclase (EC 2.7.7.65) => DgcF']\n",
      "\n",
      "SSO SSO:000023912 Uncharacterized protein YneG\n",
      "b1523 ['Uncharacterized protein YneG']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01425 glutaminase [EC:3.5.1.2]\n",
      "SSO SSO:000003165 Glutaminase (EC 3.5.1.2)\n",
      "b1524 ['Glutaminase (EC 3.5.1.2)']\n",
      "\n",
      "KO KO:K08324 succinate-semialdehyde dehydrogenase [EC:1.2.1.16 1.2.1.24]\n",
      "b1525 ['Succinate-semialdehyde dehydrogenase [NAD] (EC 1.2.1.24)', 'Succinate-semialdehyde dehydrogenase [NADP+] (EC 1.2.1.79)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K21959 LysR family transcriptional regulator, cell division regulator\n",
      "SSO SSO:000004587 LysR family transcriptional regulator YneJ\n",
      "b1526 ['LysR family transcriptional regulator YneJ']\n",
      "\n",
      "SSO SSO:000028467 Uncharacterized protein YneK\n",
      "b1527 ['Uncharacterized protein YneK']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015524 obsolete L-arabinose/beta-D-thiogalactopyranoside:hydrogen antiporter activity\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08159 MFS transporter, DHA1 family, L-arabinose/isopropyl-beta-D-thiogalactopyranoside export protein\n",
      "b1528 ['Probable sugar efflux transporter']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K05595 multiple antibiotic resistance protein\n",
      "SSO SSO:000008659 UPF0056 inner membrane protein MarC\n",
      "b1529 ['UPF0056 inner membrane protein MarC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03712 MarR family transcriptional regulator, multiple antibiotic resistance protein MarR\n",
      "SSO SSO:000005031 Multiple antibiotic resistance protein MarR\n",
      "b1530 ['Multiple antibiotic resistance protein MarR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K13632 AraC family transcriptional regulator, mar-sox-rob regulon activator\n",
      "SSO SSO:000005029 Multiple antibiotic resistance protein MarA\n",
      "b1531 ['Multiple antibiotic resistance protein MarA']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K13630 multiple antibiotic resistance protein MarB\n",
      "SSO SSO:000005030 Multiple antibiotic resistance protein MarB\n",
      "b1532 ['Multiple antibiotic resistance protein MarB']\n",
      "\n",
      "KO KO:K15268 O-acetylserine/cysteine efflux transporter\n",
      "SSO SSO:000021459 Permease of the drug/metabolite transporter (DMT) superfamily\n",
      "b1533 ['Permease of the drug/metabolite transporter (DMT) superfamily']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000023657 Putative transport protein\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b1534 ['Putative transport protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4599 ['hypothetical protein']\n",
      "\n",
      "KO KO:K13069 diguanylate cyclase [EC:2.7.7.65]\n",
      "b1535 ['Diguanylate cyclase (EC 2.7.7.65) => Zinc-regulated DgcZ']\n",
      "\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "b1536 ['Putative periplasmic protein']\n",
      "\n",
      "SSO SSO:000005307 Nicotinamide-nucleotide amidase paralog YdeJ, no functional activity\n",
      "b1537 ['Nicotinamide-nucleotide amidase paralog YdeJ, no functional activity']\n",
      "\n",
      "KO KO:K01284 peptidyl-dipeptidase Dcp [EC:3.4.15.5]\n",
      "SSO SSO:000002348 Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)\n",
      "b1538 ['Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)']\n",
      "\n",
      "KO KO:K16066 3-hydroxy acid dehydrogenase / malonic semialdehyde reductase [EC:1.1.1.381 1.1.1.-]\n",
      "b1539 ['NADP-dependent 3-hydroxy acid dehydrogenase YdfG (EC 1.1.1.381)', '3-hydroxypropionate dehydrogenase (EC 1.1.1.298)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K22293 GntR family transcriptional regulator, rspAB operon transcriptional repressor\n",
      "b1540 ['Transcriptional regulator YdfH, GntR family']\n",
      "\n",
      "SSO SSO:000023586 Putative selenium-binding protein YdfZ\n",
      "b1541 ['Putative selenium-binding protein YdfZ']\n",
      "\n",
      "KO KO:K00040 fructuronate reductase [EC:1.1.1.57]\n",
      "SSO SSO:000002034 D-mannonate oxidoreductase (EC 1.1.1.57)\n",
      "b1542 ['D-mannonate oxidoreductase (EC 1.1.1.57)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08173 MFS transporter, MHS family, metabolite:H+ symporter\n",
      "SSO SSO:000040011 Uncharacterized MFS-type transporter YdfJ\n",
      "b4600 ['Uncharacterized MFS-type transporter YdfJ']\n",
      "\n",
      "SSO SSO:000028669 Uncharacterized protein, YnaE family\n",
      "b1544 ['Uncharacterized protein, YnaE family']\n",
      "\n",
      "KO KO:K14060 putative DNA-invertase from lambdoid prophage Rac\n",
      "SSO SSO:000039146 Serine recombinase, PinQ/PinR-type\n",
      "b1545 ['Serine recombinase, PinQ/PinR-type']\n",
      "\n",
      "SSO SSO:000038366 Phage tail fiber, tail fiber assembly protein Tfa\n",
      "b1546 ['Phage tail fiber, tail fiber assembly protein Tfa']\n",
      "\n",
      "SSO SSO:000038365 Phage tail fiber, side tail fiber protein Stf\n",
      "b1547 ['Phage tail fiber, side tail fiber protein Stf']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1548 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000002149 DNA-damage-inducible protein I\n",
      "b4533 ['DNA-damage-inducible protein I']\n",
      "\n",
      "SSO SSO:000038245 Phage protein YbcV\n",
      "b1549 ['Phage protein YbcV']\n",
      "\n",
      "b1550 ['GnsB protein']\n",
      "\n",
      "SSO SSO:000028471 Uncharacterized protein YnfN\n",
      "b1551 ['Uncharacterized protein YnfN']\n",
      "\n",
      "KO KO:K03704 cold shock protein (beta-ribbon, CspA family)\n",
      "b1552 ['Cold shock protein of CSP family => CspB/CspG/CspI (naming convention as in E.coli)']\n",
      "\n",
      "b1553 ['Uncharacterized protein RzpQ']\n",
      "\n",
      "b4689 ['phage-tail assembly-like protein']\n",
      "\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "KO KO:K01185 lysozyme [EC:3.2.1.17]\n",
      "SSO SSO:000037947 Phage lysozyme R (EC 3.2.1.17)\n",
      "b1554 ['Phage lysozyme R (EC 3.2.1.17)']\n",
      "\n",
      "SSO SSO:000028667 Uncharacterized protein, YdfR family\n",
      "b1555 ['Uncharacterized protein, YdfR family']\n",
      "\n",
      "SSO SSO:000038049 Phage holin/antiholin component S\n",
      "b1556 ['Phage holin/antiholin component S']\n",
      "\n",
      "KO KO:K03704 cold shock protein (beta-ribbon, CspA family)\n",
      "b1557 ['Cold shock protein of CSP family => CspB/CspG/CspI (naming convention as in E.coli)']\n",
      "\n",
      "b1558 ['Cold shock protein of CSP family => CspF/CspH (naming convention as in E.coli)']\n",
      "\n",
      "SSO SSO:000021495 Phage antitermination protein Q\n",
      "b1559 ['Phage antitermination protein Q']\n",
      "\n",
      "SSO SSO:000038250 Phage protein YdfU family\n",
      "b1560 ['Phage protein YdfU family']\n",
      "\n",
      "SSO SSO:000034658 Antitoxin Rem\n",
      "b1561 ['Antitoxin Rem']\n",
      "\n",
      "KO KO:K18919 protein HokC/D\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1562 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K06218 mRNA interferase RelE/StbE\n",
      "SSO SSO:000009178 mRNA interferase RelE\n",
      "b1563 ['mRNA interferase RelE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K18918 RHH-type transcriptional regulator, rel operon repressor / antitoxin RelB\n",
      "SSO SSO:000006983 RelB/StbD replicon stabilization protein (antitoxin to RelE/StbE)\n",
      "b1564 ['RelB/StbD replicon stabilization protein (antitoxin to RelE/StbE)']\n",
      "\n",
      "SSO SSO:000041445 Uncharacterized protein YdfV\n",
      "b1565 ['Uncharacterized protein YdfV']\n",
      "\n",
      "SSO SSO:000022702 Protein FlxA (FliA regulated)\n",
      "b1566 ['Protein FlxA (FliA regulated)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1567 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1568 ['hypothetical protein']\n",
      "\n",
      "KO KO:K22302 transcriptional repressor of cell division inhibition gene dicB\n",
      "b1569 ['Protein DicC, repressor of division inhibition gene dicB']\n",
      "\n",
      "KO KO:K22300 HTH-type transcriptional regulator, cell division transcriptional repressor\n",
      "SSO SSO:000038411 Phage transcriptional regulator cro\n",
      "b1570 ['Phage transcriptional regulator cro']\n",
      "\n",
      "SSO SSO:000038031 Phage head, head-DNA stabilization protein D\n",
      "b1571 ['Phage head, head-DNA stabilization protein D']\n",
      "\n",
      "SSO SSO:000028216 Uncharacterized protein YdfB\n",
      "b1572 ['Uncharacterized protein YdfB']\n",
      "\n",
      "SSO SSO:000028665 Uncharacterized protein, YdfC family\n",
      "b1573 ['Uncharacterized protein, YdfC family']\n",
      "\n",
      "KO KO:K22304 cell division inhibition protein DicB\n",
      "SSO SSO:000012027 Cell division inhibition protein DicB\n",
      "b1575 ['Cell division inhibition protein DicB']\n",
      "\n",
      "SSO SSO:000028666 Uncharacterized protein, YdfD family\n",
      "b1576 ['Uncharacterized protein, YdfD family']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1577 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1578 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1579 ['hypothetical protein']\n",
      "\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K08322 L-gulonate 5-dehydrogenase [EC:1.1.1.380]\n",
      "SSO SSO:000025114 Starvation sensing protein RspB\n",
      "b1580 ['Starvation sensing protein RspB']\n",
      "\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K08323 mannonate dehydratase [EC:4.2.1.8]\n",
      "SSO SSO:000004709 Mannonate dehydratase (EC 4.2.1.8)\n",
      "b1581 ['Mannonate dehydratase (EC 4.2.1.8)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K09771 small multidrug resistance family-3 protein\n",
      "SSO SSO:000026494 UPF0060 membrane protein YnfA\n",
      "b1582 ['UPF0060 membrane protein YnfA']\n",
      "\n",
      "SSO SSO:000026684 UPF0482 protein YnfB\n",
      "b1583 ['UPF0482 protein YnfB']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "KO KO:K00657 diamine N-acetyltransferase [EC:2.3.1.57]\n",
      "SSO SSO:000007623 Spermidine N1-acetyltransferase (EC 2.3.1.57)\n",
      "b1584 ['Spermidine N1-acetyltransferase (EC 2.3.1.57)']\n",
      "\n",
      "SSO SSO:000039916 UPF0257 lipoprotein YnfC\n",
      "b1585 ['UPF0257 lipoprotein YnfC']\n",
      "\n",
      "SSO SSO:000028470 Uncharacterized protein YnfE\n",
      "b1586 ['Uncharacterized protein YnfE']\n",
      "\n",
      "KO KO:K07306 anaerobic dimethyl sulfoxide reductase subunit A [EC:1.8.5.3]\n",
      "KO KO:K07309 Tat-targeted selenate reductase subunit YnfE [EC:1.97.1.9]\n",
      "SSO SSO:000010651 Anaerobic selenate reductase, molybdenum cofactor-containing periplasmic protein\n",
      "b1587 ['Anaerobic selenate reductase, molybdenum cofactor-containing periplasmic protein']\n",
      "\n",
      "KO KO:K07306 anaerobic dimethyl sulfoxide reductase subunit A [EC:1.8.5.3]\n",
      "KO KO:K07310 Tat-targeted selenate reductase subunit YnfF [EC:1.97.1.9]\n",
      "SSO SSO:000010651 Anaerobic selenate reductase, molybdenum cofactor-containing periplasmic protein\n",
      "b1588 ['Anaerobic selenate reductase, molybdenum cofactor-containing periplasmic protein']\n",
      "\n",
      "KO KO:K07307 anaerobic dimethyl sulfoxide reductase subunit B (DMSO reductase iron- sulfur subunit)\n",
      "KO KO:K07311 Tat-targeted selenate reductase subunit YnfG\n",
      "SSO SSO:000000979 Anaerobic dimethyl sulfoxide reductase chain B , iron-sulfur binding subunit (EC 1.8.5.3)\n",
      "b1589 ['Anaerobic dimethyl sulfoxide reductase chain B (EC 1.8.5.3), iron-sulfur binding subunit']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K07308 anaerobic dimethyl sulfoxide reductase subunit C (DMSO reductase anchor subunit)\n",
      "KO KO:K07312 Tat-targeted selenate reductase subunit YnfH\n",
      "SSO SSO:000010650 Anaerobic selenate reductase protein YnfH\n",
      "SSO SSO:000031305 oxidoreductase, membrane subunit\n",
      "b1590 ['Anaerobic selenate reductase protein YnfH']\n",
      "\n",
      "KO KO:K23349 Unknown\n",
      "SSO SSO:000010646 Anaerobic dimethyl sulfoxide reductase chaperone DmsD\n",
      "b1591 ['Anaerobic dimethyl sulfoxide reductase chaperone DmsD']\n",
      "\n",
      "KO KO:K03281 chloride channel protein, CIC family\n",
      "SSO SSO:000028961 Voltage-gated ClC-type chloride channel ClcB\n",
      "b1592 ['Voltage-gated ClC-type chloride channel ClcB']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K01935 dethiobiotin synthetase [EC:6.3.3.3]\n",
      "SSO SSO:000034208 Dethiobiotin synthase BioD (EC 6.3.3.3)\n",
      "b1593 ['Dethiobiotin synthase BioD (EC 6.3.3.3)']\n",
      "\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K15545 transcriptional regulator of PTS gene\n",
      "SSO SSO:000019692 Mlc, transcriptional repressor of MalT (the transcriptional activator of maltose regulon) and manXYZ operon\n",
      "b1594 ['Mlc, transcriptional repressor of MalT (the transcriptional activator of maltose regulon) and manXYZ operon']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000004588 LysR family transcriptional regulator YnfL\n",
      "b1595 ['LysR family transcriptional regulator YnfL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08224 MFS transporter, YNFM family, putative membrane transport protein\n",
      "SSO SSO:000040013 Uncharacterized MFS-type transporter YnfM\n",
      "b1596 ['Uncharacterized MFS-type transporter YnfM']\n",
      "\n",
      "GO GO:0009268 response to pH\n",
      "SSO SSO:000010203 Acid shock protein precursor\n",
      "b1597 ['Acid shock protein precursor']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4601 ['hypothetical protein']\n",
      "\n",
      "KO KO:K04775 protease YdgD [EC:3.4.21.-]\n",
      "SSO SSO:000023495 Putative peptidase\n",
      "SSO SSO:000041511 Uncharacterized serine protease YdgD\n",
      "b1598 ['Uncharacterized serine protease YdgD']\n",
      "\n",
      "KO KO:K11742 spermidine export protein MdtI\n",
      "SSO SSO:000007625 Spermidine export protein MdtI\n",
      "b1599 ['Spermidine export protein MdtI']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11743 spermidine export protein MdtJ\n",
      "SSO SSO:000007626 Spermidine export protein MdtJ\n",
      "b1600 ['Spermidine export protein MdtJ']\n",
      "\n",
      "KO KO:K11744 AI-2 transport protein TqsA\n",
      "SSO SSO:000009969 AI-2 transport protein TqsA\n",
      "b1601 ['AI-2 transport protein TqsA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00325 H+-translocating NAD(P) transhydrogenase subunit beta [EC:1.6.1.2 7.1.1.1]\n",
      "SSO SSO:000005166 NAD(P) transhydrogenase subunit beta (EC 1.6.1.2)\n",
      "b1602 ['NAD(P) transhydrogenase subunit beta (EC 1.6.1.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00324 H+-translocating NAD(P) transhydrogenase subunit alpha [EC:1.6.1.2 7.1.1.1]\n",
      "b1603 ['NAD(P) transhydrogenase N-domain of subunit alpha (EC 1.6.1.2)', 'NAD(P) transhydrogenase C-domain of subunit alpha (EC 1.6.1.2)']\n",
      "\n",
      "SSO SSO:000038715 Protein YdgH\n",
      "b1604 ['Protein YdgH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03758 arginine:ornithine antiporter / lysine permease\n",
      "SSO SSO:000001050 Arginine/ornithine antiporter ArcD\n",
      "b1605 ['Arginine/ornithine antiporter ArcD']\n",
      "\n",
      "KO KO:K13938 dihydromonapterin reductase / dihydrofolate reductase [EC:1.5.1.50 1.5.1.3]\n",
      "SSO SSO:000002867 FolM Alternative dihydrofolate reductase 1\n",
      "b1606 ['FolM Alternative dihydrofolate reductase 1']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K02442 membrane protein GlpM\n",
      "SSO SSO:000036385 Inner membrane protein YdgC\n",
      "b1607 ['Inner membrane protein YdgC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07661 two-component system, OmpR family, response regulator RstA\n",
      "SSO SSO:000039846 Two-component transcriptional response regulator RstA\n",
      "b1608 ['Two-component transcriptional response regulator RstA']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07639 two-component system, OmpR family, sensor histidine kinase RstB [EC:2.7.13.3]\n",
      "SSO SSO:000007413 Sensory histidine kinase in two-component regulatory system with RstA\n",
      "b1609 ['Sensory histidine kinase in two-component regulatory system with RstA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K10748 DNA replication terminus site-binding protein\n",
      "SSO SSO:000012872 DNA replication terminus site-binding protein\n",
      "b1610 ['DNA replication terminus site-binding protein']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00027 malate dehydrogenase (oxaloacetate-decarboxylating) [EC:1.1.1.38]\n",
      "KO KO:K01679 fumarate hydratase, class II [EC:4.2.1.2]\n",
      "SSO SSO:000002952 Fumarate hydratase class II (EC 4.2.1.2)\n",
      "b1611 ['Fumarate hydratase class II (EC 4.2.1.2)']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "KO KO:K01676 fumarate hydratase, class I [EC:4.2.1.2]\n",
      "SSO SSO:000002950 Fumarate hydratase class I, aerobic (EC 4.2.1.2)\n",
      "b1612 ['Fumarate hydratase class I, aerobic (EC 4.2.1.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01809 mannose-6-phosphate isomerase [EC:5.3.1.8]\n",
      "SSO SSO:000004712 Mannose-6-phosphate isomerase (EC 5.3.1.8)\n",
      "b1613 ['Mannose-6-phosphate isomerase (EC 5.3.1.8)']\n",
      "\n",
      "SSO SSO:000023053 Putative GTP-binding protein YdgA\n",
      "b1614 ['Putative GTP-binding protein YdgA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16140 putative glucuronide porin\n",
      "SSO SSO:000003137 Glucuronide transport facilitator UidC\n",
      "b1615 ['Glucuronide transport facilitator UidC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03292 glycoside/pentoside/hexuronide:cation symporter, GPH family\n",
      "KO KO:K16139 glucuronide carrier protein\n",
      "SSO SSO:000003138 Glucuronide transporter UidB\n",
      "b1616 ['Glucuronide transporter UidB']\n",
      "\n",
      "EC EC:3.2.1.146 Beta-galactofuranosidase.\n",
      "EC EC:3.2.1.152 Mannosylglycoprotein endo-beta-mannosidase.\n",
      "EC EC:3.2.1.165 Exo-1,4-beta-D-glucosaminidase.\n",
      "EC EC:3.2.1.23 Beta-galactosidase.\n",
      "EC EC:3.2.1.25 Beta-mannosidase.\n",
      "EC EC:3.2.1.31 Beta-glucuronidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.55 Non-reducing end alpha-L-arabinofuranosidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01195 beta-glucuronidase [EC:3.2.1.31]\n",
      "SSO SSO:000001185 Beta-glucuronidase (EC 3.2.1.31)\n",
      "b1617 ['beta-glucuronidase (EC 3.2.1.31)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K16138 TetR/AcrR family transcriptional regulator, repressor for uid operon\n",
      "SSO SSO:000008267 Transcriptional repressor UidR\n",
      "SSO SSO:000025979 Transcriptional repressor\n",
      "b1618 ['Transcriptional repressor UidR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00059 3-oxoacyl-[acyl-carrier protein] reductase [EC:1.1.1.100]\n",
      "KO KO:K00076 7-alpha-hydroxysteroid dehydrogenase [EC:1.1.1.159]\n",
      "SSO SSO:000009760 7-alpha-hydroxysteroid dehydrogenase\n",
      "b1619 ['7-alpha-hydroxysteroid dehydrogenase (EC 1.1.1.159)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K16136 LacI family transcriptional regulator, maltose regulon regulatory protein\n",
      "SSO SSO:000019314 Maltose regulon regulatory protein MalI (repressor for malXY)\n",
      "b1620 ['Maltose regulon regulatory protein MalI (repressor for malXY)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02791 PTS system, maltose/glucose-specific IIC component\n",
      "b1621 ['PTS system, maltose and glucose-specific IIC component', 'PTS system, maltose and glucose-specific IIB component (EC 2.7.1.208)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K14155 cysteine-S-conjugate beta-lyase [EC:4.4.1.13]\n",
      "b1622 ['Cystathionine beta-lyase MalY (EC 4.4.1.8)', 'Maltose regulon modulator']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01488 adenosine deaminase [EC:3.5.4.4]\n",
      "SSO SSO:000000811 Adenosine deaminase (EC 3.5.4.4)\n",
      "b1623 ['Adenosine deaminase (EC 3.5.4.4)']\n",
      "\n",
      "KO KO:K16044 scyllo-inositol 2-dehydrogenase (NADP+) [EC:1.1.1.371]\n",
      "SSO SSO:000027385 Uncharacterized oxidoreductase YdgJ\n",
      "b1624 ['Uncharacterized oxidoreductase YdgJ']\n",
      "\n",
      "KO KO:K22718 divisome-associated membrane protein\n",
      "SSO SSO:000029455 beta-lactam resistance protein\n",
      "b4409 ['beta-lactam resistance protein']\n",
      "\n",
      "KO KO:K05839 haemolysin expression modulating protein\n",
      "SSO SSO:000012281 Cnu protein\n",
      "b1625 ['Cnu protein']\n",
      "\n",
      "SSO SSO:000036386 Inner membrane protein YdgK\n",
      "b1626 ['Inner membrane protein YdgK']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03617 Na+-translocating ferredoxin:NAD+ oxidoreductase subunit A\n",
      "SSO SSO:000002448 Electron transport complex protein RnfA\n",
      "b1627 ['Electron transport complex protein RnfA']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K03616 Na+-translocating ferredoxin:NAD+ oxidoreductase subunit B [EC:7.2.1.2]\n",
      "SSO SSO:000002449 Electron transport complex protein RnfB\n",
      "b1628 ['Electron transport complex protein RnfB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03615 Na+-translocating ferredoxin:NAD+ oxidoreductase subunit C [EC:7.2.1.2]\n",
      "SSO SSO:000002450 Electron transport complex protein RnfC\n",
      "b1629 ['Electron transport complex protein RnfC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03614 Na+-translocating ferredoxin:NAD+ oxidoreductase subunit D [EC:7.2.1.2]\n",
      "SSO SSO:000002451 Electron transport complex protein RnfD\n",
      "b1630 ['Electron transport complex protein RnfD']\n",
      "\n",
      "KO KO:K03612 Na+-translocating ferredoxin:NAD+ oxidoreductase subunit G\n",
      "SSO SSO:000002453 Electron transport complex protein RnfG\n",
      "b1631 ['Electron transport complex protein RnfG']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03613 Na+-translocating ferredoxin:NAD+ oxidoreductase subunit E\n",
      "SSO SSO:000002452 Electron transport complex protein RnfE\n",
      "b1632 ['Electron transport complex protein RnfE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0009314 response to radiation\n",
      "KO KO:K10773 endonuclease III [EC:4.2.99.18]\n",
      "SSO SSO:000002472 Endonuclease III (EC 4.2.99.18)\n",
      "b1633 ['Endonuclease III (EC 4.2.99.18)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03305 proton-dependent oligopeptide transporter, POT family\n",
      "SSO SSO:000013068 Di-tripeptide/H+ symporter DtpA\n",
      "b1634 ['Di-tripeptide/H+ symporter DtpA']\n",
      "\n",
      "KO KO:K00799 glutathione S-transferase [EC:2.5.1.18]\n",
      "SSO SSO:000003209 Glutathione S-transferase (EC 2.5.1.18)\n",
      "b1635 ['Glutathione S-transferase (EC 2.5.1.18)']\n",
      "\n",
      "KO KO:K00868 pyridoxine kinase [EC:2.7.1.35]\n",
      "SSO SSO:000006808 Pyridoxal kinase (EC 2.7.1.35)\n",
      "b1636 ['Pyridoxal kinase (EC 2.7.1.35)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01866 tyrosyl-tRNA synthetase [EC:6.1.1.1]\n",
      "SSO SSO:000008591 Tyrosyl-tRNA synthetase (EC 6.1.1.1)\n",
      "b1637 ['Tyrosyl-tRNA synthetase (EC 6.1.1.1)']\n",
      "\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "KO KO:K00275 pyridoxamine 5'-phosphate oxidase [EC:1.4.3.5]\n",
      "SSO SSO:000006811 Pyridoxamine 5'-phosphate oxidase (EC 1.4.3.5)\n",
      "b1638 [\"Pyridoxamine 5'-phosphate oxidase (EC 1.4.3.5)\"]\n",
      "\n",
      "SSO SSO:000003871 Inhibitor of vertebrate c-type lysozyme, outer membrane => MliC\n",
      "b1639 ['Inhibitor of vertebrate c-type lysozyme, outer membrane => MliC']\n",
      "\n",
      "KO KO:K09001 anhydro-N-acetylmuramic acid kinase [EC:2.7.1.170]\n",
      "SSO SSO:000000991 Anhydro-N-acetylmuramic acid kinase (EC 2.7.1.-)\n",
      "b1640 ['Anhydro-N-acetylmuramic acid kinase (EC 2.7.1.170)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06077 outer membrane lipoprotein SlyB\n",
      "SSO SSO:000020875 Outer membrane lipoprotein\n",
      "b1641 ['Outer membrane lipoprotein pcp precursor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K06075 MarR family transcriptional regulator, transcriptional regulator for hemolysin\n",
      "SSO SSO:000025852 Transcriptional regulator SlyA\n",
      "b1642 ['Transcriptional regulator SlyA']\n",
      "\n",
      "SSO SSO:000007973 Tetrapartite efflux system component, FusD-like\n",
      "b1643 ['Tetrapartite efflux system component, FusD-like']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000007977 Tetrapartite efflux system, membrane fusion component FusE-like\n",
      "b1644 ['Tetrapartite efflux system, membrane fusion component FusE-like']\n",
      "\n",
      "SSO SSO:000007975 Tetrapartite efflux system, inner membrane component FusBC-like\n",
      "b1645 ['Tetrapartite efflux system, inner membrane component FusBC-like']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K04565 superoxide dismutase, Cu-Zn family [EC:1.15.1.1]\n",
      "SSO SSO:000007860 Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)\n",
      "b1646 ['Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)']\n",
      "\n",
      "SSO SSO:000020964 Oxidoreductase YdhF\n",
      "b1647 ['Oxidoreductase YdhF']\n",
      "\n",
      "KO KO:K06938 uncharacterized protein\n",
      "SSO SSO:000028220 Uncharacterized protein YdhL\n",
      "b1648 ['Uncharacterized protein YdhL']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K16137 TetR/AcrR family transcriptional regulator, transcriptional repressor for nem operon\n",
      "SSO SSO:000025984 Transcriptional repressor NemR, AcrR family\n",
      "b1649 ['Transcriptional repressor NemR, AcrR family']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K10680 N-ethylmaleimide reductase [EC:1.-.-.-]\n",
      "SSO SSO:000020229 N-ethylmaleimide reductase\n",
      "b1650 ['N-ethylmaleimide reductase']\n",
      "\n",
      "GO GO:0009438 methylglyoxal metabolic process\n",
      "KO KO:K01759 lactoylglutathione lyase [EC:4.4.1.5]\n",
      "SSO SSO:000004390 Lactoylglutathione lyase (EC 4.4.1.5)\n",
      "b1651 ['Lactoylglutathione lyase (EC 4.4.1.5)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K03683 ribonuclease T [EC:3.1.13.-]\n",
      "SSO SSO:000007095 Ribonuclease T (EC 3.1.13.-)\n",
      "b1652 ['Ribonuclease T']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K03724 ATP-dependent helicase Lhr and Lhr-like helicase [EC:3.6.4.-]\n",
      "SSO SSO:000022385 Probable ATP-dependent helicase lhr (EC 3.6.1.-)\n",
      "SSO SSO:000023025 Putative ATP-dependent helicase\n",
      "b1653 ['Probable ATP-dependent helicase lhr (EC 3.6.1.-)']\n",
      "\n",
      "KO KO:K07390 monothiol glutaredoxin\n",
      "SSO SSO:000036908 Monothiol glutaredoxin GrxD\n",
      "b1654 ['Monothiol glutaredoxin GrxD']\n",
      "\n",
      "KO KO:K19303 murein DD-endopeptidase [EC:3.4.-.-]\n",
      "KO KO:K21471 peptidoglycan DL-endopeptidase CwlO [EC:3.4.-.-]\n",
      "SSO SSO:000020124 Murein DD-endopeptidase MepH\n",
      "b1655 ['Murein DD-endopeptidase MepH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K04564 superoxide dismutase, Fe-Mn family [EC:1.15.1.1]\n",
      "SSO SSO:000007862 Superoxide dismutase [Fe] (EC 1.15.1.1)\n",
      "b1656 ['Superoxide dismutase [Fe] (EC 1.15.1.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K19577 MFS transporter, DHA1 family, inner membrane transport protein\n",
      "SSO SSO:000018441 Inner membrane transport protein YdhP\n",
      "b1657 ['Inner membrane transport protein YdhP']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4602 ['hypothetical protein']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006545 glycine biosynthetic process\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "KO KO:K03604 LacI family transcriptional regulator, purine nucleotide synthesis repressor\n",
      "SSO SSO:000006581 Purine nucleotide synthesis repressor\n",
      "b1658 ['Purine nucleotide synthesis repressor']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000019129 LysR-family transcriptional regulator YdhB\n",
      "b1659 ['LysR-family transcriptional regulator YdhB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07552 MFS transporter, DHA1 family, multidrug resistance protein\n",
      "SSO SSO:000018440 Inner membrane transport protein YdhC\n",
      "b1660 ['Inner membrane transport protein YdhC']\n",
      "\n",
      "KO KO:K00574 cyclopropane-fatty-acyl-phospholipid synthase [EC:2.1.1.79]\n",
      "SSO SSO:000001839 Cyclopropane-fatty-acyl-phospholipid synthase (EC 2.1.1.79)\n",
      "b1661 ['Cyclopropane-fatty-acyl-phospholipid synthase (EC 2.1.1.79)']\n",
      "\n",
      "KO KO:K00793 riboflavin synthase [EC:2.5.1.9]\n",
      "SSO SSO:000007065 Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)\n",
      "b1662 ['Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03327 multidrug resistance protein, MATE family\n",
      "SSO SSO:000005016 Multidrug efflux transporter MdtK/NorM (MATE family)\n",
      "b1663 ['Multidrug efflux transporter MdtK/NorM (MATE family)']\n",
      "\n",
      "SSO SSO:000028221 Uncharacterized protein YdhQ\n",
      "b1664 ['Uncharacterized protein YdhQ']\n",
      "\n",
      "SSO SSO:000023442 Putative monooxygenase\n",
      "b1667 ['Putative monooxygenase YdhR']\n",
      "\n",
      "SSO SSO:000028222 Uncharacterized protein YdhS\n",
      "b1668 ['Uncharacterized protein YdhS']\n",
      "\n",
      "SSO SSO:000028223 Uncharacterized protein YdhT\n",
      "b1669 ['Uncharacterized protein YdhT']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08354 thiosulfate reductase cytochrome b subunit\n",
      "SSO SSO:000025635 Thiosulfate reductase subunit like\n",
      "b1670 ['Thiosulfate reductase subunit like']\n",
      "\n",
      "KO KO:K04014 protein NrfC\n",
      "SSO SSO:000027077 Uncharacterized ferredoxin-like protein YdhX\n",
      "b1671 ['Uncharacterized ferredoxin-like protein YdhX']\n",
      "\n",
      "SSO SSO:000028224 Uncharacterized protein YdhW\n",
      "b1672 ['Uncharacterized protein YdhW']\n",
      "\n",
      "KO KO:K03738 aldehyde:ferredoxin oxidoreductase [EC:1.2.7.5]\n",
      "SSO SSO:000023489 Putative oxidoreductase subunit\n",
      "b1673 ['Uncharacterized molybdopterin-containing oxidoreductase YdhV']\n",
      "\n",
      "SSO SSO:000027078 Uncharacterized ferredoxin-like protein YdhY\n",
      "b1674 ['Uncharacterized ferredoxin-like protein YdhY']\n",
      "\n",
      "SSO SSO:000035993 Fumarate hydratase FumD (EC 4.2.1.2)\n",
      "b1675 ['Fumarate hydratase FumD (EC 4.2.1.2)']\n",
      "\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00873 pyruvate kinase [EC:2.7.1.40]\n",
      "SSO SSO:000006852 Pyruvate kinase (EC 2.7.1.40)\n",
      "b1676 ['Pyruvate kinase (EC 2.7.1.40)']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06078 murein lipoprotein\n",
      "SSO SSO:000019287 Major outer membrane lipoprotein Lpp\n",
      "b1677 ['Major outer membrane lipoprotein Lpp']\n",
      "\n",
      "KO KO:K19234 L,D-transpeptidase YnhG\n",
      "b1678 ['L,D-transpeptidase => YnhG']\n",
      "\n",
      "KO KO:K02426 cysteine desulfuration protein SufE\n",
      "SSO SSO:000025281 Sulfur acceptor protein \n",
      "b1679 ['Sulfur acceptor protein => iron-sulfur cluster assembly SufE']\n",
      "\n",
      "KO KO:K11717 cysteine desulfurase / selenocysteine lyase [EC:2.8.1.7 4.4.1.16]\n",
      "b1680 ['Cysteine desulfurase (EC 2.8.1.7) => SufS']\n",
      "\n",
      "KO KO:K09015 Fe-S cluster assembly protein SufD\n",
      "SSO SSO:000004026 Iron-sulfur cluster assembly protein SufD\n",
      "b1681 ['Iron-sulfur cluster assembly protein SufD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K09013 Fe-S cluster assembly ATP-binding protein\n",
      "SSO SSO:000004024 Iron-sulfur cluster assembly ATPase protein SufC\n",
      "b1682 ['Iron-sulfur cluster assembly ATPase protein SufC']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K07033 uncharacterized protein\n",
      "KO KO:K09014 Fe-S cluster assembly protein SufB\n",
      "SSO SSO:000004025 Iron-sulfur cluster assembly protein SufB\n",
      "b1683 ['Iron-sulfur cluster assembly protein SufB']\n",
      "\n",
      "KO KO:K05997 Fe-S cluster assembly protein SufA\n",
      "SSO SSO:000018626 Iron-sulfur cluster assembly iron binding protein SufA\n",
      "b1684 ['Iron-sulfur cluster assembly iron binding protein SufA']\n",
      "\n",
      "SSO SSO:000028225 Uncharacterized protein YdiH\n",
      "b1685 ['Uncharacterized protein YdiH']\n",
      "\n",
      "KO KO:K19222 1,4-dihydroxy-2-naphthoyl-CoA hydrolase [EC:3.1.2.28]\n",
      "SSO SSO:000000036 1,4-dihydroxy-2-naphthoyl-CoA hydrolase in menaquinone biosynthesis (EC 3.1.2.28)\n",
      "b1686 ['1,4-dihydroxy-2-naphthoyl-CoA hydrolase (EC 3.1.2.28) in menaquinone biosynthesis']\n",
      "\n",
      "SSO SSO:000028226 Uncharacterized protein YdiJ\n",
      "b1687 ['Uncharacterized protein YdiJ']\n",
      "\n",
      "SSO SSO:000026510 UPF0118 inner membrane protein YdiK\n",
      "b1688 ['UPF0118 inner membrane protein YdiK']\n",
      "\n",
      "SSO SSO:000028227 Uncharacterized protein YdiL\n",
      "b1689 ['Uncharacterized protein YdiL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018442 Inner membrane transport protein YdiM\n",
      "b1690 ['Inner membrane transport protein YdiM']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018443 Inner membrane transport protein YdiN\n",
      "b1691 ['Inner membrane transport protein YdiN']\n",
      "\n",
      "KO KO:K00014 shikimate dehydrogenase [EC:1.1.1.25]\n",
      "KO KO:K05887 quinate/shikimate dehydrogenase [EC:1.1.1.282]\n",
      "SSO SSO:000007465 Shikimate/quinate 5-dehydrogenase I beta (EC 1.1.1.282)\n",
      "b1692 ['Shikimate/quinate 5-dehydrogenase I beta (EC 1.1.1.282)']\n",
      "\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K03785 3-dehydroquinate dehydratase I [EC:4.2.1.10]\n",
      "SSO SSO:000000274 3-dehydroquinate dehydratase I (EC 4.2.1.10)\n",
      "SSO SSO:000009609 3-dehydroquinate dehydratase\n",
      "b1693 ['3-dehydroquinate dehydratase I (EC 4.2.1.10)']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01026 propionate CoA-transferase [EC:2.8.3.1]\n",
      "KO KO:K19709 acetate CoA-transferase [EC:2.8.3.8]\n",
      "b1694 ['Acyl-CoA:acetate CoA-transferase, alpha subunit (EC 2.8.3.8)', 'Acyl-CoA:acetate CoA-transferase, beta subunit (EC 2.8.3.8)', 'Acyl-CoA:acetate CoA transferase (EC 2.8.3.8)']\n",
      "\n",
      "KO KO:K08297 crotonobetainyl-CoA dehydrogenase [EC:1.3.8.13]\n",
      "SSO SSO:000010253 Acyl-CoA dehydrogenase YdiO\n",
      "SSO SSO:000023140 Putative acyl-CoA dehydrogenase\n",
      "b1695 ['Acyl-CoA dehydrogenase YdiO']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000028752 Uncharacterized transcriptional regulator YdiP, AraC family\n",
      "b1696 ['Uncharacterized transcriptional regulator YdiP, AraC family']\n",
      "\n",
      "KO KO:K03521 electron transfer flavoprotein beta subunit\n",
      "SSO SSO:000002435 Electron transfer flavoprotein, beta subunit YdiQ\n",
      "b1697 ['Electron transfer flavoprotein, beta subunit YdiQ']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K03522 electron transfer flavoprotein alpha subunit\n",
      "SSO SSO:000002429 Electron transfer flavoprotein, alpha subunit YdiR\n",
      "b1698 ['Electron transfer flavoprotein, alpha subunit YdiR']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K00313 electron transfer flavoprotein-quinone oxidoreductase [EC:1.5.5.-]\n",
      "SSO SSO:000002444 Electron transfer flavoprotein-quinone oxidoreductase YdiS\n",
      "b1699 ['Electron transfer flavoprotein-quinone oxidoreductase YdiS']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K03855 ferredoxin like protein\n",
      "SSO SSO:000002713 Ferredoxin-like protein ydiT\n",
      "b1700 ['Ferredoxin-like protein YdiT']\n",
      "\n",
      "KO KO:K12507 acyl-CoA synthetase [EC:6.2.1.-]\n",
      "b1701 ['Short-chain-fatty-acid--CoA ligase']\n",
      "\n",
      "GO GO:0006094 gluconeogenesis\n",
      "KO KO:K01007 pyruvate, water dikinase [EC:2.7.9.2]\n",
      "SSO SSO:000005958 Phosphoenolpyruvate synthase (EC 2.7.9.2)\n",
      "b1702 ['Phosphoenolpyruvate synthase (EC 2.7.9.2)']\n",
      "\n",
      "KO KO:K09773 [pyruvate, water dikinase]-phosphate phosphotransferase / [pyruvate, water dikinase] kinase [EC:2.7.4.28 2.7.11.33]\n",
      "SSO SSO:000005959 Phosphoenolpyruvate synthase regulatory protein\n",
      "b1703 ['Phosphoenolpyruvate synthase regulatory protein']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "KO KO:K01626 3-deoxy-7-phosphoheptulonate synthase [EC:2.5.1.54]\n",
      "SSO SSO:000000169 2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I alpha (EC 2.5.1.54)\n",
      "b1704 ['2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I alpha (EC 2.5.1.54)']\n",
      "\n",
      "SSO SSO:000036257 Hemin uptake protein HemP/HmuP\n",
      "b1705 ['Hemin uptake protein HemP/HmuP']\n",
      "\n",
      "KO KO:K08997 uncharacterized protein\n",
      "SSO SSO:000026495 UPF0061 protein YdiU\n",
      "b1706 ['UPF0061 protein YdiU']\n",
      "\n",
      "SSO SSO:000023185 Putative anti-FlhC(2)FlhD(4) factor YdiV\n",
      "b1707 ['Putative anti-FlhC(2)FlhD(4) factor YdiV']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K13695 probable lipoprotein NlpC\n",
      "SSO SSO:000022469 Probable endopeptidase NlpC\n",
      "b1708 ['Probable endopeptidase NlpC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06074 vitamin B12 transport system ATP-binding protein [EC:3.6.3.33]\n",
      "SSO SSO:000041668 Vitamin B12 ABC transporter (EC 7.6.2.8), ATP-binding protein BtuD\n",
      "b1709 ['Vitamin B12 ABC transporter, ATP-binding protein BtuD']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K00432 glutathione peroxidase [EC:1.11.1.9]\n",
      "SSO SSO:000003224 Glutathione peroxidase (EC 1.11.1.9)\n",
      "b1710 ['Glutathione peroxidase (EC 1.11.1.9)', 'Thioredoxin peroxidase (EC 1.11.1.15)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02015 iron complex transport system permease protein\n",
      "KO KO:K06073 vitamin B12 transport system permease protein\n",
      "SSO SSO:000041669 Vitamin B12 ABC transporter (EC 7.6.2.8), permease protein BtuC\n",
      "b1711 ['Vitamin B12 ABC transporter, permease protein BtuC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K04764 integration host factor subunit alpha\n",
      "SSO SSO:000003942 Integration host factor alpha subunit\n",
      "b1712 ['Integration host factor alpha subunit']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01890 phenylalanyl-tRNA synthetase beta chain [EC:6.1.1.20]\n",
      "SSO SSO:000005902 Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)\n",
      "b1713 ['Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01889 phenylalanyl-tRNA synthetase alpha chain [EC:6.1.1.20]\n",
      "SSO SSO:000005899 Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)\n",
      "b1714 ['Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "SSO SSO:000021587 Phenylalanyl--tRNA ligase operon leader peptide\n",
      "b1715 ['Phenylalanyl--tRNA ligase operon leader peptide']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02887 large subunit ribosomal protein L20\n",
      "SSO SSO:000004240 LSU ribosomal protein L20p\n",
      "b1716 ['LSU ribosomal protein L20p']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02916 large subunit ribosomal protein L35\n",
      "SSO SSO:000004305 LSU ribosomal protein L35p\n",
      "b1717 ['LSU ribosomal protein L35p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02520 translation initiation factor IF-3\n",
      "SSO SSO:000008313 Translation initiation factor 3\n",
      "SSO SSO:000033641 translation initiation factor IF-3\n",
      "b1718 ['Translation initiation factor 3']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01868 threonyl-tRNA synthetase [EC:6.1.1.3]\n",
      "SSO SSO:000008062 Threonyl-tRNA synthetase (EC 6.1.1.3)\n",
      "b1719 ['Threonyl-tRNA synthetase (EC 6.1.1.3)']\n",
      "\n",
      "SSO SSO:000023182 Putative ankyrin repeat protein B\n",
      "b4494 ['Putative ankyrin repeat protein B']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4535 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07283 putative salt-induced outer membrane protein\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "b1722 ['Putative outer membrane protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K16370 6-phosphofructokinase 2 [EC:2.7.1.11]\n",
      "SSO SSO:000000454 6-phosphofructokinase class II (EC 2.7.1.11)\n",
      "b1723 ['6-phosphofructokinase class II (EC 2.7.1.11)']\n",
      "\n",
      "SSO SSO:000028228 Uncharacterized protein YdiZ\n",
      "b1724 ['Uncharacterized protein YdiZ']\n",
      "\n",
      "KO KO:K15523 protein-ribulosamine 3-kinase [EC:2.7.1.172]\n",
      "SSO SSO:000007157 Ribulosamine/erythrulosamine 3-kinase potentially involved in protein deglycation\n",
      "b1725 ['Ribulosamine/erythrulosamine 3-kinase potentially involved in protein deglycation']\n",
      "\n",
      "SSO SSO:000023417 Putative membrane protein YniB\n",
      "b1726 ['Putative membrane protein YniB']\n",
      "\n",
      "KO KO:K19270 sugar-phosphatase [EC:3.1.3.23]\n",
      "b1727 ['2-deoxyglucose-6-phosphate hydrolase (EC 3.1.3.68)', 'Mannitol-1-phosphatase (EC 3.1.3.22)', 'Sorbitol-6-phosphatase (EC 3.1.3.50)']\n",
      "\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K07038 inner membrane protein\n",
      "SSO SSO:000004785 Membrane-bound metal-dependent hydrolase YdjM, induced during SOS response\n",
      "b1728 ['Membrane-bound metal-dependent hydrolase YdjM, induced during SOS response']\n",
      "\n",
      "KO KO:K06956 uncharacterized protein\n",
      "SSO SSO:000018755 L-cystine uptake protein TcyP, sodium:anion symporter family\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b1729 ['L-cystine uptake protein TcyP, sodium:anion symporter family']\n",
      "\n",
      "SSO SSO:000028231 Uncharacterized protein YdjO\n",
      "b1730 ['Uncharacterized protein YdjO']\n",
      "\n",
      "KO KO:K15722 cell division activator\n",
      "SSO SSO:000012025 Cell division activator CedA\n",
      "b1731 ['Cell division activator CedA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K03781 catalase [EC:1.11.1.6]\n",
      "SSO SSO:000035014 Catalase KatE-intracellular protease (EC 1.11.1.6)\n",
      "b1732 ['Catalase KatE-intracellular protease (EC 1.11.1.6)']\n",
      "\n",
      "KO KO:K03478 chitin disaccharide deacetylase [EC:3.5.1.105]\n",
      "SSO SSO:000005071 N,N'-diacetylchitobiose utilization operon protein YdjC\n",
      "SSO SSO:000012176 Chitooligosaccharide deacetylase\n",
      "b1733 [\"N,N'-diacetylchitobiose utilization operon protein YdjC\"]\n",
      "\n",
      "EC EC:3.2.1.122 Maltose-6'-phosphate glucosidase.\n",
      "EC EC:3.2.1.139 Alpha-glucuronidase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.22 Alpha-galactosidase.\n",
      "EC EC:3.2.1.67 Galacturan 1,4-alpha-galacturonidase.\n",
      "EC EC:3.2.1.86 6-phospho-beta-glucosidase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01222 6-phospho-beta-glucosidase [EC:3.2.1.86]\n",
      "SSO SSO:000005072 N,N'-diacetylchitobiose-specific 6-phospho-beta-glucosidase (EC 3.2.1.86)\n",
      "b1734 [\"N,N'-diacetylchitobiose-specific 6-phospho-beta-glucosidase (EC 3.2.1.86)\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03490 AraC family transcriptional regulator, dual regulator of chb operon\n",
      "SSO SSO:000005073 N,N'-diacetylchitobiose-specific regulator ChbR, AraC family\n",
      "b1735 [\"N,N'-diacetylchitobiose-specific regulator ChbR, AraC family\"]\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02759 PTS system, cellobiose-specific IIA component [EC:2.7.1.196 2.7.1.205]\n",
      "SSO SSO:000005665 PTS system, N,N'-diacetylchitobiose-specific IIA component (EC 2.7.1.69)\n",
      "b1736 [\"PTS system, N,N'-diacetylchitobiose-specific IIA component (EC 2.7.1.196)\"]\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02761 PTS system, cellobiose-specific IIC component\n",
      "SSO SSO:000005667 PTS system, N,N'-diacetylchitobiose-specific IIC component\n",
      "b1737 [\"PTS system, N,N'-diacetylchitobiose-specific IIC component\"]\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02760 PTS system, cellobiose-specific IIB component [EC:2.7.1.196 2.7.1.205]\n",
      "SSO SSO:000005666 PTS system, N,N'-diacetylchitobiose-specific IIB component (EC 2.7.1.69)\n",
      "b1738 [\"PTS system, N,N'-diacetylchitobiose-specific IIB component (EC 2.7.1.196)\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K04064 osmotically inducible lipoprotein OsmE\n",
      "SSO SSO:000020857 Osmotically-inducible lipoprotein E\n",
      "b1739 ['Osmotically-inducible lipoprotein E']\n",
      "\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "KO KO:K01916 NAD+ synthase [EC:6.3.1.5]\n",
      "SSO SSO:000005161 NAD synthetase (EC 6.3.1.5)\n",
      "b1740 ['NAD synthetase (EC 6.3.1.5)']\n",
      "\n",
      "KO KO:K05984 excinuclease Cho [EC:3.1.25.-]\n",
      "SSO SSO:000002616 Excinuclease cho (excinuclease ABC alternative C subunit)\n",
      "b1741 ['Excinuclease cho (excinuclease ABC alternative C subunit)']\n",
      "\n",
      "KO KO:K09975 uncharacterized protein\n",
      "SSO SSO:000007760 Stresses-induced protein Ves (HutD)\n",
      "b1742 ['Stresses-induced protein Ves (HutD)']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K06006 periplasmic protein CpxP/Spy\n",
      "SSO SSO:000024989 Spheroplast protein Y\n",
      "b1743 ['Spheroplast protein Y']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K05526 succinylglutamate desuccinylase [EC:3.5.1.96]\n",
      "SSO SSO:000025208 Succinylglutamate desuccinylase (EC 3.5.1.96)\n",
      "b1744 ['Succinylglutamate desuccinylase (EC 3.5.1.96)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01484 succinylarginine dihydrolase [EC:3.5.3.23]\n",
      "SSO SSO:000025207 Succinylarginine dihydrolase (EC 3.5.3.23)\n",
      "b1745 ['Succinylarginine dihydrolase (EC 3.5.3.23)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K06447 succinylglutamic semialdehyde dehydrogenase [EC:1.2.1.71]\n",
      "SSO SSO:000025210 Succinylglutamic semialdehyde dehydrogenase (EC 1.2.1.71)\n",
      "b1746 ['Succinylglutamic semialdehyde dehydrogenase (EC 1.2.1.71)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00673 arginine N-succinyltransferase [EC:2.3.1.109]\n",
      "SSO SSO:000001043 Arginine N-succinyltransferase (EC 2.3.1.109)\n",
      "b1747 ['Arginine N-succinyltransferase (EC 2.3.1.109)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00821 acetylornithine/N-succinyldiaminopimelate aminotransferase [EC:2.6.1.11 2.6.1.17]\n",
      "KO KO:K00840 succinylornithine aminotransferase [EC:2.6.1.81]\n",
      "SSO SSO:000010168 Succinylornithine transaminase (EC 2.6.1.81)\n",
      "b1748 ['Succinylornithine transaminase (EC 2.6.1.81)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0009314 response to radiation\n",
      "KO KO:K01142 exodeoxyribonuclease III [EC:3.1.11.2]\n",
      "SSO SSO:000002619 Exodeoxyribonuclease III (EC 3.1.11.2)\n",
      "SSO SSO:000013531 Exonuclease III\n",
      "b1749 ['Exodeoxyribonuclease III (EC 3.1.11.2)']\n",
      "\n",
      "SSO SSO:000002227 DedA family inner membrane protein YdjX\n",
      "b1750 ['DedA family inner membrane protein YdjX']\n",
      "\n",
      "SSO SSO:000003599 Hypothetical protein YdjY\n",
      "b1751 ['Hypothetical protein YdjY']\n",
      "\n",
      "SSO SSO:000013029 DedA family inner membrane protein YdjZ\n",
      "b1752 ['DedA family inner membrane protein YdjZ']\n",
      "\n",
      "SSO SSO:000003602 Hypothetical protein YnjA\n",
      "b1753 ['Hypothetical protein YnjA']\n",
      "\n",
      "KO KO:K05777 putative thiamine transport system substrate-binding protein\n",
      "SSO SSO:000000504 ABC transporter, substrate-binding protein YnjB\n",
      "b1754 ['ABC transporter, substrate-binding protein YnjB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05778 putative thiamine transport system permease protein\n",
      "SSO SSO:000000501 ABC transporter, permease protein YnjC\n",
      "b1755 ['ABC transporter, permease protein YnjC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K05779 putative thiamine transport system ATP-binding protein\n",
      "SSO SSO:000000496 ABC transporter, ATP-binding protein YnjD\n",
      "b1756 ['ABC transporter, ATP-binding protein YnjD']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K21028 molybdopterin synthase sulfurtransferase [EC:2.8.1.11]\n",
      "SSO SSO:000008051 Thiosulfate sulfurtransferase, rhodanese (EC 2.8.1.1)\n",
      "b1757 ['Thiosulfate sulfurtransferase, rhodanese (EC 2.8.1.1)']\n",
      "\n",
      "SSO SSO:000011529 CDP-alcohol phosphatidyltransferase family protein\n",
      "b1758 ['CDP-alcohol phosphatidyltransferase family protein']\n",
      "\n",
      "KO KO:K03574 8-oxo-dGTP diphosphatase [EC:3.6.1.55]\n",
      "KO KO:K08320 (d)CTP diphosphatase [EC:3.6.1.65]\n",
      "SSO SSO:000000427 5-methyl-dCTP pyrophosphohydrolase (EC 3.6.1.-)\n",
      "b1759 ['5-methyl-dCTP pyrophosphohydrolase (EC 3.6.1.-)']\n",
      "\n",
      "SSO SSO:000028472 Uncharacterized protein YnjH\n",
      "b1760 ['Uncharacterized protein YnjH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "KO KO:K00262 glutamate dehydrogenase (NADP+) [EC:1.4.1.4]\n",
      "SSO SSO:000005240 NADP-specific glutamate dehydrogenase (EC 1.4.1.4)\n",
      "b1761 ['NADP-specific glutamate dehydrogenase (EC 1.4.1.4)']\n",
      "\n",
      "SSO SSO:000018422 Inner membrane protein YnjI\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b1762 ['Inner membrane protein YnjI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03169 DNA topoisomerase III [EC:5.99.1.2]\n",
      "SSO SSO:000002117 DNA topoisomerase III (EC 5.99.1.2)\n",
      "b1763 ['DNA topoisomerase III (EC 5.99.1.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01008 selenide, water dikinase [EC:2.7.9.3]\n",
      "SSO SSO:000007382 Selenide,water dikinase (EC 2.7.9.3)\n",
      "SSO SSO:000039125 Selenophosphate synthase\n",
      "b1764 ['Selenide,water dikinase (EC 2.7.9.3)']\n",
      "\n",
      "SSO SSO:000023079 Putative NAD(P)H nitroreductase YdjA\n",
      "b1765 ['Putative NAD(P)H nitroreductase YdjA']\n",
      "\n",
      "KO KO:K04773 protease IV [EC:3.4.21.-]\n",
      "SSO SSO:000007521 Signal peptide peptidase SppA (protease 4)\n",
      "SSO SSO:000022672 Protease IV (Signal peptide peptidase) (EC 3.4.21.-)\n",
      "b1766 ['Signal peptide peptidase SppA (protease 4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01424 L-asparaginase [EC:3.5.1.1]\n",
      "SSO SSO:000004126 L-asparaginase I, cytoplasmic (EC 3.5.1.1)\n",
      "b1767 ['L-asparaginase I, cytoplasmic (EC 3.5.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "KO KO:K08281 nicotinamidase/pyrazinamidase [EC:3.5.1.19 3.5.1.-]\n",
      "SSO SSO:000005298 Nicotinamidase (EC 3.5.1.19)\n",
      "b1768 ['Nicotinamidase (EC 3.5.1.19)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08369 MFS transporter, putative metabolite:H+ symporter\n",
      "SSO SSO:000026928 Uncharacterized MFS-type transporter YdjE\n",
      "b1769 ['Uncharacterized MFS-type transporter YdjE']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000028753 Uncharacterized transcriptional regulator YdjF, DeoR family\n",
      "b1770 ['Uncharacterized transcriptional regulator YdjF, DeoR family']\n",
      "\n",
      "KO KO:K18471 methylglyoxal reductase [EC:1.1.1.-]\n",
      "SSO SSO:000027386 Uncharacterized oxidoreductase YdjG\n",
      "SSO SSO:000043542 methylglyoxal reductase (NADH-dependent) (EC 1.1.1.78)\n",
      "b1771 ['Uncharacterized oxidoreductase YdjG']\n",
      "\n",
      "SSO SSO:000023380 Putative kinase\n",
      "SSO SSO:000028700 Uncharacterized sugar kinase YdjH\n",
      "b1772 ['Uncharacterized sugar kinase YdjH']\n",
      "\n",
      "KO KO:K01624 fructose-bisphosphate aldolase, class II [EC:4.1.2.13]\n",
      "SSO SSO:000023146 Putative aldolase YdjI\n",
      "b1773 ['Putative aldolase YdjI']\n",
      "\n",
      "KO KO:K00008 L-iditol 2-dehydrogenase [EC:1.1.1.14]\n",
      "SSO SSO:000003683 Hypothetical zinc-type alcohol dehydrogenase-like protein YdjJ\n",
      "b1774 ['Hypothetical zinc-type alcohol dehydrogenase-like protein YdjJ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08369 MFS transporter, putative metabolite:H+ symporter\n",
      "SSO SSO:000026929 Uncharacterized MFS-type transporter YdjK\n",
      "b1775 ['Uncharacterized MFS-type transporter YdjK']\n",
      "\n",
      "SSO SSO:000023485 Putative oxidoreductase YdjL\n",
      "b1776 ['Putative oxidoreductase YdjL']\n",
      "\n",
      "KO KO:K09916 uncharacterized protein\n",
      "SSO SSO:000028233 Uncharacterized protein YeaC\n",
      "b1777 ['Uncharacterized protein YeaC']\n",
      "\n",
      "KO KO:K07305 peptide-methionine (R)-S-oxide reductase [EC:1.8.4.12]\n",
      "SSO SSO:000033963 Peptide-methionine (R)-S-oxide reductase MsrB (EC 1.8.4.12)\n",
      "b1778 ['Peptide-methionine (R)-S-oxide reductase MsrB (EC 1.8.4.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "KO KO:K00134 glyceraldehyde 3-phosphate dehydrogenase [EC:1.2.1.12]\n",
      "SSO SSO:000005187 NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)\n",
      "b1779 ['NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)']\n",
      "\n",
      "KO KO:K01792 glucose-6-phosphate 1-epimerase [EC:5.1.3.15]\n",
      "SSO SSO:000010389 Aldose 1-epimerase family protein YeaD\n",
      "b1780 ['Aldose 1-epimerase family protein YeaD']\n",
      "\n",
      "SSO SSO:000006689 Putative oxidoreductase YeaE, aldo/keto reductase family\n",
      "b1781 ['Putative oxidoreductase YeaE, aldo/keto reductase family']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07274 MipA family protein\n",
      "SSO SSO:000019932 MltA-interacting protein MipA\n",
      "b1782 ['MltA-interacting protein MipA']\n",
      "\n",
      "KO KO:K07180 serine protein kinase\n",
      "SSO SSO:000028234 Uncharacterized protein YeaG\n",
      "b1783 ['Uncharacterized protein YeaG']\n",
      "\n",
      "KO KO:K09786 uncharacterized protein\n",
      "SSO SSO:000026587 UPF0229 protein YeaH\n",
      "b1784 ['UPF0229 protein YeaH']\n",
      "\n",
      "b1785 ['Diguanylate cyclase (EC 2.7.7.65) => DgcX']\n",
      "\n",
      "SSO SSO:000032478 putative diguanylate cyclase\n",
      "b1786 ['Diguanylate cyclase (EC 2.7.7.65) => DgcJ']\n",
      "\n",
      "KO KO:K19055 Ala-tRNA(Pro) deacylase [EC:3.1.1.-]\n",
      "SSO SSO:000028235 Uncharacterized protein YeaK\n",
      "b1787 ['Uncharacterized protein YeaK']\n",
      "\n",
      "SSO SSO:000028475 Uncharacterized protein YoaI\n",
      "b1788 ['Uncharacterized protein YoaI']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000026709 UPF0756 membrane protein YeaL\n",
      "b1789 ['UPF0756 membrane protein YeaL']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "b1790 ['Uncharacterized transcriptional regulator YeaM, AraC family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042886 amide transport\n",
      "KO KO:K03449 MFS transporter, CP family, cyanate transporter\n",
      "SSO SSO:000033998 2-nitroimidazole transporter NimT\n",
      "b1791 ['2-nitroimidazole transporter NimT']\n",
      "\n",
      "b1792 ['FIG016027: protein of unknown function YeaO']\n",
      "\n",
      "KO KO:K09712 uncharacterized protein\n",
      "SSO SSO:000028474 Uncharacterized protein YoaF\n",
      "b1793 ['Uncharacterized protein YoaF']\n",
      "\n",
      "KO KO:K13069 diguanylate cyclase [EC:2.7.7.65]\n",
      "SSO SSO:000013106 Diguanylate cyclase\n",
      "b1794 ['Diguanylate cyclase (EC 2.7.7.65) => DgcP']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4676 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4675 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000026676 UPF0410 protein YeaQ\n",
      "b1795 ['UPF0410 protein YeaQ']\n",
      "\n",
      "SSO SSO:000022739 Protein YoaG\n",
      "b1796 ['Protein YoaG']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "SSO SSO:000028236 Uncharacterized protein YeaR\n",
      "b1797 ['Uncharacterized protein YeaR']\n",
      "\n",
      "KO KO:K11250 leucine efflux protein\n",
      "SSO SSO:000018896 Leucine efflux protein\n",
      "b1798 ['Leucine efflux protein']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K16135 LysR family transcriptional regulator, transcriptional activator for dmlA\n",
      "SSO SSO:000039577 Transcriptional regulator DmlR, LysR family\n",
      "b1799 ['Transcriptional regulator DmlR, LysR family']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K07246 tartrate dehydrogenase/decarboxylase / D-malate dehydrogenase [EC:1.1.1.93 4.1.1.73 1.1.1.83]\n",
      "SSO SSO:000002033 D-malate dehydrogenase [decarboxylating] (EC 1.1.1.83)\n",
      "b1800 ['D-malate dehydrogenase [decarboxylating] (EC 1.1.1.83)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000034797 Betaine/carnitine/choline transporter (BCCT) family\n",
      "b1801 ['Betaine/carnitine/choline transporter (BCCT) family']\n",
      "\n",
      "KO KO:K22443 carnitine monooxygenase subunit [EC:1.14.13.239]\n",
      "SSO SSO:000035003 Carnitine monooxygenase, oxygenase component CntA (EC 1.14.13.239)\n",
      "b1802 ['Carnitine monooxygenase, oxygenase component CntA']\n",
      "\n",
      "KO KO:K22444 carnitine monooxygenase subunit [EC:1.14.13.239]\n",
      "SSO SSO:000035004 Carnitine monooxygenase, reductase component CntB (EC 1.14.13.239)\n",
      "b1803 ['Carnitine monooxygenase, reductase component CntB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006401 RNA catabolic process\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03684 ribonuclease D [EC:3.1.13.5]\n",
      "SSO SSO:000007071 Ribonuclease D (EC 3.1.26.3)\n",
      "b1804 ['Ribonuclease D (EC 3.1.26.3)']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01897 long-chain acyl-CoA synthetase [EC:6.2.1.3]\n",
      "SSO SSO:000004543 Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)\n",
      "b1805 ['Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07285 outer membrane lipoprotein\n",
      "SSO SSO:000027205 Uncharacterized lipoprotein YeaY\n",
      "b1806 ['Uncharacterized lipoprotein YeaY']\n",
      "\n",
      "KO KO:K14742 tRNA threonylcarbamoyladenosine biosynthesis protein TsaB\n",
      "SSO SSO:000042087 tRNA threonylcarbamoyladenosine biosynthesis protein TsaB\n",
      "b1807 ['tRNA threonylcarbamoyladenosine biosynthesis protein TsaB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03722 ATP-dependent DNA helicase DinG [EC:3.6.4.12]\n",
      "SSO SSO:000002341 DinG family ATP-dependent helicase YoaA\n",
      "b1808 ['DinG family ATP-dependent helicase YoaA']\n",
      "\n",
      "KO KO:K09022 2-iminobutanoate/2-iminopropanoate deaminase [EC:3.5.99.10]\n",
      "SSO SSO:000007170 RidA/YER057c/UK114 superfamily, group 2, YoaB-like protein\n",
      "b1809 ['RidA/YER057c/UK114 superfamily, group 2, YoaB-like protein']\n",
      "\n",
      "SSO SSO:000028473 Uncharacterized protein YoaC\n",
      "b1810 ['Uncharacterized protein YoaC']\n",
      "\n",
      "KO KO:K09917 uncharacterized protein\n",
      "SSO SSO:000026567 UPF0181 protein YoaH\n",
      "b1811 ['UPF0181 protein YoaH']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K01665 para-aminobenzoate synthetase component I [EC:2.6.1.85]\n",
      "SSO SSO:000005734 Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)\n",
      "b1812 ['Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)']\n",
      "\n",
      "KO KO:K17879 peroxisomal coenzyme A diphosphatase NUDT7 [EC:3.6.1.-]\n",
      "SSO SSO:000008699 Uncharacterized Nudix hydrolase NudL\n",
      "b1813 ['Uncharacterized Nudix hydrolase NudL']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01752 L-serine dehydratase [EC:4.3.1.17]\n",
      "SSO SSO:000042530 L-serine dehydratase 1 (EC 4.3.1.17)\n",
      "b1814 ['L-serine dehydratase, beta subunit (EC 4.3.1.17)', 'L-serine dehydratase, alpha subunit (EC 4.3.1.17)']\n",
      "\n",
      "KO KO:K21090 c-di-GMP phosphodiesterase [EC:3.1.4.52]\n",
      "b1815 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000026482 UPF0053 inner membrane protein YoaE\n",
      "b1816 ['UPF0053 inner membrane protein YoaE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02794 PTS system, mannose-specific IIB component [EC:2.7.1.191]\n",
      "b1817 ['PTS system, mannose-specific IIA component (EC 2.7.1.191)', 'PTS system, mannose-specific IIB component (EC 2.7.1.191)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02795 PTS system, mannose-specific IIC component\n",
      "SSO SSO:000005711 PTS system, mannose-specific IIC component (EC 2.7.1.69)\n",
      "b1818 ['PTS system, mannose-specific IIC component']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02796 PTS system, mannose-specific IID component\n",
      "SSO SSO:000005712 PTS system, mannose-specific IID component (EC 2.7.1.69)\n",
      "b1819 ['PTS system, mannose-specific IID component']\n",
      "\n",
      "SSO SSO:000026606 UPF0266 membrane protein YobD\n",
      "b1820 ['UPF0266 membrane protein YobD']\n",
      "\n",
      "KO KO:K23242 Unknown\n",
      "SSO SSO:000023404 Putative manganese efflux pump MntP\n",
      "b1821 ['Putative manganese efflux pump MntP']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00563 23S rRNA (guanine745-N1)-methyltransferase [EC:2.1.1.187]\n",
      "SSO SSO:000000236 23S rRNA (guanine(745)-N(1))-methyltransferase (EC 2.1.1.187)\n",
      "b1822 ['23S rRNA (guanine(745)-N(1))-methyltransferase (EC 2.1.1.187)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03704 cold shock protein (beta-ribbon, CspA family)\n",
      "b1823 ['Cold shock protein of CSP family => CspC (naming convention as in E.coli)']\n",
      "\n",
      "SSO SSO:000028477 Uncharacterized protein YobF\n",
      "b1824 ['Uncharacterized protein YobF']\n",
      "\n",
      "SSO SSO:000028239 Uncharacterized protein YebO\n",
      "b1825 ['Uncharacterized protein YebO']\n",
      "\n",
      "SSO SSO:000021594 PhoP/PhoQ regulator MgrB\n",
      "b1826 ['PhoP/PhoQ regulator MgrB']\n",
      "\n",
      "SSO SSO:000028478 Uncharacterized protein YobH\n",
      "b4536 ['Uncharacterized protein YobH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K13641 IclR family transcriptional regulator, acetate operon repressor\n",
      "KO KO:K19333 IclR family transcriptional regulator, KDG regulon repressor\n",
      "SSO SSO:000008201 Transcriptional regulator KdgR, KDG operon repressor\n",
      "b1827 ['Transcriptional regulator KdgR, KDG operon repressor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08169 MFS transporter, DHA2 family, multidrug resistance protein\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000028791 Uncharacterized transporter YebQ, major facilitator superfamily (MFS)\n",
      "b1828 ['Uncharacterized transporter YebQ, major facilitator superfamily (MFS)']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03799 heat shock protein HtpX [EC:3.4.24.-]\n",
      "SSO SSO:000022669 Protease HtpX\n",
      "SSO SSO:000032499 putative endopeptidase\n",
      "b1829 ['Protease HtpX']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03797 carboxyl-terminal processing protease [EC:3.4.21.102]\n",
      "b1830 ['Tail-specific protease precursor (EC 3.4.21.102)']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K03607 ProP effector\n",
      "SSO SSO:000038896 RNA chaperone ProQ\n",
      "b1831 ['RNA chaperone ProQ']\n",
      "\n",
      "KO KO:K08968 L-methionine (R)-S-oxide reductase [EC:1.8.4.14]\n",
      "SSO SSO:000002926 Free methionine-(R)-sulfoxide reductase, contains GAF domain\n",
      "b1832 ['Free methionine-(R)-sulfoxide reductase, contains GAF domain']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03808 paraquat-inducible protein A\n",
      "b1833 ['Inner membrane protein YebS']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K06192 paraquat-inducible protein B\n",
      "b1834 ['Uncharacterized protein YebT']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K11392 16S rRNA (cytosine1407-C5)-methyltransferase [EC:2.1.1.178]\n",
      "SSO SSO:000000056 16S rRNA (cytosine(1407)-C(5))-methyltransferase (EC 2.1.1.178)\n",
      "b1835 ['16S rRNA (cytosine(1407)-C(5))-methyltransferase (EC 2.1.1.178)']\n",
      "\n",
      "SSO SSO:000023244 Putative cytoplasmic protein YebV\n",
      "b1836 ['Putative cytoplasmic protein YebV']\n",
      "\n",
      "SSO SSO:000041447 Uncharacterized protein YebW\n",
      "b1837 ['Uncharacterized protein YebW']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K07313 serine/threonine protein phosphatase 1 [EC:3.1.3.16]\n",
      "SSO SSO:000024593 Serine/threonine protein phosphatase 1 (EC 3.1.3.16)\n",
      "b1838 ['Serine/threonine protein phosphatase 1 (EC 3.1.3.16)']\n",
      "\n",
      "SSO SSO:000015419 FIG00923183: possible exported protein YebY\n",
      "b1839 ['FIG00923183: possible exported protein YebY']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K07245 copper resistance protein D\n",
      "SSO SSO:000001769 Copper resistance protein CopD\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b1840 ['Copper resistance protein CopD']\n",
      "\n",
      "KO KO:K07156 copper resistance protein C\n",
      "SSO SSO:000001768 Copper resistance protein CopC\n",
      "b1841 ['Copper resistance protein CopC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02345 DNA polymerase III subunit theta [EC:2.7.7.7]\n",
      "SSO SSO:000012813 DNA polymerase III theta subunit (EC 2.7.7.7)\n",
      "b1842 ['DNA polymerase III theta subunit (EC 2.7.7.7)']\n",
      "\n",
      "SSO SSO:000028476 Uncharacterized protein YobB\n",
      "b1843 ['Uncharacterized protein YobB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K02342 DNA polymerase III subunit epsilon [EC:2.7.7.7]\n",
      "KO KO:K10857 exodeoxyribonuclease X [EC:3.1.11.-]\n",
      "SSO SSO:000013528 Exodeoxyribonuclease X\n",
      "b1844 ['Exodeoxyribonuclease X']\n",
      "\n",
      "KO KO:K01354 oligopeptidase B [EC:3.4.21.83]\n",
      "SSO SSO:000006411 Protease II (EC 3.4.21.83)\n",
      "b1845 ['Protease II (EC 3.4.21.83)']\n",
      "\n",
      "SSO SSO:000013747 FIG004088: inner membrane protein YebE\n",
      "b1846 ['FIG004088: inner membrane protein YebE']\n",
      "\n",
      "SSO SSO:000024888 Small secreted protein YebF\n",
      "b1847 ['Small secreted protein YebF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K09918 uncharacterized protein\n",
      "SSO SSO:000029126 YebG, DNA damage-inducible gene in SOS regulon, expressed in stationary phase\n",
      "b1848 ['YebG, DNA damage-inducible gene in SOS regulon, expressed in stationary phase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K08289 phosphoribosylglycinamide formyltransferase 2 [EC:2.1.2.2]\n",
      "SSO SSO:000006047 Phosphoribosylglycinamide formyltransferase 2 (EC 2.1.2.-)\n",
      "SSO SSO:000035962 Formate-dependent phosphoribosylglycinamide formyltransferase\n",
      "b1849 ['Formate-dependent phosphoribosylglycinamide formyltransferase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009255 Entner-Doudoroff pathway through 6-phosphogluconate\n",
      "GO GO:0009436 glyoxylate catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01625 2-dehydro-3-deoxyphosphogluconate aldolase / (4S)-4-hydroxy-2-oxoglutarate aldolase [EC:4.1.2.14 4.1.3.42]\n",
      "b1850 ['4-hydroxy-2-oxoglutarate aldolase (EC 4.1.3.16)', '2-dehydro-3-deoxyphosphogluconate aldolase (EC 4.1.2.14)']\n",
      "\n",
      "GO GO:0009255 Entner-Doudoroff pathway through 6-phosphogluconate\n",
      "KO KO:K01690 phosphogluconate dehydratase [EC:4.2.1.12]\n",
      "SSO SSO:000005968 Phosphogluconate dehydratase (EC 4.2.1.12)\n",
      "b1851 ['Phosphogluconate dehydratase (EC 4.2.1.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009051 pentose-phosphate shunt, oxidative branch\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00036 glucose-6-phosphate 1-dehydrogenase [EC:1.1.1.49 1.1.1.363]\n",
      "SSO SSO:000003127 Glucose-6-phosphate 1-dehydrogenase (EC 1.1.1.49)\n",
      "b1852 ['Glucose-6-phosphate 1-dehydrogenase (EC 1.1.1.49)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K19337 RpiR family transcriptional regulator, carbohydrate utilization regulator\n",
      "SSO SSO:000017203 Phosphogluconate repressor HexR, RpiR family\n",
      "b1853 ['Phosphogluconate repressor HexR, RpiR family']\n",
      "\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00873 pyruvate kinase [EC:2.7.1.40]\n",
      "SSO SSO:000006852 Pyruvate kinase (EC 2.7.1.40)\n",
      "b1854 ['Pyruvate kinase (EC 2.7.1.40)']\n",
      "\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02517 Kdo2-lipid IVA lauroyltransferase [EC:2.3.1.241]\n",
      "KO KO:K02560 lauroyl-Kdo2-lipid IVA myristoyltransferase [EC:2.3.1.243]\n",
      "SSO SSO:000004454 Lipid A biosynthesis myristoyltransferase (EC 2.3.1.243)\n",
      "b1855 ['Lipid A biosynthesis myristoyltransferase (EC 2.3.1.243)']\n",
      "\n",
      "KO KO:K19304 murein DD-endopeptidase [EC:3.4.24.-]\n",
      "SSO SSO:000020125 Murein DD-endopeptidase MepM\n",
      "b1856 ['Murein DD-endopeptidase MepM']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K09815 zinc transport system substrate-binding protein\n",
      "SSO SSO:000029177 Zinc ABC transporter, substrate-binding protein ZnuA\n",
      "b1857 ['Zinc ABC transporter, substrate-binding protein ZnuA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K09817 zinc transport system ATP-binding protein [EC:3.6.3.-]\n",
      "SSO SSO:000029173 Zinc ABC transporter, ATP-binding protein ZnuC\n",
      "b1858 ['Zinc ABC transporter, ATP-binding protein ZnuC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K09816 zinc transport system permease protein\n",
      "SSO SSO:000029175 Zinc ABC transporter, permease protein ZnuB\n",
      "b1859 ['Zinc ABC transporter, permease protein ZnuB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03551 holliday junction DNA helicase RuvB [EC:3.6.4.12]\n",
      "SSO SSO:000036305 Holliday junction ATP-dependent DNA helicase RuvB (EC 3.6.4.12)\n",
      "b1860 ['Holliday junction ATP-dependent DNA helicase RuvB (EC 3.6.4.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03550 holliday junction DNA helicase RuvA [EC:3.6.4.12]\n",
      "SSO SSO:000017857 Holliday junction ATP-dependent DNA helicase ruvA\n",
      "b1861 ['Holliday junction ATP-dependent DNA helicase RuvA (EC 3.6.4.12)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4677 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028237 Uncharacterized protein YebB\n",
      "b1862 ['Uncharacterized protein YebB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K01159 crossover junction endodeoxyribonuclease RuvC [EC:3.1.22.4]\n",
      "SSO SSO:000001785 Crossover junction endodeoxyribonuclease RuvC (EC 3.1.22.4)\n",
      "b1863 ['Crossover junction endodeoxyribonuclease RuvC (EC 3.1.22.4)']\n",
      "\n",
      "KO KO:K18189 translational activator of cytochrome c oxidase 1\n",
      "SSO SSO:000022599 Probable transcriptional regulatory protein YebC\n",
      "b1864 ['Probable transcriptional regulatory protein YebC']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K08310 dihydroneopterin triphosphate diphosphatase [EC:3.6.1.67]\n",
      "SSO SSO:000002304 Dihydroneopterin triphosphate pyrophosphohydrolase type 2\n",
      "b1865 ['Dihydroneopterin triphosphate pyrophosphohydrolase type 2']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01876 aspartyl-tRNA synthetase [EC:6.1.1.12]\n",
      "SSO SSO:000001106 Aspartyl-tRNA synthetase (EC 6.1.1.12)\n",
      "b1866 ['Aspartyl-tRNA synthetase (EC 6.1.1.12)']\n",
      "\n",
      "SSO SSO:000018641 Isochorismatase family protein\n",
      "SSO SSO:000018642 Isochorismatase family protein YecD\n",
      "b1867 ['Isochorismatase family protein YecD']\n",
      "\n",
      "SSO SSO:000026716 UPF0759 protein YecE\n",
      "b1868 ['UPF0759 protein YecE']\n",
      "\n",
      "KO KO:K07136 uncharacterized protein\n",
      "SSO SSO:000018390 Inner membrane protein YecN\n",
      "b1869 ['Inner membrane protein YecN']\n",
      "\n",
      "KO KO:K15256 tRNA (cmo5U34)-methyltransferase [EC:2.1.1.-]\n",
      "SSO SSO:000034963 Carboxy-S-adenosyl-L-methionine synthase\n",
      "b1870 ['Carboxy-S-adenosyl-L-methionine synthase']\n",
      "\n",
      "KO KO:K15257 tRNA (mo5U34)-methyltransferase [EC:2.1.1.-]\n",
      "SSO SSO:000042082 tRNA ho5U(34) carboxymethyltransferase\n",
      "b1871 ['tRNA ho5U(34) carboxymethyltransferase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K07812 trimethylamine-N-oxide reductase (cytochrome c) [EC:1.7.2.3]\n",
      "SSO SSO:000008379 Trimethylamine-N-oxide reductase , TorZ (EC 1.7.2.3)\n",
      "b1872 ['Trimethylamine-N-oxide reductase (EC 1.7.2.3), TorZ']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K07821 trimethylamine-N-oxide reductase (cytochrome c), cytochrome c-type subunit TorY\n",
      "SSO SSO:000008381 Trimethylamine-N-oxide reductase associated c-type cytochrome, TorY\n",
      "b1873 ['Trimethylamine-N-oxide reductase associated c-type cytochrome, TorY']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K06201 copper homeostasis protein\n",
      "SSO SSO:000001962 Cytoplasmic copper homeostasis protein CutC\n",
      "b1874 ['Cytoplasmic copper homeostasis protein CutC']\n",
      "\n",
      "KO KO:K09907 uncharacterized protein\n",
      "SSO SSO:000022729 Protein YecM\n",
      "b1875 ['Protein YecM']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01887 arginyl-tRNA synthetase [EC:6.1.1.19]\n",
      "SSO SSO:000001053 Arginyl-tRNA synthetase (EC 6.1.1.19)\n",
      "b1876 ['Arginyl-tRNA synthetase (EC 6.1.1.19)']\n",
      "\n",
      "SSO SSO:000028244 Uncharacterized protein YecT\n",
      "b1877 ['Uncharacterized protein YecT']\n",
      "\n",
      "KO KO:K03516 flagellar protein FlhE\n",
      "SSO SSO:000002828 Flagellar protein FlhE\n",
      "b1878 ['Flagellar protein FlhE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02400 flagellar biosynthesis protein FlhA\n",
      "SSO SSO:000002800 Flagellar biosynthesis protein FlhA\n",
      "b1879 ['Flagellar biosynthesis protein FlhA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02401 flagellar biosynthetic protein FlhB\n",
      "SSO SSO:000002801 Flagellar biosynthesis protein FlhB\n",
      "b1880 ['Flagellar biosynthesis protein FlhB']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03414 chemotaxis protein CheZ\n",
      "SSO SSO:000012157 Chemotaxis response - phosphatase CheZ\n",
      "b1881 ['Chemotaxis response - phosphatase CheZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03413 two-component system, chemotaxis family, chemotaxis protein CheY\n",
      "SSO SSO:000012156 Chemotaxis regulator - transmits chemoreceptor signals to flagellar motor components CheY\n",
      "b1882 ['Chemotaxis regulator - transmits chemoreceptor signals to flagellar motor components CheY']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03412 two-component system, chemotaxis family, protein-glutamate methylesterase/glutaminase [EC:3.1.1.61 3.5.1.44]\n",
      "SSO SSO:000012161 Chemotaxis response regulator protein-glutamate methylesterase CheB (EC 3.1.1.61)\n",
      "b1883 ['Chemotaxis response regulator protein-glutamate methylesterase CheB (EC 3.1.1.61)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K00575 chemotaxis protein methyltransferase CheR [EC:2.1.1.80]\n",
      "SSO SSO:000001514 Chemotaxis protein methyltransferase CheR (EC 2.1.1.80)\n",
      "b1884 ['Chemotaxis protein methyltransferase CheR (EC 2.1.1.80)']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03406 methyl-accepting chemotaxis protein\n",
      "KO KO:K05877 methyl-accepting chemotaxis protein IV, peptide sensor receptor\n",
      "SSO SSO:000019575 Methyl-accepting chemotaxis protein IV (dipeptide chemoreceptor protein)\n",
      "b1885 ['Methyl-accepting chemotaxis protein IV (dipeptide chemoreceptor protein)']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03406 methyl-accepting chemotaxis protein\n",
      "KO KO:K05874 methyl-accepting chemotaxis protein I, serine sensor receptor\n",
      "KO KO:K05875 methyl-accepting chemotaxis protein II, aspartate sensor receptor\n",
      "SSO SSO:000019571 Methyl-accepting chemotaxis protein II (aspartate chemoreceptor protein)\n",
      "b1886 ['Methyl-accepting chemotaxis protein II (aspartate chemoreceptor protein)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03408 purine-binding chemotaxis protein CheW\n",
      "SSO SSO:000021928 Positive regulator of CheA protein activity (CheW)\n",
      "SSO SSO:000022992 Purine-binding chemotaxis protein\n",
      "b1887 ['Positive regulator of CheA protein activity (CheW)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03407 two-component system, chemotaxis family, sensor kinase CheA [EC:2.7.13.3]\n",
      "SSO SSO:000007525 Signal transduction histidine kinase CheA (EC 2.7.3.-)\n",
      "b1888 ['Signal transduction histidine kinase CheA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02557 chemotaxis protein MotB\n",
      "SSO SSO:000002817 Flagellar motor rotation protein MotB\n",
      "b1889 ['Flagellar motor rotation protein MotB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02556 chemotaxis protein MotA\n",
      "SSO SSO:000002816 Flagellar motor rotation protein MotA\n",
      "b1890 ['Flagellar motor rotation protein MotA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02402 flagellar transcriptional activator FlhC\n",
      "SSO SSO:000002836 Flagellar transcriptional activator FlhC\n",
      "b1891 ['Flagellar transcriptional activator FlhC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02403 flagellar transcriptional activator FlhD\n",
      "SSO SSO:000002837 Flagellar transcriptional activator FlhD\n",
      "b1892 ['Flagellar transcriptional activator FlhD']\n",
      "\n",
      "KO KO:K07480 insertion element IS1 protein InsB\n",
      "SSO SSO:000018188 IS1 protein InsB\n",
      "b1893 ['IS1 protein InsB']\n",
      "\n",
      "SSO SSO:000018187 IS1 protein InsA\n",
      "b1894 ['IS1 protein InsA']\n",
      "\n",
      "KO KO:K14064 universal stress protein C\n",
      "SSO SSO:000008824 Universal stress protein C\n",
      "SSO SSO:000028839 Universal stress protein\n",
      "b1895 ['Universal stress protein C']\n",
      "\n",
      "EC EC:2.4.1.15 Alpha,alpha-trehalose-phosphate synthase (UDP-forming).\n",
      "EC EC:2.4.1.213 Glucosylglycerol-phosphate synthase.\n",
      "EC EC:3.1.3.12 Trehalose-phosphatase.\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K00697 trehalose 6-phosphate synthase [EC:2.4.1.15 2.4.1.347]\n",
      "SSO SSO:000010459 Alpha,alpha-trehalose-phosphate synthase [UDP-forming] (EC 2.4.1.15)\n",
      "SSO SSO:000026177 Trehalose-6-phosphate synthase\n",
      "b1896 ['Alpha,alpha-trehalose-phosphate synthase [UDP-forming] (EC 2.4.1.15)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K01087 trehalose 6-phosphate phosphatase [EC:3.1.3.12]\n",
      "SSO SSO:000010460 Trehalose-6-phosphate phosphatase (EC 3.1.3.12)\n",
      "b1897 ['Trehalose-6-phosphate phosphatase (EC 3.1.3.12)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10440 ribose transport system permease protein\n",
      "KO KO:K10538 L-arabinose transport system permease protein\n",
      "SSO SSO:000036485 L-arabinose ABC transporter, permease protein AraH\n",
      "b4460 ['L-arabinose ABC transporter, permease protein AraH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10539 L-arabinose transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000036484 L-arabinose ABC transporter, ATP-binding protein AraG\n",
      "b1900 ['L-arabinose ABC transporter, ATP-binding protein AraG']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10537 L-arabinose transport system substrate-binding protein\n",
      "SSO SSO:000036486 L-arabinose ABC transporter, substrate-binding protein AraF\n",
      "b1901 ['L-arabinose ABC transporter, substrate-binding protein AraF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K02255 ferritin-like protein 2\n",
      "SSO SSO:000034744 Bacterial non-heme ferritin (EC 1.16.3.2)\n",
      "b1902 ['Bacterial non-heme ferritin (EC 1.16.3.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000012984 DUF2766 family protein\n",
      "b4537 ['FIG00510289: hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4663 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028243 Uncharacterized protein YecR\n",
      "b1904 ['Uncharacterized protein YecR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K02217 ferritin [EC:1.16.3.2]\n",
      "SSO SSO:000034744 Bacterial non-heme ferritin (EC 1.16.3.2)\n",
      "b1905 ['Bacterial non-heme ferritin (EC 1.16.3.2)']\n",
      "\n",
      "SSO SSO:000028242 Uncharacterized protein YecH\n",
      "b1906 ['Uncharacterized protein YecH']\n",
      "\n",
      "GO GO:0006571 tyrosine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03834 tyrosine-specific transport protein\n",
      "SSO SSO:000008590 Tyrosine-specific transport protein\n",
      "b1907 ['Tyrosine-specific transport protein']\n",
      "\n",
      "KO KO:K07039 uncharacterized protein\n",
      "SSO SSO:000028240 Uncharacterized protein YecA\n",
      "b1908 ['Uncharacterized protein YecA']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00995 CDP-diacylglycerol---glycerol-3-phosphate 3-phosphatidyltransferase [EC:2.7.8.5]\n",
      "SSO SSO:000001301 CDP-diacylglycerol--glycerol-3-phosphate 3-phosphatidyltransferase (EC 2.7.8.5)\n",
      "b1912 ['CDP-diacylglycerol--glycerol-3-phosphate 3-phosphatidyltransferase (EC 2.7.8.5)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0009314 response to radiation\n",
      "KO KO:K03703 excinuclease ABC subunit C\n",
      "SSO SSO:000002613 Excinuclease ABC subunit C\n",
      "b1913 ['Excinuclease ABC subunit C']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07689 two-component system, NarL family, invasion response regulator UvrY\n",
      "SSO SSO:000001159 BarA-associated response regulator UvrY (= GacA = SirA)\n",
      "b1914 ['BarA-associated response regulator UvrY (= GacA = SirA)']\n",
      "\n",
      "SSO SSO:000028241 Uncharacterized protein YecF\n",
      "b1915 ['Uncharacterized protein YecF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07782 LuxR family transcriptional regulator, quorum-sensing system regulator SdiA\n",
      "b1916 ['N-(3-oxohexanoyl)-L-homoserine lactone-binding transcriptional activator', 'N-(3-oxooctanoyl)-L-homoserine lactone-binding transcriptional activator']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02028 polar amino acid transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10010 L-cystine transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000018744 L-cystine ABC transporter (wide substrate range), ATP-binding protein YecC\n",
      "b1917 ['L-cystine ABC transporter (wide substrate range), ATP-binding protein YecC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02029 polar amino acid transport system permease protein\n",
      "KO KO:K10009 L-cystine transport system permease protein\n",
      "SSO SSO:000018745 L-cystine ABC transporter (wide substrate range), permease protein YecS\n",
      "b1918 ['L-cystine ABC transporter (wide substrate range), permease protein YecS']\n",
      "\n",
      "KO KO:K05396 D-cysteine desulfhydrase [EC:4.4.1.15]\n",
      "SSO SSO:000002014 D-cysteine desulfhydrase (EC 4.4.1.15)\n",
      "b1919 ['D-cysteine desulfhydrase (EC 4.4.1.15)']\n",
      "\n",
      "GO GO:0019344 cysteine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "GO GO:0042883 cysteine transport\n",
      "KO KO:K02030 polar amino acid transport system substrate-binding protein\n",
      "KO KO:K02424 L-cystine transport system substrate-binding protein\n",
      "SSO SSO:000004138 L-cystine ABC transporter (wide substrate range), substrate-binding protein FliY\n",
      "b1920 ['L-cystine ABC transporter (wide substrate range), substrate-binding protein FliY']\n",
      "\n",
      "KO KO:K02425 regulator of sigma S factor FliZ\n",
      "SSO SSO:000006971 Regulator of sigma S factor FliZ\n",
      "b1921 ['Regulator of sigma S factor FliZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02405 RNA polymerase sigma factor for flagellar operon FliA\n",
      "SSO SSO:000006913 RNA polymerase sigma factor for flagellar operon\n",
      "b1922 ['RNA polymerase sigma factor for flagellar operon']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02406 flagellin\n",
      "SSO SSO:000002840 Flagellin (FliC)\n",
      "b1923 ['Flagellin FliC']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02407 flagellar hook-associated protein 2\n",
      "SSO SSO:000002810 Flagellar cap protein FliD\n",
      "b1924 ['Flagellar cap protein FliD']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02422 flagellar protein FliS\n",
      "SSO SSO:000002807 Flagellar biosynthesis protein FliS\n",
      "b1925 ['Flagellar biosynthesis protein FliS']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02423 flagellar protein FliT\n",
      "SSO SSO:000002808 Flagellar biosynthesis protein FliT\n",
      "b1926 ['Flagellar biosynthesis protein FliT']\n",
      "\n",
      "EC EC:2.4.1.18 1,4-alpha-glucan branching enzyme.\n",
      "EC EC:2.4.1.19 Cyclomaltodextrin glucanotransferase.\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "EC EC:2.4.1.4 Amylosucrase.\n",
      "EC EC:2.4.1.7 Sucrose phosphorylase.\n",
      "EC EC:2.4.99.16 Starch synthase (maltosyl-transferring).\n",
      "EC EC:3.2.1.1 Alpha-amylase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.116 Glucan 1,4-alpha-maltotriohydrolase.\n",
      "EC EC:3.2.1.133 Glucan 1,4-alpha-maltohydrolase.\n",
      "EC EC:3.2.1.135 Neopullulanase.\n",
      "EC EC:3.2.1.141 4-alpha-D-((1->4)-alpha-D-glucano)trehalose trehalohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.33 Amylo-alpha-1,6-glucosidase.\n",
      "EC EC:3.2.1.41 Pullulanase.\n",
      "EC EC:3.2.1.54 Cyclomaltodextrinase.\n",
      "EC EC:3.2.1.60 Glucan 1,4-alpha-maltotetraohydrolase.\n",
      "EC EC:3.2.1.68 Isoamylase.\n",
      "EC EC:3.2.1.70 Glucan 1,6-alpha-glucosidase.\n",
      "EC EC:3.2.1.93 Alpha,alpha-phosphotrehalase.\n",
      "EC EC:3.2.1.98 Glucan 1,4-alpha-maltohexaosidase.\n",
      "EC EC:5.4.99.11 Isomaltulose synthase.\n",
      "EC EC:5.4.99.15 (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase.\n",
      "EC EC:5.4.99.16 Maltose alpha-D-glucosyltransferase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01176 alpha-amylase [EC:3.2.1.1]\n",
      "SSO SSO:000001961 Cytoplasmic alpha-amylase (EC 3.2.1.1)\n",
      "b1927 ['Cytoplasmic alpha-amylase (EC 3.2.1.1)']\n",
      "\n",
      "SSO SSO:000018986 Lipoprotein\n",
      "SSO SSO:000027206 Uncharacterized lipoprotein YedD\n",
      "b1928 ['Uncharacterized lipoprotein YedD']\n",
      "\n",
      "KO KO:K07112 uncharacterized protein\n",
      "SSO SSO:000026666 UPF0394 inner membrane protein YedE\n",
      "b1929 ['UPF0394 inner membrane protein YedE']\n",
      "\n",
      "KO KO:K04085 tRNA 2-thiouridine synthesizing protein A [EC:2.8.1.-]\n",
      "SSO SSO:000026471 UPF0033 protein YedF\n",
      "b1930 ['UPF0033 protein YedF']\n",
      "\n",
      "SSO SSO:000023114 Putative SOS response-associated peptidase YedK\n",
      "b1931 ['Putative SOS response-associated peptidase YedK']\n",
      "\n",
      "KO KO:K03829 putative acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000026954 Uncharacterized N-acetyltransferase YedL\n",
      "b1932 ['Uncharacterized N-acetyltransferase YedL']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4495 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1936 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02408 flagellar hook-basal body complex protein FliE\n",
      "SSO SSO:000002814 Flagellar hook-basal body complex protein FliE\n",
      "b1937 ['Flagellar hook-basal body complex protein FliE']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02409 flagellar M-ring protein FliF\n",
      "SSO SSO:000002788 Flagellar M-ring protein FliF\n",
      "b1938 ['Flagellar M-ring protein FliF']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02410 flagellar motor switch protein FliG\n",
      "SSO SSO:000002818 Flagellar motor switch protein FliG\n",
      "b1939 ['Flagellar motor switch protein FliG']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02411 flagellar assembly protein FliH\n",
      "SSO SSO:000002791 Flagellar assembly protein FliH\n",
      "b1940 ['Flagellar assembly protein FliH']\n",
      "\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019861 obsolete flagellum\n",
      "KO KO:K02412 flagellum-specific ATP synthase [EC:3.6.3.50]\n",
      "SSO SSO:000002843 Flagellum-specific ATP synthase FliI\n",
      "b1941 ['Flagellum-specific ATP synthase FliI']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02413 flagellar FliJ protein\n",
      "SSO SSO:000002829 Flagellar protein FliJ\n",
      "SSO SSO:000016806 Flagellar protein\n",
      "b1942 ['Flagellar protein FliJ']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02414 flagellar hook-length control protein FliK\n",
      "SSO SSO:000002815 Flagellar hook-length control protein FliK\n",
      "b1943 ['Flagellar hook-length control protein FliK']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02415 flagellar FliL protein\n",
      "SSO SSO:000002792 Flagellar basal body-associated protein FliL\n",
      "SSO SSO:000029931 flagellar biosynthesis protein\n",
      "b1944 ['Flagellar basal body-associated protein FliL']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02416 flagellar motor switch protein FliM\n",
      "SSO SSO:000002819 Flagellar motor switch protein FliM\n",
      "b1945 ['Flagellar motor switch protein FliM']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02417 flagellar motor switch protein FliN/FliY\n",
      "SSO SSO:000002820 Flagellar motor switch protein FliN\n",
      "b1946 ['Flagellar motor switch protein FliN']\n",
      "\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02418 flagellar protein FliO/FliZ\n",
      "SSO SSO:000002803 Flagellar biosynthesis protein FliO\n",
      "SSO SSO:000029931 flagellar biosynthesis protein\n",
      "b1947 ['Flagellar biosynthesis protein FliO']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K02419 flagellar biosynthetic protein FliP\n",
      "SSO SSO:000002804 Flagellar biosynthesis protein FliP\n",
      "SSO SSO:000029931 flagellar biosynthesis protein\n",
      "b1948 ['Flagellar biosynthesis protein FliP']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009296 obsolete flagellum assembly\n",
      "GO GO:0019861 obsolete flagellum\n",
      "KO KO:K02420 flagellar biosynthetic protein FliQ\n",
      "SSO SSO:000002805 Flagellar biosynthesis protein FliQ\n",
      "SSO SSO:000029931 flagellar biosynthesis protein\n",
      "b1949 ['Flagellar biosynthesis protein FliQ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019861 obsolete flagellum\n",
      "KO KO:K02421 flagellar biosynthetic protein FliR\n",
      "SSO SSO:000002806 Flagellar biosynthesis protein FliR\n",
      "b1950 ['Flagellar biosynthesis protein FliR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009248 K antigen biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K07781 LuxR family transcriptional regulator, capsular biosynthesis positive transcription factor\n",
      "SSO SSO:000012335 Colanic acid capsular biosynthesis activation accesory protein RcsA, co-regulator with RcsB\n",
      "b1951 ['Colanic acid capsular biosynthesis activation accesory protein RcsA, co-regulator with RcsB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000013252 DsrB protein\n",
      "b1952 ['DsrB protein']\n",
      "\n",
      "SSO SSO:000028482 Uncharacterized protein YodD\n",
      "b1953 ['Uncharacterized protein YodD']\n",
      "\n",
      "KO KO:K07026 mannosyl-3-phosphoglycerate phosphatase [EC:3.1.3.70]\n",
      "SSO SSO:000023406 Putative mannosyl-3-phosphoglycerate phosphatase (EC 3.1.3.70)\n",
      "b1955 ['Putative mannosyl-3-phosphoglycerate phosphatase (EC 3.1.3.70)']\n",
      "\n",
      "KO KO:K21085 diguanylate cyclase [EC:2.7.7.65]\n",
      "b1956 ['Diguanylate cyclase (EC 2.7.7.65) => DgcQ']\n",
      "\n",
      "SSO SSO:000028481 Uncharacterized protein YodC\n",
      "b1957 ['Uncharacterized protein YodC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K09781 uncharacterized protein\n",
      "SSO SSO:000018391 Inner membrane protein YedI\n",
      "b1958 ['Inner membrane protein YedI']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000027121 Uncharacterized inner membrane transporter YedA\n",
      "b1959 ['Uncharacterized inner membrane transporter YedA']\n",
      "\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K07458 DNA mismatch endonuclease, patch repair protein [EC:3.1.-.-]\n",
      "SSO SSO:000008915 Very-short-patch mismatch repair endonuclease (G-T specific)\n",
      "b1960 ['Very-short-patch mismatch repair endonuclease (G-T specific)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006306 DNA methylation\n",
      "KO KO:K00558 DNA (cytosine-5)-methyltransferase 1 [EC:2.1.1.37]\n",
      "SSO SSO:000002148 DNA-cytosine methyltransferase (EC 2.1.1.37)\n",
      "b1961 ['DNA-cytosine methyltransferase (EC 2.1.1.37)']\n",
      "\n",
      "KO KO:K06950 uncharacterized protein\n",
      "SSO SSO:000028245 Uncharacterized protein YedJ\n",
      "b1962 ['Uncharacterized protein YedJ']\n",
      "\n",
      "SSO SSO:000018392 Inner membrane protein YedR\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b1963 ['Inner membrane protein YedR']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4496 ['hypothetical protein']\n",
      "\n",
      "KO KO:K05523 D-lactate dehydratase / protein deglycase [EC:4.2.1.130 3.5.1.124]\n",
      "SSO SSO:000002030 D-lactate dehydratase (EC 4.2.1.130)\n",
      "b1967 ['D-lactate dehydratase (EC 4.2.1.130)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "SSO SSO:000006773 Putative two component system histidine kinase YedV\n",
      "b1968 ['Putative two component system histidine kinase YedV']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07665 two-component system, OmpR family, copper resistance phosphate regulon response regulator CusR\n",
      "SSO SSO:000006775 Putative two-component system response regulator YedW\n",
      "b1969 ['Putative two-component system response regulator YedW']\n",
      "\n",
      "KO KO:K07127 5-hydroxyisourate hydrolase [EC:3.5.2.17]\n",
      "SSO SSO:000000422 5-hydroxyisourate hydrolase (EC 3.5.2.17)\n",
      "SSO SSO:000044172 hydroxyisourate hydrolase (EC 3.5.2.17)\n",
      "b1970 ['5-hydroxyisourate hydrolase (EC 3.5.2.17)']\n",
      "\n",
      "KO KO:K07147 methionine sulfoxide reductase catalytic subunit [EC:1.8.-.-]\n",
      "SSO SSO:000038753 Protein-methionine-sulfoxide reductase catalytic subunit MsrP\n",
      "b1971 ['Protein-methionine-sulfoxide reductase catalytic subunit MsrP']\n",
      "\n",
      "KO KO:K17247 methionine sulfoxide reductase heme-binding subunit\n",
      "SSO SSO:000038754 Protein-methionine-sulfoxide reductase heme-binding subunit MsrQ\n",
      "b1972 ['Protein-methionine-sulfoxide reductase heme-binding subunit MsrQ']\n",
      "\n",
      "SSO SSO:000019506 Metal-binding protein ZinT\n",
      "b1973 ['Metal-binding protein ZinT']\n",
      "\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K12262 cytochrome b561\n",
      "SSO SSO:000012614 Cytochrome b561 homolog 1\n",
      "b1974 ['Cytochrome b561 homolog 1']\n",
      "\n",
      "KO KO:K09933 MtfA peptidase\n",
      "SSO SSO:000022708 Protein MtfA\n",
      "b1976 ['Protein MtfA']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K13735 adhesin/invasin\n",
      "KO KO:K20276 large repetitive protein\n",
      "SSO SSO:000018569 Invasin\n",
      "SSO SSO:000023144 Putative adhesin\n",
      "b1978 ['Invasin']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4497 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08172 MFS transporter, MHS family, shikimate and dehydroshikimate transport protein\n",
      "SSO SSO:000024619 Shikimate transporter\n",
      "b1981 ['Shikimate transporter']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01241 AMP nucleosidase [EC:3.2.2.4]\n",
      "SSO SSO:000000536 AMP nucleosidase (EC 3.2.2.4)\n",
      "b1982 ['AMP nucleosidase (EC 3.2.2.4)']\n",
      "\n",
      "KO KO:K18189 translational activator of cytochrome c oxidase 1\n",
      "SSO SSO:000025968 Transcriptional regulatory protein YeeN\n",
      "b1983 ['Transcriptional regulatory protein YeeN']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03327 multidrug resistance protein, MATE family\n",
      "SSO SSO:000028792 Uncharacterized transporter YeeO\n",
      "b1985 ['Uncharacterized transporter YeeO']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0019344 cysteine biosynthetic process\n",
      "KO KO:K13635 LysR family transcriptional regulator, cys regulon transcriptional activator\n",
      "b1987 ['Alkanesulfonate utilization operon LysR-family regulator Cbl']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K19338 LysR family transcriptional regulator, nitrogen assimilation regulatory protein\n",
      "SSO SSO:000005348 Nitrogen assimilation regulatory protein Nac\n",
      "b1988 ['Nitrogen assimilation regulatory protein Nac']\n",
      "\n",
      "KO KO:K16291 L,D-transpeptidase ErfK/SrfK\n",
      "b1990 ['L,D-transpeptidase => ErfK']\n",
      "\n",
      "KO KO:K00768 nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase [EC:2.4.2.21]\n",
      "SSO SSO:000005317 Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)\n",
      "b1991 ['Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)']\n",
      "\n",
      "KO KO:K02233 adenosylcobinamide-GDP ribazoletransferase [EC:2.7.8.26]\n",
      "SSO SSO:000001627 Cobalamin synthase (EC 2.7.8.26)\n",
      "b1992 ['Cobalamin synthase (EC 2.7.8.26)']\n",
      "\n",
      "KO KO:K02231 adenosylcobinamide kinase / adenosylcobinamide-phosphate guanylyltransferase [EC:2.7.1.156 2.7.7.62]\n",
      "b1993 ['Adenosylcobinamide kinase (EC 2.7.1.156)', 'Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4639 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b1994 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4640 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4641 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4582 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000026095 Transposase InsD for insertion element IS2\n",
      "b1996 ['Transposase InsD for insertion element IS2']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026094 Transposase InsC for insertion element IS2\n",
      "b1997 ['Transposase InsC for insertion element IS2']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b1999 ['hypothetical protein']\n",
      "\n",
      "KO KO:K12687 antigen 43\n",
      "SSO SSO:000010715 Antigen 43\n",
      "b2000 ['Antigen 43']\n",
      "\n",
      "SSO SSO:000018394 Inner membrane protein YeeR\n",
      "b2001 ['Inner membrane protein YeeR']\n",
      "\n",
      "KO KO:K03630 DNA repair protein RadC\n",
      "SSO SSO:000026712 UPF0758 protein YeeS\n",
      "b2002 ['UPF0758 protein YeeS']\n",
      "\n",
      "SSO SSO:000028249 Uncharacterized protein YeeT\n",
      "b2003 ['Uncharacterized protein YeeT']\n",
      "\n",
      "KO KO:K18838 cytoskeleton bundling-enhancing protein CbeA and related proteins\n",
      "SSO SSO:000029130 YeeU protein (antitoxin to YeeV)\n",
      "b2004 ['YeeU protein (antitoxin to YeeV)']\n",
      "\n",
      "KO KO:K18837 cytoskeleton-binding toxin CbtA and related proteins\n",
      "SSO SSO:000029131 YeeV toxin protein\n",
      "b2005 ['YeeV toxin protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2006 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4642 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4538 ['hypothetical protein']\n",
      "\n",
      "KO KO:K09802 uncharacterized protein\n",
      "SSO SSO:000008667 UPF0265 protein YeeX\n",
      "b2007 ['UPF0265 protein YeeX']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000018393 Inner membrane protein YeeA\n",
      "b2008 ['Inner membrane protein YeeA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K07470 DNA gyrase inhibitor\n",
      "SSO SSO:000012760 DNA gyrase inhibitory protein\n",
      "b2009 ['DNA gyrase inhibitory protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K07258 serine-type D-Ala-D-Ala carboxypeptidase (penicillin-binding protein 5/6) [EC:3.4.16.4]\n",
      "SSO SSO:000001995 D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)\n",
      "b2010 ['D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K01141 exodeoxyribonuclease I [EC:3.1.11.1]\n",
      "SSO SSO:000002618 Exodeoxyribonuclease I (EC 3.1.11.1)\n",
      "b2011 ['Exodeoxyribonuclease I (EC 3.1.11.1)']\n",
      "\n",
      "KO KO:K04085 tRNA 2-thiouridine synthesizing protein A [EC:2.8.1.-]\n",
      "SSO SSO:000026472 UPF0033 protein YeeD\n",
      "b2012 ['UPF0033 protein YeeD']\n",
      "\n",
      "KO KO:K07112 uncharacterized protein\n",
      "SSO SSO:000026667 UPF0394 inner membrane protein YeeE\n",
      "b2013 ['UPF0394 inner membrane protein YeeE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K14052 putrescine importer\n",
      "SSO SSO:000019059 Low-affinity putrescine importer PlaP\n",
      "b2014 ['Low-affinity putrescine importer PlaP']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4678 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000019130 LysR-family transcriptional regulator YeeY\n",
      "b2015 ['LysR-family transcriptional regulator YeeY']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000022730 Protein YeeZ\n",
      "b2016 ['Protein YeeZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K19158 toxin YoeB [EC:3.1.-.-]\n",
      "SSO SSO:000008988 YoeB toxin protein\n",
      "b4539 ['YoeB toxin protein']\n",
      "\n",
      "KO KO:K19159 antitoxin YefM\n",
      "SSO SSO:000008980 YefM protein (antitoxin to YoeB)\n",
      "b2017 ['YefM protein (antitoxin to YoeB)']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "SSO SSO:000030126 his operon leader peptide\n",
      "b2018 ['his operon leader peptide']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00765 ATP phosphoribosyltransferase [EC:2.4.2.17]\n",
      "SSO SSO:000000542 ATP phosphoribosyltransferase (EC 2.4.2.17)\n",
      "b2019 ['ATP phosphoribosyltransferase (EC 2.4.2.17) => HisGl']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "KO KO:K00013 histidinol dehydrogenase [EC:1.1.1.23]\n",
      "SSO SSO:000003446 Histidinol dehydrogenase (EC 1.1.1.23)\n",
      "b2020 ['Histidinol dehydrogenase (EC 1.1.1.23)']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "KO KO:K00817 histidinol-phosphate aminotransferase [EC:2.6.1.9]\n",
      "SSO SSO:000003449 Histidinol-phosphate aminotransferase (EC 2.6.1.9)\n",
      "b2021 ['Histidinol-phosphate aminotransferase (EC 2.6.1.9)']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "KO KO:K01089 imidazoleglycerol-phosphate dehydratase / histidinol-phosphatase [EC:4.2.1.19 3.1.3.15]\n",
      "b2022 ['Histidinol-phosphatase (EC 3.1.3.15)', 'Imidazoleglycerol-phosphate dehydratase (EC 4.2.1.19)']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "KO KO:K02501 imidazole glycerol-phosphate synthase subunit HisH [EC:4.3.2.10]\n",
      "SSO SSO:000036369 Imidazole glycerol phosphate synthase amidotransferase subunit HisH\n",
      "b2023 ['Imidazole glycerol phosphate synthase amidotransferase subunit HisH']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01814 phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase [EC:5.3.1.16]\n",
      "SSO SSO:000006041 Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)\n",
      "b2024 ['Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02500 imidazole glycerol-phosphate synthase subunit HisF [EC:4.3.2.10]\n",
      "SSO SSO:000003694 Imidazole glycerol phosphate synthase cyclase subunit (EC 4.1.3.-)\n",
      "b2025 ['Imidazole glycerol phosphate synthase cyclase subunit']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K11755 phosphoribosyl-ATP pyrophosphohydrolase / phosphoribosyl-AMP cyclohydrolase [EC:3.6.1.31 3.5.4.19]\n",
      "b2026 ['Phosphoribosyl-AMP cyclohydrolase (EC 3.5.4.19)', 'Phosphoribosyl-ATP pyrophosphatase (EC 3.6.1.31)']\n",
      "\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K05789 chain length determinant protein (polysaccharide antigen chain regulator)\n",
      "SSO SSO:000005465 O-antigen chain length determinant protein WzzB\n",
      "SSO SSO:000042178 regulator of length of O-antigen component of lipopolysaccharide chains\n",
      "b2027 ['O-antigen chain length determinant protein WzzB']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00012 UDPglucose 6-dehydrogenase [EC:1.1.1.22]\n",
      "SSO SSO:000008645 UDP-glucose 6-dehydrogenase (EC 1.1.1.22)\n",
      "b2028 ['UDP-glucose 6-dehydrogenase (EC 1.1.1.22)']\n",
      "\n",
      "GO GO:0009051 pentose-phosphate shunt, oxidative branch\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00033 6-phosphogluconate dehydrogenase [EC:1.1.1.44 1.1.1.343]\n",
      "SSO SSO:000000456 6-phosphogluconate dehydrogenase, decarboxylating (EC 1.1.1.44)\n",
      "b2029 ['6-phosphogluconate dehydrogenase, decarboxylating (EC 1.1.1.44)']\n",
      "\n",
      "SSO SSO:000024109 Rhamnosyltransferase WbbL\n",
      "b4571 ['Rhamnosyltransferase WbbL']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b2030 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "EC EC:2.4.1.13 Sucrose synthase.\n",
      "EC EC:2.4.1.131 GDP-Man:Man(3)GlcNAc(2)-PP-dolichol alpha-1,2-mannosyltransferase.\n",
      "EC EC:2.4.1.132 GDP-Man:Man(1)GlcNAc(2)-PP-dolichol alpha-1,3-mannosyltransferase.\n",
      "EC EC:2.4.1.14 Sucrose-phosphate synthase.\n",
      "EC EC:2.4.1.141 N-acetylglucosaminyldiphosphodolichol N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.157 Transferred entry: 2.4.1.336 and 2.4.1.337.\n",
      "EC EC:2.4.1.208 Diglucosyl diacylglycerol synthase (1,2-linking).\n",
      "EC EC:2.4.1.231 Alpha,alpha-trehalose phosphorylase (configuration-retaining).\n",
      "EC EC:2.4.1.245 Alpha,alpha-trehalose synthase.\n",
      "EC EC:2.4.1.257 GDP-Man:Man(2)GlcNAc(2)-PP-dolichol alpha-1,6-mannosyltransferase.\n",
      "EC EC:2.4.1.283 2-deoxystreptamine N-acetyl-D-glucosaminyltransferase.\n",
      "EC EC:2.4.1.285 UDP-GlcNAc:ribostamycin N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.290 N,N'-diacetylbacillosaminyl-diphospho-undecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.306 UDP-GalNAc:alpha-D-GalNAc-diphosphoundecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.52 Poly(glycerol-phosphate) alpha-glucosyltransferase.\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.57 Transferred entry: 2.4.1.345.\n",
      "SSO SSO:000018973 Lipopolysaccharide biosynthesis protein\n",
      "SSO SSO:000023337 Putative glycosyltransferase WbbK\n",
      "b2032 ['Putative glycosyltransferase WbbK']\n",
      "\n",
      "GO GO:0009103 lipopolysaccharide biosynthetic process\n",
      "KO KO:K08280 lipopolysaccharide O-acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000018971 Lipopolysaccharide biosynthesis O-acetyl transferase WbbJ\n",
      "b2033 ['Lipopolysaccharide biosynthesis O-acetyl transferase WbbJ']\n",
      "\n",
      "SSO SSO:000011078 Beta-1,6-galactofuranosyltransferase WbbI\n",
      "b2034 ['Beta-1,6-galactofuranosyltransferase WbbI']\n",
      "\n",
      "GO GO:0008755 O antigen polymerase activity\n",
      "GO GO:0009103 lipopolysaccharide biosynthetic process\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "SSO SSO:000020696 O antigen polymerase\n",
      "b2035 ['O-antigen polymerase']\n",
      "\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01854 UDP-galactopyranose mutase [EC:5.4.99.9]\n",
      "SSO SSO:000008641 UDP-galactopyranose mutase (EC 5.4.99.9)\n",
      "b2036 ['UDP-galactopyranose mutase (EC 5.4.99.9)']\n",
      "\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "SSO SSO:000019445 Membrane protein involved in the export of O-antigen, teichoic acid lipoteichoic acids\n",
      "b2037 ['Membrane protein involved in the export of O-antigen, teichoic acid lipoteichoic acids']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009103 lipopolysaccharide biosynthetic process\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0019305 dTDP-rhamnose biosynthetic process\n",
      "GO GO:0042803 protein homodimerization activity\n",
      "KO KO:K01790 dTDP-4-dehydrorhamnose 3,5-epimerase [EC:5.1.3.13]\n",
      "SSO SSO:000009089 dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)\n",
      "b2038 ['dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)']\n",
      "\n",
      "GO GO:0000287 magnesium ion binding\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009103 lipopolysaccharide biosynthetic process\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0019305 dTDP-rhamnose biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00973 glucose-1-phosphate thymidylyltransferase [EC:2.7.7.24]\n",
      "SSO SSO:000003126 Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)\n",
      "b2039 ['Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "KO KO:K00067 dTDP-4-dehydrorhamnose reductase [EC:1.1.1.133]\n",
      "SSO SSO:000009090 dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)\n",
      "b2040 ['dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01710 dTDP-glucose 4,6-dehydratase [EC:4.2.1.46]\n",
      "SSO SSO:000009095 dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)\n",
      "b2041 ['dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "KO KO:K00963 UTP--glucose-1-phosphate uridylyltransferase [EC:2.7.7.9]\n",
      "SSO SSO:000008674 UTP--glucose-1-phosphate uridylyltransferase (EC 2.7.7.9)\n",
      "b2042 ['UTP--glucose-1-phosphate uridylyltransferase (EC 2.7.7.9)']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K16711 colanic acid biosynthesis protein WcaM\n",
      "SSO SSO:000012333 Colanic acid biosynthesis protein WcaM\n",
      "b2043 ['Colanic acid biosynthesis protein WcaM']\n",
      "\n",
      "EC EC:2.4.1.13 Sucrose synthase.\n",
      "EC EC:2.4.1.131 GDP-Man:Man(3)GlcNAc(2)-PP-dolichol alpha-1,2-mannosyltransferase.\n",
      "EC EC:2.4.1.132 GDP-Man:Man(1)GlcNAc(2)-PP-dolichol alpha-1,3-mannosyltransferase.\n",
      "EC EC:2.4.1.14 Sucrose-phosphate synthase.\n",
      "EC EC:2.4.1.141 N-acetylglucosaminyldiphosphodolichol N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.157 Transferred entry: 2.4.1.336 and 2.4.1.337.\n",
      "EC EC:2.4.1.208 Diglucosyl diacylglycerol synthase (1,2-linking).\n",
      "EC EC:2.4.1.231 Alpha,alpha-trehalose phosphorylase (configuration-retaining).\n",
      "EC EC:2.4.1.245 Alpha,alpha-trehalose synthase.\n",
      "EC EC:2.4.1.257 GDP-Man:Man(2)GlcNAc(2)-PP-dolichol alpha-1,6-mannosyltransferase.\n",
      "EC EC:2.4.1.283 2-deoxystreptamine N-acetyl-D-glucosaminyltransferase.\n",
      "EC EC:2.4.1.285 UDP-GlcNAc:ribostamycin N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.290 N,N'-diacetylbacillosaminyl-diphospho-undecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.306 UDP-GalNAc:alpha-D-GalNAc-diphosphoundecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.52 Poly(glycerol-phosphate) alpha-glucosyltransferase.\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.57 Transferred entry: 2.4.1.345.\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K16703 colanic acid/amylovoran biosynthesis glycosyltransferase [EC:2.4.-.-]\n",
      "SSO SSO:000012332 Colanic acid biosynthesis glycosyl transferase WcaL\n",
      "b2044 ['Colanic acid biosynthesis glycosyl transferase WcaL']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K16710 colanic acid/amylovoran biosynthesis protein\n",
      "b2045 ['Polysaccharide pyruvyl transferase => Colanic acid protein WcaK']\n",
      "\n",
      "KO KO:K16695 lipopolysaccharide exporter\n",
      "SSO SSO:000018974 Lipopolysaccharide biosynthesis protein WzxC\n",
      "b2046 ['Lipopolysaccharide biosynthesis protein WzxC']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K03606 putative colanic acid biosysnthesis UDP-glucose lipid carrier transferase\n",
      "SSO SSO:000008650 UDP-glucose:undecaprenyl-phosphate glucose-1-phosphate transferase (EC 2.7.8.31)\n",
      "b2047 ['UDP-glucose:undecaprenyl-phosphate glucose-1-phosphate transferase (EC 2.7.8.31)']\n",
      "\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01840 phosphomannomutase [EC:5.4.2.8]\n",
      "SSO SSO:000005992 Phosphomannomutase (EC 5.4.2.8)\n",
      "b2048 ['Phosphomannomutase (EC 5.4.2.8) => Colanic acid']\n",
      "\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00971 mannose-1-phosphate guanylyltransferase [EC:2.7.7.13]\n",
      "KO KO:K16011 mannose-1-phosphate guanylyltransferase / mannose-6-phosphate isomerase [EC:2.7.7.13 5.3.1.8]\n",
      "SSO SSO:000019340 Mannose-1-phosphate guanyltransferase\n",
      "b2049 ['Mannose-1-phosphate guanylyltransferase (EC 2.7.7.13) => Colanic acid', 'Mannose-6-phosphate isomerase (EC 5.3.1.8) => Colanic acid']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K03208 colanic acid biosynthesis glycosyl transferase WcaI\n",
      "SSO SSO:000012334 Colanic acid biosysnthesis glycosyl transferase WcaI\n",
      "b2050 ['Colanic acid biosysnthesis glycosyl transferase WcaI']\n",
      "\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K03207 colanic acid biosynthesis protein WcaH [EC:3.6.1.-]\n",
      "SSO SSO:000002974 GDP-mannose mannosyl hydrolase (EC 3.6.1.-)\n",
      "b2051 ['GDP-mannose mannosyl hydrolase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02377 GDP-L-fucose synthase [EC:1.1.1.271]\n",
      "SSO SSO:000002969 GDP-L-fucose synthetase (EC 1.1.1.271)\n",
      "b2052 ['GDP-L-fucose synthetase (EC 1.1.1.271)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009269 response to desiccation\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01711 GDPmannose 4,6-dehydratase [EC:4.2.1.47]\n",
      "SSO SSO:000002972 GDP-mannose 4,6-dehydratase (EC 4.2.1.47)\n",
      "b2053 ['GDP-mannose 4,6-dehydratase (EC 4.2.1.47)']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K03818 putative colanic acid biosynthesis acetyltransferase WcaF [EC:2.3.1.-]\n",
      "SSO SSO:000012328 Colanic acid biosynthesis acetyltransferase WcaF (EC 2.3.1.-)\n",
      "SSO SSO:000023143 Putative acyltransferase\n",
      "b2054 ['Colanic acid biosynthesis acetyltransferase WcaF (EC 2.3.1.-)']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K13683 putative colanic acid biosynthesis glycosyltransferase [EC:2.4.-.-]\n",
      "SSO SSO:000012331 Colanic acid biosynthesis glycosyl transferase WcaE\n",
      "b2055 ['Colanic acid biosynthesis glycosyl transferase WcaE']\n",
      "\n",
      "KO KO:K13620 putative colanic acid polymerase\n",
      "SSO SSO:000012336 Colanic acid polymerase WcaD\n",
      "b2056 ['Colanic acid polymerase WcaD']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K13684 putative colanic acid biosynthesis glycosyltransferase [EC:2.4.-.-]\n",
      "SSO SSO:000012330 Colanic acid biosynthesis glycosyl transferase WcaC\n",
      "b2057 ['Colanic acid biosynthesis glycosyl transferase WcaC']\n",
      "\n",
      "EC EC:2.3.1.30 Serine O-acetyltransferase.\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K00640 serine O-acetyltransferase [EC:2.3.1.30]\n",
      "KO KO:K03819 putative colanic acid biosynthesis acetyltransferase WcaB [EC:2.3.1.-]\n",
      "SSO SSO:000012327 Colanic acid biosynthesis acetyltransferase WcaB (EC 2.3.1.-)\n",
      "SSO SSO:000023143 Putative acyltransferase\n",
      "b2058 ['Colanic acid biosynthesis acetyltransferase WcaB (EC 2.3.1.-)']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "SSO SSO:000012329 Colanic acid biosynthesis glycosyl transferase WcaA\n",
      "b2059 ['Colanic acid biosynthesis glycosyl transferase WcaA']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "KO KO:K16692 tyrosine-protein kinase Etk/Wzc [EC:2.7.10.-]\n",
      "b2060 ['Tyrosine-protein kinase (EC 2.7.10.2) => Wzc']\n",
      "\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "b2061 ['Low molecular weight protein-tyrosine-phosphatase (EC 3.1.3.48) => Wzb']\n",
      "\n",
      "KO KO:K01991 polysaccharide biosynthesis/export protein\n",
      "b2062 ['Polysaccharide export lipoprotein => Wza']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000023206 Putative capsular polysaccharide transport protein YegH\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b2063 ['Putative capsular polysaccharide transport protein YegH']\n",
      "\n",
      "GO GO:0009103 lipopolysaccharide biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K07289 AsmA protein\n",
      "SSO SSO:000010849 AsmA protein\n",
      "b2064 ['AsmA protein']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01494 dCTP deaminase [EC:3.5.4.13]\n",
      "SSO SSO:000002243 Deoxycytidine triphosphate deaminase (EC 3.5.4.13)\n",
      "b2065 ['Deoxycytidine triphosphate deaminase (EC 3.5.4.13)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00876 uridine kinase [EC:2.7.1.48]\n",
      "SSO SSO:000008877 Uridine kinase (EC 2.7.1.48)\n",
      "b2066 ['Uridine kinase (EC 2.7.1.48)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K21084 diguanylate cyclase [EC:2.7.7.65]\n",
      "SSO SSO:000032478 putative diguanylate cyclase\n",
      "b2067 ['Diguanylate cyclase (EC 2.7.7.65) => DgcE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "SSO SSO:000002130 DNA-3-methyladenine glycosylase II (EC 3.2.2.21)\n",
      "b2068 ['DNA-3-methyladenine glycosylase II (EC 3.2.2.21)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04046 hypothetical chaperone protein\n",
      "SSO SSO:000027053 Uncharacterized chaperone protein YegD\n",
      "b2069 ['Uncharacterized chaperone protein YegD']\n",
      "\n",
      "SSO SSO:000028251 Uncharacterized protein YegI\n",
      "b2070 ['Uncharacterized protein YegI']\n",
      "\n",
      "SSO SSO:000028252 Uncharacterized protein YegJ\n",
      "b2071 ['Uncharacterized protein YegJ']\n",
      "\n",
      "b2072 ['Uncharacterized protein YegK']\n",
      "\n",
      "SSO SSO:000028253 Uncharacterized protein YegL\n",
      "b2073 ['Uncharacterized protein YegL']\n",
      "\n",
      "SSO SSO:000024889 Small toxic protein IbsA\n",
      "b4667 ['Small toxic protein IbsA']\n",
      "\n",
      "SSO SSO:000024890 Small toxic protein IbsB\n",
      "b4668 ['Small toxic protein IbsB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03585 membrane fusion protein, multidrug efflux system\n",
      "KO KO:K07799 membrane fusion protein, multidrug efflux system\n",
      "SSO SSO:000004981 Multidrug efflux system MdtABC-TolC, membrane fusion component MdtA\n",
      "b2074 ['Multidrug efflux system MdtABC-TolC, membrane fusion component MdtA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07788 multidrug efflux pump\n",
      "SSO SSO:000004978 Multidrug efflux system MdtABC-TolC, inner-membrane proton/drug antiporter MdtB (RND type)\n",
      "b2075 ['Multidrug efflux system MdtABC-TolC, inner-membrane proton/drug antiporter MdtB (RND type)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07789 multidrug efflux pump\n",
      "SSO SSO:000004980 Multidrug efflux system MdtABC-TolC, inner-membrane proton/drug antiporter MdtC (RND type)\n",
      "b2076 ['Multidrug efflux system MdtABC-TolC, inner-membrane proton/drug antiporter MdtC (RND type)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K18326 MFS transporter, DHA2 family, multidrug resistance protein\n",
      "SSO SSO:000008809 Uncharacterized transporter MdtD of major facilitator superfamily (MFS)\n",
      "b2077 ['Uncharacterized transporter MdtD of major facilitator superfamily (MFS)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07642 two-component system, OmpR family, sensor histidine kinase BaeS [EC:2.7.13.3]\n",
      "SSO SSO:000024529 Sensory histidine kinase BaeS\n",
      "b2078 ['Sensory histidine kinase BaeS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07664 two-component system, OmpR family, response regulator BaeR\n",
      "SSO SSO:000024037 Response regulator BaeR\n",
      "b2079 ['Response regulator BaeR']\n",
      "\n",
      "KO KO:K09946 uncharacterized protein\n",
      "b2080 ['UPF0339 protein YegP.N', 'UPF0339 protein YegP.C']\n",
      "\n",
      "KO KO:K08303 putative protease [EC:3.4.-.-]\n",
      "SSO SSO:000023495 Putative peptidase\n",
      "b2081 ['Uncharacterized protease YegQ']\n",
      "\n",
      "b2082 ['Phage protein Ogr']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2083 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028254 Uncharacterized protein YegR\n",
      "b2085 ['Uncharacterized protein YegR']\n",
      "\n",
      "KO KO:K07029 diacylglycerol kinase (ATP) [EC:2.7.1.107]\n",
      "SSO SSO:000018960 Lipid kinase YegS\n",
      "b2086 ['Lipid kinase YegS']\n",
      "\n",
      "KO KO:K02436 DeoR family transcriptional regulator, galactitol utilization operon repressor\n",
      "SSO SSO:000017036 Galactitol utilization operon repressor\n",
      "b4498 ['Galactitol utilization operon repressor']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026096 Transposase InsE for insertion sequence IS3E\n",
      "b2088 ['Transposase InsE for insertion sequence IS3E']\n",
      "\n",
      "KO KO:K07497 putative transposase\n",
      "b2089 ['Transposase InsF for insertion sequence IS3', 'Mobile element protein']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00094 galactitol-1-phosphate 5-dehydrogenase [EC:1.1.1.251]\n",
      "SSO SSO:000017037 Galactitol-1-phosphate 5-dehydrogenase (EC 1.1.1.251)\n",
      "b2091 ['Galactitol-1-phosphate 5-dehydrogenase (EC 1.1.1.251)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02775 PTS system, galactitol-specific IIC component\n",
      "SSO SSO:000021191 PTS system, galactitol-specific IIC component\n",
      "b2092 ['PTS system, galactitol-specific IIC component']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02774 PTS system, galactitol-specific IIB component [EC:2.7.1.200]\n",
      "SSO SSO:000021190 PTS system, galactitol-specific IIB component (EC 2.7.1.69)\n",
      "b2093 ['PTS system, galactitol-specific IIB component (EC 2.7.1.200)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02773 PTS system, galactitol-specific IIA component [EC:2.7.1.200]\n",
      "SSO SSO:000021189 PTS system, galactitol-specific IIA component\n",
      "b2094 ['PTS system, galactitol-specific IIA component (EC 2.7.1.200)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K16371 D-tagatose-1,6-bisphosphate aldolase subunit GatZ/KbaZ\n",
      "b2095 ['Tagatose-6-phosphate kinase GatZ (EC 2.7.1.144)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01624 fructose-bisphosphate aldolase, class II [EC:4.1.2.13]\n",
      "KO KO:K08302 tagatose 1,6-diphosphate aldolase GatY/KbaY [EC:4.1.2.40]\n",
      "SSO SSO:000007902 Tagatose-1,6-bisphosphate aldolase GatY (EC 4.1.2.40)\n",
      "b2096 ['Tagatose-1,6-bisphosphate aldolase GatY (EC 4.1.2.40)']\n",
      "\n",
      "GO GO:0006096 glycolytic process\n",
      "KO KO:K11645 fructose-bisphosphate aldolase, class I [EC:4.1.2.13]\n",
      "SSO SSO:000002934 Fructose-bisphosphate aldolase class I (EC 4.1.2.13)\n",
      "b2097 ['Fructose-bisphosphate aldolase class I (EC 4.1.2.13)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05820 MFS transporter, PPP family, 3-phenylpropionic acid transporter\n",
      "SSO SSO:000023457 Putative nucleoside transporter YegT\n",
      "b2098 ['Putative nucleoside transporter YegT']\n",
      "\n",
      "SSO SSO:000009961 ADP-ribosylglycohydrolase family protein\n",
      "SSO SSO:000009962 ADP-ribosylglycohydrolase family protein YegU\n",
      "b2099 ['ADP-ribosylglycohydrolase family protein YegU']\n",
      "\n",
      "SSO SSO:000023380 Putative kinase\n",
      "SSO SSO:000028701 Uncharacterized sugar kinase YegV, PfkB family\n",
      "b2100 ['Uncharacterized sugar kinase YegV, PfkB family']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03710 GntR family transcriptional regulator\n",
      "b2101 ['Uncharacterized HTH-type transcriptional regulator YegW']\n",
      "\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "KO KO:K07273 lysozyme\n",
      "SSO SSO:000004617 Lyzozyme M1 (1,4-beta-N-acetylmuramidase) (EC 3.2.1.17)\n",
      "b2102 ['Lyzozyme M1 (1,4-beta-N-acetylmuramidase) (EC 3.2.1.17)']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K00941 hydroxymethylpyrimidine/phosphomethylpyrimidine kinase [EC:2.7.1.49 2.7.4.7]\n",
      "SSO SSO:000003533 Hydroxymethylpyrimidine phosphate kinase ThiD (EC 2.7.4.7)\n",
      "b2103 ['Hydroxymethylpyrimidine phosphate kinase ThiD (EC 2.7.4.7)']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K00878 hydroxyethylthiazole kinase [EC:2.7.1.50]\n",
      "SSO SSO:000003523 Hydroxyethylthiazole kinase (EC 2.7.1.50)\n",
      "b2104 ['Hydroxyethylthiazole kinase (EC 2.7.1.50)']\n",
      "\n",
      "KO KO:K23240 Unknown\n",
      "SSO SSO:000008266 Transcriptional repressor RcnR\n",
      "b2105 ['Transcriptional repressor RcnR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K08970 nickel/cobalt exporter\n",
      "SSO SSO:000005296 Nickel/cobalt efflux transporter RcnA\n",
      "b2106 ['Nickel/cobalt efflux transporter RcnA']\n",
      "\n",
      "KO KO:K23243 Unknown\n",
      "SSO SSO:000005297 Nickel/cobalt homeostasis protein RcnB\n",
      "b2107 ['Nickel/cobalt homeostasis protein RcnB']\n",
      "\n",
      "SSO SSO:000008724 Uncharacterized fimbrial-like protein YehA\n",
      "b2108 ['Uncharacterized fimbrial-like protein YehA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07347 outer membrane usher protein\n",
      "SSO SSO:000005607 Outer membrane usher protein YehB\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "b2109 ['Outer membrane usher protein YehB']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000006324 Probable fimbrial chaperone YehC\n",
      "b2110 ['Probable fimbrial chaperone YehC']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000008725 Uncharacterized fimbrial-like protein YehD\n",
      "b2111 ['Uncharacterized fimbrial-like protein YehD']\n",
      "\n",
      "SSO SSO:000028255 Uncharacterized protein YehE\n",
      "b2112 ['Uncharacterized protein YehE']\n",
      "\n",
      "KO KO:K03593 ATP-binding protein involved in chromosome partitioning\n",
      "SSO SSO:000029238 [4Fe-4S] cluster assembly scaffold protein Mrp (=ApbC)\n",
      "b2113 ['[4Fe-4S] cluster assembly scaffold protein Mrp (=ApbC)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01874 methionyl-tRNA synthetase [EC:6.1.1.10]\n",
      "SSO SSO:000004843 Methionyl-tRNA synthetase (EC 6.1.1.10)\n",
      "b2114 ['Methionyl-tRNA synthetase (EC 6.1.1.10)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4499 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028256 Uncharacterized protein YehI\n",
      "b2118 ['Uncharacterized protein YehI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000028257 Uncharacterized protein YehK\n",
      "b4541 ['Uncharacterized protein YehK']\n",
      "\n",
      "GO GO:0004009 obsolete ATP-binding cassette (ABC) transporter activity\n",
      "SSO SSO:000028258 Uncharacterized protein YehL\n",
      "b2119 ['Uncharacterized protein YehL']\n",
      "\n",
      "SSO SSO:000028259 Uncharacterized protein YehM\n",
      "b2120 ['Uncharacterized protein YehM']\n",
      "\n",
      "SSO SSO:000028260 YehP, CoxE-like von Willebrand factor type A (vWA) domain-containing protein\n",
      "b2121 ['YehP, CoxE-like von Willebrand factor type A (vWA) domain-containing protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2122 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000027207 Uncharacterized lipoprotein YehR\n",
      "b2123 ['Uncharacterized lipoprotein YehR']\n",
      "\n",
      "SSO SSO:000028262 Uncharacterized protein YehS\n",
      "b2124 ['Uncharacterized protein YehS']\n",
      "\n",
      "KO KO:K02477 two-component system, LytTR family, response regulator\n",
      "SSO SSO:000039836 Two-component transcriptional response regulator BtsR\n",
      "b2125 ['Two-component transcriptional response regulator BtsR']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K02478 two-component system, LytTR family, sensor kinase [EC:2.7.13.3]\n",
      "SSO SSO:000039127 Sensor histidine kinase BtsS\n",
      "b2126 ['Sensor histidine kinase BtsS']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K21089 MerR family transcriptional regulator, activator of the csg genes\n",
      "SSO SSO:000017602 HTH-type transcriptional regulator MlrA\n",
      "b2127 ['HTH-type transcriptional regulator MlrA']\n",
      "\n",
      "SSO SSO:000026665 UPF0387 membrane protein YohO\n",
      "SSO SSO:000030665 putative membrane protein\n",
      "b4542 ['UPF0387 membrane protein YohO']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05846 osmoprotectant transport system permease protein\n",
      "SSO SSO:000020844 Osmoprotectant ABC transporter inner membrane protein YehW\n",
      "b2128 ['Osmoprotectant ABC transporter inner membrane protein YehW']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K05847 osmoprotectant transport system ATP-binding protein\n",
      "SSO SSO:000020842 Osmoprotectant ABC transporter ATP-binding subunit YehX\n",
      "b2129 ['Osmoprotectant ABC transporter ATP-binding subunit YehX']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05846 osmoprotectant transport system permease protein\n",
      "SSO SSO:000020845 Osmoprotectant ABC transporter permease protein YehY\n",
      "b2130 ['Osmoprotectant ABC transporter permease protein YehY']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K05845 osmoprotectant transport system substrate-binding protein\n",
      "SSO SSO:000020843 Osmoprotectant ABC transporter binding protein YehZ\n",
      "b2131 ['Osmoprotectant ABC transporter binding protein YehZ']\n",
      "\n",
      "EC EC:3.2.1.120 Oligoxyloglucan beta-glycosidase.\n",
      "EC EC:3.2.1.126 Coniferin beta-glucosidase.\n",
      "EC EC:3.2.1.21 Beta-glucosidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.45 Glucosylceramidase.\n",
      "EC EC:3.2.1.52 Beta-N-acetylhexosaminidase.\n",
      "EC EC:3.2.1.55 Non-reducing end alpha-L-arabinofuranosidase.\n",
      "EC EC:3.2.1.58 Glucan 1,3-beta-glucosidase.\n",
      "EC EC:3.2.1.74 Glucan 1,4-beta-glucosidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K05349 beta-glucosidase [EC:3.2.1.21]\n",
      "SSO SSO:000021425 Periplasmic beta-glucosidase\n",
      "b2132 ['Periplasmic beta-glucosidase (EC 3.2.1.21)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K03777 D-lactate dehydrogenase (quinone) [EC:1.1.5.12]\n",
      "SSO SSO:000002031 D-lactate dehydrogenase (EC 1.1.1.28)\n",
      "b2133 ['D-lactate dehydrogenase (EC 1.1.1.28)']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042493 response to drug\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K07262 serine-type D-Ala-D-Ala endopeptidase (penicillin-binding protein 7) [EC:3.4.21.-]\n",
      "SSO SSO:000005038 Murein-DD-endopeptidase (EC 3.4.99.-)\n",
      "SSO SSO:000035324 D-alanyl-D-alanine endopeptidase\n",
      "b2134 ['Murein-DD-endopeptidase (EC 3.4.99.-)']\n",
      "\n",
      "SSO SSO:000018423 Inner membrane protein YohC\n",
      "b2135 ['Inner membrane protein YohC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000018423 Inner membrane protein YohC\n",
      "b2136 ['Inner membrane protein YohC']\n",
      "\n",
      "SSO SSO:000027395 Uncharacterized oxidoreductase YohF\n",
      "b2137 ['Uncharacterized oxidoreductase YohF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K15550 outer membrane protein, multidrug efflux system\n",
      "SSO SSO:000023448 Putative multidrug resistance outer membrane protein MdtQ\n",
      "b2139 ['Putative multidrug resistance outer membrane protein MdtQ']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4679 ['hypothetical protein']\n",
      "\n",
      "KO KO:K05541 tRNA-dihydrouridine synthase C [EC:1.-.-.-]\n",
      "SSO SSO:000009397 tRNA-dihydrouridine(16) synthase\n",
      "b2140 ['tRNA-dihydrouridine(16) synthase']\n",
      "\n",
      "KO KO:K06518 holin-like protein\n",
      "SSO SSO:000026620 UPF0299 membrane protein YohJ\n",
      "b2141 ['UPF0299 membrane protein YohJ']\n",
      "\n",
      "SSO SSO:000018424 Inner membrane protein YohK\n",
      "b2142 ['Inner membrane protein YohK']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01489 cytidine deaminase [EC:3.5.4.5]\n",
      "SSO SSO:000001873 Cytidine deaminase (EC 3.5.4.5)\n",
      "SSO SSO:000029772 cytidine/deoxycytidine deaminase\n",
      "b2143 ['Cytidine deaminase (EC 3.5.4.5)']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03748 SanA protein\n",
      "SSO SSO:000024400 SanA protein\n",
      "b2144 ['SanA protein']\n",
      "\n",
      "SSO SSO:000028264 Uncharacterized protein YeiS\n",
      "b2145 ['Uncharacterized protein YeiS']\n",
      "\n",
      "KO KO:K17722 dihydropyrimidine dehydrogenase (NAD+) subunit PreT [EC:1.3.1.1]\n",
      "SSO SSO:000005186 NAD-dependent dihydropyrimidine dehydrogenase subunit PreT (EC 1.3.1.1)\n",
      "b2146 ['NAD-dependent dihydropyrimidine dehydrogenase subunit PreT (EC 1.3.1.1)']\n",
      "\n",
      "KO KO:K17723 dihydropyrimidine dehydrogenase (NAD+) subunit PreA [EC:1.3.1.1]\n",
      "SSO SSO:000005185 NAD-dependent dihydropyrimidine dehydrogenase subunit PreA (EC 1.3.1.1)\n",
      "b2147 ['NAD-dependent dihydropyrimidine dehydrogenase subunit PreA (EC 1.3.1.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10541 methyl-galactoside transport system permease protein\n",
      "SSO SSO:000036034 Galactose/methyl galactoside ABC transporter, permease protein MglC (EC 3.6.3.17)\n",
      "b2148 ['Galactose/methyl galactoside ABC transporter, permease protein MglC (EC 3.6.3.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10441 ribose transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10542 methyl-galactoside transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000036033 Galactose/methyl galactoside ABC transporter, ATP-binding protein MglA (EC 3.6.3.17)\n",
      "b2149 ['Galactose/methyl galactoside ABC transporter, ATP-binding protein MglA (EC 3.6.3.17)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10540 methyl-galactoside transport system substrate-binding protein\n",
      "SSO SSO:000036035 Galactose/methyl galactoside ABC transporter, substrate-binding protein MglB (EC 3.6.3.17)\n",
      "b2150 ['Galactose/methyl galactoside ABC transporter, substrate-binding protein MglB (EC 3.6.3.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "b2151 ['Mgl repressor and galactose ultrainduction factor GalS, HTH-type transcriptional regulator']\n",
      "\n",
      "KO KO:K07148 uncharacterized protein\n",
      "SSO SSO:000028263 Uncharacterized protein YeiB\n",
      "b2152 ['Uncharacterized protein YeiB']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K01495 GTP cyclohydrolase IA [EC:3.5.4.16]\n",
      "SSO SSO:000002983 GTP cyclohydrolase I type 1 (EC 3.5.4.16)\n",
      "SSO SSO:000017010 GTP cyclohydrolase I( EC:3.5.4.16 )\n",
      "b2153 ['GTP cyclohydrolase I (EC 3.5.4.16) type 1']\n",
      "\n",
      "EC EC:2.3.1.122 Trehalose O-mycolyltransferase.\n",
      "EC EC:2.3.1.20 Diacylglycerol O-acyltransferase.\n",
      "EC EC:3.1.1.1 Carboxylesterase.\n",
      "EC EC:3.1.1.72 Acetylxylan esterase.\n",
      "EC EC:3.1.1.73 Feruloyl esterase.\n",
      "EC EC:3.1.2.12 S-formylglutathione hydrolase.\n",
      "KO KO:K01070 S-formylglutathione hydrolase [EC:3.1.2.12]\n",
      "SSO SSO:000007202 S-formylglutathione hydrolase (EC 3.1.2.12)\n",
      "b2154 ['S-formylglutathione hydrolase (EC 3.1.2.12)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "KO KO:K16089 outer membrane receptor for ferrienterochelin and colicins\n",
      "SSO SSO:000012348 Colicin I receptor\n",
      "b2155 ['Colicin I receptor']\n",
      "\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11733 lysine-specific permease\n",
      "SSO SSO:000004603 Lysine-specific permease\n",
      "b2156 ['Lysine-specific permease']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K21900 LysR family transcriptional regulator, transcriptional activator of the cysJI operon\n",
      "SSO SSO:000008222 Transcriptional regulator YeiE, LysR family\n",
      "b2157 ['Transcriptional regulator YeiE, LysR family']\n",
      "\n",
      "SSO SSO:000026635 UPF0324 inner membrane protein YeiH\n",
      "b2158 ['UPF0324 inner membrane protein YeiH']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0009314 response to radiation\n",
      "KO KO:K01151 deoxyribonuclease IV [EC:3.1.21.2]\n",
      "SSO SSO:000002473 Endonuclease IV (EC 3.1.21.2)\n",
      "b2159 ['Endonuclease IV (EC 3.1.21.2)']\n",
      "\n",
      "KO KO:K16328 pseudouridine kinase [EC:2.7.1.83]\n",
      "SSO SSO:000023380 Putative kinase\n",
      "SSO SSO:000028702 Uncharacterized sugar kinase YeiI\n",
      "b2160 ['Uncharacterized sugar kinase YeiI']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16325 nucleoside permease\n",
      "SSO SSO:000005426 Nucleoside permease NupX\n",
      "SSO SSO:000020657 Nucleoside permease\n",
      "b2161 ['Nucleoside permease NupX']\n",
      "\n",
      "KO KO:K01239 purine nucleosidase [EC:3.2.2.1]\n",
      "KO KO:K10213 ribosylpyrimidine nucleosidase [EC:3.2.2.8]\n",
      "SSO SSO:000006828 Pyrimidine-specific ribonucleoside hydrolase RihB (EC 3.2.2.8)\n",
      "b2162 ['Pyrimidine-specific ribonucleoside hydrolase RihB (EC 3.2.2.8)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K16326 CRP/FNR family transcriptional regulator, putaive post-exponential-phase nitrogen-starvation regulator\n",
      "b2163 ['Regulatory protein YeiL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16324 putative pseudouridine transporter\n",
      "SSO SSO:000006569 Pseudouridine transporter PsuT (putative)\n",
      "b2164 ['Pseudouridine transporter PsuT (putative)']\n",
      "\n",
      "KO KO:K16329 pseudouridylate synthase [EC:4.2.1.70]\n",
      "SSO SSO:000006560 Pseudouridine 5'-phosphate glycosidase (EC 4.2.1.70)\n",
      "b2165 [\"Pseudouridine 5'-phosphate glycosidase (EC 4.2.1.70)\"]\n",
      "\n",
      "KO KO:K16328 pseudouridine kinase [EC:2.7.1.83]\n",
      "SSO SSO:000006561 Pseudouridine kinase (EC 2.7.1.83)\n",
      "b2166 ['Pseudouridine kinase (EC 2.7.1.83)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02770 PTS system, fructose-specific IIC component\n",
      "b2167 ['PTS system, inactive IIB catalytic domain', 'PTS system, fructose-specific IIB component (EC 2.7.1.202)', 'PTS system, fructose-specific IIC component']\n",
      "\n",
      "GO GO:0006096 glycolytic process\n",
      "KO KO:K00882 1-phosphofructokinase [EC:2.7.1.56]\n",
      "SSO SSO:000000049 1-phosphofructokinase (EC 2.7.1.56)\n",
      "b2168 ['1-phosphofructokinase (EC 2.7.1.56)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02784 phosphocarrier protein HPr\n",
      "KO KO:K11183 phosphocarrier protein FPr\n",
      "b2169 ['PTS system, fructose-specific IIA component (EC 2.7.1.202)', 'Fructose-specific phosphocarrier protein HPr']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03291 MFS transporter, SET family, sugar efflux transporter\n",
      "SSO SSO:000025235 Sugar efflux transporter B\n",
      "b2170 ['Sugar efflux transporter B']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06940 uncharacterized protein\n",
      "SSO SSO:000039914 UPF0153 protein YeiW\n",
      "b4502 ['UPF0153 protein YeiW']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02356 elongation factor P\n",
      "SSO SSO:000002464 Elongation factor P-like protein\n",
      "b2171 ['Elongation factor P-like protein']\n",
      "\n",
      "KO KO:K00040 fructuronate reductase [EC:1.1.1.57]\n",
      "SSO SSO:000027387 Uncharacterized oxidoreductase YeiQ\n",
      "b2172 ['Uncharacterized oxidoreductase YeiQ']\n",
      "\n",
      "SSO SSO:000041752 Zinc-binding GTPase YeiR\n",
      "b2173 ['Zinc-binding GTPase YeiR']\n",
      "\n",
      "KO KO:K19803 Kdo2-lipid A phosphotransferase [EC:2.7.4.29]\n",
      "SSO SSO:000018950 Lipid A 1-diphosphate synthase\n",
      "b2174 ['Lipid A 1-diphosphate synthase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K13694 murein DD-endopeptidase / murein LD-carboxypeptidase [EC:3.4.-.- 3.4.17.13]\n",
      "b2175 ['Murein DD-endopeptidase MepS', 'Murein LD-carboxypeptidase (EC 3.4.17.13)']\n",
      "\n",
      "KO KO:K21090 c-di-GMP phosphodiesterase [EC:3.1.4.52]\n",
      "b2176 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeN']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K13893 microcin C transport system substrate-binding protein\n",
      "SSO SSO:000000503 ABC transporter, substrate-binding protein YejA\n",
      "b2177 ['ABC transporter, substrate-binding protein YejA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02033 peptide/nickel transport system permease protein\n",
      "KO KO:K13894 microcin C transport system permease protein\n",
      "SSO SSO:000000499 ABC transporter, permease protein YejB\n",
      "b2178 ['ABC transporter, permease protein YejB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02034 peptide/nickel transport system permease protein\n",
      "KO KO:K13895 microcin C transport system permease protein\n",
      "SSO SSO:000000500 ABC transporter, permease protein YejE\n",
      "b2179 ['ABC transporter, permease protein YejE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02031 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K02032 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K13896 microcin C transport system ATP-binding protein\n",
      "SSO SSO:000000495 ABC transporter, ATP-binding protein YejF\n",
      "b2180 ['ABC transporter, ATP-binding protein YejF']\n",
      "\n",
      "SSO SSO:000028265 Uncharacterized protein YejG\n",
      "b2181 ['Uncharacterized protein YejG']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K07552 MFS transporter, DHA1 family, multidrug resistance protein\n",
      "b2182 ['Multidrug resistance transporter => Bicyclomycin resistance protein Bcr-1']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K06183 16S rRNA pseudouridine516 synthase [EC:5.4.99.19]\n",
      "SSO SSO:000039105 SSU rRNA pseudouridine(516) synthase (EC 5.4.99.19)\n",
      "b2183 ['SSU rRNA pseudouridine(516) synthase (EC 5.4.99.19)']\n",
      "\n",
      "KO KO:K19789 DNA repair protein RadD\n",
      "SSO SSO:000000606 ATP-dependent RNA helicase YejH\n",
      "b2184 ['ATP-dependent RNA helicase YejH']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02897 large subunit ribosomal protein L25\n",
      "SSO SSO:000004259 LSU ribosomal protein L25p\n",
      "b2185 ['LSU ribosomal protein L25p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06899 nucleoid-associated protein\n",
      "SSO SSO:000005418 Nucleoid-associated protein NdpA\n",
      "b2186 ['Nucleoid-associated protein NdpA']\n",
      "\n",
      "KO KO:K09904 uncharacterized protein\n",
      "SSO SSO:000026652 UPF0352 protein YejL\n",
      "b2187 ['UPF0352 protein YejL']\n",
      "\n",
      "KO KO:K07014 uncharacterized protein\n",
      "SSO SSO:000018395 Inner membrane protein YejM, alkaline phosphatase superfamily\n",
      "b2188 ['Inner membrane protein YejM, alkaline phosphatase superfamily']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000027378 Uncharacterized outer membrane protein YejO\n",
      "b2190 ['Uncharacterized outer membrane protein YejO']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b2192 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K07685 two-component system, NarL family, nitrate/nitrite response regulator NarP\n",
      "SSO SSO:000020538 Nitrate/nitrite response regulator protein NarP\n",
      "b2193 ['Nitrate/nitrite response regulator protein NarP']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02200 cytochrome c-type biogenesis protein CcmH\n",
      "b2194 ['Cytochrome c heme lyase subunit CcmL', 'Cytochrome c heme lyase subunit CcmH']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02199 cytochrome c biogenesis protein CcmG, thiol:disulfide interchange protein DsbE\n",
      "SSO SSO:000001923 Cytochrome c-type biogenesis protein CcmG/DsbE, thiol:disulfide oxidoreductase\n",
      "b2195 ['Cytochrome c-type biogenesis protein CcmG/DsbE, thiol:disulfide oxidoreductase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02198 cytochrome c-type biogenesis protein CcmF\n",
      "SSO SSO:000001903 Cytochrome c heme lyase subunit CcmF\n",
      "b2196 ['Cytochrome c heme lyase subunit CcmF']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02197 cytochrome c-type biogenesis protein CcmE\n",
      "SSO SSO:000001922 Cytochrome c-type biogenesis protein CcmE, heme chaperone\n",
      "b2197 ['Cytochrome c-type biogenesis protein CcmE, heme chaperone']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02196 heme exporter protein D\n",
      "SSO SSO:000001921 Cytochrome c-type biogenesis protein CcmD, interacts with CcmCE\n",
      "b2198 ['Cytochrome c-type biogenesis protein CcmD, interacts with CcmCE']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02195 heme exporter protein C\n",
      "SSO SSO:000001920 Cytochrome c-type biogenesis protein CcmC, putative heme lyase for CcmE\n",
      "b2199 ['Cytochrome c-type biogenesis protein CcmC, putative heme lyase for CcmE']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02194 heme exporter protein B\n",
      "SSO SSO:000000489 ABC transporter involved in cytochrome c biogenesis, CcmB subunit\n",
      "b2200 ['ABC transporter involved in cytochrome c biogenesis, CcmB subunit']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K02193 heme exporter protein A [EC:3.6.3.41]\n",
      "SSO SSO:000000488 ABC transporter involved in cytochrome c biogenesis, ATPase component CcmA\n",
      "b2201 ['ABC transporter involved in cytochrome c biogenesis, ATPase component CcmA']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02569 cytochrome c-type protein NapC\n",
      "SSO SSO:000001929 Cytochrome c-type protein NapC\n",
      "b2202 ['Cytochrome c-type protein NapC']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02568 cytochrome c-type protein NapB\n",
      "SSO SSO:000005330 Nitrate reductase cytochrome c550-type subunit\n",
      "b2203 ['Nitrate reductase cytochrome c550-type subunit']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02574 ferredoxin-type protein NapH\n",
      "SSO SSO:000006143 Polyferredoxin NapH (periplasmic nitrate reductase)\n",
      "b2204 ['Polyferredoxin NapH (periplasmic nitrate reductase)']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02573 ferredoxin-type protein NapG\n",
      "SSO SSO:000002716 Ferredoxin-type protein NapG (periplasmic nitrate reductase)\n",
      "b2205 ['Ferredoxin-type protein NapG (periplasmic nitrate reductase)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02567 periplasmic nitrate reductase NapA [EC:1.7.99.-]\n",
      "b2206 ['Periplasmic nitrate reductase (EC 1.7.99.4)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02570 periplasmic nitrate reductase NapD\n",
      "SSO SSO:000005843 Periplasmic nitrate reductase component NapD\n",
      "b2207 ['Periplasmic nitrate reductase component NapD']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02572 ferredoxin-type protein NapF\n",
      "SSO SSO:000002715 Ferredoxin-type protein NapF (periplasmic nitrate reductase)\n",
      "b2208 ['Ferredoxin-type protein NapF (periplasmic nitrate reductase)']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K08276 ecotin\n",
      "b2209 ['Proteinase inhibitor I11, ecotin precursor']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "KO KO:K00116 malate dehydrogenase (quinone) [EC:1.1.5.4]\n",
      "SSO SSO:000004677 Malate:quinone oxidoreductase (EC 1.1.5.4)\n",
      "b2210 ['Malate:quinone oxidoreductase (EC 1.1.5.4)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06159 multidrug/microcin transport system ATP-binding/permease protein\n",
      "SSO SSO:000000518 ABC-type siderophore export system, fused ATPase and permease components\n",
      "b2211 ['ABC-type siderophore export system, fused ATPase and permease components']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03919 DNA oxidative demethylase [EC:1.14.11.33]\n",
      "SSO SSO:000010503 Alpha-ketoglutarate-dependent dioxygenase AlkB (EC 1.14.11.33)\n",
      "b2212 ['Alpha-ketoglutarate-dependent dioxygenase AlkB (EC 1.14.11.33)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K00567 methylated-DNA-[protein]-cysteine S-methyltransferase [EC:2.1.1.63]\n",
      "KO KO:K10778 AraC family transcriptional regulator, regulatory protein of adaptative response / methylated-DNA-[protein]-cysteine methyltransferase [EC:2.1.1.63]\n",
      "b2213 ['Methylphosphotriester-DNA--protein-cysteine S-methyltransferase (EC 2.1.1.n11)', 'ADA regulatory protein', 'Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K03734 FAD:protein FMN transferase [EC:2.7.1.180]\n",
      "SSO SSO:000013628 FAD:protein FMN transferase (EC 2.7.1.180)\n",
      "b2214 ['FAD:protein FMN transferase (EC 2.7.1.180)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K09475 outer membrane pore protein C\n",
      "SSO SSO:000005574 Outer membrane porin OmpC\n",
      "b2215 ['Outer membrane porin OmpC']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K07676 two-component system, NarL family, sensor histidine kinase RcsD [EC:2.7.13.3]\n",
      "SSO SSO:000006054 Phosphotransferase RcsD\n",
      "b2216 ['Phosphotransferase RcsD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K07687 two-component system, NarL family, captular synthesis response regulator RcsB\n",
      "SSO SSO:000002132 DNA-binding capsular synthesis response regulator RcsB\n",
      "b2217 ['DNA-binding capsular synthesis response regulator RcsB']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009242 colanic acid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K07677 two-component system, NarL family, capsular synthesis sensor histidine kinase RcsC [EC:2.7.13.3]\n",
      "SSO SSO:000007394 Sensor histidine kinase RcsC (EC 2.7.13.3)\n",
      "b2218 ['Sensor histidine kinase RcsC (EC 2.7.13.3)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02482 two-component system, NtrC family, sensor kinase [EC:2.7.13.3]\n",
      "KO KO:K07710 two-component system, NtrC family, sensor histidine kinase AtoS [EC:2.7.13.3]\n",
      "SSO SSO:000007527 Signal transduction histidine-protein kinase AtoS ## response regulator AtoC (EC 2.7.13.3)\n",
      "b2219 ['Signal transduction histidine-protein kinase AtoS (EC 2.7.13.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K02481 two-component system, NtrC family, response regulator\n",
      "KO KO:K07713 two-component system, NtrC family, response regulator HydG\n",
      "KO KO:K07714 two-component system, NtrC family, response regulator AtoC\n",
      "SSO SSO:000000674 Acetoacetate metabolism regulatory protein AtoC\n",
      "b2220 ['Acetoacetate metabolism regulatory protein AtoC']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01034 acetate CoA/acetoacetate CoA-transferase alpha subunit [EC:2.8.3.8 2.8.3.9]\n",
      "SSO SSO:000010161 Acetyl-CoA:acetoacetyl-CoA transferase alpha subunit (EC 2.8.3.-)\n",
      "SSO SSO:000034473 Acyl-CoA:acetate CoA-transferase, alpha subunit (EC 2.8.3.8)\n",
      "b2221 ['Acyl-CoA:acetate CoA-transferase, alpha subunit (EC 2.8.3.8)']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01035 acetate CoA/acetoacetate CoA-transferase beta subunit [EC:2.8.3.8 2.8.3.9]\n",
      "SSO SSO:000010162 Acetyl-CoA:acetoacetyl-CoA transferase, beta subunit (EC 2.8.3.8)\n",
      "SSO SSO:000034474 Acyl-CoA:acetate CoA-transferase, beta subunit (EC 2.8.3.8)\n",
      "b2222 ['Acyl-CoA:acetate CoA-transferase, beta subunit (EC 2.8.3.8)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02106 short-chain fatty acids transporter\n",
      "SSO SSO:000024621 Short chain fatty acids transporter\n",
      "b2223 ['Short-chain fatty acids transporter']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K00626 acetyl-CoA C-acetyltransferase [EC:2.3.1.9]\n",
      "SSO SSO:000000702 Acetyl-CoA acetyltransferase (EC 2.3.1.9)\n",
      "b2224 ['Acetyl-CoA acetyltransferase (EC 2.3.1.9)']\n",
      "\n",
      "SSO SSO:000028273 Uncharacterized protein YfaP\n",
      "b2225 ['Uncharacterized protein YfaP']\n",
      "\n",
      "SSO SSO:000028274 Uncharacterized protein YfaQ\n",
      "b2226 ['Uncharacterized protein YfaQ']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4500 ['hypothetical protein']\n",
      "\n",
      "KO KO:K09934 uncharacterized protein\n",
      "SSO SSO:000028275 Uncharacterized protein YfaT\n",
      "b2229 ['Uncharacterized protein YfaT']\n",
      "\n",
      "SSO SSO:000028272 Uncharacterized protein YfaA\n",
      "b2230 ['Uncharacterized protein YfaA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K02469 DNA gyrase subunit A [EC:5.99.1.3]\n",
      "SSO SSO:000002067 DNA gyrase subunit A (EC 5.99.1.3)\n",
      "b2231 ['DNA gyrase subunit A (EC 5.99.1.3)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00568 2-polyprenyl-6-hydroxyphenyl methylase / 3-demethylubiquinone-9 3-methyltransferase [EC:2.1.1.222 2.1.1.64]\n",
      "b2232 ['3-demethylubiquinol 3-O-methyltransferase (EC 2.1.1.64)', '2-polyprenyl-6-hydroxyphenyl methylase (EC 2.1.1.222)']\n",
      "\n",
      "KO KO:K07279 autotransporter family porin\n",
      "SSO SSO:000010324 Adhesin\n",
      "b2233 ['Uncharacterized protein YfaL']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00525 ribonucleoside-diphosphate reductase alpha chain [EC:1.17.4.1]\n",
      "SSO SSO:000007102 Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)\n",
      "b2234 ['Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00526 ribonucleoside-diphosphate reductase beta chain [EC:1.17.4.1]\n",
      "SSO SSO:000007103 Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)\n",
      "b2235 ['Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)']\n",
      "\n",
      "KO KO:K11107 ferredoxin\n",
      "SSO SSO:000029134 YfaE: ferredoxin involved with ribonucleotide reductase diferric-tyrosyl radical (Y*) cofactor maintenance\n",
      "b2236 ['YfaE: ferredoxin involved with ribonucleotide reductase diferric-tyrosyl radical (Y*) cofactor maintenance']\n",
      "\n",
      "SSO SSO:000029012 Weak-acid-inducible locus InaA\n",
      "b2237 ['Weak-acid-inducible locus InaA']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2238 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01126 glycerophosphoryl diester phosphodiesterase [EC:3.1.4.46]\n",
      "SSO SSO:000034575 Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46) periplasmic (secreted in GramPositives)\n",
      "b2239 ['Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46) periplasmic (secreted in GramPositives)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02445 MFS transporter, OPA family, glycerol-3-phosphate transporter\n",
      "SSO SSO:000017332 Glycerol-3-phosphate transporter\n",
      "b2240 ['Glycerol-3-phosphate transporter']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00111 glycerol-3-phosphate dehydrogenase [EC:1.1.5.3]\n",
      "SSO SSO:000000980 Anaerobic glycerol-3-phosphate dehydrogenase subunit A (EC 1.1.5.3)\n",
      "b2241 ['Anaerobic glycerol-3-phosphate dehydrogenase subunit A (EC 1.1.5.3)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00112 glycerol-3-phosphate dehydrogenase subunit B [EC:1.1.5.3]\n",
      "SSO SSO:000000981 Anaerobic glycerol-3-phosphate dehydrogenase subunit B (EC 1.1.5.3)\n",
      "b2242 ['Anaerobic glycerol-3-phosphate dehydrogenase subunit B (EC 1.1.5.3)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00113 glycerol-3-phosphate dehydrogenase subunit C [EC:1.1.5.3]\n",
      "SSO SSO:000000982 Anaerobic glycerol-3-phosphate dehydrogenase subunit C (EC 1.1.5.3)\n",
      "b2243 ['Anaerobic glycerol-3-phosphate dehydrogenase subunit C (EC 1.1.5.3)']\n",
      "\n",
      "KO KO:K24090 Unknown\n",
      "SSO SSO:000008765 Uncharacterized protein YfaD\n",
      "b2244 ['Uncharacterized protein YfaD']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4543 ['hypothetical protein']\n",
      "\n",
      "KO KO:K02510 4-hydroxy-2-oxoheptanedioate aldolase [EC:4.1.2.52]\n",
      "KO KO:K12660 2-dehydro-3-deoxy-L-rhamnonate aldolase [EC:4.1.2.53]\n",
      "SSO SSO:000000174 2-keto-3-deoxy-L-rhamnonate aldolase (EC 4.1.2.53)\n",
      "b2245 ['2-keto-3-deoxy-L-rhamnonate aldolase (EC 4.1.2.53)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000004166 L-rhamnonate transporter (predicted by genome context)\n",
      "b2246 ['L-rhamnonate transporter (predicted by genome context)']\n",
      "\n",
      "KO KO:K12661 L-rhamnonate dehydratase [EC:4.2.1.90]\n",
      "SSO SSO:000004165 L-rhamnonate dehydratase (EC 4.2.1.90)\n",
      "b2247 ['L-rhamnonate dehydratase (EC 4.2.1.90)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000004167 L-rhamnonate utilization transcriptional regulator (predicted by genome context)\n",
      "b2248 ['L-rhamnonate utilization transcriptional regulator (predicted by genome context)']\n",
      "\n",
      "KO KO:K03742 nicotinamide-nucleotide amidase [EC:3.5.1.42]\n",
      "b2249 ['ADP-ribose pyrophosphatase of COG1058 family (EC 3.6.1.13)', 'Nicotinamide-nucleotide amidase paralog YfaY, no functional activity']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000008738 Uncharacterized outer membrane protein YfaZ\n",
      "b2250 ['Uncharacterized outer membrane protein YfaZ']\n",
      "\n",
      "KO KO:K12944 nucleoside triphosphatase [EC:3.6.1.-]\n",
      "SSO SSO:000006823 Pyrimidine deoxynucleoside triphosphate (dYTP) pyrophosphohydrolase NudI\n",
      "SSO SSO:000042990 nucleoside-triphosphatase (EC 3.6.1.15)\n",
      "b2251 ['Pyrimidine deoxynucleoside triphosphate (dYTP) pyrophosphohydrolase NudI']\n",
      "\n",
      "b2252 ['Lipopolysaccharide core heptose(II)-phosphate phosphatase', 'Polymyxin resistance protein PmrG']\n",
      "\n",
      "KO KO:K07806 UDP-4-amino-4-deoxy-L-arabinose-oxoglutarate aminotransferase [EC:2.6.1.87]\n",
      "SSO SSO:000008608 UDP-4-amino-4-deoxy-L-arabinose--oxoglutarate aminotransferase (EC 2.6.1.-)\n",
      "b2253 ['UDP-4-amino-4-deoxy-L-arabinose--oxoglutarate aminotransferase (EC 2.6.1.87)']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "KO KO:K10012 undecaprenyl-phosphate 4-deoxy-4-formamido-L-arabinose transferase [EC:2.4.2.53]\n",
      "SSO SSO:000008819 Undecaprenyl-phosphate 4-deoxy-4-formamido-L-arabinose transferase (EC 2.4.2.53)\n",
      "b2254 ['Undecaprenyl-phosphate 4-deoxy-4-formamido-L-arabinose transferase (EC 2.4.2.53)']\n",
      "\n",
      "KO KO:K10011 UDP-4-amino-4-deoxy-L-arabinose formyltransferase / UDP-glucuronic acid dehydrogenase (UDP-4-keto-hexauronic acid decarboxylating) [EC:2.1.2.13 1.1.1.305]\n",
      "b2255 ['UDP-4-amino-4-deoxy-L-arabinose formyltransferase (EC 2.1.2.13)', 'UDP-glucuronic acid oxidase (UDP-4-keto-hexauronic acid decarboxylating) (EC 1.1.1.305)']\n",
      "\n",
      "KO KO:K13014 undecaprenyl phosphate-alpha-L-ara4FN deformylase [EC:3.5.1.-]\n",
      "SSO SSO:000000358 4-deoxy-4-formamido-L-arabinose-phosphoundecaprenol deformylase ArnD (EC 3.5.1.n3)\n",
      "b2256 ['4-deoxy-4-formamido-L-arabinose-phosphoundecaprenol deformylase ArnD (EC 3.5.1.n3)']\n",
      "\n",
      "EC EC:2.4.2.43 Lipid IV(A) 4-amino-4-deoxy-L-arabinosyltransferase.\n",
      "GO GO:0006950 response to stress\n",
      "KO KO:K07264 4-amino-4-deoxy-L-arabinose transferase [EC:2.4.2.43]\n",
      "SSO SSO:000009664 4-amino-4-deoxy-L-arabinose transferase\n",
      "b2257 ['Undecaprenyl phosphate-alpha-4-amino-4-deoxy-L-arabinose arabinosyl transferase (EC 2.4.2.43)', 'Polymyxin resistance protein PmrK']\n",
      "\n",
      "GO GO:0016021 integral component of membrane\n",
      "KO KO:K12962 undecaprenyl phosphate-alpha-L-ara4N flippase subunit ArnE\n",
      "SSO SSO:000008815 Undecaprenyl phosphate-aminoarabinose flippase subunit ArnE\n",
      "b4544 ['Undecaprenyl phosphate-aminoarabinose flippase subunit ArnE']\n",
      "\n",
      "KO KO:K12963 undecaprenyl phosphate-alpha-L-ara4N flippase subunit ArnF\n",
      "SSO SSO:000008816 Undecaprenyl phosphate-aminoarabinose flippase subunit ArnF\n",
      "b2258 ['Undecaprenyl phosphate-aminoarabinose flippase subunit ArnF']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K19238 signal transduction protein PmrD\n",
      "SSO SSO:000039168 Signal transduction protein PmrD, part of PhoPQ-PmrD-PmrAB multicomponent regulatory system\n",
      "b2259 ['Signal transduction protein PmrD, part of PhoPQ-PmrD-PmrAB multicomponent regulatory system']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K01911 O-succinylbenzoic acid---CoA ligase [EC:6.2.1.26]\n",
      "SSO SSO:000005474 O-succinylbenzoic acid--CoA ligase (EC 6.2.1.26)\n",
      "SSO SSO:000043193 o-succinylbenzoate---CoA ligase (EC 6.2.1.26)\n",
      "b2260 ['O-succinylbenzoic acid--CoA ligase (EC 6.2.1.26)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02549 O-succinylbenzoate synthase [EC:4.2.1.113]\n",
      "SSO SSO:000005473 O-succinylbenzoate synthase (EC 4.2.1.113)\n",
      "b2261 ['O-succinylbenzoate synthase (EC 4.2.1.113)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K01661 naphthoate synthase [EC:4.1.3.36]\n",
      "SSO SSO:000005287 Naphthoate synthase (EC 4.1.3.36)\n",
      "b2262 ['Naphthoate synthase (EC 4.1.3.36)']\n",
      "\n",
      "KO KO:K08680 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase [EC:4.2.99.20]\n",
      "SSO SSO:000000219 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase (EC 4.2.99.20)\n",
      "b2263 ['2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase (EC 4.2.99.20)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02551 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylate synthase [EC:2.2.1.9]\n",
      "SSO SSO:000000218 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acid synthase (EC 2.2.1.9)\n",
      "b2264 ['2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acid synthase (EC 2.2.1.9)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02552 menaquinone-specific isochorismate synthase [EC:5.4.4.2]\n",
      "b2265 ['Isochorismate synthase (EC 5.4.4.2)', 'Menaquinone-specific isochorismate synthase (EC 5.4.4.2)']\n",
      "\n",
      "KO KO:K05594 ElaB protein\n",
      "SSO SSO:000013339 ElaB protein\n",
      "b2266 ['ElaB protein']\n",
      "\n",
      "KO KO:K02348 ElaA protein\n",
      "SSO SSO:000020215 N-acetyltransferase ElaA\n",
      "b2267 ['N-acetyltransferase ElaA']\n",
      "\n",
      "KO KO:K00784 ribonuclease Z [EC:3.1.26.11]\n",
      "SSO SSO:000007097 Ribonuclease Z (EC 3.1.26.11)\n",
      "b2268 ['Ribonuclease Z (EC 3.1.26.11)']\n",
      "\n",
      "KO KO:K18015 deubiquitinase [EC:3.4.22.-]\n",
      "SSO SSO:000002265 Deubiquitinating protease ElaD\n",
      "b2269 ['Deubiquitinating protease ElaD']\n",
      "\n",
      "KO KO:K07114 Ca-activated chloride channel homolog\n",
      "SSO SSO:000023724 Putative von Willebrand factor, vWF type A domain protein STM2315\n",
      "b2270 ['Putative von Willebrand factor, vWF type A domain protein STM2315']\n",
      "\n",
      "SSO SSO:000028276 Uncharacterized protein YfbL\n",
      "b2271 ['Uncharacterized protein YfbL']\n",
      "\n",
      "SSO SSO:000028277 Uncharacterized protein YfbM\n",
      "b2272 ['Uncharacterized protein YfbM']\n",
      "\n",
      "SSO SSO:000028278 Uncharacterized protein YfbN\n",
      "b2273 ['Uncharacterized protein YfbN']\n",
      "\n",
      "SSO SSO:000028279 Uncharacterized protein YfbO\n",
      "b2274 ['Uncharacterized protein YfbO']\n",
      "\n",
      "SSO SSO:000028280 Uncharacterized protein YfbP\n",
      "b2275 ['Uncharacterized protein YfbP']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00343 NADH-quinone oxidoreductase subunit N [EC:7.1.1.2]\n",
      "SSO SSO:000005236 NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)\n",
      "b2276 ['NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00342 NADH-quinone oxidoreductase subunit M [EC:7.1.1.2]\n",
      "SSO SSO:000005235 NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)\n",
      "b2277 ['NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00341 NADH-quinone oxidoreductase subunit L [EC:7.1.1.2]\n",
      "SSO SSO:000005234 NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)\n",
      "b2278 ['NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00340 NADH-quinone oxidoreductase subunit K [EC:7.1.1.2]\n",
      "SSO SSO:000005233 NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)\n",
      "b2279 ['NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00339 NADH-quinone oxidoreductase subunit J [EC:7.1.1.2]\n",
      "SSO SSO:000005232 NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)\n",
      "b2280 ['NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00338 NADH-quinone oxidoreductase subunit I [EC:7.1.1.2]\n",
      "SSO SSO:000005231 NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)\n",
      "b2281 ['NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00337 NADH-quinone oxidoreductase subunit H [EC:7.1.1.2]\n",
      "SSO SSO:000005230 NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)\n",
      "b2282 ['NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00336 NADH-quinone oxidoreductase subunit G [EC:7.1.1.2]\n",
      "SSO SSO:000005228 NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)\n",
      "b2283 ['NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00335 NADH-quinone oxidoreductase subunit F [EC:7.1.1.2]\n",
      "SSO SSO:000005227 NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)\n",
      "b2284 ['NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00334 NADH-quinone oxidoreductase subunit E [EC:7.1.1.2]\n",
      "SSO SSO:000005226 NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)\n",
      "b2285 ['NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K13378 NADH-quinone oxidoreductase subunit C/D [EC:7.1.1.2]\n",
      "b2286 ['NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)', 'NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00331 NADH-quinone oxidoreductase subunit B [EC:7.1.1.2]\n",
      "SSO SSO:000005222 NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)\n",
      "b2287 ['NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00330 NADH-quinone oxidoreductase subunit A [EC:7.1.1.2]\n",
      "SSO SSO:000005215 NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)\n",
      "b2288 ['NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000004590 LysR family transcriptional regulator lrhA\n",
      "b2289 ['LysR family transcriptional regulator LrhA']\n",
      "\n",
      "KO KO:K14260 alanine-synthesizing transaminase [EC:2.6.1.66 2.6.1.2]\n",
      "SSO SSO:000010359 Alanine transaminase (EC 2.6.1.2)\n",
      "b2290 ['Alanine transaminase (EC 2.6.1.2)']\n",
      "\n",
      "KO KO:K08722 5'-deoxynucleotidase [EC:3.1.3.89]\n",
      "SSO SSO:000000399 5'-nucleotidase (EC 3.1.3.5)\n",
      "SSO SSO:000009715 5'-deoxynucleotidase YfbR (EC 3.1.3.89)\n",
      "b2291 [\"5'-deoxynucleotidase YfbR (EC 3.1.3.89)\"]\n",
      "\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000028793 Uncharacterized transporter YfbS\n",
      "b2292 ['Uncharacterized transporter YfbS']\n",
      "\n",
      "KO KO:K19270 sugar-phosphatase [EC:3.1.3.23]\n",
      "SSO SSO:000036271 Hexitol phosphatase HxpA (EC 3.1.3.22) (EC 3.1.3.23) (EC 3.1.3.50)\n",
      "b2293 ['Hexitol phosphatase HxpA (EC 3.1.3.22) (EC 3.1.3.23) (EC 3.1.3.50)']\n",
      "\n",
      "KO KO:K09161 uncharacterized protein\n",
      "SSO SSO:000026622 UPF0304 protein yfbU\n",
      "b2294 ['UPF0304 protein yfbU']\n",
      "\n",
      "KO KO:K09899 uncharacterized protein\n",
      "SSO SSO:000026576 UPF0208 membrane protein YfbV\n",
      "b2295 ['UPF0208 membrane protein YfbV']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042867 pyruvate catabolic process\n",
      "GO GO:0045733 acetate catabolic process\n",
      "KO KO:K00925 acetate kinase [EC:2.7.2.1]\n",
      "SSO SSO:000000670 Acetate kinase (EC 2.7.2.1)\n",
      "b2296 ['Acetate kinase (EC 2.7.2.1)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042867 pyruvate catabolic process\n",
      "GO GO:0045733 acetate catabolic process\n",
      "KO KO:K13788 phosphate acetyltransferase [EC:2.3.1.8]\n",
      "SSO SSO:000005916 Phosphate acetyltransferase (EC 2.3.1.8)\n",
      "b2297 ['BioD-like N-terminal domain', 'Phosphate acetyltransferase (EC 2.3.1.8)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000028281 Uncharacterized protein YfcC\n",
      "b2298 ['Uncharacterized protein YfcC']\n",
      "\n",
      "SSO SSO:000026978 Uncharacterized Nudix hydrolase YfcD\n",
      "SSO SSO:000032288 putative NUDIX hydrolase\n",
      "b2299 ['Uncharacterized Nudix hydrolase YfcD']\n",
      "\n",
      "KO KO:K07095 uncharacterized protein\n",
      "SSO SSO:000021636 Phosphodiesterase YfcE\n",
      "b2300 ['Phosphodiesterase YfcE']\n",
      "\n",
      "KO KO:K00799 glutathione S-transferase [EC:2.5.1.18]\n",
      "SSO SSO:000003209 Glutathione S-transferase (EC 2.5.1.18)\n",
      "SSO SSO:000006328 Probable glutathione S-transferase , YfcF homolog (EC 2.5.1.18)\n",
      "b2301 ['Probable glutathione S-transferase (EC 2.5.1.18), YfcF homolog']\n",
      "\n",
      "KO KO:K00799 glutathione S-transferase [EC:2.5.1.18]\n",
      "KO KO:K11209 GSH-dependent disulfide-bond oxidoreductase [EC:1.8.4.-]\n",
      "SSO SSO:000006329 Probable glutathione S-transferase , YfcG homolog (EC 2.5.1.18)\n",
      "b2302 ['Probable glutathione S-transferase (EC 2.5.1.18), YfcG homolog']\n",
      "\n",
      "KO KO:K07589 D-erythro-7,8-dihydroneopterin triphosphate epimerase [EC:5.1.99.7]\n",
      "SSO SSO:000002016 D-erythro-7,8-dihydroneopterin triphosphate epimerase\n",
      "b2303 ['D-erythro-7,8-dihydroneopterin triphosphate epimerase']\n",
      "\n",
      "KO KO:K07071 uncharacterized protein\n",
      "SSO SSO:000012028 Cell division inhibitor Slr1223 (YfcH in EC), contains epimerase/dehydratase and DUF1731 domains\n",
      "b2304 ['Cell division inhibitor Slr1223 (YfcH in EC), contains epimerase/dehydratase and DUF1731 domains']\n",
      "\n",
      "KO KO:K24087 Unknown\n",
      "b2305 ['Uncharacterized protein YfcI']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K02028 polar amino acid transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10017 histidine transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000017801 Histidine ABC transporter, ATP-binding protein HisP (TC 3.A.1.3.1)\n",
      "b2306 ['Histidine ABC transporter, ATP-binding protein HisP (TC 3.A.1.3.1)']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02029 polar amino acid transport system permease protein\n",
      "KO KO:K10015 histidine transport system permease protein\n",
      "SSO SSO:000017803 Histidine ABC transporter, permease protein HisM (TC 3.A.1.3.1)\n",
      "b2307 ['Histidine ABC transporter, permease protein HisM (TC 3.A.1.3.1)']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10016 histidine transport system permease protein\n",
      "SSO SSO:000017804 Histidine ABC transporter, permease protein HisQ (TC 3.A.1.3.1)\n",
      "b2308 ['Histidine ABC transporter, permease protein HisQ (TC 3.A.1.3.1)']\n",
      "\n",
      "GO GO:0000105 histidine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02030 polar amino acid transport system substrate-binding protein\n",
      "KO KO:K10013 lysine/arginine/ornithine transport system substrate-binding protein\n",
      "KO KO:K10014 histidine transport system substrate-binding protein\n",
      "SSO SSO:000036284 Histidine ABC transporter, substrate-binding protein HisJ (TC 3.A.1.3.1)\n",
      "b2309 ['Histidine ABC transporter, substrate-binding protein HisJ (TC 3.A.1.3.1)']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0015597 obsolete histidine/arginine/lysine/ornithine porter activity\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02030 polar amino acid transport system substrate-binding protein\n",
      "KO KO:K10013 lysine/arginine/ornithine transport system substrate-binding protein\n",
      "SSO SSO:000036705 Lysine-arginine-ornithine-binding ABC transporter, substrate-binding protein ArgT (TC 3.A.1.3.1)\n",
      "b2310 ['Lysine-arginine-ornithine-binding ABC transporter, substrate-binding protein ArgT (TC 3.A.1.3.1)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K03186 flavin prenyltransferase [EC:2.5.1.129]\n",
      "SSO SSO:000002845 Flavin prenyltransferase UbiX\n",
      "b2311 ['Flavin prenyltransferase UbiX']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K00764 amidophosphoribosyltransferase [EC:2.4.2.14]\n",
      "SSO SSO:000000945 Amidophosphoribosyltransferase (EC 2.4.2.14)\n",
      "b2312 ['Amidophosphoribosyltransferase (EC 2.4.2.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03558 membrane protein required for colicin V production\n",
      "SSO SSO:000001680 Colicin V production protein\n",
      "b2313 ['Colicin V production protein']\n",
      "\n",
      "KO KO:K03749 DedD protein\n",
      "SSO SSO:000007419 Septum-associated cell division protein DedD\n",
      "b2314 ['Septum-associated cell division protein DedD']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K11754 dihydrofolate synthase / folylpolyglutamate synthase [EC:6.3.2.12 6.3.2.17]\n",
      "b2315 ['Dihydrofolate synthase (EC 6.3.2.12)', 'Folylpolyglutamate synthase (EC 6.3.2.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01963 acetyl-CoA carboxylase carboxyl transferase subunit beta [EC:6.4.1.2 2.1.3.15]\n",
      "SSO SSO:000000718 Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)\n",
      "b2316 ['Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03975 membrane-associated protein\n",
      "SSO SSO:000002229 DedA protein\n",
      "b2317 ['DedA protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K06173 tRNA pseudouridine38-40 synthase [EC:5.4.99.12]\n",
      "SSO SSO:000009365 tRNA pseudouridine(38-40) synthase (EC 5.4.99.12)\n",
      "b2318 ['tRNA pseudouridine(38-40) synthase (EC 5.4.99.12)']\n",
      "\n",
      "KO KO:K00133 aspartate-semialdehyde dehydrogenase [EC:1.2.1.11]\n",
      "SSO SSO:000039126 Semialdehyde dehydrogenase, USG-1 protein\n",
      "b2319 ['Semialdehyde dehydrogenase, USG-1 protein']\n",
      "\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "KO KO:K03473 erythronate-4-phosphate dehydrogenase [EC:1.1.1.290]\n",
      "SSO SSO:000002559 Erythronate-4-phosphate dehydrogenase (EC 1.1.1.290)\n",
      "b2320 ['Erythronate-4-phosphate dehydrogenase (EC 1.1.1.290)']\n",
      "\n",
      "SSO SSO:000002831 Flagellar regulator flk\n",
      "b2321 ['Flagellar regulator flk']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000026585 UPF0226 protein YfcJ\n",
      "b2322 ['UPF0226 protein YfcJ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00647 3-oxoacyl-[acyl-carrier-protein] synthase I [EC:2.3.1.41]\n",
      "SSO SSO:000000332 3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)\n",
      "b2323 ['3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)']\n",
      "\n",
      "KO KO:K15461 tRNA 5-methylaminomethyl-2-thiouridine biosynthesis bifunctional protein [EC:2.1.1.61 1.5.-.-]\n",
      "b2324 ['tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)', 'FAD-dependent cmnm(5)s(2)U34 oxidoreductase']\n",
      "\n",
      "SSO SSO:000028282 Uncharacterized protein YfcL\n",
      "b2325 ['Uncharacterized protein YfcL']\n",
      "\n",
      "KO KO:K09906 elongation factor P hydroxylase [EC:1.14.-.-]\n",
      "SSO SSO:000008304 Translation elongation factor P Lys34 hydroxylase\n",
      "b2326 ['Translation elongation factor P Lys34 hydroxylase']\n",
      "\n",
      "KO KO:K07090 uncharacterized protein\n",
      "SSO SSO:000039768 Transmembrane protein YfcA\n",
      "b2327 ['Transmembrane protein YfcA']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K07261 penicillin-insensitive murein DD-endopeptidase [EC:3.4.24.-]\n",
      "SSO SSO:000005038 Murein-DD-endopeptidase (EC 3.4.99.-)\n",
      "SSO SSO:000021309 Penicillin-insensitive murein endopeptidase\n",
      "b2328 ['Penicillin-insensitive murein endopeptidase']\n",
      "\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K01736 chorismate synthase [EC:4.2.3.5]\n",
      "SSO SSO:000001558 Chorismate synthase (EC 4.2.3.5)\n",
      "b2329 ['Chorismate synthase (EC 4.2.3.5)']\n",
      "\n",
      "KO KO:K07320 ribosomal protein L3 glutamine methyltransferase [EC:2.1.1.298]\n",
      "SSO SSO:000007129 Ribosomal protein L3 N(5)-glutamine methyltransferase (EC 2.1.1.298)\n",
      "b2330 ['Ribosomal protein L3 N(5)-glutamine methyltransferase (EC 2.1.1.298)']\n",
      "\n",
      "SSO SSO:000026509 UPF0115 protein YfcN\n",
      "b2331 ['UPF0115 protein YfcN']\n",
      "\n",
      "SSO SSO:000028146 Uncharacterized protein YadU in stf fimbrial cluster\n",
      "b2332 ['Uncharacterized protein YadU in stf fimbrial cluster']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000004924 Minor fimbrial subunit StfG\n",
      "b2333 ['Minor fimbrial subunit StfG']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000004923 Minor fimbrial subunit StfF\n",
      "b2334 ['Minor fimbrial subunit StfF']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000004922 Minor fimbrial subunit StfE\n",
      "b2335 ['Minor fimbrial subunit StfE']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K12519 chaperone protein PapD\n",
      "SSO SSO:000005840 Periplasmic fimbrial chaperone StfD\n",
      "b2336 ['Periplasmic fimbrial chaperone StfD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4661 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000004664 Major fimbrial subunit StfA\n",
      "b2339 ['Major fimbrial subunit StfA']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K08296 phosphohistidine phosphatase [EC:3.1.3.-]\n",
      "SSO SSO:000021663 Phosphohistidine phosphatase\n",
      "SSO SSO:000021664 Phosphohistidine phosphatase SixA\n",
      "b2340 ['Phosphohistidine phosphatase SixA']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01782 3-hydroxyacyl-CoA dehydrogenase / enoyl-CoA hydratase / 3-hydroxybutyryl-CoA epimerase [EC:1.1.1.35 4.2.1.17 5.1.2.3]\n",
      "b2341 ['Enoyl-CoA hydratase (EC 4.2.1.17)', '3-hydroxyacyl-CoA dehydrogenase (EC 1.1.1.35)', '3-hydroxybutyryl-CoA epimerase (EC 5.1.2.3)']\n",
      "\n",
      "KO KO:K00626 acetyl-CoA C-acetyltransferase [EC:2.3.1.9]\n",
      "SSO SSO:000000312 3-ketoacyl-CoA thiolase (EC 2.3.1.16)\n",
      "b2342 ['3-ketoacyl-CoA thiolase (EC 2.3.1.16)']\n",
      "\n",
      "SSO SSO:000026661 UPF0381 protein YfcZ\n",
      "b2343 ['UPF0381 protein YfcZ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K06076 long-chain fatty acid transport protein\n",
      "SSO SSO:000019041 Long-chain fatty acid transport protein\n",
      "b2344 ['Long-chain fatty acid transport protein']\n",
      "\n",
      "GO GO:0006259 DNA metabolic process\n",
      "SSO SSO:000028283 Uncharacterized protein YfdF\n",
      "b2345 ['Uncharacterized protein YfdF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04754 phospholipid-binding lipoprotein MlaA\n",
      "SSO SSO:000005610 Outer-membrane-phospholipid-binding lipoprotein MlaA\n",
      "b2346 ['Outer-membrane-phospholipid-binding lipoprotein MlaA']\n",
      "\n",
      "KO KO:K21990 formate-nitrite transporter family protein\n",
      "SSO SSO:000018396 Inner membrane protein YfdC\n",
      "b2347 ['Inner membrane protein YfdC']\n",
      "\n",
      "b2349 ['Putative prophage CPS-53 integrase']\n",
      "\n",
      "SSO SSO:000011033 Bactoprenol-linked glucose translocase\n",
      "b2350 ['Bactoprenol-linked glucose translocase']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "KO KO:K20534 polyisoprenyl-phosphate glycosyltransferase [EC:2.4.-.-]\n",
      "SSO SSO:000011031 Bactoprenol glucosyl transferase\n",
      "b2351 ['Bactoprenol glucosyl transferase']\n",
      "\n",
      "SSO SSO:000028284 Uncharacterized protein YfdI\n",
      "b2352 ['Uncharacterized protein YfdI']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2353 ['hypothetical protein']\n",
      "\n",
      "b2354 ['Tail fiber assembly protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2355 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2356 ['hypothetical protein']\n",
      "\n",
      "b2357 ['Uncharacterized protein YfdN']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2358 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028285 Uncharacterized protein YfdP\n",
      "b2359 ['Uncharacterized protein YfdP']\n",
      "\n",
      "SSO SSO:000028286 Uncharacterized protein YfdQ\n",
      "b2360 ['Uncharacterized protein YfdQ']\n",
      "\n",
      "KO KO:K06952 uncharacterized protein\n",
      "SSO SSO:000011673 COG1896: Predicted hydrolases of HD superfamily\n",
      "b2361 ['COG1896: Predicted hydrolases of HD superfamily']\n",
      "\n",
      "SSO SSO:000021489 Phage EaE protein\n",
      "b2362 ['Phage EaE protein']\n",
      "\n",
      "SSO SSO:000028287 Uncharacterized protein YfdT\n",
      "b2363 ['Uncharacterized protein YfdT']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4545 ['hypothetical protein']\n",
      "\n",
      "b4501 ['Gene 18 protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03566 LysR family transcriptional regulator, glycine cleavage system transcriptional activator\n",
      "KO KO:K13636 LysR family transcriptional regulator, D-serine deaminase activator\n",
      "SSO SSO:000012710 D-serine dehydratase transcriptional activator\n",
      "b2364 ['D-serine dehydratase transcriptional activator']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03299 gluconate:H+ symporter, GntP family\n",
      "KO KO:K13629 D-serine transporter\n",
      "SSO SSO:000012711 D-serine permease DsdX\n",
      "b2365 ['D-serine permease DsdX']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01753 D-serine dehydratase [EC:4.3.1.18]\n",
      "SSO SSO:000002043 D-serine ammonia-lyase (EC 4.3.1.18)\n",
      "SSO SSO:000002044 D-serine dehydratase (EC 4.3.1.18)\n",
      "b2366 ['D-serine ammonia-lyase (EC 4.3.1.18)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03446 MFS transporter, DHA2 family, multidrug resistance protein\n",
      "KO KO:K07786 MFS transporter, DHA2 family, multidrug resistance protein\n",
      "SSO SSO:000004976 Multidrug efflux system EmrKY-TolC, inner-membrane proton/drug antiporter EmrY (MFS type)\n",
      "b2367 ['Multidrug efflux system EmrKY-TolC, inner-membrane proton/drug antiporter EmrY (MFS type)']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03543 membrane fusion protein, multidrug efflux system\n",
      "KO KO:K07797 multidrug resistance protein K\n",
      "SSO SSO:000004977 Multidrug efflux system EmrKY-TolC, membrane fusion component EmrK\n",
      "b2368 ['Multidrug efflux system EmrKY-TolC, membrane fusion component EmrK']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07690 two-component system, NarL family, response regulator EvgA\n",
      "SSO SSO:000006180 Positive transcription regulator EvgA\n",
      "b2369 ['Positive transcription regulator EvgA']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07679 two-component system, NarL family, sensor histidine kinase EvgS [EC:2.7.13.3]\n",
      "SSO SSO:000003495 Hybrid sensory histidine kinase in two-component regulatory system with EvgA\n",
      "b2370 ['Hybrid sensory histidine kinase in two-component regulatory system with EvgA']\n",
      "\n",
      "KO KO:K18702 CoA:oxalate CoA-transferase [EC:2.8.3.19]\n",
      "SSO SSO:000029136 YfdE protein\n",
      "b2371 ['YfdE protein']\n",
      "\n",
      "KO KO:K07088 uncharacterized protein\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000028794 Uncharacterized transporter YfdV\n",
      "b2372 ['Uncharacterized transporter YfdV']\n",
      "\n",
      "KO KO:K01577 oxalyl-CoA decarboxylase [EC:4.1.1.8]\n",
      "KO KO:K12261 2-hydroxyacyl-CoA lyase 1 [EC:4.1.-.-]\n",
      "SSO SSO:000005620 Oxalyl-CoA decarboxylase (EC 4.1.1.8)\n",
      "b2373 ['Oxalyl-CoA decarboxylase (EC 4.1.1.8)']\n",
      "\n",
      "KO KO:K07749 formyl-CoA transferase [EC:2.8.3.16]\n",
      "SSO SSO:000002901 Formyl-coenzyme A transferase (EC 2.8.3.16)\n",
      "b2374 ['Formyl-coenzyme A transferase (EC 2.8.3.16)']\n",
      "\n",
      "SSO SSO:000022731 Protein YfdX\n",
      "b2375 ['Protein YfdX']\n",
      "\n",
      "SSO SSO:000027213 Uncharacterized lipoprotein YpdI\n",
      "b2376 ['Uncharacterized lipoprotein YpdI']\n",
      "\n",
      "SSO SSO:000028288 Uncharacterized protein YfdY\n",
      "b2377 ['Uncharacterized protein YfdY']\n",
      "\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02517 Kdo2-lipid IVA lauroyltransferase [EC:2.3.1.241]\n",
      "KO KO:K12974 KDO2-lipid IV(A) palmitoleoyltransferase [EC:2.3.1.242]\n",
      "SSO SSO:000004455 Lipid A biosynthesis palmitoleoyltransferase (EC 2.3.1.242)\n",
      "b2378 ['Lipid A biosynthesis palmitoleoyltransferase (EC 2.3.1.242)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b4680 ['hypothetical protein']\n",
      "\n",
      "KO KO:K14261 alanine-synthesizing transaminase [EC:2.6.1.-]\n",
      "SSO SSO:000036106 Glutamate-pyruvate aminotransferase AlaC (EC 2.6.1.2)\n",
      "b2379 ['Glutamate-pyruvate aminotransferase AlaC (EC 2.6.1.2)']\n",
      "\n",
      "KO KO:K02478 two-component system, LytTR family, sensor kinase [EC:2.7.13.3]\n",
      "SSO SSO:000024484 Sensor histidine kinase YpdA\n",
      "b2380 ['Sensor histidine kinase YpdA']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K02477 two-component system, LytTR family, response regulator\n",
      "KO KO:K07705 two-component system, LytTR family, response regulator LytT\n",
      "SSO SSO:000039852 Two-component transcriptional response regulator YpdB\n",
      "b2381 ['Two-component transcriptional response regulator YpdB']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000028763 Uncharacterized transcriptional regulator YpdC, AraC family\n",
      "b2382 ['Uncharacterized transcriptional regulator YpdC, AraC family']\n",
      "\n",
      "GO GO:0009401 phosphoenolpyruvate-dependent sugar phosphotransferase system\n",
      "KO KO:K23991 Unknown\n",
      "SSO SSO:000005964 Phosphoenolpyruvate-protein phosphotransferase of PTS system (EC 2.7.3.9)\n",
      "b2383 ['Phosphoenolpyruvate-protein phosphotransferase of PTS system (EC 2.7.3.9)']\n",
      "\n",
      "SSO SSO:000005403 Nondeblocking aminopeptidase YpdE (X-X-[^PR]- specific)\n",
      "SSO SSO:000010609 Aminopeptidase\n",
      "b2384 ['Nondeblocking aminopeptidase YpdE (X-X-[^PR]- specific)']\n",
      "\n",
      "KO KO:K01271 Xaa-Pro dipeptidase [EC:3.4.13.9]\n",
      "KO KO:K08326 aminopeptidase [EC:3.4.11.-]\n",
      "SSO SSO:000000971 Aminopeptidase YpdF (MP-, MA-, MS-, AP-, NP- specific)\n",
      "SSO SSO:000008954 Xaa-Pro aminopeptidase (EC 3.4.11.9)\n",
      "b2385 ['Aminopeptidase YpdF (MP-, MA-, MS-, AP-, NP- specific)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000021183 PTS system, fructose-like IIC component FryC\n",
      "b2386 ['PTS system, fructose-like IIC component FryC']\n",
      "\n",
      "GO GO:0009401 phosphoenolpyruvate-dependent sugar phosphotransferase system\n",
      "KO KO:K11202 PTS system, fructose-specific IIB-like component [EC:2.7.1.-]\n",
      "SSO SSO:000021181 PTS system, fructose-like IIB component FryB (EC 2.7.1.69)\n",
      "b2387 ['PTS system, fructose-like IIB component FryB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000003097 Glucokinase (EC 2.7.1.2)\n",
      "b2388 ['Glucokinase (EC 2.7.1.2)']\n",
      "\n",
      "SSO SSO:000012182 Chloride channel protein\n",
      "b2389 ['Chloride channel protein']\n",
      "\n",
      "SSO SSO:000028496 Uncharacterized protein YpeC\n",
      "b2390 ['Uncharacterized protein YpeC']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03322 manganese transport protein\n",
      "SSO SSO:000019329 Manganese transport protein MntH\n",
      "b2392 ['Manganese transport protein MntH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11535 nucleoside transport protein\n",
      "SSO SSO:000005277 Na+ dependent nucleoside transporter NupC\n",
      "b2393 ['Na+ dependent nucleoside transporter NupC']\n",
      "\n",
      "SSO SSO:000026100 Transposase InsL for insertion sequence element IS186\n",
      "b2394 ['Transposase InsL for insertion sequence element IS186']\n",
      "\n",
      "SSO SSO:000032478 putative diguanylate cyclase\n",
      "b2395 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeA']\n",
      "\n",
      "SSO SSO:000028658 Uncharacterized protein yfeC\n",
      "b2398 ['Uncharacterized protein yfeC']\n",
      "\n",
      "SSO SSO:000028658 Uncharacterized protein yfeC\n",
      "b2399 ['Uncharacterized protein yfeC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01885 glutamyl-tRNA synthetase [EC:6.1.1.17]\n",
      "SSO SSO:000003186 Glutamyl-tRNA synthetase (EC 6.1.1.17)\n",
      "b2400 ['Glutamyl-tRNA synthetase (EC 6.1.1.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000008965 Xanthosine operon regulatory protein XapR, LysR family\n",
      "b2405 ['Xanthosine operon regulatory protein XapR, LysR family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11537 MFS transporter, NHS family, xanthosine permease\n",
      "SSO SSO:000029045 Xanthosine permease\n",
      "b2406 ['Xanthosine permease']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K03783 purine-nucleoside phosphorylase [EC:2.4.2.1]\n",
      "KO KO:K03815 xanthosine phosphorylase [EC:2.4.2.-]\n",
      "SSO SSO:000029046 Xanthosine phosphorylase (EC 2.4.2.1)\n",
      "b2407 ['Xanthosine phosphorylase (EC 2.4.2.1)']\n",
      "\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "SSO SSO:000027379 Uncharacterized outer membrane protein YfeN\n",
      "b2408 ['Uncharacterized outer membrane protein YfeN']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K17737 LysR family transcriptional regulator, carnitine catabolism transcriptional activator\n",
      "SSO SSO:000008223 Transcriptional regulator YfeR, LysR family\n",
      "b2409 ['Transcriptional regulator YfeR, LysR family']\n",
      "\n",
      "KO KO:K14347 solute carrier family 10 (sodium/bile acid cotransporter), member 7\n",
      "SSO SSO:000028290 Uncharacterized protein YfeH\n",
      "b2410 ['Uncharacterized protein YfeH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K09954 uncharacterized protein\n",
      "SSO SSO:000028495 Uncharacterized protein YpeB\n",
      "b4546 ['Uncharacterized protein YpeB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K01972 DNA ligase (NAD+) [EC:6.5.1.2]\n",
      "SSO SSO:000002073 DNA ligase (NAD(+)) (EC 6.5.1.2)\n",
      "b2411 ['DNA ligase (NAD(+)) (EC 6.5.1.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03528 cell division protein ZipA\n",
      "SSO SSO:000012037 Cell division protein ZipA\n",
      "b2412 ['Cell division protein ZipA']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06203 CysZ protein\n",
      "SSO SSO:000007815 Sulfate transporter, CysZ-type\n",
      "b2413 ['Sulfate transporter, CysZ-type']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0019344 cysteine biosynthetic process\n",
      "KO KO:K01738 cysteine synthase [EC:2.5.1.47]\n",
      "SSO SSO:000001864 Cysteine synthase (EC 2.5.1.47)\n",
      "b2414 ['Cysteine synthase (EC 2.5.1.47)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02784 phosphocarrier protein HPr\n",
      "SSO SSO:000006056 Phosphotransferase system, phosphocarrier protein HPr\n",
      "b2415 ['Phosphotransferase system, phosphocarrier protein HPr']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K08483 phosphotransferase system, enzyme I, PtsI [EC:2.7.3.9]\n",
      "SSO SSO:000005964 Phosphoenolpyruvate-protein phosphotransferase of PTS system (EC 2.7.3.9)\n",
      "b2416 ['Phosphoenolpyruvate-protein phosphotransferase of PTS system (EC 2.7.3.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02777 PTS system, sugar-specific IIA component [EC:2.7.1.-]\n",
      "SSO SSO:000005696 PTS system, glucose-specific IIA component (EC 2.7.1.69)\n",
      "b2417 ['PTS system, glucose-specific IIA component (EC 2.7.1.199)']\n",
      "\n",
      "KO KO:K00868 pyridoxine kinase [EC:2.7.1.35]\n",
      "SSO SSO:000006808 Pyridoxal kinase (EC 2.7.1.35)\n",
      "b2418 ['Pyridoxal kinase (EC 2.7.1.35)']\n",
      "\n",
      "SSO SSO:000028291 Uncharacterized protein YfeK\n",
      "b2419 ['Uncharacterized protein YfeK']\n",
      "\n",
      "SSO SSO:000019995 Molybdate metabolism regulator\n",
      "SSO SSO:000028996 WGR domain protein\n",
      "b2420 ['Molybdate metabolism regulator']\n",
      "\n",
      "GO GO:0019344 cysteine biosynthetic process\n",
      "KO KO:K12339 S-sulfo-L-cysteine synthase (O-acetyl-L-serine-dependent) [EC:2.5.1.144]\n",
      "SSO SSO:000001865 Cysteine synthase B (EC 2.5.1.47)\n",
      "b2421 ['Cysteine synthase B (EC 2.5.1.47)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0015419 sulfate transmembrane-transporting ATPase activity\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K02045 sulfate transport system ATP-binding protein [EC:3.6.3.25]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000007807 Sulfate and thiosulfate import ATP-binding protein CysA (EC 3.6.3.25)\n",
      "b2422 ['Sulfate and thiosulfate import ATP-binding protein CysA (EC 3.6.3.25)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02047 sulfate transport system permease protein\n",
      "SSO SSO:000007811 Sulfate transport system permease protein CysW\n",
      "b2423 ['Sulfate transport system permease protein CysW']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02046 sulfate transport system permease protein\n",
      "SSO SSO:000007810 Sulfate transport system permease protein CysT\n",
      "b2424 ['Sulfate transport system permease protein CysT']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02048 sulfate transport system substrate-binding protein\n",
      "SSO SSO:000007806 Sulfate and thiosulfate binding protein CysP\n",
      "b2425 ['Sulfate and thiosulfate binding protein CysP']\n",
      "\n",
      "KO KO:K00059 3-oxoacyl-[acyl-carrier protein] reductase [EC:1.1.1.100]\n",
      "SSO SSO:000020970 Oxidoreductase ucpA\n",
      "b2426 ['Oxidoreductase UcpA']\n",
      "\n",
      "KO KO:K15835 RpiR family transcriptional regulator, murPQ operon repressor\n",
      "SSO SSO:000025824 Transcriptional regulator MurR, RpiR family\n",
      "b2427 ['Transcriptional regulator MurR, RpiR family']\n",
      "\n",
      "KO KO:K07106 N-acetylmuramic acid 6-phosphate etherase [EC:4.2.1.126]\n",
      "SSO SSO:000005114 N-acetylmuramic acid 6-phosphate etherase (EC 4.2.-.-)\n",
      "b2428 ['N-acetylmuramic acid 6-phosphate etherase (EC 4.2.1.126)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K11192 PTS system, N-acetylmuramic acid-specific IIC component\n",
      "b2429 ['PTS system, N-acetylmuramic acid-specific IIB component (EC 2.7.1.192)', 'PTS system, N-acetylmuramic acid-specific IIC component']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K21469 serine-type D-Ala-D-Ala carboxypeptidase [EC:3.4.16.4]\n",
      "b2430 ['UPF0214 protein YfeW']\n",
      "\n",
      "KO KO:K07223 porphyrinogen peroxidase [EC:1.11.1.-]\n",
      "SSO SSO:000022175 Predicted dye-decolorizing peroxidase (DyP), YfeX-like subgroup\n",
      "b2431 ['Predicted dye-decolorizing peroxidase (DyP), YfeX-like subgroup']\n",
      "\n",
      "SSO SSO:000006264 Predicted outer membrane lipoprotein YfeY\n",
      "b2432 ['Predicted outer membrane lipoprotein YfeY']\n",
      "\n",
      "SSO SSO:000003882 Inner membrane protein YfeZ\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b2433 ['Inner membrane protein YfeZ']\n",
      "\n",
      "SSO SSO:000000724 Acetyltransferase YpeA\n",
      "b2434 ['Acetyltransferase YpeA']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01448 N-acetylmuramoyl-L-alanine amidase [EC:3.5.1.28]\n",
      "SSO SSO:000005115 N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)\n",
      "b2435 ['N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00228 coproporphyrinogen III oxidase [EC:1.3.3.3]\n",
      "SSO SSO:000001780 Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)\n",
      "SSO SSO:000029759 coproporphyrinogen III oxidase\n",
      "b2436 ['Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K04033 AraC family transcriptional regulator, ethanolamine operon transcriptional activator\n",
      "SSO SSO:000002566 Ethanolamine operon regulatory protein\n",
      "b2437 ['Ethanolamine operon regulatory protein']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04025 ethanolamine utilization protein EutK\n",
      "SSO SSO:000002570 Ethanolamine utilization polyhedral-body-like protein EutK\n",
      "b2438 ['Ethanolamine utilization polyhedral-body-like protein EutK']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04026 ethanolamine utilization protein EutL\n",
      "SSO SSO:000002571 Ethanolamine utilization polyhedral-body-like protein EutL\n",
      "b2439 ['Ethanolamine utilization polyhedral-body-like protein EutL']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K03736 ethanolamine ammonia-lyase small subunit [EC:4.3.1.7]\n",
      "SSO SSO:000002565 Ethanolamine ammonia-lyase light chain (EC 4.3.1.7)\n",
      "b2440 ['Ethanolamine ammonia-lyase light chain (EC 4.3.1.7)']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K03735 ethanolamine ammonia-lyase large subunit [EC:4.3.1.7]\n",
      "SSO SSO:000002564 Ethanolamine ammonia-lyase heavy chain (EC 4.3.1.7)\n",
      "b2441 ['Ethanolamine ammonia-lyase heavy chain (EC 4.3.1.7)']\n",
      "\n",
      "SSO SSO:000018491 Integrase\n",
      "b2442 ['Integrase']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2443 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2444 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2445 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2446 ['hypothetical protein']\n",
      "\n",
      "b2447 ['FIG00640748: hypothetical protein']\n",
      "\n",
      "b2448 ['CPZ-55 prophage', 'predicted protein']\n",
      "\n",
      "b2449 ['CPZ-55 prophage', 'predicted protein']\n",
      "\n",
      "b2450 ['FIG00639312: hypothetical protein']\n",
      "\n",
      "KO KO:K04019 ethanolamine utilization protein EutA\n",
      "SSO SSO:000002575 Ethanolamine utilization protein EutA\n",
      "b2451 ['Ethanolamine utilization protein EutA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04023 ethanolamine transporter\n",
      "SSO SSO:000002567 Ethanolamine permease\n",
      "b2452 ['Ethanolamine permease']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04022 alcohol dehydrogenase\n",
      "SSO SSO:000002576 Ethanolamine utilization protein EutG\n",
      "b2453 ['Ethanolamine utilization protein EutG']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04024 ethanolamine utilization protein EutJ\n",
      "SSO SSO:000002577 Ethanolamine utilization protein EutJ\n",
      "b2454 ['Ethanolamine utilization protein EutJ']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04021 aldehyde dehydrogenase\n",
      "b2455 ['Acetaldehyde dehydrogenase (EC 1.2.1.10)', 'Acetaldehyde dehydrogenase (EC 1.2.1.10), ethanolamine utilization cluster']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K04028 ethanolamine utilization protein EutN\n",
      "SSO SSO:000002573 Ethanolamine utilization polyhedral-body-like protein EutN\n",
      "b2456 ['Ethanolamine utilization polyhedral-body-like protein EutN']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K04027 ethanolamine utilization protein EutM\n",
      "SSO SSO:000002572 Ethanolamine utilization polyhedral-body-like protein EutM\n",
      "b2457 ['Ethanolamine utilization polyhedral-body-like protein EutM']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04020 phosphotransacetylase\n",
      "SSO SSO:000005916 Phosphate acetyltransferase (EC 2.3.1.8)\n",
      "SSO SSO:000005917 Phosphate acetyltransferase , ethanolamine utilization-specific (EC 2.3.1.8)\n",
      "b2458 ['Phosphate acetyltransferase (EC 2.3.1.8), ethanolamine utilization-specific']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04032 ethanolamine utilization cobalamin adenosyltransferase [EC:2.5.1.17]\n",
      "b2459 ['ATP:Cob(I)alamin adenosyltransferase (EC 2.5.1.17)', 'ATP:Cob(I)alamin adenosyltransferase (EC 2.5.1.17), ethanolamine utilization']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04030 ethanolamine utilization protein EutQ\n",
      "SSO SSO:000002579 Ethanolamine utilization protein EutQ\n",
      "b2460 ['Ethanolamine utilization protein EutQ']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04029 ethanolamine utilization protein EutP\n",
      "SSO SSO:000002578 Ethanolamine utilization protein EutP\n",
      "b2461 ['Ethanolamine utilization protein EutP']\n",
      "\n",
      "GO GO:0009310 amine catabolic process\n",
      "KO KO:K04031 ethanolamine utilization protein EutS\n",
      "SSO SSO:000002574 Ethanolamine utilization polyhedral-body-like protein EutS\n",
      "b2462 ['Ethanolamine utilization polyhedral-body-like protein EutS']\n",
      "\n",
      "KO KO:K00029 malate dehydrogenase (oxaloacetate-decarboxylating)(NADP+) [EC:1.1.1.40]\n",
      "SSO SSO:000005239 NADP-dependent malic enzyme (EC 1.1.1.40)\n",
      "b2463 ['NADP-dependent malic enzyme (EC 1.1.1.40)']\n",
      "\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "KO KO:K00616 transaldolase [EC:2.2.1.2]\n",
      "SSO SSO:000008135 Transaldolase (EC 2.2.1.2)\n",
      "b2464 ['Transaldolase (EC 2.2.1.2)']\n",
      "\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00615 transketolase [EC:2.2.1.1]\n",
      "SSO SSO:000008287 Transketolase (EC 2.2.1.1)\n",
      "b2465 ['Transketolase (EC 2.2.1.1)']\n",
      "\n",
      "SSO SSO:000012963 DUF1176 domain-containing protein\n",
      "b2466 ['DUF1176 domain-containing protein']\n",
      "\n",
      "KO KO:K12945 GDP-mannose pyrophosphatase NudK [EC:3.6.1.-]\n",
      "SSO SSO:000002975 GDP-mannose pyrophosphatase NudK\n",
      "b2467 ['GDP-mannose pyrophosphatase NudK']\n",
      "\n",
      "KO KO:K00266 glutamate synthase (NADPH) small chain [EC:1.4.1.13]\n",
      "KO KO:K05796 electron transport protein HydN\n",
      "SSO SSO:000022693 Protein AegA\n",
      "b2468 ['Protein AegA']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07674 two-component system, NarL family, nitrate/nitrite sensor histidine kinase NarQ [EC:2.7.13.3]\n",
      "SSO SSO:000020540 Nitrate/nitrite sensor protein NarQ\n",
      "b2469 ['Nitrate/nitrite sensor protein NarQ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K18138 multidrug efflux pump\n",
      "KO KO:K18324 multidrug efflux pump\n",
      "SSO SSO:000000965 Aminoglycosides efflux system AcrAD-TolC, inner-membrane proton/drug antiporter AcrD (RND type)\n",
      "b2470 ['Aminoglycosides efflux system AcrAD-TolC, inner-membrane proton/drug antiporter AcrD (RND type)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4606 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000016525 FIG138056: a glutathione-dependent thiol reductase\n",
      "b2471 ['FIG138056: a glutathione-dependent thiol reductase']\n",
      "\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K01439 succinyl-diaminopimelate desuccinylase [EC:3.5.1.18]\n",
      "SSO SSO:000005145 N-succinyl-L,L-diaminopimelate desuccinylase (EC 3.5.1.18)\n",
      "b2472 ['N-succinyl-L,L-diaminopimelate desuccinylase (EC 3.5.1.18)']\n",
      "\n",
      "GO GO:0031225 anchored component of membrane\n",
      "SSO SSO:000026657 UPF0370 protein YpfN\n",
      "b4547 ['UPF0370 protein YpfN']\n",
      "\n",
      "KO KO:K06999 phospholipase/carboxylesterase\n",
      "SSO SSO:000013479 Esterase YpfH\n",
      "b2473 ['Esterase YpfH']\n",
      "\n",
      "KO KO:K06957 tRNA(Met) cytidine acetyltransferase [EC:2.3.1.193]\n",
      "SSO SSO:000009355 tRNA cytosine(34) acetyltransferase (EC 2.3.1.193)\n",
      "b2474 ['tRNA cytosine(34) acetyltransferase (EC 2.3.1.193)']\n",
      "\n",
      "KO KO:K07054 uncharacterized protein\n",
      "SSO SSO:000008989 YpfJ protein, zinc metalloprotease superfamily\n",
      "b2475 ['YpfJ protein, zinc metalloprotease superfamily']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K01923 phosphoribosylaminoimidazole-succinocarboxamide synthase [EC:6.3.2.6]\n",
      "SSO SSO:000006038 Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)\n",
      "b2476 ['Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07287 outer membrane protein assembly factor BamC\n",
      "SSO SSO:000005533 Outer membrane beta-barrel assembly protein BamC\n",
      "b2477 ['Outer membrane beta-barrel assembly protein BamC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K01714 4-hydroxy-tetrahydrodipicolinate synthase [EC:4.3.3.7]\n",
      "SSO SSO:000000366 4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)\n",
      "SSO SSO:000029835 dihydrodipicolinate synthase\n",
      "b2478 ['4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03567 glycine cleavage system transcriptional repressor\n",
      "SSO SSO:000003273 Glycine cleavage system transcriptional antiactivator GcvR\n",
      "b2479 ['Glycine cleavage system transcriptional antiactivator GcvR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K03564 peroxiredoxin Q/BCP [EC:1.11.1.15]\n",
      "SSO SSO:000008031 Thiol peroxidase, Bcp-type (EC 1.11.1.15)\n",
      "b2480 ['Thiol peroxidase, Bcp-type (EC 1.11.1.15)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K12136 hydrogenase-4 component A [EC:1.-.-.-]\n",
      "SSO SSO:000003502 Hydrogenase-4 component A (EC 1.-.-.-)\n",
      "b2481 ['Hydrogenase-4 component A']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K12137 hydrogenase-4 component B [EC:1.-.-.-]\n",
      "SSO SSO:000003503 Hydrogenase-4 component B (EC 1.-.-.-)\n",
      "b2482 ['Hydrogenase-4 component B']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K12138 hydrogenase-4 component C [EC:1.-.-.-]\n",
      "SSO SSO:000003504 Hydrogenase-4 component C (EC 1.-.-.-)\n",
      "b2483 ['Hydrogenase-4 component C']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K12139 hydrogenase-4 component D [EC:1.-.-.-]\n",
      "SSO SSO:000003505 Hydrogenase-4 component D (EC 1.-.-.-)\n",
      "b2484 ['Hydrogenase-4 component D']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K12140 hydrogenase-4 component E [EC:1.-.-.-]\n",
      "SSO SSO:000003506 Hydrogenase-4 component E (EC 1.-.-.-)\n",
      "b2485 ['Hydrogenase-4 component E']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K12141 hydrogenase-4 component F [EC:1.-.-.-]\n",
      "SSO SSO:000003507 Hydrogenase-4 component F (EC 1.-.-.-)\n",
      "b2486 ['Hydrogenase-4 component F']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "SSO SSO:000003508 Hydrogenase-4 component G (EC 1.-.-.-)\n",
      "b2487 ['Hydrogenase-4 component G']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K12143 hydrogenase-4 component H\n",
      "SSO SSO:000003509 Hydrogenase-4 component H (EC 1.-.-.-)\n",
      "b2488 ['Hydrogenase-4 component H']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K12144 hydrogenase-4 component I [EC:1.-.-.-]\n",
      "SSO SSO:000003510 Hydrogenase-4 component I (EC 1.-.-.-)\n",
      "b2489 ['Hydrogenase-4 component I']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K12145 hydrogenase-4 component J [EC:1.-.-.-]\n",
      "SSO SSO:000003511 Hydrogenase-4 maturation protease\n",
      "b2490 ['Hydrogenase-4 maturation protease']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K12146 hydrogenase-4 transcriptional activator\n",
      "KO KO:K15836 formate hydrogenlyase transcriptional activator\n",
      "SSO SSO:000003513 Hydrogenase-4 transcriptional activator\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2491 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03459 formate transporter\n",
      "KO KO:K06212 formate transporter\n",
      "SSO SSO:000035959 Formate efflux transporter FocA\n",
      "b2492 ['Formate efflux transporter FocA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03548 putative permease\n",
      "SSO SSO:000006718 Putative permease PerM (= YfgO)\n",
      "b2493 ['Putative permease PerM (= YfgO)']\n",
      "\n",
      "KO KO:K01423 beta-barrel assembly-enhancing protease [EC:3.4.-.-]\n",
      "b2494 ['Exported zinc metalloprotease YfgC precursor']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K00537 arsenate reductase (glutaredoxin) [EC:1.20.4.1]\n",
      "SSO SSO:000041448 Uncharacterized protein YfgD, not an arsenate reductase\n",
      "b2495 ['Uncharacterized protein YfgD, not an arsenate reductase']\n",
      "\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K10763 DnaA-homolog protein\n",
      "SSO SSO:000035520 DnaA inactivator Hda (shorter homolog of DnaA)\n",
      "b2496 ['DnaA inactivator Hda (shorter homolog of DnaA)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02824 uracil permease\n",
      "SSO SSO:000008848 Uracil permease\n",
      "b2497 ['Uracil permease', 'Uracil:proton symporter UraA']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00761 uracil phosphoribosyltransferase [EC:2.4.2.9]\n",
      "SSO SSO:000008849 Uracil phosphoribosyltransferase (EC 2.4.2.9)\n",
      "b2498 ['Uracil phosphoribosyltransferase (EC 2.4.2.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K01933 phosphoribosylformylglycinamidine cyclo-ligase [EC:6.3.3.1]\n",
      "SSO SSO:000006042 Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)\n",
      "b2499 ['Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K11175 phosphoribosylglycinamide formyltransferase 1 [EC:2.1.2.2]\n",
      "SSO SSO:000006046 Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)\n",
      "b2500 ['Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K00937 polyphosphate kinase [EC:2.7.4.1]\n",
      "SSO SSO:000006159 Polyphosphate kinase (EC 2.7.4.1)\n",
      "b2501 ['Polyphosphate kinase (EC 2.7.4.1)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K01524 exopolyphosphatase / guanosine-5'-triphosphate,3'-diphosphate pyrophosphatase [EC:3.6.1.11 3.6.1.40]\n",
      "SSO SSO:000002629 Exopolyphosphatase (EC 3.6.1.11)\n",
      "b2502 ['Exopolyphosphatase (EC 3.6.1.11)']\n",
      "\n",
      "b2503 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeF']\n",
      "\n",
      "SSO SSO:000028292 Uncharacterized protein YfgG\n",
      "b2504 ['Uncharacterized protein YfgG']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K06077 outer membrane lipoprotein SlyB\n",
      "SSO SSO:000027208 Uncharacterized lipoprotein YfgH\n",
      "b2505 ['Uncharacterized lipoprotein YfgH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000028293 Uncharacterized protein YfgI\n",
      "b2506 ['Uncharacterized protein YfgI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01951 GMP synthase (glutamine-hydrolysing) [EC:6.3.5.2]\n",
      "b2507 ['GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)', 'GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00088 IMP dehydrogenase [EC:1.1.1.205]\n",
      "SSO SSO:000044002 IMP dehydrogenase (EC 1.1.1.205)\n",
      "b2508 [\"Inosine-5'-monophosphate dehydrogenase (EC 1.1.1.205)\", 'CBS domain']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K03601 exodeoxyribonuclease VII large subunit [EC:3.1.11.6]\n",
      "SSO SSO:000002623 Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)\n",
      "SSO SSO:000013534 Exonuclease VII, large subunit\n",
      "b2509 ['Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)']\n",
      "\n",
      "GO GO:0042966 biotin carboxyl carrier protein biosynthetic process\n",
      "SSO SSO:000028294 Uncharacterized protein YfgJ\n",
      "b2510 ['Uncharacterized protein YfgJ']\n",
      "\n",
      "KO KO:K03977 GTPase\n",
      "SSO SSO:000002993 GTP-binding protein EngA\n",
      "b2511 ['GTP-binding protein EngA']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K17713 outer membrane protein assembly factor BamB\n",
      "SSO SSO:000005532 Outer membrane beta-barrel assembly protein BamB\n",
      "b2512 ['Outer membrane beta-barrel assembly protein BamB']\n",
      "\n",
      "SSO SSO:000039910 UPF0070 protein YfgM\n",
      "b2513 ['UPF0070 protein YfgM']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01892 histidyl-tRNA synthetase [EC:6.1.1.21]\n",
      "SSO SSO:000003450 Histidyl-tRNA synthetase (EC 6.1.1.21)\n",
      "b2514 ['Histidyl-tRNA synthetase (EC 6.1.1.21)']\n",
      "\n",
      "KO KO:K03526 (E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase [EC:1.17.7.1 1.17.7.3]\n",
      "SSO SSO:000000048 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate synthase (EC 1.17.7.1)\n",
      "SSO SSO:000033866 (E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase (flavodoxin) (EC 1.17.7.3)\n",
      "b2515 ['(E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase (flavodoxin) (EC 1.17.7.3)']\n",
      "\n",
      "KO KO:K15539 cytoskeleton protein RodZ\n",
      "SSO SSO:000012669 Cytoskeleton protein RodZ\n",
      "b2516 ['Cytoskeleton protein RodZ']\n",
      "\n",
      "KO KO:K06941 23S rRNA (adenine2503-C2)-methyltransferase [EC:2.1.1.192]\n",
      "b2517 ['23S rRNA (adenine(2503)-C(2))-methyltransferase', 'tRNA (adenine(37)-C(2))-methyltransferase (EC 2.1.1.192)']\n",
      "\n",
      "GO GO:0009152 purine ribonucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K00940 nucleoside-diphosphate kinase [EC:2.7.4.6]\n",
      "SSO SSO:000005425 Nucleoside diphosphate kinase (EC 2.7.4.6)\n",
      "b2518 ['Nucleoside diphosphate kinase (EC 2.7.4.6)']\n",
      "\n",
      "EC EC:2.4.1.129 Peptidoglycan glycosyltransferase.\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K05367 penicillin-binding protein 1C [EC:2.4.1.129]\n",
      "SSO SSO:000005776 Penicillin-insensitive transglycosylase & transpeptidase PBP-1C (EC 2.4.2.-)\n",
      "b2519 ['Penicillin-insensitive transglycosylase (EC 2.4.2.-) & transpeptidase PBP-1C']\n",
      "\n",
      "KO KO:K06894 uncharacterized protein\n",
      "SSO SSO:000010472 Alpha-2-macroglobulin\n",
      "b2520 ['Alpha-2-macroglobulin']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K01011 thiosulfate/3-mercaptopyruvate sulfurtransferase [EC:2.8.1.1 2.8.1.2]\n",
      "SSO SSO:000000314 3-mercaptopyruvate sulfurtransferase (EC 2.8.1.2)\n",
      "b2521 ['3-mercaptopyruvate sulfurtransferase (EC 2.8.1.2)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "SSO SSO:000022721 Protein SseB\n",
      "b2522 ['Protein SseB']\n",
      "\n",
      "KO KO:K07751 PepB aminopeptidase [EC:3.4.11.23]\n",
      "SSO SSO:000005782 Peptidase B (EC 3.4.11.23)\n",
      "b2523 ['Peptidase B (EC 3.4.11.23)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000022705 Protein IscX, believed to be involved in assembly of Fe-S clusters\n",
      "b2524 ['Protein IscX, believed to be involved in assembly of Fe-S clusters']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K04755 ferredoxin, 2Fe-2S\n",
      "SSO SSO:000002704 Ferredoxin, 2Fe-2S\n",
      "b2525 ['Ferredoxin, 2Fe-2S']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04044 molecular chaperone HscA\n",
      "SSO SSO:000001506 Chaperone protein HscA\n",
      "b2526 ['Chaperone protein HscA']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04082 molecular chaperone HscB\n",
      "SSO SSO:000001507 Chaperone protein HscB\n",
      "b2527 ['Chaperone protein HscB']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K13628 iron-sulfur cluster assembly protein\n",
      "SSO SSO:000018625 Iron-sulfur cluster assembly iron binding protein IscA\n",
      "b2528 ['Iron-sulfur cluster assembly iron binding protein IscA']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K04488 nitrogen fixation protein NifU and related proteins\n",
      "KO KO:K22068 iron-sulfur cluster assembly enzyme ISCU, mitochondrial\n",
      "SSO SSO:000004027 Iron-sulfur cluster assembly scaffold protein IscU\n",
      "b2529 ['Iron-sulfur cluster assembly scaffold protein IscU']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K04487 cysteine desulfurase [EC:2.8.1.7]\n",
      "b2530 ['Cysteine desulfurase (EC 2.8.1.7) => IscS']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K13643 Rrf2 family transcriptional regulator, iron-sulfur cluster assembly transcription factor\n",
      "SSO SSO:000004031 Iron-sulfur cluster regulator IscR\n",
      "b2531 ['Iron-sulfur cluster regulator IscR']\n",
      "\n",
      "KO KO:K02533 tRNA/rRNA methyltransferase [EC:2.1.1.-]\n",
      "KO KO:K15396 tRNA (cytidine32/uridine32-2'-O)-methyltransferase [EC:2.1.1.200]\n",
      "SSO SSO:000009333 tRNA (cytidine(32)/uridine(32)-2'-O)-methyltransferase (EC 2.1.1.200)\n",
      "b2532 [\"tRNA (cytidine(32)/uridine(32)-2'-O)-methyltransferase (EC 2.1.1.200)\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K01092 myo-inositol-1(or 4)-monophosphatase [EC:3.1.3.25]\n",
      "SSO SSO:000003916 Inositol-1-monophosphatase (EC 3.1.3.25)\n",
      "SSO SSO:000018454 Inositol monophosphatase\n",
      "b2533 ['Inositol-1-monophosphatase (EC 3.1.3.25)']\n",
      "\n",
      "KO KO:K06889 uncharacterized protein\n",
      "SSO SSO:000028297 Uncharacterized protein YfhR\n",
      "b2534 ['Uncharacterized protein YfhR']\n",
      "\n",
      "SSO SSO:000025115 Stationary phase inducible protein CsiE\n",
      "b2535 ['Stationary phase inducible protein CsiE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05820 MFS transporter, PPP family, 3-phenylpropionic acid transporter\n",
      "SSO SSO:000022375 Probable 3-phenylpropionic acid transporter\n",
      "b2536 ['Probable 3-phenylpropionic acid transporter']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05817 LysR family transcriptional regulator, hca operon transcriptional activator\n",
      "SSO SSO:000017642 Hca operon (3-phenylpropionic acid catabolism) transcriptional activator HcaR\n",
      "b2537 ['Hca operon (3-phenylpropionic acid catabolism) transcriptional activator HcaR']\n",
      "\n",
      "SSO SSO:000017642 Hca operon (3-phenylpropionic acid catabolism) transcriptional activator HcaR\n",
      "b4706 ['Hca operon (3-phenylpropionic acid catabolism) transcriptional activator HcaR']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K05708 3-phenylpropionate/trans-cinnamate dioxygenase subunit alpha [EC:1.14.12.19]\n",
      "SSO SSO:000009640 3-phenylpropionate dioxygenase alpha subunit (EC 1.14.1.-)\n",
      "b2538 ['3-phenylpropionate dioxygenase, alpha subunit (EC 1.14.12.19)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K05709 3-phenylpropionate/trans-cinnamate dioxygenase subunit beta [EC:1.14.12.19]\n",
      "SSO SSO:000009641 3-phenylpropionate dioxygenase beta subunit (EC 1.14.1.-)\n",
      "b2539 ['3-phenylpropionate dioxygenase, beta subunit (EC 1.14.12.19)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K05710 3-phenylpropionate/trans-cinnamate dioxygenase ferredoxin component\n",
      "SSO SSO:000009642 3-phenylpropionate dioxygenase ferredoxin subunit\n",
      "b2540 ['3-phenylpropionate dioxygenase ferredoxin subunit']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K05711 2,3-dihydroxy-2,3-dihydrophenylpropionate dehydrogenase [EC:1.3.1.87]\n",
      "SSO SSO:000000085 2,3-dihydroxy-2,3-dihydrophenylpropionate dehydrogenase (EC 1.3.1.56)\n",
      "b2541 ['2,3-dihydroxy-2,3-dihydro-phenylpropionate dehydrogenase (EC 1.3.1.87)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00529 3-phenylpropionate/trans-cinnamate dioxygenase ferredoxin reductase component [EC:1.18.1.3]\n",
      "SSO SSO:000009643 3-phenylpropionate dioxygenase ferredoxin--NAD(+) reductase component (EC 1.18.1.3)\n",
      "b2542 ['3-phenylpropionate dioxygenase ferredoxin--NAD(+) reductase component (EC 1.18.1.3)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K15977 putative oxidoreductase\n",
      "SSO SSO:000018425 Inner membrane protein YphA\n",
      "b2543 ['Inner membrane protein YphA']\n",
      "\n",
      "KO KO:K01785 aldose 1-epimerase [EC:5.1.3.3]\n",
      "SSO SSO:000010390 Aldose 1-epimerase family protein YphB\n",
      "b2544 ['Aldose 1-epimerase family protein YphB']\n",
      "\n",
      "SSO SSO:000018172 Hypothetical zinc-type alcohol dehydrogenase-like protein YphC\n",
      "b2545 ['Hypothetical zinc-type alcohol dehydrogenase-like protein YphC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10440 ribose transport system permease protein\n",
      "KO KO:K23546 Unknown\n",
      "SSO SSO:000022321 Predicted sugar ABC transport system, permease protein YphD\n",
      "b2546 ['Predicted sugar ABC transport system, permease protein YphD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K23547 Unknown\n",
      "SSO SSO:000022319 Predicted sugar ABC transport system, ATP-binding protein YphE\n",
      "b2547 ['Predicted sugar ABC transport system, ATP-binding protein YphE']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K23545 Unknown\n",
      "SSO SSO:000022320 Predicted sugar ABC transport system, periplasmic binding protein YphF precursor\n",
      "b2548 ['Predicted sugar ABC transport system, periplasmic binding protein YphF precursor']\n",
      "\n",
      "SSO SSO:000028498 Uncharacterized protein YphG, TPR-domain containing\n",
      "b2549 ['Uncharacterized protein YphG, TPR-domain containing']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000023084 Putative NAGC-like transcriptional regulator\n",
      "b2550 ['Putative NAGC-like transcriptional regulator']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006545 glycine biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K00600 glycine hydroxymethyltransferase [EC:2.1.2.1]\n",
      "SSO SSO:000007424 Serine hydroxymethyltransferase (EC 2.1.2.1)\n",
      "b2551 ['Serine hydroxymethyltransferase (EC 2.1.2.1)']\n",
      "\n",
      "KO KO:K05916 nitric oxide dioxygenase [EC:1.14.12.17]\n",
      "b2552 ['Flavohemoglobin', 'Nitric oxide dioxygenase (EC 1.14.12.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "KO KO:K04751 nitrogen regulatory protein P-II 1\n",
      "KO KO:K04752 nitrogen regulatory protein P-II 2\n",
      "SSO SSO:000005355 Nitrogen regulatory protein P-II\n",
      "b2553 ['Nitrogen regulatory protein P-II']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K07715 two-component system, NtrC family, response regulator GlrR\n",
      "SSO SSO:000008284 Transcriptional response regulatory protein GlrR\n",
      "b2554 ['Transcriptional response regulatory protein GlrR']\n",
      "\n",
      "SSO SSO:000028296 Uncharacterized protein YfhG\n",
      "b2555 ['Uncharacterized protein YfhG']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K07711 two-component system, NtrC family, sensor histidine kinase GlrK [EC:2.7.13.3]\n",
      "SSO SSO:000007390 Sensor histidine kinase GlrK\n",
      "b2556 ['Sensor histidine kinase GlrK']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K01952 phosphoribosylformylglycinamidine synthase [EC:6.3.5.3]\n",
      "KO KO:K23265 Unknown\n",
      "b2557 ['Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)', 'Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)']\n",
      "\n",
      "EC EC:3.2.1.14 Chitinase.\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K18691 membrane-bound lytic murein transglycosylase F [EC:4.2.2.-]\n",
      "SSO SSO:000019479 Membrane-bound lytic murein transglycosylase F (EC 4.2.2.n1)\n",
      "b2558 ['Membrane-bound lytic murein transglycosylase F (EC 4.2.2.n1)']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K11991 tRNA(adenine34) deaminase [EC:3.5.4.33]\n",
      "SSO SSO:000009404 tRNA-specific adenosine-34 deaminase (EC 3.5.4.33)\n",
      "b2559 ['tRNA-specific adenosine-34 deaminase (EC 3.5.4.33)']\n",
      "\n",
      "KO KO:K18697 phosphatidylglycerophosphatase C [EC:3.1.3.27]\n",
      "SSO SSO:000021623 Phosphatidylglycerophosphatase C (EC 3.1.3.27)\n",
      "b2560 ['Phosphatidylglycerophosphatase C (EC 3.1.3.27)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000026882 Uncharacterized HTH-type transcriptional regulator YfhH\n",
      "b2561 ['Uncharacterized HTH-type transcriptional regulator YfhH']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "SSO SSO:000027079 Uncharacterized ferredoxin-like protein YfhL\n",
      "b2562 ['Uncharacterized ferredoxin-like protein YfhL']\n",
      "\n",
      "SSO SSO:000024893 Small toxic protein ShoB\n",
      "b4687 ['Small toxic protein ShoB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00997 holo-[acyl-carrier protein] synthase [EC:2.7.8.7]\n",
      "SSO SSO:000003470 Holo-[acyl-carrier protein] synthase (EC 2.7.8.7)\n",
      "b2563 ['Holo-[acyl-carrier-protein] synthase (EC 2.7.8.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "KO KO:K03474 pyridoxine 5-phosphate synthase [EC:2.6.99.2]\n",
      "SSO SSO:000006816 Pyridoxine 5'-phosphate synthase (EC 2.6.99.2)\n",
      "b2564 [\"Pyridoxine 5'-phosphate synthase (EC 2.6.99.2)\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03584 DNA repair protein RecO (recombination protein O)\n",
      "SSO SSO:000002096 DNA recombination and repair protein RecO\n",
      "b2565 ['DNA recombination and repair protein RecO']\n",
      "\n",
      "GO GO:0007049 cell cycle\n",
      "KO KO:K03595 GTPase\n",
      "SSO SSO:000002995 GTP-binding protein Era\n",
      "b2566 ['GTP-binding protein Era']\n",
      "\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K03685 ribonuclease III [EC:3.1.26.3]\n",
      "SSO SSO:000007084 Ribonuclease III (EC 3.1.26.3)\n",
      "b2567 ['Ribonuclease III (EC 3.1.26.3)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03100 signal peptidase I [EC:3.4.21.89]\n",
      "SSO SSO:000007517 Signal peptidase I (EC 3.4.21.89)\n",
      "b2568 ['Signal peptidase I (EC 3.4.21.89)']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K03596 GTP-binding protein LepA\n",
      "SSO SSO:000008302 Translation elongation factor LepA\n",
      "b2569 ['Translation elongation factor LepA']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K03803 sigma-E factor negative regulatory protein RseC\n",
      "SSO SSO:000007512 Sigma factor RpoE regulatory protein RseC\n",
      "b2570 ['Sigma factor RpoE regulatory protein RseC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K03598 sigma-E factor negative regulatory protein RseB\n",
      "SSO SSO:000007511 Sigma factor RpoE negative regulatory protein RseB precursor\n",
      "b2571 ['Sigma factor RpoE negative regulatory protein RseB precursor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K03597 sigma-E factor negative regulatory protein RseA\n",
      "SSO SSO:000007510 Sigma factor RpoE negative regulatory protein RseA\n",
      "SSO SSO:000010682 Anti-sigma factor\n",
      "b2572 ['Sigma factor RpoE negative regulatory protein RseA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K03088 RNA polymerase sigma-70 factor, ECF subfamily\n",
      "SSO SSO:000006903 RNA polymerase sigma factor RpoE\n",
      "b2573 ['RNA polymerase sigma factor RpoE']\n",
      "\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "KO KO:K00278 L-aspartate oxidase [EC:1.4.3.16]\n",
      "SSO SSO:000004128 L-aspartate oxidase (EC 1.4.3.16)\n",
      "b2574 ['L-aspartate oxidase (EC 1.4.3.16)']\n",
      "\n",
      "KO KO:K15460 tRNA1Val (adenine37-N6)-methyltransferase [EC:2.1.1.223]\n",
      "SSO SSO:000009370 tRNA(1)(Val) (adenine(37)-N(6))-methyltransferase (EC 2.1.1.223)\n",
      "b2575 ['tRNA(1)(Val) (adenine(37)-N(6))-methyltransferase (EC 2.1.1.223)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K05590 ATP-dependent RNA helicase SrmB [EC:3.6.4.13]\n",
      "SSO SSO:000000599 ATP-dependent RNA helicase SrmB\n",
      "SSO SSO:000009996 ATP dependent RNA helicase\n",
      "b2576 ['ATP-dependent RNA helicase SrmB']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000039696 Transcriptional regulator YfiE, LysR family\n",
      "b2577 ['Transcriptional regulator YfiE, LysR family']\n",
      "\n",
      "KO KO:K11249 cysteine/O-acetylserine efflux protein\n",
      "SSO SSO:000012572 Cysteine/O-acetylserine efflux protein\n",
      "b2578 ['Cysteine/O-acetylserine efflux protein']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K06866 autonomous glycyl radical cofactor\n",
      "b2579 ['Autonomous glycyl radical cofactor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03648 uracil-DNA glycosylase [EC:3.2.2.27]\n",
      "SSO SSO:000008851 Uracil-DNA glycosylase, family 1\n",
      "SSO SSO:000028867 Uracil DNA glycosylase\n",
      "b2580 ['Uracil-DNA glycosylase, family 1 (EC 3.2.2.27)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03214 RNA methyltransferase, TrmH family [EC:2.1.1.-]\n",
      "b2581 ['Uncharacterized tRNA/rRNA methyltransferase YfiF']\n",
      "\n",
      "KO KO:K03671 thioredoxin 1\n",
      "KO KO:K03672 thioredoxin 2 [EC:1.8.1.8]\n",
      "SSO SSO:000025619 Thioredoxin 2 (EC 1.8.1.8)\n",
      "b2582 ['Thioredoxin 2']\n",
      "\n",
      "KO KO:K05812 DTW domain-containing protein\n",
      "SSO SSO:000027059 Uncharacterized conserved protein YfiP, contains DTW domain\n",
      "b2583 ['Uncharacterized conserved protein YfiP, contains DTW domain']\n",
      "\n",
      "KO KO:K09181 acetyltransferase\n",
      "SSO SSO:000038725 Protein lysine acetyltransferase Pat\n",
      "b2584 ['Protein lysine acetyltransferase Pat (EC 2.3.1.-)']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00998 CDP-diacylglycerol---serine O-phosphatidyltransferase [EC:2.7.8.8]\n",
      "SSO SSO:000001303 CDP-diacylglycerol--serine O-phosphatidyltransferase (EC 2.7.8.8)\n",
      "b2585 ['CDP-diacylglycerol--serine O-phosphatidyltransferase (EC 2.7.8.8)']\n",
      "\n",
      "KO KO:K05811 putative lipoprotein\n",
      "SSO SSO:000020814 putative lipoprotein\n",
      "b2586 ['Putative outer membrane lipoprotein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03761 MFS transporter, MHS family, alpha-ketoglutarate permease\n",
      "SSO SSO:000010501 Alpha-ketoglutarate permease\n",
      "SSO SSO:000010502 Alpha-ketoglutarate transporter\n",
      "b2587 ['Alpha-ketoglutarate permease']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03695 ATP-dependent Clp protease ATP-binding subunit ClpB\n",
      "SSO SSO:000035081 Chaperone protein ClpB (ATP-dependent unfoldase)\n",
      "b2592 ['Chaperone protein ClpB (ATP-dependent unfoldase)']\n",
      "\n",
      "KO KO:K05810 polyphenol oxidase [EC:1.10.3.-]\n",
      "b2593 ['FIG00003370: Multicopper polyphenol oxidase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K06180 23S rRNA pseudouridine1911/1915/1917 synthase [EC:5.4.99.23]\n",
      "SSO SSO:000036575 LSU rRNA pseudouridine(1911/1915/1917) synthase (EC 5.4.99.23)\n",
      "b2594 ['LSU rRNA pseudouridine(1911/1915/1917) synthase (EC 5.4.99.23)']\n",
      "\n",
      "KO KO:K05807 outer membrane protein assembly factor BamD\n",
      "SSO SSO:000005534 Outer membrane beta-barrel assembly protein BamD\n",
      "b2595 ['Outer membrane beta-barrel assembly protein BamD']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K05809 ribosome-associated inhibitor A\n",
      "SSO SSO:000007151 Ribosome-associated inhibitor A\n",
      "b2597 ['Ribosome-associated inhibitor A']\n",
      "\n",
      "GO GO:0009094 L-phenylalanine biosynthetic process\n",
      "SSO SSO:000021576 Phe operon leader peptide\n",
      "b2598 ['Phe operon leader peptide']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006571 tyrosine biosynthetic process\n",
      "GO GO:0009094 L-phenylalanine biosynthetic process\n",
      "KO KO:K04518 prephenate dehydratase [EC:4.2.1.51]\n",
      "KO KO:K14170 chorismate mutase / prephenate dehydratase [EC:5.4.99.5 4.2.1.51]\n",
      "b2599 ['Chorismate mutase I (EC 5.4.99.5)', 'Prephenate dehydratase (EC 4.2.1.51)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006571 tyrosine biosynthetic process\n",
      "GO GO:0009094 L-phenylalanine biosynthetic process\n",
      "KO KO:K14187 chorismate mutase / prephenate dehydrogenase [EC:5.4.99.5 1.3.1.12]\n",
      "b2600 ['Chorismate mutase I (EC 5.4.99.5)', 'Cyclohexadienyl dehydrogenase (EC 1.3.1.12)(EC 1.3.1.43)']\n",
      "\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K01626 3-deoxy-7-phosphoheptulonate synthase [EC:2.5.1.54]\n",
      "SSO SSO:000000169 2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I alpha (EC 2.5.1.54)\n",
      "b2601 ['2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I alpha (EC 2.5.1.54)']\n",
      "\n",
      "SSO SSO:000018986 Lipoprotein\n",
      "SSO SSO:000028298 Uncharacterized protein YfiL\n",
      "b2602 ['Uncharacterized protein YfiL']\n",
      "\n",
      "SSO SSO:000028299 Uncharacterized protein YfiR\n",
      "b2603 ['Uncharacterized protein YfiR']\n",
      "\n",
      "KO KO:K21021 diguanylate cyclase [EC:2.7.7.65]\n",
      "b2604 ['Diguanylate cyclase (EC 2.7.7.65) => DgcN']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "b2605 ['Integral membrane protein YfiB']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02884 large subunit ribosomal protein L19\n",
      "SSO SSO:000004233 LSU ribosomal protein L19p\n",
      "b2606 ['LSU ribosomal protein L19p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00554 tRNA (guanine37-N1)-methyltransferase [EC:2.1.1.228]\n",
      "SSO SSO:000009340 tRNA (guanine(37)-N(1))-methyltransferase (EC 2.1.1.228)\n",
      "b2607 ['tRNA (guanine(37)-N(1))-methyltransferase (EC 2.1.1.228)']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K02860 16S rRNA processing protein RimM\n",
      "SSO SSO:000000065 16S rRNA processing protein RimM\n",
      "b2608 ['16S rRNA processing protein RimM']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02959 small subunit ribosomal protein S16\n",
      "SSO SSO:000007266 SSU ribosomal protein S16p\n",
      "b2609 ['SSU ribosomal protein S16p']\n",
      "\n",
      "KO KO:K03106 signal recognition particle subunit SRP54 [EC:3.6.5.4]\n",
      "SSO SSO:000007523 Signal recognition particle protein Ffh\n",
      "b2610 ['Signal recognition particle protein Ffh']\n",
      "\n",
      "SSO SSO:000018426 Inner membrane protein YpjD\n",
      "b2611 ['Inner membrane protein YpjD']\n",
      "\n",
      "KO KO:K03699 putative hemolysin\n",
      "SSO SSO:000023416 Putative membrane protein YfjD\n",
      "b4461 ['Putative membrane protein YfjD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03687 molecular chaperone GrpE\n",
      "SSO SSO:000003351 Heat shock protein GrpE\n",
      "SSO SSO:000017646 Heat shock protein\n",
      "b2614 ['Heat shock protein GrpE']\n",
      "\n",
      "KO KO:K00858 NAD+ kinase [EC:2.7.1.23]\n",
      "SSO SSO:000005159 NAD kinase (EC 2.7.1.23)\n",
      "b2615 ['NAD kinase (EC 2.7.1.23)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03631 DNA repair protein RecN (Recombination protein N)\n",
      "SSO SSO:000002109 DNA repair protein RecN\n",
      "b2616 ['DNA repair protein RecN']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06186 outer membrane protein assembly factor BamE\n",
      "SSO SSO:000005535 Outer membrane beta-barrel assembly protein BamE\n",
      "b2617 ['Outer membrane beta-barrel assembly protein BamE']\n",
      "\n",
      "KO KO:K09801 uncharacterized protein\n",
      "b2618 ['UPF0125 protein RatB']\n",
      "\n",
      "KO KO:K18588 coenzyme Q-binding protein COQ10\n",
      "SSO SSO:000039037 Ribosome association toxin RatA\n",
      "b2619 ['Ribosome association toxin RatA']\n",
      "\n",
      "KO KO:K03664 SsrA-binding protein\n",
      "SSO SSO:000009411 tmRNA-binding protein SmpB\n",
      "b2620 ['tmRNA-binding protein SmpB']\n",
      "\n",
      "b2622 ['Prophage CP4-57 integrase']\n",
      "\n",
      "SSO SSO:000032592 putative histone\n",
      "b2623 ['putative histone']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07733 prophage regulatory protein\n",
      "b2624 ['Prophage CP4-57 regulatory protein alpA']\n",
      "\n",
      "SSO SSO:000028300 Uncharacterized protein YfjI\n",
      "b2625 ['Uncharacterized protein YfjI']\n",
      "\n",
      "SSO SSO:000028301 Uncharacterized protein YfjJ\n",
      "b2626 ['Uncharacterized protein YfjJ']\n",
      "\n",
      "SSO SSO:000010027 ATP-dependent DNA helicase\n",
      "b2627 ['ATP-dependent DNA helicase']\n",
      "\n",
      "SSO SSO:000028302 Uncharacterized protein YfjL\n",
      "b2628 ['Uncharacterized protein YfjL']\n",
      "\n",
      "SSO SSO:000028303 Uncharacterized protein YfjM\n",
      "b2629 ['Uncharacterized protein YfjM']\n",
      "\n",
      "SSO SSO:000009177 mRNA endoribonuclease LS\n",
      "b2630 ['mRNA endoribonuclease LS']\n",
      "\n",
      "SSO SSO:000010730 Antitoxin RnlB\n",
      "b2631 ['Antitoxin RnlB']\n",
      "\n",
      "KO KO:K06946 uncharacterized protein\n",
      "SSO SSO:000028304 Uncharacterized protein YfjP\n",
      "b2632 ['Uncharacterized protein YfjP']\n",
      "\n",
      "SSO SSO:000026658 UPF0380 proteins YafZ and homologs\n",
      "b2633 ['UPF0380 proteins YafZ and homologs']\n",
      "\n",
      "SSO SSO:000028754 Uncharacterized transcriptional regulator YfjR, DeoR family\n",
      "b2634 ['Uncharacterized transcriptional regulator YfjR, DeoR family']\n",
      "\n",
      "b2635 ['predicted inner membrane protein']\n",
      "\n",
      "SSO SSO:000027201 Uncharacterized lipoprotein YafY\n",
      "b2636 ['Uncharacterized lipoprotein YafY']\n",
      "\n",
      "SSO SSO:000028427 Uncharacterized protein YkfB\n",
      "b2637 ['Uncharacterized protein YkfB']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2638 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2641 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028306 Uncharacterized protein YfjW\n",
      "b2642 ['Uncharacterized protein YfjW']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4644 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000010725 Antirestriction protein klcA\n",
      "b2643 ['Antirestriction protein klcA']\n",
      "\n",
      "KO KO:K03630 DNA repair protein RadC\n",
      "SSO SSO:000026713 UPF0758 protein YfjY\n",
      "b2644 ['UPF0758 protein YfjY']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "b4548 ['Uncharacterized protein YkfH']\n",
      "\n",
      "KO KO:K18838 cytoskeleton bundling-enhancing protein CbeA and related proteins\n",
      "SSO SSO:000029139 YfjZ protein (antitoxin to YpjF)\n",
      "b2645 ['YfjZ protein (antitoxin to YpjF)']\n",
      "\n",
      "KO KO:K18837 cytoskeleton-binding toxin CbtA and related proteins\n",
      "SSO SSO:000029159 YpjF toxin protein\n",
      "b2646 ['YpjF toxin protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000027380 Uncharacterized outer membrane protein YpjA\n",
      "b2647 ['Uncharacterized outer membrane protein YpjA']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2648 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2650 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4462 ['hypothetical protein']\n",
      "\n",
      "KO KO:K15737 glutarate hydroxylase [EC:1.14.11.-]\n",
      "SSO SSO:000001416 Carbon starvation induced protein CsiD\n",
      "b2659 ['Carbon starvation induced protein CsiD']\n",
      "\n",
      "KO KO:K15736 L-2-hydroxyglutarate oxidase [EC:1.1.5.-]\n",
      "SSO SSO:000004108 L-2-hydroxyglutarate oxidase (EC 1.1.3.15)\n",
      "b2660 ['L-2-hydroxyglutarate oxidase (EC 1.1.3.15)']\n",
      "\n",
      "GO GO:0009447 putrescine catabolic process\n",
      "GO GO:0009450 gamma-aminobutyric acid catabolic process\n",
      "KO KO:K00135 succinate-semialdehyde dehydrogenase / glutarate-semialdehyde dehydrogenase [EC:1.2.1.16 1.2.1.79 1.2.1.20]\n",
      "SSO SSO:000007783 Succinate-semialdehyde dehydrogenase [NADP+] (EC 1.2.1.16)\n",
      "b2661 ['Succinate-semialdehyde dehydrogenase [NAD(P)+] (EC 1.2.1.16)']\n",
      "\n",
      "GO GO:0009447 putrescine catabolic process\n",
      "GO GO:0009450 gamma-aminobutyric acid catabolic process\n",
      "KO KO:K00823 4-aminobutyrate aminotransferase [EC:2.6.1.19]\n",
      "KO KO:K07250 4-aminobutyrate aminotransferase / (S)-3-amino-2-methylpropionate transaminase / 5-aminovalerate transaminase [EC:2.6.1.19 2.6.1.22 2.6.1.48]\n",
      "SSO SSO:000003026 Gamma-aminobutyrate:alpha-ketoglutarate aminotransferase (EC 2.6.1.19)\n",
      "b2662 ['Gamma-aminobutyrate:alpha-ketoglutarate aminotransferase (EC 2.6.1.19)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11735 GABA permease\n",
      "SSO SSO:000009116 gamma-aminobutyrate (GABA) permease\n",
      "b2663 ['gamma-aminobutyrate (GABA) permease']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K15735 GntR family transcriptional regulator, carbon starvation induced regulator\n",
      "SSO SSO:000001794 CsiR, transcriptional repressor of CsiD\n",
      "b2664 ['CsiR, transcriptional repressor of CsiD']\n",
      "\n",
      "SSO SSO:000026888 Uncharacterized LysM domain protein YgaU\n",
      "b2665 ['Uncharacterized LysM domain protein YgaU']\n",
      "\n",
      "SSO SSO:000026490 UPF0057 membrane protein YqaE\n",
      "b2666 ['UPF0057 membrane protein YqaE']\n",
      "\n",
      "KO KO:K22042 ArsR family transcriptional regulator, virulence genes transcriptional regulator\n",
      "SSO SSO:000025868 Transcriptional regulator YgaV, ArsR family\n",
      "b2667 ['Transcriptional regulator YgaV, ArsR family']\n",
      "\n",
      "SSO SSO:000024116 Rhodanese-domain-containing inner membrane protein YgaP\n",
      "b2668 ['Rhodanese-domain-containing inner membrane protein YgaP']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03746 DNA-binding protein H-NS\n",
      "KO KO:K11685 DNA-binding protein StpA\n",
      "SSO SSO:000012907 DNA-binding protein StpA\n",
      "b2669 ['DNA-binding protein StpA']\n",
      "\n",
      "SSO SSO:000018731 L-alanine exporter AlaE\n",
      "b2670 ['L-alanine exporter AlaE']\n",
      "\n",
      "SSO SSO:000028312 Uncharacterized protein YgaC\n",
      "b2671 ['Uncharacterized protein YgaC']\n",
      "\n",
      "KO KO:K05594 ElaB protein\n",
      "SSO SSO:000013339 ElaB protein\n",
      "b2672 ['ElaB protein']\n",
      "\n",
      "KO KO:K06191 glutaredoxin-like protein NrdH\n",
      "SSO SSO:000009130 glutaredoxin-like protein NrdH, required for reduction of Ribonucleotide reductase class Ib\n",
      "b2673 ['Glutaredoxin-like protein NrdH, required for reduction of Ribonucleotide reductase class Ib']\n",
      "\n",
      "KO KO:K03647 protein involved in ribonucleotide reduction\n",
      "SSO SSO:000007107 Ribonucleotide reduction protein NrdI\n",
      "b2674 ['Ribonucleotide reduction protein NrdI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00525 ribonucleoside-diphosphate reductase alpha chain [EC:1.17.4.1]\n",
      "SSO SSO:000007104 Ribonucleotide reductase of class Ib (aerobic), alpha subunit (EC 1.17.4.1)\n",
      "b2675 ['Ribonucleotide reductase of class Ib (aerobic), alpha subunit (EC 1.17.4.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00526 ribonucleoside-diphosphate reductase beta chain [EC:1.17.4.1]\n",
      "SSO SSO:000007105 Ribonucleotide reductase of class Ib (aerobic), beta subunit (EC 1.17.4.1)\n",
      "b2676 ['Ribonucleotide reductase of class Ib (aerobic), beta subunit (EC 1.17.4.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "GO GO:0015596 obsolete glycine betaine/proline porter activity\n",
      "KO KO:K02000 glycine betaine/proline transport system ATP-binding protein [EC:3.6.3.32]\n",
      "b2677 ['Glycine betaine/L-proline transport ATP-binding protein ProV (TC 3.A.1.12.1)']\n",
      "\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015596 obsolete glycine betaine/proline porter activity\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02001 glycine betaine/proline transport system permease protein\n",
      "b2678 ['Glycine betaine/L-proline transport system permease protein ProW (TC 3.A.1.12.1)']\n",
      "\n",
      "GO GO:0006545 glycine biosynthetic process\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "GO GO:0006578 amino-acid betaine biosynthetic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0015596 obsolete glycine betaine/proline porter activity\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02002 glycine betaine/proline transport system substrate-binding protein\n",
      "b2679 ['Glycine betaine/L-proline transport substrate-binding protein ProX (TC 3.A.1.12.1)']\n",
      "\n",
      "GO GO:0006950 response to stress\n",
      "SSO SSO:000040008 Uncharacterized MFS-type transporter STM2812\n",
      "b2681 ['Uncharacterized MFS-type transporter STM2812']\n",
      "\n",
      "b2682 ['Inner membrane protein YgaZ']\n",
      "\n",
      "b2683 ['Uncharacterized protein YgaH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K15974 MarR family transcriptional regulator, negative regulator of the multidrug operon emrRAB\n",
      "SSO SSO:000005025 Multidrug resistance regulator EmrR (MprA)\n",
      "b2684 ['Multidrug resistance regulator EmrR (MprA)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03543 membrane fusion protein, multidrug efflux system\n",
      "SSO SSO:000004975 Multidrug efflux system EmrAB-OMF, membrane fusion component EmrA\n",
      "b2685 ['Multidrug efflux system EmrAB-OMF, membrane fusion component EmrA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03446 MFS transporter, DHA2 family, multidrug resistance protein\n",
      "SSO SSO:000004974 Multidrug efflux system EmrAB-OMF, inner-membrane proton/drug antiporter EmrB (MFS type)\n",
      "b2686 ['Multidrug efflux system EmrAB-OMF, inner-membrane proton/drug antiporter EmrB (MFS type)']\n",
      "\n",
      "KO KO:K07173 S-ribosylhomocysteine lyase [EC:4.4.1.21]\n",
      "SSO SSO:000007208 S-ribosylhomocysteine lyase (EC 4.4.1.21)\n",
      "b2687 ['S-ribosylhomocysteine lyase (EC 4.4.1.21)', 'Autoinducer-2 production protein LuxS']\n",
      "\n",
      "GO GO:0006750 glutathione biosynthetic process\n",
      "KO KO:K01919 glutamate--cysteine ligase [EC:6.3.2.2]\n",
      "SSO SSO:000003160 Glutamate--cysteine ligase (EC 6.3.2.2)\n",
      "b2688 ['Glutamate--cysteine ligase (EC 6.3.2.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000018427 Inner membrane protein YqaA\n",
      "b2689 ['Inner membrane protein YqaA']\n",
      "\n",
      "SSO SSO:000035973 Fructose-1-phosphate phosphatase YqaB\n",
      "b2690 ['Fructose-1-phosphate phosphatase YqaB (EC 3.1.3.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0006401 RNA catabolic process\n",
      "GO GO:0009386 translational attenuation\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03563 carbon storage regulator\n",
      "SSO SSO:000001418 Carbon storage regulator\n",
      "b2696 ['Carbon storage regulator']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01872 alanyl-tRNA synthetase [EC:6.1.1.7]\n",
      "SSO SSO:000000850 Alanyl-tRNA synthetase (EC 6.1.1.7)\n",
      "b2697 ['Alanyl-tRNA synthetase (EC 6.1.1.7)']\n",
      "\n",
      "KO KO:K03565 regulatory protein\n",
      "SSO SSO:000006977 Regulatory protein RecX\n",
      "b2698 ['Regulatory protein RecX']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0006508 proteolysis\n",
      "GO GO:0008233 peptidase activity\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03553 recombination protein RecA\n",
      "SSO SSO:000006953 RecA protein\n",
      "b2699 ['RecA protein']\n",
      "\n",
      "KO KO:K03743 nicotinamide-nucleotide amidase [EC:3.5.1.42]\n",
      "SSO SSO:000005306 Nicotinamide-nucleotide amidase (EC 3.5.1.42)\n",
      "b2700 ['Nicotinamide-nucleotide amidase (EC 3.5.1.42)']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K08305 membrane-bound lytic murein transglycosylase B [EC:4.2.2.-]\n",
      "SSO SSO:000004782 Membrane-bound lytic murein transglycosylase B (EC 3.2.1.-)\n",
      "b2701 ['Membrane-bound lytic murein transglycosylase B']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02783 PTS system, glucitol/sorbitol-specific IIC component\n",
      "SSO SSO:000005694 PTS system, glucitol/sorbitol-specific IIC component (EC 2.7.1.69)\n",
      "b2702 ['PTS system, glucitol/sorbitol-specific IIC component']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02782 PTS system, glucitol/sorbitol-specific IIB component [EC:2.7.1.198]\n",
      "b2703 ['PTS system, glucitol/sorbitol-specific IIB component (EC 2.7.1.198)', 'PTS system, glucitol/sorbitol-specific IIC component 2']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02781 PTS system, glucitol/sorbitol-specific IIA component [EC:2.7.1.198]\n",
      "SSO SSO:000005692 PTS system, glucitol/sorbitol-specific IIA component (EC 2.7.1.69)\n",
      "b2704 ['PTS system, glucitol/sorbitol-specific IIA component (EC 2.7.1.198)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00068 sorbitol-6-phosphate 2-dehydrogenase [EC:1.1.1.140]\n",
      "SSO SSO:000017873 Sorbitol-6-phosphate 2-dehydrogenase (EC 1.1.1.140)\n",
      "b2705 ['Sorbitol-6-phosphate 2-dehydrogenase (EC 1.1.1.140)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02466 glucitol operon activator protein\n",
      "b2706 ['Glucitol operon activator protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02468 DeoR family transcriptional regulator, glucitol operon repressor\n",
      "b2707 ['Glucitol operon repressor']\n",
      "\n",
      "KO KO:K02467 arabinose 5-phosphate isomerase [EC:5.3.1.13]\n",
      "KO KO:K06041 arabinose-5-phosphate isomerase [EC:5.3.1.13]\n",
      "SSO SSO:000002013 D-arabinose 5-phosphate isomerase (EC 5.3.1.13)\n",
      "b2708 ['D-arabinose-5-phosphate isomerase (EC 5.3.1.13)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K12266 anaerobic nitric oxide reductase transcription regulator\n",
      "SSO SSO:000010648 Anaerobic nitric oxide reductase transcription regulator NorR\n",
      "b2709 ['Anaerobic nitric oxide reductase transcription regulator NorR']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K12264 anaerobic nitric oxide reductase flavorubredoxin\n",
      "SSO SSO:000010647 Anaerobic nitric oxide reductase flavorubredoxin\n",
      "b2710 ['Anaerobic nitric oxide reductase flavorubredoxin']\n",
      "\n",
      "KO KO:K12265 nitric oxide reductase FlRd-NAD(+) reductase [EC:1.18.1.-]\n",
      "SSO SSO:000020543 Nitric oxide reductase FlRd-NAD(+) reductase (EC 1.18.1.-)\n",
      "b2711 ['Nitric oxide reductase FlRd-NAD(+) reductase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04656 hydrogenase maturation protein HypF\n",
      "b2712 ['Acylphosphate phosphohydrolase (EC 3.6.1.7)', '[NiFe] hydrogenase metallocenter assembly protein HypF']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K05796 electron transport protein HydN\n",
      "SSO SSO:000013343 Electron transport protein HydN\n",
      "b2713 ['Electron transport protein HydN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "KO KO:K03487 LacI family transcriptional regulator, asc operon repressor\n",
      "b2714 ['AscBF operon repressor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02753 PTS system, beta-glucoside (arbutin/salicin/cellobiose)-specific IIC component\n",
      "b2715 ['PTS system, arbutin-, cellobiose-, and salicin-specific IIB component', 'PTS system, arbutin-, cellobiose-, and salicin-specific IIC component']\n",
      "\n",
      "EC EC:3.2.1.105 3-alpha-(S)-strictosidine beta-glucosidase.\n",
      "EC EC:3.2.1.108 Lactase.\n",
      "EC EC:3.2.1.117 Amygdalin beta-glucosidase.\n",
      "EC EC:3.2.1.118 Prunasin beta-glucosidase.\n",
      "EC EC:3.2.1.119 Vicianin beta-glucosidase.\n",
      "EC EC:3.2.1.125 Raucaffricine beta-glucosidase.\n",
      "EC EC:3.2.1.147 Thioglucosidase.\n",
      "EC EC:3.2.1.149 Beta-primeverosidase.\n",
      "EC EC:3.2.1.161 Beta-apiosyl-beta-glucosidase.\n",
      "EC EC:3.2.1.175 Beta-D-glucopyranosyl abscisate beta-glucosidase.\n",
      "EC EC:3.2.1.182 4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl glucosidebeta-D-glucosidase.\n",
      "EC EC:3.2.1.21 Beta-glucosidase.\n",
      "EC EC:3.2.1.23 Beta-galactosidase.\n",
      "EC EC:3.2.1.25 Beta-mannosidase.\n",
      "EC EC:3.2.1.31 Beta-glucuronidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.38 Beta-D-fucosidase.\n",
      "EC EC:3.2.1.62 Glycosylceramidase.\n",
      "EC EC:3.2.1.74 Glucan 1,4-beta-glucosidase.\n",
      "EC EC:3.2.1.85 6-phospho-beta-galactosidase.\n",
      "EC EC:3.2.1.86 6-phospho-beta-glucosidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01223 6-phospho-beta-glucosidase [EC:3.2.1.86]\n",
      "SSO SSO:000000451 6-phospho-beta-glucosidase (EC 3.2.1.86)\n",
      "b2716 ['6-phospho-beta-glucosidase (EC 3.2.1.86)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K08315 hydrogenase 3 maturation protease [EC:3.4.23.51]\n",
      "SSO SSO:000035960 Formate hydrogenlyase maturation protease (EC 3.4.23.51)\n",
      "b2717 ['Formate hydrogenlyase maturation protease (EC 3.4.23.51)']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K15834 formate hydrogenlyase maturation protein HycH\n",
      "SSO SSO:000009112 formate hydrogenlyase maturation protein HycH\n",
      "b2718 ['Formate hydrogenlyase maturation protein HycH']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15832 formate hydrogenlyase subunit 7\n",
      "SSO SSO:000002895 Formate hydrogenlyase subunit 7\n",
      "b2719 ['Formate hydrogenlyase subunit 7']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15831 formate hydrogenlyase subunit 6\n",
      "SSO SSO:000002894 Formate hydrogenlyase subunit 6\n",
      "b2720 ['Formate hydrogenlyase subunit 6']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15830 formate hydrogenlyase subunit 5\n",
      "SSO SSO:000002893 Formate hydrogenlyase subunit 5\n",
      "b2721 ['Formate hydrogenlyase subunit 5']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15829 formate hydrogenlyase subunit 4\n",
      "SSO SSO:000002892 Formate hydrogenlyase subunit 4\n",
      "b2722 ['Formate hydrogenlyase subunit 4']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15828 formate hydrogenlyase subunit 3\n",
      "SSO SSO:000002891 Formate hydrogenlyase subunit 3\n",
      "b2723 ['Formate hydrogenlyase subunit 3']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K15827 formate hydrogenlyase subunit 2\n",
      "SSO SSO:000002890 Formate hydrogenlyase subunit 2\n",
      "b2724 ['Formate hydrogenlyase subunit 2']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K15833 formate hydrogenlyase regulatory protein HycA\n",
      "SSO SSO:000002889 Formate hydrogenlyase regulatory protein HycA\n",
      "b2725 ['Formate hydrogenlyase regulatory protein HycA']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04651 hydrogenase nickel incorporation protein HypA/HybF\n",
      "SSO SSO:000009025 [NiFe] hydrogenase nickel incorporation protein HypA\n",
      "b2726 ['[NiFe] hydrogenase nickel incorporation protein HypA']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04652 hydrogenase nickel incorporation protein HypB\n",
      "SSO SSO:000009026 [NiFe] hydrogenase nickel incorporation-associated protein HypB\n",
      "b2727 ['[NiFe] hydrogenase nickel incorporation-associated protein HypB']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04653 hydrogenase expression/formation protein HypC\n",
      "SSO SSO:000009019 [NiFe] hydrogenase metallocenter assembly protein HypC\n",
      "b2728 ['[NiFe] hydrogenase metallocenter assembly protein HypC']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04654 hydrogenase expression/formation protein HypD\n",
      "SSO SSO:000009020 [NiFe] hydrogenase metallocenter assembly protein HypD\n",
      "b2729 ['[NiFe] hydrogenase metallocenter assembly protein HypD']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04655 hydrogenase expression/formation protein HypE\n",
      "SSO SSO:000009021 [NiFe] hydrogenase metallocenter assembly protein HypE\n",
      "b2730 ['[NiFe] hydrogenase metallocenter assembly protein HypE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K15836 formate hydrogenlyase transcriptional activator\n",
      "SSO SSO:000016876 Formate hydrogenlyase transcriptional activator\n",
      "b2731 ['Formate hydrogenlyase transcriptional activator']\n",
      "\n",
      "SSO SSO:000028313 Uncharacterized protein YgbA\n",
      "b2732 ['Uncharacterized protein YgbA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03555 DNA mismatch repair protein MutS\n",
      "SSO SSO:000002076 DNA mismatch repair protein MutS\n",
      "b2733 ['DNA mismatch repair protein MutS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K07313 serine/threonine protein phosphatase 1 [EC:3.1.3.16]\n",
      "KO KO:K07314 serine/threonine protein phosphatase 2 [EC:3.1.3.16]\n",
      "SSO SSO:000024594 Serine/threonine protein phosphatase 2 (EC 3.1.3.16)\n",
      "SSO SSO:000024599 Serine/threonine specific protein phosphatase 2 (EC 3.1.3.16)\n",
      "b2734 ['Serine/threonine protein phosphatase 2 (EC 3.1.3.16)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000036558 L-threonate/D-erythronate transcriptional regulator, DeoR family\n",
      "b2735 ['L-threonate/D-erythronate transcriptional regulator, DeoR family']\n",
      "\n",
      "KO KO:K00020 3-hydroxyisobutyrate dehydrogenase [EC:1.1.1.31]\n",
      "KO KO:K08319 L-threonate 2-dehydrogenase [EC:1.1.1.411]\n",
      "SSO SSO:000023250 Putative dehydrogenase\n",
      "b2736 ['L-threonate dehydrogenase (NAD+)']\n",
      "\n",
      "KO KO:K21948 3-dehydrotetronate 4-kinase [EC:2.7.1.217]\n",
      "SSO SSO:000034079 3-oxo-tetronate kinase\n",
      "b2737 ['3-oxo-tetronate kinase']\n",
      "\n",
      "KO KO:K22130 3-dehydro-4-phosphotetronate decarboxylase [EC:4.1.1.104]\n",
      "SSO SSO:000034078 3-oxo-tetronate 4-phosphate decarboxylase\n",
      "b2738 ['3-oxo-tetronate 4-phosphate decarboxylase']\n",
      "\n",
      "KO KO:K01816 hydroxypyruvate isomerase [EC:5.3.1.22]\n",
      "KO KO:K22131 2-dehydrotetronate isomerase [EC:5.3.1.35]\n",
      "SSO SSO:000033966 2-dehydrotetronate isomerase (EC 5.3.1.35)\n",
      "b2739 ['2-dehydrotetronate isomerase (EC 5.3.1.35)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03299 gluconate:H+ symporter, GntP family\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000036560 L-threonate/D-erythronate, proton symporter\n",
      "b2740 ['L-threonate/D-erythronate, proton symporter']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K03087 RNA polymerase nonessential primary-like sigma factor\n",
      "SSO SSO:000006908 RNA polymerase sigma factor RpoS\n",
      "b2741 ['RNA polymerase sigma factor RpoS']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06194 lipoprotein NlpD\n",
      "SSO SSO:000005037 Murein hydrolase activator NlpD\n",
      "b2742 ['Murein hydrolase activator NlpD']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K00573 protein-L-isoaspartate(D-aspartate) O-methyltransferase [EC:2.1.1.77]\n",
      "SSO SSO:000006532 Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)\n",
      "b2743 ['Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)']\n",
      "\n",
      "GO GO:0006950 response to stress\n",
      "KO KO:K03787 5'-nucleotidase [EC:3.1.3.5]\n",
      "b2744 [\"5'-nucleotidase SurE (EC 3.1.3.5)\", 'Exopolyphosphatase (EC 3.6.1.11)']\n",
      "\n",
      "KO KO:K06176 tRNA pseudouridine13 synthase [EC:5.4.99.27]\n",
      "SSO SSO:000009364 tRNA pseudouridine(13) synthase (EC 5.4.99.27)\n",
      "b2745 ['tRNA pseudouridine(13) synthase (EC 5.4.99.27)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01770 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase [EC:4.6.1.12]\n",
      "SSO SSO:000000115 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase (EC 4.6.1.12)\n",
      "b2746 ['2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase (EC 4.6.1.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00991 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase [EC:2.7.7.60]\n",
      "SSO SSO:000000116 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase (EC 2.7.7.60)\n",
      "SSO SSO:000009677 4-diphosphocytidyl-2C-methyl-D-erythritol synthase\n",
      "b2747 ['2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase (EC 2.7.7.60)']\n",
      "\n",
      "KO KO:K05589 cell division protein FtsB\n",
      "SSO SSO:000001464 Cell division protein DivIC (FtsB), stabilizes FtsL against RasP cleavage\n",
      "SSO SSO:000012031 Cell division protein\n",
      "b2748 ['Cell division protein DivIC (FtsB), stabilizes FtsL against RasP cleavage']\n",
      "\n",
      "SSO SSO:000018398 Inner membrane protein YgbE\n",
      "b2749 ['Inner membrane protein YgbE']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K00860 adenylylsulfate kinase [EC:2.7.1.25]\n",
      "SSO SSO:000000826 Adenylylsulfate kinase (EC 2.7.1.25)\n",
      "b2750 ['Adenylylsulfate kinase (EC 2.7.1.25)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K00956 sulfate adenylyltransferase subunit 1 [EC:2.7.7.4]\n",
      "SSO SSO:000007804 Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)\n",
      "b2751 ['Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K00957 sulfate adenylyltransferase subunit 2 [EC:2.7.7.4]\n",
      "SSO SSO:000007805 Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)\n",
      "b2752 ['Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K09612 alkaline phosphatase isozyme conversion protein [EC:3.4.11.-]\n",
      "SSO SSO:000010407 Alkaline phosphatase isozyme conversion protein\n",
      "b2753 ['Alkaline phosphatase isozyme conversion protein']\n",
      "\n",
      "KO KO:K09951 CRISPR-associated protein Cas2\n",
      "SSO SSO:000001341 CRISPR-associated protein Cas2\n",
      "b2754 ['CRISPR-associated protein Cas2']\n",
      "\n",
      "KO KO:K15342 CRISP-associated protein Cas1\n",
      "SSO SSO:000001340 CRISPR-associated protein Cas1\n",
      "b2755 ['CRISPR-associated protein Cas1']\n",
      "\n",
      "KO KO:K19126 CRISPR system Cascade subunit CasE\n",
      "SSO SSO:000001369 CRISPR-associated protein, Cse3 family\n",
      "b2756 ['CRISPR-associated protein, Cse3 family']\n",
      "\n",
      "KO KO:K19125 CRISPR system Cascade subunit CasD\n",
      "SSO SSO:000001360 CRISPR-associated protein, Cas5e family\n",
      "b2757 ['CRISPR-associated protein, Cas5e family']\n",
      "\n",
      "KO KO:K19124 CRISPR system Cascade subunit CasC\n",
      "SSO SSO:000001370 CRISPR-associated protein, Cse4 family\n",
      "b2758 ['CRISPR-associated protein, Cse4 family']\n",
      "\n",
      "KO KO:K19046 CRISPR system Cascade subunit CasB\n",
      "SSO SSO:000001368 CRISPR-associated protein, Cse2 family\n",
      "b2759 ['CRISPR-associated protein, Cse2 family']\n",
      "\n",
      "KO KO:K19123 CRISPR system Cascade subunit CasA\n",
      "SSO SSO:000001367 CRISPR-associated protein, Cse1 family\n",
      "b2760 ['CRISPR-associated protein, Cse1 family']\n",
      "\n",
      "KO KO:K07012 CRISPR-associated endonuclease/helicase Cas3 [EC:3.1.-.- 3.6.4.-]\n",
      "SSO SSO:000001337 CRISPR-associated helicase Cas3\n",
      "b2761 ['CRISPR-associated helicase Cas3']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K00390 phosphoadenosine phosphosulfate reductase [EC:1.8.4.8 1.8.4.10]\n",
      "SSO SSO:000005950 Phosphoadenylyl-sulfate reductase [thioredoxin] (EC 1.8.4.8)\n",
      "b2762 ['Phosphoadenylyl-sulfate reductase [thioredoxin] (EC 1.8.4.8)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K00381 sulfite reductase (NADPH) hemoprotein beta-component [EC:1.8.1.2]\n",
      "SSO SSO:000007822 Sulfite reductase [NADPH] hemoprotein beta-component (EC 1.8.1.2)\n",
      "b2763 ['Sulfite reductase [NADPH] hemoprotein beta-component (EC 1.8.1.2)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K00380 sulfite reductase (NADPH) flavoprotein alpha-component [EC:1.8.1.2]\n",
      "SSO SSO:000007821 Sulfite reductase [NADPH] flavoprotein alpha-component (EC 1.8.1.2)\n",
      "b2764 ['Sulfite reductase [NADPH] flavoprotein alpha-component (EC 1.8.1.2)']\n",
      "\n",
      "KO KO:K01737 6-pyruvoyltetrahydropterin/6-carboxytetrahydropterin synthase [EC:4.2.3.12 4.1.2.50]\n",
      "SSO SSO:000009744 6-carboxy-5,6,7,8-tetrahydropterin synthase (EC 4.1.2.50)\n",
      "b2765 ['6-carboxy-5,6,7,8-tetrahydropterin synthase (EC 4.1.2.50)']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K00313 electron transfer flavoprotein-quinone oxidoreductase [EC:1.5.5.-]\n",
      "SSO SSO:000002445 Electron transfer flavoprotein-quinone oxidoreductase YgcN\n",
      "b2766 ['Electron transfer flavoprotein-quinone oxidoreductase YgcN']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K03855 ferredoxin like protein\n",
      "SSO SSO:000002711 Ferredoxin-like protein YgcO\n",
      "b2767 ['Ferredoxin-like protein YgcO']\n",
      "\n",
      "KO KO:K02443 glycerol uptake operon antiterminator\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2768 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K03522 electron transfer flavoprotein alpha subunit\n",
      "SSO SSO:000002430 Electron transfer flavoprotein, alpha subunit YgcQ\n",
      "b2769 ['Electron transfer flavoprotein, alpha subunit YgcQ']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K03521 electron transfer flavoprotein beta subunit\n",
      "SSO SSO:000002436 Electron transfer flavoprotein, beta subunit YgcR\n",
      "b2770 ['Electron transfer flavoprotein, beta subunit YgcR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08368 MFS transporter, putative metabolite transport protein\n",
      "SSO SSO:000018373 Inner membrane metabolite transport protein YgcS\n",
      "b2771 ['Inner membrane metabolite transport protein YgcS']\n",
      "\n",
      "KO KO:K00803 alkyldihydroxyacetonephosphate synthase [EC:2.5.1.26]\n",
      "SSO SSO:000022093 Predicted FAD containing dehydrogenase\n",
      "b4463 ['Predicted FAD containing dehydrogenase']\n",
      "\n",
      "SSO SSO:000027390 Uncharacterized oxidoreductase YgcW\n",
      "b2774 ['Uncharacterized oxidoreductase YgcW']\n",
      "\n",
      "SSO SSO:000018428 Inner membrane protein YqcE\n",
      "b2775 ['Inner membrane protein YqcE']\n",
      "\n",
      "SSO SSO:000023380 Putative kinase\n",
      "SSO SSO:000028703 Uncharacterized sugar kinase YgcE (EC 2.7.1.-)\n",
      "b2776 ['Uncharacterized sugar kinase YgcE (EC 2.7.1.-)']\n",
      "\n",
      "GO GO:0046138 obsolete coenzyme and prosthetic group biosynthetic process\n",
      "KO KO:K10026 7-carboxy-7-deazaguanine synthase [EC:4.3.99.3]\n",
      "SSO SSO:000009761 7-carboxy-7-deazaguanine synthase (EC 4.3.99.3)\n",
      "b2777 ['7-carboxy-7-deazaguanine synthase (EC 4.3.99.3)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4682 ['hypothetical protein']\n",
      "\n",
      "KO KO:K06872 uncharacterized protein\n",
      "SSO SSO:000028315 Uncharacterized protein YgcG\n",
      "b2778 ['Uncharacterized protein YgcG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006096 glycolytic process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K01689 enolase [EC:4.2.1.11]\n",
      "SSO SSO:000002536 Enolase (EC 4.2.1.11)\n",
      "b2779 ['Enolase (EC 4.2.1.11)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01937 CTP synthase [EC:6.3.4.2]\n",
      "SSO SSO:000001378 CTP synthase (EC 6.3.4.2)\n",
      "b2780 ['CTP synthase (EC 6.3.4.2)']\n",
      "\n",
      "KO KO:K04765 nucleoside triphosphate diphosphatase [EC:3.6.1.9]\n",
      "SSO SSO:000005427 Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)\n",
      "SSO SSO:000020652 Nucleoside Triphosphate Pyrophosphohydrolase (EC 3.6.1.8)\n",
      "b2781 ['Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)']\n",
      "\n",
      "KO KO:K07171 mRNA interferase MazF [EC:3.1.-.-]\n",
      "SSO SSO:000006354 Programmed cell death toxin MazF\n",
      "b2782 ['Programmed cell death toxin MazF']\n",
      "\n",
      "KO KO:K07172 antitoxin MazE\n",
      "SSO SSO:000006349 Programmed cell death antitoxin MazE\n",
      "b2783 ['Programmed cell death antitoxin MazE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00951 GTP pyrophosphokinase [EC:2.7.6.5]\n",
      "b2784 [\"Inactive (p)ppGpp 3'-pyrophosphohydrolase domain\", 'GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase I']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03215 23S rRNA (uracil1939-C5)-methyltransferase [EC:2.1.1.190]\n",
      "SSO SSO:000000241 23S rRNA (uracil(1939)-C(5))-methyltransferase (EC 2.1.1.190)\n",
      "b2785 ['23S rRNA (uracil(1939)-C(5))-methyltransferase (EC 2.1.1.190)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07678 two-component system, NarL family, sensor histidine kinase BarA [EC:2.7.13.3]\n",
      "KO KO:K20974 two-component system, sensor histidine kinase [EC:2.7.13.3]\n",
      "SSO SSO:000007528 Signal transduction histidine-protein kinase BarA (EC 2.7.13.3)\n",
      "b2786 ['Signal transduction histidine-protein kinase BarA (EC 2.7.13.3)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01706 glucarate dehydratase [EC:4.2.1.40]\n",
      "SSO SSO:000003095 Glucarate dehydratase (EC 4.2.1.40)\n",
      "b2787 ['Glucarate dehydratase (EC 4.2.1.40)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K13918 glucarate dehydratase-related protein\n",
      "SSO SSO:000003096 Glucarate dehydratase related protein YgcY\n",
      "b2788 ['Glucarate dehydratase related protein YgcY']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03535 MFS transporter, ACS family, glucarate transporter\n",
      "SSO SSO:000002021 D-glucarate transporter\n",
      "b2789 ['D-glucarate transporter']\n",
      "\n",
      "KO KO:K06205 MioC protein\n",
      "SSO SSO:000003568 Hypothetical flavoprotein YqcA (clustered with tRNA pseudouridine synthase C)\n",
      "b2790 ['Hypothetical flavoprotein YqcA (clustered with tRNA pseudouridine synthase C)']\n",
      "\n",
      "KO KO:K06175 tRNA pseudouridine65 synthase [EC:5.4.99.26]\n",
      "SSO SSO:000009367 tRNA pseudouridine(65) synthase (EC 5.4.99.26)\n",
      "b2791 ['tRNA pseudouridine(65) synthase (EC 5.4.99.26)']\n",
      "\n",
      "SSO SSO:000028508 Uncharacterized protein YqcC\n",
      "b2792 ['Uncharacterized protein YqcC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K15723 SecY interacting protein Syd\n",
      "SSO SSO:000025337 Syd protein\n",
      "b2793 ['Syd protein']\n",
      "\n",
      "KO KO:K06879 7-cyano-7-deazaguanine reductase [EC:1.7.1.13]\n",
      "SSO SSO:000020356 NADPH-dependent 7-cyano-7-deazaguanine reductase (EC 1.7.1.13)\n",
      "b2794 ['NADPH-dependent 7-cyano-7-deazaguanine reductase (EC 1.7.1.13)']\n",
      "\n",
      "KO KO:K06966 pyrimidine/purine-5'-nucleotide nucleosidase [EC:3.2.2.10 3.2.2.-]\n",
      "SSO SSO:000038858 Pyrimidine/purine nucleotide 5'-monophosphate nucleosidase PpnN (EC 3.2.2.4) (EC 3.2.2.10)\n",
      "b2795 [\"Pyrimidine/purine nucleotide 5'-monophosphate nucleosidase PpnN (EC 3.2.2.4) (EC 3.2.2.10)\"]\n",
      "\n",
      "GO GO:0006564 L-serine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03837 serine transporter\n",
      "SSO SSO:000007440 Serine transporter\n",
      "b2796 ['Serine transporter']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01752 L-serine dehydratase [EC:4.3.1.17]\n",
      "b2797 ['L-serine dehydratase, beta subunit (EC 4.3.1.17)', 'L-serine dehydratase, alpha subunit (EC 4.3.1.17)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K01146 protein Xni\n",
      "SSO SSO:000035931 Flap endonuclease Xni\n",
      "b2798 ['Flap endonuclease Xni']\n",
      "\n",
      "GO GO:0009441 glycolate metabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00048 lactaldehyde reductase [EC:1.1.1.77]\n",
      "SSO SSO:000004378 Lactaldehyde reductase (EC 1.1.1.77)\n",
      "b2799 ['Lactaldehyde reductase (EC 1.1.1.77)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01628 L-fuculose-phosphate aldolase [EC:4.1.2.17]\n",
      "SSO SSO:000004149 L-fuculose phosphate aldolase (EC 4.1.2.17)\n",
      "b2800 ['L-fuculose phosphate aldolase (EC 4.1.2.17)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02429 MFS transporter, FHS family, L-fucose permease\n",
      "SSO SSO:000002948 Fucose permease\n",
      "b2801 ['Fucose permease']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01818 L-fucose/D-arabinose isomerase [EC:5.3.1.25 5.3.1.3]\n",
      "SSO SSO:000004144 L-fucose isomerase (EC 5.3.1.25)\n",
      "b2802 ['L-fucose isomerase (EC 5.3.1.25)', 'D-Arabinose ketol-isomerase (EC 5.3.1.3)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00879 L-fuculokinase [EC:2.7.1.51]\n",
      "SSO SSO:000004148 L-fuculokinase (EC 2.7.1.51)\n",
      "b2803 ['L-fuculokinase (EC 2.7.1.51)']\n",
      "\n",
      "KO KO:K02431 L-fucose mutarotase [EC:5.1.3.29]\n",
      "SSO SSO:000004145 L-fucose mutarotase (EC 5.1.3.29)\n",
      "b2804 ['L-fucose mutarotase (EC 5.1.3.29)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02430 DeoR family transcriptional regulator, L-fucose operon activator\n",
      "SSO SSO:000004147 L-fucose operon activator\n",
      "b2805 ['L-fucose operon activator']\n",
      "\n",
      "KO KO:K06968 23S rRNA (cytidine2498-2'-O)-methyltransferase [EC:2.1.1.186]\n",
      "SSO SSO:000000230 23S rRNA (cytidine(2498)-2'-O)-methyltransferase (EC 2.1.1.186)\n",
      "b2806 [\"23S rRNA (cytidine(2498)-2'-O)-methyltransferase (EC 2.1.1.186)\"]\n",
      "\n",
      "SSO SSO:000013644 FIG00018398: hypothetical regulator\n",
      "b2807 ['FIG00018398: hypothetical regulator']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03566 LysR family transcriptional regulator, glycine cleavage system transcriptional activator\n",
      "SSO SSO:000003272 Glycine cleavage system transcriptional activator GcvA\n",
      "b2808 ['Glycine cleavage system transcriptional activator GcvA']\n",
      "\n",
      "SSO SSO:000027209 Uncharacterized lipoprotein YgdI\n",
      "b2809 ['Uncharacterized lipoprotein YgdI']\n",
      "\n",
      "GO GO:0006520 cellular amino acid metabolic process\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K01766 cysteine sulfinate desulfinase [EC:4.4.1.-]\n",
      "b2810 ['Cysteine desulfurase (EC 2.8.1.7) => sulfur transfer pathway protein CsdA']\n",
      "\n",
      "KO KO:K02426 cysteine desulfuration protein SufE\n",
      "b2811 ['Sulfur acceptor protein => sulfur transfer pathway protein CsdE']\n",
      "\n",
      "KO KO:K22132 tRNA threonylcarbamoyladenosine dehydratase\n",
      "b2812 ['HesA/MoeB/ThiF family protein => sulfur transfer pathway protein CsdL']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K08304 membrane-bound lytic murein transglycosylase A [EC:4.2.2.-]\n",
      "SSO SSO:000019475 Membrane-bound lytic murein transglycosylase A\n",
      "b2813 ['Membrane-bound lytic murein transglycosylase A']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01448 N-acetylmuramoyl-L-alanine amidase [EC:3.5.1.28]\n",
      "SSO SSO:000005115 N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)\n",
      "b2817 ['N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K14682 amino-acid N-acetyltransferase [EC:2.3.1.1]\n",
      "SSO SSO:000005110 N-acetylglutamate synthase (EC 2.3.1.1)\n",
      "b2818 ['N-acetylglutamate synthase (EC 2.3.1.1)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03581 exodeoxyribonuclease V alpha subunit [EC:3.1.11.5]\n",
      "SSO SSO:000002620 Exodeoxyribonuclease V alpha chain (EC 3.1.11.5)\n",
      "b2819 ['Exodeoxyribonuclease V alpha chain (EC 3.1.11.5)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03582 exodeoxyribonuclease V beta subunit [EC:3.1.11.5]\n",
      "SSO SSO:000002621 Exodeoxyribonuclease V beta chain (EC 3.1.11.5)\n",
      "b2820 ['Exodeoxyribonuclease V beta chain (EC 3.1.11.5)']\n",
      "\n",
      "KO KO:K01407 protease III [EC:3.4.24.55]\n",
      "SSO SSO:000006412 Protease III precursor (EC 3.4.24.55)\n",
      "SSO SSO:000038698 Protease III (EC 3.4.24.55)\n",
      "b2821 ['Protease III precursor (EC 3.4.24.55)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03583 exodeoxyribonuclease V gamma subunit [EC:3.1.11.5]\n",
      "SSO SSO:000002622 Exodeoxyribonuclease V gamma chain (EC 3.1.11.5)\n",
      "b2822 ['Exodeoxyribonuclease V gamma chain (EC 3.1.11.5)']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K02681 prepilin peptidase dependent protein C\n",
      "b2823 ['FIG004136: Prepilin peptidase dependent protein C precursor']\n",
      "\n",
      "SSO SSO:000028316 Uncharacterized protein YgdB\n",
      "b2824 ['Uncharacterized protein YgdB']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K02680 prepilin peptidase dependent protein B\n",
      "b2825 ['FIG004819: Prepilin peptidase dependent protein B precursor']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K02679 prepilin peptidase dependent protein A\n",
      "b2826 ['Prepilin peptidase dependent protein A precursor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00560 thymidylate synthase [EC:2.1.1.45]\n",
      "SSO SSO:000008068 Thymidylate synthase (EC 2.1.1.45)\n",
      "b2827 ['Thymidylate synthase (EC 2.1.1.45)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K13292 phosphatidylglycerol---prolipoprotein diacylglyceryl transferase [EC:2.5.1.145]\n",
      "SSO SSO:000006370 Prolipoprotein diacylglyceryl transferase (EC 2.4.99.-)\n",
      "b2828 ['Prolipoprotein diacylglyceryl transferase']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K08484 phosphotransferase system, enzyme I, PtsP [EC:2.7.3.9]\n",
      "SSO SSO:000013654 FIG001592: Phosphocarrier protein kinase/phosphorylase, nitrogen regulation associated\n",
      "b2829 ['FIG001592: Phosphocarrier protein kinase/phosphorylase, nitrogen regulation associated']\n",
      "\n",
      "KO KO:K08311 putative (di)nucleoside polyphosphate hydrolase [EC:3.6.1.-]\n",
      "SSO SSO:000000810 Adenosine (5')-pentaphospho-(5'')-adenosine pyrophosphohydrolase (EC 3.6.1.-)\n",
      "b2830 [\"Adenosine (5')-pentaphospho-(5'')-adenosine pyrophosphohydrolase\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03573 DNA mismatch repair protein MutH\n",
      "SSO SSO:000002074 DNA mismatch repair endonuclease MutH\n",
      "b2831 ['DNA mismatch repair endonuclease MutH']\n",
      "\n",
      "SSO SSO:000026481 UPF0053 inner membrane protein YgdQ\n",
      "b2832 ['UPF0053 inner membrane protein YgdQ']\n",
      "\n",
      "SSO SSO:000027210 Uncharacterized lipoprotein YgdR\n",
      "b2833 ['Uncharacterized lipoprotein YgdR']\n",
      "\n",
      "SSO SSO:000025482 Tas protein, an NADP(H)-dependent aldo-keto reductase\n",
      "b2834 ['Tas protein, an NADP(H)-dependent aldo-keto reductase']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K08227 MFS transporter, LPLT family, lysophospholipid transporter\n",
      "SSO SSO:000019152 Lysophospholipid transporter LplT\n",
      "b2835 ['Lysophospholipid transporter LplT']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05939 acyl-[acyl-carrier-protein]-phospholipid O-acyltransferase / long-chain-fatty-acid--[acyl-carrier-protein] ligase [EC:2.3.1.40 6.2.1.20]\n",
      "b2836 ['2-acylglycerophosphoethanolamine acyltransferase (EC 2.3.1.40)', 'Acyl-[acyl-carrier-protein] synthetase (EC 6.2.1.20)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "SSO SSO:000003010 Galactose operon repressor, GalR-LacI family of transcriptional regulators\n",
      "b2837 ['Galactose operon repressor, GalR-LacI family of transcriptional regulators']\n",
      "\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K01586 diaminopimelate decarboxylase [EC:4.1.1.20]\n",
      "SSO SSO:000002280 Diaminopimelate decarboxylase (EC 4.1.1.20)\n",
      "b2838 ['Diaminopimelate decarboxylase (EC 4.1.1.20)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000008191 Transcriptional activator protein LysR\n",
      "b2839 ['Transcriptional activator protein LysR']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K01779 aspartate racemase [EC:5.1.1.13]\n",
      "SSO SSO:000028317 Uncharacterized protein YgeA of aspartate/glutamate/hydantoin racemase family\n",
      "b2840 ['Uncharacterized protein YgeA of aspartate/glutamate/hydantoin racemase family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015524 obsolete L-arabinose/beta-D-thiogalactopyranoside:hydrogen antiporter activity\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02100 MFS transporter, SP family, arabinose:H+ symporter\n",
      "KO KO:K08137 MFS transporter, SP family, galactose:H+ symporter\n",
      "SSO SSO:000030971 low-affinity L-arabinose transport system proton symport protein\n",
      "b2841 ['low-affinity L-arabinose transport system proton symport protein']\n",
      "\n",
      "KO KO:K00065 2-dehydro-3-deoxy-D-gluconate 5-dehydrogenase [EC:1.1.1.127]\n",
      "b2842 ['2-dehydro-3-deoxy-D-gluconate 5-dehydrogenase (EC 1.1.1.127)', '2-deoxy-D-gluconate 3-dehydrogenase (EC 1.1.1.125)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01815 4-deoxy-L-threo-5-hexosulose-uronate ketol-isomerase [EC:5.3.1.17]\n",
      "SSO SSO:000000359 4-deoxy-L-threo-5-hexosulose-uronate ketol-isomerase (EC 5.3.1.17)\n",
      "b2843 ['4-deoxy-L-threo-5-hexosulose-uronate ketol-isomerase (EC 5.3.1.17)']\n",
      "\n",
      "KO KO:K00626 acetyl-CoA C-acetyltransferase [EC:2.3.1.9]\n",
      "SSO SSO:000000702 Acetyl-CoA acetyltransferase (EC 2.3.1.9)\n",
      "b2844 ['Acetyl-CoA acetyltransferase (EC 2.3.1.9)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018444 Inner membrane transport protein YqeG\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b2845 ['Inner membrane transport protein YqeG']\n",
      "\n",
      "SSO SSO:000028509 Uncharacterized protein YqeH\n",
      "b2846 ['Uncharacterized protein YqeH']\n",
      "\n",
      "SSO SSO:000028510 Uncharacterized protein YqeI\n",
      "b2847 ['Uncharacterized protein YqeI']\n",
      "\n",
      "SSO SSO:000029161 YqeJ protein\n",
      "b2848 ['YqeJ protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4683 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028511 Uncharacterized protein YqeK\n",
      "b2849 ['Uncharacterized protein YqeK']\n",
      "\n",
      "SSO SSO:000028318 Uncharacterized protein YgeF\n",
      "b2850 ['Uncharacterized protein YgeF']\n",
      "\n",
      "b2851 ['Uncharacterized protein YgeG']\n",
      "\n",
      "KO KO:K22486 transcriptional regulator HilA, main transcriptional regulator of SPI1\n",
      "b2852 ['Uncharacterized protein YgeH']\n",
      "\n",
      "SSO SSO:000028319 Uncharacterized protein YgeI\n",
      "b2853 ['Uncharacterized protein YgeI']\n",
      "\n",
      "SSO SSO:000003771 IncI1 plasmid conjugative transfer putative membrane protein PilT\n",
      "b2854 ['IncI1 plasmid conjugative transfer putative membrane protein PilT']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2856 ['hypothetical protein']\n",
      "\n",
      "b2858 ['Putative uncharacterized protein YgeN']\n",
      "\n",
      "SSO SSO:000005625 Oxygen-regulated invasion protein OrgA\n",
      "b2859 ['Oxygen-regulated invasion protein OrgA']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000026095 Transposase InsD for insertion element IS2\n",
      "b2860 ['Transposase InsD for insertion element IS2']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026094 Transposase InsC for insertion element IS2\n",
      "b2861 ['Transposase InsC for insertion element IS2']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2863 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K06194 lipoprotein NlpD\n",
      "KO KO:K12943 lipoprotein YgeR\n",
      "SSO SSO:000027211 Uncharacterized lipoprotein YgeR\n",
      "b2865 ['Uncharacterized lipoprotein YgeR']\n",
      "\n",
      "KO KO:K00087 xanthine dehydrogenase molybdenum-binding subunit [EC:1.17.1.4]\n",
      "SSO SSO:000008959 Xanthine dehydrogenase, molybdenum binding subunit (EC 1.17.1.4)\n",
      "b2866 ['Xanthine dehydrogenase, molybdenum binding subunit (EC 1.17.1.4)']\n",
      "\n",
      "KO KO:K13479 xanthine dehydrogenase FAD-binding subunit [EC:1.17.1.4]\n",
      "SSO SSO:000008958 Xanthine dehydrogenase, FAD binding subunit (EC 1.17.1.4)\n",
      "b2867 ['Xanthine dehydrogenase, FAD binding subunit (EC 1.17.1.4)']\n",
      "\n",
      "KO KO:K13480 xanthine dehydrogenase iron-sulfur-binding subunit\n",
      "SSO SSO:000008957 Xanthine dehydrogenase iron-sulfur subunit (EC 1.17.1.4)\n",
      "b2868 ['Xanthine dehydrogenase iron-sulfur subunit (EC 1.17.1.4)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000008803 Uncharacterized sigma-54-dependent transcriptional regulator YgeV\n",
      "b2869 ['Uncharacterized sigma-54-dependent transcriptional regulator YgeV']\n",
      "\n",
      "SSO SSO:000001102 Aspartate/ornithine carbamoyltransferase family protein protein YgeW\n",
      "b2870 ['Aspartate/ornithine carbamoyltransferase family protein protein YgeW']\n",
      "\n",
      "KO KO:K01751 diaminopropionate ammonia-lyase [EC:4.3.1.15]\n",
      "SSO SSO:000002283 Diaminopropionate ammonia-lyase (EC 4.3.1.15)\n",
      "b2871 ['Diaminopropionate ammonia-lyase (EC 4.3.1.15)']\n",
      "\n",
      "SSO SSO:000006638 Putative deacetylase YgeY\n",
      "SSO SSO:000023495 Putative peptidase\n",
      "b2872 ['Putative deacetylase YgeY']\n",
      "\n",
      "KO KO:K01464 dihydropyrimidinase [EC:3.5.2.2]\n",
      "SSO SSO:000012706 D-phenylhydantoinase\n",
      "b2873 ['D-phenylhydantoinase']\n",
      "\n",
      "KO KO:K00926 carbamate kinase [EC:2.7.2.2]\n",
      "SSO SSO:000011922 Carbamate kinase-like protein YqeA\n",
      "b2874 ['Carbamate kinase-like protein YqeA']\n",
      "\n",
      "KO KO:K07402 xanthine dehydrogenase accessory factor\n",
      "b2875 ['Xanthine and CO dehydrogenases maturation factor, XdhC/CoxF family', 'Selenium-dependent molybdenum hydroxylase system protein YqeB']\n",
      "\n",
      "SSO SSO:000000658 Accessory protein YqeC in selenium-dependent molybdenum hydroxylase maturation\n",
      "b2876 ['Accessory protein YqeC in selenium-dependent molybdenum hydroxylase maturation']\n",
      "\n",
      "KO KO:K07141 molybdenum cofactor cytidylyltransferase [EC:2.7.7.76]\n",
      "SSO SSO:000004937 Molybdenum cofactor cytidylyltransferase (EC 2.7.7.76)\n",
      "b2877 ['Molybdenum cofactor cytidylyltransferase (EC 2.7.7.76)']\n",
      "\n",
      "KO KO:K12527 putative selenate reductase [EC:1.97.1.9]\n",
      "SSO SSO:000027020 Uncharacterized [4Fe4S] protein YgfK\n",
      "b2878 ['Uncharacterized [4Fe4S] protein YgfK']\n",
      "\n",
      "GO GO:0007049 cell cycle\n",
      "SSO SSO:000038785 Putative aminohydrolase SsnA\n",
      "b2879 ['Putative aminohydrolase SsnA']\n",
      "\n",
      "KO KO:K12529 putative selenate reductase FAD-binding subunit\n",
      "SSO SSO:000008766 Uncharacterized protein YgfM\n",
      "b2880 ['Uncharacterized protein YgfM']\n",
      "\n",
      "KO KO:K12528 putative selenate reductase molybdopterin-binding subunit\n",
      "SSO SSO:000006185 Possible hypoxanthine oxidase XdhD (EC 1.-.-.-)\n",
      "b2881 ['Possible hypoxanthine oxidase XdhD (EC 1.-.-.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K16346 xanthine permease XanQ\n",
      "SSO SSO:000008960 Xanthine permease\n",
      "b2882 ['Xanthine permease', 'Xanthine:proton symporter XanQ']\n",
      "\n",
      "KO KO:K01487 guanine deaminase [EC:3.5.4.3]\n",
      "SSO SSO:000003324 Guanine deaminase (EC 3.5.4.3)\n",
      "b2883 ['Guanine deaminase (EC 3.5.4.3)']\n",
      "\n",
      "KO KO:K06901 putative MFS transporter, AGZA family, xanthine/uracil permease\n",
      "SSO SSO:000017489 Guanine/hypoxanthine permease GhxQ\n",
      "b4464 ['Guanine/hypoxanthine permease GhxQ']\n",
      "\n",
      "KO KO:K00196 anaerobic carbon-monoxide dehydrogenase iron sulfur subunit\n",
      "SSO SSO:000023280 Putative electron transport protein ygfS\n",
      "b2886 ['Putative electron transport protein YgfS']\n",
      "\n",
      "KO KO:K00266 glutamate synthase (NADPH) small chain [EC:1.4.1.13]\n",
      "SSO SSO:000027021 Uncharacterized [4Fe4S] protein YgfT\n",
      "b2887 ['Uncharacterized [4Fe4S] protein YgfT']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03458 nucleobase:cation symporter-2, NCS2 family\n",
      "SSO SSO:000041621 Uric acid:proton symporter UacT\n",
      "b2888 ['Uric acid:proton symporter UacT']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4684 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01823 isopentenyl-diphosphate Delta-isomerase [EC:5.3.3.2]\n",
      "SSO SSO:000004065 Isopentenyl-diphosphate Delta-isomerase (EC 5.3.3.2)\n",
      "b2889 ['Isopentenyl-diphosphate Delta-isomerase (EC 5.3.3.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K04567 lysyl-tRNA synthetase, class II [EC:6.1.1.6]\n",
      "SSO SSO:000004610 Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)\n",
      "b2890 ['Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02836 peptide chain release factor 2\n",
      "b2891 ['Peptide chain release factor 2', 'programmed frameshift-containing']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K07462 single-stranded-DNA-specific exonuclease [EC:3.1.-.-]\n",
      "SSO SSO:000007570 Single-stranded-DNA-specific exonuclease RecJ (EC 3.1.-.-)\n",
      "b2892 ['Single-stranded-DNA-specific exonuclease RecJ']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03981 thiol:disulfide interchange protein DsbC [EC:5.3.4.1]\n",
      "SSO SSO:000008034 Thiol:disulfide interchange protein DsbC\n",
      "b2893 ['Thiol:disulfide interchange protein DsbC']\n",
      "\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K04763 integrase/recombinase XerD\n",
      "SSO SSO:000007586 Site-specific tyrosine recombinase\n",
      "SSO SSO:000007588 Site-specific tyrosine recombinase XerD\n",
      "b2894 ['Site-specific tyrosine recombinase XerD']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K03840 flavodoxin II\n",
      "SSO SSO:000002851 Flavodoxin 2\n",
      "b2895 ['Flavodoxin 2']\n",
      "\n",
      "KO KO:K19168 toxin CptA\n",
      "b2896 ['Putative inner membrane protein']\n",
      "\n",
      "KO KO:K09159 antitoxin CptB\n",
      "SSO SSO:000007778 Succinate dehydrogenase flavin-adding protein, antitoxin of CptAB toxin-antitoxin\n",
      "b2897 ['Succinate dehydrogenase flavin-adding protein, antitoxin of CptAB toxin-antitoxin']\n",
      "\n",
      "KO KO:K06980 tRNA-modifying protein YgfZ\n",
      "SSO SSO:000042093 tRNA-modifying protein YgfZ\n",
      "b2898 ['tRNA-modifying protein YgfZ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K11068 hemolysin III\n",
      "SSO SSO:000016295 FIG01964566: Predicted membrane protein, hemolysin III homolog\n",
      "b2899 ['FIG01964566: Predicted membrane protein, hemolysin III homolog']\n",
      "\n",
      "KO KO:K09900 uncharacterized protein\n",
      "SSO SSO:000006928 RNA-binding domain protein\n",
      "b2900 ['RNA-binding domain protein']\n",
      "\n",
      "EC EC:3.2.1.105 3-alpha-(S)-strictosidine beta-glucosidase.\n",
      "EC EC:3.2.1.108 Lactase.\n",
      "EC EC:3.2.1.117 Amygdalin beta-glucosidase.\n",
      "EC EC:3.2.1.118 Prunasin beta-glucosidase.\n",
      "EC EC:3.2.1.119 Vicianin beta-glucosidase.\n",
      "EC EC:3.2.1.125 Raucaffricine beta-glucosidase.\n",
      "EC EC:3.2.1.147 Thioglucosidase.\n",
      "EC EC:3.2.1.149 Beta-primeverosidase.\n",
      "EC EC:3.2.1.161 Beta-apiosyl-beta-glucosidase.\n",
      "EC EC:3.2.1.175 Beta-D-glucopyranosyl abscisate beta-glucosidase.\n",
      "EC EC:3.2.1.182 4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl glucosidebeta-D-glucosidase.\n",
      "EC EC:3.2.1.21 Beta-glucosidase.\n",
      "EC EC:3.2.1.23 Beta-galactosidase.\n",
      "EC EC:3.2.1.25 Beta-mannosidase.\n",
      "EC EC:3.2.1.31 Beta-glucuronidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.38 Beta-D-fucosidase.\n",
      "EC EC:3.2.1.62 Glycosylceramidase.\n",
      "EC EC:3.2.1.74 Glucan 1,4-beta-glucosidase.\n",
      "EC EC:3.2.1.85 6-phospho-beta-galactosidase.\n",
      "EC EC:3.2.1.86 6-phospho-beta-glucosidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01223 6-phospho-beta-glucosidase [EC:3.2.1.86]\n",
      "SSO SSO:000000451 6-phospho-beta-glucosidase (EC 3.2.1.86)\n",
      "b2901 ['6-phospho-beta-glucosidase (EC 3.2.1.86)']\n",
      "\n",
      "KO KO:K00059 3-oxoacyl-[acyl-carrier protein] reductase [EC:1.1.1.100]\n",
      "SSO SSO:000005624 Oxidoreductase, short-chain dehydrogenase/reductase family (EC 1.1.1.-)\n",
      "b2902 ['Oxidoreductase, short-chain dehydrogenase/reductase family']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00281 glycine dehydrogenase [EC:1.4.4.2]\n",
      "SSO SSO:000003274 Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)\n",
      "b2903 ['Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K02437 glycine cleavage system H protein\n",
      "SSO SSO:000003268 Glycine cleavage system H protein\n",
      "b2904 ['Glycine cleavage system H protein']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00605 aminomethyltransferase [EC:2.1.2.10]\n",
      "SSO SSO:000000966 Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)\n",
      "b2905 ['Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)']\n",
      "\n",
      "KO KO:K18800 2-polyprenylphenol 6-hydroxylase [EC:1.14.13.240]\n",
      "SSO SSO:000034018 2-polyprenylphenol hydroxylase (EC 1.14.13.240)\n",
      "b2906 ['2-polyprenylphenol hydroxylase']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K03185 2-octaprenyl-6-methoxyphenol hydroxylase [EC:1.14.13.-]\n",
      "SSO SSO:000000216 2-polyprenyl-6-methoxyphenol hydroxylase\n",
      "b2907 ['2-polyprenyl-6-methoxyphenol hydroxylase']\n",
      "\n",
      "KO KO:K01262 Xaa-Pro aminopeptidase [EC:3.4.11.9]\n",
      "SSO SSO:000008954 Xaa-Pro aminopeptidase (EC 3.4.11.9)\n",
      "b2908 ['Xaa-Pro aminopeptidase (EC 3.4.11.9)']\n",
      "\n",
      "KO KO:K09895 uncharacterized protein\n",
      "SSO SSO:000026555 UPF0149 exported protein YgfB\n",
      "b2909 ['UPF0149 exported protein YgfB']\n",
      "\n",
      "KO KO:K09888 cell division protein ZapA\n",
      "SSO SSO:000008993 Z-ring-associated protein ZapA\n",
      "b2910 ['Z-ring-associated protein ZapA']\n",
      "\n",
      "KO KO:K01934 5-formyltetrahydrofolate cyclo-ligase [EC:6.3.3.2]\n",
      "SSO SSO:000000421 5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)\n",
      "b2912 ['5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4665 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006564 L-serine biosynthetic process\n",
      "KO KO:K00058 D-3-phosphoglycerate dehydrogenase / 2-oxoglutarate reductase [EC:1.1.1.95 1.1.1.399]\n",
      "SSO SSO:000001979 D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)\n",
      "b2913 ['D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)']\n",
      "\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01807 ribose 5-phosphate isomerase A [EC:5.3.1.6]\n",
      "SSO SSO:000007110 Ribose 5-phosphate isomerase A (EC 5.3.1.6)\n",
      "b2914 ['Ribose-5-phosphate isomerase A (EC 5.3.1.6)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2915 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05596 LysR family transcriptional regulator, chromosome initiation inhibitor\n",
      "SSO SSO:000025793 Transcriptional regulator ArgP, LysR family\n",
      "b2916 ['Transcriptional regulator ArgP, LysR family']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01847 methylmalonyl-CoA mutase [EC:5.4.99.2]\n",
      "SSO SSO:000004891 Methylmalonyl-CoA mutase (EC 5.4.99.2)\n",
      "b2917 ['Methylmalonyl-CoA mutase (EC 5.4.99.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07588 LAO/AO transport system kinase [EC:2.7.-.-]\n",
      "SSO SSO:000008982 YgfD: protein that forms a complex with the methylmalonyl-CoA mutase in a pathway for conversion of succinyl-CoA to propionyl-CoA\n",
      "b2918 ['YgfD: protein that forms a complex with the methylmalonyl-CoA mutase in a pathway for conversion of succinyl-CoA to propionyl-CoA']\n",
      "\n",
      "KO KO:K11264 methylmalonyl-CoA decarboxylase [EC:4.1.1.-]\n",
      "SSO SSO:000019619 Methylmalonyl-CoA decarboxylase (EC 4.1.1.41)\n",
      "b2919 ['Methylmalonyl-CoA decarboxylase (EC 4.1.1.41)']\n",
      "\n",
      "KO KO:K22214 propionyl-CoA:succinyl-CoA transferase [EC:2.8.3.-]\n",
      "SSO SSO:000022662 Propionyl-CoA:succinyl-CoA transferase\n",
      "b2920 ['Propionyl-CoA:succinyl-CoA transferase']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000019102 LysR family transcriptional regulator YgfI\n",
      "b2921 ['LysR family transcriptional regulator YgfI']\n",
      "\n",
      "KO KO:K09807 uncharacterized protein\n",
      "SSO SSO:000022864 Protein of unknown function DUF541\n",
      "b2922 ['Protein of unknown function DUF541']\n",
      "\n",
      "KO KO:K06895 L-lysine exporter family protein LysE/ArgO\n",
      "SSO SSO:000010793 Arginine exporter protein ArgO\n",
      "b2923 ['Arginine exporter protein ArgO']\n",
      "\n",
      "KO KO:K03442 small conductance mechanosensitive channel\n",
      "SSO SSO:000022787 Protein involved in stability of MscS mechanosensitive channel\n",
      "b2924 ['Protein involved in stability of MscS mechanosensitive channel']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01624 fructose-bisphosphate aldolase, class II [EC:4.1.2.13]\n",
      "SSO SSO:000002935 Fructose-bisphosphate aldolase class II (EC 4.1.2.13)\n",
      "b2925 ['Fructose-bisphosphate aldolase class II (EC 4.1.2.13)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006096 glycolytic process\n",
      "KO KO:K00927 phosphoglycerate kinase [EC:2.7.2.3]\n",
      "SSO SSO:000005970 Phosphoglycerate kinase (EC 2.7.2.3)\n",
      "b2926 ['Phosphoglycerate kinase (EC 2.7.2.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "KO KO:K00134 glyceraldehyde 3-phosphate dehydrogenase [EC:1.2.1.12]\n",
      "KO KO:K03472 D-erythrose 4-phosphate dehydrogenase [EC:1.2.1.72]\n",
      "SSO SSO:000002017 D-erythrose-4-phosphate dehydrogenase (EC 1.2.1.72)\n",
      "b2927 ['D-erythrose-4-phosphate dehydrogenase (EC 1.2.1.72)']\n",
      "\n",
      "KO KO:K02173 putative kinase\n",
      "SSO SSO:000041551 Uncharacterized sugar kinase YggC\n",
      "b2928 ['Uncharacterized sugar kinase YggC']\n",
      "\n",
      "KO KO:K02562 mannitol operon repressor\n",
      "SSO SSO:000035994 Fumarate hydratase FumE (EC 4.2.1.2)\n",
      "b2929 ['Fumarate hydratase FumE (EC 4.2.1.2)']\n",
      "\n",
      "SSO SSO:000002931 Fructose-1,6-bisphosphatase, GlpX type (EC 3.1.3.11)\n",
      "b2930 ['Fructose-1,6-bisphosphatase, GlpX type (EC 3.1.3.11)']\n",
      "\n",
      "KO KO:K19956 L-sorbose 1-phosphate reductase [EC:1.1.1.-]\n",
      "b4465 ['Uncharacterized protein YggP']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02800 PTS system, mannitol-specific IIC component\n",
      "b2933 ['PTS system, mannitol-specific IIC component', 'PTS system, mannitol-specific IIB component (EC 2.7.1.197)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02821 PTS system, ascorbate-specific IIA component [EC:2.7.1.194]\n",
      "b2934 ['PTS system, mannitol-specific cryptic IIA component (EC 2.7.1.197)']\n",
      "\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00615 transketolase [EC:2.2.1.1]\n",
      "SSO SSO:000008287 Transketolase (EC 2.2.1.1)\n",
      "b2935 ['Transketolase (EC 2.2.1.1)']\n",
      "\n",
      "KO KO:K07387 metalloprotease [EC:3.4.24.-]\n",
      "b2936 ['Putative metalloprotease yggG (EC 3.4.24.-)']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "KO KO:K01480 agmatinase [EC:3.5.3.11]\n",
      "SSO SSO:000000843 Agmatinase (EC 3.5.3.11)\n",
      "b2937 ['Agmatinase (EC 3.5.3.11)']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01585 arginine decarboxylase [EC:4.1.1.19]\n",
      "SSO SSO:000001222 Biosynthetic arginine decarboxylase (EC 4.1.1.19)\n",
      "b2938 ['Biosynthetic arginine decarboxylase (EC 4.1.1.19)']\n",
      "\n",
      "SSO SSO:000028514 Uncharacterized protein YqgB\n",
      "b2939 ['Uncharacterized protein YqgB']\n",
      "\n",
      "SSO SSO:000028515 Uncharacterized protein YqgC\n",
      "b2940 ['Uncharacterized protein YqgC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006556 S-adenosylmethionine biosynthetic process\n",
      "KO KO:K00789 S-adenosylmethionine synthetase [EC:2.5.1.6]\n",
      "SSO SSO:000007196 S-adenosylmethionine synthetase (EC 2.5.1.6)\n",
      "b2942 ['S-adenosylmethionine synthetase (EC 2.5.1.6)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08137 MFS transporter, SP family, galactose:H+ symporter\n",
      "SSO SSO:000017046 Galactose-proton symport\n",
      "b2943 ['Galactose-proton symport']\n",
      "\n",
      "KO KO:K02742 SprT protein\n",
      "SSO SSO:000022720 Protein SprT\n",
      "b2944 ['Protein SprT']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K01150 deoxyribonuclease I [EC:3.1.21.1]\n",
      "b2945 ['Endonuclease I precursor (EC 3.1.21.1)', 'Extracellular deoxyribonuclease Dns (EC 3.1.21.-)']\n",
      "\n",
      "KO KO:K09761 16S rRNA (uracil1498-N3)-methyltransferase [EC:2.1.1.193]\n",
      "SSO SSO:000000064 16S rRNA (uracil(1498)-N(3))-methyltransferase (EC 2.1.1.193)\n",
      "b2946 ['16S rRNA (uracil(1498)-N(3))-methyltransferase (EC 2.1.1.193)']\n",
      "\n",
      "GO GO:0006750 glutathione biosynthetic process\n",
      "KO KO:K01920 glutathione synthase [EC:6.3.2.3]\n",
      "SSO SSO:000003227 Glutathione synthetase (EC 6.3.2.3)\n",
      "b2947 ['Glutathione synthetase (EC 6.3.2.3)']\n",
      "\n",
      "KO KO:K07735 putative transcriptional regulator\n",
      "SSO SSO:000008668 UPF0301 protein YqgE\n",
      "b2948 ['UPF0301 protein YqgE']\n",
      "\n",
      "KO KO:K07447 putative holliday junction resolvase [EC:3.1.-.-]\n",
      "SSO SSO:000034541 Putative pre-16S rRNA nuclease YqgF\n",
      "b2949 ['Putative pre-16S rRNA nuclease YqgF']\n",
      "\n",
      "SSO SSO:000008429 Twitching motility protein PilT\n",
      "b2950 ['Twitching motility protein PilT']\n",
      "\n",
      "KO KO:K06997 PLP dependent protein\n",
      "b2951 ['Pyridoxal phosphate-containing protein YggS']\n",
      "\n",
      "KO KO:K02221 YggT family protein\n",
      "SSO SSO:000001462 Cell division integral membrane protein, YggT and half-length relatives\n",
      "b2952 ['Cell division integral membrane protein, YggT and half-length relatives']\n",
      "\n",
      "KO KO:K09131 uncharacterized protein\n",
      "SSO SSO:000026595 UPF0235 protein VC0458\n",
      "b2953 ['UPF0235 protein VC0458']\n",
      "\n",
      "KO KO:K02428 XTP/dITP diphosphohydrolase [EC:3.6.1.66]\n",
      "SSO SSO:000005420 Nucleoside 5-triphosphatase RdgB (dHAPTP, dITP, XTP-specific) (EC 3.6.1.15)\n",
      "b2954 ['Nucleoside 5-triphosphatase RdgB (dHAPTP, dITP, XTP-specific) (EC 3.6.1.66)']\n",
      "\n",
      "KO KO:K02495 oxygen-independent coproporphyrinogen III oxidase [EC:1.3.98.3]\n",
      "SSO SSO:000037447 Oxygen-independent coproporphyrinogen-III oxidase-like protein YggW\n",
      "b2955 ['Oxygen-independent coproporphyrinogen-III oxidase-like protein YggW']\n",
      "\n",
      "SSO SSO:000023153 Putative alpha helix chain\n",
      "b2956 ['Putative alpha helix chain']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01424 L-asparaginase [EC:3.5.1.1]\n",
      "SSO SSO:000004125 L-asparaginase (EC 3.5.1.1)\n",
      "b2957 ['L-asparaginase (EC 3.5.1.1)']\n",
      "\n",
      "SSO SSO:000013737 FIG004016: Uncharacterized protein YggN\n",
      "b2958 ['FIG004016: Uncharacterized protein YggN']\n",
      "\n",
      "KO KO:K09923 uncharacterized protein\n",
      "SSO SSO:000013662 FIG002060: uncharacterized protein YggL\n",
      "b2959 ['FIG002060: uncharacterized protein YggL']\n",
      "\n",
      "KO KO:K03439 tRNA (guanine-N7-)-methyltransferase [EC:2.1.1.33]\n",
      "SSO SSO:000009341 tRNA (guanine(46)-N(7))-methyltransferase (EC 2.1.1.33)\n",
      "b2960 ['tRNA (guanine(46)-N(7))-methyltransferase (EC 2.1.1.33)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03575 A/G-specific adenine glycosylase [EC:3.2.2.31]\n",
      "SSO SSO:000000466 A/G-specific adenine glycosylase (EC 3.2.2.-)\n",
      "b2961 ['A/G-specific adenine glycosylase (EC 3.2.2.-)']\n",
      "\n",
      "b2962 ['FIG001341: Probable Fe(2+)-trafficking protein YggX']\n",
      "\n",
      "EC EC:3.2.1.14 Chitinase.\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08306 membrane-bound lytic murein transglycosylase C [EC:4.2.2.-]\n",
      "SSO SSO:000019477 Membrane-bound lytic murein transglycosylase C (EC 3.2.1.n1)\n",
      "b2963 ['Membrane-bound lytic murein transglycosylase C (EC 3.2.1.n1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03289 MFS transporter, NHS family, nucleoside permease\n",
      "SSO SSO:000020659 Nucleoside permease NupG\n",
      "SSO SSO:000020661 Nucleoside transporter\n",
      "b2964 ['Nucleoside permease NupG']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "SSO SSO:000005516 Ornithine decarboxylase (EC 4.1.1.17)\n",
      "b2965 ['Ornithine decarboxylase (EC 4.1.1.17)']\n",
      "\n",
      "KO KO:K07150 uncharacterized protein\n",
      "SSO SSO:000023363 Putative inner membrane protein YqgA\n",
      "b2966 ['Putative inner membrane protein YqgA']\n",
      "\n",
      "KO KO:K02462 general secretion pathway protein M\n",
      "SSO SSO:000009125 general secretion pathway protein M\n",
      "b2968 ['General secretion pathway protein M']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2969 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2970 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000018014 Hypothetical lipoprotein yghG precursor\n",
      "b2971 ['Hypothetical lipoprotein yghG precursor']\n",
      "\n",
      "KO KO:K02654 leader peptidase (prepilin peptidase) / N-methyltransferase [EC:3.4.23.43 2.1.1.-]\n",
      "b2972 ['Leader peptidase (Prepilin peptidase) (EC 3.4.23.43)', 'N-methyltransferase (EC 2.1.1.-)']\n",
      "\n",
      "KO KO:K10939 accessory colonization factor AcfD\n",
      "SSO SSO:000023389 Putative lipoprotein AcfD-like\n",
      "b4466 ['Putative lipoprotein AcfD-like']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02550 glycolate permease\n",
      "KO KO:K03303 lactate permease\n",
      "SSO SSO:000003298 Glycolate permease\n",
      "b2975 ['Glycolate permease']\n",
      "\n",
      "GO GO:0009436 glyoxylate catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01638 malate synthase [EC:2.3.3.9]\n",
      "SSO SSO:000004673 Malate synthase G (EC 2.3.3.9)\n",
      "b2976 ['Malate synthase G (EC 2.3.3.9)']\n",
      "\n",
      "KO KO:K11477 glc operon protein GlcG\n",
      "SSO SSO:000003586 Hypothetical protein GlcG in glycolate utilization operon\n",
      "b2977 ['Hypothetical protein GlcG in glycolate utilization operon']\n",
      "\n",
      "KO KO:K11473 glycolate oxidase iron-sulfur subunit\n",
      "SSO SSO:000003295 Glycolate dehydrogenase , iron-sulfur subunit GlcF (EC 1.1.99.14)\n",
      "b4467 ['Glycolate dehydrogenase (EC 1.1.99.14), iron-sulfur subunit GlcF']\n",
      "\n",
      "KO KO:K11472 glycolate oxidase FAD binding subunit\n",
      "SSO SSO:000003294 Glycolate dehydrogenase , FAD-binding subunit GlcE (EC 1.1.99.14)\n",
      "b4468 ['Glycolate dehydrogenase (EC 1.1.99.14), FAD-binding subunit GlcE']\n",
      "\n",
      "EC EC:1.1.3.38 Vanillyl-alcohol oxidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00104 glycolate oxidase [EC:1.1.3.15]\n",
      "SSO SSO:000003296 Glycolate dehydrogenase , subunit GlcD (EC 1.1.99.14)\n",
      "b2979 ['Glycolate dehydrogenase (EC 1.1.99.14), subunit GlcD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "KO KO:K11474 GntR family transcriptional regulator, glc operon transcriptional activator\n",
      "SSO SSO:000003299 Glycolate utilization operon transcriptional activator GlcC\n",
      "b2980 ['Glycolate utilization operon transcriptional activator GlcC']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b2981 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b2982 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "SSO SSO:000003883 Inner membrane protein YghQ, probably involved in polysaccharide biosynthesis\n",
      "b2983 ['Inner membrane protein YghQ, probably involved in polysaccharide biosynthesis']\n",
      "\n",
      "SSO SSO:000012434 Conserved protein YghR, with nucleoside triphosphate hydrolase domain\n",
      "SSO SSO:000023019 Putative ATP-binding protein\n",
      "b2984 ['Conserved protein YghR, with nucleoside triphosphate hydrolase domain']\n",
      "\n",
      "SSO SSO:000012399 Conserved ATP-binding protein YghS\n",
      "SSO SSO:000023019 Putative ATP-binding protein\n",
      "b2985 ['Conserved ATP-binding protein YghS']\n",
      "\n",
      "SSO SSO:000012435 Conserved protein YghT, with nucleoside triphosphate hydrolase domain\n",
      "SSO SSO:000023019 Putative ATP-binding protein\n",
      "b2986 ['Conserved protein YghT, with nucleoside triphosphate hydrolase domain']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16322 low-affinity inorganic phosphate transporter\n",
      "SSO SSO:000019057 Low-affinity inorganic phosphate transporter\n",
      "b2987 ['Low-affinity inorganic phosphate transporter']\n",
      "\n",
      "GO GO:0006596 polyamine biosynthetic process\n",
      "KO KO:K01460 glutathionylspermidine amidase/synthetase [EC:3.5.1.78 6.3.1.8]\n",
      "SSO SSO:000043543 Glutathionylspermidine synthase (EC 6.3.1.8) / Glutathionylspermidine amidohydrolase (EC 3.5.1.78)\n",
      "b2988 ['Glutathionylspermidine synthase (EC 6.3.1.8)', 'Glutathionylspermidine amidohydrolase (EC 3.5.1.78)']\n",
      "\n",
      "KO KO:K00799 glutathione S-transferase [EC:2.5.1.18]\n",
      "KO KO:K11209 GSH-dependent disulfide-bond oxidoreductase [EC:1.8.4.-]\n",
      "SSO SSO:000008696 Uncharacterized GST-like protein yghU associated with glutathionylspermidine synthetase/amidase\n",
      "SSO SSO:000023112 Putative S-transferase\n",
      "b2989 ['Uncharacterized GST-like protein yghU associated with glutathionylspermidine synthetase/amidase']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04653 hydrogenase expression/formation protein HypC\n",
      "SSO SSO:000009018 [NiFe] hydrogenase metallocenter assembly protein HybG\n",
      "b2990 ['[NiFe] hydrogenase metallocenter assembly protein HybG']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04651 hydrogenase nickel incorporation protein HypA/HybF\n",
      "SSO SSO:000009024 [NiFe] hydrogenase nickel incorporation protein HybF\n",
      "b2991 ['[NiFe] hydrogenase nickel incorporation protein HybF']\n",
      "\n",
      "SSO SSO:000017945 Hydrogenase-2 operon protein hybE\n",
      "b2992 ['Hydrogenase-2 operon protein HybE']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K03605 hydrogenase maturation protease [EC:3.4.23.-]\n",
      "SSO SSO:000017941 Hydrogenase maturation protease HybD\n",
      "b2993 ['Hydrogenase maturation protease HybD']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06281 hydrogenase large subunit [EC:1.12.99.6]\n",
      "SSO SSO:000037154 Ni/Fe-hydrogenase 2 large subunit HybC (EC 1.12.99.6)\n",
      "b2994 ['Ni/Fe-hydrogenase 2 large subunit HybC (EC 1.12.99.6)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000037153 Ni/Fe-hydrogenase 2 b-type cytochrome subunit HybB\n",
      "b2995 ['Ni/Fe-hydrogenase 2 b-type cytochrome subunit HybB']\n",
      "\n",
      "KO KO:K00124 formate dehydrogenase iron-sulfur subunit\n",
      "SSO SSO:000036339 Hydrogenase-2 operon protein HybA\n",
      "b2996 ['Hydrogenase-2 operon protein HybA']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06282 hydrogenase small subunit [EC:1.12.99.6]\n",
      "SSO SSO:000037155 Ni/Fe-hydrogenase 2 small subunit HybO (EC 1.12.99.6)\n",
      "b2997 ['Ni/Fe-hydrogenase 2 small subunit HybO (EC 1.12.99.6)']\n",
      "\n",
      "SSO SSO:000028321 Uncharacterized protein YghW\n",
      "b2998 ['Uncharacterized protein YghW']\n",
      "\n",
      "KO KO:K01061 carboxymethylenebutenolidase [EC:3.1.1.45]\n",
      "SSO SSO:000013098 Dienelactone hydrolase family protein\n",
      "b4658 ['Dienelactone hydrolase family protein']\n",
      "\n",
      "GO GO:0005244 voltage-gated ion channel activity\n",
      "KO KO:K19265 L-glyceraldehyde 3-phosphate reductase [EC:1.1.1.-]\n",
      "SSO SSO:000018761 L-glyceraldehyde 3-phosphate reductase\n",
      "b3001 ['L-glyceraldehyde 3-phosphate reductase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000026508 UPF0114 protein YqhA\n",
      "b3002 ['UPF0114 protein YqhA']\n",
      "\n",
      "SSO SSO:000027391 Uncharacterized oxidoreductase YghA\n",
      "b3003 ['Uncharacterized oxidoreductase YghA']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K03559 biopolymer transport protein ExbD\n",
      "SSO SSO:000039497 TonB-ExbBD energy transducing system, ExbD subunit\n",
      "b3005 ['TonB-ExbBD energy transducing system, ExbD subunit']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03561 biopolymer transport protein ExbB\n",
      "SSO SSO:000039496 TonB-ExbBD energy transducing system, ExbB subunit\n",
      "b3006 ['TonB-ExbBD energy transducing system, ExbB subunit']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "SSO SSO:000001851 Cystathionine beta-lyase (EC 4.4.1.8)\n",
      "b3008 ['Cystathionine beta-lyase (EC 4.4.1.8)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03975 membrane-associated protein\n",
      "SSO SSO:000013030 DedA family inner membrane protein YghB\n",
      "b3009 ['DedA family inner membrane protein YghB']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000008224 Transcriptional regulator YqhC, positively regulates YqhD and DkgA\n",
      "b3010 ['Transcriptional regulator YqhC, positively regulates YqhD and DkgA']\n",
      "\n",
      "KO KO:K08325 NADP-dependent alcohol dehydrogenase [EC:1.1.-.-]\n",
      "SSO SSO:000005241 NADPH-dependent broad range aldehyde dehydrogenase YqhD\n",
      "b3011 ['NADPH-dependent broad range aldehyde dehydrogenase YqhD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K06221 2,5-diketo-D-gluconate reductase A [EC:1.1.1.346]\n",
      "b3012 ['Methylglyoxal reductase, dihydroxyacetone producing', '2,5-didehydrogluconate reductase (2-dehydro-D-gluconate-forming) (EC 1.1.1.274)']\n",
      "\n",
      "SSO SSO:000028519 Uncharacterized protein YqhG\n",
      "b3013 ['Uncharacterized protein YqhG']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K06078 murein lipoprotein\n",
      "SSO SSO:000027215 Uncharacterized lipoprotein YqhH\n",
      "b3014 ['Uncharacterized lipoprotein YqhH']\n",
      "\n",
      "SSO SSO:000026631 UPF0313 [4Fe-4S] protein YgiQ\n",
      "SSO SSO:000033055 radical SAM superfamily protein\n",
      "b4469 ['UPF0313 [4Fe-4S] protein YgiQ']\n",
      "\n",
      "KO KO:K04753 suppressor of ftsI\n",
      "SSO SSO:000012036 Cell division protein FtsP\n",
      "b3017 ['Cell division protein FtsP']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00655 1-acyl-sn-glycerol-3-phosphate acyltransferase [EC:2.3.1.51]\n",
      "SSO SSO:000009471 1-acyl-sn-glycerol-3-phosphate acyltransferase\n",
      "b3018 ['Acyl-CoA:1-acyl-sn-glycerol-3-phosphate acyltransferase (EC 2.3.1.51)', 'Acyl-ACP:1-acyl-sn-glycerol-3-phosphate acyltransferase (EC 2.3.1.n4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02621 topoisomerase IV subunit A [EC:5.99.1.-]\n",
      "SSO SSO:000012881 DNA topoisomerase IV subunit A\n",
      "b3019 ['DNA topoisomerase IV subunit A (EC 5.99.1.3)']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "KO KO:K15580 oligopeptide transport system substrate-binding protein\n",
      "SSO SSO:000022459 Probable deoxycholate-binding periplasmic protein YgiS\n",
      "b3020 ['Probable deoxycholate-binding periplasmic protein YgiS']\n",
      "\n",
      "KO KO:K13655 HTH-type transcriptional regulator / antitoxin MqsA\n",
      "b3021 ['Antitoxin YgiT']\n",
      "\n",
      "KO KO:K13651 motility quorum-sensing regulator / GCU-specific mRNA interferase toxin\n",
      "b3022 ['Toxin mRNA interferase YgiU']\n",
      "\n",
      "SSO SSO:000025870 Transcriptional regulator YgiV, AraC family\n",
      "b3023 ['Transcriptional regulator YgiV, AraC family']\n",
      "\n",
      "SSO SSO:000022937 Protein ygiW precursor\n",
      "b3024 ['Protein ygiW precursor']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07666 two-component system, OmpR family, response regulator QseB\n",
      "SSO SSO:000008464 Two-component system response regulator QseB\n",
      "b3025 ['Two-component system response regulator QseB']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K07645 two-component system, OmpR family, sensor histidine kinase QseC [EC:2.7.13.3]\n",
      "SSO SSO:000007412 Sensory histidine kinase QseC\n",
      "b3026 ['Sensory histidine kinase QseC']\n",
      "\n",
      "SSO SSO:000018436 Inner membrane protein ygiZ\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b3027 ['Inner membrane protein ygiZ']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03923 modulator of drug activity B\n",
      "SSO SSO:000019989 Modulator of drug activity B\n",
      "b3028 ['Modulator of drug activity B']\n",
      "\n",
      "SSO SSO:000041450 Uncharacterized protein YgiN\n",
      "b3029 ['Uncharacterized protein YgiN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02622 topoisomerase IV subunit B [EC:5.99.1.-]\n",
      "SSO SSO:000012882 DNA topoisomerase IV subunit B\n",
      "b3030 ['DNA topoisomerase IV subunit B (EC 5.99.1.3)']\n",
      "\n",
      "KO KO:K07000 uncharacterized protein\n",
      "SSO SSO:000002561 Esterase YqiA\n",
      "b3031 ['Esterase YqiA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008233 peptidase activity\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K03651 3',5'-cyclic-AMP phosphodiesterase [EC:3.1.4.53]\n",
      "SSO SSO:000000256 3',5'-cyclic-nucleotide phosphodiesterase (EC 3.1.4.17)\n",
      "b3032 [\"3',5'-cyclic-nucleotide phosphodiesterase (EC 3.1.4.17)\"]\n",
      "\n",
      "KO KO:K09920 uncharacterized protein\n",
      "SSO SSO:000008781 Uncharacterized protein YqiB\n",
      "b3033 ['Uncharacterized protein YqiB']\n",
      "\n",
      "EC EC:3.6.1.13 ADP-ribose diphosphatase.\n",
      "KO KO:K01515 ADP-ribose pyrophosphatase [EC:3.6.1.13]\n",
      "SSO SSO:000000533 ADP-ribose pyrophosphatase (EC 3.6.1.13)\n",
      "b3034 ['ADP-ribose pyrophosphatase (EC 3.6.1.13)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K12340 outer membrane protein\n",
      "SSO SSO:000005536 Outer membrane channel TolC (OpmH)\n",
      "b3035 ['Outer membrane channel TolC (OpmH)']\n",
      "\n",
      "SSO SSO:000026681 UPF0441 protein YgiB\n",
      "b3037 ['UPF0441 protein YgiB']\n",
      "\n",
      "SSO SSO:000023138 Putative acid--amine ligase YgiC\n",
      "b3038 ['Putative acid--amine ligase YgiC']\n",
      "\n",
      "KO KO:K15777 4,5-DOPA dioxygenase extradiol [EC:1.13.11.-]\n",
      "SSO SSO:000009656 4,5-DOPA dioxygenase extradiol (EC 1.13.11.29), but not physiological substrate\n",
      "b3039 ['4,5-DOPA dioxygenase extradiol (EC 1.13.11.29), but not physiological substrate']\n",
      "\n",
      "KO KO:K07238 zinc transporter, ZIP family\n",
      "SSO SSO:000029193 Zinc transporter ZupT\n",
      "SSO SSO:000033846 zinc transporter\n",
      "b3040 ['Zinc transporter ZupT']\n",
      "\n",
      "KO KO:K02858 3,4-dihydroxy 2-butanone 4-phosphate synthase [EC:4.1.99.12]\n",
      "SSO SSO:000000263 3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)\n",
      "b3041 ['3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)']\n",
      "\n",
      "KO KO:K09806 uncharacterized protein\n",
      "b3042 ['Uncharacterized protein YqiC']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "SSO SSO:000027083 Uncharacterized fimbrial-like protein YgiL\n",
      "b3043 ['Uncharacterized fimbrial-like protein YgiL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K12518 outer membrane usher protein PapC\n",
      "SSO SSO:000006686 Putative outer membrane usher protein YqiG\n",
      "b3046 ['Putative outer membrane usher protein YqiG']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026094 Transposase InsC for insertion element IS2\n",
      "b3044 ['Transposase InsC for insertion element IS2']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000026095 Transposase InsD for insertion element IS2\n",
      "b3045 ['Transposase InsD for insertion element IS2']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000008711 Uncharacterized fimbrial chaperone YqiH\n",
      "b3047 ['Uncharacterized fimbrial chaperone YqiH']\n",
      "\n",
      "SSO SSO:000008729 Uncharacterized fimbrial-system-associated protein YqiI\n",
      "SSO SSO:000016760 Fimbrial protein\n",
      "b3048 ['Uncharacterized fimbrial-system-associated protein YqiI']\n",
      "\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K22720 surface composition regulator\n",
      "SSO SSO:000003292 Glycogen synthesis protein GlgS\n",
      "b3049 ['Glycogen synthesis protein GlgS']\n",
      "\n",
      "SSO SSO:000018429 Inner membrane protein YqiJ\n",
      "b3050 ['Inner membrane protein YqiJ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07192 flotillin\n",
      "SSO SSO:000018430 Inner membrane protein YqiK\n",
      "b3051 ['Inner membrane protein YqiK']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4664 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4666 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K03272 D-beta-D-heptose 7-phosphate kinase / D-beta-D-heptose 1-phosphate adenosyltransferase [EC:2.7.1.167 2.7.7.70]\n",
      "b3052 ['D-glycero-beta-D-manno-heptose 1-phosphate adenylyltransferase (EC 2.7.7.70)', 'D-glycero-beta-D-manno-heptose-7-phosphate kinase (EC 2.7.1.167)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K00982 [glutamine synthetase] adenylyltransferase / [glutamine synthetase]-adenylyl-L-tyrosine phosphorylase [EC:2.7.7.42 2.7.7.89]\n",
      "b3053 ['Glutamine synthetase adenylyl-L-tyrosine phosphorylase (EC 2.7.7.89)', 'Glutamate-ammonia-ligase adenylyltransferase (EC 2.7.7.42)']\n",
      "\n",
      "KO KO:K18446 triphosphatase [EC:3.6.1.25]\n",
      "SSO SSO:000018449 Inorganic triphosphatase (EC 3.6.1.25)\n",
      "b3054 ['Inorganic triphosphatase (EC 3.6.1.25)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K07184 SH3 domain protein\n",
      "SSO SSO:000001077 Arylsulfatase (EC 3.1.6.1)\n",
      "SSO SSO:000024331 SH3 domain protein\n",
      "b3055 ['Arylsulfatase (EC 3.1.6.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00974 tRNA nucleotidyltransferase (CCA-adding enzyme) [EC:2.7.7.72 3.1.3.- 3.1.4.-]\n",
      "SSO SSO:000001295 CCA tRNA nucleotidyltransferase (EC 2.7.7.72)\n",
      "b3056 ['CCA tRNA nucleotidyltransferase (EC 2.7.7.72)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K06153 undecaprenyl-diphosphatase [EC:3.6.1.27]\n",
      "SSO SSO:000008817 Undecaprenyl-diphosphatase (EC 3.6.1.27)\n",
      "b3057 ['Undecaprenyl-diphosphatase (EC 3.6.1.27)']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K01633 7,8-dihydroneopterin aldolase/epimerase/oxygenase [EC:4.1.2.25 5.1.99.8 1.13.11.81]\n",
      "SSO SSO:000002299 Dihydroneopterin aldolase (EC 4.1.2.25)\n",
      "b3058 ['Dihydroneopterin aldolase (EC 4.1.2.25)']\n",
      "\n",
      "KO KO:K08591 acyl phosphate:glycerol-3-phosphate acyltransferase [EC:2.3.1.275]\n",
      "SSO SSO:000000756 Acyl-ACP:glycerol-3-phosphate O-acyltransferase (EC 2.3.1.n5)\n",
      "b3059 ['Acyl-ACP:glycerol-3-phosphate O-acyltransferase (EC 2.3.1.n5)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K16135 LysR family transcriptional regulator, transcriptional activator for dmlA\n",
      "KO KO:K21742 LysR family transcriptional regulator, transcriptional activator for ttdABT operon\n",
      "SSO SSO:000039534 Transcriptional activator TtdR, LysR family\n",
      "b3060 ['Transcriptional activator TtdR, LysR family']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K03779 L(+)-tartrate dehydratase alpha subunit [EC:4.2.1.32]\n",
      "SSO SSO:000004097 L(+)-tartrate dehydratase alpha subunit (EC 4.2.1.32)\n",
      "b3061 ['L(+)-tartrate dehydratase alpha subunit (EC 4.2.1.32)']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K03780 L(+)-tartrate dehydratase beta subunit [EC:4.2.1.32]\n",
      "SSO SSO:000004098 L(+)-tartrate dehydratase beta subunit (EC 4.2.1.32)\n",
      "b3062 ['L(+)-tartrate dehydratase beta subunit (EC 4.2.1.32)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11106 L-tartrate/succinate antiporter\n",
      "SSO SSO:000018791 L-tartrate/succinate antiporter\n",
      "b3063 ['L-tartrate/succinate antiporter']\n",
      "\n",
      "KO KO:K01409 N6-L-threonylcarbamoyladenine synthase [EC:2.3.1.234]\n",
      "SSO SSO:000005069 N(6)-L-threonylcarbamoyladenine synthase (EC 2.3.1.234)\n",
      "b3064 ['N(6)-L-threonylcarbamoyladenine synthase (EC 2.3.1.234)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02970 small subunit ribosomal protein S21\n",
      "SSO SSO:000007289 SSU ribosomal protein S21p\n",
      "b3065 ['SSU ribosomal protein S21p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02316 DNA primase [EC:2.7.7.101]\n",
      "SSO SSO:000002090 DNA primase (EC 2.7.7.-)\n",
      "SSO SSO:000035413 DNA primase DnaG\n",
      "b3066 ['DNA primase DnaG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03086 RNA polymerase primary sigma factor\n",
      "SSO SSO:000006902 RNA polymerase sigma factor RpoD\n",
      "b3067 ['RNA polymerase sigma factor RpoD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03649 double-stranded uracil-DNA glycosylase [EC:3.2.2.28]\n",
      "SSO SSO:000016933 G/U mismatch-specific uracil DNA glycosylase (EC 3.2.2.28)\n",
      "b3068 ['G/U mismatch-specific uracil DNA glycosylase (EC 3.2.2.28)']\n",
      "\n",
      "KO KO:K07229 ferric-chelate reductase (NADPH) [EC:1.16.1.9]\n",
      "SSO SSO:000009170 iron-chelator utilization protein\n",
      "SSO SSO:000032895 putative siderophore-interacting protein\n",
      "b3070 ['Iron-chelator utilization protein']\n",
      "\n",
      "KO KO:K10947 PadR family transcriptional regulator, regulatory protein PadR\n",
      "SSO SSO:000025875 Transcriptional regulator YqjI, PadR family\n",
      "b3071 ['Transcriptional regulator YqjI, PadR family']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03406 methyl-accepting chemotaxis protein\n",
      "KO KO:K03776 aerotaxis receptor\n",
      "SSO SSO:000010338 Aerotaxis sensor receptor protein\n",
      "b3072 ['Aerotaxis sensor receptor protein']\n",
      "\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "KO KO:K09251 putrescine aminotransferase [EC:2.6.1.82]\n",
      "SSO SSO:000006781 Putrescine aminotransferase (EC 2.6.1.82)\n",
      "b3073 ['Putrescine aminotransferase (EC 2.6.1.82)']\n",
      "\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "SSO SSO:000009382 tRNA-binding protein YgjH\n",
      "b3074 ['tRNA-binding protein YgjH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K12113 LacI family transcriptional regulator, ebg operon repressor\n",
      "SSO SSO:000002604 Evolved beta-D-galactosidase transcriptional repressor\n",
      "SSO SSO:000025979 Transcriptional repressor\n",
      "b3075 ['Evolved beta-D-galactosidase transcriptional repressor']\n",
      "\n",
      "EC EC:3.2.1.146 Beta-galactofuranosidase.\n",
      "EC EC:3.2.1.152 Mannosylglycoprotein endo-beta-mannosidase.\n",
      "EC EC:3.2.1.165 Exo-1,4-beta-D-glucosaminidase.\n",
      "EC EC:3.2.1.23 Beta-galactosidase.\n",
      "EC EC:3.2.1.25 Beta-mannosidase.\n",
      "EC EC:3.2.1.31 Beta-glucuronidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.55 Non-reducing end alpha-L-arabinofuranosidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01190 beta-galactosidase [EC:3.2.1.23]\n",
      "KO KO:K12111 evolved beta-galactosidase subunit alpha [EC:3.2.1.23]\n",
      "SSO SSO:000002605 Evolved beta-D-galactosidase, alpha subunit\n",
      "b3076 ['Evolved beta-D-galactosidase, alpha subunit']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K12112 evolved beta-galactosidase subunit beta\n",
      "SSO SSO:000002606 Evolved beta-D-galactosidase, beta subunit\n",
      "b3077 ['Evolved beta-D-galactosidase, beta subunit']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K20265 glutamate:GABA antiporter\n",
      "SSO SSO:000003878 Inner membrane permease YgjI, clustered with evolved beta-galactosidase\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b3078 ['Inner membrane permease YgjI, clustered with evolved beta-galactosidase']\n",
      "\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "SSO SSO:000028322 Uncharacterized protein YgjJ\n",
      "b3079 ['Uncharacterized protein YgjJ']\n",
      "\n",
      "EC EC:3.2.1.106 Mannosyl-oligosaccharide glucosidase.\n",
      "EC EC:3.2.1.170 Mannosylglycerate hydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.208 Glucosylglycerate hydrolase.\n",
      "EC EC:3.2.1.28 Alpha,alpha-trehalase.\n",
      "EC EC:3.2.1.84 Glucan 1,3-alpha-glucosidase.\n",
      "KO KO:K03931 putative isomerase\n",
      "SSO SSO:000000920 Alpha-glucosidase (EC 3.2.1.20)\n",
      "SSO SSO:000017233 Glucosidase YgjK\n",
      "b3080 ['Glucosidase YgjK']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K00219 2,4-dienoyl-CoA reductase (NADPH2) [EC:1.3.1.34]\n",
      "SSO SSO:000000102 2,4-dienoyl-CoA reductase [NADPH] (EC 1.3.1.34)\n",
      "b3081 ['2,4-dienoyl-CoA reductase [NADPH] (EC 1.3.1.34)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K18831 HTH-type transcriptional regulator / antitoxin HigA\n",
      "SSO SSO:000001018 Antitoxin HigA\n",
      "b3082 ['Antitoxin HigA']\n",
      "\n",
      "KO KO:K19166 mRNA interferase HigB [EC:3.1.-.-]\n",
      "SSO SSO:000030989 mRNA interferase HigB\n",
      "b3083 ['mRNA interferase HigB']\n",
      "\n",
      "KO KO:K00564 16S rRNA (guanine1207-N2)-methyltransferase [EC:2.1.1.172]\n",
      "KO KO:K11391 23S rRNA (guanine1835-N2)-methyltransferase [EC:2.1.1.174]\n",
      "SSO SSO:000000232 23S rRNA (guanine(1835)-N(2))-methyltransferase (EC 2.1.1.174)\n",
      "b3084 ['23S rRNA (guanine(1835)-N(2))-methyltransferase (EC 2.1.1.174)']\n",
      "\n",
      "KO KO:K07043 uncharacterized protein\n",
      "SSO SSO:000039943 UTP pyrophosphatase (EC 3.6.1.9)\n",
      "b3085 ['UTP pyrophosphatase (EC 3.6.1.9)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03748 SanA protein\n",
      "SSO SSO:000024400 SanA protein\n",
      "b3086 ['SanA protein']\n",
      "\n",
      "KO KO:K22230 scyllo-inositol 2-dehydrogenase (NADP+) [EC:1.1.1.-]\n",
      "SSO SSO:000027392 Uncharacterized oxidoreductase YgjR\n",
      "b3087 ['Uncharacterized oxidoreductase YgjR']\n",
      "\n",
      "GO GO:0006950 response to stress\n",
      "KO KO:K05794 tellurite resistance protein TerC\n",
      "SSO SSO:000010405 Alkaline induced inner membrane protein Alx\n",
      "b3088 ['Alkaline induced inner membrane protein Alx']\n",
      "\n",
      "GO GO:0006564 L-serine biosynthetic process\n",
      "GO GO:0009088 threonine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07862 serine/threonine transporter\n",
      "SSO SSO:000024609 Serine/threonine:Na+ symporter SstT\n",
      "b3089 ['Serine/threonine:Na+ symporter SstT']\n",
      "\n",
      "SSO SSO:000018399 Inner membrane protein YgjV\n",
      "b3090 ['Inner membrane protein YgjV']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01685 altronate hydrolase [EC:4.2.1.7]\n",
      "SSO SSO:000000936 Altronate dehydratase (EC 4.2.1.7)\n",
      "SSO SSO:000029347 altronate hydrolase( EC:4.2.1.7 )\n",
      "b3091 ['Altronate dehydratase (EC 4.2.1.7)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01812 glucuronate isomerase [EC:5.3.1.12]\n",
      "SSO SSO:000008883 Uronate isomerase (EC 5.3.1.12)\n",
      "b3092 ['Uronate isomerase (EC 5.3.1.12)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08191 MFS transporter, ACS family, hexuronate transporter\n",
      "SSO SSO:000003427 Hexuronate transporter\n",
      "b3093 ['Hexuronate transporter']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "KO KO:K19775 GntR family transcriptional regulator, hexuronate regulon transcriptional repressor\n",
      "SSO SSO:000003428 Hexuronate utilization operon transcriptional repressor ExuR\n",
      "b3094 ['Hexuronate utilization operon transcriptional repressor ExuR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03975 membrane-associated protein\n",
      "SSO SSO:000013032 DedA family inner membrane protein YqjA\n",
      "b3095 ['DedA family inner membrane protein YqjA']\n",
      "\n",
      "SSO SSO:000028522 Uncharacterized protein YqjB\n",
      "b3096 ['Uncharacterized protein YqjB']\n",
      "\n",
      "SSO SSO:000021441 Periplasmic protein YqjC\n",
      "b3097 ['Periplasmic protein YqjC']\n",
      "\n",
      "KO KO:K05594 ElaB protein\n",
      "SSO SSO:000013339 ElaB protein\n",
      "b3098 ['ElaB protein']\n",
      "\n",
      "SSO SSO:000021510 Phage holin\n",
      "b3099 ['Phage holin']\n",
      "\n",
      "SSO SSO:000018433 Inner membrane protein YqjK\n",
      "b3100 ['Inner membrane protein YqjK']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K15977 putative oxidoreductase\n",
      "SSO SSO:000018432 Inner membrane protein YqjF\n",
      "b3101 ['Inner membrane protein YqjF']\n",
      "\n",
      "KO KO:K07393 glutathionyl-hydroquinone reductase [EC:1.8.5.7]\n",
      "SSO SSO:000003216 Glutathione S-transferase, omega (EC 2.5.1.18)\n",
      "SSO SSO:000023112 Putative S-transferase\n",
      "b3102 ['Glutathione S-transferase, omega (EC 2.5.1.18)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000018400 Inner membrane protein YhaH\n",
      "b3103 ['Inner membrane protein YhaH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000018401 Inner membrane protein YhaI\n",
      "b3104 ['Inner membrane protein YhaI']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000025403 TRANSCRIPTIONAL REGULATOR (LYSR FAMILY)\n",
      "b3105 ['Transcriptional regulator, LysR family']\n",
      "\n",
      "KO KO:K06911 uncharacterized protein\n",
      "b3106 ['Quercetin 2,3-dioxygenase (EC 1.13.11.24) => YhhW']\n",
      "\n",
      "SSO SSO:000028327 Uncharacterized protein YhaL\n",
      "b3107 ['Uncharacterized protein YhaL']\n",
      "\n",
      "SSO SSO:000026686 UPF0597 protein YhaM\n",
      "b4470 ['UPF0597 protein YhaM']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b3110 ['Inner membrane transport protein YhaO']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01752 L-serine dehydratase [EC:4.3.1.17]\n",
      "b4471 ['L-serine dehydratase, beta subunit (EC 4.3.1.17)', 'L-serine dehydratase, alpha subunit (EC 4.3.1.17)']\n",
      "\n",
      "KO KO:K09022 2-iminobutanoate/2-iminopropanoate deaminase [EC:3.5.99.10]\n",
      "SSO SSO:000033990 2-iminobutanoate/2-iminopropanoate deaminase RidA/TdcF (EC 3.5.99.10)\n",
      "b3113 ['2-iminobutanoate/2-iminopropanoate deaminase RidA/TdcF (EC 3.5.99.10)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00656 formate C-acetyltransferase [EC:2.3.1.54]\n",
      "b3114 ['2-ketobutyrate formate-lyase (EC 2.3.1.-)', 'Pyruvate formate-lyase (EC 2.3.1.54)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00925 acetate kinase [EC:2.7.2.1]\n",
      "KO KO:K00932 propionate kinase [EC:2.7.2.15]\n",
      "b3115 ['Acetate kinase (EC 2.7.2.1)', 'Propionate kinase (EC 2.7.2.15)']\n",
      "\n",
      "GO GO:0009088 threonine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03838 threonine transporter\n",
      "SSO SSO:000004183 L-threonine transporter, anaerobically inducible\n",
      "b3116 ['L-threonine transporter, anaerobically inducible']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01754 threonine dehydratase [EC:4.3.1.19]\n",
      "b3117 ['Threonine dehydratase, catabolic (EC 4.3.1.19)', 'L-serine dehydratase, (PLP)-dependent (EC 4.3.1.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07592 LysR family transcriptional regulator, tdc operon transcriptional activator\n",
      "SSO SSO:000008056 Threonine catabolic operon transcriptional activator TdcA\n",
      "b3118 ['Threonine catabolic operon transcriptional activator TdcA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07591 threonine dehydratase operon activator protein\n",
      "SSO SSO:000008057 Threonine catabolic operon transcriptional activator TdcR\n",
      "b3119 ['Threonine catabolic operon transcriptional activator TdcR']\n",
      "\n",
      "SSO SSO:000028324 Uncharacterized protein YhaB\n",
      "b3120 ['Uncharacterized protein YhaB']\n",
      "\n",
      "SSO SSO:000028325 Uncharacterized protein YhaC\n",
      "b3121 ['Uncharacterized protein YhaC']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00865 glycerate 2-kinase [EC:2.7.1.165]\n",
      "SSO SSO:000003241 Glycerate-2-kinase (EC 2.7.1.165)\n",
      "b3124 ['Glycerate-2-kinase (EC 2.7.1.165)']\n",
      "\n",
      "GO GO:0009441 glycolate metabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00020 3-hydroxyisobutyrate dehydrogenase [EC:1.1.1.31]\n",
      "KO KO:K00042 2-hydroxy-3-oxopropionate reductase [EC:1.1.1.60]\n",
      "SSO SSO:000000159 2-hydroxy-3-oxopropionate reductase (EC 1.1.1.60)\n",
      "b3125 ['2-hydroxy-3-oxopropionate reductase (EC 1.1.1.60)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01630 2-dehydro-3-deoxyglucarate aldolase [EC:4.1.2.20]\n",
      "KO KO:K02510 4-hydroxy-2-oxoheptanedioate aldolase [EC:4.1.2.52]\n",
      "SSO SSO:000000146 2-dehydro-3-deoxyglucarate aldolase (EC 4.1.2.20)\n",
      "b3126 ['2-dehydro-3-deoxyglucarate aldolase (EC 4.1.2.20)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03535 MFS transporter, ACS family, glucarate transporter\n",
      "KO KO:K12299 MFS transporter, ACS family, probable galactarate transporter\n",
      "b3127 ['D-galactarate transporter', 'D-glucarate transporter']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01708 galactarate dehydratase [EC:4.2.1.42]\n",
      "SSO SSO:000002019 D-galactarate dehydratase (EC 4.2.1.42)\n",
      "b3128 ['D-galactarate dehydratase (EC 4.2.1.42)']\n",
      "\n",
      "KO KO:K19156 antitoxin PrlF\n",
      "SSO SSO:000010729 Antitoxin PrlF\n",
      "b3129 ['Antitoxin PrlF']\n",
      "\n",
      "KO KO:K19155 toxin YhaV [EC:3.1.-.-]\n",
      "SSO SSO:000025725 Toxin YhaV\n",
      "b3130 ['Toxin YhaV']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K02081 DeoR family transcriptional regulator, aga operon transcriptional repressor\n",
      "SSO SSO:000008274 Transcriptional repressor of aga operon\n",
      "b3131 ['Transcriptional repressor of aga operon']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K16371 D-tagatose-1,6-bisphosphate aldolase subunit GatZ/KbaZ\n",
      "SSO SSO:000002047 D-tagatose-1,6-bisphosphate aldolase subunit KbaZ\n",
      "b3132 ['D-tagatose-1,6-bisphosphate aldolase subunit KbaZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K02745 PTS system, N-acetylgalactosamine-specific IIB component [EC:2.7.1.-]\n",
      "SSO SSO:000005670 PTS system, N-acetylgalactosamine-specific IIB component (EC 2.7.1.69)\n",
      "b3133 ['PTS system, N-acetylgalactosamine-specific IIB component']\n",
      "\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b3134 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b3135 ['hypothetical protein']\n",
      "\n",
      "KO KO:K02082 tagatose-6-phosphate ketose/aldose isomerase [EC:5.-.-.-]\n",
      "SSO SSO:000003008 Galactosamine-6-phosphate isomerase AgaS\n",
      "b3136 ['Galactosamine-6-phosphate isomerase AgaS']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01624 fructose-bisphosphate aldolase, class II [EC:4.1.2.13]\n",
      "KO KO:K08302 tagatose 1,6-diphosphate aldolase GatY/KbaY [EC:4.1.2.40]\n",
      "SSO SSO:000002046 D-tagatose-1,6-bisphosphate aldolase subunit KbaY (EC 4.1.2.40)\n",
      "b3137 ['D-tagatose-1,6-bisphosphate aldolase subunit KbaY (EC 4.1.2.40)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K10984 PTS system, galactosamine-specific IIB component [EC:2.7.1.-]\n",
      "SSO SSO:000005689 PTS system, galactosamine-specific IIB component (EC 2.7.1.69)\n",
      "b3138 ['PTS system, galactosamine-specific IIB component']\n",
      "\n",
      "KO KO:K10985 PTS system, galactosamine-specific IIC component\n",
      "SSO SSO:000005690 PTS system, galactosamine-specific IIC component (EC 2.7.1.69)\n",
      "b3139 ['PTS system, galactosamine-specific IIC component']\n",
      "\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K10986 PTS system, galactosamine-specific IID component\n",
      "SSO SSO:000005691 PTS system, galactosamine-specific IID component (EC 2.7.1.69)\n",
      "b3140 ['PTS system, galactosamine-specific IID component']\n",
      "\n",
      "KO KO:K02080 galactosamine-6-phosphate isomerase [EC:5.3.1.-]\n",
      "KO KO:K02564 glucosamine-6-phosphate deaminase [EC:3.5.99.6]\n",
      "SSO SSO:000003009 Galactosamine-6-phosphate isomerase [deaminating]\n",
      "b3141 ['Galactosamine-6-phosphate isomerase [deaminating]']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "SSO SSO:000008726 Uncharacterized fimbrial-like protein YraH\n",
      "b3142 ['Uncharacterized fimbrial-like protein YraH']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K07346 fimbrial chaperone protein\n",
      "SSO SSO:000006325 Probable fimbrial chaperone YraI\n",
      "b3143 ['Probable fimbrial chaperone YraI']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07347 outer membrane usher protein\n",
      "SSO SSO:000005608 Outer membrane usher protein YraJ\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "b3144 ['Outer membrane usher protein YraJ']\n",
      "\n",
      "GO GO:0009289 pilus\n",
      "SSO SSO:000008727 Uncharacterized fimbrial-like protein YraK\n",
      "b3145 ['Uncharacterized fimbrial-like protein YraK']\n",
      "\n",
      "KO KO:K07056 16S rRNA (cytidine1402-2'-O)-methyltransferase [EC:2.1.1.198]\n",
      "SSO SSO:000000053 16S rRNA (cytidine(1402)-2'-O)-methyltransferase (EC 2.1.1.198)\n",
      "b3146 [\"16S rRNA (cytidine(1402)-2'-O)-methyltransferase (EC 2.1.1.198)\"]\n",
      "\n",
      "KO KO:K07121 uncharacterized protein\n",
      "SSO SSO:000021308 Penicillin-binding protein activator LpoA\n",
      "b3147 ['Penicillin-binding protein activator LpoA']\n",
      "\n",
      "KO KO:K07460 putative endonuclease\n",
      "SSO SSO:000026500 UPF0102 protein YraN\n",
      "b3148 ['UPF0102 protein YraN']\n",
      "\n",
      "KO KO:K03271 D-sedoheptulose 7-phosphate isomerase [EC:5.3.1.28]\n",
      "KO KO:K12961 DnaA initiator-associating protein\n",
      "SSO SSO:000013191 DnaA initiator-associating protein DiaA\n",
      "b3149 ['DnaA initiator-associating protein DiaA']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K04065 hyperosmotically inducible periplasmic protein\n",
      "SSO SSO:000020875 Outer membrane lipoprotein\n",
      "SSO SSO:000028534 Uncharacterized protein YraP\n",
      "b3150 ['Uncharacterized protein YraP']\n",
      "\n",
      "KO KO:K07089 uncharacterized protein\n",
      "SSO SSO:000027321 Uncharacterized membrane protein, YraQ family\n",
      "b3151 ['Uncharacterized membrane protein, YraQ family']\n",
      "\n",
      "KO KO:K17290 oxidoreductase [EC:1.1.1.-]\n",
      "SSO SSO:000023458 Putative nucleoside-diphosphate-sugar epimerase\n",
      "b3152 ['Uncharacterized protein YraR']\n",
      "\n",
      "KO KO:K05520 protease I [EC:3.5.1.124]\n",
      "SSO SSO:000025585 ThiJ/PfpI family protein YhbO\n",
      "b3153 ['ThiJ/PfpI family protein YhbO']\n",
      "\n",
      "KO KO:K09979 uncharacterized protein\n",
      "SSO SSO:000026625 UPF0306 protein YhbP\n",
      "b3154 ['UPF0306 protein YhbP']\n",
      "\n",
      "KO KO:K07461 putative endonuclease\n",
      "SSO SSO:000008664 UPF0213 protein YhbQ\n",
      "b3155 ['UPF0213 protein YhbQ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03824 putative acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000026956 Uncharacterized N-acetyltransferase YhbS\n",
      "b3156 ['Uncharacterized N-acetyltransferase YhbS']\n",
      "\n",
      "SSO SSO:000024314 SCP-2 sterol transfer family protein\n",
      "SSO SSO:000028331 Uncharacterized protein YhbT\n",
      "b3157 ['Uncharacterized protein YhbT']\n",
      "\n",
      "KO KO:K08303 putative protease [EC:3.4.-.-]\n",
      "SSO SSO:000008741 Uncharacterized protease YhbU\n",
      "b3158 ['Uncharacterized protease YhbU']\n",
      "\n",
      "SSO SSO:000008740 Uncharacterized peptidase U32 family member YhbV\n",
      "b3159 ['Uncharacterized peptidase U32 family member YhbV']\n",
      "\n",
      "SSO SSO:000019077 Luciferase-like monooxygenase YhbW\n",
      "b3160 ['Luciferase-like monooxygenase YhbW']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03835 tryptophan-specific transport protein\n",
      "SSO SSO:000026227 Tryptophan-specific transport protein\n",
      "b3161 ['Tryptophan-specific transport protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K05592 ATP-dependent RNA helicase DeaD [EC:3.6.4.13]\n",
      "SSO SSO:000009996 ATP dependent RNA helicase\n",
      "SSO SSO:000035396 DEAD-box ATP-dependent RNA helicase DeaD (= CshA) (EC 3.6.4.13)\n",
      "b3162 ['DEAD-box ATP-dependent RNA helicase DeaD (= CshA) (EC 3.6.4.13)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4685 ['hypothetical protein']\n",
      "\n",
      "KO KO:K05803 lipoprotein NlpI\n",
      "SSO SSO:000019001 Lipoprotein NlpI\n",
      "b3163 ['Lipoprotein NlpI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K00962 polyribonucleotide nucleotidyltransferase [EC:2.7.7.8]\n",
      "SSO SSO:000006161 Polyribonucleotide nucleotidyltransferase (EC 2.7.7.8)\n",
      "b3164 ['Polyribonucleotide nucleotidyltransferase (EC 2.7.7.8)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02956 small subunit ribosomal protein S15\n",
      "SSO SSO:000007262 SSU ribosomal protein S15p (S13e)\n",
      "b3165 ['SSU ribosomal protein S15p (S13e)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03177 tRNA pseudouridine55 synthase [EC:5.4.99.25]\n",
      "SSO SSO:000009366 tRNA pseudouridine(55) synthase (EC 5.4.99.25)\n",
      "b3166 ['tRNA pseudouridine(55) synthase (EC 5.4.99.25)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K02834 ribosome-binding factor A\n",
      "SSO SSO:000007152 Ribosome-binding factor A\n",
      "b3167 ['Ribosome-binding factor A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02519 translation initiation factor IF-2\n",
      "SSO SSO:000008310 Translation initiation factor 2\n",
      "SSO SSO:000033640 translation initiation factor IF-2\n",
      "b3168 ['Translation initiation factor 2']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K02600 N utilization substance protein A\n",
      "SSO SSO:000008177 Transcription termination protein NusA\n",
      "b3169 ['Transcription termination protein NusA']\n",
      "\n",
      "KO KO:K09748 ribosome maturation factor RimP\n",
      "SSO SSO:000001142 Bacterial ribosome SSU maturation protein RimP\n",
      "b3170 ['Bacterial ribosome SSU maturation protein RimP']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K01940 argininosuccinate synthase [EC:6.3.4.5]\n",
      "SSO SSO:000001052 Argininosuccinate synthase (EC 6.3.4.5)\n",
      "b3172 ['Argininosuccinate synthase (EC 6.3.4.5)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K19353 heptose-I-phosphate ethanolaminephosphotransferase [EC:2.7.8.-]\n",
      "SSO SSO:000023518 Putative phosphoethanolamine transferase YhbX\n",
      "b3173 ['Putative phosphoethanolamine transferase YhbX']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03075 preprotein translocase subunit SecG\n",
      "SSO SSO:000038743 Protein translocase membrane subunit SecG\n",
      "b3175 ['Protein translocase membrane subunit SecG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K03431 phosphoglucosamine mutase [EC:5.4.2.10]\n",
      "SSO SSO:000005969 Phosphoglucosamine mutase (EC 5.4.2.10)\n",
      "b3176 ['Phosphoglucosamine mutase (EC 5.4.2.10)']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K00796 dihydropteroate synthase [EC:2.5.1.15]\n",
      "SSO SSO:000002313 Dihydropteroate synthase (EC 2.5.1.15)\n",
      "b3177 ['Dihydropteroate synthase (EC 2.5.1.15)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03798 cell division protease FtsH [EC:3.4.24.-]\n",
      "SSO SSO:000035044 Cell division-associated, ATP-dependent zinc metalloprotease FtsH\n",
      "b3178 ['Cell division-associated, ATP-dependent zinc metalloprotease FtsH']\n",
      "\n",
      "KO KO:K02427 23S rRNA (uridine2552-2'-O)-methyltransferase [EC:2.1.1.166]\n",
      "SSO SSO:000000244 23S rRNA (uridine(2552)-2'-O)-methyltransferase (EC 2.1.1.166)\n",
      "b3179 [\"23S rRNA (uridine(2552)-2'-O)-methyltransferase (EC 2.1.1.166)\"]\n",
      "\n",
      "KO KO:K07574 RNA-binding protein\n",
      "SSO SSO:000023854 RNA-binding protein YhbY\n",
      "b3180 ['RNA-binding protein YhbY']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K03624 transcription elongation factor GreA\n",
      "SSO SSO:000008140 Transcription elongation factor GreA\n",
      "b3181 ['Transcription elongation factor GreA']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042493 response to drug\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K07259 serine-type D-Ala-D-Ala carboxypeptidase/endopeptidase (penicillin-binding protein 4) [EC:3.4.16.4 3.4.21.-]\n",
      "SSO SSO:000001995 D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)\n",
      "b3182 ['D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)']\n",
      "\n",
      "KO KO:K03979 GTPase [EC:3.6.5.-]\n",
      "SSO SSO:000002996 GTP-binding protein Obg\n",
      "b3183 ['GTP-binding protein Obg']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000027122 Uncharacterized inner membrane transporter YhbE\n",
      "b3184 ['Uncharacterized inner membrane transporter YhbE']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02899 large subunit ribosomal protein L27\n",
      "SSO SSO:000004263 LSU ribosomal protein L27p\n",
      "b3185 ['LSU ribosomal protein L27p']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02888 large subunit ribosomal protein L21\n",
      "SSO SSO:000004244 LSU ribosomal protein L21p\n",
      "b3186 ['LSU ribosomal protein L21p']\n",
      "\n",
      "KO KO:K02523 octaprenyl-diphosphate synthase [EC:2.5.1.90]\n",
      "SSO SSO:000005482 Octaprenyl diphosphate synthase (EC 2.5.1.90)\n",
      "b3187 ['Octaprenyl diphosphate synthase (EC 2.5.1.90)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07724 Ner family transcriptional regulator\n",
      "SSO SSO:000025238 Sugar fermentation stimulation protein B\n",
      "b3188 ['Sugar fermentation stimulation protein B']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K00790 UDP-N-acetylglucosamine 1-carboxyvinyltransferase [EC:2.5.1.7]\n",
      "SSO SSO:000008616 UDP-N-acetylglucosamine 1-carboxyvinyltransferase (EC 2.5.1.7)\n",
      "b3189 ['UDP-N-acetylglucosamine 1-carboxyvinyltransferase (EC 2.5.1.7)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000010205 Acid stress protein IbaG\n",
      "b3190 ['Acid stress protein IbaG']\n",
      "\n",
      "KO KO:K07122 phospholipid transport system transporter-binding protein\n",
      "SSO SSO:000005990 Phospholipid ABC transporter-binding protein MlaB\n",
      "b3191 ['Phospholipid ABC transporter-binding protein MlaB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K07323 phospholipid transport system substrate-binding protein\n",
      "SSO SSO:000005988 Phospholipid ABC transporter shuttle protein MlaC\n",
      "b3192 ['Phospholipid ABC transporter shuttle protein MlaC']\n",
      "\n",
      "KO KO:K02067 phospholipid/cholesterol/gamma-HCH transport system substrate-binding protein\n",
      "SSO SSO:000005989 Phospholipid ABC transporter substrate-binding protein MlaD\n",
      "b3193 ['Phospholipid ABC transporter substrate-binding protein MlaD']\n",
      "\n",
      "KO KO:K02066 phospholipid/cholesterol/gamma-HCH transport system permease protein\n",
      "SSO SSO:000005987 Phospholipid ABC transporter permease protein MlaE\n",
      "b3194 ['Phospholipid ABC transporter permease protein MlaE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02065 phospholipid/cholesterol/gamma-HCH transport system ATP-binding protein\n",
      "SSO SSO:000005986 Phospholipid ABC transporter ATP-binding protein MlaF\n",
      "b3195 ['Phospholipid ABC transporter ATP-binding protein MlaF']\n",
      "\n",
      "GO GO:0005432 calcium:sodium antiporter activity\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07301 cation:H+ antiporter\n",
      "SSO SSO:000003886 Inner membrane protein YrbG, predicted calcium/sodium:proton antiporter\n",
      "b3196 ['Inner membrane protein YrbG, predicted calcium/sodium:proton antiporter']\n",
      "\n",
      "KO KO:K06041 arabinose-5-phosphate isomerase [EC:5.3.1.13]\n",
      "SSO SSO:000002013 D-arabinose 5-phosphate isomerase (EC 5.3.1.13)\n",
      "b3197 ['D-arabinose-5-phosphate isomerase (EC 5.3.1.13)']\n",
      "\n",
      "KO KO:K03270 3-deoxy-D-manno-octulosonate 8-phosphate phosphatase (KDO 8-P phosphatase) [EC:3.1.3.45]\n",
      "SSO SSO:000000279 3-deoxy-D-manno-octulosonate 8-phosphate phosphatase (EC 3.1.3.45)\n",
      "b3198 ['3-deoxy-D-manno-octulosonate 8-phosphate phosphatase (EC 3.1.3.45)']\n",
      "\n",
      "KO KO:K11719 lipopolysaccharide export system protein LptC\n",
      "SSO SSO:000004486 Lipopolysaccharide export system protein LptC\n",
      "b3199 ['Lipopolysaccharide export system protein LptC']\n",
      "\n",
      "KO KO:K09774 lipopolysaccharide export system protein LptA\n",
      "SSO SSO:000036654 Lipopolysaccharide export system protein LptA\n",
      "b3200 ['Lipopolysaccharide export system protein LptA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06861 lipopolysaccharide export system ATP-binding protein [EC:3.6.3.-]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000004474 Lipopolysaccharide ABC transporter, ATP-binding protein LptB\n",
      "b3201 ['Lipopolysaccharide ABC transporter, ATP-binding protein LptB']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K03092 RNA polymerase sigma-54 factor\n",
      "SSO SSO:000006914 RNA polymerase sigma-54 factor RpoN\n",
      "b3202 ['RNA polymerase sigma-54 factor RpoN']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K05808 putative sigma-54 modulation protein\n",
      "SSO SSO:000007143 Ribosome hibernation promoting factor Hpf\n",
      "b3203 ['Ribosome hibernation promoting factor Hpf']\n",
      "\n",
      "KO KO:K02806 PTS system, nitrogen regulatory IIA component [EC:2.7.1.-]\n",
      "SSO SSO:000005654 PTS IIA-like nitrogen-regulatory protein PtsN (EC 2.7.1.69 )\n",
      "b3204 ['PTS IIA-like nitrogen-regulatory protein PtsN']\n",
      "\n",
      "KO KO:K06958 RNase adapter protein RapZ\n",
      "SSO SSO:000006939 RNase adapter protein RapZ\n",
      "b3205 ['RNase adapter protein RapZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K08485 phosphocarrier protein NPr\n",
      "SSO SSO:000005952 Phosphocarrier protein, nitrogen regulation associated\n",
      "b3206 ['Phosphocarrier protein, nitrogen regulation associated']\n",
      "\n",
      "SSO SSO:000028535 Uncharacterized protein YrbL\n",
      "b3207 ['Uncharacterized protein YrbL']\n",
      "\n",
      "EC EC:2.4.1.129 Peptidoglycan glycosyltransferase.\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03814 monofunctional glycosyltransferase [EC:2.4.1.129]\n",
      "SSO SSO:000004949 Monofunctional biosynthetic peptidoglycan transglycosylase (EC 2.4.2.-)\n",
      "b3208 ['Monofunctional biosynthetic peptidoglycan transglycosylase']\n",
      "\n",
      "SSO SSO:000013415 Enhancing lycopene biosynthesis protein 2\n",
      "b3209 ['Enhancing lycopene biosynthesis protein 2']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07648 two-component system, OmpR family, aerobic respiration control sensor histidine kinase ArcB [EC:2.7.13.3]\n",
      "SSO SSO:000000841 Aerobic respiration control sensor protein arcB (EC 2.7.3.-)\n",
      "b3210 ['Aerobic respiration control sensor protein ArcB (EC 2.7.13.3)']\n",
      "\n",
      "KO KO:K07139 uncharacterized protein\n",
      "SSO SSO:000041767 [4Fe-4S]-AdoMet protein YhcC\n",
      "b3211 ['[4Fe-4S]-AdoMet protein YhcC']\n",
      "\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K00265 glutamate synthase (NADPH) large chain [EC:1.4.1.13]\n",
      "SSO SSO:000003155 Glutamate synthase [NADPH] large chain (EC 1.4.1.13)\n",
      "b3212 ['Glutamate synthase [NADPH] large chain (EC 1.4.1.13)']\n",
      "\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K00266 glutamate synthase (NADPH) small chain [EC:1.4.1.13]\n",
      "SSO SSO:000003157 Glutamate synthase [NADPH] small chain (EC 1.4.1.13)\n",
      "b3213 ['Glutamate synthase [NADPH] small chain (EC 1.4.1.13)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "SSO SSO:000005354 Nitrogen regulatory protein GltF\n",
      "SSO SSO:000031394 periplasmic protein\n",
      "b3214 ['Nitrogen regulatory protein GltF']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000027081 Uncharacterized fimbrial chaperone YhcA\n",
      "b3215 ['Uncharacterized fimbrial chaperone YhcA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000027381 Uncharacterized outer membrane usher protein YhcD\n",
      "b3216 ['Uncharacterized outer membrane usher protein YhcD']\n",
      "\n",
      "b4569 ['PTS system, mannitol-specific IIABC component (EIIABC-Mtl) (Mannitol- permease IIABC component) (Phosphotransferase enzyme II, ABC component) (EC 2.7.1.69)']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b3218 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000028332 Uncharacterized protein YhcF\n",
      "b3219 ['Uncharacterized protein YhcF']\n",
      "\n",
      "SSO SSO:000023701 Putative uncharacterized protein YhcG\n",
      "b3220 ['Putative uncharacterized protein YhcG']\n",
      "\n",
      "KO KO:K19334 biofilm protein TabA\n",
      "b3221 ['Putative sugar isomerase involved in processing of exogenous sialic acid']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K00885 N-acylmannosamine kinase [EC:2.7.1.60]\n",
      "SSO SSO:000005112 N-acetylmannosamine kinase (EC 2.7.1.60)\n",
      "b3222 ['N-acetylmannosamine kinase (EC 2.7.1.60)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K01788 N-acylglucosamine-6-phosphate 2-epimerase [EC:5.1.3.9]\n",
      "SSO SSO:000005113 N-acetylmannosamine-6-phosphate 2-epimerase (EC 5.1.3.9)\n",
      "b3223 ['N-acetylmannosamine-6-phosphate 2-epimerase (EC 5.1.3.9)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K03290 MFS transporter, SHS family, sialic acid transporter\n",
      "SSO SSO:000007470 Sialic acid transporter (permease) NanT\n",
      "b3224 ['Sialic acid transporter (permease) NanT']\n",
      "\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K01639 N-acetylneuraminate lyase [EC:4.1.3.3]\n",
      "SSO SSO:000005118 N-acetylneuraminate lyase (EC 4.1.3.3)\n",
      "b3225 ['N-acetylneuraminate lyase (EC 4.1.3.3)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "KO KO:K22104 GntR family transcriptional regulator, sialic acid-inducible nan operon repressor\n",
      "SSO SSO:000009416 transcriptional regulator NanR\n",
      "b3226 ['Transcriptional regulator NanR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03326 C4-dicarboxylate transporter, DcuC family\n",
      "SSO SSO:000023232 Putative cryptic C4-dicarboxylate transporter DcuD\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b3227 ['Putative cryptic C4-dicarboxylate transporter DcuD']\n",
      "\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K03600 stringent starvation protein B\n",
      "SSO SSO:000035135 ClpXP protease specificity-enhancing factor SspB\n",
      "b3228 ['ClpXP protease specificity-enhancing factor SspB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K00799 glutathione S-transferase [EC:2.5.1.18]\n",
      "KO KO:K03599 stringent starvation protein A\n",
      "SSO SSO:000025160 Stringent starvation protein A\n",
      "b3229 ['Stringent starvation protein A']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02996 small subunit ribosomal protein S9\n",
      "SSO SSO:000007345 SSU ribosomal protein S9p (S16e)\n",
      "b3230 ['SSU ribosomal protein S9p (S16e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02871 large subunit ribosomal protein L13\n",
      "SSO SSO:000004208 LSU ribosomal protein L13p (L13Ae)\n",
      "b3231 ['LSU ribosomal protein L13p (L13Ae)']\n",
      "\n",
      "KO KO:K06916 cell division protein ZapE\n",
      "SSO SSO:000001478 Cell division protein ZapE\n",
      "b3232 ['Cell division protein ZapE']\n",
      "\n",
      "KO KO:K09908 uncharacterized protein\n",
      "b3233 ['Putative cytochrome d ubiquinol oxidase subunit III (EC 1.10.3.-) (Cytochrome bd-I oxidase subunit III)']\n",
      "\n",
      "KO KO:K04771 serine protease Do [EC:3.4.21.107]\n",
      "KO KO:K04772 serine protease DegQ [EC:3.4.21.-]\n",
      "SSO SSO:000020933 Outer membrane stress sensor protease DegQ, serine protease\n",
      "b3234 ['Outer membrane stress sensor protease DegQ, serine protease']\n",
      "\n",
      "GO GO:0006508 proteolysis\n",
      "GO GO:0008233 peptidase activity\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K04691 serine protease DegS [EC:3.4.21.-]\n",
      "SSO SSO:000020934 Outer membrane stress sensor protease DegS\n",
      "b3235 ['Outer membrane stress sensor protease DegS']\n",
      "\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009436 glyoxylate catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000004670 Malate dehydrogenase (EC 1.1.1.37)\n",
      "b3236 ['Malate dehydrogenase (EC 1.1.1.37)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03402 transcriptional regulator of arginine metabolism\n",
      "SSO SSO:000010795 Arginine pathway regulatory protein ArgR, repressor of arg regulon\n",
      "b3237 ['Arginine pathway regulatory protein ArgR, repressor of arg regulon']\n",
      "\n",
      "b3238 ['probable exported protein YPO3518']\n",
      "\n",
      "KO KO:K03623 ribonuclease inhibitor\n",
      "SSO SSO:000001160 Barstar, ribonuclease (Barnase) inhibitor\n",
      "b3239 ['Barstar, ribonuclease (Barnase) inhibitor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03468 p-hydroxybenzoic acid efflux pump subunit AaeB\n",
      "SSO SSO:000000644 AaeAB efflux system for hydroxylated, aromatic carboxylic acids, inner membrane subunit AaeB\n",
      "b3240 ['AaeAB efflux system for hydroxylated, aromatic carboxylic acids, inner membrane subunit AaeB']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15548 p-hydroxybenzoic acid efflux pump subunit AaeA\n",
      "SSO SSO:000000645 AaeAB efflux system for hydroxylated, aromatic carboxylic acids, membrane fusion component AaeA\n",
      "b3241 ['AaeAB efflux system for hydroxylated, aromatic carboxylic acids, membrane fusion component AaeA']\n",
      "\n",
      "KO KO:K21695 protein AaeX\n",
      "SSO SSO:000000646 AaeX protein, function unknown\n",
      "b3242 ['AaeX protein, function unknown']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K21698 LysR family transcriptional regulator, transcriptional activator for aaeXAB operon\n",
      "SSO SSO:000039542 Transcriptional regulator AaeR, LysR family\n",
      "b3243 ['Transcriptional regulator AaeR, LysR family']\n",
      "\n",
      "KO KO:K03568 TldD protein\n",
      "SSO SSO:000008072 TldD protein, part of TldE/TldD proteolytic complex\n",
      "SSO SSO:000023495 Putative peptidase\n",
      "b3244 ['TldD protein, part of TldE/TldD proteolytic complex']\n",
      "\n",
      "SSO SSO:000028336 Uncharacterized protein YhdP\n",
      "b4472 ['Uncharacterized protein YhdP']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K08301 ribonuclease G [EC:3.1.26.-]\n",
      "SSO SSO:000007075 Ribonuclease G\n",
      "b3247 ['Ribonuclease G']\n",
      "\n",
      "KO KO:K06287 septum formation protein\n",
      "SSO SSO:000007416 Septum formation protein Maf\n",
      "b3248 ['Septum formation protein Maf']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03571 rod shape-determining protein MreD\n",
      "SSO SSO:000007179 Rod shape-determining protein MreD\n",
      "b3249 ['Rod shape-determining protein MreD']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03570 rod shape-determining protein MreC\n",
      "SSO SSO:000007178 Rod shape-determining protein MreC\n",
      "b3250 ['Rod shape-determining protein MreC']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03569 rod shape-determining protein MreB and related proteins\n",
      "SSO SSO:000007177 Rod shape-determining protein MreB\n",
      "b3251 ['Rod shape-determining protein MreB']\n",
      "\n",
      "KO KO:K18765 RNase E specificity factor CsrD\n",
      "SSO SSO:000023873 RNase E specificity factor CsrD\n",
      "b3252 ['RNase E specificity factor CsrD']\n",
      "\n",
      "KO KO:K19745 acrylyl-CoA reductase (NADPH) [EC:1.3.1.-]\n",
      "SSO SSO:000000741 Acryloyl-CoA reductase AcuI/YhdH (EC 1.3.1.84)\n",
      "b3253 ['Acryloyl-CoA reductase AcuI/YhdH (EC 1.3.1.84)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4697 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4646 ['hypothetical protein']\n",
      "\n",
      "GO GO:0042966 biotin carboxyl carrier protein biosynthetic process\n",
      "KO KO:K02160 acetyl-CoA carboxylase biotin carboxyl carrier protein\n",
      "SSO SSO:000001226 Biotin carboxyl carrier protein of acetyl-CoA carboxylase\n",
      "b3255 ['Biotin carboxyl carrier protein of acetyl-CoA carboxylase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K01961 acetyl-CoA carboxylase, biotin carboxylase subunit [EC:6.4.1.2 6.3.4.14]\n",
      "SSO SSO:000001232 Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)\n",
      "b3256 ['Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)']\n",
      "\n",
      "SSO SSO:000028337 Uncharacterized protein YhdT\n",
      "b3257 ['Uncharacterized protein YhdT']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015937 coenzyme A biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K14392 sodium/pantothenate symporter\n",
      "SSO SSO:000005731 Pantothenate:Na+ symporter (TC 2.A.21.1.1)\n",
      "b3258 ['Pantothenate:Na+ symporter (TC 2.A.21.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K02687 ribosomal protein L11 methyltransferase [EC:2.1.1.-]\n",
      "SSO SSO:000007126 Ribosomal protein L11 methyltransferase (EC 2.1.1.-)\n",
      "b3259 ['Ribosomal protein L11 methyltransferase']\n",
      "\n",
      "KO KO:K05540 tRNA-dihydrouridine synthase B [EC:1.-.-.-]\n",
      "SSO SSO:000009357 tRNA dihydrouridine synthase B (EC 1.-.-.-)\n",
      "SSO SSO:000033453 tRNA-dihydrouridine synthase DusB\n",
      "b3260 ['tRNA-dihydrouridine synthase DusB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03557 Fis family transcriptional regulator, factor for inversion stimulation protein\n",
      "SSO SSO:000002135 DNA-binding protein Fis\n",
      "b3261 ['DNA-binding protein Fis']\n",
      "\n",
      "KO KO:K00571 site-specific DNA-methyltransferase (adenine-specific) [EC:2.1.1.72]\n",
      "KO KO:K07319 adenine-specific DNA-methyltransferase [EC:2.1.1.72]\n",
      "SSO SSO:000008477 Type II restriction adenine-specific methylase (EC 2.1.1.72)\n",
      "b3262 ['Type II restriction adenine-specific methylase (EC 2.1.1.72)']\n",
      "\n",
      "SSO SSO:000028338 Uncharacterized protein YhdU\n",
      "SSO SSO:000030665 putative membrane protein\n",
      "b3263 ['Uncharacterized protein YhdU']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K18140 TetR/AcrR family transcriptional regulator, acrEF/envCD operon repressor\n",
      "SSO SSO:000006314 Probable acrEF/envCD operon repressor EnvR\n",
      "b3264 ['Probable acrEF/envCD operon repressor EnvR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03585 membrane fusion protein, multidrug efflux system\n",
      "KO KO:K18141 membrane fusion protein, multidrug efflux system\n",
      "SSO SSO:000004973 Multidrug efflux system AcrEF-TolC, membrane fusion component AcrE\n",
      "b3265 ['Multidrug efflux system AcrEF-TolC, membrane fusion component AcrE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K18138 multidrug efflux pump\n",
      "KO KO:K18142 multidrug efflux pump\n",
      "SSO SSO:000004972 Multidrug efflux system AcrEF-TolC, inner-membrane proton/drug antiporter AcrF (RND type)\n",
      "b3266 ['Multidrug efflux system AcrEF-TolC, inner-membrane proton/drug antiporter AcrF (RND type)']\n",
      "\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "SSO SSO:000027261 Uncharacterized membrane protein YhdV\n",
      "b3267 ['Uncharacterized membrane protein YhdV']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K09969 general L-amino acid transport system substrate-binding protein\n",
      "SSO SSO:000034598 Amino acid ABC transporter, substrate-binding protein YhdW\n",
      "b3268 ['Amino acid ABC transporter, substrate-binding protein YhdW']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K09970 general L-amino acid transport system permease protein\n",
      "SSO SSO:000034595 Amino acid ABC transporter, permease protein 1, YhdX\n",
      "b3269 ['Amino acid ABC transporter, permease protein 1, YhdX']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K09971 general L-amino acid transport system permease protein\n",
      "SSO SSO:000034596 Amino acid ABC transporter, permease protein 2, YhdY\n",
      "b3270 ['Amino acid ABC transporter, permease protein 2, YhdY']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02028 polar amino acid transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K09972 general L-amino acid transport system ATP-binding protein [EC:7.4.2.1]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000010567 Amino acid ABC transporter, ATP-binding protein YhdZ\n",
      "b3271 ['Amino acid ABC transporter, ATP-binding protein YhdZ']\n",
      "\n",
      "KO KO:K01726 gamma-carbonic anhydrase [EC:4.2.1.-]\n",
      "SSO SSO:000006458 Protein YrdA\n",
      "b3279 ['Protein YrdA']\n",
      "\n",
      "SSO SSO:000028536 Uncharacterized protein YrdB\n",
      "b3280 ['Uncharacterized protein YrdB']\n",
      "\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K00014 shikimate dehydrogenase [EC:1.1.1.25]\n",
      "SSO SSO:000007459 Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)\n",
      "b3281 ['Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)']\n",
      "\n",
      "KO KO:K07566 L-threonylcarbamoyladenylate synthase [EC:2.7.7.87]\n",
      "SSO SSO:000039483 Threonylcarbamoyl-AMP synthase (EC 2.7.7.87)\n",
      "b3282 ['Threonylcarbamoyl-AMP synthase (EC 2.7.7.87)']\n",
      "\n",
      "KO KO:K07479 putative DNA topoisomerase\n",
      "SSO SSO:000008782 Uncharacterized protein YrdD\n",
      "b3283 ['Uncharacterized protein YrdD']\n",
      "\n",
      "KO KO:K03747 Smg protein\n",
      "SSO SSO:000006512 Protein of unknown function Smg\n",
      "b3284 ['Protein of unknown function Smg']\n",
      "\n",
      "KO KO:K04096 DNA processing protein\n",
      "SSO SSO:000007181 Rossmann fold nucleotide-binding protein Smf possibly involved in DNA uptake\n",
      "b4473 ['Rossmann fold nucleotide-binding protein Smf possibly involved in DNA uptake']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K01462 peptide deformylase [EC:3.5.1.88]\n",
      "SSO SSO:000005794 Peptide deformylase (EC 3.5.1.88)\n",
      "b3287 ['Peptide deformylase (EC 3.5.1.88)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00604 methionyl-tRNA formyltransferase [EC:2.1.2.9]\n",
      "SSO SSO:000004840 Methionyl-tRNA formyltransferase (EC 2.1.2.9)\n",
      "b3288 ['Methionyl-tRNA formyltransferase (EC 2.1.2.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03500 16S rRNA (cytosine967-C5)-methyltransferase [EC:2.1.1.176]\n",
      "SSO SSO:000000057 16S rRNA (cytosine(967)-C(5))-methyltransferase (EC 2.1.1.176)\n",
      "b3289 ['16S rRNA (cytosine(967)-C(5))-methyltransferase (EC 2.1.1.176)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03499 trk system potassium uptake protein\n",
      "SSO SSO:000008386 Trk potassium uptake system protein TrkA\n",
      "b3290 ['Trk potassium uptake system protein TrkA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03282 large conductance mechanosensitive channel\n",
      "SSO SSO:000018870 Large-conductance mechanosensitive channel\n",
      "b3291 ['Large-conductance mechanosensitive channel']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K09890 alternative ribosome-rescue factor\n",
      "SSO SSO:000000935 Alternative ribosome-rescue factor A\n",
      "b4550 ['Alternative ribosome-rescue factor A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K13638 MerR family transcriptional regulator, Zn(II)-responsive regulator of zntA\n",
      "SSO SSO:000025877 Transcriptional regulator ZntR, MerR family\n",
      "b3292 ['Transcriptional regulator ZntR, MerR family']\n",
      "\n",
      "SSO SSO:000028335 Uncharacterized protein YhdN\n",
      "b3293 ['Uncharacterized protein YhdN']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02879 large subunit ribosomal protein L17\n",
      "SSO SSO:000004224 LSU ribosomal protein L17p\n",
      "b3294 ['LSU ribosomal protein L17p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03040 DNA-directed RNA polymerase subunit alpha [EC:2.7.7.6]\n",
      "SSO SSO:000002173 DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)\n",
      "b3295 ['DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02986 small subunit ribosomal protein S4\n",
      "b3296 ['SSU ribosomal protein S4p (S9e)', 'SSU ribosomal protein S4p (S9e), zinc-independent']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02948 small subunit ribosomal protein S11\n",
      "SSO SSO:000007243 SSU ribosomal protein S11p (S14e)\n",
      "b3297 ['SSU ribosomal protein S11p (S14e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02952 small subunit ribosomal protein S13\n",
      "SSO SSO:000007251 SSU ribosomal protein S13p (S18e)\n",
      "b3298 ['SSU ribosomal protein S13p (S18e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02919 large subunit ribosomal protein L36\n",
      "b3299 ['LSU ribosomal protein L36p', 'LSU ribosomal protein L36p, zinc-dependent']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03076 preprotein translocase subunit SecY\n",
      "SSO SSO:000006526 Protein translocase subunit SecY\n",
      "b3300 ['Protein translocase subunit SecY']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02876 large subunit ribosomal protein L15\n",
      "SSO SSO:000004216 LSU ribosomal protein L15p (L27Ae)\n",
      "b3301 ['LSU ribosomal protein L15p (L27Ae)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02907 large subunit ribosomal protein L30\n",
      "SSO SSO:000004280 LSU ribosomal protein L30p (L7e)\n",
      "b3302 ['LSU ribosomal protein L30p (L7e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02988 small subunit ribosomal protein S5\n",
      "SSO SSO:000007329 SSU ribosomal protein S5p (S2e)\n",
      "b3303 ['SSU ribosomal protein S5p (S2e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02881 large subunit ribosomal protein L18\n",
      "SSO SSO:000004229 LSU ribosomal protein L18p (L5e)\n",
      "b3304 ['LSU ribosomal protein L18p (L5e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02933 large subunit ribosomal protein L6\n",
      "SSO SSO:000004355 LSU ribosomal protein L6p (L9e)\n",
      "b3305 ['LSU ribosomal protein L6p (L9e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "GO GO:0009386 translational attenuation\n",
      "KO KO:K02994 small subunit ribosomal protein S8\n",
      "SSO SSO:000007341 SSU ribosomal protein S8p (S15Ae)\n",
      "b3306 ['SSU ribosomal protein S8p (S15Ae)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02954 small subunit ribosomal protein S14\n",
      "b3307 ['SSU ribosomal protein S14p (S29e)', 'SSU ribosomal protein S14p (S29e), zinc-independent']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02931 large subunit ribosomal protein L5\n",
      "SSO SSO:000004352 LSU ribosomal protein L5p (L11e)\n",
      "b3308 ['LSU ribosomal protein L5p (L11e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02895 large subunit ribosomal protein L24\n",
      "SSO SSO:000004256 LSU ribosomal protein L24p (L26e)\n",
      "b3309 ['LSU ribosomal protein L24p (L26e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02874 large subunit ribosomal protein L14\n",
      "SSO SSO:000004212 LSU ribosomal protein L14p (L23e)\n",
      "b3310 ['LSU ribosomal protein L14p (L23e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02961 small subunit ribosomal protein S17\n",
      "SSO SSO:000007270 SSU ribosomal protein S17p (S11e)\n",
      "b3311 ['SSU ribosomal protein S17p (S11e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02904 large subunit ribosomal protein L29\n",
      "SSO SSO:000004273 LSU ribosomal protein L29p (L35e)\n",
      "b3312 ['LSU ribosomal protein L29p (L35e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02878 large subunit ribosomal protein L16\n",
      "SSO SSO:000004219 LSU ribosomal protein L16p (L10e)\n",
      "b3313 ['LSU ribosomal protein L16p (L10e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02982 small subunit ribosomal protein S3\n",
      "SSO SSO:000007319 SSU ribosomal protein S3p (S3e)\n",
      "b3314 ['SSU ribosomal protein S3p (S3e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02890 large subunit ribosomal protein L22\n",
      "SSO SSO:000004247 LSU ribosomal protein L22p (L17e)\n",
      "b3315 ['LSU ribosomal protein L22p (L17e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02965 small subunit ribosomal protein S19\n",
      "SSO SSO:000007280 SSU ribosomal protein S19p (S15e)\n",
      "b3316 ['SSU ribosomal protein S19p (S15e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02886 large subunit ribosomal protein L2\n",
      "SSO SSO:000004276 LSU ribosomal protein L2p (L8e)\n",
      "b3317 ['LSU ribosomal protein L2p (L8e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02892 large subunit ribosomal protein L23\n",
      "SSO SSO:000004252 LSU ribosomal protein L23p (L23Ae)\n",
      "b3318 ['LSU ribosomal protein L23p (L23Ae)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "GO GO:0006412 translation\n",
      "GO GO:0009386 translational attenuation\n",
      "KO KO:K02926 large subunit ribosomal protein L4\n",
      "SSO SSO:000004342 LSU ribosomal protein L4p (L1e)\n",
      "b3319 ['LSU ribosomal protein L4p (L1e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02906 large subunit ribosomal protein L3\n",
      "SSO SSO:000004323 LSU ribosomal protein L3p (L3e)\n",
      "b3320 ['LSU ribosomal protein L3p (L3e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02946 small subunit ribosomal protein S10\n",
      "SSO SSO:000007239 SSU ribosomal protein S10p (S20e)\n",
      "b3321 ['SSU ribosomal protein S10p (S20e)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "SSO SSO:000021792 PioO protein\n",
      "b3322 ['PioO protein']\n",
      "\n",
      "SSO SSO:000003071 General secretion pathway protein A\n",
      "b3323 ['General secretion pathway protein A']\n",
      "\n",
      "KO KO:K02452 general secretion pathway protein C\n",
      "SSO SSO:000003073 General secretion pathway protein C\n",
      "b3324 ['General secretion pathway protein C']\n",
      "\n",
      "KO KO:K02453 general secretion pathway protein D\n",
      "SSO SSO:000003074 General secretion pathway protein D\n",
      "b3325 ['General secretion pathway protein D']\n",
      "\n",
      "KO KO:K02454 general secretion pathway protein E\n",
      "SSO SSO:000003075 General secretion pathway protein E\n",
      "b3326 ['General secretion pathway protein E']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02455 general secretion pathway protein F\n",
      "SSO SSO:000003076 General secretion pathway protein F\n",
      "b3327 ['General secretion pathway protein F']\n",
      "\n",
      "KO KO:K02456 general secretion pathway protein G\n",
      "SSO SSO:000009121 general secretion pathway protein G\n",
      "b3328 ['General secretion pathway protein G']\n",
      "\n",
      "KO KO:K02457 general secretion pathway protein H\n",
      "SSO SSO:000009122 general secretion pathway protein H\n",
      "b3329 ['General secretion pathway protein H']\n",
      "\n",
      "KO KO:K02458 general secretion pathway protein I\n",
      "SSO SSO:000009123 general secretion pathway protein I\n",
      "b3330 ['General secretion pathway protein I']\n",
      "\n",
      "KO KO:K02459 general secretion pathway protein J\n",
      "SSO SSO:000003077 General secretion pathway protein J\n",
      "b3331 ['General secretion pathway protein J']\n",
      "\n",
      "KO KO:K02460 general secretion pathway protein K\n",
      "SSO SSO:000003078 General secretion pathway protein K\n",
      "b3332 ['General secretion pathway protein K']\n",
      "\n",
      "KO KO:K02461 general secretion pathway protein L\n",
      "SSO SSO:000009124 general secretion pathway protein L\n",
      "b3333 ['General secretion pathway protein L']\n",
      "\n",
      "KO KO:K02462 general secretion pathway protein M\n",
      "SSO SSO:000009125 general secretion pathway protein M\n",
      "b3334 ['General secretion pathway protein M']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02654 leader peptidase (prepilin peptidase) / N-methyltransferase [EC:3.4.23.43 2.1.1.-]\n",
      "SSO SSO:000018880 Leader peptidase HopD\n",
      "b3335 ['Leader peptidase HopD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K03594 bacterioferritin [EC:1.16.3.1]\n",
      "SSO SSO:000001157 Bacterioferritin (EC 1.16.3.1)\n",
      "b3336 ['Bacterioferritin (EC 1.16.3.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K02192 bacterioferritin-associated ferredoxin\n",
      "SSO SSO:000001158 Bacterioferritin-associated ferredoxin\n",
      "b3337 ['Bacterioferritin-associated ferredoxin']\n",
      "\n",
      "GO GO:0009057 macromolecule catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K13381 bifunctional chitinase/lysozyme [EC:3.2.1.14 3.2.1.17]\n",
      "SSO SSO:000012169 Chitinase (EC 3.2.1.14)\n",
      "b3338 ['Chitinase (EC 3.2.1.14)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K02358 elongation factor Tu\n",
      "SSO SSO:000008307 Translation elongation factor Tu\n",
      "b3339 ['Translation elongation factor Tu']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02355 elongation factor G\n",
      "SSO SSO:000008297 Translation elongation factor G\n",
      "b3340 ['Translation elongation factor G']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02992 small subunit ribosomal protein S7\n",
      "SSO SSO:000007337 SSU ribosomal protein S7p (S5e)\n",
      "b3341 ['SSU ribosomal protein S7p (S5e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02950 small subunit ribosomal protein S12\n",
      "SSO SSO:000007247 SSU ribosomal protein S12p (S23e)\n",
      "b3342 ['SSU ribosomal protein S12p (S23e)']\n",
      "\n",
      "KO KO:K07237 tRNA 2-thiouridine synthesizing protein B\n",
      "SSO SSO:000009350 tRNA 5-methylaminomethyl-2-thiouridine synthase subunit TusB\n",
      "b3343 ['tRNA 5-methylaminomethyl-2-thiouridine synthase subunit TusB']\n",
      "\n",
      "KO KO:K07236 tRNA 2-thiouridine synthesizing protein C\n",
      "SSO SSO:000009351 tRNA 5-methylaminomethyl-2-thiouridine synthase subunit TusC\n",
      "b3344 ['tRNA 5-methylaminomethyl-2-thiouridine synthase subunit TusC']\n",
      "\n",
      "KO KO:K07235 tRNA 2-thiouridine synthesizing protein D [EC:2.8.1.-]\n",
      "SSO SSO:000009352 tRNA 5-methylaminomethyl-2-thiouridine synthase subunit TusD\n",
      "b3345 ['tRNA 5-methylaminomethyl-2-thiouridine synthase subunit TusD']\n",
      "\n",
      "b3346 ['YheO-like PAS domain']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03772 FKBP-type peptidyl-prolyl cis-trans isomerase FkpA [EC:5.2.1.8]\n",
      "SSO SSO:000002672 FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)\n",
      "b3347 ['FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)']\n",
      "\n",
      "KO KO:K03745 SlyX protein\n",
      "SSO SSO:000022718 Protein SlyX\n",
      "b3348 ['Protein SlyX']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03775 FKBP-type peptidyl-prolyl cis-trans isomerase SlyD [EC:5.2.1.8]\n",
      "SSO SSO:000002674 FKBP-type peptidyl-prolyl cis-trans isomerase SlyD (EC 5.2.1.8)\n",
      "b3349 ['FKBP-type peptidyl-prolyl cis-trans isomerase SlyD (EC 5.2.1.8)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K07070 uncharacterized protein\n",
      "SSO SSO:000001965 Cytoplasmic protein, probably associated with glutathione-regulated potassium-efflux\n",
      "b4551 ['Cytoplasmic protein, probably associated with glutathione-regulated potassium-efflux']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015386 potassium:proton antiporter activity\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03455 monovalent cation:H+ antiporter-2, CPA2 family\n",
      "KO KO:K11747 glutathione-regulated potassium-efflux system protein KefB\n",
      "SSO SSO:000003232 Glutathione-regulated potassium-efflux system protein KefB\n",
      "b3350 ['Glutathione-regulated potassium-efflux system protein KefB']\n",
      "\n",
      "KO KO:K11748 glutathione-regulated potassium-efflux system ancillary protein KefG\n",
      "SSO SSO:000003231 Glutathione-regulated potassium-efflux system ancillary protein KefG\n",
      "b3351 ['Glutathione-regulated potassium-efflux system ancillary protein KefG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06158 ATP-binding cassette, subfamily F, member 3\n",
      "SSO SSO:000011185 Bis-ABC ATPase YheS\n",
      "b3352 ['Bis-ABC ATPase YheS']\n",
      "\n",
      "KO KO:K07019 uncharacterized protein\n",
      "SSO SSO:000023287 Putative esterase YheT functionally coupled to phosphoribulokinase homolog\n",
      "b3353 ['Putative esterase YheT functionally coupled to phosphoribulokinase homolog']\n",
      "\n",
      "KO KO:K09898 uncharacterized protein\n",
      "SSO SSO:000026607 UPF0270 protein YheU\n",
      "b3354 ['UPF0270 protein YheU']\n",
      "\n",
      "GO GO:0006006 glucose metabolic process\n",
      "KO KO:K00855 phosphoribulokinase [EC:2.7.1.19]\n",
      "SSO SSO:000006049 Phosphoribulokinase homolog, function unknown (EC 2.7.1.19)\n",
      "b3355 ['Phosphoribulokinase homolog, function unknown']\n",
      "\n",
      "KO KO:K07397 putative redox protein\n",
      "SSO SSO:000020835 OsmC family protein\n",
      "SSO SSO:000022732 Protein YhfA\n",
      "b3356 ['Protein YhfA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K10914 CRP/FNR family transcriptional regulator, cyclic AMP receptor protein\n",
      "SSO SSO:000001826 Cyclic AMP receptor protein\n",
      "b3357 ['Cyclic AMP receptor protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008152 metabolic process\n",
      "SSO SSO:000028342 Uncharacterized protein YhfK\n",
      "b3358 ['Uncharacterized protein YhfK']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K00821 acetylornithine/N-succinyldiaminopimelate aminotransferase [EC:2.6.1.11 2.6.1.17]\n",
      "b3359 ['N-acetylornithine aminotransferase (EC 2.6.1.11)', 'N-succinyl-L,L-diaminopimelate aminotransferase (EC 2.6.1.17)']\n",
      "\n",
      "GO GO:0046656 folic acid biosynthetic process\n",
      "KO KO:K01658 anthranilate synthase component II [EC:4.1.3.27]\n",
      "KO KO:K01664 para-aminobenzoate synthetase component II [EC:2.6.1.85]\n",
      "SSO SSO:000005733 Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)\n",
      "b3360 ['Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)']\n",
      "\n",
      "KO KO:K04095 cell filamentation protein\n",
      "SSO SSO:000001489 Cell filamentation protein fic\n",
      "b3361 ['Cell filamentation protein Fic']\n",
      "\n",
      "SSO SSO:000022097 Predicted Fic antitoxin YfhG\n",
      "b3362 ['Predicted Fic antitoxin YfhG']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000005810 Peptidyl-prolyl cis-trans isomerase PpiA precursor (EC 5.2.1.8)\n",
      "b3363 ['Peptidyl-prolyl cis-trans isomerase PpiA precursor (EC 5.2.1.8)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06141 MFS transporter, TsgA protein\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000024467 Selenite- and tellurite-inducible protein TsgA\n",
      "b3364 ['Selenite- and tellurite-inducible protein TsgA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00362 nitrite reductase (NADH) large subunit [EC:1.7.1.15]\n",
      "SSO SSO:000005343 Nitrite reductase [NAD(P)H] large subunit (EC 1.7.1.4)\n",
      "b3365 ['Nitrite reductase [NAD(P)H] large subunit (EC 1.7.1.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00363 nitrite reductase (NADH) small subunit [EC:1.7.1.15]\n",
      "SSO SSO:000005344 Nitrite reductase [NAD(P)H] small subunit (EC 1.7.1.4)\n",
      "SSO SSO:000037210 Nitrite reductase (NADH), small subunit (EC 1.7.1.15)\n",
      "b3366 ['Nitrite reductase [NAD(P)H] small subunit (EC 1.7.1.4)']\n",
      "\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02598 nitrite transporter\n",
      "SSO SSO:000020559 Nitrite transporter NirC\n",
      "b3367 ['Nitrite transporter NirC']\n",
      "\n",
      "KO KO:K02302 uroporphyrin-III C-methyltransferase / precorrin-2 dehydrogenase / sirohydrochlorin ferrochelatase [EC:2.1.1.107 1.3.1.76 4.99.1.4]\n",
      "KO KO:K02303 uroporphyrin-III C-methyltransferase [EC:2.1.1.107]\n",
      "KO KO:K02304 precorrin-2 dehydrogenase / sirohydrochlorin ferrochelatase [EC:1.3.1.76 4.99.1.4]\n",
      "b3368 ['Precorrin-2 oxidase (EC 1.3.1.76)', 'Sirohydrochlorin ferrochelatase activity of CysG (EC 4.99.1.4)', 'Uroporphyrinogen-III methyltransferase (EC 2.1.1.107)']\n",
      "\n",
      "SSO SSO:000028343 Uncharacterized protein YhfL\n",
      "b3369 ['Uncharacterized protein YhfL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K19540 fructoselysine transporter\n",
      "SSO SSO:000016905 Fructoselysine/psicoselysine transporter FrlA\n",
      "b3370 ['Fructoselysine/psicoselysine transporter FrlA']\n",
      "\n",
      "KO KO:K10708 fructoselysine 6-phosphate deglycase [EC:3.5.-.-]\n",
      "SSO SSO:000016904 Fructoselysine-6-phosphate deglycase\n",
      "b3371 ['Fructoselysine-6-phosphate deglycase']\n",
      "\n",
      "KO KO:K10709 fructoselysine 3-epimerase [EC:5.1.3.41]\n",
      "SSO SSO:000016902 Fructoselysine 3-epimerase\n",
      "b4474 ['Fructoselysine 3-epimerase']\n",
      "\n",
      "KO KO:K10710 fructoselysine 6-kinase [EC:2.7.1.218]\n",
      "SSO SSO:000016903 Fructoselysine 6-kinase\n",
      "b3374 ['Fructoselysine 6-kinase']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03710 GntR family transcriptional regulator\n",
      "KO KO:K10711 GntR family transcriptional regulator, frlABCD operon transcriptional regulator\n",
      "SSO SSO:000025894 Transcriptional regulator of fructoselysine utilization operon FrlR\n",
      "b3375 ['Transcriptional regulator of fructoselysine utilization operon FrlR']\n",
      "\n",
      "SSO SSO:000008796 Uncharacterized pyridoxal phosphate protein YhfS\n",
      "b3376 ['Uncharacterized pyridoxal phosphate protein YhfS']\n",
      "\n",
      "SSO SSO:000008768 Uncharacterized protein YhfT\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b3377 ['Uncharacterized protein YhfT']\n",
      "\n",
      "SSO SSO:000008769 Uncharacterized protein YhfU\n",
      "b3378 ['Uncharacterized protein YhfU']\n",
      "\n",
      "KO KO:K07048 phosphotriesterase-related protein\n",
      "SSO SSO:000006057 Phosphotriesterase like protein Php\n",
      "SSO SSO:000021733 Phosphotriesterase homology protein\n",
      "b3379 ['Phosphotriesterase like protein Php']\n",
      "\n",
      "KO KO:K01839 phosphopentomutase [EC:5.4.2.7]\n",
      "SSO SSO:000006029 Phosphopentomutase like protein YhfW\n",
      "b3380 ['Phosphopentomutase like protein YhfW']\n",
      "\n",
      "SSO SSO:000008701 Uncharacterized alanine racemase domain-containing protein YhfX\n",
      "b3381 ['Uncharacterized alanine racemase domain-containing protein YhfX']\n",
      "\n",
      "SSO SSO:000005721 PTS-regulatory domain-containing protein YhfY\n",
      "b3382 ['PTS-regulatory domain-containing protein YhfY']\n",
      "\n",
      "SSO SSO:000003601 Hypothetical protein YhfZ\n",
      "b3383 ['Hypothetical protein YhfZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01867 tryptophanyl-tRNA synthetase [EC:6.1.1.2]\n",
      "SSO SSO:000008401 Tryptophanyl-tRNA synthetase (EC 6.1.1.2)\n",
      "b3384 ['Tryptophanyl-tRNA synthetase (EC 6.1.1.2)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01091 phosphoglycolate phosphatase [EC:3.1.3.18]\n",
      "SSO SSO:000005983 Phosphoglycolate phosphatase (EC 3.1.3.18)\n",
      "b3385 ['Phosphoglycolate phosphatase (EC 3.1.3.18)']\n",
      "\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01783 ribulose-phosphate 3-epimerase [EC:5.1.3.1]\n",
      "SSO SSO:000007162 Ribulose-phosphate 3-epimerase (EC 5.1.3.1)\n",
      "b3386 ['Ribulose-phosphate 3-epimerase (EC 5.1.3.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006306 DNA methylation\n",
      "KO KO:K06223 DNA adenine methylase [EC:2.1.1.72]\n",
      "SSO SSO:000004866 Methyl-directed repair DNA adenine methylase (EC 2.1.1.72)\n",
      "b3387 ['Methyl-directed repair DNA adenine methylase (EC 2.1.1.72)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03112 DamX protein\n",
      "SSO SSO:000007418 Septum-associated cell division protein DamX\n",
      "b3388 ['Septum-associated cell division protein DamX']\n",
      "\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K01735 3-dehydroquinate synthase [EC:4.2.3.4]\n",
      "SSO SSO:000000276 3-dehydroquinate synthase (EC 4.2.3.4)\n",
      "b3389 ['3-dehydroquinate synthase (EC 4.2.3.4)']\n",
      "\n",
      "GO GO:0009423 chorismate biosynthetic process\n",
      "KO KO:K00891 shikimate kinase [EC:2.7.1.71]\n",
      "SSO SSO:000007462 Shikimate kinase I (EC 2.7.1.71)\n",
      "b3390 ['Shikimate kinase I (EC 2.7.1.71)']\n",
      "\n",
      "KO KO:K02507 protein transport protein HofQ\n",
      "SSO SSO:000008574 Type IV pilus biogenesis protein PilQ\n",
      "b3391 ['Type IV pilus biogenesis protein PilQ']\n",
      "\n",
      "KO KO:K12291 pilus assembly protein HofP\n",
      "SSO SSO:000008573 Type IV pilus biogenesis protein PilP\n",
      "b3392 ['Type IV pilus biogenesis protein PilP']\n",
      "\n",
      "KO KO:K12290 pilus assembly protein HofO\n",
      "SSO SSO:000008571 Type IV pilus biogenesis protein PilO\n",
      "b3393 ['Type IV pilus biogenesis protein PilO']\n",
      "\n",
      "KO KO:K12289 pilus assembly protein HofN\n",
      "SSO SSO:000008569 Type IV pilus biogenesis protein PilN\n",
      "b3394 ['Type IV pilus biogenesis protein PilN']\n",
      "\n",
      "KO KO:K12288 pilus assembly protein HofM\n",
      "SSO SSO:000008567 Type IV pilus biogenesis protein PilM\n",
      "b3395 ['Type IV pilus biogenesis protein PilM']\n",
      "\n",
      "EC EC:2.4.1.129 Peptidoglycan glycosyltransferase.\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K05366 penicillin-binding protein 1A [EC:2.4.1.129 3.4.16.4]\n",
      "SSO SSO:000005028 Multimodular transpeptidase-transglycosylase (EC 3.4.-.-) (EC 2.4.1.129)\n",
      "b3396 ['Multimodular transpeptidase-transglycosylase (EC 2.4.1.129) (EC 3.4.-.-)']\n",
      "\n",
      "KO KO:K08312 ADP-ribose diphosphatase [EC:3.6.1.-]\n",
      "SSO SSO:000000525 ADP compounds hydrolase NudE (EC 3.6.1.-)\n",
      "b3397 ['ADP compounds hydrolase NudE']\n",
      "\n",
      "SSO SSO:000003692 IgaA: a membrane protein that prevents overactivation of the Rcs regulatory system\n",
      "b3398 ['IgaA: a membrane protein that prevents overactivation of the Rcs regulatory system']\n",
      "\n",
      "KO KO:K20881 5'-nucleotidase [EC:3.1.3.5]\n",
      "SSO SSO:000002982 GMP/IMP nucleotidase YrfG\n",
      "b3399 ['GMP/IMP nucleotidase YrfG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K04762 ribosome-associated heat shock protein Hsp15\n",
      "SSO SSO:000007150 Ribosome-associated heat shock protein implicated in the recycling of the 50S subunit (S4 paralog)\n",
      "b3400 ['Ribosome-associated heat shock protein implicated in the recycling of the 50S subunit (S4 paralog)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04083 molecular chaperone Hsp33\n",
      "SSO SSO:000034094 33 kDa chaperonin HslO\n",
      "b3401 ['33 kDa chaperonin HslO']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000028344 Uncharacterized protein YhgE\n",
      "b3402 ['Uncharacterized protein YhgE']\n",
      "\n",
      "GO GO:0006094 gluconeogenesis\n",
      "KO KO:K01610 phosphoenolpyruvate carboxykinase (ATP) [EC:4.1.1.49]\n",
      "SSO SSO:000005953 Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)\n",
      "b3403 ['Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07638 two-component system, OmpR family, osmolarity sensor histidine kinase EnvZ [EC:2.7.13.3]\n",
      "SSO SSO:000005524 Osmolarity sensory histidine kinase EnvZ\n",
      "b3404 ['Osmolarity sensory histidine kinase EnvZ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07659 two-component system, OmpR family, phosphate regulon response regulator OmpR\n",
      "SSO SSO:000008463 Two-component system response regulator OmpR\n",
      "b3405 ['Two-component system response regulator OmpR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K04760 transcription elongation factor GreB\n",
      "SSO SSO:000008141 Transcription elongation factor GreB\n",
      "b3406 ['Transcription elongation factor GreB']\n",
      "\n",
      "KO KO:K06959 protein Tex\n",
      "SSO SSO:000008137 Transcription accessory protein (S1 RNA-binding domain)\n",
      "b3407 ['Transcription accessory protein (S1 RNA-binding domain)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K04758 ferrous iron transport protein A\n",
      "SSO SSO:000035911 Ferrous iron transporter-associated protein FeoA\n",
      "b3408 ['Ferrous iron transporter-associated protein FeoA']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04759 ferrous iron transport protein B\n",
      "SSO SSO:000035917 Ferrous iron transporter FeoB\n",
      "b3409 ['Ferrous iron transporter FeoB']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K07490 ferrous iron transport protein C\n",
      "SSO SSO:000035918 Ferrous iron-sensing transcriptional regulator FeoC\n",
      "b3410 ['Ferrous iron-sensing transcriptional regulator FeoC']\n",
      "\n",
      "KO KO:K01157 Unknown\n",
      "b3411 ['Uncharacterized protein YhgA']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K02170 pimeloyl-[acyl-carrier protein] methyl ester esterase [EC:3.1.1.85]\n",
      "SSO SSO:000006114 Pimeloyl-[acyl-carrier protein] methyl ester esterase BioH (EC 3.1.1.85)\n",
      "b3412 ['Pimeloyl-[acyl-carrier protein] methyl ester esterase BioH (EC 3.1.1.85)']\n",
      "\n",
      "KO KO:K02242 competence protein ComFC\n",
      "b3413 ['Competence protein F homolog, phosphoribosyltransferase domain', 'protein YhgH required for utilization of DNA as sole source of carbon and energy']\n",
      "\n",
      "KO KO:K07400 Fe/S biogenesis protein NfuA\n",
      "SSO SSO:000041765 [4Fe-4S] cluster carrier protein NfuA\n",
      "b3414 ['[4Fe-4S] cluster carrier protein NfuA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03299 gluconate:H+ symporter, GntP family\n",
      "KO KO:K06155 Gnt-I system high-affinity gluconate transporter\n",
      "SSO SSO:000003436 High-affinity gluconate transporter GntT\n",
      "b3415 ['High-affinity gluconate transporter GntT']\n",
      "\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00705 4-alpha-glucanotransferase [EC:2.4.1.25]\n",
      "SSO SSO:000000349 4-alpha-glucanotransferase (amylomaltase) (EC 2.4.1.25)\n",
      "b3416 ['4-alpha-glucanotransferase (amylomaltase) (EC 2.4.1.25)']\n",
      "\n",
      "EC EC:2.4.1.1 Glycogen phosphorylase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00688 glycogen phosphorylase [EC:2.4.1.1]\n",
      "SSO SSO:000004700 Maltodextrin phosphorylase (EC 2.4.1.1)\n",
      "b3417 ['Maltodextrin phosphorylase (EC 2.4.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03556 LuxR family transcriptional regulator, maltose regulon positive regulatory protein\n",
      "SSO SSO:000008189 Transcriptional activator of maltose regulon, MalT\n",
      "b3418 ['Transcriptional activator of maltose regulon, MalT']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016070 RNA metabolic process\n",
      "KO KO:K01974 RNA 3'-terminal phosphate cyclase (ATP) [EC:6.5.1.4]\n",
      "SSO SSO:000006885 RNA 3'-terminal phosphate cyclase (EC 6.5.1.4)\n",
      "b4475 [\"RNA 3'-terminal phosphate cyclase (EC 6.5.1.4)\"]\n",
      "\n",
      "KO KO:K14415 tRNA-splicing ligase RtcB (3'-phosphate/5'-hydroxy nucleic acid ligase) [EC:6.5.1.8]\n",
      "SSO SSO:000006926 RNA-2',3'-PO4:RNA-5'-OH ligase\n",
      "b3421 [\"RNA-2',3'-PO4:RNA-5'-OH ligase\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K14414 transcriptional regulatory protein RtcR\n",
      "SSO SSO:000025967 Transcriptional regulatory protein RtcR\n",
      "b3422 ['Transcriptional regulatory protein RtcR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02444 DeoR family transcriptional regulator, glycerol-3-phosphate regulon repressor\n",
      "SSO SSO:000003259 Glycerol-3-phosphate regulon repressor GlpR\n",
      "b3423 ['Glycerol-3-phosphate regulon repressor GlpR']\n",
      "\n",
      "KO KO:K02441 GlpG protein\n",
      "SSO SSO:000007062 Rhomboid protease GlpG (EC 3.4.21.105)\n",
      "b3424 ['Rhomboid protease GlpG (EC 3.4.21.105)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02439 thiosulfate sulfurtransferase [EC:2.8.1.1]\n",
      "SSO SSO:000008049 Thiosulfate sulfurtransferase GlpE (EC 2.8.1.1)\n",
      "b3425 ['Thiosulfate sulfurtransferase GlpE (EC 2.8.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K00111 glycerol-3-phosphate dehydrogenase [EC:1.1.5.3]\n",
      "SSO SSO:000000839 Aerobic glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)\n",
      "b3426 ['Aerobic glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b3427 ['hypothetical protein']\n",
      "\n",
      "EC EC:2.4.1.1 Glycogen phosphorylase.\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00688 glycogen phosphorylase [EC:2.4.1.1]\n",
      "SSO SSO:000003290 Glycogen phosphorylase (EC 2.4.1.1)\n",
      "b3428 ['Glycogen phosphorylase (EC 2.4.1.1)']\n",
      "\n",
      "EC EC:2.4.1.11 Glycogen(starch) synthase.\n",
      "EC EC:2.4.1.183 Alpha-1,3-glucan synthase.\n",
      "EC EC:2.4.1.21 Starch synthase.\n",
      "EC EC:2.4.1.242 NDP-glucose--starch glucosyltransferase.\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00703 starch synthase [EC:2.4.1.21]\n",
      "SSO SSO:000003291 Glycogen synthase, ADP-glucose transglucosylase (EC 2.4.1.21)\n",
      "SSO SSO:000017356 Glycogen synthase (EC 2.4.1.11)\n",
      "b3429 ['Glycogen synthase, ADP-glucose transglucosylase (EC 2.4.1.21)']\n",
      "\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00975 glucose-1-phosphate adenylyltransferase [EC:2.7.7.27]\n",
      "SSO SSO:000003124 Glucose-1-phosphate adenylyltransferase (EC 2.7.7.27)\n",
      "b3430 ['Glucose-1-phosphate adenylyltransferase (EC 2.7.7.27)']\n",
      "\n",
      "EC EC:2.4.1.18 1,4-alpha-glucan branching enzyme.\n",
      "EC EC:2.4.1.19 Cyclomaltodextrin glucanotransferase.\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "EC EC:2.4.1.4 Amylosucrase.\n",
      "EC EC:2.4.1.7 Sucrose phosphorylase.\n",
      "EC EC:2.4.99.16 Starch synthase (maltosyl-transferring).\n",
      "EC EC:3.2.1.1 Alpha-amylase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.116 Glucan 1,4-alpha-maltotriohydrolase.\n",
      "EC EC:3.2.1.133 Glucan 1,4-alpha-maltohydrolase.\n",
      "EC EC:3.2.1.135 Neopullulanase.\n",
      "EC EC:3.2.1.141 4-alpha-D-((1->4)-alpha-D-glucano)trehalose trehalohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.33 Amylo-alpha-1,6-glucosidase.\n",
      "EC EC:3.2.1.41 Pullulanase.\n",
      "EC EC:3.2.1.54 Cyclomaltodextrinase.\n",
      "EC EC:3.2.1.60 Glucan 1,4-alpha-maltotetraohydrolase.\n",
      "EC EC:3.2.1.68 Isoamylase.\n",
      "EC EC:3.2.1.70 Glucan 1,6-alpha-glucosidase.\n",
      "EC EC:3.2.1.93 Alpha,alpha-phosphotrehalase.\n",
      "EC EC:3.2.1.98 Glucan 1,4-alpha-maltohexaosidase.\n",
      "EC EC:5.4.99.11 Isomaltulose synthase.\n",
      "EC EC:5.4.99.15 (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase.\n",
      "EC EC:5.4.99.16 Maltose alpha-D-glucosyltransferase.\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01214 isoamylase [EC:3.2.1.68]\n",
      "KO KO:K02438 glycogen debranching enzyme [EC:3.2.1.196]\n",
      "SSO SSO:000003289 Glycogen debranching enzyme (EC 3.2.1.-)\n",
      "SSO SSO:000018920 Limit dextrin alpha-1,6-maltotetraose-hydrolase (EC 3.2.1.196)\n",
      "b3431 ['Limit dextrin alpha-1,6-maltotetraose-hydrolase (EC 3.2.1.196)']\n",
      "\n",
      "EC EC:2.4.1.18 1,4-alpha-glucan branching enzyme.\n",
      "EC EC:2.4.1.19 Cyclomaltodextrin glucanotransferase.\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "EC EC:2.4.1.4 Amylosucrase.\n",
      "EC EC:2.4.1.7 Sucrose phosphorylase.\n",
      "EC EC:2.4.99.16 Starch synthase (maltosyl-transferring).\n",
      "EC EC:3.2.1.1 Alpha-amylase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.116 Glucan 1,4-alpha-maltotriohydrolase.\n",
      "EC EC:3.2.1.133 Glucan 1,4-alpha-maltohydrolase.\n",
      "EC EC:3.2.1.135 Neopullulanase.\n",
      "EC EC:3.2.1.141 4-alpha-D-((1->4)-alpha-D-glucano)trehalose trehalohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.33 Amylo-alpha-1,6-glucosidase.\n",
      "EC EC:3.2.1.41 Pullulanase.\n",
      "EC EC:3.2.1.54 Cyclomaltodextrinase.\n",
      "EC EC:3.2.1.60 Glucan 1,4-alpha-maltotetraohydrolase.\n",
      "EC EC:3.2.1.68 Isoamylase.\n",
      "EC EC:3.2.1.70 Glucan 1,6-alpha-glucosidase.\n",
      "EC EC:3.2.1.93 Alpha,alpha-phosphotrehalase.\n",
      "EC EC:3.2.1.98 Glucan 1,4-alpha-maltohexaosidase.\n",
      "EC EC:5.4.99.11 Isomaltulose synthase.\n",
      "EC EC:5.4.99.15 (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase.\n",
      "EC EC:5.4.99.16 Maltose alpha-D-glucosyltransferase.\n",
      "GO GO:0000271 polysaccharide biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00700 1,4-alpha-glucan branching enzyme [EC:2.4.1.18]\n",
      "SSO SSO:000000034 1,4-alpha-glucan (glycogen) branching enzyme, GH-13-type (EC 2.4.1.18)\n",
      "SSO SSO:000009466 1,4-alpha-glucan branching enzyme\n",
      "b3432 ['1,4-alpha-glucan (glycogen) branching enzyme, GH-13-type (EC 2.4.1.18)']\n",
      "\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "GO GO:0009088 threonine biosynthetic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0009090 homoserine biosynthetic process\n",
      "KO KO:K00133 aspartate-semialdehyde dehydrogenase [EC:1.2.1.11]\n",
      "SSO SSO:000001100 Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)\n",
      "b3433 ['Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K05595 multiple antibiotic resistance protein\n",
      "SSO SSO:000004721 MarC family integral membrane protein\n",
      "b3434 ['MarC family integral membrane protein']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03299 gluconate:H+ symporter, GntP family\n",
      "KO KO:K06156 Gnt-I system low-affinity gluconate transporter\n",
      "SSO SSO:000004553 Low-affinity gluconate/H+ symporter GntU\n",
      "b4476 ['Low-affinity gluconate/H+ symporter GntU']\n",
      "\n",
      "GO GO:0006006 glucose metabolic process\n",
      "KO KO:K00851 gluconokinase [EC:2.7.1.12]\n",
      "SSO SSO:000003108 Gluconokinase (EC 2.7.1.12)\n",
      "b3437 ['Gluconokinase (EC 2.7.1.12)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009255 Entner-Doudoroff pathway through 6-phosphogluconate\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K06145 LacI family transcriptional regulator, gluconate utilization system Gnt-I transcriptional repressor\n",
      "SSO SSO:000003106 Gluconate utilization system Gnt-I transcriptional repressor\n",
      "b3438 ['Gluconate utilization system Gnt-I transcriptional repressor']\n",
      "\n",
      "KO KO:K06911 uncharacterized protein\n",
      "b3439 ['Quercetin 2,3-dioxygenase (EC 1.13.11.24) => YhhW']\n",
      "\n",
      "KO KO:K16044 scyllo-inositol 2-dehydrogenase (NADP+) [EC:1.1.1.371]\n",
      "SSO SSO:000027393 Uncharacterized oxidoreductase YhhX\n",
      "b3440 ['Uncharacterized oxidoreductase YhhX']\n",
      "\n",
      "KO KO:K03825 putative acetyltransferase [EC:2.3.1.-]\n",
      "b3441 ['L-methionine N-acetyltransferase', 'L-phenylalanine N-acetyltransferase (EC 2.3.1.53)']\n",
      "\n",
      "KO KO:K06887 uncharacterized protein\n",
      "b3442 ['T6SS component Hcp', 'endonuclease']\n",
      "\n",
      "SSO SSO:000023709 Putative uncharacterized protein YrhA\n",
      "b3443 ['Putative uncharacterized protein YrhA']\n",
      "\n",
      "SSO SSO:000018187 IS1 protein InsA\n",
      "b3444 ['IS1 protein InsA']\n",
      "\n",
      "KO KO:K07480 insertion element IS1 protein InsB\n",
      "SSO SSO:000018188 IS1 protein InsB\n",
      "b3445 ['IS1 protein InsB']\n",
      "\n",
      "SSO SSO:000028538 Uncharacterized protein YrhB\n",
      "b3446 ['Uncharacterized protein YrhB']\n",
      "\n",
      "KO KO:K00681 gamma-glutamyltranspeptidase / glutathione hydrolase [EC:2.3.2.2 3.4.19.13]\n",
      "SSO SSO:000003036 Gamma-glutamyltranspeptidase (EC 2.3.2.2)\n",
      "b3447 ['Gamma-glutamyltranspeptidase (EC 2.3.2.2)', 'Glutathione hydrolase (EC 3.4.19.13)']\n",
      "\n",
      "SSO SSO:000028345 Uncharacterized protein YhhA\n",
      "b3448 ['Uncharacterized protein YhhA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "KO KO:K01126 glycerophosphoryl diester phosphodiesterase [EC:3.1.4.46]\n",
      "SSO SSO:000003262 Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)\n",
      "b3449 ['Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K05816 sn-glycerol 3-phosphate transport system ATP-binding protein [EC:3.6.3.20]\n",
      "KO KO:K10112 multiple sugar transport system ATP-binding protein\n",
      "SSO SSO:000017321 Glycerol-3-phosphate ABC transporter, ATP-binding protein UgpC (TC 3.A.1.1.3)\n",
      "b3450 ['Glycerol-3-phosphate ABC transporter, ATP-binding protein UgpC (TC 3.A.1.1.3)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05815 sn-glycerol 3-phosphate transport system permease protein\n",
      "SSO SSO:000017324 Glycerol-3-phosphate ABC transporter, permease protein UgpE (TC 3.A.1.1.3)\n",
      "b3451 ['Glycerol-3-phosphate ABC transporter, permease protein UgpE (TC 3.A.1.1.3)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05814 sn-glycerol 3-phosphate transport system permease protein\n",
      "SSO SSO:000017323 Glycerol-3-phosphate ABC transporter, permease protein UgpA (TC 3.A.1.1.3)\n",
      "b3452 ['Glycerol-3-phosphate ABC transporter, permease protein UgpA (TC 3.A.1.1.3)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02027 multiple sugar transport system substrate-binding protein\n",
      "KO KO:K05813 sn-glycerol 3-phosphate transport system substrate-binding protein\n",
      "SSO SSO:000036152 Glycerol-3-phosphate ABC transporter (EC 7.6.2.10), substrate-binding protein UgpB\n",
      "b3453 ['Glycerol-3-phosphate ABC transporter, substrate-binding protein UgpB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K01996 branched-chain amino acid transport system ATP-binding protein\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000034836 Branched-chain amino acid ABC transporter, ATP-binding protein LivF (TC 3.A.1.4.1)\n",
      "b3454 ['Branched-chain amino acid ABC transporter, ATP-binding protein LivF (TC 3.A.1.4.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K01995 branched-chain amino acid transport system ATP-binding protein\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000034837 Branched-chain amino acid ABC transporter, ATP-binding protein LivG (TC 3.A.1.4.1)\n",
      "b3455 ['Branched-chain amino acid ABC transporter, ATP-binding protein LivG (TC 3.A.1.4.1)']\n",
      "\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01998 branched-chain amino acid transport system permease protein\n",
      "SSO SSO:000034839 Branched-chain amino acid ABC transporter, permease protein LivM (TC 3.A.1.4.1)\n",
      "b3456 ['Branched-chain amino acid ABC transporter, permease protein LivM (TC 3.A.1.4.1)']\n",
      "\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01997 branched-chain amino acid transport system permease protein\n",
      "SSO SSO:000034838 Branched-chain amino acid ABC transporter, permease protein LivH (TC 3.A.1.4.1)\n",
      "b3457 ['Branched-chain amino acid ABC transporter, permease protein LivH (TC 3.A.1.4.1)']\n",
      "\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01999 branched-chain amino acid transport system substrate-binding protein\n",
      "SSO SSO:000036607 Leucine-specific ABC transporter, substrate-binding protein LivK (TC 3.A.1.4.1)\n",
      "b3458 ['Leucine-specific ABC transporter, substrate-binding protein LivK (TC 3.A.1.4.1)']\n",
      "\n",
      "SSO SSO:000000706 Acetyl-CoA sensor PanM, required for maturation of L-aspartate decarboxylase\n",
      "b3459 ['Acetyl-CoA sensor PanM, required for maturation of L-aspartate decarboxylase']\n",
      "\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01999 branched-chain amino acid transport system substrate-binding protein\n",
      "SSO SSO:000034840 Branched-chain amino acid ABC transporter, substrate-binding protein LivJ (TC 3.A.1.4.1)\n",
      "b3460 ['Branched-chain amino acid ABC transporter, substrate-binding protein LivJ (TC 3.A.1.4.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K03089 RNA polymerase sigma-32 factor\n",
      "SSO SSO:000006904 RNA polymerase sigma factor RpoH\n",
      "b3461 ['RNA polymerase sigma factor RpoH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K09811 cell division transport system permease protein\n",
      "SSO SSO:000035066 Cell-division-associated, ABC-transporter-like signaling protein FtsX\n",
      "b3462 ['Cell-division-associated, ABC-transporter-like signaling protein FtsX']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K09812 cell division transport system ATP-binding protein\n",
      "SSO SSO:000035065 Cell-division-associated, ABC-transporter-like signaling protein FtsE\n",
      "b3463 ['Cell-division-associated, ABC-transporter-like signaling protein FtsE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03110 fused signal recognition particle receptor\n",
      "SSO SSO:000007524 Signal recognition particle receptor FtsY\n",
      "b3464 ['Signal recognition particle receptor FtsY']\n",
      "\n",
      "KO KO:K08316 16S rRNA (guanine966-N2)-methyltransferase [EC:2.1.1.171]\n",
      "SSO SSO:000000063 16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)\n",
      "b3465 ['16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)']\n",
      "\n",
      "KO KO:K08993 putative membrane protein\n",
      "SSO SSO:000028347 Uncharacterized protein YhhL\n",
      "b3466 ['Uncharacterized protein YhhL']\n",
      "\n",
      "SSO SSO:000028348 Uncharacterized protein YhhM\n",
      "b3467 ['Uncharacterized protein YhhM']\n",
      "\n",
      "SSO SSO:000027264 Uncharacterized membrane protein YhhN\n",
      "b3468 ['Uncharacterized membrane protein YhhN']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015662 ATPase activity, coupled to transmembrane movement of ions, phosphorylative mechanism\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01534 Cd2+/Zn2+-exporting ATPase [EC:3.6.3.3 3.6.3.5]\n",
      "b3469 ['Lead, cadmium, zinc and mercury transporting ATPase (EC 3.6.3.3) (EC 3.6.3.5)', 'Copper-translocating P-type ATPase (EC 3.6.3.4)']\n",
      "\n",
      "KO KO:K04085 tRNA 2-thiouridine synthesizing protein A [EC:2.8.1.-]\n",
      "SSO SSO:000009353 tRNA 5-methylaminomethyl-2-thiouridine synthesis sulfur carrier protein TusA\n",
      "b3470 ['tRNA 5-methylaminomethyl-2-thiouridine synthesis sulfur carrier protein TusA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K09125 uncharacterized protein\n",
      "SSO SSO:000023539 Putative preQ0 transporter YhhQ\n",
      "b3471 ['Putative preQ0 transporter YhhQ']\n",
      "\n",
      "SSO SSO:000020814 putative lipoprotein\n",
      "b3472 ['DcrB protein precursor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000026586 UPF0226 protein YhhS\n",
      "b3473 ['UPF0226 protein YhhS']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11744 AI-2 transport protein TqsA\n",
      "SSO SSO:000026511 UPF0118 inner membrane protein YhhT\n",
      "b3474 ['UPF0118 inner membrane protein YhhT']\n",
      "\n",
      "KO KO:K06133 4'-phosphopantetheinyl transferase [EC:2.7.8.-]\n",
      "SSO SSO:000000345 4'-phosphopantetheinyl transferase (EC 2.7.8.-)\n",
      "b3475 [\"4'-phosphopantetheinyl transferase (EC 2.7.8.-)\"]\n",
      "\n",
      "GO GO:0042330 taxis\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "KO KO:K15584 nickel transport system substrate-binding protein\n",
      "b3476 ['Nickel ABC transporter, substrate-binding protein NikA (TC 3.A.1.5.3)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02033 peptide/nickel transport system permease protein\n",
      "KO KO:K15585 nickel transport system permease protein\n",
      "b3477 ['Nickel ABC transporter, permease protein NikB (TC 3.A.1.5.3)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02034 peptide/nickel transport system permease protein\n",
      "KO KO:K15586 nickel transport system permease protein\n",
      "b3478 ['Nickel ABC transporter, permease protein NikC (TC 3.A.1.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K15587 nickel transport system ATP-binding protein [EC:3.6.3.24]\n",
      "b3479 ['Nickel ABC transporter, ATP-binding protein NikD (TC 3.A.1.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K10824 nickel transport system ATP-binding protein [EC:3.6.3.24]\n",
      "b3480 ['Nickel ABC transporter, ATP-binding protein NikE (TC 3.A.1.5.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K07722 CopG family transcriptional regulator, nickel-responsive regulator\n",
      "SSO SSO:000020497 Nickel responsive regulator NikR\n",
      "b3481 ['Nickel responsive regulator NikR']\n",
      "\n",
      "SSO SSO:000022714 Protein RhsB\n",
      "b3482 ['Protein RhsB']\n",
      "\n",
      "SSO SSO:000028346 Uncharacterized protein YhhH\n",
      "b3483 ['Uncharacterized protein YhhH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4552 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000017503 H repeat-associated protein, YhhI family\n",
      "b3484 ['H repeat-associated protein, YhhI family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01992 ABC-2 type transport system permease protein\n",
      "SSO SSO:000018437 Inner membrane transport permease YhhJ\n",
      "b3485 ['Inner membrane transport permease YhhJ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K01990 ABC-2 type transport system ATP-binding protein\n",
      "KO KO:K01992 ABC-2 type transport system permease protein\n",
      "KO KO:K13926 ribosome-dependent ATPase\n",
      "SSO SSO:000024183 Ribosome-associated ATPase RbbA\n",
      "b3486 ['Ribosome-associated ATPase RbbA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01993 HlyD family secretion protein\n",
      "SSO SSO:000036297 HlyD-like membrane fusion protein YhiI\n",
      "b3487 ['HlyD-like membrane fusion protein YhiI']\n",
      "\n",
      "SSO SSO:000028349 Uncharacterized protein YhiJ\n",
      "b3488 ['Uncharacterized protein YhiJ']\n",
      "\n",
      "SSO SSO:000018117 Hypothetical protein YhiL\n",
      "b4660 ['Hypothetical protein YhiL']\n",
      "\n",
      "SSO SSO:000018402 Inner membrane protein YhiM\n",
      "b3491 ['Inner membrane protein YhiM']\n",
      "\n",
      "SSO SSO:000028350 Uncharacterized protein YhiN\n",
      "b3492 ['Uncharacterized protein YhiN']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16322 low-affinity inorganic phosphate transporter\n",
      "SSO SSO:000019057 Low-affinity inorganic phosphate transporter\n",
      "b3493 ['Low-affinity inorganic phosphate transporter']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0006950 response to stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06144 universal stress protein B\n",
      "SSO SSO:000008823 Universal stress protein B\n",
      "b3494 ['Universal stress protein B']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006950 response to stress\n",
      "KO KO:K06149 universal stress protein A\n",
      "SSO SSO:000008822 Universal stress protein A\n",
      "b3495 ['Universal stress protein A']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03305 proton-dependent oligopeptide transporter, POT family\n",
      "SSO SSO:000013069 Di-tripeptide/H+ symporter DtpB\n",
      "b3496 ['Di-tripeptide/H+ symporter DtpB']\n",
      "\n",
      "KO KO:K15984 16S rRNA (guanine1516-N2)-methyltransferase [EC:2.1.1.242]\n",
      "SSO SSO:000000060 16S rRNA (guanine(1516)-N(2))-methyltransferase (EC 2.1.1.242)\n",
      "b3497 ['16S rRNA (guanine(1516)-N(2))-methyltransferase (EC 2.1.1.242)']\n",
      "\n",
      "KO KO:K01414 oligopeptidase A [EC:3.4.24.70]\n",
      "SSO SSO:000005487 Oligopeptidase A (EC 3.4.24.70)\n",
      "b3498 ['Oligopeptidase A (EC 3.4.24.70)']\n",
      "\n",
      "KO KO:K07115 23S rRNA (adenine2030-N6)-methyltransferase [EC:2.1.1.266]\n",
      "SSO SSO:000000223 23S rRNA (adenine(2030)-N(6))-methyltransferase (EC 2.1.1.266)\n",
      "b3499 ['23S rRNA (adenine(2030)-N(6))-methyltransferase (EC 2.1.1.266)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00383 glutathione reductase (NADPH) [EC:1.8.1.7]\n",
      "SSO SSO:000003226 Glutathione reductase (EC 1.8.1.7)\n",
      "b3500 ['Glutathione reductase (EC 1.8.1.7)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4613 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K03892 ArsR family transcriptional regulator, arsenate/arsenite/antimonite-responsive transcriptional repressor\n",
      "SSO SSO:000001071 Arsenical resistance operon repressor\n",
      "b3501 ['Arsenical resistance operon repressor']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03893 arsenical pump membrane protein\n",
      "SSO SSO:000034698 Arsenite/antimonite:H+ antiporter ArsB (EC 7.3.2.7)\n",
      "b3502 ['Arsenite/antimonite:H+ antiporter ArsB']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K00537 arsenate reductase (glutaredoxin) [EC:1.20.4.1]\n",
      "SSO SSO:000001067 Arsenate reductase (EC 1.20.4.1)\n",
      "SSO SSO:000034693 Arsenate reductase (EC 1.20.4.1) glutaredoxin-coupled, glutaredoxin-like family\n",
      "b3503 ['Arsenate reductase (EC 1.20.4.1) glutaredoxin-coupled, glutaredoxin-like family']\n",
      "\n",
      "SSO SSO:000023702 Putative uncharacterized protein YhiS\n",
      "b3504 ['Putative uncharacterized protein YhiS']\n",
      "\n",
      "KO KO:K07481 transposase, IS5 family\n",
      "SSO SSO:000008346 Transposase InsH for insertion sequence element IS5\n",
      "b3505 ['Transposase InsH for insertion sequence element IS5']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K07285 outer membrane lipoprotein\n",
      "SSO SSO:000020875 Outer membrane lipoprotein\n",
      "SSO SSO:000025111 Starvation lipoprotein Slp\n",
      "b3506 ['Starvation lipoprotein Slp']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K21901 LuxR family transcriptional regulator, dicarboxylate transport regulator\n",
      "SSO SSO:000025809 Transcriptional regulator DctR, LuxR family\n",
      "b3507 ['Transcriptional regulator DctR, LuxR family']\n",
      "\n",
      "KO KO:K07507 putative Mg2+ transporter-C (MgtC) family protein\n",
      "SSO SSO:000023402 Putative magnesium transporter YhiD\n",
      "b3508 ['Putative magnesium transporter YhiD']\n",
      "\n",
      "KO KO:K19778 acid stress chaperone HdeB\n",
      "SSO SSO:000012121 Chaperone HdeB\n",
      "b3509 ['Chaperone HdeB']\n",
      "\n",
      "KO KO:K19777 acid stress chaperone HdeA\n",
      "SSO SSO:000012120 Chaperone HdeA\n",
      "b3510 ['Chaperone HdeA']\n",
      "\n",
      "KO KO:K21908 membrane protein HdeD\n",
      "SSO SSO:000019467 Membrane transporter HdeD, H-NS repressed\n",
      "b3511 ['Membrane transporter HdeD, H-NS repressed']\n",
      "\n",
      "KO KO:K21907 LuxR family transcriptional regulator, glutamate-dependent acid resistance regulator\n",
      "SSO SSO:000008181 Transcriptional activator GadE\n",
      "b3512 ['Transcriptional activator GadE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03585 membrane fusion protein, multidrug efflux system\n",
      "KO KO:K18898 membrane fusion protein, multidrug efflux system\n",
      "SSO SSO:000004983 Multidrug efflux system MdtEF-TolC, membrane fusion component MdtE\n",
      "b3513 ['Multidrug efflux system MdtEF-TolC, membrane fusion component MdtE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K18138 multidrug efflux pump\n",
      "KO KO:K18899 multidrug efflux pump\n",
      "SSO SSO:000004982 Multidrug efflux system MdtEF-TolC, inner-membrane proton/drug antiporter MdtF (RND type)\n",
      "b3514 ['Multidrug efflux system MdtEF-TolC, inner-membrane proton/drug antiporter MdtF (RND type)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K21906 AraC family transcriptional regulator, glutamate-dependent acid resistance regulator\n",
      "SSO SSO:000025812 Transcriptional regulator GadW, AraC family\n",
      "b3515 ['Transcriptional regulator GadW, AraC family']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K21905 AraC family transcriptional regulator, glutamate-dependent acid resistance regulator\n",
      "SSO SSO:000025813 Transcriptional regulator GadX, AraC family\n",
      "b3516 ['Transcriptional regulator GadX, AraC family']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006520 cellular amino acid metabolic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01580 glutamate decarboxylase [EC:4.1.1.15]\n",
      "SSO SSO:000003148 Glutamate decarboxylase (EC 4.1.1.15)\n",
      "b3517 ['Glutamate decarboxylase (EC 4.1.1.15)']\n",
      "\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K00428 cytochrome c peroxidase [EC:1.11.1.5]\n",
      "SSO SSO:000001935 Cytochrome c551 peroxidase (EC 1.11.1.5)\n",
      "b3518 ['Cytochrome c551 peroxidase (EC 1.11.1.5)']\n",
      "\n",
      "EC EC:3.2.1.28 Alpha,alpha-trehalase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01194 alpha,alpha-trehalase [EC:3.2.1.28]\n",
      "SSO SSO:000026168 Cytoplasmic trehalase (EC 3.2.1.28)\n",
      "b3519 ['Trehalase (EC 3.2.1.28)', 'Cytoplasmic trehalase (EC 3.2.1.28)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000041571 Uncharacterized transcriptional regulator YhjB, LuxR family\n",
      "b3520 ['Uncharacterized transcriptional regulator YhjB, LuxR family']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K18900 LysR family transcriptional regulator, regulator for bpeEF and oprC\n",
      "SSO SSO:000039699 Transcriptional regulator YhjC, LysR family\n",
      "b3521 ['Transcriptional regulator YhjC, LysR family']\n",
      "\n",
      "KO KO:K07058 membrane protein\n",
      "SSO SSO:000018403 Inner membrane protein YhjD\n",
      "b3522 ['Inner membrane protein YhjD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000040012 Uncharacterized MFS-type transporter YhjE\n",
      "b3523 ['Uncharacterized MFS-type transporter YhjE']\n",
      "\n",
      "KO KO:K07290 AsmA family protein\n",
      "SSO SSO:000028352 Uncharacterized protein YhjG\n",
      "b3524 ['Uncharacterized protein YhjG']\n",
      "\n",
      "KO KO:K21086 c-di-GMP phosphodiesterase [EC:3.1.4.52]\n",
      "b3525 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeH']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00874 2-dehydro-3-deoxygluconokinase [EC:2.7.1.45]\n",
      "SSO SSO:000000148 2-dehydro-3-deoxygluconokinase (EC 2.7.1.45)\n",
      "b3526 ['2-dehydro-3-deoxygluconokinase (EC 2.7.1.45)']\n",
      "\n",
      "SSO SSO:000022733 Protein YhjJ, putative peptidase\n",
      "b3527 ['Protein YhjJ, putative peptidase']\n",
      "\n",
      "GO GO:0005311 obsolete sodium:dicarboxylate/tricarboxylate symporter activity\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11103 aerobic C4-dicarboxylate transport protein\n",
      "SSO SSO:000020441 Na+/H+-dicarboxylate symporter\n",
      "b3528 ['Na+/H+-dicarboxylate symporter']\n",
      "\n",
      "KO KO:K21023 diguanylate cyclase [EC:2.7.7.65]\n",
      "b3529 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeK']\n",
      "\n",
      "KO KO:K20543 cellulose synthase operon protein C\n",
      "SSO SSO:000012095 Cellulose synthase operon protein C\n",
      "b3530 ['Cellulose synthase operon protein C']\n",
      "\n",
      "EC EC:3.2.1.132 Chitosanase.\n",
      "EC EC:3.2.1.156 Oligosaccharide reducing-end xylanase.\n",
      "EC EC:3.2.1.4 Cellulase.\n",
      "EC EC:3.2.1.73 Licheninase.\n",
      "EC EC:3.2.1.8 Endo-1,4-beta-xylanase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "GO GO:0005576 extracellular region\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K20542 endoglucanase [EC:3.2.1.4]\n",
      "SSO SSO:000019333 beta-1,4-glucanase (cellulase) (EC 3.2.1.4)\n",
      "b3531 ['beta-1,4-glucanase (cellulase) (EC 3.2.1.4)']\n",
      "\n",
      "KO KO:K20541 cellulose synthase operon protein B\n",
      "SSO SSO:000035231 Cyclic di-GMP-binding protein BcsB\n",
      "b3532 ['Cyclic di-GMP-binding protein BcsB']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "KO KO:K00694 cellulose synthase (UDP-forming) [EC:2.4.1.12]\n",
      "SSO SSO:000012093 Cellulose synthase catalytic subunit\n",
      "SSO SSO:000012094 Cellulose synthase catalytic subunit [UDP-forming]\n",
      "b3533 ['Cellulose synthase catalytic subunit [UDP-forming] (EC 2.4.1.12)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b3534 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000038792 Putative cytoplasmic protein YhjR\n",
      "b3535 ['Putative cytoplasmic protein YhjR']\n",
      "\n",
      "SSO SSO:000035073 Cellulose biosynthesis protein BcsE\n",
      "b3536 ['Cellulose biosynthesis protein BcsE']\n",
      "\n",
      "SSO SSO:000035074 Cellulose biosynthesis protein BcsF\n",
      "b3537 ['Cellulose biosynthesis protein BcsF']\n",
      "\n",
      "SSO SSO:000035075 Cellulose biosynthesis protein BcsG\n",
      "b3538 ['Cellulose biosynthesis protein BcsG']\n",
      "\n",
      "KO KO:K18862 small toxic polypeptide LdrA/B/C/D\n",
      "SSO SSO:000024879 Small inner membrane protein, YmgF family\n",
      "b4453 ['Small inner membrane protein, YmgF family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000036232 HAAAP family transport protein YhjV\n",
      "b3539 ['HAAAP family transport protein YhjV']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K02032 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K12372 dipeptide transport system ATP-binding protein\n",
      "b3540 ['Dipeptide ABC transporter, ATP-binding protein DppF (TC 3.A.1.5.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K02031 peptide/nickel transport system ATP-binding protein\n",
      "KO KO:K12371 dipeptide transport system ATP-binding protein\n",
      "b3541 ['Dipeptide ABC transporter, ATP-binding protein DppD (TC 3.A.1.5.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02034 peptide/nickel transport system permease protein\n",
      "KO KO:K12370 dipeptide transport system permease protein\n",
      "b3542 ['Dipeptide ABC transporter, permease protein DppC (TC 3.A.1.5.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02033 peptide/nickel transport system permease protein\n",
      "KO KO:K12369 dipeptide transport system permease protein\n",
      "b3543 ['Dipeptide ABC transporter, permease protein DppB (TC 3.A.1.5.2)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02035 peptide/nickel transport system substrate-binding protein\n",
      "KO KO:K12368 dipeptide transport system substrate-binding protein\n",
      "b3544 ['Dipeptide ABC transporter, substrate-binding protein DppA (TC 3.A.1.5.2)', 'Dipeptide chemoreceptor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K12975 KDO II ethanolaminephosphotransferase [EC:2.7.8.42]\n",
      "SSO SSO:000036462 Kdo2-lipid A phosphoethanolamine transferase EptB (EC 2.7.8.42)\n",
      "b3546 ['Kdo2-lipid A phosphoethanolamine transferase EptB (EC 2.7.8.42)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K08177 MFS transporter, OFA family, oxalate/formate antiporter\n",
      "SSO SSO:000023560 Putative resistance protein\n",
      "b3547 ['Putative resistance protein']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "SSO SSO:000010895 Autotransporter beta- domain protein\n",
      "SSO SSO:000041451 Uncharacterized protein YhjY\n",
      "b3548 ['Uncharacterized protein YhjY']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K01246 DNA-3-methyladenine glycosylase I [EC:3.2.2.20]\n",
      "SSO SSO:000002129 DNA-3-methyladenine glycosylase (EC 3.2.2.20)\n",
      "b3549 ['DNA-3-methyladenine glycosylase (EC 3.2.2.20)']\n",
      "\n",
      "KO KO:K03826 putative acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000040015 Uncharacterized N-acetyltransferase YiaC\n",
      "b3550 ['Uncharacterized N-acetyltransferase YiaC']\n",
      "\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "KO KO:K08351 biotin/methionine sulfoxide reductase [EC:1.-.-.-]\n",
      "SSO SSO:000011153 Biotin sulfoxide reductase\n",
      "b3551 ['Biotin sulfoxide reductase (EC 1.-.-.-)', 'Free methionine-(S)-sulfoxide reductase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03286 OmpA-OmpF porin, OOP family\n",
      "SSO SSO:000020894 Outer membrane porin, OmpA family\n",
      "b3552 ['Outer membrane porin, OmpA family']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00090 glyoxylate/hydroxypyruvate/2-ketogluconate reductase [EC:1.1.1.79 1.1.1.81 1.1.1.215]\n",
      "SSO SSO:000000177 2-ketoaldonate reductase, broad specificity (EC 1.1.1.-) (EC 1.1.1.215)\n",
      "b3553 ['2-ketoaldonate reductase, broad specificity (EC 1.1.1.215)']\n",
      "\n",
      "SSO SSO:000028355 Uncharacterized protein YiaF\n",
      "b3554 ['Uncharacterized protein YiaF']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K07726 putative transcriptional regulator\n",
      "SSO SSO:000001019 Antitoxin to RelE-like translational repressor toxin\n",
      "b3555 ['Antitoxin to RelE-like translational repressor toxin']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03704 cold shock protein (beta-ribbon, CspA family)\n",
      "b3556 ['Cold shock protein of CSP family => CspA (naming convention as in E.coli)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4647 ['hypothetical protein']\n",
      "\n",
      "KO KO:K18920 protein HokA\n",
      "SSO SSO:000039503 Toxic component of a type I toxin-antitoxin system, HokA\n",
      "b4455 ['Toxic component of a type I toxin-antitoxin system, HokA']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000019954 Mobile element protein\n",
      "b3557 ['Mobile element protein']\n",
      "\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000023665 Putative transposase InsK for insertion sequence element IS150\n",
      "b3558 ['Putative transposase InsK for insertion sequence element IS150']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01879 glycyl-tRNA synthetase beta chain [EC:6.1.1.14]\n",
      "SSO SSO:000003315 Glycyl-tRNA synthetase beta chain (EC 6.1.1.14)\n",
      "b3559 ['Glycyl-tRNA synthetase beta chain (EC 6.1.1.14)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01878 glycyl-tRNA synthetase alpha chain [EC:6.1.1.14]\n",
      "SSO SSO:000003312 Glycyl-tRNA synthetase alpha chain (EC 6.1.1.14)\n",
      "b3560 ['Glycyl-tRNA synthetase alpha chain (EC 6.1.1.14)']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000027216 Uncharacterized lipoprotein YsaB precursor\n",
      "b4553 ['Uncharacterized lipoprotein YsaB precursor']\n",
      "\n",
      "SSO SSO:000018406 Inner membrane protein YiaH\n",
      "b3561 ['Inner membrane protein YiaH']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000018404 Inner membrane protein YiaA\n",
      "b3562 ['Inner membrane protein YiaA']\n",
      "\n",
      "SSO SSO:000018405 Inner membrane protein YiaB\n",
      "b3563 ['Inner membrane protein YiaB']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00854 xylulokinase [EC:2.7.1.17]\n",
      "SSO SSO:000008974 Xylulose kinase (EC 2.7.1.17)\n",
      "SSO SSO:000033834 xylulokinase\n",
      "b3564 ['Xylulose kinase (EC 2.7.1.17)']\n",
      "\n",
      "EC EC:5.3.1.5 Xylose isomerase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01805 xylose isomerase [EC:5.3.1.5]\n",
      "SSO SSO:000029073 Xylose isomerase\n",
      "b3565 ['Xylose isomerase (EC 5.3.1.5)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10543 D-xylose transport system substrate-binding protein\n",
      "SSO SSO:000035395 D-xylose ABC transporter, substrate-binding protein XylF\n",
      "b3566 ['D-xylose ABC transporter, substrate-binding protein XylF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10545 D-xylose transport system ATP-binding protein [EC:3.6.3.17]\n",
      "SSO SSO:000035393 D-xylose ABC transporter, ATP-binding protein XylG\n",
      "b3567 ['D-xylose ABC transporter, ATP-binding protein XylG']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10544 D-xylose transport system permease protein\n",
      "SSO SSO:000035394 D-xylose ABC transporter, permease protein XylH\n",
      "b3568 ['D-xylose ABC transporter, permease protein XylH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "b3569 ['Xylose activator XylR (AraC family)']\n",
      "\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "EC EC:3.2.1.96 Mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase.\n",
      "KO KO:K03796 Bax protein\n",
      "SSO SSO:000010951 BAX protein\n",
      "b3570 ['BAX protein']\n",
      "\n",
      "EC EC:2.4.1.18 1,4-alpha-glucan branching enzyme.\n",
      "EC EC:2.4.1.19 Cyclomaltodextrin glucanotransferase.\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "EC EC:2.4.1.4 Amylosucrase.\n",
      "EC EC:2.4.1.7 Sucrose phosphorylase.\n",
      "EC EC:2.4.99.16 Starch synthase (maltosyl-transferring).\n",
      "EC EC:3.2.1.1 Alpha-amylase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.116 Glucan 1,4-alpha-maltotriohydrolase.\n",
      "EC EC:3.2.1.133 Glucan 1,4-alpha-maltohydrolase.\n",
      "EC EC:3.2.1.135 Neopullulanase.\n",
      "EC EC:3.2.1.141 4-alpha-D-((1->4)-alpha-D-glucano)trehalose trehalohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.33 Amylo-alpha-1,6-glucosidase.\n",
      "EC EC:3.2.1.41 Pullulanase.\n",
      "EC EC:3.2.1.54 Cyclomaltodextrinase.\n",
      "EC EC:3.2.1.60 Glucan 1,4-alpha-maltotetraohydrolase.\n",
      "EC EC:3.2.1.68 Isoamylase.\n",
      "EC EC:3.2.1.70 Glucan 1,6-alpha-glucosidase.\n",
      "EC EC:3.2.1.93 Alpha,alpha-phosphotrehalase.\n",
      "EC EC:3.2.1.98 Glucan 1,4-alpha-maltohexaosidase.\n",
      "EC EC:5.4.99.11 Isomaltulose synthase.\n",
      "EC EC:5.4.99.15 (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase.\n",
      "EC EC:5.4.99.16 Maltose alpha-D-glucosyltransferase.\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "GO GO:0006508 proteolysis\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01176 alpha-amylase [EC:3.2.1.1]\n",
      "SSO SSO:000000917 Alpha-amylase (EC 3.2.1.1)\n",
      "SSO SSO:000005831 Periplasmic alpha-amylase (EC 3.2.1.1)\n",
      "b3571 ['Periplasmic alpha-amylase (EC 3.2.1.1)']\n",
      "\n",
      "GO GO:0006523 alanine biosynthetic process\n",
      "KO KO:K00835 valine--pyruvate aminotransferase [EC:2.6.1.66]\n",
      "SSO SSO:000008904 Valine--pyruvate aminotransferase (EC 2.6.1.66)\n",
      "b3572 ['Valine--pyruvate aminotransferase (EC 2.6.1.66)']\n",
      "\n",
      "KO KO:K05796 electron transport protein HydN\n",
      "SSO SSO:000023279 Putative electron transport protein YsaA\n",
      "b3573 ['Putative electron transport protein YsaA']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K21602 IclR family transcriptional regulator, carbohydrate utilization repressor\n",
      "SSO SSO:000039722 Transcriptional regulator yiaJ, IclR family\n",
      "b3574 ['Transcriptional regulator yiaJ, IclR family']\n",
      "\n",
      "KO KO:K08092 3-dehydro-L-gulonate 2-dehydrogenase [EC:1.1.1.130]\n",
      "SSO SSO:000000273 3-dehydro-L-gulonate 2-dehydrogenase (EC 1.1.1.130)\n",
      "b3575 ['3-dehydro-L-gulonate 2-dehydrogenase (EC 1.1.1.130)']\n",
      "\n",
      "KO KO:K19334 biofilm protein TabA\n",
      "SSO SSO:000041452 Uncharacterized protein YiaL\n",
      "b3576 ['Uncharacterized protein YiaL']\n",
      "\n",
      "KO KO:K21394 TRAP-type transport system small permease protein\n",
      "SSO SSO:000009503 2,3-diketo-L-gulonate TRAP transporter small permease protein yiaM\n",
      "b3577 ['2,3-diketo-L-gulonate TRAP transporter small permease protein yiaM']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11690 C4-dicarboxylate transporter, DctM subunit\n",
      "KO KO:K21393 TRAP-type transport system large permease protein\n",
      "SSO SSO:000009502 2,3-diketo-L-gulonate TRAP transporter large permease protein yiaN\n",
      "b3578 ['2,3-diketo-L-gulonate TRAP transporter large permease protein yiaN']\n",
      "\n",
      "KO KO:K21395 TRAP-type transport system periplasmic protein\n",
      "SSO SSO:000009504 2,3-diketo-L-gulonate-binding periplasmic protein yiaO precursor\n",
      "b3579 ['2,3-diketo-L-gulonate-binding periplasmic protein yiaO precursor']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00880 L-xylulokinase [EC:2.7.1.53]\n",
      "SSO SSO:000004188 L-xylulose/3-keto-L-gulonate kinase (EC 2.7.1.-)\n",
      "SSO SSO:000036562 L-xylulose kinase (EC 2.7.1.53)\n",
      "b3580 ['L-xylulose/3-keto-L-gulonate kinase (EC 2.7.1.-)']\n",
      "\n",
      "KO KO:K03078 3-dehydro-L-gulonate-6-phosphate decarboxylase [EC:4.1.1.85]\n",
      "SSO SSO:000000310 3-keto-L-gulonate 6-phosphate decarboxylase (EC 4.1.1.85)\n",
      "SSO SSO:000009626 3-keto-L-gulonate 6-phosphate decarboxylase homolog\n",
      "b3581 ['3-keto-L-gulonate 6-phosphate decarboxylase homolog']\n",
      "\n",
      "KO KO:K03079 L-ribulose-5-phosphate 3-epimerase [EC:5.1.3.22]\n",
      "SSO SSO:000018794 L-xylulose 5-phosphate 3-epimerase (EC 5.1.3.-) homolog\n",
      "SSO SSO:000023067 Putative L-xylulose-5-phosphate 3-epimerase\n",
      "b3582 ['L-xylulose 5-phosphate 3-epimerase (EC 5.1.3.-) homolog']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K03077 L-ribulose-5-phosphate 4-epimerase [EC:5.1.3.4]\n",
      "SSO SSO:000004170 L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)\n",
      "SSO SSO:000004171 L-ribulose-5-phosphate 4-epimerase SgbE (EC 5.1.3.4)\n",
      "b3583 ['L-ribulose-5-phosphate 4-epimerase SgbE (EC 5.1.3.4)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4648 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4649 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07274 MipA family protein\n",
      "SSO SSO:000023468 Putative outer membrane protein\n",
      "SSO SSO:000037382 Outer membrane protein YiaT\n",
      "b3584 ['Outer membrane protein YiaT']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000039700 Transcriptional regulator YiaU, LysR family\n",
      "b3585 ['Transcriptional regulator YiaU, LysR family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018407 Inner membrane protein YiaV\n",
      "b3586 ['Inner membrane protein YiaV']\n",
      "\n",
      "SSO SSO:000017027 GTPase\n",
      "b3587 ['GTPase (EC 3.6.1.-)']\n",
      "\n",
      "GO GO:0009438 methylglyoxal metabolic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00138 aldehyde dehydrogenase [EC:1.2.1.-]\n",
      "SSO SSO:000000867 Aldehyde dehydrogenase B (EC 1.2.1.22)\n",
      "b3588 ['Aldehyde dehydrogenase B (EC 1.2.1.22)']\n",
      "\n",
      "KO KO:K13954 alcohol dehydrogenase [EC:1.1.1.1]\n",
      "SSO SSO:000000860 Alcohol dehydrogenase (EC 1.1.1.1)\n",
      "b3589 ['Alcohol dehydrogenase (EC 1.1.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K03833 selenocysteine-specific elongation factor\n",
      "SSO SSO:000007384 Selenocysteine-specific translation elongation factor\n",
      "b3590 ['Selenocysteine-specific translation elongation factor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008615 pyridoxine biosynthetic process\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K01042 L-seryl-tRNA(Ser) seleniumtransferase [EC:2.9.1.1]\n",
      "SSO SSO:000004177 L-seryl-tRNA(Sec) selenium transferase (EC 2.9.1.1)\n",
      "b3591 ['L-seryl-tRNA(Sec) selenium transferase (EC 2.9.1.1)']\n",
      "\n",
      "KO KO:K00799 glutathione S-transferase [EC:2.5.1.18]\n",
      "SSO SSO:000008695 Uncharacterized GST-like protein YibF\n",
      "SSO SSO:000017311 Glutathione-S-transferase homolog\n",
      "b3592 ['Uncharacterized GST-like protein YibF']\n",
      "\n",
      "SSO SSO:000022713 Protein RhsA\n",
      "b3593 ['Protein RhsA']\n",
      "\n",
      "SSO SSO:000022734 Protein YibA\n",
      "b3594 ['Protein YibA']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b3595 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000028356 Uncharacterized protein YibG\n",
      "b3596 ['Uncharacterized protein YibG']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4650 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4651 ['hypothetical protein']\n",
      "\n",
      "b4615 ['FIG00641942: hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4652 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018408 Inner membrane protein YibH\n",
      "b3597 ['Inner membrane protein YibH']\n",
      "\n",
      "SSO SSO:000017027 GTPase\n",
      "b3598 ['GTPase (EC 3.6.1.-)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02800 PTS system, mannitol-specific IIC component\n",
      "b3599 ['PTS system, mannitol-specific IIC component', 'PTS system, mannitol-specific IIB component (EC 2.7.1.197)', 'PTS system, mannitol-specific IIA component (EC 2.7.1.197)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00009 mannitol-1-phosphate 5-dehydrogenase [EC:1.1.1.17]\n",
      "SSO SSO:000019339 Mannitol-1-phosphate 5-dehydrogenase (EC 1.1.1.17)\n",
      "b3600 ['Mannitol-1-phosphate 5-dehydrogenase (EC 1.1.1.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02562 mannitol operon repressor\n",
      "SSO SSO:000019338 Mannitol operon repressor\n",
      "b3601 ['Mannitol operon repressor']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000028358 Uncharacterized protein YibT\n",
      "b4554 ['Uncharacterized protein YibT']\n",
      "\n",
      "KO KO:K14762 ribosome-associated protein\n",
      "SSO SSO:000028357 Uncharacterized protein YibL\n",
      "b3602 ['Uncharacterized protein YibL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00427 L-lactate permease\n",
      "KO KO:K03303 lactate permease\n",
      "SSO SSO:000018768 L-lactate permease\n",
      "b3603 ['L-lactate permease']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "KO KO:K14348 GntR family transcriptional regulator, L-lactate dehydrogenase operon regulator\n",
      "b3604 ['Lactate-responsive regulator LldR in Enterobacteria, GntR family']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00101 L-lactate dehydrogenase (cytochrome) [EC:1.1.2.3]\n",
      "SSO SSO:000004155 L-lactate dehydrogenase (EC 1.1.1.27)\n",
      "b3605 ['L-lactate dehydrogenase']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03216 tRNA (cytidine/uridine-2'-O-)-methyltransferase [EC:2.1.1.207]\n",
      "SSO SSO:000009334 tRNA (cytidine(34)-2'-O)-methyltransferase (EC 2.1.1.207)\n",
      "b3606 [\"tRNA (cytidine(34)-2'-O)-methyltransferase (EC 2.1.1.207)\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0019344 cysteine biosynthetic process\n",
      "KO KO:K00640 serine O-acetyltransferase [EC:2.3.1.30]\n",
      "SSO SSO:000007423 Serine acetyltransferase (EC 2.3.1.30)\n",
      "b3607 ['Serine acetyltransferase (EC 2.3.1.30)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K00057 glycerol-3-phosphate dehydrogenase (NAD(P)+) [EC:1.1.1.94]\n",
      "SSO SSO:000003258 Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)\n",
      "SSO SSO:000036153 Glycerol-3-phosphate dehydrogenase [NAD(+)] (EC 1.1.1.8)\n",
      "b3608 ['Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03071 preprotein translocase subunit SecB\n",
      "SSO SSO:000006534 Protein-export protein SecB (maintains pre-export unfolded state)\n",
      "b3609 ['Protein-export protein SecB (maintains pre-export unfolded state)']\n",
      "\n",
      "KO KO:K03676 glutaredoxin 3\n",
      "SSO SSO:000003203 Glutaredoxin 3 (Grx3)\n",
      "SSO SSO:000009129 glutaredoxin 3\n",
      "b3610 ['Glutaredoxin 3 (Grx3)']\n",
      "\n",
      "SSO SSO:000007059 Rhodanese-related sulfurtransferase YibN\n",
      "SSO SSO:000030672 putative Rhodanese-related sulfurtransferase\n",
      "b3611 ['Rhodanese-related sulfurtransferase YibN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K15633 2,3-bisphosphoglycerate-independent phosphoglycerate mutase [EC:5.4.2.12]\n",
      "SSO SSO:000000075 2,3-bisphosphoglycerate-independent phosphoglycerate mutase (EC 5.4.2.12)\n",
      "b3612 ['2,3-bisphosphoglycerate-independent phosphoglycerate mutase (EC 5.4.2.12)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K22719 murein hydrolase activator\n",
      "SSO SSO:000005036 Murein hydrolase activator EnvC\n",
      "b3613 ['Murein hydrolase activator EnvC']\n",
      "\n",
      "KO KO:K09798 uncharacterized protein\n",
      "SSO SSO:000006717 Putative periplasmic protein YibQ, distant homology with nucleoside diphosphatase and polysaccharide deacetylase\n",
      "SSO SSO:000023534 Putative polysaccharide deacetylase\n",
      "b3614 ['Putative periplasmic protein YibQ, distant homology with nucleoside diphosphatase and polysaccharide deacetylase']\n",
      "\n",
      "EC EC:2.4.1.117 Dolichyl-phosphate beta-glucosyltransferase.\n",
      "EC EC:2.4.1.12 Cellulose synthase (UDP-forming).\n",
      "EC EC:2.4.1.16 Chitin synthase.\n",
      "EC EC:2.4.1.199 Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase.\n",
      "EC EC:2.4.1.212 Hyaluronan synthase.\n",
      "EC EC:2.4.1.287 Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,4/1,5-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.288 Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase.\n",
      "EC EC:2.4.1.289 N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase.\n",
      "EC EC:2.4.1.34 1,3-beta-glucan synthase.\n",
      "EC EC:2.4.1.83 Dolichyl-phosphate beta-D-mannosyltransferase.\n",
      "KO KO:K19354 heptose III glucuronosyltransferase [EC:2.4.1.-]\n",
      "SSO SSO:000039900 UDP-glucuronate:LPS(HepIII) glycosyltransferase WaaH\n",
      "b3615 ['UDP-glucuronate:LPS(HepIII) glycosyltransferase WaaH']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00060 threonine 3-dehydrogenase [EC:1.1.1.103]\n",
      "SSO SSO:000018792 L-threonine 3-dehydrogenase\n",
      "b3616 ['L-threonine 3-dehydrogenase (EC 1.1.1.103)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K00639 glycine C-acetyltransferase [EC:2.3.1.29]\n",
      "SSO SSO:000000126 2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)\n",
      "SSO SSO:000044159 glycine C-acetyltransferase (EC 2.3.1.29)\n",
      "b3617 ['2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)']\n",
      "\n",
      "GO GO:0009103 lipopolysaccharide biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "SSO SSO:000004476 Lipopolysaccharide biosynthesis associated protein HtrL\n",
      "b3618 ['Lipopolysaccharide biosynthesis associated protein HtrL']\n",
      "\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K03274 ADP-L-glycero-D-manno-heptose 6-epimerase [EC:5.1.3.20]\n",
      "SSO SSO:000000526 ADP-L-glycero-D-manno-heptose-6-epimerase (EC 5.1.3.20)\n",
      "b3619 ['ADP-L-glycero-D-manno-heptose-6-epimerase (EC 5.1.3.20)']\n",
      "\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02843 heptosyltransferase II [EC:2.4.-.-]\n",
      "SSO SSO:000000531 ADP-heptose--lipooligosaccharide heptosyltransferase II (EC 2.4.1.-)\n",
      "SSO SSO:000009948 ADP-heptose:LPS heptosyltransferase II\n",
      "b3620 ['ADP-heptose--lipooligosaccharide heptosyltransferase II']\n",
      "\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02841 heptosyltransferase I [EC:2.4.-.-]\n",
      "SSO SSO:000004482 Lipopolysaccharide core heptosyltransferase I\n",
      "b3621 ['Lipopolysaccharide core heptosyltransferase I']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02847 O-antigen ligase [EC:2.4.1.-]\n",
      "SSO SSO:000005467 O-antigen ligase\n",
      "b3622 ['O-antigen ligase']\n",
      "\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K03277 heptosyltransferase IV [EC:2.4.-.-]\n",
      "SSO SSO:000004471 Lipopolysaccharide 1,2-N-acetylglucosaminetransferase (EC 2.4.1.56)\n",
      "b3623 ['Lipopolysaccharide 1,2-N-acetylglucosaminetransferase (EC 2.4.1.56)']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K12981 KDO transferase III [EC:2.4.99.-]\n",
      "SSO SSO:000004479 Lipopolysaccharide core biosynthesis protein RfaZ\n",
      "b3624 ['Lipopolysaccharide core biosynthesis protein RfaZ']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02850 heptose II phosphotransferase [EC:2.7.1.-]\n",
      "SSO SSO:000004481 Lipopolysaccharide core heptose(II) kinase RfaY\n",
      "SSO SSO:000030948 lipopolysaccharide core biosynthesis protein\n",
      "b3625 ['Lipopolysaccharide core heptose(II) kinase RfaY']\n",
      "\n",
      "EC EC:2.4.1.123 Inositol 3-alpha-galactosyltransferase.\n",
      "EC EC:2.4.1.186 Glycogenin glucosyltransferase.\n",
      "EC EC:2.4.1.43 Polygalacturonate 4-alpha-galacturonosyltransferase.\n",
      "EC EC:2.4.1.44 Lipopolysaccharide 3-alpha-galactosyltransferase.\n",
      "EC EC:2.4.1.58 Lipopolysaccharide glucosyltransferase I.\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K03276 UDP-glucose/galactose:(glucosyl)LPS alpha-1,2-glucosyl/galactosyltransferase [EC:2.4.1.-]\n",
      "SSO SSO:000008648 UDP-glucose:(glucosyl)lipopolysaccharide alpha-1,2-glucosyltransferase (EC 2.4.1.58)\n",
      "b3626 ['UDP-glucose:(glucosyl)lipopolysaccharide alpha-1,2-glucosyltransferase (EC 2.4.1.58)']\n",
      "\n",
      "EC EC:2.4.1.123 Inositol 3-alpha-galactosyltransferase.\n",
      "EC EC:2.4.1.186 Glycogenin glucosyltransferase.\n",
      "EC EC:2.4.1.43 Polygalacturonate 4-alpha-galacturonosyltransferase.\n",
      "EC EC:2.4.1.44 Lipopolysaccharide 3-alpha-galactosyltransferase.\n",
      "EC EC:2.4.1.58 Lipopolysaccharide glucosyltransferase I.\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K03275 UDP-glucose:(glucosyl)LPS alpha-1,3-glucosyltransferase [EC:2.4.1.-]\n",
      "SSO SSO:000004472 Lipopolysaccharide 1,3-galactosyltransferase (EC 2.4.1.44)\n",
      "b3627 ['Lipopolysaccharide 1,3-galactosyltransferase (EC 2.4.1.44)']\n",
      "\n",
      "EC EC:2.4.1.13 Sucrose synthase.\n",
      "EC EC:2.4.1.131 GDP-Man:Man(3)GlcNAc(2)-PP-dolichol alpha-1,2-mannosyltransferase.\n",
      "EC EC:2.4.1.132 GDP-Man:Man(1)GlcNAc(2)-PP-dolichol alpha-1,3-mannosyltransferase.\n",
      "EC EC:2.4.1.14 Sucrose-phosphate synthase.\n",
      "EC EC:2.4.1.141 N-acetylglucosaminyldiphosphodolichol N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.157 Transferred entry: 2.4.1.336 and 2.4.1.337.\n",
      "EC EC:2.4.1.208 Diglucosyl diacylglycerol synthase (1,2-linking).\n",
      "EC EC:2.4.1.231 Alpha,alpha-trehalose phosphorylase (configuration-retaining).\n",
      "EC EC:2.4.1.245 Alpha,alpha-trehalose synthase.\n",
      "EC EC:2.4.1.257 GDP-Man:Man(2)GlcNAc(2)-PP-dolichol alpha-1,6-mannosyltransferase.\n",
      "EC EC:2.4.1.283 2-deoxystreptamine N-acetyl-D-glucosaminyltransferase.\n",
      "EC EC:2.4.1.285 UDP-GlcNAc:ribostamycin N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.290 N,N'-diacetylbacillosaminyl-diphospho-undecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.306 UDP-GalNAc:alpha-D-GalNAc-diphosphoundecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.52 Poly(glycerol-phosphate) alpha-glucosyltransferase.\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.57 Transferred entry: 2.4.1.345.\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02840 UDP-D-galactose:(glucosyl)LPS alpha-1,6-D-galactosyltransferase [EC:2.4.1.-]\n",
      "SSO SSO:000004473 Lipopolysaccharide 1,6-galactosyltransferase (EC 2.4.1.-)\n",
      "b3628 ['Lipopolysaccharide 1,6-galactosyltransferase (EC 2.4.1.-)']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K12986 1,5-rhamnosyltransferase [EC:2.4.1.-]\n",
      "SSO SSO:000004478 Lipopolysaccharide core biosynthesis protein RfaS\n",
      "b3629 ['Lipopolysaccharide core biosynthesis protein RfaS']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02848 heptose I phosphotransferase [EC:2.7.1.-]\n",
      "SSO SSO:000004480 Lipopolysaccharide core heptose(I) kinase RfaP\n",
      "b3630 ['Lipopolysaccharide core heptose(I) kinase RfaP']\n",
      "\n",
      "EC EC:2.4.1.13 Sucrose synthase.\n",
      "EC EC:2.4.1.131 GDP-Man:Man(3)GlcNAc(2)-PP-dolichol alpha-1,2-mannosyltransferase.\n",
      "EC EC:2.4.1.132 GDP-Man:Man(1)GlcNAc(2)-PP-dolichol alpha-1,3-mannosyltransferase.\n",
      "EC EC:2.4.1.14 Sucrose-phosphate synthase.\n",
      "EC EC:2.4.1.141 N-acetylglucosaminyldiphosphodolichol N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.157 Transferred entry: 2.4.1.336 and 2.4.1.337.\n",
      "EC EC:2.4.1.208 Diglucosyl diacylglycerol synthase (1,2-linking).\n",
      "EC EC:2.4.1.231 Alpha,alpha-trehalose phosphorylase (configuration-retaining).\n",
      "EC EC:2.4.1.245 Alpha,alpha-trehalose synthase.\n",
      "EC EC:2.4.1.257 GDP-Man:Man(2)GlcNAc(2)-PP-dolichol alpha-1,6-mannosyltransferase.\n",
      "EC EC:2.4.1.283 2-deoxystreptamine N-acetyl-D-glucosaminyltransferase.\n",
      "EC EC:2.4.1.285 UDP-GlcNAc:ribostamycin N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.290 N,N'-diacetylbacillosaminyl-diphospho-undecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.306 UDP-GalNAc:alpha-D-GalNAc-diphosphoundecaprenol alpha-1,3-N-acetylgalactosaminyltransferase.\n",
      "EC EC:2.4.1.52 Poly(glycerol-phosphate) alpha-glucosyltransferase.\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "EC EC:2.4.1.57 Transferred entry: 2.4.1.345.\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0009432 SOS response\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02844 UDP-glucose:(heptosyl)LPS alpha-1,3-glucosyltransferase [EC:2.4.1.-]\n",
      "SSO SSO:000008649 UDP-glucose:(heptosyl) LPS alpha1,3-glucosyltransferase WaaG (EC 2.4.1.-)\n",
      "b3631 ['UDP-glucose:(heptosyl) LPS alpha1,3-glucosyltransferase WaaG']\n",
      "\n",
      "EC EC:2.4.1.56 Lipopolysaccharide N-acetylglucosaminyltransferase.\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02849 heptosyltransferase III [EC:2.4.-.-]\n",
      "SSO SSO:000004483 Lipopolysaccharide core heptosyltransferase III\n",
      "SSO SSO:000030948 lipopolysaccharide core biosynthesis protein\n",
      "b3632 ['Lipopolysaccharide core heptosyltransferase III']\n",
      "\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0009245 lipid A biosynthetic process\n",
      "GO GO:0009248 K antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02527 3-deoxy-D-manno-octulosonic-acid transferase [EC:2.4.99.12 2.4.99.13 2.4.99.14 2.4.99.15]\n",
      "SSO SSO:000000280 3-deoxy-D-manno-octulosonic acid transferase (EC 2.4.99.13) (EC 2.4.99.12)\n",
      "b3633 ['3-deoxy-D-manno-octulosonic acid transferase (EC 2.4.99.12)(EC 2.4.99.13)']\n",
      "\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00954 pantetheine-phosphate adenylyltransferase [EC:2.7.7.3]\n",
      "SSO SSO:000006021 Phosphopantetheine adenylyltransferase (EC 2.7.7.3)\n",
      "b3634 ['Phosphopantetheine adenylyltransferase (EC 2.7.7.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K10563 formamidopyrimidine-DNA glycosylase [EC:3.2.2.23 4.2.99.18]\n",
      "SSO SSO:000002875 Formamidopyrimidine-DNA glycosylase (EC 3.2.2.23)\n",
      "b3635 ['Formamidopyrimidine-DNA glycosylase (EC 3.2.2.23)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02913 large subunit ribosomal protein L33\n",
      "b3636 ['LSU ribosomal protein L33p', 'LSU ribosomal protein L33p, zinc-independent']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02902 large subunit ribosomal protein L28\n",
      "b3637 ['LSU ribosomal protein L28p', 'LSU ribosomal protein L28p, zinc-independent']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03630 DNA repair protein RadC\n",
      "SSO SSO:000026714 UPF0758 protein YicR\n",
      "SSO SSO:000039941 UPF0758 family protein\n",
      "b3638 ['UPF0758 protein YicR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0015937 coenzyme A biosynthetic process\n",
      "KO KO:K13038 phosphopantothenoylcysteine decarboxylase / phosphopantothenate---cysteine ligase [EC:4.1.1.36 6.3.2.5]\n",
      "b3639 ['Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)', 'Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01520 dUTP pyrophosphatase [EC:3.6.1.23]\n",
      "SSO SSO:000002256 Deoxyuridine 5'-triphosphate nucleotidohydrolase (EC 3.6.1.23)\n",
      "b3640 [\"Deoxyuridine 5'-triphosphate nucleotidohydrolase (EC 3.6.1.23)\"]\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05501 TetR/AcrR family transcriptional regulator\n",
      "SSO SSO:000025851 Transcriptional regulator SlmA, TetR family\n",
      "b3641 ['Transcriptional regulator SlmA, TetR family']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "KO KO:K00762 orotate phosphoribosyltransferase [EC:2.4.2.10]\n",
      "SSO SSO:000005521 Orotate phosphoribosyltransferase (EC 2.4.2.10)\n",
      "b3642 ['Orotate phosphoribosyltransferase (EC 2.4.2.10)']\n",
      "\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K00989 ribonuclease PH [EC:2.7.7.56]\n",
      "SSO SSO:000007094 Ribonuclease PH (EC 2.7.7.56)\n",
      "b3643 ['Ribonuclease PH (EC 2.7.7.56)']\n",
      "\n",
      "SSO SSO:000039939 UPF0701 protein YicC\n",
      "b3644 ['UPF0701 protein YicC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K14623 DNA-damage-inducible protein D\n",
      "SSO SSO:000012752 DNA damage-inducible protein\n",
      "SSO SSO:000012934 DNA-damage-inducible protein D\n",
      "b3645 ['DNA-damage-inducible protein D']\n",
      "\n",
      "SSO SSO:000026536 UPF0126 inner membrane protein YicG\n",
      "b3646 ['UPF0126 inner membrane protein YicG']\n",
      "\n",
      "GO GO:0006259 DNA metabolic process\n",
      "KO KO:K01972 DNA ligase (NAD+) [EC:6.5.1.2]\n",
      "SSO SSO:000002073 DNA ligase (NAD(+)) (EC 6.5.1.2)\n",
      "b3647 ['DNA ligase (NAD(+)) (EC 6.5.1.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009152 purine ribonucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00942 guanylate kinase [EC:2.7.4.8]\n",
      "SSO SSO:000003328 Guanylate kinase (EC 2.7.4.8)\n",
      "b3648 ['Guanylate kinase (EC 2.7.4.8)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03060 DNA-directed RNA polymerase subunit omega [EC:2.7.7.6]\n",
      "SSO SSO:000002185 DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)\n",
      "b3649 ['DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008233 peptidase activity\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K01139 GTP diphosphokinase / guanosine-3',5'-bis(diphosphate) 3'-diphosphatase [EC:2.7.6.5 3.1.7.2]\n",
      "b3650 [\"Guanosine-3',5'-bis(diphosphate) 3'-pyrophosphohydrolase (EC 3.1.7.2)\", 'GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00556 tRNA (guanosine-2'-O-)-methyltransferase [EC:2.1.1.34]\n",
      "SSO SSO:000009344 tRNA (guanosine(18)-2'-O)-methyltransferase (EC 2.1.1.34)\n",
      "b3651 [\"tRNA (guanosine(18)-2'-O)-methyltransferase (EC 2.1.1.34)\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K03655 ATP-dependent DNA helicase RecG [EC:3.6.4.12]\n",
      "SSO SSO:000000569 ATP-dependent DNA helicase RecG (EC 3.6.1.-)\n",
      "SSO SSO:000010027 ATP-dependent DNA helicase\n",
      "b3652 ['ATP-dependent DNA helicase RecG (EC 3.6.4.12)']\n",
      "\n",
      "GO GO:0006532 aspartate biosynthetic process\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03312 glutamate:Na+ symporter, ESS family\n",
      "SSO SSO:000007612 Sodium/glutamate symporter\n",
      "b3653 ['Sodium/glutamate symporter']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K16345 xanthine permease XanP\n",
      "SSO SSO:000008960 Xanthine permease\n",
      "b3654 ['Xanthine permease', 'Xanthine:proton symporter XanP']\n",
      "\n",
      "SSO SSO:000028359 Uncharacterized protein YicH\n",
      "b3655 ['Uncharacterized protein YicH']\n",
      "\n",
      "EC EC:2.4.1.161 Oligosaccharide 4-alpha-D-glucosyltransferase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.177 Alpha-D-xyloside xylohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.22 Alpha-galactosidase.\n",
      "EC EC:3.2.1.24 Alpha-mannosidase.\n",
      "EC EC:3.2.1.48 Sucrose alpha-glucosidase.\n",
      "EC EC:3.2.1.84 Glucan 1,3-alpha-glucosidase.\n",
      "EC EC:4.2.2.13 Exo-(1->4)-alpha-D-glucan lyase.\n",
      "KO KO:K01811 alpha-D-xyloside xylohydrolase [EC:3.2.1.177]\n",
      "SSO SSO:000023156 Putative alpha-glucosidase\n",
      "SSO SSO:000029337 alpha-xylosidase (EC 3.2.1.177)\n",
      "b3656 ['alpha-xylosidase (EC 3.2.1.177)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03292 glycoside/pentoside/hexuronide:cation symporter, GPH family\n",
      "SSO SSO:000022666 Proposed xyloside/Na+ symporter YicJ\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b3657 ['Proposed xyloside/Na+ symporter YicJ']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4653 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03291 MFS transporter, SET family, sugar efflux transporter\n",
      "SSO SSO:000025236 Sugar efflux transporter C\n",
      "b3659 ['Sugar efflux transporter C']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000027123 Uncharacterized inner membrane transporter YicL\n",
      "b3660 ['Uncharacterized inner membrane transporter YicL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02073 D-methionine transport system substrate-binding protein\n",
      "SSO SSO:000018987 Lipoprotein 28 (NlpA)\n",
      "b3661 ['Lipoprotein 28 (NlpA)']\n",
      "\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "SSO SSO:000036233 HAD phosphatase YigL\n",
      "b4555 ['HAD phosphatase YigL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03445 MFS transporter, DHA1 family, purine ribonucleoside efflux pump\n",
      "SSO SSO:000022991 Purine ribonucleoside efflux pump NepI\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b3662 ['Purine ribonucleoside efflux pump NepI']\n",
      "\n",
      "SSO SSO:000028360 Uncharacterized protein YicN\n",
      "b3663 ['Uncharacterized protein YicN']\n",
      "\n",
      "KO KO:K06901 putative MFS transporter, AGZA family, xanthine/uracil permease\n",
      "SSO SSO:000010302 Adenine permease AdeQ\n",
      "b3664 ['Adenine permease AdeQ']\n",
      "\n",
      "GO GO:0008152 metabolic process\n",
      "KO KO:K01486 adenine deaminase [EC:3.5.4.2]\n",
      "SSO SSO:000010300 Adenine deaminase (EC 3.5.4.2)\n",
      "b3665 ['Adenine deaminase (EC 3.5.4.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07784 MFS transporter, OPA family, hexose phosphate transport protein UhpT\n",
      "SSO SSO:000017773 Hexose phosphate transport protein UhpT\n",
      "b3666 ['Hexose phosphate transport protein UhpT']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07783 MFS transporter, OPA family, sugar phosphate sensor protein UhpC\n",
      "SSO SSO:000037555 Periplasmic space glucose-6-phosphate sensor protein UhpC\n",
      "b3667 ['Periplasmic space glucose-6-phosphate sensor protein UhpC']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07675 two-component system, NarL family, sensor histidine kinase UhpB [EC:2.7.13.3]\n",
      "SSO SSO:000036272 Hexose-phosphate uptake signal transduction histidine-protein kinase/phosphatase UhpB\n",
      "b3668 ['Hexose-phosphate uptake signal transduction histidine-protein kinase/phosphatase UhpB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07686 two-component system, NarL family, uhpT operon response regulator UhpA\n",
      "SSO SSO:000036273 Hexose-phosphate uptake two-component transcriptional response regulator UhpA\n",
      "b3669 ['Hexose-phosphate uptake two-component transcriptional response regulator UhpA']\n",
      "\n",
      "KO KO:K01653 acetolactate synthase I/III small subunit [EC:2.2.1.6]\n",
      "SSO SSO:000000686 Acetolactate synthase small subunit (EC 2.2.1.6)\n",
      "b3670 ['Acetolactate synthase small subunit (EC 2.2.1.6)']\n",
      "\n",
      "KO KO:K01652 acetolactate synthase I/II/III large subunit [EC:2.2.1.6]\n",
      "SSO SSO:000000685 Acetolactate synthase large subunit (EC 2.2.1.6)\n",
      "b3671 ['Acetolactate synthase large subunit (EC 2.2.1.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b3672 ['hypothetical protein']\n",
      "\n",
      "KO KO:K19148 small toxic protein TisB\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4618 ['hypothetical protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042493 response to drug\n",
      "KO KO:K08154 MFS transporter, DHA1 family, 2-module integral membrane pump EmrD\n",
      "SSO SSO:000036922 Multidrug efflux pump EmrD (of MFS type)\n",
      "b3673 ['Multidrug efflux pump EmrD (of MFS type)']\n",
      "\n",
      "SSO SSO:000028362 Uncharacterized protein YidF\n",
      "b3674 ['Uncharacterized protein YidF']\n",
      "\n",
      "SSO SSO:000018409 Inner membrane protein YidG\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b3675 ['Inner membrane protein YidG']\n",
      "\n",
      "KO KO:K00389 putative membrane protein\n",
      "SSO SSO:000018410 Inner membrane protein YidH\n",
      "b3676 ['Inner membrane protein YidH']\n",
      "\n",
      "SSO SSO:000018411 Inner membrane protein YidI\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b3677 ['Inner membrane protein YidI']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "SSO SSO:000028705 Uncharacterized sulfatase YidJ\n",
      "b3678 ['Uncharacterized sulfatase YidJ']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03307 solute:Na+ symporter, SSS family\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000028710 Uncharacterized symporter YidK\n",
      "b3679 ['Uncharacterized symporter YidK']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000028757 Uncharacterized transcriptional regulator YidL, AraC family\n",
      "b3680 ['Uncharacterized transcriptional regulator YidL, AraC family']\n",
      "\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4556 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b3683 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03482 GntR family transcriptional regulator, glv operon transcriptional regulator\n",
      "KO KO:K03710 GntR family transcriptional regulator\n",
      "b3684 ['Uncharacterized transcriptional regulator YidP, GntR family']\n",
      "\n",
      "KO KO:K07085 putative transport protein\n",
      "SSO SSO:000023659 Putative transport protein YidE\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b3685 ['Putative transport protein YidE']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K04081 molecular chaperone IbpB\n",
      "SSO SSO:000009487 16 kDa heat shock protein B\n",
      "b3686 ['16 kDa heat shock protein B']\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K04080 molecular chaperone IbpA\n",
      "SSO SSO:000009486 16 kDa heat shock protein A\n",
      "b3687 ['16 kDa heat shock protein A']\n",
      "\n",
      "SSO SSO:000020877 Outer membrane lipoprotein YidQ\n",
      "b3688 ['Outer membrane lipoprotein YidQ']\n",
      "\n",
      "SSO SSO:000028363 Uncharacterized protein YidR\n",
      "b3689 ['Uncharacterized protein YidR']\n",
      "\n",
      "SSO SSO:000028364 Uncharacterized protein YidS\n",
      "b3690 ['Uncharacterized protein YidS']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08194 MFS transporter, ACS family, D-galactonate transporter\n",
      "SSO SSO:000012694 D-galactonate transporter\n",
      "b3691 ['D-galactonate transporter']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01684 galactonate dehydratase [EC:4.2.1.6]\n",
      "SSO SSO:000003007 Galactonate dehydratase (EC 4.2.1.6)\n",
      "b4478 ['Galactonate dehydratase (EC 4.2.1.6)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01631 2-dehydro-3-deoxyphosphogalactonate aldolase [EC:4.1.2.21]\n",
      "SSO SSO:000009529 2-dehydro-3-deoxyphosphogalactonate aldolase (EC 4.1.2.21)\n",
      "b4477 ['2-dehydro-3-deoxyphosphogalactonate aldolase (EC 4.1.2.21)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00883 2-dehydro-3-deoxygalactonokinase [EC:2.7.1.58]\n",
      "SSO SSO:000009528 2-dehydro-3-deoxygalactonokinase (EC 2.7.1.58)\n",
      "b3693 ['2-dehydro-3-deoxygalactonokinase (EC 2.7.1.58)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "KO KO:K19776 GntR family transcriptional regulator, galactonate operon transcriptional repressor\n",
      "b4479 ['D-Galactonate repressor DgoR']\n",
      "\n",
      "SSO SSO:000020814 putative lipoprotein\n",
      "SSO SSO:000028365 Uncharacterized protein YidX\n",
      "b3696 ['Uncharacterized protein YidX']\n",
      "\n",
      "KO KO:K21064 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphatase [EC:3.1.3.104]\n",
      "SSO SSO:000025245 Sugar phosphatase YidA (EC 3.1.3.23)\n",
      "b3697 ['Sugar phosphatase YidA (EC 3.1.3.23)']\n",
      "\n",
      "SSO SSO:000028361 Uncharacterized protein YidB\n",
      "b3698 ['Uncharacterized protein YidB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K02470 DNA gyrase subunit B [EC:5.99.1.3]\n",
      "SSO SSO:000002068 DNA gyrase subunit B (EC 5.99.1.3)\n",
      "b3699 ['DNA gyrase subunit B (EC 5.99.1.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03629 DNA replication and repair protein RecF\n",
      "SSO SSO:000002095 DNA recombination and repair protein RecF\n",
      "b3700 ['DNA recombination and repair protein RecF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02338 DNA polymerase III subunit beta [EC:2.7.7.7]\n",
      "SSO SSO:000002080 DNA polymerase III beta subunit (EC 2.7.7.7)\n",
      "b3701 ['DNA polymerase III beta subunit (EC 2.7.7.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02313 chromosomal replication initiator protein\n",
      "SSO SSO:000001563 Chromosomal replication initiator protein DnaA\n",
      "b3702 ['Chromosomal replication initiator protein DnaA']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02914 large subunit ribosomal protein L34\n",
      "SSO SSO:000004300 LSU ribosomal protein L34p\n",
      "b3703 ['LSU ribosomal protein L34p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006401 RNA catabolic process\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03536 ribonuclease P protein component [EC:3.1.26.5]\n",
      "SSO SSO:000007089 Ribonuclease P protein component (EC 3.1.26.5)\n",
      "b3704 ['Ribonuclease P protein component (EC 3.1.26.5)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K08998 uncharacterized protein\n",
      "SSO SSO:000035545 Membrane protein insertion efficiency factor YidD\n",
      "b4557 ['Membrane protein insertion efficiency factor YidD']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03217 YidC/Oxa1 family membrane protein insertase\n",
      "SSO SSO:000036390 Inner membrane protein translocase and chaperone YidC, long form\n",
      "b3705 ['Inner membrane protein translocase and chaperone YidC, long form']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03650 tRNA modification GTPase [EC:3.6.-.-]\n",
      "SSO SSO:000033447 tRNA-5-carboxymethylaminomethyl-2-thiouridine(34) synthesis protein MnmE\n",
      "b3706 ['tRNA-5-carboxymethylaminomethyl-2-thiouridine(34) synthesis protein MnmE']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "b3707 ['leader', 'Degradation of small molecules: Amino acids']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01667 tryptophanase [EC:4.1.99.1]\n",
      "SSO SSO:000008399 Tryptophanase (EC 4.1.99.1)\n",
      "b3708 ['Tryptophanase (EC 4.1.99.1)']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03836 low affinity tryptophan permease\n",
      "SSO SSO:000019049 Low affinity tryptophan permease\n",
      "b3709 ['Low affinity tryptophan permease']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08163 MFS transporter, DHA1 family, multidrug resistance protein\n",
      "SSO SSO:000036924 Multidrug efflux pump MdtL (of MFS type)\n",
      "b3710 ['Multidrug efflux pump MdtL (of MFS type)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000025871 Transcriptional regulator YidZ, LysR family\n",
      "b3711 ['Transcriptional regulator YidZ, LysR family']\n",
      "\n",
      "SSO SSO:000028366 Uncharacterized protein YieE\n",
      "b3712 ['Uncharacterized protein YieE']\n",
      "\n",
      "KO KO:K19784 chromate reductase, NAD(P)H dehydrogenase (quinone)\n",
      "SSO SSO:000012214 Chromate reductase (EC 1.6.5.2)\n",
      "b3713 ['Chromate reductase (EC 1.6.5.2)']\n",
      "\n",
      "KO KO:K06901 putative MFS transporter, AGZA family, xanthine/uracil permease\n",
      "SSO SSO:000010301 Adenine permease AdeP\n",
      "b3714 ['Adenine permease AdeP']\n",
      "\n",
      "SSO SSO:000009753 6-phosphogluconate phosphatase\n",
      "b3715 ['6-phosphogluconate phosphatase']\n",
      "\n",
      "SSO SSO:000006260 Predicted inner membrane protein CbrB\n",
      "b3716 ['Predicted inner membrane protein CbrB']\n",
      "\n",
      "KO KO:K09925 uncharacterized protein\n",
      "b3717 ['Colicin E2 tolerance protein CbrC-like protein => CbrC']\n",
      "\n",
      "SSO SSO:000028367 Uncharacterized protein YieK\n",
      "b3718 ['Uncharacterized protein YieK']\n",
      "\n",
      "EC EC:2.3.1.122 Trehalose O-mycolyltransferase.\n",
      "EC EC:2.3.1.20 Diacylglycerol O-acyltransferase.\n",
      "EC EC:3.1.1.1 Carboxylesterase.\n",
      "EC EC:3.1.1.72 Acetylxylan esterase.\n",
      "EC EC:3.1.1.73 Feruloyl esterase.\n",
      "EC EC:3.1.2.12 S-formylglutathione hydrolase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "SSO SSO:000023726 Putative xylanase\n",
      "SSO SSO:000027094 Uncharacterized glycosyl hyrdrolase YieL\n",
      "b3719 ['Uncharacterized glycosyl hyrdrolase YieL']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K10124 carbohydrate-specific outer membrane porin\n",
      "SSO SSO:000035214 Cryptic outer membrane porin BglH\n",
      "b3720 ['Cryptic outer membrane porin BglH']\n",
      "\n",
      "EC EC:3.2.1.105 3-alpha-(S)-strictosidine beta-glucosidase.\n",
      "EC EC:3.2.1.108 Lactase.\n",
      "EC EC:3.2.1.117 Amygdalin beta-glucosidase.\n",
      "EC EC:3.2.1.118 Prunasin beta-glucosidase.\n",
      "EC EC:3.2.1.119 Vicianin beta-glucosidase.\n",
      "EC EC:3.2.1.125 Raucaffricine beta-glucosidase.\n",
      "EC EC:3.2.1.147 Thioglucosidase.\n",
      "EC EC:3.2.1.149 Beta-primeverosidase.\n",
      "EC EC:3.2.1.161 Beta-apiosyl-beta-glucosidase.\n",
      "EC EC:3.2.1.175 Beta-D-glucopyranosyl abscisate beta-glucosidase.\n",
      "EC EC:3.2.1.182 4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl glucosidebeta-D-glucosidase.\n",
      "EC EC:3.2.1.21 Beta-glucosidase.\n",
      "EC EC:3.2.1.23 Beta-galactosidase.\n",
      "EC EC:3.2.1.25 Beta-mannosidase.\n",
      "EC EC:3.2.1.31 Beta-glucuronidase.\n",
      "EC EC:3.2.1.37 Xylan 1,4-beta-xylosidase.\n",
      "EC EC:3.2.1.38 Beta-D-fucosidase.\n",
      "EC EC:3.2.1.62 Glycosylceramidase.\n",
      "EC EC:3.2.1.74 Glucan 1,4-beta-glucosidase.\n",
      "EC EC:3.2.1.85 6-phospho-beta-galactosidase.\n",
      "EC EC:3.2.1.86 6-phospho-beta-glucosidase.\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01223 6-phospho-beta-glucosidase [EC:3.2.1.86]\n",
      "SSO SSO:000000451 6-phospho-beta-glucosidase (EC 3.2.1.86)\n",
      "b3721 ['6-phospho-beta-glucosidase (EC 3.2.1.86)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02757 PTS system, beta-glucoside-specific IIC component\n",
      "b3722 ['PTS system, beta-glucoside-specific IIB component', 'PTS system, beta-glucoside-specific IIC component', 'PTS system, beta-glucoside-specific IIA component']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03488 beta-glucoside operon transcriptional antiterminator\n",
      "SSO SSO:000011104 Beta-glucoside bgl operon antiterminator, BglG family\n",
      "b3723 ['Beta-glucoside bgl operon antiterminator, BglG family']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K02039 phosphate transport system protein\n",
      "SSO SSO:000005927 Phosphate transport system regulatory protein PhoU\n",
      "b3724 ['Phosphate transport system regulatory protein PhoU']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K02036 phosphate transport system ATP-binding protein [EC:7.3.2.1]\n",
      "SSO SSO:000038495 Phosphate ABC transporter, ATP-binding protein PstB (TC 3.A.1.7.1)\n",
      "b3725 ['Phosphate ABC transporter, ATP-binding protein PstB (TC 3.A.1.7.1)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02038 phosphate transport system permease protein\n",
      "SSO SSO:000038496 Phosphate ABC transporter, permease protein PstA (TC 3.A.1.7.1)\n",
      "b3726 ['Phosphate ABC transporter, permease protein PstA (TC 3.A.1.7.1)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02037 phosphate transport system permease protein\n",
      "SSO SSO:000038497 Phosphate ABC transporter, permease protein PstC (TC 3.A.1.7.1)\n",
      "b3727 ['Phosphate ABC transporter, permease protein PstC (TC 3.A.1.7.1)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02040 phosphate transport system substrate-binding protein\n",
      "SSO SSO:000038498 Phosphate ABC transporter, substrate-binding protein PstS (TC 3.A.1.7.1)\n",
      "b3728 ['Phosphate ABC transporter, substrate-binding protein PstS (TC 3.A.1.7.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "GO GO:0046349 amino sugar biosynthetic process\n",
      "KO KO:K00820 glucosamine---fructose-6-phosphate aminotransferase (isomerizing) [EC:2.6.1.16]\n",
      "SSO SSO:000036121 Glutamine--fructose-6-phosphate aminotransferase [isomerizing] (EC 2.6.1.16)\n",
      "b3729 ['Glutamine--fructose-6-phosphate aminotransferase [isomerizing] (EC 2.6.1.16)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009310 amine catabolic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K04042 bifunctional UDP-N-acetylglucosamine pyrophosphorylase / Glucosamine-1-phosphate N-acetyltransferase [EC:2.7.7.23 2.3.1.157]\n",
      "b3730 ['N-acetylglucosamine-1-phosphate uridyltransferase (EC 2.7.7.23)', 'Glucosamine-1-phosphate N-acetyltransferase (EC 2.3.1.157)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02114 F-type H+-transporting ATPase subunit epsilon\n",
      "SSO SSO:000000554 ATP synthase epsilon chain (EC 3.6.3.14)\n",
      "b3731 ['ATP synthase epsilon chain (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02112 F-type H+/Na+-transporting ATPase subunit beta [EC:7.1.2.2 7.2.2.1]\n",
      "SSO SSO:000000551 ATP synthase beta chain (EC 3.6.3.14)\n",
      "b3732 ['ATP synthase beta chain (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02115 F-type H+-transporting ATPase subunit gamma\n",
      "SSO SSO:000000555 ATP synthase gamma chain (EC 3.6.3.14)\n",
      "b3733 ['ATP synthase gamma chain (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02111 F-type H+/Na+-transporting ATPase subunit alpha [EC:7.1.2.2 7.2.2.1]\n",
      "SSO SSO:000000549 ATP synthase alpha chain (EC 3.6.3.14)\n",
      "b3734 ['ATP synthase alpha chain (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02113 F-type H+-transporting ATPase subunit delta\n",
      "SSO SSO:000000553 ATP synthase delta chain (EC 3.6.3.14)\n",
      "b3735 ['ATP synthase delta chain (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02109 F-type H+-transporting ATPase subunit b\n",
      "SSO SSO:000000546 ATP synthase F0 sector subunit b (EC 3.6.3.14)\n",
      "b3736 ['ATP synthase F0 sector subunit b (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02110 F-type H+-transporting ATPase subunit c\n",
      "SSO SSO:000000548 ATP synthase F0 sector subunit c (EC 3.6.3.14)\n",
      "b3737 ['ATP synthase F0 sector subunit c (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02108 F-type H+-transporting ATPase subunit a\n",
      "SSO SSO:000000545 ATP synthase F0 sector subunit a (EC 3.6.3.14)\n",
      "b3738 ['ATP synthase F0 sector subunit a (EC 3.6.3.14)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000000556 ATP synthase protein I (EC 3.6.3.14 )\n",
      "b3739 ['ATP synthase protein I']\n",
      "\n",
      "KO KO:K03501 16S rRNA (guanine527-N7)-methyltransferase [EC:2.1.1.170]\n",
      "SSO SSO:000000061 16S rRNA (guanine(527)-N(7))-methyltransferase (EC 2.1.1.170)\n",
      "b3740 ['16S rRNA (guanine(527)-N(7))-methyltransferase (EC 2.1.1.170)']\n",
      "\n",
      "KO KO:K03495 tRNA uridine 5-carboxymethylaminomethyl modification enzyme\n",
      "SSO SSO:000033448 tRNA-5-carboxymethylaminomethyl-2-thiouridine(34) synthesis protein MnmG\n",
      "b3741 ['tRNA-5-carboxymethylaminomethyl-2-thiouridine(34) synthesis protein MnmG']\n",
      "\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K06205 MioC protein\n",
      "SSO SSO:000002852 Flavoprotein MioC\n",
      "b3742 ['Flavoprotein MioC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006529 asparagine biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03718 Lrp/AsnC family transcriptional regulator, regulator for asnA, asnC and gidA\n",
      "SSO SSO:000025794 Transcriptional regulator AsnC\n",
      "b3743 ['Transcriptional regulator AsnC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006529 asparagine biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01914 aspartate--ammonia ligase [EC:6.3.1.1]\n",
      "SSO SSO:000001098 Aspartate--ammonia ligase (EC 6.3.1.1)\n",
      "b3744 ['Aspartate--ammonia ligase (EC 6.3.1.1)']\n",
      "\n",
      "SSO SSO:000022724 Protein ViaA\n",
      "b3745 ['Protein ViaA']\n",
      "\n",
      "KO KO:K03924 MoxR-like ATPase [EC:3.6.3.-]\n",
      "SSO SSO:000010098 ATPase RavA\n",
      "b3746 ['ATPase RavA']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03549 KUP system potassium uptake protein\n",
      "SSO SSO:000018715 Kup system potassium uptake protein\n",
      "b3747 ['Kup system potassium uptake protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06726 D-ribose pyranase [EC:5.4.99.62]\n",
      "SSO SSO:000034830 D-ribose pyranase (EC 5.4.99.62)\n",
      "b3748 ['D-ribose pyranase (EC 5.4.99.62)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10441 ribose transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000039032 Ribose ABC transporter, ATP-binding protein RbsA (TC 3.A.1.2.1)\n",
      "b3749 ['Ribose ABC transporter, ATP-binding protein RbsA (TC 3.A.1.2.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02057 simple sugar transport system permease protein\n",
      "KO KO:K10440 ribose transport system permease protein\n",
      "SSO SSO:000039033 Ribose ABC transporter, permease protein RbsC (TC 3.A.1.2.1)\n",
      "b3750 ['Ribose ABC transporter, permease protein RbsC (TC 3.A.1.2.1)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10439 ribose transport system substrate-binding protein\n",
      "SSO SSO:000039034 Ribose ABC transporter, substrate-binding protein RbsB (TC 3.A.1.2.1)\n",
      "b3751 ['Ribose ABC transporter, substrate-binding protein RbsB (TC 3.A.1.2.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00852 ribokinase [EC:2.7.1.15]\n",
      "SSO SSO:000007068 Ribokinase (EC 2.7.1.15)\n",
      "b3752 ['Ribokinase (EC 2.7.1.15)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "b3753 ['Ribose operon repressor']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000022602 Probable transport protein HsrA\n",
      "b3754 ['Probable transport protein HsrA']\n",
      "\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "SSO SSO:000028758 Uncharacterized transcriptional regulator YieP, GntR family\n",
      "b3755 ['Uncharacterized transcriptional regulator YieP, GntR family']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K23773 Unknown\n",
      "SSO SSO:000004570 LysR family transcriptional regulator HdfR\n",
      "b4480 ['LysR family transcriptional regulator HdfR']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K09897 uncharacterized protein\n",
      "SSO SSO:000026680 UPF0438 protein YifE\n",
      "b3764 ['UPF0438 protein YifE']\n",
      "\n",
      "KO KO:K07391 magnesium chelatase family protein\n",
      "SSO SSO:000009788 AAA+ ATPase superfamily protein YifB/ComM, associated with DNA recombination\n",
      "b3765 ['AAA+ ATPase superfamily protein YifB/ComM, associated with DNA recombination']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000018320 IlvGMEDA operon leader peptide\n",
      "b3766 ['IlvGMEDA operon leader peptide']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4669 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4488 ['hypothetical protein']\n",
      "\n",
      "KO KO:K11258 acetolactate synthase II small subunit [EC:2.2.1.6]\n",
      "SSO SSO:000000686 Acetolactate synthase small subunit (EC 2.2.1.6)\n",
      "b3769 ['Acetolactate synthase small subunit (EC 2.2.1.6)']\n",
      "\n",
      "GO GO:0006523 alanine biosynthetic process\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K00826 branched-chain amino acid aminotransferase [EC:2.6.1.42]\n",
      "SSO SSO:000001260 Branched-chain amino acid aminotransferase (EC 2.6.1.42)\n",
      "b3770 ['Branched-chain amino acid aminotransferase (EC 2.6.1.42)']\n",
      "\n",
      "KO KO:K01687 dihydroxy-acid dehydratase [EC:4.2.1.9]\n",
      "SSO SSO:000002316 Dihydroxy-acid dehydratase (EC 4.2.1.9)\n",
      "b3771 ['Dihydroxy-acid dehydratase (EC 4.2.1.9)']\n",
      "\n",
      "KO KO:K01754 threonine dehydratase [EC:4.3.1.19]\n",
      "SSO SSO:000008059 Threonine dehydratase biosynthetic (EC 4.3.1.19)\n",
      "b3772 ['Threonine dehydratase biosynthetic (EC 4.3.1.19)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02521 LysR family transcriptional regulator, positive regulator for ilvC\n",
      "SSO SSO:000025817 Transcriptional regulator IlvY, LysR family\n",
      "b3773 ['Transcriptional regulator IlvY, LysR family']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00053 ketol-acid reductoisomerase [EC:1.1.1.86]\n",
      "SSO SSO:000036468 Ketol-acid reductoisomerase (NAD(P)(+)) (EC 1.1.1.383)\n",
      "b3774 ['Ketol-acid reductoisomerase (NADP(+)) (EC 1.1.1.86)']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "SSO SSO:000005811 Peptidyl-prolyl cis-trans isomerase PpiC (EC 5.2.1.8)\n",
      "b3775 ['Peptidyl-prolyl cis-trans isomerase PpiC (EC 5.2.1.8)']\n",
      "\n",
      "KO KO:K07171 mRNA interferase MazF [EC:3.1.-.-]\n",
      "b3777 ['FIG00642180: hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K03656 ATP-dependent DNA helicase Rep [EC:3.6.4.12]\n",
      "KO KO:K03657 DNA helicase II / ATP-dependent DNA helicase PcrA [EC:3.6.4.12]\n",
      "SSO SSO:000000585 ATP-dependent DNA helicase rep (EC 3.6.1.-)\n",
      "b3778 ['ATP-dependent DNA helicase Rep']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K01524 exopolyphosphatase / guanosine-5'-triphosphate,3'-diphosphate pyrophosphatase [EC:3.6.1.11 3.6.1.40]\n",
      "b3779 [\"Guanosine-5'-triphosphate,3'-diphosphate pyrophosphatase (EC 3.6.1.40)\", 'Exopolyphosphatase (EC 3.6.1.11)']\n",
      "\n",
      "KO KO:K03732 ATP-dependent RNA helicase RhlB [EC:3.6.4.13]\n",
      "SSO SSO:000000609 ATP-dependent RNA helicase rhlB (EC 3.6.1.-)\n",
      "SSO SSO:000009996 ATP dependent RNA helicase\n",
      "b3780 ['ATP-dependent RNA helicase RhlB (EC 3.6.4.13)']\n",
      "\n",
      "KO KO:K03671 thioredoxin 1\n",
      "SSO SSO:000008042 Thioredoxin (EC 1.8.1.8)\n",
      "b3781 ['Thioredoxin']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03628 transcription termination factor Rho\n",
      "SSO SSO:000008176 Transcription termination factor Rho\n",
      "b3783 ['Transcription termination factor Rho']\n",
      "\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02851 UDP-GlcNAc:undecaprenyl-phosphate/decaprenyl-phosphate GlcNAc-1-phosphate transferase [EC:2.7.8.33 2.7.8.35]\n",
      "SSO SSO:000008820 Undecaprenyl-phosphate alpha-N-acetylglucosaminyl 1-phosphate transferase (EC 2.7.8.33)\n",
      "SSO SSO:000043607 UDP-GlcNAc:undecaprenyl-phosphate GlcNAc-1-phosphate transferase (EC 2.7.8.33)\n",
      "b3784 ['Undecaprenyl-phosphate alpha-N-acetylglucosaminyl 1-phosphate transferase (EC 2.7.8.33)']\n",
      "\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K05790 lipopolysaccharide biosynthesis protein WzzE\n",
      "SSO SSO:000009173 lipopolysaccharide biosynthesis protein WzzE\n",
      "b3785 ['Lipopolysaccharide biosynthesis protein WzzE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01791 UDP-N-acetylglucosamine 2-epimerase (non-hydrolysing) [EC:5.1.3.14]\n",
      "SSO SSO:000008617 UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)\n",
      "b3786 ['UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)']\n",
      "\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02472 UDP-N-acetyl-D-mannosaminuronic acid dehydrogenase [EC:1.1.1.336]\n",
      "SSO SSO:000008610 UDP-N-acetyl-D-mannosamine dehydrogenase (EC 1.1.1.-)\n",
      "SSO SSO:000026439 UDP-N-acetyl-D-mannosaminuronic acid dehydrogenase\n",
      "b3787 ['UDP-N-acetyl-D-mannosamine dehydrogenase (EC 1.1.1.336)']\n",
      "\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K01710 dTDP-glucose 4,6-dehydratase [EC:4.2.1.46]\n",
      "SSO SSO:000009095 dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)\n",
      "b3788 ['dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)']\n",
      "\n",
      "GO GO:0009243 O antigen biosynthetic process\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K00973 glucose-1-phosphate thymidylyltransferase [EC:2.7.7.24]\n",
      "SSO SSO:000003126 Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)\n",
      "b3789 ['Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)']\n",
      "\n",
      "EC EC:2.3.1.210 dTDP-4-amino-4,6-dideoxy-D-galactose acyltransferase.\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K16704 dTDP-4-amino-4,6-dideoxy-D-galactose acyltransferase [EC:2.3.1.210]\n",
      "SSO SSO:000009094 dTDP-fucosamine acetyltransferase (EC 2.3.1.210)\n",
      "b3790 ['dTDP-fucosamine acetyltransferase (EC 2.3.1.210)']\n",
      "\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02805 dTDP-4-amino-4,6-dideoxygalactose transaminase [EC:2.6.1.59]\n",
      "SSO SSO:000009087 dTDP-4-amino-4,6-dideoxygalactose transaminase (EC 2.6.1.59)\n",
      "b3791 ['dTDP-4-amino-4,6-dideoxygalactose transaminase (EC 2.6.1.59)']\n",
      "\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K16693 enterobacterial common antigen flippase\n",
      "SSO SSO:000004458 Lipid III flippase\n",
      "b3792 ['Lipid III flippase']\n",
      "\n",
      "KO KO:K12582 dTDP-N-acetylfucosamine:lipid II N-acetylfucosaminyltransferase [EC:2.4.1.325]\n",
      "SSO SSO:000007884 TDP-N-acetylfucosamine:lipid II N-acetylfucosaminyltransferase (EC 2.4.1.325)\n",
      "b4481 ['TDP-N-acetylfucosamine:lipid II N-acetylfucosaminyltransferase (EC 2.4.1.325)']\n",
      "\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02853 enterobacterial common antigen polymerase [EC:2.4.1.-]\n",
      "SSO SSO:000006598 Putative ECA polymerase WzyE\n",
      "b3793 ['Putative ECA polymerase WzyE']\n",
      "\n",
      "GO GO:0009246 enterobacterial common antigen biosynthetic process\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K02852 UDP-N-acetyl-D-mannosaminouronate:lipid I N-acetyl-D-mannosaminouronosyltransferase [EC:2.4.1.180]\n",
      "SSO SSO:000004475 Lipopolysaccharide N-acetylmannosaminouronosyltransferase (EC 2.4.1.180)\n",
      "SSO SSO:000026440 UDP-N-acetyl-D-mannosaminuronic acid transferase\n",
      "b3794 ['Lipopolysaccharide N-acetylmannosaminouronosyltransferase (EC 2.4.1.180)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03293 amino acid transporter, AAT family\n",
      "SSO SSO:000022603 Probable transport protein YifK\n",
      "b3795 ['Probable transport protein YifK']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K06871 uncharacterized protein\n",
      "SSO SSO:000010652 Anaerobic sulfatase-maturating enzyme homolog AslB\n",
      "b3800 ['Anaerobic sulfatase-maturating enzyme homolog AslB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "SSO SSO:000001077 Arylsulfatase (EC 3.1.6.1)\n",
      "b3801 ['Arylsulfatase (EC 3.1.6.1)']\n",
      "\n",
      "KO KO:K02498 HemY protein\n",
      "SSO SSO:000027462 Uncharacterized protein EC-HemY in Proteobacteria (unrelated to HemY-type PPO in GramPositives)\n",
      "b3802 ['Uncharacterized protein EC-HemY in Proteobacteria (unrelated to HemY-type PPO in GramPositives)']\n",
      "\n",
      "KO KO:K02496 uroporphyrin-III C-methyltransferase [EC:2.1.1.107]\n",
      "SSO SSO:000008748 Uncharacterized protein EC-HemX\n",
      "b3803 ['Uncharacterized protein EC-HemX']\n",
      "\n",
      "KO KO:K01719 uroporphyrinogen-III synthase [EC:4.2.1.75]\n",
      "SSO SSO:000008887 Uroporphyrinogen-III synthase (EC 4.2.1.75)\n",
      "b3804 ['Uroporphyrinogen-III synthase (EC 4.2.1.75)']\n",
      "\n",
      "KO KO:K01749 hydroxymethylbilane synthase [EC:2.5.1.61]\n",
      "SSO SSO:000006175 Porphobilinogen deaminase (EC 2.5.1.61)\n",
      "b3805 ['Porphobilinogen deaminase (EC 2.5.1.61)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K05851 adenylate cyclase, class 1 [EC:4.6.1.1]\n",
      "SSO SSO:000000819 Adenylate cyclase (EC 4.6.1.1)\n",
      "b3806 ['Adenylate cyclase (EC 4.6.1.1)']\n",
      "\n",
      "KO KO:K06202 CyaY protein\n",
      "SSO SSO:000002925 Frataxin homolog CyaY, facilitates Fe-S cluster assembly, interacts with IscS\n",
      "b3807 ['Frataxin homolog CyaY, facilitates Fe-S cluster assembly, interacts with IscS']\n",
      "\n",
      "SSO SSO:000013087 Diaminopimelate decarboxylase and/or diaminopimelate epimerase leader peptide\n",
      "SSO SSO:000020814 putative lipoprotein\n",
      "b4558 ['Diaminopimelate decarboxylase and/or diaminopimelate epimerase leader peptide']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K01778 diaminopimelate epimerase [EC:5.1.1.7]\n",
      "SSO SSO:000002281 Diaminopimelate epimerase (EC 5.1.1.7)\n",
      "b3809 ['Diaminopimelate epimerase (EC 5.1.1.7)']\n",
      "\n",
      "KO KO:K09921 uncharacterized protein\n",
      "SSO SSO:000026848 Uncharacterized DUF484 protein YigA\n",
      "b3810 ['Uncharacterized DUF484 protein YigA']\n",
      "\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03733 integrase/recombinase XerC\n",
      "KO KO:K04763 integrase/recombinase XerD\n",
      "SSO SSO:000007586 Site-specific tyrosine recombinase\n",
      "SSO SSO:000007587 Site-specific tyrosine recombinase XerC\n",
      "b3811 ['Site-specific tyrosine recombinase XerC']\n",
      "\n",
      "KO KO:K20862 FMN hydrolase / 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphatase [EC:3.1.3.102 3.1.3.104]\n",
      "SSO SSO:000028680 Uncharacterized putative hydrolase YigB\n",
      "b3812 ['Uncharacterized putative hydrolase YigB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03657 DNA helicase II / ATP-dependent DNA helicase PcrA [EC:3.6.4.12]\n",
      "SSO SSO:000000572 ATP-dependent DNA helicase UvrD/PcrA (EC 3.6.4.12)\n",
      "b3813 ['ATP-dependent DNA helicase UvrD/PcrA (EC 3.6.4.12)']\n",
      "\n",
      "SSO SSO:000026664 UPF0386 protein YjhX\n",
      "b4482 ['UPF0386 protein YjhX']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03284 magnesium transporter\n",
      "SSO SSO:000004630 Magnesium and cobalt transport protein CorA\n",
      "b3816 ['Magnesium and cobalt transport protein CorA']\n",
      "\n",
      "SSO SSO:000028368 Uncharacterized protein YigF\n",
      "b3817 ['Uncharacterized protein YigF']\n",
      "\n",
      "SSO SSO:000018412 Inner membrane protein YigG\n",
      "b3818 ['Inner membrane protein YigG']\n",
      "\n",
      "GO GO:0042493 response to drug\n",
      "GO GO:0042892 chloramphenicol transport\n",
      "KO KO:K05786 chloramphenicol-sensitive protein RarD\n",
      "SSO SSO:000040130 Uncharacterized inner membrane protein RarD\n",
      "b3819 ['Uncharacterized inner membrane protein RarD']\n",
      "\n",
      "SSO SSO:000028369 Uncharacterized protein YigI\n",
      "b3820 ['Uncharacterized protein YigI']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K01058 phospholipase A1/A2 [EC:3.1.1.32 3.1.1.4]\n",
      "SSO SSO:000021672 Outer membrane phospholipase A\n",
      "b3821 ['Phospholipase A1 (EC 3.1.1.32) (EC 3.1.1.4)', 'Outer membrane phospholipase A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006310 DNA recombination\n",
      "KO KO:K03654 ATP-dependent DNA helicase RecQ [EC:3.6.4.12]\n",
      "SSO SSO:000000584 ATP-dependent DNA helicase recQ (EC 3.6.1.- )\n",
      "SSO SSO:000010027 ATP-dependent DNA helicase\n",
      "b3822 ['ATP-dependent DNA helicase RecQ']\n",
      "\n",
      "GO GO:0009088 threonine biosynthetic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05835 threonine efflux protein\n",
      "SSO SSO:000039472 Threonine efflux protein RhtC\n",
      "b3823 ['Threonine efflux protein RhtC']\n",
      "\n",
      "GO GO:0006564 L-serine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K05834 homoserine/homoserine lactone efflux protein\n",
      "SSO SSO:000017874 Homoserine/homoserine lactone efflux protein\n",
      "b3824 ['Homoserine/homoserine lactone efflux protein']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K01048 lysophospholipase [EC:3.1.1.5]\n",
      "SSO SSO:000019149 Lysophospholipase L2 (EC 3.1.1.5)\n",
      "b3825 ['Lysophospholipase L2 (EC 3.1.1.5)']\n",
      "\n",
      "KO KO:K11938 HMP-PP phosphatase [EC:3.6.1.-]\n",
      "KO KO:K21064 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphatase [EC:3.1.3.104]\n",
      "SSO SSO:000036233 HAD phosphatase YigL\n",
      "b3826 ['HAD phosphatase YigL']\n",
      "\n",
      "b3827 ['Uncharacterized membrane protein YigM']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03576 LysR family transcriptional regulator, regulator for metE and metH\n",
      "SSO SSO:000025769 Transcriptional activator MetR\n",
      "b3828 ['Transcriptional activator MetR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006556 S-adenosylmethionine biosynthetic process\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "KO KO:K00549 5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase [EC:2.1.1.14]\n",
      "SSO SSO:000000430 5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)\n",
      "SSO SSO:000042475 5-methyltetrahydropteroyltriglutamate---homocysteine S-methyltransferase (EC 2.1.1.14)\n",
      "b3829 ['5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)']\n",
      "\n",
      "KO KO:K01061 carboxymethylenebutenolidase [EC:3.1.1.45]\n",
      "SSO SSO:000013098 Dienelactone hydrolase family protein\n",
      "b3830 ['Dienelactone hydrolase family protein']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00757 uridine phosphorylase [EC:2.4.2.3]\n",
      "SSO SSO:000008880 Uridine phosphorylase (EC 2.4.2.3)\n",
      "b3831 ['Uridine phosphorylase (EC 2.4.2.3)']\n",
      "\n",
      "KO KO:K09760 DNA recombination protein RmuC\n",
      "SSO SSO:000002097 DNA recombination protein RmuC\n",
      "b3832 ['DNA recombination protein RmuC']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K03183 demethylmenaquinone methyltransferase / 2-methoxy-6-polyprenyl-1,4-benzoquinol methylase [EC:2.1.1.163 2.1.1.201]\n",
      "b3833 ['2-methoxy-6-polyprenyl-1,4-benzoquinol methylase (EC 2.1.1.201)', 'Demethylmenaquinone methyltransferase (EC 2.1.1.163)']\n",
      "\n",
      "KO KO:K03690 ubiquinone biosynthesis protein UbiJ\n",
      "SSO SSO:000008685 Ubiquinone biosynthesis protein UbiJ\n",
      "b3834 ['Ubiquinone biosynthesis protein UbiJ']\n",
      "\n",
      "KO KO:K03688 ubiquinone biosynthesis protein\n",
      "SSO SSO:000008686 Ubiquinone biosynthesis regulatory protein kinase UbiB\n",
      "b3835 ['Ubiquinone biosynthesis regulatory protein kinase UbiB']\n",
      "\n",
      "KO KO:K03116 sec-independent protein translocase protein TatA\n",
      "SSO SSO:000008421 Twin-arginine translocation protein TatA\n",
      "b3836 ['Twin-arginine translocation protein TatA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03117 sec-independent protein translocase protein TatB\n",
      "SSO SSO:000008424 Twin-arginine translocation protein TatB\n",
      "b3838 ['Twin-arginine translocation protein TatB']\n",
      "\n",
      "KO KO:K03118 sec-independent protein translocase protein TatC\n",
      "SSO SSO:000008425 Twin-arginine translocation protein TatC\n",
      "b3839 ['Twin-arginine translocation protein TatC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03424 TatD DNase family protein [EC:3.1.21.-]\n",
      "SSO SSO:000002254 Deoxyribonuclease TatD\n",
      "b4483 ['Deoxyribonuclease TatD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009244 lipopolysaccharide core region biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "GO GO:0009297 pilus assembly\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0042280 obsolete cell surface antigen activity, host-interacting\n",
      "KO KO:K05785 transcriptional antiterminator RfaH\n",
      "SSO SSO:000008186 Transcriptional activator RfaH\n",
      "SSO SSO:000039514 Transcription antitermination protein\n",
      "b3842 ['Transcriptional activator RfaH']\n",
      "\n",
      "KO KO:K03182 4-hydroxy-3-polyprenylbenzoate decarboxylase [EC:4.1.1.98]\n",
      "SSO SSO:000000341 3-polyprenyl-4-hydroxybenzoate carboxy-lyase (EC 4.1.1.-)\n",
      "b3843 ['3-polyprenyl-4-hydroxybenzoate carboxy-lyase (EC 4.1.1.98)']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K05368 aquacobalamin reductase / NAD(P)H-flavin reductase [EC:1.16.1.3 1.5.1.41]\n",
      "SSO SSO:000005170 NAD(P)H-flavin reductase (EC 1.16.1.3) (EC 1.5.1.29)\n",
      "b3844 ['NAD(P)H-flavin reductase (EC 1.5.1.41) (EC 1.16.1.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K00626 acetyl-CoA C-acetyltransferase [EC:2.3.1.9]\n",
      "KO KO:K00632 acetyl-CoA acyltransferase [EC:2.3.1.16]\n",
      "SSO SSO:000000312 3-ketoacyl-CoA thiolase (EC 2.3.1.16)\n",
      "b3845 ['3-ketoacyl-CoA thiolase (EC 2.3.1.16)']\n",
      "\n",
      "GO GO:0019395 fatty acid oxidation\n",
      "KO KO:K01825 3-hydroxyacyl-CoA dehydrogenase / enoyl-CoA hydratase / 3-hydroxybutyryl-CoA epimerase / enoyl-CoA isomerase [EC:1.1.1.35 4.2.1.17 5.1.2.3 5.3.3.8]\n",
      "b3846 ['Enoyl-CoA hydratase (EC 4.2.1.17)', 'Delta(3)-cis-delta(2)-trans-enoyl-CoA isomerase (EC 5.3.3.8)', '3-hydroxyacyl-CoA dehydrogenase (EC 1.1.1.35)', '3-hydroxybutyryl-CoA epimerase (EC 5.1.2.3)']\n",
      "\n",
      "SSO SSO:000006363 Proline dipeptidase (EC 3.4.13.9)\n",
      "SSO SSO:000008955 Xaa-Pro dipeptidase PepQ (EC 3.4.13.9)\n",
      "b3847 ['Xaa-Pro dipeptidase PepQ (EC 3.4.13.9)']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "b3848 ['FIG000605: protein co-occurring with transport systems (COG1739)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03498 trk system potassium uptake protein\n",
      "SSO SSO:000008388 Trk potassium uptake system protein TrkH\n",
      "b3849 ['Trk potassium uptake system protein TrkH']\n",
      "\n",
      "KO KO:K00230 menaquinone-dependent protoporphyrinogen oxidase [EC:1.3.5.3]\n",
      "SSO SSO:000006552 Protoporphyrinogen IX oxidase, oxygen-independent, HemG (EC 1.3.-.-)\n",
      "b3850 ['Protoporphyrinogen IX oxidase, oxygen-independent, HemG']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03753 molybdopterin-guanine dinucleotide biosynthesis adapter protein\n",
      "SSO SSO:000004947 Molybdopterin-guanine dinucleotide biosynthesis protein MobB\n",
      "b3856 ['Molybdopterin-guanine dinucleotide biosynthesis protein MobB']\n",
      "\n",
      "GO GO:0006777 Mo-molybdopterin cofactor biosynthetic process\n",
      "KO KO:K03752 molybdenum cofactor guanylyltransferase [EC:2.7.7.77]\n",
      "SSO SSO:000004938 Molybdenum cofactor guanylyltransferase (EC 2.7.7.77)\n",
      "b3857 ['Molybdenum cofactor guanylyltransferase (EC 2.7.7.77)']\n",
      "\n",
      "KO KO:K09896 uncharacterized protein\n",
      "SSO SSO:000006531 Protein yihD\n",
      "b3858 ['Protein YihD']\n",
      "\n",
      "SSO SSO:000008984 YihE protein, a ser/thr kinase implicated in LPS synthesis and Cpx signalling\n",
      "b3859 ['YihE protein, a ser/thr kinase implicated in LPS synthesis and Cpx signalling']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0006950 response to stress\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03673 thiol:disulfide interchange protein DsbA\n",
      "SSO SSO:000005848 Periplasmic thiol:disulfide interchange protein DsbA\n",
      "b3860 ['Periplasmic thiol:disulfide interchange protein DsbA']\n",
      "\n",
      "SSO SSO:000028370 Uncharacterized protein YihF\n",
      "b3861 ['Uncharacterized protein YihF']\n",
      "\n",
      "SSO SSO:000022420 Probable acyltransferase YihG\n",
      "b3862 ['Probable acyltransferase YihG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02335 DNA polymerase I [EC:2.7.7.7]\n",
      "SSO SSO:000002077 DNA polymerase I (EC 2.7.7.7)\n",
      "b3863 ['DNA polymerase I (EC 2.7.7.7)']\n",
      "\n",
      "GO GO:0007049 cell cycle\n",
      "KO KO:K03978 GTP-binding protein\n",
      "SSO SSO:000002994 GTP-binding protein EngB\n",
      "b3865 ['GTP-binding protein EngB']\n",
      "\n",
      "KO KO:K09894 uncharacterized protein\n",
      "SSO SSO:000013058 Der GTPase-activating protein YihI\n",
      "b3866 ['Der GTPase-activating protein YihI']\n",
      "\n",
      "KO KO:K02495 oxygen-independent coproporphyrinogen III oxidase [EC:1.3.98.3]\n",
      "SSO SSO:000001781 Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)\n",
      "b3867 ['Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)']\n",
      "\n",
      "SSO SSO:000041472 Uncharacterized protein YshB\n",
      "b4686 ['Uncharacterized protein YshB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K07712 two-component system, NtrC family, nitrogen regulation response regulator GlnG\n",
      "KO KO:K07713 two-component system, NtrC family, response regulator HydG\n",
      "KO KO:K07714 two-component system, NtrC family, response regulator AtoC\n",
      "SSO SSO:000037224 Nitrogen regulation protein NR(I), GlnG (=NtrC)\n",
      "b3868 ['Nitrogen regulation protein NR(I), GlnG (=NtrC)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07708 two-component system, NtrC family, nitrogen regulation sensor histidine kinase GlnL [EC:2.7.13.3]\n",
      "SSO SSO:000005350 Nitrogen regulation protein NtrB (EC 2.7.13.3)\n",
      "b3869 ['Nitrogen regulation protein NtrB (EC 2.7.13.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006542 glutamine biosynthetic process\n",
      "GO GO:0006807 nitrogen compound metabolic process\n",
      "KO KO:K01915 glutamine synthetase [EC:6.3.1.2]\n",
      "SSO SSO:000003169 Glutamine synthetase (EC 6.3.1.2)\n",
      "SSO SSO:000003172 Glutamine synthetase type I (EC 6.3.1.2)\n",
      "b3870 ['Glutamine synthetase type I (EC 6.3.1.2)']\n",
      "\n",
      "KO KO:K06207 GTP-binding protein\n",
      "SSO SSO:000002992 GTP-binding protein\n",
      "SSO SSO:000002998 GTP-binding protein TypA/BipA\n",
      "b3871 ['GTP-binding protein TypA/BipA']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "b3872 ['Uncharacterized transcriptional regulator YihL, GntR family']\n",
      "\n",
      "SSO SSO:000028371 Uncharacterized protein YihM\n",
      "b3873 ['Uncharacterized protein YihM']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000018413 Inner membrane protein YihN\n",
      "b3874 ['Inner membrane protein YihN']\n",
      "\n",
      "KO KO:K22110 oligogalacturonate-specific porin family protein\n",
      "SSO SSO:000007844 Sulpholipid porin\n",
      "b3875 ['Sulpholipid porin']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03292 glycoside/pentoside/hexuronide:cation symporter, GPH family\n",
      "SSO SSO:000007845 Sulphoquinovose/Na+ symporter\n",
      "b3876 ['Sulphoquinovose/Na+ symporter']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03292 glycoside/pentoside/hexuronide:cation symporter, GPH family\n",
      "SSO SSO:000000094 2,3-dihydroxypropane-1-sulfonate exporter\n",
      "b3877 ['2,3-dihydroxypropane-1-sulfonate exporter']\n",
      "\n",
      "EC EC:2.4.1.161 Oligosaccharide 4-alpha-D-glucosyltransferase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.177 Alpha-D-xyloside xylohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.22 Alpha-galactosidase.\n",
      "EC EC:3.2.1.24 Alpha-mannosidase.\n",
      "EC EC:3.2.1.48 Sucrose alpha-glucosidase.\n",
      "EC EC:3.2.1.84 Glucan 1,3-alpha-glucosidase.\n",
      "EC EC:4.2.2.13 Exo-(1->4)-alpha-D-glucan lyase.\n",
      "KO KO:K15922 sulfoquinovosidase [EC:3.2.1.199]\n",
      "SSO SSO:000007843 Sulpholipid alpha-glucosidase (proposed)\n",
      "b3878 ['Sulpholipid alpha-glucosidase (proposed)']\n",
      "\n",
      "KO KO:K01785 aldose 1-epimerase [EC:5.1.3.3]\n",
      "SSO SSO:000007798 Sugar-1-epimerase YihR\n",
      "b3879 ['Sugar-1-epimerase YihR']\n",
      "\n",
      "KO KO:K18479 sulfoquinovose isomerase [EC:5.3.1.31]\n",
      "SSO SSO:000039339 Sulfoquinovose isomerase (EC 5.3.1.31)\n",
      "b3880 ['Sulfoquinovose isomerase (EC 5.3.1.31)']\n",
      "\n",
      "KO KO:K01671 sulfofructosephosphate aldolase [EC:4.1.2.57]\n",
      "SSO SSO:000000444 6-deoxy-6-sulphofructose-1-phosphate aldolase\n",
      "b3881 ['6-deoxy-6-sulphofructose-1-phosphate aldolase']\n",
      "\n",
      "KO KO:K00020 3-hydroxyisobutyrate dehydrogenase [EC:1.1.1.31]\n",
      "KO KO:K08318 4-hydroxybutyrate dehydrogenase / sulfolactaldehyde 3-reductase [EC:1.1.1.61 1.1.1.373]\n",
      "SSO SSO:000000342 3-sulpholactaldehyde reductase\n",
      "b3882 ['3-sulpholactaldehyde reductase']\n",
      "\n",
      "KO KO:K18478 sulfofructose kinase [EC:2.7.1.184]\n",
      "SSO SSO:000000443 6-deoxy-6-sulphofructose kinase\n",
      "b3883 ['6-deoxy-6-sulphofructose kinase']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02444 DeoR family transcriptional regulator, glycerol-3-phosphate regulon repressor\n",
      "SSO SSO:000002242 DeoR-type transcriptional regulator YihW\n",
      "b3884 ['DeoR-type transcriptional regulator YihW']\n",
      "\n",
      "KO KO:K20866 glucose-1-phosphatase [EC:3.1.3.10]\n",
      "SSO SSO:000029324 alpha-D-glucose 1-phosphate phosphatase YihX (EC 3.1.3.10)\n",
      "b3885 ['alpha-D-glucose 1-phosphate phosphatase YihX (EC 3.1.3.10)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K07058 membrane protein\n",
      "SSO SSO:000003884 Inner membrane protein YihY, formerly thought to be RNase BN\n",
      "b3886 ['Inner membrane protein YihY, formerly thought to be RNase BN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K07560 D-aminoacyl-tRNA deacylase [EC:3.1.1.96]\n",
      "SSO SSO:000002007 D-aminoacyl-tRNA deacylase (EC 3.1.1.96)\n",
      "b3887 ['D-aminoacyl-tRNA deacylase (EC 3.1.1.96)']\n",
      "\n",
      "SSO SSO:000026867 Uncharacterized GNAT family acetyltransferase YiiD\n",
      "b3888 ['Uncharacterized GNAT family acetyltransferase YiiD']\n",
      "\n",
      "SSO SSO:000028372 Uncharacterized protein YiiE\n",
      "b3889 ['Uncharacterized protein YiiE']\n",
      "\n",
      "SSO SSO:000028373 Uncharacterized protein YiiF\n",
      "b3890 ['Uncharacterized protein YiiF']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02380 FdhE protein\n",
      "SSO SSO:000009111 formate dehydrogenase formation protein FdhE\n",
      "b3891 ['Formate dehydrogenase formation protein FdhE']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00127 formate dehydrogenase subunit gamma\n",
      "SSO SSO:000002882 Formate dehydrogenase O gamma subunit (EC 1.2.1.2)\n",
      "b3892 ['Formate dehydrogenase O gamma subunit (EC 1.2.1.2)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00124 formate dehydrogenase iron-sulfur subunit\n",
      "SSO SSO:000002881 Formate dehydrogenase O beta subunit (EC 1.2.1.2)\n",
      "b3893 ['Formate dehydrogenase O beta subunit (EC 1.2.1.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K08348 formate dehydrogenase-N, alpha subunit [EC:1.17.5.3]\n",
      "b3894 ['Formate dehydrogenase O alpha subunit (EC 1.2.1.2)', 'selenocysteine-containing']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K02379 FdhD protein\n",
      "SSO SSO:000036897 Sulfur carrier protein FdhD\n",
      "b3895 ['Sulfur carrier protein FdhD']\n",
      "\n",
      "SSO SSO:000028374 Uncharacterized protein YiiG\n",
      "b3896 ['Uncharacterized protein YiiG']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K18531 putative frv operon regulatory protein\n",
      "SSO SSO:000023314 Putative frv operon regulatory protein\n",
      "b3897 ['Putative frv operon regulatory protein']\n",
      "\n",
      "GO GO:0000272 polysaccharide catabolic process\n",
      "KO KO:K18530 putative aminopeptidase FrvX [EC:3.4.11.-]\n",
      "SSO SSO:000023175 Putative aminopeptidase FrvX\n",
      "SSO SSO:000023495 Putative peptidase\n",
      "b3898 ['Putative aminopeptidase FrvX']\n",
      "\n",
      "b3899 ['PTS system, fructose-specific IIB component (EC 2.7.1.202)', 'PTS system, fructose-specific IIC component']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K11201 PTS system, fructose-specific IIA-like component [EC:2.7.1.-]\n",
      "SSO SSO:000005685 PTS system, fructose-specific IIA component (EC 2.7.1.69)\n",
      "b3900 ['PTS system, fructose-specific IIA component (EC 2.7.1.202)']\n",
      "\n",
      "KO KO:K03534 L-rhamnose mutarotase [EC:5.1.3.32]\n",
      "SSO SSO:000036545 L-rhamnose mutarotase (EC 5.1.3.32)\n",
      "b3901 ['L-rhamnose mutarotase (EC 5.1.3.32)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01629 rhamnulose-1-phosphate aldolase [EC:4.1.2.19]\n",
      "SSO SSO:000007040 Rhamnulose-1-phosphate aldolase (EC 4.1.2.19)\n",
      "b3902 ['Rhamnulose-1-phosphate aldolase (EC 4.1.2.19)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01813 L-rhamnose isomerase [EC:5.3.1.14]\n",
      "SSO SSO:000018777 L-rhamnose isomerase\n",
      "b3903 ['L-rhamnose isomerase (EC 5.3.1.14)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00848 rhamnulokinase [EC:2.7.1.5]\n",
      "SSO SSO:000024110 Rhamnulokinase (EC 2.7.1.5)\n",
      "b3904 ['Rhamnulokinase (EC 2.7.1.5)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02855 AraC family transcriptional regulator, L-rhamnose operon regulatory protein RhaS\n",
      "SSO SSO:000018778 L-rhamnose operon regulatory protein RhaS\n",
      "b3905 ['L-rhamnose operon regulatory protein RhaS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02854 AraC family transcriptional regulator, L-rhamnose operon transcriptional activator RhaR\n",
      "SSO SSO:000018779 L-rhamnose operon transcriptional activator RhaR\n",
      "b3906 ['L-rhamnose operon transcriptional activator RhaR']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02856 L-rhamnose-H+ transport protein\n",
      "SSO SSO:000018781 L-rhamnose-proton symporter\n",
      "b3907 ['L-rhamnose-proton symporter']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K04564 superoxide dismutase, Fe-Mn family [EC:1.15.1.1]\n",
      "SSO SSO:000007864 Superoxide dismutase [Mn] (EC 1.15.1.1)\n",
      "b3908 ['Superoxide dismutase [Mn] (EC 1.15.1.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02526 2-keto-3-deoxygluconate permease\n",
      "SSO SSO:000000175 2-keto-3-deoxygluconate permease (KDG permease)\n",
      "b3909 ['2-keto-3-deoxygluconate permease (KDG permease)']\n",
      "\n",
      "SSO SSO:000022736 Protein YiiM\n",
      "b3910 ['Protein YiiM']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07640 two-component system, OmpR family, sensor histidine kinase CpxA [EC:2.7.13.3]\n",
      "SSO SSO:000001770 Copper sensory histidine kinase CpxA\n",
      "b3911 ['Copper sensory histidine kinase CpxA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07662 two-component system, OmpR family, response regulator CpxR\n",
      "SSO SSO:000001775 Copper-sensing two-component system response regulator CpxR\n",
      "b3912 ['Copper-sensing two-component system response regulator CpxR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K06006 periplasmic protein CpxP/Spy\n",
      "SSO SSO:000021438 Periplasmic protein CpxP\n",
      "b4484 ['Periplasmic protein CpxP']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K13283 ferrous-iron efflux pump FieF\n",
      "SSO SSO:000016720 Ferrous-iron efflux pump FieF\n",
      "b3915 ['Ferrous-iron efflux pump FieF']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00850 6-phosphofructokinase 1 [EC:2.7.1.11]\n",
      "SSO SSO:000000453 6-phosphofructokinase (EC 2.7.1.11)\n",
      "b3916 ['6-phosphofructokinase (EC 2.7.1.11)']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K23163 Unknown\n",
      "b3917 ['Sulfate-binding protein Sbp']\n",
      "\n",
      "KO KO:K01521 CDP-diacylglycerol pyrophosphatase [EC:3.6.1.26]\n",
      "SSO SSO:000001300 CDP-diacylglycerol pyrophosphatase (EC 3.6.1.26)\n",
      "b3918 ['CDP-diacylglycerol pyrophosphatase (EC 3.6.1.26)']\n",
      "\n",
      "GO GO:0006096 glycolytic process\n",
      "KO KO:K01803 triosephosphate isomerase (TIM) [EC:5.3.1.1]\n",
      "SSO SSO:000008383 Triosephosphate isomerase (EC 5.3.1.1)\n",
      "b3919 ['Triosephosphate isomerase (EC 5.3.1.1)']\n",
      "\n",
      "SSO SSO:000028375 Uncharacterized protein YiiQ\n",
      "b3920 ['Uncharacterized protein YiiQ']\n",
      "\n",
      "SSO SSO:000028376 Uncharacterized protein YiiR\n",
      "b3921 ['Uncharacterized protein YiiR']\n",
      "\n",
      "SSO SSO:000026662 UPF0381 protein YiiS\n",
      "b3922 ['UPF0381 protein YiiS']\n",
      "\n",
      "GO GO:0006950 response to stress\n",
      "KO KO:K14065 universal stress protein D\n",
      "SSO SSO:000008825 Universal stress protein D\n",
      "b3923 ['Universal stress protein D']\n",
      "\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K00528 ferredoxin/flavodoxin---NADP+ reductase [EC:1.18.1.2 1.19.1.1]\n",
      "b3924 ['Ferredoxin--NADP(+) reductase (EC 1.18.1.2)', 'Flavodoxin--NADP(+) reductase (EC 1.19.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0008652 cellular amino acid biosynthetic process\n",
      "KO KO:K02446 fructose-1,6-bisphosphatase II [EC:3.1.3.11]\n",
      "SSO SSO:000002931 Fructose-1,6-bisphosphatase, GlpX type (EC 3.1.3.11)\n",
      "b3925 ['Fructose-1,6-bisphosphatase, GlpX type (EC 3.1.3.11)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "KO KO:K00864 glycerol kinase [EC:2.7.1.30]\n",
      "SSO SSO:000003248 Glycerol kinase (EC 2.7.1.30)\n",
      "b3926 ['Glycerol kinase (EC 2.7.1.30)']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02440 glycerol uptake facilitator protein\n",
      "SSO SSO:000003249 Glycerol uptake facilitator protein\n",
      "b3927 ['Glycerol uptake facilitator protein']\n",
      "\n",
      "KO KO:K09892 cell division protein ZapB\n",
      "SSO SSO:000001475 Cell division protein ZapB\n",
      "b3928 ['Cell division protein ZapB']\n",
      "\n",
      "KO KO:K02553 regulator of ribonuclease activity A\n",
      "SSO SSO:000007073 Ribonuclease E inhibitor RraA\n",
      "b3929 ['Ribonuclease E inhibitor RraA']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02548 1,4-dihydroxy-2-naphthoate polyprenyltransferase [EC:2.5.1.74]\n",
      "SSO SSO:000000035 1,4-dihydroxy-2-naphthoate polyprenyltransferase (EC 2.5.1.74)\n",
      "SSO SSO:000009468 1,4-dihydroxy-2-naphthoate octaprenyltransferase\n",
      "b3930 ['1,4-dihydroxy-2-naphthoate polyprenyltransferase (EC 2.5.1.74)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0006508 proteolysis\n",
      "GO GO:0008233 peptidase activity\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "KO KO:K03667 ATP-dependent HslUV protease ATP-binding subunit HslU\n",
      "SSO SSO:000000616 ATP-dependent hsl protease ATP-binding subunit HslU\n",
      "b3931 ['ATP-dependent hsl protease ATP-binding subunit HslU']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0006508 proteolysis\n",
      "GO GO:0008233 peptidase activity\n",
      "KO KO:K01419 ATP-dependent HslUV protease, peptidase subunit HslV [EC:3.4.25.2]\n",
      "SSO SSO:000000624 ATP-dependent protease subunit HslV (EC 3.4.25.2)\n",
      "b3932 ['ATP-dependent protease subunit HslV (EC 3.4.25.2)']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03591 cell division protein FtsN\n",
      "SSO SSO:000012035 Cell division protein FtsN\n",
      "b3933 ['Cell division protein FtsN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02529 LacI family transcriptional regulator\n",
      "KO KO:K05499 LacI family transcriptional regulator, repressor for deo operon, udp, cdd, tsx, nupC, and nupG\n",
      "SSO SSO:000008180 Transcriptional (co)regulator CytR\n",
      "b3934 ['Transcriptional (co)regulator CytR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K04066 primosomal protein N' (replication factor Y) (superfamily II helicase) [EC:3.6.4.-]\n",
      "SSO SSO:000003354 Helicase PriA essential for oriC/DnaA-independent DNA replication\n",
      "b3935 ['Helicase PriA essential for oriC/DnaA-independent DNA replication']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02909 large subunit ribosomal protein L31\n",
      "b3936 ['LSU ribosomal protein L31p', 'LSU ribosomal protein L31p, zinc-dependent']\n",
      "\n",
      "SSO SSO:000028377 Uncharacterized protein YiiX\n",
      "b3937 ['Uncharacterized protein YiiX']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03764 MetJ family transcriptional regulator, methionine regulon repressor\n",
      "SSO SSO:000004836 Methionine repressor MetJ\n",
      "b3938 ['Methionine repressor MetJ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "KO KO:K01739 cystathionine gamma-synthase [EC:2.5.1.48]\n",
      "SSO SSO:000001855 Cystathionine gamma-synthase (EC 2.5.1.48)\n",
      "b3939 ['Cystathionine gamma-synthase (EC 2.5.1.48)']\n",
      "\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "GO GO:0009090 homoserine biosynthetic process\n",
      "KO KO:K12525 bifunctional aspartokinase / homoserine dehydrogenase 2 [EC:2.7.2.4 1.1.1.3]\n",
      "b3940 ['Aspartokinase (EC 2.7.2.4)', 'Homoserine dehydrogenase (EC 1.1.1.3)']\n",
      "\n",
      "KO KO:K00297 methylenetetrahydrofolate reductase (NADPH) [EC:1.5.1.20]\n",
      "SSO SSO:000000401 5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)\n",
      "b3941 ['5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)']\n",
      "\n",
      "EC EC:1.11.1.13 Manganese peroxidase.\n",
      "EC EC:1.11.1.14 Lignin peroxidase.\n",
      "EC EC:1.11.1.16 Versatile peroxidase.\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "KO KO:K03782 catalase-peroxidase [EC:1.11.1.21]\n",
      "SSO SSO:000001448 Catalase-peroxidase KatG (EC 1.11.1.21)\n",
      "b3942 ['Catalase-peroxidase KatG (EC 1.11.1.21)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000027124 Uncharacterized inner membrane transporter YijE\n",
      "b3943 ['Uncharacterized inner membrane transporter YijE']\n",
      "\n",
      "KO KO:K09974 uncharacterized protein\n",
      "SSO SSO:000028378 Uncharacterized protein YijF\n",
      "b3944 ['Uncharacterized protein YijF']\n",
      "\n",
      "GO GO:0006071 glycerol metabolic process\n",
      "KO KO:K00005 glycerol dehydrogenase [EC:1.1.1.6]\n",
      "SSO SSO:000003247 Glycerol dehydrogenase (EC 1.1.1.6)\n",
      "b3945 ['Glycerol dehydrogenase (EC 1.1.1.6)']\n",
      "\n",
      "KO KO:K08314 fructose-6-phosphate aldolase 2 [EC:4.1.2.-]\n",
      "SSO SSO:000035974 Fructose-6-phosphate aldolase\n",
      "b3946 ['Fructose-6-phosphate aldolase']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K23991 Unknown\n",
      "SSO SSO:000005964 Phosphoenolpyruvate-protein phosphotransferase of PTS system (EC 2.7.3.9)\n",
      "b3947 ['Phosphoenolpyruvate-protein phosphotransferase of PTS system (EC 2.7.3.9)']\n",
      "\n",
      "KO KO:K11203 PTS system, fructose-specific IIC-like component\n",
      "SSO SSO:000021182 PTS system, fructose-like IIC component FrwC\n",
      "b3949 ['PTS system, fructose-like IIC component FrwC']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02769 PTS system, fructose-specific IIB component [EC:2.7.1.202]\n",
      "KO KO:K11202 PTS system, fructose-specific IIB-like component [EC:2.7.1.-]\n",
      "SSO SSO:000021179 PTS system, fructose-like IIB component FrwB (EC 2.7.1.69)\n",
      "b3950 ['PTS system, fructose-like IIB component FrwB']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00656 formate C-acetyltransferase [EC:2.3.1.54]\n",
      "SSO SSO:000006850 Pyruvate formate-lyase (EC 2.3.1.54)\n",
      "b3951 ['Pyruvate formate-lyase (EC 2.3.1.54)']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04069 pyruvate formate lyase activating enzyme [EC:1.97.1.4]\n",
      "SSO SSO:000006851 Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)\n",
      "b3952 ['Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K11202 PTS system, fructose-specific IIB-like component [EC:2.7.1.-]\n",
      "SSO SSO:000021180 PTS system, fructose-like IIB component FrwD (EC 2.7.1.69)\n",
      "b3953 ['PTS system, fructose-like IIB component FrwD']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "SSO SSO:000028759 Uncharacterized transcriptional regulator YijO, AraC family\n",
      "b3954 ['Uncharacterized transcriptional regulator YijO, AraC family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K19353 heptose-I-phosphate ethanolaminephosphotransferase [EC:2.7.8.-]\n",
      "SSO SSO:000038514 Phosphoethanolamine transferase EptC [E.coli], specific for LPS heptose I residue\n",
      "b3955 ['Phosphoethanolamine transferase EptC [E.coli], specific for LPS heptose I residue']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K01595 phosphoenolpyruvate carboxylase [EC:4.1.1.31]\n",
      "SSO SSO:000005955 Phosphoenolpyruvate carboxylase (EC 4.1.1.31)\n",
      "b3956 ['Phosphoenolpyruvate carboxylase (EC 4.1.1.31)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K01438 acetylornithine deacetylase [EC:3.5.1.16]\n",
      "SSO SSO:000000722 Acetylornithine deacetylase (EC 3.5.1.16)\n",
      "SSO SSO:000036987 N-acetylornithine deacetylase (EC 3.5.1.16)\n",
      "b3957 ['N-acetylornithine deacetylase (EC 3.5.1.16)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K00145 N-acetyl-gamma-glutamyl-phosphate reductase [EC:1.2.1.38]\n",
      "SSO SSO:000005097 N-acetyl-gamma-glutamyl-phosphate reductase (EC 1.2.1.38)\n",
      "b3958 ['N-acetyl-gamma-glutamyl-phosphate reductase (EC 1.2.1.38)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K00930 acetylglutamate kinase [EC:2.7.2.8]\n",
      "SSO SSO:000000720 Acetylglutamate kinase (EC 2.7.2.8)\n",
      "SSO SSO:000036092 N-acetylglutamate kinase (EC 2.7.2.8)\n",
      "b3959 ['N-acetylglutamate kinase (EC 2.7.2.8)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K01755 argininosuccinate lyase [EC:4.3.2.1]\n",
      "SSO SSO:000001051 Argininosuccinate lyase (EC 4.3.2.1)\n",
      "b3960 ['Argininosuccinate lyase (EC 4.3.2.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K04761 LysR family transcriptional regulator, hydrogen peroxide-inducible genes activator\n",
      "b3961 ['Hydrogen peroxide-inducible genes activator => OxyR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009226 nucleotide-sugar biosynthetic process\n",
      "KO KO:K00322 NAD(P) transhydrogenase [EC:1.6.1.1]\n",
      "SSO SSO:000007619 Soluble pyridine nucleotide transhydrogenase (EC 1.6.1.1)\n",
      "b3962 ['Soluble pyridine nucleotide transhydrogenase (EC 1.6.1.1)']\n",
      "\n",
      "KO KO:K22105 TetR/AcrR family transcriptional regulator, fatty acid biosynthesis regulator\n",
      "SSO SSO:000028860 Unsaturated fatty acid biosythesis repressor FabR, TetR family\n",
      "b3963 ['Unsaturated fatty acid biosythesis repressor FabR, TetR family']\n",
      "\n",
      "SSO SSO:000018414 Inner membrane protein YijD\n",
      "b3964 ['Inner membrane protein YijD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00557 tRNA (uracil-5-)-methyltransferase [EC:2.1.1.35]\n",
      "b3965 ['tRNA (uracil(54)-C5)-methyltransferase (EC 2.1.1.35)', 'tmRNA (uracil(341)-C5)-methyltransferase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "KO KO:K16092 vitamin B12 transporter\n",
      "SSO SSO:000020942 Outer membrane vitamin B12 receptor BtuB\n",
      "b3966 ['Outer membrane vitamin B12 receptor BtuB']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01776 glutamate racemase [EC:5.1.1.3]\n",
      "SSO SSO:000003153 Glutamate racemase (EC 5.1.1.3)\n",
      "b3967 ['Glutamate racemase (EC 5.1.1.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K00075 UDP-N-acetylmuramate dehydrogenase [EC:1.3.1.98]\n",
      "SSO SSO:000008614 UDP-N-acetylenolpyruvoylglucosamine reductase (EC 1.1.1.158)\n",
      "b3972 ['UDP-N-acetylenolpyruvoylglucosamine reductase (EC 1.3.1.98)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009102 biotin biosynthetic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03524 BirA family transcriptional regulator, biotin operon repressor / biotin---[acetyl-CoA-carboxylase] ligase [EC:6.3.4.15]\n",
      "b3973 ['Biotin operon repressor', 'Biotin--protein ligase (EC 6.3.4.9)(EC 6.3.4.10)(EC 6.3.4.11)(EC 6.3.4.15)']\n",
      "\n",
      "GO GO:0015937 coenzyme A biosynthetic process\n",
      "KO KO:K00867 type I pantothenate kinase [EC:2.7.1.33]\n",
      "SSO SSO:000005728 Pantothenate kinase (EC 2.7.1.33)\n",
      "b3974 ['Pantothenate kinase (EC 2.7.1.33)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "GO GO:0006970 response to osmotic stress\n",
      "KO KO:K02358 elongation factor Tu\n",
      "SSO SSO:000008307 Translation elongation factor Tu\n",
      "b3980 ['Translation elongation factor Tu']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03073 preprotein translocase subunit SecE\n",
      "SSO SSO:000006524 Protein translocase subunit SecE\n",
      "b3981 ['Protein translocase subunit SecE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K02601 transcriptional antiterminator NusG\n",
      "SSO SSO:000008138 Transcription antitermination protein NusG\n",
      "b3982 ['Transcription antitermination protein NusG']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02867 large subunit ribosomal protein L11\n",
      "SSO SSO:000004201 LSU ribosomal protein L11p (L12e)\n",
      "b3983 ['LSU ribosomal protein L11p (L12e)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "GO GO:0009386 translational attenuation\n",
      "KO KO:K02863 large subunit ribosomal protein L1\n",
      "SSO SSO:000004237 LSU ribosomal protein L1p (L10Ae)\n",
      "b3984 ['LSU ribosomal protein L1p (L10Ae)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02864 large subunit ribosomal protein L10\n",
      "SSO SSO:000004197 LSU ribosomal protein L10p (P0)\n",
      "b3985 ['LSU ribosomal protein L10p (P0)']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02935 large subunit ribosomal protein L7/L12\n",
      "SSO SSO:000036584 LSU ribosomal protein L7p/L12p (P1/P2)\n",
      "b3986 ['LSU ribosomal protein L7p/L12p (P1/P2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03043 DNA-directed RNA polymerase subunit beta [EC:2.7.7.6]\n",
      "SSO SSO:000002176 DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)\n",
      "b3987 ['DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03046 DNA-directed RNA polymerase subunit beta' [EC:2.7.7.6]\n",
      "SSO SSO:000002178 DNA-directed RNA polymerase beta' subunit (EC 2.7.7.6)\n",
      "b3988 [\"DNA-directed RNA polymerase beta' subunit (EC 2.7.7.6)\"]\n",
      "\n",
      "GO GO:0009266 response to temperature stimulus\n",
      "SSO SSO:000017650 Heat shock protein C\n",
      "b3989 ['Heat shock protein C']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K03150 2-iminoacetate synthase [EC:4.1.99.19]\n",
      "SSO SSO:000000167 2-iminoacetate synthase (ThiH) (EC 4.1.99.19)\n",
      "b3990 ['2-iminoacetate synthase (ThiH) (EC 4.1.99.19)']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K03149 thiazole synthase [EC:2.8.1.10]\n",
      "SSO SSO:000008019 Thiazole synthase (EC 2.8.1.10)\n",
      "b3991 ['Thiazole synthase (EC 2.8.1.10)']\n",
      "\n",
      "KO KO:K03154 sulfur carrier protein\n",
      "SSO SSO:000007837 Sulfur carrier protein ThiS\n",
      "b4407 ['Sulfur carrier protein ThiS']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K03148 sulfur carrier protein ThiS adenylyltransferase [EC:2.7.7.73]\n",
      "KO KO:K21029 molybdopterin-synthase adenylyltransferase [EC:2.7.7.80]\n",
      "SSO SSO:000007838 Sulfur carrier protein ThiS adenylyltransferase (EC 2.7.7.73)\n",
      "b3992 ['Sulfur carrier protein ThiS adenylyltransferase (EC 2.7.7.73)']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K00788 thiamine-phosphate pyrophosphorylase [EC:2.5.1.3]\n",
      "SSO SSO:000008001 Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)\n",
      "b3993 ['Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)']\n",
      "\n",
      "GO GO:0009228 thiamine biosynthetic process\n",
      "KO KO:K03147 phosphomethylpyrimidine synthase [EC:4.1.99.17]\n",
      "SSO SSO:000038542 Phosphomethylpyrimidine synthase ThiC (EC 4.1.99.17)\n",
      "b3994 ['Phosphomethylpyrimidine synthase ThiC (EC 4.1.99.17)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K07740 regulator of sigma D\n",
      "SSO SSO:000023941 Regulator of sigma D\n",
      "b3995 ['Regulator of sigma D']\n",
      "\n",
      "KO KO:K03426 NAD+ diphosphatase [EC:3.6.1.22]\n",
      "SSO SSO:000005214 NADH pyrophosphatase (EC 3.6.1.22)\n",
      "SSO SSO:000037077 NADH pyrophosphatase (EC 3.6.1.22), decaps 5'-NAD modified RNA\n",
      "b3996 [\"NADH pyrophosphatase (EC 3.6.1.22), decaps 5'-NAD modified RNA\"]\n",
      "\n",
      "KO KO:K01599 uroporphyrinogen decarboxylase [EC:4.1.1.37]\n",
      "SSO SSO:000008885 Uroporphyrinogen III decarboxylase (EC 4.1.1.37)\n",
      "SSO SSO:000028881 Uroporphyrinogen decarboxylase\n",
      "b3997 ['Uroporphyrinogen III decarboxylase (EC 4.1.1.37)']\n",
      "\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K05982 deoxyribonuclease V [EC:3.1.21.7]\n",
      "SSO SSO:000002475 Endonuclease V (EC 3.1.21.7)\n",
      "b3998 ['Endonuclease V (EC 3.1.21.7)']\n",
      "\n",
      "KO KO:K09891 uncharacterized protein\n",
      "SSO SSO:000028383 Uncharacterized protein YjaG\n",
      "b3999 ['Uncharacterized protein YjaG']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K03530 DNA-binding protein HU-beta\n",
      "KO KO:K05787 DNA-binding protein HU-alpha\n",
      "SSO SSO:000002138 DNA-binding protein HU-alpha\n",
      "b4000 ['DNA-binding protein HU-alpha']\n",
      "\n",
      "SSO SSO:000028384 Uncharacterized protein YjaH\n",
      "b4001 ['Uncharacterized protein YjaH']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K07803 zinc resistance-associated protein\n",
      "SSO SSO:000041749 Zinc resistance-associated protein ZraP\n",
      "b4002 ['Zinc resistance-associated protein ZraP']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07709 two-component system, NtrC family, sensor histidine kinase HydH [EC:2.7.13.3]\n",
      "SSO SSO:000007410 Sensor protein of zinc sigma-54-dependent two-component system\n",
      "b4003 ['Sensor protein of zinc sigma-54-dependent two-component system']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07713 two-component system, NtrC family, response regulator HydG\n",
      "KO KO:K07714 two-component system, NtrC family, response regulator AtoC\n",
      "SSO SSO:000007036 Response regulator of zinc sigma-54-dependent two-component system\n",
      "b4004 ['Response regulator of zinc sigma-54-dependent two-component system']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "KO KO:K01945 phosphoribosylamine---glycine ligase [EC:6.3.4.13]\n",
      "SSO SSO:000006036 Phosphoribosylamine--glycine ligase (EC 6.3.4.13)\n",
      "b4005 ['Phosphoribosylamine--glycine ligase (EC 6.3.4.13)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00602 phosphoribosylaminoimidazolecarboxamide formyltransferase / IMP cyclohydrolase [EC:2.1.2.3 3.5.4.10]\n",
      "b4006 ['IMP cyclohydrolase (EC 3.5.4.10)', 'Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)']\n",
      "\n",
      "SSO SSO:000028382 Uncharacterized protein YjaA\n",
      "b4011 ['Uncharacterized protein YjaA']\n",
      "\n",
      "KO KO:K03827 putative acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000026960 Uncharacterized N-acetyltransferase YjaB\n",
      "b4012 ['Uncharacterized N-acetyltransferase YjaB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "KO KO:K00651 homoserine O-succinyltransferase/O-acetyltransferase [EC:2.3.1.46 2.3.1.31]\n",
      "SSO SSO:000003484 Homoserine O-succinyltransferase (EC 2.3.1.46)\n",
      "b4013 ['Homoserine O-succinyltransferase (EC 2.3.1.46)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006097 glyoxylate cycle\n",
      "KO KO:K01638 malate synthase [EC:2.3.3.9]\n",
      "SSO SSO:000004672 Malate synthase (EC 2.3.3.9)\n",
      "b4014 ['Malate synthase (EC 2.3.3.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006097 glyoxylate cycle\n",
      "KO KO:K01637 isocitrate lyase [EC:4.1.3.1]\n",
      "KO KO:K01638 malate synthase [EC:2.3.3.9]\n",
      "SSO SSO:000004053 Isocitrate lyase (EC 4.1.3.1)\n",
      "b4015 ['Isocitrate lyase (EC 4.1.3.1)']\n",
      "\n",
      "GO GO:0006097 glyoxylate cycle\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K00906 isocitrate dehydrogenase kinase/phosphatase [EC:2.7.11.5 3.1.3.-]\n",
      "SSO SSO:000004052 Isocitrate dehydrogenase phosphatase /kinase (EC 3.1.3.-) (EC 2.7.11.5)\n",
      "b4016 ['Isocitrate dehydrogenase phosphatase (EC 2.7.11.5)/kinase (EC 3.1.3.-)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "SSO SSO:000010666 Ankyrin repeat protein\n",
      "SSO SSO:000010674 Ankyrin-repeat protein A\n",
      "b4017 ['Ankyrin-repeat protein A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006097 glyoxylate cycle\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K13641 IclR family transcriptional regulator, acetate operon repressor\n",
      "SSO SSO:000010150 Acetate operon repressor IclR\n",
      "SSO SSO:000025979 Transcriptional repressor\n",
      "b4018 ['Acetate operon repressor IclR']\n",
      "\n",
      "GO GO:0006556 S-adenosylmethionine biosynthetic process\n",
      "GO GO:0009086 methionine biosynthetic process\n",
      "KO KO:K00548 5-methyltetrahydrofolate--homocysteine methyltransferase [EC:2.1.1.13]\n",
      "SSO SSO:000000428 5-methyltetrahydrofolate--homocysteine methyltransferase (EC 2.1.1.13)\n",
      "b4019 ['5-methyltetrahydrofolate--homocysteine methyltransferase (EC 2.1.1.13)']\n",
      "\n",
      "KO KO:K03324 phosphate:Na+ symporter\n",
      "SSO SSO:000007601 Sodium-dependent phosphate transporter\n",
      "b4020 ['Sodium-dependent phosphate transporter']\n",
      "\n",
      "KO KO:K05995 dipeptidase E [EC:3.4.13.21]\n",
      "SSO SSO:000000918 Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)\n",
      "SSO SSO:000012530 Peptidase E (alpha-aspartyl dipeptidase)\n",
      "b4021 ['Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)']\n",
      "\n",
      "KO KO:K06182 23S rRNA pseudouridine2604 synthase [EC:5.4.99.21]\n",
      "SSO SSO:000036577 LSU rRNA pseudouridine(2604) synthase (EC 5.4.99.21)\n",
      "SSO SSO:000043591 23S rRNA pseudouridine2604 synthase (EC 5.4.99.21)\n",
      "b4022 ['LSU rRNA pseudouridine(2604) synthase (EC 5.4.99.21)']\n",
      "\n",
      "SSO SSO:000028387 Uncharacterized protein YjbD\n",
      "b4023 ['Uncharacterized protein YjbD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0009090 homoserine biosynthetic process\n",
      "KO KO:K00928 aspartate kinase [EC:2.7.2.4]\n",
      "SSO SSO:000001104 Aspartokinase (EC 2.7.2.4)\n",
      "b4024 ['Aspartokinase (EC 2.7.2.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006096 glycolytic process\n",
      "KO KO:K01810 glucose-6-phosphate isomerase [EC:5.3.1.9]\n",
      "SSO SSO:000003129 Glucose-6-phosphate isomerase (EC 5.3.1.9)\n",
      "b4025 ['Glucose-6-phosphate isomerase (EC 5.3.1.9)']\n",
      "\n",
      "b4026 ['Uncharacterized protein YjbE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "b4027 ['Uncharacterized lipoprotein YjbF']\n",
      "\n",
      "b4028 ['Uncharacterized protein YjbG']\n",
      "\n",
      "b4029 ['Uncharacterized lipoprotein YjbH']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "b4620 ['hypothetical protein']\n",
      "\n",
      "KO KO:K13256 protein PsiE\n",
      "b4030 ['PsiE protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015753 D-xylose transport\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K08138 MFS transporter, SP family, xylose:H+ symportor\n",
      "SSO SSO:000012717 D-xylose proton-symporter XylE\n",
      "b4031 ['D-xylose proton-symporter XylE']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10110 maltose/maltodextrin transport system permease protein\n",
      "SSO SSO:000019305 Maltodextrin ABC transporter, permease protein MdxG\n",
      "b4032 ['Maltodextrin ABC transporter, permease protein MdxG']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10109 maltose/maltodextrin transport system permease protein\n",
      "KO KO:K15771 arabinogalactan oligomer / maltooligosaccharide transport system permease protein\n",
      "SSO SSO:000019304 Maltodextrin ABC transporter, permease protein MdxF\n",
      "b4033 ['Maltodextrin ABC transporter, permease protein MdxF']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02027 multiple sugar transport system substrate-binding protein\n",
      "KO KO:K10108 maltose/maltodextrin transport system substrate-binding protein\n",
      "KO KO:K15770 arabinogalactan oligomer / maltooligosaccharide transport system substrate-binding protein\n",
      "SSO SSO:000019306 Maltodextrin ABC transporter, substrate-binding protein MdxE\n",
      "b4034 ['Maltodextrin ABC transporter, substrate-binding protein MdxE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K10111 multiple sugar transport system ATP-binding protein [EC:3.6.3.-]\n",
      "SSO SSO:000019316 Maltose/maltodextrin transport ATP-binding protein MalK (EC 3.6.3.19)\n",
      "b4035 ['Maltose/maltodextrin transport ATP-binding protein MalK (EC 3.6.3.19)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02024 maltoporin\n",
      "SSO SSO:000019309 Maltoporin (maltose/maltodextrin high-affinity receptor, phage lambda receptor protein)\n",
      "b4036 ['Maltoporin (maltose/maltodextrin high-affinity receptor, phage lambda receptor protein)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K05775 maltose operon periplasmic protein\n",
      "SSO SSO:000019311 Maltose operon periplasmic protein MalM\n",
      "b4037 ['Maltose operon periplasmic protein MalM']\n",
      "\n",
      "SSO SSO:000029153 YjbI protein\n",
      "b4038 ['YjbI protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009060 aerobic respiration\n",
      "KO KO:K03181 chorismate--pyruvate lyase [EC:4.1.3.40]\n",
      "SSO SSO:000001559 Chorismate--pyruvate lyase (EC 4.1.3.40)\n",
      "b4039 ['Chorismate--pyruvate lyase (EC 4.1.3.40)']\n",
      "\n",
      "GO GO:0009060 aerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03179 4-hydroxybenzoate polyprenyltransferase [EC:2.5.1.39]\n",
      "SSO SSO:000000367 4-hydroxybenzoate polyprenyltransferase (EC 2.5.1.39)\n",
      "b4040 ['4-hydroxybenzoate polyprenyltransferase (EC 2.5.1.39)']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00631 glycerol-3-phosphate O-acyltransferase [EC:2.3.1.15]\n",
      "SSO SSO:000003256 Glycerol-3-phosphate acyltransferase (EC 2.3.1.15)\n",
      "SSO SSO:000017325 Glycerol-3-phosphate O-acyltransferase\n",
      "b4041 ['Glycerol-3-phosphate acyltransferase (EC 2.3.1.15)']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000002270 Diacylglycerol kinase (EC 2.7.1.107)\n",
      "b4042 ['Diacylglycerol kinase (EC 2.7.1.107)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009432 SOS response\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K01356 repressor LexA [EC:3.4.21.88]\n",
      "SSO SSO:000007226 SOS-response repressor and protease LexA (EC 3.4.21.88)\n",
      "b4043 ['SOS-response repressor and protease LexA (EC 3.4.21.88)']\n",
      "\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009314 response to radiation\n",
      "GO GO:0009432 SOS response\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03327 multidrug resistance protein, MATE family\n",
      "SSO SSO:000012935 DNA-damage-inducible protein F\n",
      "b4044 ['DNA-damage-inducible protein F']\n",
      "\n",
      "SSO SSO:000026645 UPF0337 protein YjbJ\n",
      "b4045 ['UPF0337 protein YjbJ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K09823 Fur family transcriptional regulator, zinc uptake regulator\n",
      "SSO SSO:000008999 Zinc uptake regulation protein Zur\n",
      "b4046 ['Zinc uptake regulation protein Zur']\n",
      "\n",
      "SSO SSO:000028388 Uncharacterized protein YjbL\n",
      "b4047 ['Uncharacterized protein YjbL']\n",
      "\n",
      "SSO SSO:000028389 Uncharacterized protein YjbM\n",
      "b4048 ['Uncharacterized protein YjbM']\n",
      "\n",
      "KO KO:K05539 tRNA-dihydrouridine synthase A [EC:1.-.-.-]\n",
      "SSO SSO:000009356 tRNA dihydrouridine synthase A (EC 1.-.-.-)\n",
      "SSO SSO:000033459 tRNA-dihydrouridine(20/20a) synthase (EC 1.3.1.91)\n",
      "b4049 ['tRNA-dihydrouridine(20/20a) synthase (EC 1.3.1.91)']\n",
      "\n",
      "SSO SSO:000021546 Phage shock protein G\n",
      "b4050 ['Phage shock protein G']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "KO KO:K00344 NADPH2:quinone reductase [EC:1.6.5.5]\n",
      "SSO SSO:000006879 Quinone oxidoreductase (EC 1.6.5.5)\n",
      "b4051 ['Quinone oxidoreductase (EC 1.6.5.5)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02314 replicative DNA helicase [EC:3.6.4.12]\n",
      "SSO SSO:000007005 Replicative DNA helicase (DnaB) (EC 3.6.4.12)\n",
      "SSO SSO:000024013 Replicative DNA helicase\n",
      "b4052 ['Replicative DNA helicase (DnaB) (EC 3.6.4.12)']\n",
      "\n",
      "GO GO:0006523 alanine biosynthetic process\n",
      "KO KO:K01775 alanine racemase [EC:5.1.1.1]\n",
      "SSO SSO:000000848 Alanine racemase (EC 5.1.1.1)\n",
      "b4053 ['Alanine racemase (EC 5.1.1.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006571 tyrosine biosynthetic process\n",
      "GO GO:0009094 L-phenylalanine biosynthetic process\n",
      "GO GO:0009098 leucine biosynthetic process\n",
      "KO KO:K00832 aromatic-amino-acid transaminase [EC:2.6.1.57]\n",
      "SSO SSO:000001220 Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)\n",
      "b4054 ['Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)']\n",
      "\n",
      "b4621 ['FIG00638711: hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K03788 acid phosphatase (class B) [EC:3.1.3.2]\n",
      "b4055 ['NMN phosphatase (EC 3.1.3.5)', 'Class B acid phosphatase precursor (EC 3.1.3.2)']\n",
      "\n",
      "SSO SSO:000008658 UPF0047 protein YjbQ\n",
      "b4056 ['UPF0047 protein YjbQ']\n",
      "\n",
      "SSO SSO:000028390 Uncharacterized protein YjbR\n",
      "b4057 ['Uncharacterized protein YjbR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "GO GO:0009314 response to radiation\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03701 excinuclease ABC subunit A\n",
      "SSO SSO:000002607 Excinuclease ABC subunit A\n",
      "b4058 ['Excinuclease ABC subunit A']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K03111 single-strand DNA-binding protein\n",
      "SSO SSO:000007565 Single-stranded DNA-binding protein\n",
      "b4059 ['Single-stranded DNA-binding protein']\n",
      "\n",
      "SSO SSO:000028391 Uncharacterized protein YjcB\n",
      "b4060 ['Uncharacterized protein YjcB']\n",
      "\n",
      "KO KO:K21090 c-di-GMP phosphodiesterase [EC:3.1.4.52]\n",
      "b4061 ['c-di-GMP phosphodiesterase (EC 3.1.4.52) => PdeC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K13631 AraC family transcriptional regulator, mar-sox-rob regulon activator\n",
      "SSO SSO:000002147 DNA-binding transcriptional dual regulator SoxS\n",
      "b4062 ['DNA-binding transcriptional dual regulator SoxS']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006805 xenobiotic metabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K13639 MerR family transcriptional regulator, redox-sensitive transcriptional activator SoxR\n",
      "SSO SSO:000006963 Redox-sensitive transcriptional activator SoxR\n",
      "b4063 ['Redox-sensitive transcriptional activator SoxR']\n",
      "\n",
      "KO KO:K06901 putative MFS transporter, AGZA family, xanthine/uracil permease\n",
      "SSO SSO:000017488 Guanine/hypoxanthine permease GhxP\n",
      "b4064 ['Guanine/hypoxanthine permease GhxP']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03316 monovalent cation:H+ antiporter, CPA1 family\n",
      "SSO SSO:000026976 Uncharacterized Na(+)/H(+) exchanger YjcE\n",
      "b4065 ['Uncharacterized Na(+)/H(+) exchanger YjcE']\n",
      "\n",
      "SSO SSO:000028392 Uncharacterized protein YjcF\n",
      "b4066 ['Uncharacterized protein YjcF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K14393 cation/acetate symporter\n",
      "SSO SSO:000000671 Acetate permease ActP (cation/acetate symporter)\n",
      "b4067 ['Acetate permease ActP (cation/acetate symporter)']\n",
      "\n",
      "SSO SSO:000003885 Inner membrane protein YjcH, clustering with ActP\n",
      "b4068 ['Inner membrane protein YjcH, clustering with ActP']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0045733 acetate catabolic process\n",
      "KO KO:K01895 acetyl-CoA synthetase [EC:6.2.1.1]\n",
      "SSO SSO:000000713 Acetyl-CoA synthetase (EC 6.2.1.1)\n",
      "b4069 ['Acetyl-CoA synthetase (EC 6.2.1.1)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K03385 nitrite reductase (cytochrome c-552) [EC:1.7.2.2]\n",
      "SSO SSO:000012636 Cytochrome c-552 precursor\n",
      "b4070 ['Cytochrome c552 precursor (EC 1.7.2.2)']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K04013 cytochrome c-type protein NrfB\n",
      "b4071 ['Cytochrome c-type protein NrfB precursor']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04014 protein NrfC\n",
      "b4072 ['NrfC protein']\n",
      "\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04015 protein NrfD\n",
      "SSO SSO:000005415 NrfD protein\n",
      "b4073 ['NrfD protein']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04016 cytochrome c-type biogenesis protein NrfE\n",
      "b4074 ['Cytochrome c-type heme lyase subunit nrfE, nitrite reductase complex assembly']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K02200 cytochrome c-type biogenesis protein CcmH\n",
      "KO KO:K04017 formate-dependent nitrite reductase complex subunit NrfF\n",
      "b4075 ['Formate-dependent nitrite reductase complex subunit NrfF']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K04018 formate-dependent nitrite reductase complex subunit NrfG\n",
      "b4076 ['Formate-dependent nitrite reductase complex subunit NrfG']\n",
      "\n",
      "GO GO:0006532 aspartate biosynthetic process\n",
      "GO GO:0006537 glutamate biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11102 proton glutamate symport protein\n",
      "b4077 ['Proton/glutamate symporter', 'Proton/aspartate symporter']\n",
      "\n",
      "KO KO:K07126 uncharacterized protein\n",
      "SSO SSO:000028393 Uncharacterized protein YjcO\n",
      "b4078 ['Uncharacterized protein YjcO']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "KO KO:K22015 formate dehydrogenase (acceptor) [EC:1.17.99.7]\n",
      "b4079 ['Formate dehydrogenase H (EC 1.2.1.2)', 'selenocysteine-containing']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K15550 outer membrane protein, multidrug efflux system\n",
      "SSO SSO:000020098 Multidrug resistance outer membrane protein MdtP\n",
      "b4080 ['Multidrug resistance outer membrane protein MdtP']\n",
      "\n",
      "KO KO:K15547 multidrug resistance protein MdtO\n",
      "SSO SSO:000019415 Membrane fusion component MdtO of tripartite multidrug resistance system\n",
      "b4081 ['Membrane fusion component MdtO of tripartite multidrug resistance system']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K15549 membrane fusion protein, multidrug efflux system\n",
      "SSO SSO:000018372 Inner membrane component MdtN of tripartite multidrug resistance system\n",
      "b4082 ['Inner membrane component MdtN of tripartite multidrug resistance system']\n",
      "\n",
      "KO KO:K21695 protein AaeX\n",
      "SSO SSO:000028548 Uncharacterized protein YtcA\n",
      "b4622 ['Uncharacterized protein YtcA']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "SSO SSO:000019533 Metallo-beta-lactamase superfamily protein\n",
      "SSO SSO:000023151 Putative alkyl/aryl-sulfatase YjcS\n",
      "b4083 ['Putative alkyl/aryl-sulfatase YjcS']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K00881 allose kinase [EC:2.7.1.55]\n",
      "SSO SSO:000002000 D-allose kinase (EC 2.7.1.55)\n",
      "b4084 ['D-allose kinase (EC 2.7.1.55)']\n",
      "\n",
      "KO KO:K17195 D-allulose-6-phosphate 3-epimerase [EC:5.1.3.-]\n",
      "SSO SSO:000002002 D-allulose-6-phosphate 3-epimerase (EC 5.1.3.-)\n",
      "b4085 ['D-allulose-6-phosphate 3-epimerase (EC 5.1.3.-)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K10550 D-allose transport system permease protein\n",
      "SSO SSO:000001998 D-allose ABC transporter, permease protein\n",
      "b4086 ['D-allose ABC transporter, permease protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10551 D-allose transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000001997 D-allose ABC transporter, ATP-binding protein\n",
      "b4087 ['D-allose ABC transporter, ATP-binding protein']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10439 ribose transport system substrate-binding protein\n",
      "KO KO:K10549 D-allose transport system substrate-binding protein\n",
      "SSO SSO:000001999 D-allose ABC transporter, substrate-binding protein\n",
      "b4088 ['D-allose ABC transporter, substrate-binding protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K23238 Unknown\n",
      "SSO SSO:000039712 Transcriptional regulator of D-allose utilization RpiR\n",
      "b4089 ['Transcriptional regulator of D-allose utilization RpiR']\n",
      "\n",
      "GO GO:0009052 pentose-phosphate shunt, non-oxidative branch\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01808 ribose 5-phosphate isomerase B [EC:5.3.1.6]\n",
      "b4090 ['D-allose-6-phosphate isomerase (EC 5.3.1.-)', 'Ribose-5-phosphate isomerase B (EC 5.3.1.6)']\n",
      "\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "SSO SSO:000041454 Uncharacterized protein YjdP\n",
      "b4487 ['Uncharacterized protein YjdP']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K06167 phosphoribosyl 1,2-cyclic phosphate phosphodiesterase [EC:3.1.4.55]\n",
      "SSO SSO:000006032 Phosphoribosyl 1,2-cyclic phosphodiesterase (EC 3.1.4.55)\n",
      "b4092 ['Phosphoribosyl 1,2-cyclic phosphodiesterase (EC 3.1.4.55)']\n",
      "\n",
      "KO KO:K09994 (aminoalkyl)phosphonate N-acetyltransferase [EC:2.3.1.280]\n",
      "SSO SSO:000006515 Protein phnO\n",
      "b4093 ['Protein PhnO']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K05774 ribose 1,5-bisphosphokinase [EC:2.7.4.23]\n",
      "SSO SSO:000007109 Ribose 1,5-bisphosphate phosphokinase PhnN (EC 2.7.4.23)\n",
      "b4094 ['Ribose 1,5-bisphosphate phosphokinase PhnN (EC 2.7.4.23)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06162 alpha-D-ribose 1-methylphosphonate 5-triphosphate diphosphatase [EC:3.6.1.63]\n",
      "SSO SSO:000000908 Alpha-D-ribose 1-methylphosphonate 5-triphosphate diphosphatase (EC 3.6.1.63)\n",
      "b4095 ['Alpha-D-ribose 1-methylphosphonate 5-triphosphate diphosphatase (EC 3.6.1.63)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K05780 alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnL [EC:2.7.8.37]\n",
      "SSO SSO:000000912 Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnL (EC 2.7.8.37)\n",
      "b4096 ['Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnL (EC 2.7.8.37)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K05781 putative phosphonate transport system ATP-binding protein\n",
      "SSO SSO:000006009 Phosphonates utilization ATP-binding protein PhnK\n",
      "b4097 ['Phosphonates utilization ATP-binding protein PhnK']\n",
      "\n",
      "KO KO:K06163 alpha-D-ribose 1-methylphosphonate 5-phosphate C-P lyase [EC:4.7.1.1]\n",
      "SSO SSO:000000907 Alpha-D-ribose 1-methylphosphonate 5-phosphate C-P lyase (EC 4.7.1.1)\n",
      "b4098 ['Alpha-D-ribose 1-methylphosphonate 5-phosphate C-P lyase (EC 4.7.1.1)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K06164 alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnI [EC:2.7.8.37]\n",
      "SSO SSO:000000911 Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnI (EC 2.7.8.37)\n",
      "b4099 ['Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnI (EC 2.7.8.37)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K06165 alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnH [EC:2.7.8.37]\n",
      "SSO SSO:000000910 Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnH (EC 2.7.8.37)\n",
      "b4100 ['Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnH (EC 2.7.8.37)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K06165 alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnH [EC:2.7.8.37]\n",
      "KO KO:K06166 alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnG [EC:2.7.8.37]\n",
      "SSO SSO:000000909 Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnG (EC 2.7.8.37)\n",
      "b4101 ['Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnG (EC 2.7.8.37)']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K02043 GntR family transcriptional regulator, phosphonate transport system regulatory protein\n",
      "SSO SSO:000008209 Transcriptional regulator PhnF\n",
      "b4102 ['Transcriptional regulator PhnF']\n",
      "\n",
      "KO KO:K02042 phosphonate transport system permease protein\n",
      "SSO SSO:000005999 Phosphonate ABC transporter permease protein PhnE (TC 3.A.1.9.1)\n",
      "b4104 ['Phosphonate ABC transporter permease protein PhnE (TC 3.A.1.9.1)']\n",
      "\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02044 phosphonate transport system substrate-binding protein\n",
      "SSO SSO:000006004 Phosphonate ABC transporter substrate-binding protein PhnD (TC 3.A.1.9.1)\n",
      "b4105 ['Phosphonate ABC transporter substrate-binding protein PhnD (TC 3.A.1.9.1)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K02041 phosphonate transport system ATP-binding protein [EC:7.3.2.2]\n",
      "KO KO:K02042 phosphonate transport system permease protein\n",
      "SSO SSO:000005996 Phosphonate ABC transporter ATP-binding protein PhnC (TC 3.A.1.9.1)\n",
      "b4106 ['Phosphonate ABC transporter ATP-binding protein PhnC (TC 3.A.1.9.1)']\n",
      "\n",
      "KO KO:K04750 PhnB protein\n",
      "SSO SSO:000022711 Protein PhnB\n",
      "b4107 ['Protein PhnB']\n",
      "\n",
      "KO KO:K06193 protein PhnA\n",
      "SSO SSO:000022710 Protein PhnA\n",
      "b4108 ['Protein PhnA']\n",
      "\n",
      "SSO SSO:000012254 Clamp-binding protein CrfC\n",
      "b4109 ['Clamp-binding protein CrfC']\n",
      "\n",
      "SSO SSO:000028394 Uncharacterized protein YjcZ\n",
      "b4110 ['Uncharacterized protein YjcZ']\n",
      "\n",
      "GO GO:0006561 proline biosynthetic process\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03762 MFS transporter, MHS family, proline/betaine transporter\n",
      "SSO SSO:000004114 L-Proline/Glycine betaine transporter ProP\n",
      "b4111 ['L-Proline/Glycine betaine transporter ProP']\n",
      "\n",
      "SSO SSO:000023415 Putative membrane protein PmrR\n",
      "b4703 ['Putative membrane protein PmrR']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07643 two-component system, OmpR family, sensor histidine kinase BasS [EC:2.7.13.3]\n",
      "SSO SSO:000039133 Sensor protein BasS/PmrB (activates BasR/PmrA)\n",
      "b4112 ['Sensor protein BasS/PmrB (activates BasR/PmrA)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K02483 two-component system, OmpR family, response regulator\n",
      "KO KO:K07771 two-component system, OmpR family, response regulator BasR\n",
      "SSO SSO:000039834 Two-component transcriptional regulatory protein BasR/PmrA (activated by BasS/PmrB)\n",
      "b4113 ['Two-component transcriptional regulatory protein BasR/PmrA (activated by BasS/PmrB)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K03760 lipid A ethanolaminephosphotransferase [EC:2.7.8.43]\n",
      "SSO SSO:000036632 Lipid A phosphoethanolamine transferase EptA/PmrC (EC 2.7.8.43)\n",
      "b4114 ['Lipid A phosphoethanolamine transferase EptA/PmrC (EC 2.7.8.43)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015496 putrescine:ornithine antiporter activity\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03759 arginine:agmatine antiporter\n",
      "SSO SSO:000001048 Arginine/agmatine antiporter\n",
      "b4115 ['Arginine/agmatine antiporter']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03755 AraC family transcriptional regulator, transcriptional activator of adiA\n",
      "SSO SSO:000008240 Transcriptional regulator of catabolic arginine decarboxylase (adiA)\n",
      "b4116 ['Transcriptional regulator of catabolic arginine decarboxylase (adiA)']\n",
      "\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "KO KO:K01584 arginine decarboxylase [EC:4.1.1.19]\n",
      "SSO SSO:000001044 Arginine decarboxylase (EC 4.1.1.19)\n",
      "SSO SSO:000001045 Arginine decarboxylase, catabolic (EC 4.1.1.19)\n",
      "b4117 ['Arginine decarboxylase, catabolic (EC 4.1.1.19)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K23237 Unknown\n",
      "SSO SSO:000019401 Melibiose operon regulatory protein MelR, AraC family\n",
      "b4118 ['Melibiose operon regulatory protein MelR, AraC family']\n",
      "\n",
      "EC EC:3.2.1.122 Maltose-6'-phosphate glucosidase.\n",
      "EC EC:3.2.1.139 Alpha-glucuronidase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.22 Alpha-galactosidase.\n",
      "EC EC:3.2.1.67 Galacturan 1,4-alpha-galacturonidase.\n",
      "EC EC:3.2.1.86 6-phospho-beta-glucosidase.\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K07406 alpha-galactosidase [EC:3.2.1.22]\n",
      "SSO SSO:000000919 Alpha-galactosidase (EC 3.2.1.22)\n",
      "b4119 ['alpha-galactosidase (EC 3.2.1.22)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03292 glycoside/pentoside/hexuronide:cation symporter, GPH family\n",
      "KO KO:K11104 melibiose permease\n",
      "SSO SSO:000019400 Melibiose carrier protein, Na+/melibiose symporter\n",
      "b4120 ['Melibiose carrier protein, Na+/melibiose symporter']\n",
      "\n",
      "KO KO:K08984 putative membrane protein\n",
      "SSO SSO:000018415 Inner membrane protein YjdF\n",
      "b4121 ['Inner membrane protein YjdF']\n",
      "\n",
      "GO GO:0006099 tricarboxylic acid cycle\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K01676 fumarate hydratase, class I [EC:4.2.1.2]\n",
      "SSO SSO:000002951 Fumarate hydratase class I, anaerobic (EC 4.2.1.2)\n",
      "b4122 ['Fumarate hydratase class I, anaerobic (EC 4.2.1.2)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07792 anaerobic C4-dicarboxylate transporter DcuB\n",
      "SSO SSO:000010641 Anaerobic C4-dicarboxylate transporter DcuB\n",
      "b4123 ['Anaerobic C4-dicarboxylate transporter DcuB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K07703 two-component system, CitB family, response regulator DcuR\n",
      "SSO SSO:000016916 Fumarate respiration two-component response regulator DcuR\n",
      "b4124 ['Fumarate respiration two-component response regulator DcuR']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K07701 two-component system, CitB family, sensor histidine kinase DcuS [EC:2.7.13.3]\n",
      "SSO SSO:000016915 Fumarate respiration sensor kinase protein DcuS\n",
      "b4125 ['Fumarate respiration sensor kinase protein DcuS']\n",
      "\n",
      "SSO SSO:000028396 Uncharacterized protein YjdI\n",
      "b4126 ['Uncharacterized protein YjdI']\n",
      "\n",
      "KO KO:K06975 uncharacterized protein\n",
      "SSO SSO:000028398 Uncharacterized protein YjdJ, possible GNAT acetyltransferase\n",
      "b4127 ['Uncharacterized protein YjdJ, possible GNAT acetyltransferase']\n",
      "\n",
      "KO KO:K18840 antitoxin GhoS\n",
      "SSO SSO:000002477 Endoribonuclease GhoS\n",
      "b4128 ['Endoribonuclease GhoS']\n",
      "\n",
      "GO GO:0016021 integral component of membrane\n",
      "KO KO:K18839 toxin GhoT\n",
      "SSO SSO:000008112 Toxin GhoT\n",
      "b4559 ['Toxin GhoT']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K04567 lysyl-tRNA synthetase, class II [EC:6.1.1.6]\n",
      "SSO SSO:000004610 Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)\n",
      "b4129 ['Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03305 proton-dependent oligopeptide transporter, POT family\n",
      "SSO SSO:000013070 Di-tripeptide/H+ symporter DtpC\n",
      "b4130 ['Di-tripeptide/H+ symporter DtpC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K01582 lysine decarboxylase [EC:4.1.1.18]\n",
      "SSO SSO:000004600 Lysine decarboxylase, inducible (EC 4.1.1.18)\n",
      "b4131 ['Lysine decarboxylase, inducible (EC 4.1.1.18)']\n",
      "\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03757 cadaverine:lysine antiporter\n",
      "SSO SSO:000019145 Lysine/cadaverine antiporter membrane protein CadB\n",
      "b4132 ['Lysine/cadaverine antiporter membrane protein CadB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K03765 transcriptional activator of cad operon\n",
      "SSO SSO:000025777 Transcriptional activator of cad operon\n",
      "b4133 ['Transcriptional activator of cad operon']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4654 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000025872 Transcriptional regulator YjdC, AcrR family\n",
      "b4135 ['Transcriptional regulator YjdC, AcrR family']\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K04084 thiol:disulfide interchange protein DsbD [EC:1.8.1.8]\n",
      "SSO SSO:000001926 Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)\n",
      "b4136 ['Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)']\n",
      "\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K03926 periplasmic divalent cation tolerance protein\n",
      "SSO SSO:000005838 Periplasmic divalent cation tolerance protein cutA\n",
      "b4137 ['Periplasmic divalent cation tolerance protein CutA']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07791 anaerobic C4-dicarboxylate transporter DcuA\n",
      "SSO SSO:000010640 Anaerobic C4-dicarboxylate transporter DcuA\n",
      "b4138 ['Anaerobic C4-dicarboxylate transporter DcuA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006520 cellular amino acid metabolic process\n",
      "GO GO:0006529 asparagine biosynthetic process\n",
      "GO GO:0009089 lysine biosynthetic process via diaminopimelate\n",
      "KO KO:K01744 aspartate ammonia-lyase [EC:4.3.1.1]\n",
      "SSO SSO:000001094 Aspartate ammonia-lyase (EC 4.3.1.1)\n",
      "b4139 ['Aspartate ammonia-lyase (EC 4.3.1.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07113 UPF0716 protein FxsA\n",
      "SSO SSO:000026693 UPF0716 protein FxsA\n",
      "b4140 ['UPF0716 protein FxsA']\n",
      "\n",
      "KO KO:K16263 amino acid efflux transporter\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000036528 L-methionine and branched-chain amino acid export protein YjeH\n",
      "b4141 ['L-methionine and branched-chain amino acid export protein YjeH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04078 chaperonin GroES\n",
      "SSO SSO:000036241 Heat shock protein 10 kDa family chaperone GroES\n",
      "b4142 ['Heat shock protein 10 kDa family chaperone GroES']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K04077 chaperonin GroEL\n",
      "SSO SSO:000035012 Heat shock protein 60 kDa family chaperone GroEL\n",
      "b4143 ['Heat shock protein 60 kDa family chaperone GroEL']\n",
      "\n",
      "SSO SSO:000028399 Uncharacterized protein YjeI\n",
      "b4144 ['Uncharacterized protein YjeI']\n",
      "\n",
      "SSO SSO:000028400 Uncharacterized protein YjeJ\n",
      "b4145 ['Uncharacterized protein YjeJ']\n",
      "\n",
      "KO KO:K19810 L-lysine 2,3-aminomutase [EC:5.4.3.-]\n",
      "SSO SSO:000004595 Lysine 2,3-aminomutase (EC 5.4.3.2)\n",
      "b4146 ['Lysine 2,3-aminomutase (EC 5.4.3.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02356 elongation factor P\n",
      "SSO SSO:000008303 Translation elongation factor P\n",
      "b4147 ['Translation elongation factor P']\n",
      "\n",
      "KO KO:K16347 entericidin A\n",
      "SSO SSO:000013424 Entericidin A\n",
      "b4410 ['Entericidin A']\n",
      "\n",
      "KO KO:K16348 entericidin B\n",
      "SSO SSO:000013425 Entericidin B\n",
      "b4411 ['Entericidin B']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11741 quaternary ammonium compound-resistance protein SugE\n",
      "b4148 ['Small multidrug resistance (SMR) efflux transporter => SugE, quaternary ammonium compounds']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K03098 apolipoprotein D and lipocalin family protein\n",
      "SSO SSO:000036639 Lipocalin Blc\n",
      "b4149 ['Lipocalin Blc']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042493 response to drug\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K01467 beta-lactamase class C [EC:3.5.2.6]\n",
      "b4150 ['Class C beta-lactamase (EC 3.5.2.6) => BlaEC family']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00247 fumarate reductase subunit D\n",
      "SSO SSO:000002961 Fumarate reductase subunit D\n",
      "b4151 ['Fumarate reductase subunit D']\n",
      "\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K00246 fumarate reductase subunit C\n",
      "SSO SSO:000002960 Fumarate reductase subunit C\n",
      "b4152 ['Fumarate reductase subunit C']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00240 succinate dehydrogenase / fumarate reductase, iron-sulfur subunit [EC:1.3.5.1 1.3.5.4]\n",
      "KO KO:K00245 fumarate reductase iron-sulfur subunit [EC:1.3.5.4]\n",
      "KO KO:K00246 fumarate reductase subunit C\n",
      "SSO SSO:000002958 Fumarate reductase iron-sulfur protein (EC 1.3.5.4)\n",
      "b4153 ['Fumarate reductase iron-sulfur protein (EC 1.3.5.4)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006113 fermentation\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K00239 succinate dehydrogenase / fumarate reductase, flavoprotein subunit [EC:1.3.5.1 1.3.5.4]\n",
      "KO KO:K00244 fumarate reductase flavoprotein subunit [EC:1.3.5.4]\n",
      "SSO SSO:000002957 Fumarate reductase flavoprotein subunit (EC 1.3.99.1)\n",
      "b4154 ['Fumarate reductase flavoprotein subunit (EC 1.3.5.4)']\n",
      "\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K04568 elongation factor P--(R)-beta-lysine ligase [EC:6.3.1.-]\n",
      "SSO SSO:000039758 Translation elongation factor P Lys34--(R)-beta-lysine ligase\n",
      "b4155 ['Translation elongation factor P Lys34--(R)-beta-lysine ligase']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K20265 glutamate:GABA antiporter\n",
      "SSO SSO:000018445 Inner membrane transporter YjeM\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b4156 ['Inner membrane transporter YjeM']\n",
      "\n",
      "SSO SSO:000028401 Uncharacterized protein YjeN\n",
      "b4157 ['Uncharacterized protein YjeN']\n",
      "\n",
      "SSO SSO:000018416 Inner membrane protein YjeO\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b4158 ['Inner membrane protein YjeO']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K22051 miniconductance mechanosensitive channel\n",
      "SSO SSO:000019680 Miniconductance mechanosensitive channel MscM\n",
      "b4159 ['Miniconductance mechanosensitive channel MscM']\n",
      "\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01613 phosphatidylserine decarboxylase [EC:4.1.1.65]\n",
      "SSO SSO:000005945 Phosphatidylserine decarboxylase (EC 4.1.1.65)\n",
      "b4160 ['Phosphatidylserine decarboxylase (EC 4.1.1.65)']\n",
      "\n",
      "KO KO:K06949 ribosome biogenesis GTPase / thiamine phosphate phosphatase [EC:3.6.1.- 3.1.3.100]\n",
      "SSO SSO:000007149 Ribosome small subunit biogenesis RbfA-release protein RsgA\n",
      "b4161 ['Ribosome small subunit biogenesis RbfA-release protein RsgA']\n",
      "\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K13288 oligoribonuclease [EC:3.1.-.-]\n",
      "SSO SSO:000000260 3'-to-5' oligoribonuclease (orn)\n",
      "b4162 [\"3'-to-5' oligoribonuclease (orn)\"]\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4670 ['hypothetical protein']\n",
      "\n",
      "KO KO:K18979 epoxyqueuosine reductase [EC:1.17.99.6]\n",
      "SSO SSO:000035656 Epoxyqueuosine reductase (EC 1.17.99.6) QueG\n",
      "b4166 ['Epoxyqueuosine reductase (EC 1.17.99.6) QueG']\n",
      "\n",
      "KO KO:K17758 ADP-dependent NAD(P)H-hydrate dehydratase [EC:4.2.1.136]\n",
      "KO KO:K23997 Unknown\n",
      "b4167 ['NAD(P)H-hydrate epimerase (EC 5.1.99.6)', 'ADP-dependent (S)-NAD(P)H-hydrate dehydratase (EC 4.2.1.136)']\n",
      "\n",
      "KO KO:K06925 tRNA threonylcarbamoyladenosine biosynthesis protein TsaE\n",
      "SSO SSO:000034427 tRNA threonylcarbamoyladenosine biosynthesis protein TsaE\n",
      "b4168 ['tRNA threonylcarbamoyladenosine biosynthesis protein TsaE']\n",
      "\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K01448 N-acetylmuramoyl-L-alanine amidase [EC:3.5.1.28]\n",
      "SSO SSO:000005115 N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)\n",
      "b4169 ['N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006281 DNA repair\n",
      "KO KO:K03572 DNA mismatch repair protein MutL\n",
      "SSO SSO:000002075 DNA mismatch repair protein MutL\n",
      "b4170 ['DNA mismatch repair protein MutL']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00791 tRNA dimethylallyltransferase [EC:2.5.1.75]\n",
      "SSO SSO:000009358 tRNA dimethylallyltransferase (EC 2.5.1.75)\n",
      "b4171 ['tRNA dimethylallyltransferase (EC 2.5.1.75)']\n",
      "\n",
      "GO GO:0009386 translational attenuation\n",
      "KO KO:K03666 host factor-I protein\n",
      "SSO SSO:000006929 RNA-binding protein Hfq\n",
      "b4172 ['RNA-binding protein Hfq']\n",
      "\n",
      "KO KO:K03665 GTPase\n",
      "SSO SSO:000007142 Ribosome LSU-associated GTP-binding protein HflX\n",
      "b4173 ['Ribosome LSU-associated GTP-binding protein HflX']\n",
      "\n",
      "KO KO:K04088 membrane protease subunit HflK [EC:3.4.-.-]\n",
      "SSO SSO:000003430 HflK protein\n",
      "b4174 ['HflK protein']\n",
      "\n",
      "KO KO:K04087 membrane protease subunit HflC [EC:3.4.-.-]\n",
      "SSO SSO:000003429 HflC protein\n",
      "b4175 ['HflC protein']\n",
      "\n",
      "KO KO:K09937 uncharacterized protein\n",
      "SSO SSO:000006664 Putative inner membrane protein YjeT (clustered with HflC)\n",
      "b4176 ['Putative inner membrane protein YjeT (clustered with HflC)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01939 adenylosuccinate synthase [EC:6.3.4.4]\n",
      "SSO SSO:000000824 Adenylosuccinate synthetase (EC 6.3.4.4)\n",
      "b4177 ['Adenylosuccinate synthetase (EC 6.3.4.4)']\n",
      "\n",
      "KO KO:K13771 Rrf2 family transcriptional regulator, nitric oxide-sensitive transcriptional repressor\n",
      "SSO SSO:000020561 Nitrite-sensitive transcriptional repressor NsrR\n",
      "b4178 ['Nitrite-sensitive transcriptional repressor NsrR']\n",
      "\n",
      "GO GO:0006401 RNA catabolic process\n",
      "KO KO:K12573 ribonuclease R [EC:3.1.-.-]\n",
      "SSO SSO:000000259 3'-to-5' exoribonuclease RNase R\n",
      "b4179 [\"3'-to-5' exoribonuclease RNase R\"]\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K03218 23S rRNA (guanosine2251-2'-O)-methyltransferase [EC:2.1.1.185]\n",
      "SSO SSO:000000238 23S rRNA (guanosine(2251)-2'-O)-methyltransferase (EC 2.1.1.185)\n",
      "b4180 [\"23S rRNA (guanosine(2251)-2'-O)-methyltransferase (EC 2.1.1.185)\"]\n",
      "\n",
      "KO KO:K09980 uncharacterized protein\n",
      "SSO SSO:000028405 Uncharacterized protein YjfI\n",
      "b4181 ['Uncharacterized protein YjfI']\n",
      "\n",
      "SSO SSO:000022976 PspA/IM30 family protein\n",
      "SSO SSO:000028406 Uncharacterized protein YjfJ\n",
      "b4182 ['Uncharacterized protein YjfJ']\n",
      "\n",
      "SSO SSO:000028407 Uncharacterized protein YjfK\n",
      "b4183 ['Uncharacterized protein YjfK']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K08989 putative membrane protein\n",
      "SSO SSO:000026695 UPF0719 inner membrane protein YjfL\n",
      "b4184 ['UPF0719 inner membrane protein YjfL']\n",
      "\n",
      "SSO SSO:000028408 Uncharacterized protein YjfM\n",
      "b4185 ['Uncharacterized protein YjfM']\n",
      "\n",
      "SSO SSO:000023139 Putative acid--amine ligase YjfC\n",
      "b4186 ['Putative acid--amine ligase YjfC']\n",
      "\n",
      "GO GO:0006950 response to stress\n",
      "KO KO:K09456 putative acyl-CoA dehydrogenase\n",
      "SSO SSO:000010432 Alkylation response protein AidB, acyl-CoA dehydrogenase family\n",
      "b4187 ['Alkylation response protein AidB, acyl-CoA dehydrogenase family']\n",
      "\n",
      "SSO SSO:000023703 Putative uncharacterized protein YjfN\n",
      "b4188 ['Putative uncharacterized protein YjfN']\n",
      "\n",
      "SSO SSO:000018990 Lipoprotein BsmA\n",
      "b4189 ['Lipoprotein BsmA']\n",
      "\n",
      "SSO SSO:000029155 YjfP protein\n",
      "b4190 ['YjfP protein']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03477 DeoR family transcriptional regulator, ulaG and ulaABCDEF operon transcriptional repressor\n",
      "SSO SSO:000034700 Ascorbate utilization transcriptional regulator UlaR, DeoR family\n",
      "b4191 ['Ascorbate utilization transcriptional regulator UlaR, DeoR family']\n",
      "\n",
      "KO KO:K03476 L-ascorbate 6-phosphate lactonase [EC:3.1.1.-]\n",
      "SSO SSO:000036490 L-ascorbate-6-phosphate lactonase UlaG\n",
      "b4192 ['L-ascorbate-6-phosphate lactonase UlaG']\n",
      "\n",
      "GO GO:0009401 phosphoenolpyruvate-dependent sugar phosphotransferase system\n",
      "KO KO:K03475 PTS system, ascorbate-specific IIC component\n",
      "SSO SSO:000037474 PTS system, ascorbate-specific IIC component\n",
      "b4193 ['PTS system, ascorbate-specific IIC component']\n",
      "\n",
      "GO GO:0009401 phosphoenolpyruvate-dependent sugar phosphotransferase system\n",
      "KO KO:K02822 PTS system, ascorbate-specific IIB component [EC:2.7.1.194]\n",
      "SSO SSO:000037473 PTS system, ascorbate-specific IIB component (EC 2.7.1.194)\n",
      "b4194 ['PTS system, ascorbate-specific IIB component (EC 2.7.1.194)']\n",
      "\n",
      "KO KO:K02821 PTS system, ascorbate-specific IIA component [EC:2.7.1.194]\n",
      "SSO SSO:000037472 PTS system, ascorbate-specific IIA component\n",
      "b4195 ['PTS system, ascorbate-specific IIA component']\n",
      "\n",
      "KO KO:K03078 3-dehydro-L-gulonate-6-phosphate decarboxylase [EC:4.1.1.85]\n",
      "SSO SSO:000000310 3-keto-L-gulonate 6-phosphate decarboxylase (EC 4.1.1.85)\n",
      "SSO SSO:000034066 3-keto-L-gulonate-6-phosphate decarboxylase UlaD (EC 4.1.1.85)\n",
      "b4196 ['3-keto-L-gulonate-6-phosphate decarboxylase UlaD (EC 4.1.1.85)']\n",
      "\n",
      "KO KO:K03079 L-ribulose-5-phosphate 3-epimerase [EC:5.1.3.22]\n",
      "SSO SSO:000004169 L-ribulose-5-phosphate 3-epimerase UlaE (EC 5.1.3.22)\n",
      "SSO SSO:000004187 L-xylulose 5-phosphate 3-epimerase (EC 5.1.3.-)\n",
      "b4197 ['L-ribulose-5-phosphate 3-epimerase UlaE (EC 5.1.3.22)']\n",
      "\n",
      "KO KO:K03077 L-ribulose-5-phosphate 4-epimerase [EC:5.1.3.4]\n",
      "SSO SSO:000004170 L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)\n",
      "SSO SSO:000036548 L-ribulose-5-phosphate 4-epimerase UlaF (EC 5.1.3.4)\n",
      "b4198 ['L-ribulose-5-phosphate 4-epimerase UlaF (EC 5.1.3.4)']\n",
      "\n",
      "b4199 ['Uncharacterized protein YjfY']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02990 small subunit ribosomal protein S6\n",
      "SSO SSO:000007333 SSU ribosomal protein S6p\n",
      "b4200 ['SSU ribosomal protein S6p']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02686 primosomal replication protein N\n",
      "SSO SSO:000022366 Primosomal replication protein N\n",
      "b4201 ['Primosomal replication protein N']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02963 small subunit ribosomal protein S18\n",
      "b4202 ['SSU ribosomal protein S18p', 'SSU ribosomal protein S18p, zinc-independent']\n",
      "\n",
      "GO GO:0003735 structural constituent of ribosome\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02939 large subunit ribosomal protein L9\n",
      "SSO SSO:000004365 LSU ribosomal protein L9p\n",
      "b4203 ['LSU ribosomal protein L9p']\n",
      "\n",
      "SSO SSO:000028410 Uncharacterized protein YjfZ\n",
      "b4204 ['Uncharacterized protein YjfZ']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4205 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07269 uncharacterized protein\n",
      "b4206 ['Uncharacterized protein YtfB']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03772 FKBP-type peptidyl-prolyl cis-trans isomerase FkpA [EC:5.2.1.8]\n",
      "KO KO:K03773 FKBP-type peptidyl-prolyl cis-trans isomerase FklB [EC:5.2.1.8]\n",
      "SSO SSO:000002671 FKBP-type peptidyl-prolyl cis-trans isomerase FklB (EC 5.2.1.8)\n",
      "b4207 ['FKBP-type peptidyl-prolyl cis-trans isomerase FklB (EC 5.2.1.8)']\n",
      "\n",
      "GO GO:0006164 purine nucleotide biosynthetic process\n",
      "GO GO:0006523 alanine biosynthetic process\n",
      "GO GO:0006545 glycine biosynthetic process\n",
      "GO GO:0006750 glutathione biosynthetic process\n",
      "GO GO:0009063 cellular amino acid catabolic process\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K11737 D-serine/D-alanine/glycine transporter\n",
      "SSO SSO:000002045 D-serine/D-alanine/glycine transporter\n",
      "b4208 ['D-serine/D-alanine/glycine transporter']\n",
      "\n",
      "KO KO:K07322 regulator of cell morphogenesis and NO signaling\n",
      "SSO SSO:000024001 Repair of Iron Centers di-iron protein\n",
      "b4209 ['Repair of Iron Centers di-iron protein']\n",
      "\n",
      "SSO SSO:000018434 Inner membrane protein YtfF\n",
      "b4210 ['Inner membrane protein YtfF']\n",
      "\n",
      "KO KO:K19267 NAD(P)H dehydrogenase (quinone) [EC:1.6.5.2]\n",
      "SSO SSO:000005248 NADPH:quinone oxidoreductase (EC 1.6.5.5)\n",
      "b4211 ['NAD(P)H dehydrogenase (quinone) 2 (EC 1.6.5.2)']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "b4212 ['Redox-sensing transcriptional regulator QorR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01119 2',3'-cyclic-nucleotide 2'-phosphodiesterase / 3'-nucleotidase [EC:3.1.4.16 3.1.3.6]\n",
      "b4213 [\"2',3'-cyclic-nucleotide 2'-phosphodiesterase (EC 3.1.4.16)\", \"3'-nucleotidase (EC 3.1.3.6)\"]\n",
      "\n",
      "GO GO:0006790 sulfur compound metabolic process\n",
      "KO KO:K01082 3'(2'), 5'-bisphosphate nucleotidase [EC:3.1.3.7]\n",
      "SSO SSO:000000255 3'(2'),5'-bisphosphate nucleotidase (EC 3.1.3.7)\n",
      "b4214 [\"3'(2'),5'-bisphosphate nucleotidase (EC 3.1.3.7)\"]\n",
      "\n",
      "SSO SSO:000028550 Uncharacterized protein YtfI\n",
      "b4215 ['Uncharacterized protein YtfI']\n",
      "\n",
      "KO KO:K07109 uncharacterized protein\n",
      "SSO SSO:000028551 Uncharacterized protein YtfJ\n",
      "b4216 ['Uncharacterized protein YtfJ']\n",
      "\n",
      "SSO SSO:000028552 Uncharacterized protein YtfK\n",
      "b4217 ['Uncharacterized protein YtfK']\n",
      "\n",
      "KO KO:K03699 putative hemolysin\n",
      "SSO SSO:000026483 UPF0053 inner membrane protein YtfL\n",
      "b4218 ['UPF0053 inner membrane protein YtfL']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K07304 peptide-methionine (S)-S-oxide reductase [EC:1.8.4.11]\n",
      "SSO SSO:000031064 methionine sulfoxide reductase A\n",
      "SSO SSO:000035652 Peptide-methionine (S)-S-oxide reductase MsrA (EC 1.8.4.11)\n",
      "b4219 ['Peptide-methionine (S)-S-oxide reductase MsrA (EC 1.8.4.11)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07278 translocation and assembly module TamA\n",
      "SSO SSO:000005537 Outer membrane component of TAM transport system\n",
      "b4220 ['Outer membrane component of TAM transport system']\n",
      "\n",
      "KO KO:K09800 translocation and assembly module TamB\n",
      "SSO SSO:000003876 Inner membrane component of TAM transport system\n",
      "b4221 ['Inner membrane component of TAM transport system']\n",
      "\n",
      "KO KO:K19761 gamma-glutamylaminecyclotransferase [EC:2.3.2.-]\n",
      "SSO SSO:000008662 UPF0131 protein YtfP\n",
      "b4222 ['UPF0131 protein YtfP']\n",
      "\n",
      "KO KO:K07172 antitoxin MazE\n",
      "KO KO:K18842 antitoxin ChpS\n",
      "SSO SSO:000006348 Programmed cell death antitoxin ChpS\n",
      "b4224 ['Programmed cell death antitoxin ChpS']\n",
      "\n",
      "KO KO:K07171 mRNA interferase MazF [EC:3.1.-.-]\n",
      "KO KO:K18841 mRNA interferase ChpB [EC:3.1.-.-]\n",
      "SSO SSO:000006353 Programmed cell death toxin ChpB\n",
      "b4225 ['Programmed cell death toxin ChpB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006793 phosphorus metabolic process\n",
      "KO KO:K01507 inorganic pyrophosphatase [EC:3.6.1.1]\n",
      "SSO SSO:000003905 Inorganic pyrophosphatase (EC 3.6.1.1)\n",
      "b4226 ['Inorganic pyrophosphatase (EC 3.6.1.1)']\n",
      "\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K10439 ribose transport system substrate-binding protein\n",
      "KO KO:K23508 Unknown\n",
      "SSO SSO:000036029 Galactofuranose ABC transporter, substrate-binding protein\n",
      "b4227 ['Galactofuranose ABC transporter, substrate-binding protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02056 simple sugar transport system ATP-binding protein [EC:3.6.3.17]\n",
      "KO KO:K10820 monosaccharide-transporting ATPase [EC:3.6.3.17]\n",
      "KO KO:K10822 nonpolar-amino-acid-transporting ATPase [EC:7.4.2.2]\n",
      "SSO SSO:000036026 Galactofuranose ABC transporter, ATP-binding protein\n",
      "b4485 ['Galactofuranose ABC transporter, ATP-binding protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K23509 Unknown\n",
      "SSO SSO:000036027 Galactofuranose ABC transporter, permease protein 1\n",
      "b4230 ['Galactofuranose ABC transporter, permease protein 1']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K23509 Unknown\n",
      "SSO SSO:000036028 Galactofuranose ABC transporter, permease protein 2\n",
      "b4231 ['Galactofuranose ABC transporter, permease protein 2']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "KO KO:K03841 fructose-1,6-bisphosphatase I [EC:3.1.3.11]\n",
      "SSO SSO:000002932 Fructose-1,6-bisphosphatase, type I (EC 3.1.3.11)\n",
      "b4232 ['Fructose-1,6-bisphosphatase, type I (EC 3.1.3.11)']\n",
      "\n",
      "GO GO:0000270 peptidoglycan metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K02558 UDP-N-acetylmuramate: L-alanyl-gamma-D-glutamyl-meso-diaminopimelate ligase [EC:6.3.2.45]\n",
      "SSO SSO:000008628 UDP-N-acetylmuramate:L-alanyl-gamma-D-glutamyl-meso-diaminopimelate ligase (EC 6.3.2.-)\n",
      "b4233 ['UDP-N-acetylmuramate:L-alanyl-gamma-D-glutamyl-meso-diaminopimelate ligase (EC 6.3.2.-)']\n",
      "\n",
      "GO GO:0004009 obsolete ATP-binding cassette (ABC) transporter activity\n",
      "KO KO:K09889 ribosome-associated protein\n",
      "SSO SSO:000008669 UPF0307 protein YjgA\n",
      "b4234 ['UPF0307 protein YjgA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K03592 PmbA protein\n",
      "SSO SSO:000008074 TldE protein, part of TldE/TldD proteolytic complex\n",
      "b4235 ['TldE protein, part of TldE/TldD proteolytic complex']\n",
      "\n",
      "GO GO:0009055 electron carrier activity\n",
      "GO GO:0017004 cytochrome complex assembly\n",
      "KO KO:K15536 soluble cytochrome b562\n",
      "SSO SSO:000024964 Soluble cytochrome b562\n",
      "b4236 ['Soluble cytochrome b562']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K04068 anaerobic ribonucleoside-triphosphate reductase activating protein [EC:1.97.1.4]\n",
      "SSO SSO:000007100 Ribonucleotide reductase of class III (anaerobic), activating protein (EC 1.97.1.4)\n",
      "b4237 ['Ribonucleotide reductase of class III (anaerobic), activating protein (EC 1.97.1.4)']\n",
      "\n",
      "KO KO:K21636 ribonucleoside-triphosphate reductase (formate) [EC:1.1.98.6]\n",
      "SSO SSO:000007101 Ribonucleotide reductase of class III (anaerobic), large subunit (EC 1.17.4.2)\n",
      "b4238 ['Ribonucleotide reductase of class III (anaerobic), large subunit (EC 1.17.4.2)']\n",
      "\n",
      "EC EC:2.4.1.18 1,4-alpha-glucan branching enzyme.\n",
      "EC EC:2.4.1.19 Cyclomaltodextrin glucanotransferase.\n",
      "EC EC:2.4.1.25 4-alpha-glucanotransferase.\n",
      "EC EC:2.4.1.4 Amylosucrase.\n",
      "EC EC:2.4.1.7 Sucrose phosphorylase.\n",
      "EC EC:2.4.99.16 Starch synthase (maltosyl-transferring).\n",
      "EC EC:3.2.1.1 Alpha-amylase.\n",
      "EC EC:3.2.1.10 Oligo-1,6-glucosidase.\n",
      "EC EC:3.2.1.116 Glucan 1,4-alpha-maltotriohydrolase.\n",
      "EC EC:3.2.1.133 Glucan 1,4-alpha-maltohydrolase.\n",
      "EC EC:3.2.1.135 Neopullulanase.\n",
      "EC EC:3.2.1.141 4-alpha-D-((1->4)-alpha-D-glucano)trehalose trehalohydrolase.\n",
      "EC EC:3.2.1.20 Alpha-glucosidase.\n",
      "EC EC:3.2.1.33 Amylo-alpha-1,6-glucosidase.\n",
      "EC EC:3.2.1.41 Pullulanase.\n",
      "EC EC:3.2.1.54 Cyclomaltodextrinase.\n",
      "EC EC:3.2.1.60 Glucan 1,4-alpha-maltotetraohydrolase.\n",
      "EC EC:3.2.1.68 Isoamylase.\n",
      "EC EC:3.2.1.70 Glucan 1,6-alpha-glucosidase.\n",
      "EC EC:3.2.1.93 Alpha,alpha-phosphotrehalase.\n",
      "EC EC:3.2.1.98 Glucan 1,4-alpha-maltohexaosidase.\n",
      "EC EC:5.4.99.11 Isomaltulose synthase.\n",
      "EC EC:5.4.99.15 (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase.\n",
      "EC EC:5.4.99.16 Maltose alpha-D-glucosyltransferase.\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01226 trehalose-6-phosphate hydrolase [EC:3.2.1.93]\n",
      "SSO SSO:000026176 Trehalose-6-phosphate hydrolase (EC 3.2.1.93)\n",
      "b4239 ['Trehalose-6-phosphate hydrolase (EC 3.2.1.93)']\n",
      "\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02819 PTS system, trehalose-specific IIC component\n",
      "b4240 ['PTS system, trehalose-specific IIB component (EC 2.7.1.201)', 'PTS system, trehalose-specific IIC component']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006006 glucose metabolic process\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03485 LacI family transcriptional regulator, trehalose operon repressor\n",
      "b4241 ['Trehalose operon transcriptional repressor']\n",
      "\n",
      "SSO SSO:000019664 MgtA leader peptide\n",
      "b4702 ['MgtA leader peptide']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0015662 ATPase activity, coupled to transmembrane movement of ions, phosphorylative mechanism\n",
      "GO GO:0015986 ATP synthesis coupled proton transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01531 Mg2+-importing ATPase [EC:3.6.3.2]\n",
      "SSO SSO:000004911 Mg(2+) transport ATPase, P-type (EC 3.6.3.2)\n",
      "SSO SSO:000044077 Magnesium transporter\n",
      "b4242 ['Mg(2+) transport ATPase, P-type (EC 3.6.3.2)']\n",
      "\n",
      "KO KO:K09022 2-iminobutanoate/2-iminopropanoate deaminase [EC:3.5.99.10]\n",
      "SSO SSO:000033990 2-iminobutanoate/2-iminopropanoate deaminase RidA/TdcF (EC 3.5.99.10)\n",
      "b4243 ['2-iminobutanoate/2-iminopropanoate deaminase RidA/TdcF (EC 3.5.99.10)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "KO KO:K00610 aspartate carbamoyltransferase regulatory subunit\n",
      "SSO SSO:000001096 Aspartate carbamoyltransferase regulatory chain (PyrI)\n",
      "b4244 ['Aspartate carbamoyltransferase regulatory chain (PyrI)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K00609 aspartate carbamoyltransferase catalytic subunit [EC:2.1.3.2]\n",
      "SSO SSO:000001095 Aspartate carbamoyltransferase (EC 2.1.3.2)\n",
      "b4245 ['Aspartate carbamoyltransferase (EC 2.1.3.2)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006221 pyrimidine nucleotide biosynthetic process\n",
      "SSO SSO:000023737 PyrBI operon leader peptide\n",
      "b4246 ['PyrBI operon leader peptide']\n",
      "\n",
      "GO GO:0006412 translation\n",
      "KO KO:K09022 2-iminobutanoate/2-iminopropanoate deaminase [EC:3.5.99.10]\n",
      "SSO SSO:000007175 RidA/YER057c/UK114 superfamily, group 7, YjgH-like protein\n",
      "b4248 ['RidA/YER057c/UK114 superfamily, group 7, YjgH-like protein']\n",
      "\n",
      "KO KO:K19336 cyclic-di-GMP-binding biofilm dispersal mediator protein\n",
      "SSO SSO:000012549 Cyclic-di-GMP-binding biofilm dispersal mediator protein\n",
      "b4249 ['Cyclic-di-GMP-binding biofilm dispersal mediator protein']\n",
      "\n",
      "KO KO:K19335 TetR/AcrR family transcriptional regulator, repressor for divergent bdcA\n",
      "SSO SSO:000023057 Putative HTH-type transcriptional regulator YjgJ, TetR family\n",
      "b4251 ['Putative HTH-type transcriptional regulator YjgJ, TetR family']\n",
      "\n",
      "KO KO:K19334 biofilm protein TabA\n",
      "SSO SSO:000022737 Protein YjgK, linked to biofilm formation\n",
      "b4252 ['Protein YjgK, linked to biofilm formation']\n",
      "\n",
      "SSO SSO:000022738 Protein YjgL, putative CCAAT-box DNA binding protein subunit B\n",
      "b4253 ['Protein YjgL, putative CCAAT-box DNA binding protein subunit B']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006526 arginine biosynthetic process\n",
      "KO KO:K00611 ornithine carbamoyltransferase [EC:2.1.3.3]\n",
      "SSO SSO:000005514 Ornithine carbamoyltransferase (EC 2.1.3.3)\n",
      "b4254 ['Ornithine carbamoyltransferase (EC 2.1.3.3)']\n",
      "\n",
      "KO KO:K09893 regulator of ribonuclease activity B\n",
      "SSO SSO:000007074 Ribonuclease E inhibitor RraB\n",
      "b4255 ['Ribonuclease E inhibitor RraB']\n",
      "\n",
      "KO KO:K03828 putative acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000039731 Uncharacterized N-acetyltransferase YjgM\n",
      "b4256 ['Uncharacterized N-acetyltransferase YjgM']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000018417 Inner membrane protein YjgN\n",
      "b4257 ['Inner membrane protein YjgN']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "KO KO:K01873 valyl-tRNA synthetase [EC:6.1.1.9]\n",
      "SSO SSO:000008905 Valyl-tRNA synthetase (EC 6.1.1.9)\n",
      "b4258 ['Valyl-tRNA synthetase (EC 6.1.1.9)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02339 DNA polymerase III subunit chi [EC:2.7.7.7]\n",
      "SSO SSO:000002081 DNA polymerase III chi subunit (EC 2.7.7.7)\n",
      "b4259 ['DNA polymerase III chi subunit (EC 2.7.7.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K01255 leucyl aminopeptidase [EC:3.4.11.1]\n",
      "SSO SSO:000001970 Cytosol aminopeptidase PepA (EC 3.4.11.1)\n",
      "b4260 ['Cytosol aminopeptidase PepA (EC 3.4.11.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K07091 lipopolysaccharide export system permease protein\n",
      "SSO SSO:000004484 Lipopolysaccharide export system permease protein LptF\n",
      "b4261 ['Lipopolysaccharide export system permease protein LptF']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "KO KO:K11720 lipopolysaccharide export system permease protein\n",
      "SSO SSO:000004485 Lipopolysaccharide export system permease protein LptG\n",
      "b4262 ['Lipopolysaccharide export system permease protein LptG']\n",
      "\n",
      "KO KO:K06915 uncharacterized protein\n",
      "SSO SSO:000028413 Uncharacterized protein YjgR\n",
      "b4263 ['Uncharacterized protein YjgR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K06145 LacI family transcriptional regulator, gluconate utilization system Gnt-I transcriptional repressor\n",
      "KO KO:K06146 LacI family transcriptional regulator, gluconate utilization system Gnt-II transcriptional activator\n",
      "SSO SSO:000006178 Positive regulator of L-idonate catabolism\n",
      "b4264 ['Positive regulator of L-idonate catabolism']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03299 gluconate:H+ symporter, GntP family\n",
      "KO KO:K06155 Gnt-I system high-affinity gluconate transporter\n",
      "KO KO:K06157 Gnt-II system L-idonate transporter\n",
      "SSO SSO:000004154 L-idonate, D-gluconate, 5-keto-D-gluconate transporter\n",
      "b4265 ['L-idonate, D-gluconate, 5-keto-D-gluconate transporter']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00046 gluconate 5-dehydrogenase [EC:1.1.1.69]\n",
      "SSO SSO:000000426 5-keto-D-gluconate 5-reductase (EC 1.1.1.69)\n",
      "b4266 ['5-keto-D-gluconate 5-reductase (EC 1.1.1.69)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00098 L-idonate 5-dehydrogenase [EC:1.1.1.264]\n",
      "SSO SSO:000004153 L-idonate 5-dehydrogenase (EC 1.1.1.264)\n",
      "b4267 ['L-idonate 5-dehydrogenase (EC 1.1.1.264)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00851 gluconokinase [EC:2.7.1.12]\n",
      "SSO SSO:000003108 Gluconokinase (EC 2.7.1.12)\n",
      "b4268 ['Gluconokinase (EC 2.7.1.12)']\n",
      "\n",
      "KO KO:K12957 uncharacterized zinc-type alcohol dehydrogenase-like protein [EC:1.-.-.-]\n",
      "b4269 ['Cinnamyl alcohol dehydrogenase/reductase (EC 1.1.1.195)', 'Alcohol dehydrogenase (EC 1.1.1.1)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4271 ['hypothetical protein']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "SSO SSO:000026094 Transposase InsC for insertion element IS2\n",
      "b4272 ['Transposase InsC for insertion element IS2']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000026095 Transposase InsD for insertion element IS2\n",
      "b4273 ['Transposase InsD for insertion element IS2']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4575 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000019954 Mobile element protein\n",
      "b4277 ['Mobile element protein']\n",
      "\n",
      "KO KO:K07495 putative transposase\n",
      "SSO SSO:000026098 Transposase InsG for insertion sequence element IS4\n",
      "b4278 ['Transposase InsG for insertion sequence element IS4']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000023429 Putative metabolite transport protein YjhB\n",
      "b4279 ['Putative metabolite transport protein YjhB']\n",
      "\n",
      "SSO SSO:000027394 Uncharacterized oxidoreductase YjhC\n",
      "b4280 ['Uncharacterized oxidoreductase YjhC']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4655 ['hypothetical protein']\n",
      "\n",
      "b4281 ['KpLE2 phage-like element', 'predicted protein (pseudogene)']\n",
      "\n",
      "SSO SSO:000026890 Uncharacterized MFS-type transporter\n",
      "b4282 ['Uncharacterized MFS-type transporter']\n",
      "\n",
      "KO KO:K07483 transposase\n",
      "KO KO:K07497 putative transposase\n",
      "SSO SSO:000026102 Transposase InsO for insertion sequence element IS911\n",
      "b4623 ['Transposase InsO for insertion sequence element IS911']\n",
      "\n",
      "KO KO:K07482 transposase, IS30 family\n",
      "SSO SSO:000026099 Transposase InsI for insertion sequence element IS30\n",
      "b4284 ['Transposase InsI for insertion sequence element IS30']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4561 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4286 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006826 iron ion transport\n",
      "KO KO:K02013 iron complex transport system ATP-binding protein [EC:3.6.3.34]\n",
      "KO KO:K23184 Unknown\n",
      "SSO SSO:000004006 Iron(III) dicitrate transport ATP-binding protein FecE (TC 3.A.1.14.1)\n",
      "b4287 ['Iron(III) dicitrate transport ATP-binding protein FecE (TC 3.A.1.14.1)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02015 iron complex transport system permease protein\n",
      "KO KO:K23182 Unknown\n",
      "SSO SSO:000004009 Iron(III) dicitrate transport system permease protein FecD (TC 3.A.1.14.1)\n",
      "b4288 ['Iron(III) dicitrate transport system permease protein FecD (TC 3.A.1.14.1)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K02015 iron complex transport system permease protein\n",
      "KO KO:K23183 Unknown\n",
      "SSO SSO:000004008 Iron(III) dicitrate transport system permease protein FecC (TC 3.A.1.14.1)\n",
      "b4289 ['Iron(III) dicitrate transport system permease protein FecC (TC 3.A.1.14.1)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K02016 iron complex transport system substrate-binding protein\n",
      "KO KO:K23181 Unknown\n",
      "SSO SSO:000004010 Iron(III) dicitrate transport system, periplasmic iron-binding protein FecB (TC 3.A.1.14.1)\n",
      "b4290 ['Iron(III) dicitrate transport system, periplasmic iron-binding protein FecB (TC 3.A.1.14.1)']\n",
      "\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K02014 iron complex outermembrane recepter protein\n",
      "KO KO:K16091 Fe(3+) dicitrate transport protein\n",
      "SSO SSO:000004007 Iron(III) dicitrate transport protein FecA\n",
      "b4291 ['Iron(III) dicitrate transport protein FecA']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0006826 iron ion transport\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07165 transmembrane sensor\n",
      "SSO SSO:000004005 Iron(III) dicitrate transmembrane sensor protein FecR\n",
      "b4292 ['Iron(III) dicitrate transmembrane sensor protein FecR']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "KO KO:K03088 RNA polymerase sigma-70 factor, ECF subfamily\n",
      "KO KO:K23514 Unknown\n",
      "SSO SSO:000006901 RNA polymerase sigma factor FecI\n",
      "b4293 ['RNA polymerase sigma factor FecI']\n",
      "\n",
      "SSO SSO:000018187 IS1 protein InsA\n",
      "b4294 ['IS1 protein InsA']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4576 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000028760 Uncharacterized transcriptional regulator YjhU, SorC family\n",
      "b4295 ['Uncharacterized transcriptional regulator YjhU, SorC family']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03299 gluconate:H+ symporter, GntP family\n",
      "SSO SSO:000003104 Gluconate permease, Bsu4004 homolog\n",
      "SSO SSO:000023662 Putative transporter\n",
      "b4296 ['Gluconate permease, Bsu4004 homolog']\n",
      "\n",
      "KO KO:K22396 xylonate dehydratase [EC:4.2.1.82]\n",
      "SSO SSO:000032474 putative dehydratase\n",
      "b4297 ['Uncharacterized dehydratase YjhG']\n",
      "\n",
      "KO KO:K01714 4-hydroxy-tetrahydrodipicolinate synthase [EC:4.3.3.7]\n",
      "KO KO:K22397 2-dehydro-3-deoxy-D-pentonate aldolase [EC:4.1.2.28]\n",
      "b4298 ['Uncharacterized lyase YjhH']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000026883 Uncharacterized HTH-type transcriptional regulator YjhI, IclR family\n",
      "b4299 ['Uncharacterized HTH-type transcriptional regulator YjhI, IclR family']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "SSO SSO:000023593 Putative sgc region transcriptional regulator\n",
      "b4300 ['Putative sgc region transcriptional regulator']\n",
      "\n",
      "KO KO:K14587 protein sgcE [EC:5.1.3.-]\n",
      "SSO SSO:000022913 Protein sgcE (EC 5.1.3.-)\n",
      "b4301 ['Protein sgcE (EC 5.1.3.-)']\n",
      "\n",
      "KO KO:K02821 PTS system, ascorbate-specific IIA component [EC:2.7.1.194]\n",
      "SSO SSO:000021158 PTS system, IIA component\n",
      "b4302 ['PTS system, IIA component']\n",
      "\n",
      "KO KO:K06971 uncharacterized protein\n",
      "SSO SSO:000041298 Uncharacterized protein STM1615\n",
      "b4303 ['Uncharacterized protein STM1615']\n",
      "\n",
      "GO GO:0009401 phosphoenolpyruvate-dependent sugar phosphotransferase system\n",
      "KO KO:K02775 PTS system, galactitol-specific IIC component\n",
      "SSO SSO:000021159 PTS system, IIC component\n",
      "b4304 ['PTS system, IIC component']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K02774 PTS system, galactitol-specific IIB component [EC:2.7.1.200]\n",
      "SSO SSO:000021149 PTS system IIB component\n",
      "b4565 ['PTS system, IIB component']\n",
      "\n",
      "KO KO:K20609 putative aminopeptidase [EC:3.4.11.-]\n",
      "SSO SSO:000023592 Putative sgc region protein sgcX\n",
      "b4305 ['Putative sgc region protein sgcX']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4656 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000007212 SAM-dependent methyltransferase (EC 2.1.1.-)\n",
      "b4306 ['SAM-dependent methyltransferase']\n",
      "\n",
      "GO GO:0016070 RNA metabolic process\n",
      "KO KO:K03824 putative acetyltransferase [EC:2.3.1.-]\n",
      "SSO SSO:000010176 Acetyltransferase\n",
      "b4307 ['Acetyltransferase']\n",
      "\n",
      "KO KO:K09982 uncharacterized protein\n",
      "SSO SSO:000026664 UPF0386 protein YjhX\n",
      "b4566 ['UPF0386 protein YjhX']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4657 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4308 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000022378 Probable 9-O-acetyl-N-acetylneuraminic acid deacetylase\n",
      "b4309 ['Probable 9-O-acetyl-N-acetylneuraminic acid deacetylase']\n",
      "\n",
      "KO KO:K17948 N-acetylneuraminate epimerase [EC:5.1.3.24]\n",
      "SSO SSO:000005117 N-acetylneuraminate epimerase (EC 5.1.3.24)\n",
      "b4310 ['N-acetylneuraminate epimerase (EC 5.1.3.24)']\n",
      "\n",
      "KO KO:K22110 oligogalacturonate-specific porin family protein\n",
      "SSO SSO:000020212 N-acetylneuraminic acid outer membrane channel protein NanC\n",
      "b4311 ['N-acetylneuraminic acid outer membrane channel protein NanC']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0009289 pilus\n",
      "GO GO:0009297 pilus assembly\n",
      "KO KO:K07357 type 1 fimbriae regulatory protein FimB\n",
      "SSO SSO:000009422 type 1 fimbriae regulatory protein FimB\n",
      "b4312 ['Type 1 fimbriae regulatory protein FimB']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006310 DNA recombination\n",
      "GO GO:0009289 pilus\n",
      "GO GO:0009297 pilus assembly\n",
      "KO KO:K07358 type 1 fimbriae regulatory protein FimE\n",
      "SSO SSO:000008466 Type 1 fimbriae regulatory protein FimE\n",
      "b4313 ['Type 1 fimbriae regulatory protein FimE']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "SSO SSO:000008582 Type-1 fimbrial protein, A chain\n",
      "b4314 ['Type-1 fimbrial protein, A chain']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "KO KO:K07351 fimbrial protein\n",
      "SSO SSO:000002784 Fimbrin-like protein FimI\n",
      "b4315 ['Fimbrin-like protein FimI']\n",
      "\n",
      "GO GO:0006457 protein folding\n",
      "GO GO:0009289 pilus\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K07346 fimbrial chaperone protein\n",
      "SSO SSO:000001505 Chaperone protein FimC\n",
      "b4316 ['Chaperone protein FimC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0009279 cell outer membrane\n",
      "KO KO:K07347 outer membrane usher protein\n",
      "SSO SSO:000005603 Outer membrane usher protein FimD\n",
      "b4317 ['Outer membrane usher protein FimD']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07345 major type 1 subunit fimbrin (pilin)\n",
      "KO KO:K07348 minor fimbrial subunit\n",
      "SSO SSO:000006431 Protein FimF (regulates length and adhesion of type 1 fimbriae)\n",
      "b4318 ['Protein FimF (regulates length and adhesion of type 1 fimbriae)']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07349 minor fimbrial subunit\n",
      "SSO SSO:000006432 Protein FimG (regulates length and adhesion of type 1 fimbriae)\n",
      "b4319 ['Protein FimG (regulates length and adhesion of type 1 fimbriae)']\n",
      "\n",
      "GO GO:0009101 glycoprotein biosynthetic process\n",
      "GO GO:0009289 pilus\n",
      "KO KO:K07350 minor fimbrial subunit\n",
      "SSO SSO:000006433 Protein FimH (regulates length and adhesion of type 1 fimbriae, and mediates mannose binding)\n",
      "b4320 ['Protein FimH (regulates length and adhesion of type 1 fimbriae, and mediates mannose binding)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K03299 gluconate:H+ symporter, GntP family\n",
      "KO KO:K16321 high-affinity gluconate transporter\n",
      "SSO SSO:000002940 Fructuronate transporter GntP\n",
      "b4321 ['Fructuronate transporter GntP']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K01686 mannonate dehydratase [EC:4.2.1.8]\n",
      "SSO SSO:000004709 Mannonate dehydratase (EC 4.2.1.8)\n",
      "b4322 ['Mannonate dehydratase (EC 4.2.1.8)']\n",
      "\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "KO KO:K00040 fructuronate reductase [EC:1.1.1.57]\n",
      "SSO SSO:000002034 D-mannonate oxidoreductase (EC 1.1.1.57)\n",
      "b4323 ['D-mannonate oxidoreductase (EC 1.1.1.57)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K05799 GntR family transcriptional regulator, transcriptional repressor for pyruvate dehydrogenase complex\n",
      "KO KO:K13637 GntR family transcriptional regulator, uxu operon transcriptional repressor\n",
      "SSO SSO:000008892 Uxu operon transcriptional regulator\n",
      "b4324 ['Uxu operon transcriptional regulator']\n",
      "\n",
      "SSO SSO:000028414 Uncharacterized protein YjiC\n",
      "b4325 ['Uncharacterized protein YjiC']\n",
      "\n",
      "KO KO:K21637 anti-adapter protein IraD\n",
      "b4326 ['YjiD protein']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K21645 LysR family transcriptional regulator, hypochlorite-specific transcription factor HypT\n",
      "SSO SSO:000019131 LysR-family transcriptional regulator YjiE\n",
      "b4327 ['LysR-family transcriptional regulator YjiE']\n",
      "\n",
      "KO KO:K01305 beta-aspartyl-dipeptidase (metallo-type) [EC:3.4.19.-]\n",
      "b4328 ['Isoaspartyl aminopeptidase (EC 3.4.19.5)', 'Asp-X dipeptidase']\n",
      "\n",
      "SSO SSO:000018418 Inner membrane protein YjiG\n",
      "b4329 ['Inner membrane protein YjiG']\n",
      "\n",
      "SSO SSO:000028415 Uncharacterized protein YjiH\n",
      "b4330 ['Uncharacterized protein YjiH']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K07559 putative RNA 2'-phosphotransferase [EC:2.7.1.-]\n",
      "SSO SSO:000006933 RNA:NAD 2'-phosphotransferase\n",
      "b4331 [\"RNA:NAD 2'-phosphotransferase\"]\n",
      "\n",
      "SSO SSO:000027185 Uncharacterized integral membrane protein YjiJ\n",
      "b4332 ['Uncharacterized integral membrane protein YjiJ']\n",
      "\n",
      "GO GO:0009279 cell outer membrane\n",
      "SSO SSO:000027186 Uncharacterized integral membrane protein YjiK\n",
      "b4333 ['Uncharacterized integral membrane protein YjiK']\n",
      "\n",
      "KO KO:K23876 Unknown\n",
      "SSO SSO:000028416 Uncharacterized protein YjiL\n",
      "b4334 ['Uncharacterized protein YjiL']\n",
      "\n",
      "SSO SSO:000028417 Uncharacterized protein YjiM\n",
      "b4335 ['Uncharacterized protein YjiM']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "SSO SSO:000027119 Uncharacterized inner membrane protein YjiN\n",
      "SSO SSO:000041756 Zinc-type alcohol dehydrogenase-like protein\n",
      "b4336 ['Uncharacterized inner membrane protein YjiN']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K23241 Unknown\n",
      "SSO SSO:000036925 Multidrug efflux pump MdtM (of MFS type)\n",
      "b4337 ['Multidrug efflux pump MdtM (of MFS type)']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4339 ['hypothetical protein']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "b4340 ['DNA-binding transcriptional regulator, MocR family', 'aminotransferase domain']\n",
      "\n",
      "SSO SSO:000028418 Uncharacterized protein YjiS\n",
      "b4341 ['Uncharacterized protein YjiS']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4342 ['hypothetical protein']\n",
      "\n",
      "SSO SSO:000009137 hypothetical protein\n",
      "b4486 ['hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K19147 5-methylcytosine-specific restriction enzyme subunit McrC\n",
      "SSO SSO:000022707 Protein McrC, which modifies the specificity of McrB restriction\n",
      "b4345 ['Protein McrC, which modifies the specificity of McrB restriction']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K07452 5-methylcytosine-specific restriction enzyme B [EC:3.1.21.-]\n",
      "SSO SSO:000009734 5-methylcytosine-specific restriction enzyme McrB\n",
      "b4346 ['5-methylcytosine-specific restriction enzyme McrB']\n",
      "\n",
      "GO GO:0009432 SOS response\n",
      "KO KO:K19048 toxic protein SymE\n",
      "SSO SSO:000025722 Toxic protein SymE\n",
      "b4347 ['Toxic protein SymE']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006306 DNA methylation\n",
      "KO KO:K01154 type I restriction enzyme, S subunit [EC:3.1.21.3]\n",
      "SSO SSO:000026411 Type-1 restriction enzyme EcoKI specificity protein\n",
      "b4348 ['Type-1 restriction enzyme EcoKI specificity protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006306 DNA methylation\n",
      "KO KO:K03427 type I restriction enzyme M protein [EC:2.1.1.72]\n",
      "SSO SSO:000026328 Type I restriction enzyme EcoKI methylase protein (EC 2.1.1.72)\n",
      "b4349 ['Type I restriction enzyme EcoKI methylase protein (EC 2.1.1.72)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006308 DNA catabolic process\n",
      "KO KO:K01153 type I restriction enzyme, R subunit [EC:3.1.21.3]\n",
      "SSO SSO:000026329 Type I restriction enzyme EcoKI restriction protein (EC 3.1.21.3)\n",
      "b4350 ['Type I restriction enzyme EcoKI restriction protein (EC 3.1.21.3)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006306 DNA methylation\n",
      "KO KO:K07448 restriction system protein\n",
      "SSO SSO:000004964 Mrr restriction system protein\n",
      "b4351 ['Mrr restriction system protein']\n",
      "\n",
      "KO KO:K02234 cobalamin biosynthesis protein CobW\n",
      "SSO SSO:000036814 Metal-binding GTPase YjiA\n",
      "b4352 ['Metal-binding GTPase YjiA']\n",
      "\n",
      "SSO SSO:000028673 Uncharacterized proteins YbdD and YjiX\n",
      "b4353 ['Uncharacterized proteins YbdD and YjiX']\n",
      "\n",
      "GO GO:0042594 response to starvation\n",
      "KO KO:K06200 carbon starvation protein\n",
      "KO KO:K23676 Unknown\n",
      "SSO SSO:000023662 Putative transporter\n",
      "SSO SSO:000038864 Pyruvate:H+ symporter BtsT\n",
      "b4354 ['Pyruvate:H+ symporter BtsT']\n",
      "\n",
      "GO GO:0019866 organelle inner membrane\n",
      "GO GO:0042330 taxis\n",
      "KO KO:K03406 methyl-accepting chemotaxis protein\n",
      "KO KO:K05874 methyl-accepting chemotaxis protein I, serine sensor receptor\n",
      "SSO SSO:000019570 Methyl-accepting chemotaxis protein I (serine chemoreceptor protein)\n",
      "b4355 ['Methyl-accepting chemotaxis protein I (serine chemoreceptor protein)']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K23016 MFS transporter, ACS family, L-galactonate transporter\n",
      "SSO SSO:000004150 L-galactonate transporter\n",
      "b4356 ['L-galactonate transporter']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "KO KO:K23015 GntR family transcriptional regulator, transcriptional activator for L-galactonate catabolism\n",
      "SSO SSO:000006968 Regulator of L-galactonate catabolism LgoR, GntR family\n",
      "b4357 ['Regulator of L-galactonate catabolism LgoR, GntR family']\n",
      "\n",
      "KO KO:K23007 L-galactonate 5-dehydrogenase [EC:1.1.1.414]\n",
      "SSO SSO:000004151 L-galactonate-5-dehydrogenase\n",
      "b4358 ['L-galactonate-5-dehydrogenase']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0008654 phospholipid biosynthetic process\n",
      "GO GO:0009250 glucan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K01002 phosphoglycerol transferase [EC:2.7.8.20]\n",
      "SSO SSO:000005938 Phosphatidylglycerol--membrane-oligosaccharide glycerophosphotransferase (EC 2.7.8.20)\n",
      "b4359 ['Phosphatidylglycerol--membrane-oligosaccharide glycerophosphotransferase (EC 2.7.8.20)']\n",
      "\n",
      "SSO SSO:000023329 Putative glycoprotein/receptor\n",
      "b4360 ['Putative glycoprotein/receptor']\n",
      "\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02315 DNA replication protein DnaC\n",
      "SSO SSO:000035422 DNA replication helicase loader DnaC/DnaI\n",
      "b4361 ['DNA replication helicase loader DnaC/DnaI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02317 DNA replication protein DnaT\n",
      "SSO SSO:000022364 Primosomal protein I\n",
      "b4362 ['Primosomal protein I']\n",
      "\n",
      "SSO SSO:000013659 FIG001826: putative inner membrane protein\n",
      "b4363 ['FIG001826: putative inner membrane protein']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "SSO SSO:000016326 FIG023911: putative membrane protein\n",
      "b4364 ['FIG023911: putative membrane protein']\n",
      "\n",
      "SSO SSO:000023555 Putative regulatory protein\n",
      "b4365 ['Putative regulatory protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016052 carbohydrate catabolic process\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K22650 luxR family transcriptional regulator, regulator of transport and utilization of aryl beta-glucosides\n",
      "SSO SSO:000025779 Transcriptional activator protein bglJ\n",
      "b4366 ['Transcriptional activator protein bglJ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006457 protein folding\n",
      "KO KO:K13255 ferric iron reductase protein FhuF\n",
      "SSO SSO:000002735 Ferric reductase (1.6.99.14)\n",
      "b4367 ['Ferric reductase (1.6.99.14)']\n",
      "\n",
      "GO GO:0016021 integral component of membrane\n",
      "b4567 ['FIG00638078: hypothetical protein']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K00564 16S rRNA (guanine1207-N2)-methyltransferase [EC:2.1.1.172]\n",
      "SSO SSO:000000058 16S rRNA (guanine(1207)-N(2))-methyltransferase (EC 2.1.1.172)\n",
      "b4371 ['16S rRNA (guanine(1207)-N(2))-methyltransferase (EC 2.1.1.172)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "KO KO:K02344 DNA polymerase III subunit psi [EC:2.7.7.7]\n",
      "SSO SSO:000012807 DNA polymerase III psi subunit (EC 2.7.7.7)\n",
      "b4372 ['DNA polymerase III psi subunit (EC 2.7.7.7)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006464 cellular protein modification process\n",
      "KO KO:K03789 [ribosomal protein S18]-alanine N-acetyltransferase [EC:2.3.1.266]\n",
      "SSO SSO:000007139 Ribosomal-protein-S18p-alanine acetyltransferase (EC 2.3.1.-)\n",
      "SSO SSO:000041789 [ribosomal protein S18]-alanine N-acetyltransferase (EC 2.3.1.266)\n",
      "b4373 ['Ribosomal-protein-S18p-alanine acetyltransferase (EC 2.3.1.128)']\n",
      "\n",
      "KO KO:K07025 putative hydrolase of the HAD superfamily\n",
      "KO KO:K08723 5'-nucleotidase [EC:3.1.3.5]\n",
      "SSO SSO:000006817 Pyrimidine 5'-nucleotidase YjjG (EC 3.1.3.5)\n",
      "b4374 [\"Pyrimidine 5'-nucleotidase YjjG (EC 3.1.3.5)\"]\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006412 translation\n",
      "KO KO:K02837 peptide chain release factor 3\n",
      "SSO SSO:000005791 Peptide chain release factor 3 (EC 4.1.1.68 )\n",
      "SSO SSO:000021386 Peptide chain release factor RF-3\n",
      "b4375 ['Peptide chain release factor 3']\n",
      "\n",
      "GO GO:0006970 response to osmotic stress\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K04065 hyperosmotically inducible periplasmic protein\n",
      "SSO SSO:000005527 Osmotically inducible protein OsmY\n",
      "b4376 ['Osmotically inducible protein OsmY']\n",
      "\n",
      "GO GO:0016021 integral component of membrane\n",
      "SSO SSO:000039934 UPF0391 membrane protein YtjA\n",
      "b4568 ['UPF0391 membrane protein YtjA']\n",
      "\n",
      "GO GO:0006355 regulation of transcription, DNA-templated\n",
      "SSO SSO:000028419 Uncharacterized protein YjjU\n",
      "b4377 ['Uncharacterized protein YjjU']\n",
      "\n",
      "KO KO:K03424 TatD DNase family protein [EC:3.1.21.-]\n",
      "SSO SSO:000006642 Putative deoxyribonuclease YjjV\n",
      "b4378 ['Putative deoxyribonuclease YjjV']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009061 anaerobic respiration\n",
      "KO KO:K04069 pyruvate formate lyase activating enzyme [EC:1.97.1.4]\n",
      "SSO SSO:000028683 Uncharacterized radical SAM protein YjjW\n",
      "b4379 ['Uncharacterized radical SAM protein YjjW']\n",
      "\n",
      "SSO SSO:000027109 Uncharacterized glycyl radical protein YjjI\n",
      "b4380 ['Uncharacterized glycyl radical protein YjjI']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01619 deoxyribose-phosphate aldolase [EC:4.1.2.4]\n",
      "SSO SSO:000002255 Deoxyribose-phosphate aldolase (EC 4.1.2.4)\n",
      "b4381 ['Deoxyribose-phosphate aldolase (EC 4.1.2.4)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K00758 thymidine phosphorylase [EC:2.4.2.4]\n",
      "SSO SSO:000008066 Thymidine phosphorylase (EC 2.4.2.4)\n",
      "b4382 ['Thymidine phosphorylase (EC 2.4.2.4)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K01839 phosphopentomutase [EC:5.4.2.7]\n",
      "SSO SSO:000006028 Phosphopentomutase (EC 5.4.2.7)\n",
      "b4383 ['Phosphopentomutase (EC 5.4.2.7)']\n",
      "\n",
      "GO GO:0015949 nucleobase-containing small molecule interconversion\n",
      "KO KO:K03784 purine-nucleoside phosphorylase [EC:2.4.2.1]\n",
      "SSO SSO:000006580 Purine nucleoside phosphorylase (EC 2.4.2.1)\n",
      "b4384 ['Purine nucleoside phosphorylase (EC 2.4.2.1)']\n",
      "\n",
      "KO KO:K07154 serine/threonine-protein kinase HipA [EC:2.7.11.1]\n",
      "SSO SSO:000032823 putative protein kinase\n",
      "b4385 ['Putative kinase YjjJ']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006418 tRNA aminoacylation for protein translation\n",
      "GO GO:0042158 lipoprotein biosynthetic process\n",
      "KO KO:K03800 lipoate---protein ligase [EC:6.3.1.20]\n",
      "SSO SSO:000004465 Lipoate-protein ligase A\n",
      "b4386 ['Lipoate-protein ligase A']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07186 membrane protein\n",
      "SSO SSO:000022719 Protein Smp\n",
      "b4387 ['Protein Smp']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006545 glycine biosynthetic process\n",
      "GO GO:0006564 L-serine biosynthetic process\n",
      "KO KO:K01079 phosphoserine phosphatase [EC:3.1.3.3]\n",
      "SSO SSO:000006051 Phosphoserine phosphatase (EC 3.1.3.3)\n",
      "b4388 ['Phosphoserine phosphatase (EC 3.1.3.3)']\n",
      "\n",
      "KO KO:K04485 DNA repair protein RadA/Sms\n",
      "SSO SSO:000002107 DNA repair protein RadA\n",
      "SSO SSO:000012853 DNA repair protein\n",
      "b4389 ['DNA repair protein RadA']\n",
      "\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0009435 NAD biosynthetic process\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K06211 HTH-type transcriptional regulator, transcriptional repressor of NAD biosynthesis genes [EC:2.7.7.1 2.7.1.22]\n",
      "b4390 ['NadR transcriptional regulator', 'Nicotinamide-nucleotide adenylyltransferase, NadR family (EC 2.7.7.1)', 'Ribosylnicotinamide kinase (EC 2.7.1.22)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "KO KO:K06020 energy-dependent translational throttle protein EttA\n",
      "SSO SSO:000002535 Energy-dependent translational throttle protein EttA\n",
      "b4391 ['Energy-dependent translational throttle protein EttA']\n",
      "\n",
      "EC EC:3.2.1.14 Chitinase.\n",
      "EC EC:3.2.1.17 Lysozyme.\n",
      "GO GO:0009252 peptidoglycan biosynthetic process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0042597 periplasmic space\n",
      "KO KO:K08309 soluble lytic murein transglycosylase [EC:4.2.2.-]\n",
      "SSO SSO:000039197 Soluble lytic murein transglycosylase (EC 4.2.2.n1)\n",
      "b4392 ['Soluble lytic murein transglycosylase (EC 4.2.2.n1)']\n",
      "\n",
      "GO GO:0000162 tryptophan biosynthetic process\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K03720 TrpR family transcriptional regulator, trp operon repressor\n",
      "SSO SSO:000008281 Transcriptional repressor protein TrpR\n",
      "b4393 ['Transcriptional repressor protein TrpR']\n",
      "\n",
      "KO KO:K01529 Unknown\n",
      "SSO SSO:000003910 Inosine/xanthosine triphosphatase (EC 3.6.1.-)\n",
      "b4394 ['Inosine/xanthosine triphosphatase']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006094 gluconeogenesis\n",
      "GO GO:0006096 glycolytic process\n",
      "KO KO:K15634 probable phosphoglycerate mutase [EC:5.4.2.12]\n",
      "SSO SSO:000005972 Phosphoglycerate mutase (EC 5.4.2.1)\n",
      "SSO SSO:000021599 Phosphatase\n",
      "b4395 ['Phosphoglycerate mutase (EC 5.4.2.11)']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006261 DNA-dependent DNA replication\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K05804 AraC family transcriptional regulator, mar-sox-rob regulon activator\n",
      "SSO SSO:000002146 DNA-binding transcriptional dual regulator Rob\n",
      "b4396 ['DNA-binding transcriptional dual regulator Rob']\n",
      "\n",
      "KO KO:K05805 CreA protein\n",
      "SSO SSO:000001759 Conserved uncharacterized protein CreA\n",
      "SSO SSO:000023504 Putative periplasmic protein\n",
      "b4397 ['Conserved uncharacterized protein CreA']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "KO KO:K07663 two-component system, OmpR family, catabolic regulation response regulator CreB\n",
      "SSO SSO:000007021 Response regulator CreB of two-component signal transduction system CreBC\n",
      "b4398 ['Response regulator CreB of two-component signal transduction system CreBC']\n",
      "\n",
      "GO GO:0006464 cellular protein modification process\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K07641 two-component system, OmpR family, sensor histidine kinase CreC [EC:2.7.13.3]\n",
      "SSO SSO:000007411 Sensory histidine kinase CreC of two-component signal transduction system CreBC\n",
      "b4399 ['Sensory histidine kinase CreC of two-component signal transduction system CreBC']\n",
      "\n",
      "GO GO:0009274 peptidoglycan-based cell wall\n",
      "GO GO:0019866 organelle inner membrane\n",
      "KO KO:K06143 inner membrane protein\n",
      "SSO SSO:000003879 Inner membrane protein CreD\n",
      "SSO SSO:000036376 Inner membrane protein\n",
      "b4400 ['Inner membrane protein CreD']\n",
      "\n",
      "GO GO:0005737 cytoplasm\n",
      "GO GO:0006350 Unknown\n",
      "GO GO:0016563 obsolete transcription activator activity\n",
      "GO GO:0016564 obsolete transcription repressor activity\n",
      "KO KO:K07773 two-component system, OmpR family, aerobic respiration control protein ArcA\n",
      "SSO SSO:000034520 Aerobic respiration control response regulator ArcA\n",
      "b4401 ['Aerobic respiration control response regulator ArcA']\n",
      "\n",
      "SSO SSO:000028420 Uncharacterized protein YjjY\n",
      "b4402 ['Uncharacterized protein YjjY']\n",
      "\n",
      "GO GO:0009451 RNA modification\n",
      "KO KO:K02533 tRNA/rRNA methyltransferase [EC:2.1.1.-]\n",
      "SSO SSO:000028712 Uncharacterized tRNA/rRNA methyltransferase LasT\n",
      "b4403 ['Uncharacterized tRNA/rRNA methyltransferase LasT']\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for f in o['features']:\n",
    "    for term in f['ontology_terms']:\n",
    "        for k in f['ontology_terms'][term]:\n",
    "            print(term, k, o_present[term][k])\n",
    "    print(f['id'], f['functions'])\n",
    "    print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "a8b3241c-b571-42c3-84dc-c755a29bc192",
   "metadata": {},
   "outputs": [],
   "source": [
    "o_events = o['ontology_events']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "id": "3a17c329-e6cd-4012-a6fc-e26faf514b71",
   "metadata": {},
   "outputs": [],
   "source": [
    "o_present = o['ontologies_present']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ca12a45e-0f64-4795-b81b-5765df44257b",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "1e9910b0-6288-4534-a8eb-337e6fa0166d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'GO': {'GO:0009451': [0]},\n",
       " 'KO': {'KO:K02533': [2]},\n",
       " 'SSO': {'SSO:000028712': [5]}}"
      ]
     },
     "execution_count": 111,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f['ontology_terms']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5b7f10ae-a9d8-4b6d-afed-69814bc3c2d0",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
